{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "3e630a32-efb8-452b-a1fa-d28536b8d778",
   "metadata": {
    "id": "3e630a32-efb8-452b-a1fa-d28536b8d778"
   },
   "source": [
    "# dbGaP Reporter\n",
    "This notebook queries the database of Genotypes and Phenotypes [dbGaP](https://www.ncbi.nlm.nih.gov/gap/) for studies and reports access requests for datasets. The number of access requests per study is a measure of data reuse. In addition, this notebook queries [Europe PMC](https://europepmc.org/) for publications and preprints that cite or mention dbGaP accession numbers.\n",
    "\n",
    "Created: 2023-03-09\n",
    "\n",
    "Author : Peter W. Rose (pwrose@ucsd.edu)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "0eb4cad0-4bed-486e-87c6-e2dfe9faeccb",
   "metadata": {
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "id": "0eb4cad0-4bed-486e-87c6-e2dfe9faeccb",
    "outputId": "fdb1eeda-fdc2-4a35-d067-393dd8081161"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Query: \"COVID-19\"\n"
     ]
    }
   ],
   "source": [
    "#@title Enter a query term and then select **Run All** from the **Runtime** menu. For an exact match enclose query term in quotes. { run: \"auto\", vertical-output: true, form-width: \"50%\", display-mode: \"form\" }\n",
    "#@markdown ### Enter query term or dbGap accession number\n",
    "query = \"\\\"COVID-19\\\"\" #@param {type:\"string\"}\n",
    "print(f\"Query: {query}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "ddab1f36-cbd8-4d20-a993-642e5c9467c0",
   "metadata": {
    "cellView": "form",
    "id": "ddab1f36-cbd8-4d20-a993-642e5c9467c0",
    "tags": []
   },
   "outputs": [],
   "source": [
    "%%capture\n",
    "#@title Installing software on Google Colab\n",
    "![ ! -f \"installed\" ] && pip -q install selenium\n",
    "![ ! -f \"installed\" ] && apt-get update\n",
    "![ ! -f \"installed\" ] && apt-get install firefox && touch installed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "2991fd05-389b-4203-ba4b-1617ba4d7666",
   "metadata": {
    "cellView": "form",
    "id": "2991fd05-389b-4203-ba4b-1617ba4d7666",
    "tags": []
   },
   "outputs": [],
   "source": [
    "#@title Importing packages\n",
    "import os\n",
    "import shutil\n",
    "import glob\n",
    "import time\n",
    "from tqdm import tqdm\n",
    "import pandas as pd\n",
    "from selenium import webdriver\n",
    "from selenium.webdriver.common.by import By\n",
    "from selenium.webdriver.firefox.options import Options\n",
    "from google.colab import data_table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "131d482b-6ab3-411c-b7e0-fc42d42a4ec2",
   "metadata": {
    "cellView": "form",
    "colab": {
     "base_uri": "https://localhost:8080/"
    },
    "id": "131d482b-6ab3-411c-b7e0-fc42d42a4ec2",
    "outputId": "ff454a17-1267-404f-96a4-989f0575ae89",
    "tags": []
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 15/15 [00:15<00:00,  1.00s/it]\n"
     ]
    }
   ],
   "source": [
    "#@title Running query\n",
    "TMP_DIR = \"/tmp\"\n",
    "filepath = os.path.join(TMP_DIR, \"studies.csv\")\n",
    "\n",
    "def driversetup(download_dir):\n",
    "    options = Options()\n",
    "    #run Selenium in headless mode\n",
    "    options.add_argument(\"--headless\")\n",
    "    options.add_argument(\"--no-sandbox\")\n",
    "    # https://stackoverflow.com/questions/60170311/how-to-switch-download-directory-using-selenium-firefox-python\n",
    "    # 0: download to the desktop, 1 download to the default \"Downloads\" directory, 2 use specified directory\n",
    "    options.set_preference(\"browser.download.folderList\", 2)\n",
    "    options.set_preference(\"browser.download.manager.showWhenStarting\", False)\n",
    "    options.set_preference(\"browser.download.dir\", download_dir)\n",
    "    options.set_preference(\"browser.helperApps.neverAsk.saveToDisk\", \"text/csv\")\n",
    "    \n",
    "    # https://stackoverflow.com/questions/42204897/how-to-set-up-a-selenium-python-environment-for-firefox\n",
    "    driver = webdriver.Firefox(options=options)\n",
    "    driver.implicitly_wait(5)\n",
    "\n",
    "    return driver\n",
    "\n",
    "def download_dbgap_studies(query, filepath):\n",
    "    # clean up any previously downloaded csv files\n",
    "    files = glob.glob(os.path.join(TMP_DIR, \"*.csv\"))\n",
    "    for file in files:\n",
    "        os.remove(file)\n",
    "    \n",
    "    # download csv file\n",
    "    driver = driversetup(TMP_DIR)\n",
    "    driver.get(f\"https://www.ncbi.nlm.nih.gov/gap/advanced_search/?TERM={query}\")\n",
    "    time.sleep(3)\n",
    "    #print(\"Running: \", driver.title)\n",
    "    button = driver.find_element(By.CLASS_NAME, \"svr_container\")\n",
    "    time.sleep(3)\n",
    "    button.click()\n",
    "    # wait until download is completed\n",
    "    for step in tqdm(range(15)):\n",
    "        time.sleep(1)\n",
    "    #time.sleep(15)\n",
    "    driver.close()\n",
    "                  \n",
    "    # move downloaded csv file to a standard location\n",
    "    move_studies_file(filepath)\n",
    "    \n",
    "def move_studies_file(filepath):\n",
    "    \"\"\" Move downloaded file to a specified standard location\"\"\"\n",
    "    # the file name of the downloaded csv file is unknown in advance,\n",
    "    # but there should be only one csv file.\n",
    "    files = glob.glob(os.path.join(TMP_DIR, \"*.csv\"))\n",
    "    if len(files) == 1:\n",
    "        shutil.move(files[0], filepath)\n",
    "    else:\n",
    "        print(\"query error\")\n",
    "        \n",
    "filepath = \"studies.csv\"\n",
    "download_dbgap_studies(query, filepath)\n",
    "\n",
    "studies = pd.read_csv(filepath, usecols=[\"accession\", \"name\", \"description\", \"Study Design\", \"Study Consent\",])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "82a725e0-c840-4854-a927-69b07836258b",
   "metadata": {
    "cellView": "form",
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 695
    },
    "id": "82a725e0-c840-4854-a927-69b07836258b",
    "outputId": "8518cce4-efe8-4bcd-f149-a1c76ca903bc",
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of studies for \"COVID-19\": 175\n"
     ]
    },
    {
     "data": {
      "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/99dac6621f6ae8c4/data_table.js\";\n\n      window.createDataTable({\n        data: [[\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\"],\n [\"phs002245.v1.p1\",\n\"Genetic Determinants of Susceptibility to Severe COVID-19 Infection\",\n\"The goal of the project is to identify genetic susceptibility variants associated with acute COVID-19 infection susceptibility and severity using whole genome sequencing. The target of the project will include  WGS data from whole blood samples of up to approximately 2000 COVID-19 patients (including critical/severe patient subgroup, asymptomatic/mild\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002258.v1.p1\",\n\"Shotgun Transcriptome and Isothermal Profiling of SARS-CoV-2 Infection    Reveals Unique Host Responses, Viral Diversification, and Drug    Interactions\",\n\"In less than nine months, the Severe Acute Respiratory Syndrome Coronavirus 2 (SARS-CoV-2) has killed hundreds of thousands of people, including 23,000 in New York City (NYC) alone. The pandemic's   COVID-19 disease. No inclusion/exclusion criteria in the IRB. Principal Investigator: Christopher E. Mason, Weill Cornell Medicine, New York, NY, USA\",\n\"Metagenomics\",\n\"GRU-IRB-PUB-COL --- General research use (irb, pub, col)\"],\n [\"phs002299.v1.p1\",\n\"PETAL Network: Outcomes Related to COVID-19 Treated With    Hydroxychloroquine Among Inpatients With Symptomatic Disease (ORCHID)    Trial\",\n\"ORCHID was a multicenter, blinded, placebo-controlled randomized trial conducted at 34 hospitals in the US between April 2 and June 19, 2020. Adults hospitalized with respiratory symptoms from severe acute  respiratory syndrome coronavirus 2 infection were enrolled, with the last outcome assessment on July 17, 2020. The planned sample size was 510\",\n\"Clinical Trial\",\n\"HMB --- Health/medical/biomedical\"],\n [\"phs002300.v1.p1\",\n\"Successful Clinical Response in Pneumonia Therapy (SCRIPT)\",\n\"The goal of this study is to iteratively identify, validate and refine biomarkers to predict clinical failure, and identify novel targets for therapy, beyond traditional antibiotics, for patients with severe  pneumonia requiring intubation, including severe community-acquired pneumonia (CAP), hospital-acquired pneumonia, and ventilator-associated pneumonia\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-IRB --- Health/medical/biomedical (irb)\"],\n [\"phs002363.v1.p1\",\n\"PETAL Repository of Electronic Data COVID-19 Observational Study (RED    CORAL)\",\n\"To describe characteristics, treatment, and outcomes among patients hospitalized with COVID-19 early in the pandemic, 1480 consecutive adult inpatients with laboratory-confirmed, symptomatic SARS-CoV-2 infection admitted to 57 US hospitals from  March 1 to April 1, 2020 were studied. It was found that in a geographically diverse early-pandemic COVID-19 cohort with complete hospital folllow-up\",\n\"Control Set\",\n\"HMB --- Health/medical/biomedical\"],\n [\"phs002433.v1.p1\",\n\"Nasopharyngeal RNASeq Comparing SARS-CoV-2+ Patients and SARS-CoV-2    Negative Control Subjects\",\n\"The goal of this analysis was to compare RNAs enriched in the upper respiratory tract of SARS-CoV-2 patients and SARS-CoV-2 negative controls. RNA-sequencing of RNA isolated from the viral transport  medium associated with nasopharyngeal swab samples from SARS-CoV-2+ patients and SARS-COV-2 negative controls. Nasopharyngeal (NP) swab samples were\",\n\"Collection\",\n\"HMB-IRB --- Health/medical/biomedical (irb)\"],\n [\"phs002516.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: IBM Covid19    Contact Tracing and Data Exchange Tools\",\n\"The goal of this project is to develop both contact tracing and secure data exchange tools. The contact tracing solution securely combines data from a variety of sources (including manual  self-report data and mobile device surveys) to enable tracing of contacts with individuals that have tested positive for COVID-19 or have been\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002519.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Covidseeker and COVID-19 Citizen Science: Leveraging    Citizen Science and Real-Time Geospatial Temporal Mobile Data for    Digital Contact Tracing and SARS-CoV-2 Hotspotting\",\n\"The Covidseeker and COVID-19 Citizen Science Study integrates a retrospectively-determined geolocation digital program into an established digital infrastructure housed within the NIH-funded Eureka platform to enroll SARS-CoV-2 positive and negative  individuals. The solution also leverages the ongoing COVID-19 Citizen Science Study cohort both in-person and remotely to test and implement existing\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002520.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Community Based COVID-19 Testing Evaluation\",\n\"We have used a community-based participatory research approach guided by the Social Ecological Model to target testing strategies that will have the greatest impact for the Latinx and Marshallese communities  in Northwest Arkansas. Our specific aims are: Aim 1: leverage and fully engage our long-standing community-based partnerships to increase COVID-19\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002521.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Creating a Sustainable Infrastructure for SARS-CoV-2 Testing at    Syringe Exchange Programs\",\n\"This study leverages a current partnership with HIV Alliance - an Oregon-based nonprofit dedicated to supporting individuals living with HIV/AIDS and preventing new HIV infections - to embed a COVID-19   as well as vaccination, when available, for people who inject drugs. This project has the potential to enhance COVID-19 testing access and reach for a\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002522.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): MOF-SCENT:    Metal-Organic Frameworks for Screening COVID-19 by Electronic-Nose    Technology to Improve Selectivity and Time Response\",\n\"This proposal describes the design, fabrication, and proof-of-concept testing of a novel MOF-SCENT system (Metal-Organic Frameworks for Screening COVID-19 by Electronic-Nose Technology) that features the following advantages: (1) Highly selective   dramatically increase testing capacity for COVID-19 because it can be rapidly integrated into and adopted by the public and healthcare systems by deploying a\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002523.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Multi-Modal    Wireless COVID Monitoring and Infection Alerts for Concentrated    Populations\",\n\"We plan to extend and improve our COH/PHD system in using wearables for the early detection of COVID-19 infection. With the over 5,000 patients who have been recruited in our   backgrounds (Aim 1). We will further integrate and optimize MyPHD and COH to enable real-time online detection of COVID-19 (Aim 2). We will deploy MyPHD/COH on\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002524.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Validation of    Smart Masks for Surveillance of COVID-19\",\n\"Vulnerable populations do not just need testing - they need surveillance. The ideal surveillance tool would operate in the background with minimal involvement of the population to be tested; it   when the wearer has been exposed to biomarkers (proteases) of COVID-19. This is a novel approach to surveillance because most people are already wearing\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002525.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): SF-RAD:    Development and Proof-of-Concept Implementation of the South Florida    Miami RADx-rad SARS-CoV-2 Wastewater-Based Surveillance Infrastructure\",\n\"The University of Miami (UM), with three primary campuses in Miami, Florida, is geographically spread within one of the worst current COVID-19 hotbeds. UM has deployed an elaborate human surveillance   UM and that treats COVID-19 patients. To augment this COVID-19 monitoring system, UM has deployed a pilot wastewater surveillance program for detecting\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002526.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Community Driven Approach to Mitigate COVID-19 Disparities in Hawaii's    Vulnerable Populations\",\n\"The long-term goal of this proposal is to eliminate COVID-19 deiparities among Hawaii's racial/ethnically diverse yet vulnerable populations including Native Hawaiian and other Pacific Islanders (NHPI) living in rural and   long-standing health disparities and socioeconomic challenges. NHPIs suffer from increased infection and mortality rates attributed to COVID-19. To date\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002527.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Bioinformatics    Framework for Wastewater-based Surveillance of Infectious Diseases\",\n\"COVID-19 is expected to become one of the largest mass casualty events in the history of the United States (U.S.). Assessment of the true burden of disease in the population   communication originally developed for our parent award to quickly provide a data stream and scientific resource for managing the COVID-19 epidemic in the U.S. In\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002533.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Digital Health Solutions for COVID-19: COVID Community    Action and Research Engagement (COVID-CARE)\",\n\"Vibrent Health will expand the Vibrent Digital Health Solutions Platform (DHSP) implementation to additional populations among diverse user groups for additional validation of the technology's performance, usability, and reliability in   decision-making in the COVID-19 pandemic response. This includes the aims to innovate a technology that can differentiate between COVID-19 and flu (or other\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002534.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: Team SAE\",\n\"The goal of this project is to develop a smartphone-based platform to monitor and support individuals with COVID-19 symptoms (who may need testing) and those who have already tested positive.   consisting of vital data, medical history and questionnaire data, and participant COVID-19 viral and antibody testing results collected during the study. DOI\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002535.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Personalized Analytics and Wearable Biosensor Platform for    Early Detection of Covid-19 Decompensation (DECODE)\",\n\"The goal of this project is to develop an artificial intelligence-based data analytics and cloud computing platform, paired with U.S. Food and Drug Administration (FDA)-cleared wearable devices, to create a  personalized baseline index that could indicate a change in health status for patients who have tested COVID-19 positive. The project involves the\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002537.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): COVID-19 Experience Study (C19EX) Survey\",\n\"This was conducted virtually through the Achievement studies platform during the current COVID-19 pandemic. Participants were asked to complete a survey every day to capture information about whether they had  experienced symptoms corresponding to respiratory, ILI, and COVID-19 infections within the previous 24 hour period, their experiences with the\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002538.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): ILI Labels and Longitudinal Novel Engagement with Symptom    Surveillance (ILLNESS) Study\",\n\"This study is a prospective observational study, approximately seven months in duration. Participants were asked to complete a weekly survey online asking about their ILI (influenza-like illness) experience over the  past seven days. The primary objective of this study was to collect a population-wide, longitudinal dataset of ILI events (with onset and recovery\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002539.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Large Scale Flu Surveillance Study (LSFS)\",\n\"The purpose of this study was to better understand behavioral and physiological functioning in relation to recent self-reported influenza and influenza-like-illness (ILI), including coronavirus disease (COVID-19). Over 65,000 Achievement members  responded to a weekly one-click survey asking if they had experienced ILI within the previous 7-day period. If they responded no, they were given the\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002540.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: SAFER-COVID -    Integration of Testing and Digital Health\",\n\"SAFER-COVID provides a set of self-management tools to consumers to track symptoms, test results, vaccine record, and environmental factors, such as exposure to others. Consumers may choose to integrate data  from wearables and electronic health records (EHR) for self management. These data are used to enable multiple use cases such as return-to-work\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002542.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Wastewater    Assessment for Coronavirus in Kentucky: Implementing Enhanced    Surveillance Technology\",\n\"Developing next generation technology to simplify wastewater RNA extraction and quantification will make wastewater surveillance feasible and scalable in facilities and rural communities. Wastewater testing for SARS-CoV-2 in rural communities  using field-friendly technology will provide important information to local authorities and citizens about the spread and trend of SARS-CoV-2\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002543.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Minimal    False-Alarm Touch-Based Detection of SARS-CoV-2 Virus Particles Using    Poly-Aptamers\",\n\"Reliable real-time detection of SARS-CoV-2 virus represents a considerable challenge, with available tools requiring extensive sample preparation and/or expensive lab-based equipment to obtain accurate results. The objective in this proposal  is to build a touch-screen sensor array to directly capture, detect, and identify model SARS-CoV-2 virus particles with minimal false alarms. This\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002544.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): AFS/SERS    Saliva-Based SARS-CoV-2 Earliest Infection and Antibodies Detection\",\n\"This U18 application is responsive to the NIH's RADx-rad Emergency Response to the COVID-pandemic for new or non-traditional technologies developed to single extracellular vesicle, exosome and extracellular RNA (exRNA) isolation  and analysis and reposition them for detection of SARS-CoV-2. The applicant's group is a grantee in the NIH Common Fund \\\"Extraceullar RNA\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002546.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Development of an    Automated Diagnostic Platform for SARS-CoV-2 Monitoring in Vulnerable    Areas\",\n\"The devastation caused by emerging pathogens with fast transmission capacity, such as SARS-CoV-2, has demonstrated the importance of preparedness for future viral outbreaks; this includes the ability for fast deployment  of in-situ testing tools and epidemiological surveillance with high temporal and spatial resolution; particularly in places that are most vulnerable\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002549.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Severity    Predictors Using Immunology and Transcriptomics in Saliva Using Multi    Neural Network Intelligence in SARS-CoV2 Infection in Children (SPITS    MISC)\",\n\"Children have been disproportionately less impacted by the Coronavirus Disease 2019 (COVID-19) caused by the Severe Acute Respiratory Syndrome Coronavirus 2 (SAR-CoV-2) compared to adults. However, severe illnesses including Multisystem  Inflammatory Syndrome (MIS-C) and respiratory failure have occurred in a small proportion of children with SARS-CoV-2 infection. Nearly 80% of\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002550.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Direct    Bioelectronic Detection of SARS-CoV-2 From Saliva Using Singlemolecule    Field-effect Transistor Array\",\n\"Nucleic acid test-standard for diagnostics testing for COVID-19, usually performed in specialized laboratories. Most are based on reverse-transcription quantitative polymerase chain reaction (qRT-PCR). The time required for specimen transport and  processing results in a turnaround time that is typically several days. The few rapid ( 1hour) point-of-care (POC) tests are more expensive, still\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002551.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Marshallese:    Alternate Surveillance for COVID-19 in a Unique Population\",\n\"Marshallese Pacific Islanders bear a disproportionate burden of COVID-19 infection, hospitalization, and death, with rates 4 to 25 times higher than those of other US racial and ethnic groups in   of 5 COVID-19 cases in this area. Similarly, Marshallese represent just 1% of the population in Spokane County, Washington, but were nearly 30% of\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002553.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Characterization    of Multisystem Inflammatory Syndrome in Children and its Relationship    to Kawasaki Disease\",\n\"As the COVID-19 pandemic evolved in early 2020, case reports appeared describing children with unusual febrile illnesses with elevated inflammatory markers and multi-system involvement that is now termed Multisystem Inflammatory  , protein and antibody datasets from children with COVID-19, MIS-C, and with other febrile illnesses. Next, we will use these data to devise tests to\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002561.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Rapid,    Sensitive, Point-of-Care, Antigen-Based Diagnostics for SARS-CoV-2\",\n\"A rapid, sensitive, low-cost diagnostic device for early detection of novel Coronavirus Disease (COVID-19) could significantly change the paradigm in real-time and efficient management of patients and available healthcare resources,   most important method for the diagnosis of COVID-19 at its early stage of infection and before the appearance of clinical symptoms. However, qPCR-based\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002563.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Handheld    Microchip for GC Analysis of Breath to Screen for COVID-19\",\n\"The COVID-19 pandemic has caused unprecedented societal suffering and economic disruption. Although current COVID-19 diagnostic testing technologies are critical for curbing the spread of the virus and preventing future outbreaks,   provide test results. The overall goal of the project is to develop a mobile breath analysis technology for rapid screening for COVID-19 using a handheld\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002565.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad):    Touchscreen-Compatible, Real-Time Electrochemical Sensing of    SARS-CoV-2\",\n\"We are developing an electrochemical assay for detecting SARS-CoV-2. We will be generating data that includes a signal for a given virus or spike (S) protein concentration, as well as  binding constants for molecular recognition agents. Note for data in RADx: Instructions for requesting individual-level data are available on RADx\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002567.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP): NJ    HEROES TOO: New Jersey Healthcare Essential Worker Outreach and    Education Study Testing Overlooked Occupations\",\n\"The COVID-19 pandemic disproportionately impacts members of under-represented minority (URM) communities. Unfortunately, URM and other vulnerable communities remain profoundly under-tested. As such, new approaches are needed to disseminate and accomplish  , 2nd in the US (after NY) in the per capita rate of COVID-19 deaths, and 5th in total cases and represents a highly diverse state with substantial Black\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002568.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Reaching Communities through the Design of Information Visualizations    (ReDIVis) Toolbox for Return of COVID-19 Results (RCR)\",\n\"The overall goal of Reaching Communities through the Design of Information Visualizations (ReDIVis) Toolbox for Return of COVID-19 Results (RCR) is to decrease health disparities related to COVID-19 by enabling   and used; specifically, how the results influence decision making and COVID-19 mitigation behaviors (e.g., social distancing, self-isolation, mask\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002569.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): COVID-19 Network    of Networks Expanding Clinical and Translational Approaches to Predict    Severe Illness in Children (CONNECT to Predict SIck Children)\",\n\"This study's objective is to understand which children are at the highest risk for experiencing severe consequences from SARS-CoV-2 infection. The study will use clinical, epidemiologic, and sociodemographic data, alongside  specific biomarkers, and create machine-learning models designed to predict severe consequences. In order to accomplish this, we will establish a\",\n\"Case-Control\",\n\"GRU --- General research use\"],\n [\"phs002570.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Multimodal    Platform for Oral Screening of COVID-19\",\n\"This project will develop a multiplexed sensor platform based on nanomaterials capable of rapid simultaneous detection of multiple components of viral particles in a field-applicable electrochemical device. The device has  synthetic biosensors that, in a multimodal, rapid, accurate, inexpensive, field-deployable test, will detect multiple SARS-CoV-2 biomarkers. Note for\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002572.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A SARS-CoV-2    Breathalyzer for Direct Virus Detection\",\n\"A substantial and growing body of evidence has demonstrated that COVID-19 is transmitted by human-emitted airborne particles; therefore, it is critical to rapidly screen individuals to determine whether they are   bring the current COVID-19 pandemic under control before this SARS-CoV-2 breathalyzer is commercially available. Note for data in RADx: Instructions for\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002573.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Detection and    Automatic Privacy-Protected Contact Tracing System Designed for    COVID-19\",\n\"The goal of the project is to develop a pipeline for automatic detection and tracing of active SARS-CoV-2 infection in a cost-effective, highly accurate, real-time, portable, and super-encrypted manner. Our  integrated approach first captures viral particles from patient samples to activate fluorescence reporters, followed by signal detection using a\",\n\"Case-Control\",\n\"GRU --- General research use\"],\n [\"phs002575.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    SCALE UP Utah: A Community-Academic Partnership to Address COVID-19    Among Utah Community Health Centers\",\n\"Racial/ethnic minority, low socioeconomic status (SES), and rural populations suffer profound health inequities across a wide variety of diseases and conditions, as well as a disproportionate burden of the negative  health consequences of the COVID-19 pandemic. Latinos make up 14% of the Utah population vs. 40% of Utah's COVID-19 cases, and the case rate is over\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002577.v1.p1\",\n\"The Collaboration to Assess Risk and Identify LoNG-term outcomes for    Children with COVID (CARING for Children with COVID) -    NICHD-2019-POP02: Pharmacokinetics, Pharmacodynamics, and Safety    Profile of Understudied Drugs Administered to Children per Standard of    Care\",\n\"The goals of the NIH's Collaboration to Assess Risk and Identify LoNG-term outcomes for Children with COVID (CARING for Children with COVID) effort are to foster collaborative and accelerated pediatric  COVID research to understand the spectrum of pediatric SARS-CoV-2 infection and pediatric COVID-19, and identify MIS-C risk factors and long term\",\n\"Case Set\",\nNaN],\n [\"phs002578.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx - UP):    COVID-19 Testing and Linkage to Care with African American Church and    Health Agency Partners\",\n\"This 2-arm clustered, randomized community trial will test a multi-level, COVID-19 testing and linkage to care (eg, health insurance, medical appointments, community resources, contact tracing) intervention against a non-tailored, attention-control  condition on uptake of COVID-19 testing with adult African American church-affiliated members at 6 months. Contact tracing approval (beliefs and\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002581.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Adapting Community-Based Task-Shifting for the COVID-19 Response Among    Underserved Populations in Piedmont, North Carolina (ACT UP)\",\n\"In North Carolina and nationally, Black and Hispanic/Latinx communities have been disproportionately affected by COVID-19 due to multi-level risk factors and barriers to testing, prevention, and care, but have not   with COVID-19 testing and linkage to care, vaccination, and other preventative measures in rural central North Carolina, and to evaluate the effects of\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002582.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Understanding the COVID-19, Racism, and Violence Syndemic and its    Effects on COVID-19 Testing Disparities\",\n\"Black communities in the US are experiencing three converging public health crises: COVID-19, systemic racism, and community and police violence. Black Americans are significantly less likely to receive a needed  COVID-19 test and have higher rates of COVID-19 infection and death than white Americans. Additionally, Black communities have experienced a surge in\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002583.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Rapid    Breathalyzer Diagnostics Platform for COVID-19\",\n\"We propose to develop a novel testing platform that detects SARS-CoV-2 virions in a patient's breath. When a person exhales into the COVID breathalyzer, droplets and other emitted particles are   and challenges, not only as the virus causing COVID-19 evolves in future years, but also to provide multiplexing capabilities to detect multiple viruses\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002584.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Scaling Up SARS-COV-2 Testing to Serve Latinx Communities\",\n\"The global SARS-CoV-2 pandemic that causes the severe respiratory illness COVID-19 is the worst health crisis the United States has faced in a century. Although this highly contagious virus has   communities. Nationally, Latinx people with COVID-19 are hospitalized at four times the rate of Whites and have much higher rates of morbidity and mortality\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002585.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): AICORE-kids\",\n\"This work is directed at characterizing pediatric COVID-19 and stratifying incoming patients by projected (future) disease severity. Such stratification has several implications: immediately improving treatment planning, and as disease mechanistic  pathways are uncovered, directing treatment. Predicting future severity will inform the risks of outpatient treatment - to the patients themselves\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002588.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Effective,    Reagent-Free Detection of the Odor Signature of COVID-19 Infection    Using a Nano-Enabled Sensor Array\",\n\"COVID-19 presents a public health emergency. There is a critical need for rapid, not reagent intensive, non-invasive testing technologies. This program will lead to the production of a prototype system  to diagnose COVID-19 infection using the body odor signature of the disease. Our goal is to maximize societal impact by creating a validated\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002600.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Portable GC    Detector for COVID Diagnostics\",\n\"The data herein combines GC-MS and GC-DMS analysis of exhaled breath vapor compounds. The intent of this study is to develop a portable GC-DMS system to diagnose SARS-CoV-2 infections from  a breath sample. The data was collected from hundreds of subjects in an IRB-sanctioned study. Breath vapor samples were collected from COVID-19\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002601.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Community Network-Driven COVID-19 Testing of Vulnerable Populations in    the Central US (C3)\",\n\"Community network-driven COVID-19 testing of vulnerable populations in the Central US (C3) will implement and evaluate a COVID-19 testing approach that combines an evidence-based Social Network Testing Strategy (SNS) with  community developed COVID-19 public health messages (SNS+). C3 will engage two disenfranchised populations across rural and urban sites in states\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002602.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Virus Counter:    Rapid and Sensitive Diagnostics Based on Digital Detection of    Individual Pathogens\",\n\"To help diversify the approaches that can be used to sensitively detect SARS-CoV-2 and improve response resilience, we will pivot an established virus detection system that has already shown the   diagnostic can be performed in the presence of influenza. A patient with flu-like symptoms can be diagnosed with flu or COVID-19, or both as feared that these\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002603.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Diagnosis of    MIS-C in Febrile Children\",\n\"The recent emergence of SARS-CoV2 and resultant pandemic of COVID-19 disease has overwhelmed global health systems and led to over 200,000 American deaths to date. While initial reports suggested that  SARS-CoV-2 infection in children was generally being, a novel post-inflammatory syndrome known as multi system inflammatory syndrome in children (MIS\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002604.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Tracking the    COVID-19 Epidemic in Sewage (TRACES)\",\n\"Wastewater based testing (WBT) holds great promise for cost-effective population surveillance and transmission tracking of SARS-CoV-2, but optimal sampling modalities and protocols are unknown. Taking advantage of a diverse inner  city urban campus encompassing undergraduate, postgraduate dorms, research buildings and medical facilities, this project includes optimization of\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002608.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Increasing Representation of Black Communities in SARS-CoV-2    Serosurveys by Understanding Barriers and Motivations for    Participation (Represent ATL)\",\n\"Racial minorities have disproportionate risk for SARS-CoV-2 diagnoses and adverse outcomes including death. Burden of disease in Black populations is likely underestimated due to sub-optimal access to, and usage of,  SARS-CoV-2 testing. Serosurveys, which use probability-based methods to select persons for SARS-CoV-2 testing and an accompanying survey, have\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002609.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Identifying    Biomarker Signatures of Prognostic Value for Multisystem Inflammatory    Syndrome in Children (MIS-C)\",\n\"The purpose of this study is to develop a tool that can diagnose multisystem inflammatory syndrome in children (MIS-C.) Some children and young adults who have had Covid-19 develop this  , skin, eyes, and intestines. MIS-C has symptoms similar to several diseases, including Covid-19 and Kawasaki disease, making diagnosis difficult\",\n\"Case-Control\",\n\"GRU --- General research use\"],\n [\"phs002616.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Bridging the Evidence-to-Practice Gap: Evaluating Practice    Facilitation as a Strategy to Accelerate Translation of a    Systems-Level Adherence Intervention into Safety Net Practices/Project    Evidence\",\n\"The COVID-19 pandemic is a major public health crisis, revealing deep and persistent racial/ethnic disparities in health. In New York City (NYC), one of the most affected cities in the  U.S., Black and Latinx communities suffered the highest rates of hospitalizations and mortality from COVID-19. Age-adjusted death rates for Blacks\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs002618.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Rapid Acceleration of Acceptable COVID Testing and Care Options for    NYC Public Housing Residents (RADx NYCHA)\",\n\"The Rapid Acceleration of Diagnostics for Underserved Populations (RADx-UP) initiative supports supplements to individual NIH awards to increase COVID-19 testing among underserved populations. Leveraging the infrastructure of a current project   Grossman School of Medicine, and community partnerships, this proposal aims to execute community-engaged investigations to improve COVID-19 testing for NYCHA\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002619.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Improving the Response of Local Urban and Rural Communities to    Disparities in Covid-19 Testing\",\n\"Local health departments are employing widely varying approaches to address the dramatic health inequities associated with the COVID-19 pandemic. Learning collaboratives have been effective in other settings for bringing entities  together to support joint problem solving and reduce variance in approach. These collaboratives build upon shared experiences to help their members\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002628.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: Clear2Go - A    Digital Identity Wallet for Health Status\",\n\"Clear2Go is a solution/app that provides digital, non-refutable cryptographic proof of testing or vaccination that can be used to evaluate risk of allowing individuals to return to normal work, travel,   it with their COVID-19 test and vaccination records. The person can then prove their status by utilizing the credentials with a QRCode or NFC tap. DOI\",\n\"Clinical Trial\",\n\"GRU --- General research use\"],\n [\"phs002631.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Rapid Olfactory    Tools for Telemedicine-Friendly COVID-19 Screening and Survey\",\n\"The COVID-19 pandemic is the most devastating infectious disease outbreak in a century, particularly in underserved and minoritized communities. In 2020 alone, it costed a million lives. It continues to   COVID-19. Rapid screening tools can identify potentially infected individuals who can then be isolated/quarantined from the uninfected and directed\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002633.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Transforming Community Engagement to Increase SARS-CoV-2 Testing in    Underserved Populations in Baton Rouge (TEST UP-BR)\",\n\"In the US, COVID-19 has unveiled a disproportionate health burden in low income and underserved segments of society. In Louisiana, some of the greatest health and economic consequences are evident   research in our region; it is uniquely positioned to lead a community-engaged testing research project to determine differences in COVID-19 testing rates\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002634.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    COVID-19 Treatment Cascade Optimization Study\",\n\"This study tests an adaptation of the HIV Continuum of Prevention, Care, and Treatment Framework (CoPCT) for use in tracking COVID-19 testing and follow-up in a medically and socially vulnerable   study will use the Multiphase Optimization Strategy (MOST) to optimize an adaptive intervention that successfully navigates COVID-19 medically or socially\",\n\"Clinical Trial\",\n\"GRU --- General research use\"],\n [\"phs002642.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A    Confectionary-Based Screening Tool for Assessing Chemosensory Loss in    COVID-19 Patients\",\n\"The goals of this Emergency Competitive Revision are to develop (Aim 1) and deploy (Aim 2) a novel, objective, psychophysical smell and taste screening test to detect the onset of  COVID-19 in at-risk populations. Complementary to the temperature screening procedures currently in use, this tool is predicted to promote early\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002643.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    COVID-19 Large Network Center\",\n\"Ohio includes a very diverse population, with many minority, underserved, and vulnerable communities (MUVP). The need to increase testing in Ohio's MUVP is great - testing rates are only at   intervene to reduce disparities in COVID-19 education, testing, contact tracing, follow-up and treatment among MUVP in Ohio in a bi-directional academic-state\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002644.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Randomized COVID-19 Testing in Vulnerable Communities and Risk Tool    Creation (FightCOVIDMKE)\",\n\"Fight COVID MKE will use randomized COVID antibody testing in Milwaukee County, Wisconsin to measure the population proportion infected and how this proportion varies over time and with patient demographic  and health characteristics. Through partnerships with 20 primary care health centers and with local churches and community agencies to conduct\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002645.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Alive Church Network: Increasing COVID-19 Testing in Chicago's African    American Testing Deserts\",\n\"The epidemic of novel coronavirus disease 2019 (COVID-19) has caused an unprecedented public health crisis in the United States. African Americans (AA) have been disproportionately impacted, as systemic inequities have  contributed to increased exposure and vulnerability to COVID-19. Evidence suggests that AAs are delaying testing and care for COVID-19, which\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002647.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Supporting COVID-19 Prevention and Testing for Marginalized and    Minoritized Youth and Young Adults\",\n\"This project will use a geographically diverse quantitative survey to assess patterns and disparities in COVID-19 testing and preventive behaviors longitudinally across 6 months, as well as survey the impact   acceptability. Given the lack of messaging guidelines for this population, this will be the first evidence-based messaging intervention for COVID-19 prevention\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002648.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Keeping Rural Minority 'Essential' Workplaces Open Safely During the    COVID-19 Pandemic: The Role of Frequent Point-of-Care Molecular    Workplace Surveillance for Miners (The Miners' Pandemic Project)\",\n\"There are limited studies on the spread of SARS-CoV-2 infection in rural essential workers. This gap in knowledge must be addressed to develop and implement novel pandemic strategies to keep   population of high-risk, rural, essential workers who are susceptible and vulnerable to COVID-19, and who are predominantly racial/ethnic minorities in New\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002649.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Communities Fighting COVID-19\",\n\"Given observed disparities in COVID-19 morbidity, hospitalization, and mortality across race, ethnicity, and socioeconomic status, there is a great need to increase testing access and uptake with rapid return of  test results. We propose a community health worker (CHW)-led approach to facilitate COVID-19 testing for underserved populations, with a focus on\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs002650.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Building Resiliency and Vital Equity (BRAVE)\",\n\"This project will collaboratively partner with American Indian populations to establish COVID-19 services and resources for American Indian communities. The Building Resilience and Vital Equity (BRAVE) project seeks to partner  with American Indian tribal communities in North Carolina to establish COVID-19 services and resources for American Indian communities. The goal of\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002651.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Community-Engaged Research on COVID-19 Testing Among Underserved    and/or Vulnerable Populations\",\n\"The SARS-CoV-2 virus, responsible for the COVID-19 pandemic, has caused disastrous and unprecedented public health and economic consequences in the U.S., seriously affecting Americans' physical and mental health. Death rates  attributable to COVID-19 among minority populations are several-fold higher than among predominantly White counties. South Florida, and specifically\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002656.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Eliminating COVID-19 Disparities in Arizona in Partnership with    Vulnerable/Underserved Communities\",\n\"Arizona has one of the highest COVID-19 positivity test rates (approximately 19%) in the U.S. Positivity rates are disproportionately higher among Arizona's Latinx, American Indian, and African American communities. The   in vulnerable and underserved communities across Arizona. The intervention aims at reducing disparities in COVID-19 diagnostics, education, wraparound\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002657.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Early Detection,    Containment, and Management of COVID-19 in Dialysis Facilities Using    Multi-Modal Data Sources\",\n\"With older age and multiple comorbidities, dialysis patients are at high risk for serious complications, even death, from COVID-19. There is a large disproportionate representation of minorities, especially Blacks and   shelter in place. There is a critical need to characterize COVID-19 transmission pathways in dialysis patients and clinics, identify potential coronavirus\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002658.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    LITE CONNECT: Addressing Testing Gaps and Epidemiologic Disparities of    COVID-19 Among Transgender People in the United States\",\n\"Transgender (TG) people are an NIH-designated health disparities population with high morbidity and mortality across multiple health conditions, including HIV infection, mental health, and substance use. These conditions are a   Syndrome Coronavirus 2 (SARS-CoV-2) and Coronavirus Disease 2019 (COVID-19) was recognized as a global pandemic. To date, no data exists on COVID-19 disease\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002660.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Juntos (Together): A Community-Led Approach to Enhance COVID-19    Testing Among Vulnerable Latinos\",\n\"Latinos are among the most heavily impacted communities by the COVID-19 pandemic in the U.S., with rates more than 3 times higher than among non-Hispanic whites. To address this disparity,  , immigration status, stigmatization or loss of job/income), and to expand access to free COVID-19 testing in community settings. Our preliminary findings show\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002661.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Back to ECE Safely with SAGE: Reducing COVID-19 Transmission in    Hispanic and Low-Income Preschoolers\",\n\"The overarching goal of this study is to determine the acceptability, feasibility, efficacy, and scalability of back-to-early childhood education (ECE) testing in young children (3-5 years). We will also determine  whether outdoor garden-based education can help close the education gap experienced by children who have been out of school during the pandemic. The\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002662.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Developing Novel Strategies to Increase COVID-19 Testing Among    Underserved and Vulnerable Populations in West Virginia through    Community and State Partnerships\",\n\"West Virginia (WV) is rated by Becker's Hospital Review as the state with the sixth most vulnerable population to the novel coronavirus (SARS CoV-2) while a second group at Wallethub,  %). Through school and university closures in March 2020, along with shutdown of non-essential businesses, WV had relatively few COVID-19 cases and deaths until\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs002679.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Wastewater    Detection of COVID-19\",\n\"When faced with a pandemic such as SARS-Coronavirus-2 (SAR-CoV-2), the virus responsible for COVID-19, timely risk assessment and action are required to prevent public health impacts to entire communities. Because   sewersheds for genetic indicators of COVID-19 in wastewater and to provide additional, population-level information about virus circulation that is not\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002682.v1.p1\",\n\"Rapid Acceleration of Diagnostics - TECH (RADx-TECH): 2183 SARS-CoV-2    Rapid Antigen Test\",\n\"There are two device studies under the 2183 banner. The 2183b study uses an enhanced visual display compared to the original study (2183a), so it has been renamed. These prospective  clinical studies seek to examine the performance of the 2183 device (a and b), a lateral flow immunoassay for the point-of-care (POC) detection of\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002685.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): DNA Star    SAS-CoV-2 Rapid Test\",\n\"Automated, rapid diagnostics with little sample collection and preparation are needed to identify and trace affected persons in times when hyper-infectious pathogens cause pandemics. Frequent, low cost and highly scalable  testing is the only way to gain visibility on the magnitude of the pandemic and ultimately control the spread of the disease. We propose the\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002686.v1.p1\",\n\"Immunophenotyping in a COVID-19 Cohort (IMPACC) Transcriptomics and    Genotyping Assays\",\n\"IMPACC (IMmunoPhenotyping Assessment in a COVID-19 Cohort) is a prospective longitudinal cohort study (NCT04378777) launched in May 2020 in response to the worldwide pandemic caused by the severe acute respiratory   hospitalized COVID-19 patients in the study within 48 hours of their hospital admission. Clinical data and biological samples (blood and mid-turbinate nasal\",\n\"Prospective Longitudinal Cohort\",\n\"DS-COV --- Disease-specific (covid-19)\"],\n [\"phs002689.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Rapid Saliva    Antigen Test for SARs-CoV-2 Detection\",\n\"The pandemic caused by COVID-19 has been associated with a very high disease burden as well as high global financial costs. With this pandemic came massive shut downs of both  private and public businesses including schools, retail stores, and restaurants. In order to be able to safely open these places of business and have\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002694.v2.p1\",\n\"COVID-19 ACTIV-4 ACUTE: A Multicenter, Adaptive, Randomized Controlled    Platform Trial of the Safety and Efficacy of Antithrombotic Strategies    in Hospitalized Adults with COVID-19 (ACTIV4A)\",\n\"This is a randomized, open label, adaptive platform trial to compare the effectiveness of antithrombotic strategies for prevention of adverse outcomes in COVID-19 positive inpatients. Inclusion Criteria In order to  be eligible to participate in this study, an individual must meet all of the following criteria: &#x02265; 18 years of age Hospitalized for COVID-19\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002699.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): SCENTinel: A    Rapid Smell Test for COVID-19 Surveillance\",\n\"Smell loss is a predominant symptom of COVID-19, and initial evidence based on self-reports suggests that chemosensory loss is a sensitive predictor of COVID-19 in the general population, more so   patients with COVID-19 by 20% compared to an objective test. Therefore, we propose testing and deploying a rapid and objective measure of smell ability, the\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs002700.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Nanobody-based    Electrochemical Biosensor for Real-Time Detection of Aerosolized    SARS-CoV2\",\n\"Coronavirus 2019 (COVID-19) has afflicted 6.2 million Americans and killed 190,000 as of early September 2020 (WHO website); a roughly 3% mortality. Between a shortage in testing and unidentified asymptomatic   contaminated surface or by inhaling aerosolized virus. Unfortunately, COVID-19 is likely to be prevalent well into 2021 and beyond. We must increase our ability\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002702.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Scalable    Aptamer-Based Electrochemical Biosensor for Rapid Detection of    SARS-CoV-2 from Saliva\",\n\"This study is designed to develop an aptamer-based electrochemical detection platform with SARS-CoV-2 as the model system to demonstrate its performance. As this study is a technology development, the data  generated do not involve anything related to human samples or human subjects. The project's goals will be relayed as a data summary in the form of a\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002704.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Use of Behavioral Economics in Repeat SARS-CoV-2 Antibody Testing in    Disadvantaged Communities\",\n\"The rapid spread of the SARS-CoV-2 virus has greatly impacted underserved populations. This project aims to understand social and behavioral determinants of COVID-19 testing, and variations within sub-groups of this   promoting repeated testing. Messaging will focus upon either (1a) household risk, or (1b) personal risk of COVID-19. They are also randomly assigned to an\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002707.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Understanding Factors Influencing COVID-19 Testing and Vaccination in    Immigrant Low-Income and Homeless Populations and Testing Targeted    Interventions\",\n\"Working with our community partners, we will create and evaluate the impact of a public health intervention in the area of Cumberland County, Maine to provide outreach, testing, and vaccination   homelessness. This will include a longitudinal study to understand beliefs about and knowledge of COVID-19, and barriers and beliefs about testing and\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs002708.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Community-Driven Optimization of COVID-19 Testing to Reach and Engage    Underserved Areas for Testing Equity (CO-CREATE)\",\n\"UC San Diego's Superfund Research Center (SRC), supported by the NIEHS Superfund Research Program since 2000, is engaged in research and Public Health initiatives that examine the cumulative impacts of   conditions. These same communities are being ravaged by COVID-19, which is caused by infection by the SARS-CoV-2 virus. With California becoming one the global\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002709.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Multiparametric    Integrated Molecular Detection of SARS-CoV-2 from Biofluids by    Adapting Single Extracellular Vesicle Characterization Technologies\",\n\"COVID-19, a global pandemic due to its rapid progression, is caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). There are different assays for COVID-19, including nucleic acid, serological,   nasopharyngeal secretions and saliva from COVID-19 patients. Note for data in RADx: Instructions for requesting individual-level data are available on RADx Data\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002710.v1.p1\",\n\"COVID-19 Outpatient Thrombosis Prevention Trial (ACTIV-4B)\",\n\"An adaptive randomized double-blind placebo-controlled platform trial to compare the effectiveness of anticoagulation with antiplatelet agents and with placebo to prevent thrombotic events in patients diagnosed with COVID-19 who are  not admitted to hospital as COVID-19 related symptoms are currently stable. Inclusion criteria: Age between 40 and 80 years inclusive Documentation\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002711.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Community-Engaged Approaches to Testing in Community and Healthcare    Settings for Underserved Populations (CATCH-UP)\",\n\"The pandemic caused by the novel coronavirus, SARS-CoV2, has resulted in substantial global morbidity and mortality, including in Oklahoma, and has caused unprecedented interruptions in all aspects of our lives.  COVID-19 has demonstrated considerable disparities based on age and certain chronic illnesses, as well as other social determinants of health. The\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002713.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Using Health Communication to Increase COVID-19 Testing and    Vaccination in Underserved Populations\",\n\"Many low-income Americans, racial and ethnic minorities, and other marginalized groups live in information-poor environments, disproportionately exposed to misinformation about COVID-19, and distrusting medical, government, and scientific institutions and leaders.   contexts in which people live must be considered, especially among groups hit hardest by the COVID-19 pandemic. Our approach builds on strong evidence from\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002720.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Exploring Barriers and Facilitators to Women Who Use Drugs (WWUD)    Awareness, Acceptance and Uptake of COVID-19 Testing (The CARE Study)\",\n\"In the U.S., women who use illicit drugs (WWUD) have disproportionately high rates of infectious (e.g., HIV, HCV), chronic, and pulmonary diseases that elevate their risk for COVID-19. Their health  , discrimination). These prevailing social conditions can also undermine WWUDs' ability to protect themselves from COVID-19 (e.g., washing hands, social distancing\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002729.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Improved    Scalability, Sensitivity, and Interpretability of Pathogen Detection,    Including SARS-CoV-2, in Wastewater using High-Throughput, Highly    Multiplexed Digital Array PCR Technology\",\n\"Presently, the application of molecular technologies, such as, RT-qPCR and digital PCR (dPCR) to quantify SARS-CoV-2 and related targets in wastewater is cumbersome, time consuming, and costly. While progress has  been made in the development of methods and in the interpretation of data, much remains to be improved for the technology to be used as a public\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002736.v1.p1\",\n\"Rapid Acceleration of Diagnostics - TECH (RADx-TECH): Rapid Manufacturing    Scale-up for Compact Point-of-Care EUA SARS-CoV-2 RT-PCR Test\",\n\"This prospective clinical study seeks to examine the performance of the device, a sample-to-answer nucleic acid PCR amplification platform solution for the point-of-care (POC) detection of SARS-CoV-2, compared to the  Roche 6800 Cobas PCR for SARS-CoV-2 assay. Secondary outcomes include usability and interpretability of results among POC device operators. The goal\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002737.v1.p1\",\n\"Rapid Acceleration of Diagnostics - TECH (RADx-TECH): Accurate, Sensitive    and Affordable POC and At Home Testing Device\",\n\"The purpose of this study is to test the performance of the Saliva Test device in detecting SARS CoV-2 in symptomatic people for Emergency Use Authorization (EUA) as a point-of-care  (POC) test. The 2322 Saliva Test is a rapid real-time RT-PCR test intended for the qualitative detection of RNA from the SARS-CoV-2 virus in saliva\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002738.v1.p1\",\n\"Rapid Acceleration of Diagnostics - TECH (RADx-TECH): Ultra-Sensitive    Immunoassay Test for Quantitation of SARS-CoV-2 Antigen\",\n\"The 5534 study prospective clinical study seeks to examine the performance of the 5534 device, for the detection of SARS-CoV-2 nucleocapsid protein antigen compared to the Roche 6800 Cobas PCR  for SARS-CoV-2 assay. The goal of the 5534 study is for EUA approval that the device to use among symptomatic persons by trained healthcare personnel\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002739.v1.p1\",\n\"Rapid Acceleration of Diagnostics - TECH (RADx-TECH): COVID-19 Test At    Home Feasibility Sub-Study\",\n\"The purpose of this feasibility study is to gather knowledge that will inform a future larger study using at-home SARS-CoV-2 testing and digital tools. This study will: 1) Test the  feasibility of daily home SARS-CoV-2 testing using the the Quidel QuickVue device; 2) Test whether a smartphone app (CareEvolution MyDataHelps\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002740.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Support for Safe Return to In-Person School: COVID-19 Testing,    Learning, and Consultation\",\n\"The School TLC Study will develop and implement COVID-19 testing approaches in highly disadvantaged public school settings that address barriers to testing and returning to in-person school among vulnerable populations.  The study includes a COVID-19 test preference study and formative needs assessment to inform ongoing testing approaches and services to support in\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002741.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Community-Engaged Research to Improve the COVID-19 Testing Cascade    Among Underserved Populations in the US Caribbean Territories\",\n\"SARS-CoV-2, the virus that causes COVID-19, has ravaged underserved and vulnerable populations in the U.S., including in the U.S. territories of Puerto Rico (PR) and the U.S. Virgin Islands (USVI).   from COVID-19. Nearly 30% of ECS participants report food insecurity.These ECS data were recently requested by the Congressional Ways and Means Committee\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002742.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Puipuia le Ola: Increasing Reach and Uptake of COVID-19 Testing Among    Pacific Islanders in Hawaii and Guam\",\n\"Non-Native Hawaiian Pacific Islanders (PI), defined as indigenous people having origins in Guam, the Commonwealth of the Northern Mariana Islands, American Samoa, Federated States of Micronesia, Republic of the Marshall  Islands and Republic of Palau, have among the highest COVID-19-associated morbidity and mortality rates in the U.S. PI also suffer from medical co\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002743.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Addressing Low-Income Housing Resident Mistrust in COVID-19 Guidance\",\n\"The purpose of this project is to study how perceived legitimacy of important institutions and mistrust in sources of COVID-19 information affect compliance with public health guidance designed to mitigate  the impact of COVID-19, including maintaining physical distancing, mask wearing, testing, and vaccination. Creating capacity to interact with study\",\n\"Control Set\",\n\"GRU --- General research use\"],\n [\"phs002744.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Rolosense: An    Innovative Platform for Automatic Mobile Phone Readout of Active    SARS-CoV-2\",\n\"The ultimate goal of this proposal is to develop a novel platform technology for automatic surveillance and tracing of airborne SARS-CoV-2 virus particles in real time. The centerpiece of this  proposal is the \\\"Rolosense\\\" technology, which leverages a DNA micromotor as the virus sensing and transduction material (VSTM) that can be detected\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002745.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Epidemiologic Intelligence Network (EpI-Net) to Promote COVID-19    Testing and Prevention Practices Among Socially Vulnerable Communities    in Puerto Rico\",\n\"In Puerto Rico (PR), the first COVID-19 case was confirmed early March 2020. To date cases have exponentially increased partly due to the lifting of locked-down restriction by the PR  government, limited COVID-19 testing, and contact tracing. Puerto Ricans living on the island are particularly vulnerable to the current overlapping\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002747.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): COVID-19    Detection through Scent Analysis with a Compact Gas Chromatography    Device\",\n\"Recent studies, including ours, have suggested that a person's breath may used to diagnose COVID-19 infection and even to monitor its progress. In contrast to immunological and genetic based methods   analyses on up to 760 patients, and to identify and validate COVID-19 biomarkers in breath. Thus, in coordination with the RADx-rad Data Coordination Center\",\n\"Methods\",\n\"GRU --- General research use\"],\n [\"phs002752.v1.p1\",\n\"Clinical Trial of COVID-19 Convalescent Plasma in Outpatients (C3PO)\",\n\"The Clinical Trial of COVID-19 Convalescent Plasma in Outpatients (C3PO) is a multicenter, randomized, single blind, two-arm, placebo controlled Phase III trial with blinded outcome assessments to establish the safety  and efficacy of single dose of convalescent plasma for preventing the progression from mild to severe COVID-19 illness. Note for data in BioData\",\n\"Clinical Trial\",\n\"GRU --- General research use\"],\n [\"phs002759.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Assessing Testing Strategies for Safe Return to K-12 Schools in an    Underserved Population\",\n\"Children with intellectual and developmental disabilities (IDD) are at major risk of irreversible harm from the Coronavirus Infectious Diseases 2019 (COVID-19) pandemic, particularly those from underserved populations. Not only are  they at dramatically higher risk of becoming infected with Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) and death from COVID-19, but\",\n\"Case-Control\",\n\"GRU --- General research use\"],\n [\"phs002760.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    COVID Self-Testing Through Rapid Network Distribution (C-STRAND)\",\n\"COVID-19, the disease caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), has led to a global pandemic and has exacerbated existing health inequities among vulnerable populations. Despite higher rates  of COVID-19 in Black and Latinx individuals compared to White individuals, rates of testing in predominately non-White, low-income communities are\",\n\"Clinical Trial\",\n\"GRU --- General research use\"],\n [\"phs002761.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Addressing COVID-19 Testing Disparities in Vulnerable Populations    Using a Community JITAI (Just in Time Adaptive Intervention)    Approach-A UTHealth CTSA Program\",\n\"The novel coronavirus that causes COVID-19 has negatively impacted global health status and economies worldwide. In the USA, vulnerable populations experience significant disparities in terms of both infection and mortality   incidence and mortality. For example, Hispanics comprise 29.4% of the state's population, but 40.1% of COVID-19 cases and 47.9% of confirmed fatalities. The\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs002763.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Investigating the Effectiveness of COVID-19 Testing Choices, Community    Engagement, and Culturally-Embedded mHealth Literacy Delivery in a    Medically-Underserved, Community-Based Sample\",\n\"This project will employ sentinel and community-based epidemiological surveillance and participatory research methods to evaluate whether a person-centered, rapid COVID-19 testing intervention (at-home swab and send, or on-site point of  care testing), coupled with a novel mobile health (mHealth) COVID-19 Literacy and Outreach Suite of apps and videos, and will serve to increase the\",\n\"Collection\",\n\"GRU --- NULL\"],\n [\"phs002775.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Developing a Realtime Monitoring System and Program to Improve    COVID-19 Testing for Latinx Populations\",\n\"Nationally, Latinx individuals have four times the rate of SARS-CoV-2 diagnoses, compared to Non-Hispanic Whites. Furthermore, Latinx populations may face barriers in accessing SARS-CoV-2 tests, receiving test results, interpreting test   teams (&#x0201c;Promotoras Teams&#x0201d;) with the largest Latinx community organization in Rhode Island to address identified barriers to COVID-19\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs002776.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Getting Asian Americans INFORMED To Facilitate COVID-19 Testing and    Vaccination\",\n\"Getting Asian Americans informed to facilitate COVID-19 testing and vaccinations is a timely project to identify and address sociocultural, ethical and behavioral barriers, and to enable Asian Americans to make  well-informed decisions about getting tested for COVID-19. Asian Americans have been among the highest COVID-19 mortality rates when measured in case\",\n\"Case-Control\",\n\"GRU --- General research use\"],\n [\"phs002777.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Social, Ethical, and Behavioral Implications Research on COVID-19:    Testing and Vaccine Uptake Among Rural Latino Migrants in Southwest    Florida\",\n\"Latinos in Florida account for over 30% of all COVID-19 cases and hospitalizations. Most Latinos live in rural areas, work as migrant farmworkers, and have higher rates of underlying medical   potentially leading to increased risk of COVID-19 exposure, morbidity and mortality. Our community partner, the Hispanic Services Council (HSC), trains local\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs002778.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Microfluidic    Isolation and Characterization of SARS-CoV-2 and Virus Related    Exosomes\",\n\"Robust, efficient, and reliable testing for SARS-CoV-2 is extraordinarily challenging due to our lack of ultra-sensitive assays and constantly evolving knowledge of the virus. Standard PCR based assays still result  in very high false negative rates in the earliest days of infection. Microfluidic processing of clinical samples is low cost and shows great promise\",\n\"Case-Control\",\n\"HMB --- Health/medical/biomedical\"],\n [\"phs002780.v1.p1\",\n\"IFN-&#x003b3; and TNF-&#x003b1; drive a CXCL10+ CCL2+ Macrophage Phenotype Expanded in    Severe COVID-19 Lungs and Inflammatory Diseases with Tissue    Inflammation\",\n\"We explored how the human macrophages response to different inflammatory factors, focusing particularly on the effects of antiviral interferons (IFN-&#x003b2; and IFN-&#x003b3;), pro-inflammatory cytokines such as TNF-&#x003b1;, and other mediators   TNF-&#x003b1; synergistically driven inflammatory macrophage phenotype expanded in severe COVID-19 lungs and other inflamed disease tissues. Related\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002781.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Discovery and    Clinical Validation of Host Biomarkers of Disease Severity and    Multi-system Inflammatory Syndrome in Children (MIS-C) and COVID-19\",\n\"SARS-CoV-2 infection exhibits a wide range of clinical outcomes in both children and adults, from asymptomatic and mild disease to severe viral pneumonia, ARDS, acute kidney injury, thrombotic disorders, and   ( 21) and that nearly 40% of those who are admitted with Covid-19 require ICU support, including mechanical ventilation. One of the most severe\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002782.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): NIEHS    Diagnostic-Prognostic RNAseq\",\n\"Infectious disease outbreaks like Coronavirus Disease 2019 (COVID-19) can overwhelm healthcare systems when screening tools are scarce or lacking. In the face of an ongoing COVID-19 pandemic and with single-plex  , long queue times, backlogs in COVID-19 diagnoses, and delayed access to specialized treatment for COVID-19 patients. The goal of this RADx funded project\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002786.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Lupus Education Alliance Program (LEAP)\",\n\"People living with an autoimmune disease, including those with systemic lupus erythematosus (SLE), have been especially impacted by the COVID-19 pandemic. The purpose of this research project is to identify  barriers and facilitators to COVID-19 testing and vaccination among people living with SLE using both qualitative and quantitative methods. We will\",\n\"Case-Control\",\n\"GRU --- General research use\"],\n [\"phs002791.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Organizations to Reduce, Advance, and Lead for Equity (ORALE) Against    COVID-19\",\n\"The COVID-19 pandemic has disproportionately affected the already underserved Latinx communities. The overall goal of the proposed testing research project is to decrease disparities in the morbidity and mortality patterns  for COVID-19 through engaged partnerships with several communities in central California that are both historically underserved and vulnerable to\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002799.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Leveraging Social Networks to Increase COVID-19 Testing Uptake: A    Comparison of Credible Messenger and Chain Referral Recruitment    Approaches\",\n\"Until the advent of treatment or a vaccine, our ability to contain COVID-19 must rely on widespread identification of (asymptomatic) positive cases, their subsequent quarantine, and contact tracing of those  , uptake, delivery and sustainment) COVID-19 point of service, rapid-testing among a highly vulnerable and often underserved population, those who use\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002800.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Testing for COVID-19 in High Risk Children with Intellectual and    Developmental Disabilities (COV-IDD)\",\n\"1. Virological Testing: We will establish a nasal-swab FDA-approved testing regimen to monitor and identify disease outbreaks in a specialized school setting serving 450 children with moderate-to-severe intellectual and developmental  disabilities (IDD) and the 500 staff supporting them, a setting at ultra-high risk for COVID-19 disease transmission. We will rapidly identify\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002807.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Monitoring COVID-19 and Building Capacity with Northern Plains Tribes    and the Future of Pandemics\",\n\"Monitoring COVID-19 and Building Capacity with Northern Plains Tribes and the Future of Pandemics will upgrade and sustain local capacity at the Native BioData Consortium (NBDC), an Indigenous-led research group,  with Lab Protocols to improve COVID-19 testing outcomes for Tribal populations by creating an improved diagnostics laboratory/local data resource and\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002821.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Harm Reduction Coalition San Diego - OnPoint\",\n\"The project aims to increase access and uptake of COVID019 testing and vaccination among people who inject drugs (PWID), a population highly vulnerable to COVID-19. The proposed RADx-UP project, LinkUP,  , 2020. To date, 31% of La Frontera PWID have tested SARS-CoV-2 antibody+, of whom 65% had not previously been tested for COVID-19. Only 3% had been\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002825.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    SCALE UP COUNTS: A Health Information Technology Approach to    Increasing COVID-19 Testing in Elementary and Middle Schools and    Serving Disadvantaged Communities\",\n\"Racial/ethnic minority and low socioeconomic status (SES) populations suffer profound health inequities across a wide variety of conditions as well as a disproportionate burden of the negative health consequences of  the COVID-19 pandemic. Latinos make up 14% of the Utah population vs. 21% of Utah's COVID-19 cases (50% higher) and the case rate is over 3-fold\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002868.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Obesity Health Disparities Research Center (OHDRC) COVID-19 Testing    Model Among Vulnerable Populations From Community Engagement to    Follow-Up\",\n\"The overall goal of the Obesity Health Disparities Research Center (OHDRC) COVID-19 Testing Model of Vulnerable Populations: From Community Engagement to Follow-Up is to implement a three-component mobile community-based testing  model and evaluate its impact for improved reach, access, acceptance, uptake, and appropriate follow-up to COVID-19 testing. The proposed model\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002869.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    COVID-19 Testing in Underserved and Vulnerable Populations Receiving    Care in San Diego Community Health\",\n\"Pronounced inequities and disparities in coronavirus disease COVID-19 morbidity and mortality have been reported, largely due to comorbid conditions and social determinants of health. Approximately 95% of COVID-19 related deaths   experienced the greatest burden of COVID-19 disease and deaths. Furthermore, testing challenges to date are evident, including long turnaround of test results\",\n\"Clinical Trial\",\n\"GRU --- General research use\"],\n [\"phs002870.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    SARS-CoV-2 Screening and Diagnostic Testing for Return to K-12 Schools\",\n\"By leveraging the infrastructure and community partnerships formed through the ABC Science Collaborative, this project will assess the effectiveness of rapid surveillance testing in reducing within-school transmission and restoring trust  among Black and Latino families to return to in-person school, as well as assess the uptake of school-based testing and time to safe school return\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002871.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP): A    Dynamic COVID-19 Community-Engaged Testing Strategy in Alabama (COVID    COMET AL)\",\n\"This study seeks to understand the factors associated with COVID-19 morbidity and mortality disparities and to lay the foundation to reduce disparities for those underserved and vulnerable populations that are  disproportionately affected by, have the highest infection rates of, and/or are most at risk for adverse outcomes from the COVID-19 pandemic. This\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs002872.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Implementation Science Center in Cancer Control Equity: A Competitive    Revision to Accelerate COVID Testing in Vulnerable Communities -    Project 08\",\n\"This competitive revision submitted to the RADx-UP Initiative (NOT-20-121) aims to extend the infrastructure of our P50 Implementation Science Center for Cancer Control Equity (ISCCCE) to accelerate COVID testing in  -engaged efforts to COVID-19 testing, and to build an infrastructure that will support future COVID-related mitigation and prevention efforts in the\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002873.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    The Cherokee Nation Community-Driven Program for Testing and Contact    Tracing (Cherokee PROTECT)\",\n\"The Cherokee Nation Community-Driven Program for Testing and Contact Tracing (Cherokee PROTECT) unites tribal, academic, and community partners under the leadership of the Cherokee Nation (CN) to solve a dire  need for COVID-19 testing, contact tracing, and culturally informed education in underserved and vulnerable rural populations. Since the beginning of\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs002874.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    COVID-19: Implementing Community-Based Approaches to Increase    SARS-CoV-2 Testing Among an Underserved and Vulnerable Hispanic    Population\",\n\"Hispanics living in the United States have been negatively affected by the COVID-19 pandemic in a disproportionate manner, including significantly higher infection and hospitalization rates compared with non-Hispanic whites. The   foundational data for understanding testing barriers and developing platforms for assessing future COVID-19 vaccine uptake willingness for El Paso residents. The\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs002875.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Assessing Vaccine Hesitancy and a Pharmacist Led Intervention Model to    Increase COVID-19 Vaccine Uptake Among African Americans\",\n\"The investigators at Xavier University of Louisiana are utilizing a multicenter approach including a clinic, health care institution, and community survey-based research project to assess the beliefs and behaviors regarding  willingness to be vaccinated with a prospective COVID-19 vaccine among African Americans in the southeastern Louisiana region. The COVID-19 pandemic\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002878.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Communities Fighting COVID!: Returning Our Kids Back to School Safely\",\n\"Data for this study come from the Communities Fighting COVID!: Returning Our Kids Back to School Safely study, which is part of the National Institutes of Health RADx-UP Initiative, was  supported by the National Institute of Child Health and Human Development of the National Institutes of Health under Award Number 1OT2HD108112-01\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002904.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    ReSET: Restarting Safe Education and Testing for Children with Medical    Complexity\",\n\"Our objective is to increase safe return to school for CMC (children with medical complexity) and school personnel through 3 complementary approaches. Aim 1 proposes to determine the feasibility of  in-home and school-based testing strategies for CMC, as well as to gain understanding of parent perceptions about in-person school attendance\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs002905.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Safety, Testing/Transmission, and Outcomes in Pregnancies with    COVID-19 (STOP COVID-19)\",\n\"Pregnant women are a vulnerable and high-risk population, as COVID-19 is associated with an increased risk of preterm birth, cesarean section, and maternal intensive care. The objectives of this study   will evaluate patients' and providers' perceptions of SARS-CoV-2 testing during pregnancy and the influence of COVID-19 on maternal care-seeking behavior\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002906.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP): A    Nurse-Community Health Worker-Family Partnership Model: Addressing    Uptake of COVID-19 Testing and Control Measures\",\n\"New York City (NYC) is a global epicenter of the SARS-CoV2/COVID-19 pandemic, with 222,000 cases and more than 25,000 deaths. Neighborhood-level disparities in cases and deaths in NYC can be   1.5 times more likely to test positive and more than twice as likely to die as a result of COVID-19. The community of Mott Haven is located the South\",\n\"Case-Control\",\n\"GRU --- General research use\"],\n [\"phs002917.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Reducing Ethical and Social Prejudicial Effects of COVID-19 Testing in    Underserved Populations (RESPECT-UP)\",\n\"The community-engaged project, Reducing Ethical and Social Prejudicial Effects of COVID-19 Testing in Underserved Populations (RESPECT-UP), aims to understand the social, ethical, and behavioral factors surrounding COVID-19 testing in vulnerable   Toolkits, multilevel strategies to reduce COVID-19 disparities, including inequities in testing, while considering vaccine availability. Note for data in\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002920.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP): A    Community Health Worker Intervention to Identify and Decrease Barriers    to Pre-Procedural COVID-19 Testing Among Los Angeles County Department    of Health Safety-Net Patients\",\n\"African American and Latinx communities nationally and in California not only bear a disproportionate burden of COVID-19 positive cases and deaths but are also not taking part in COVID-19 testing   challenge for its majority Latinx and African American patients. With the current pandemic, COVID-19 testing for patients has become integral to receiving\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002924.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Broad-Spectrum    Detection of VOC and Non-VOC Biomarkers from Patient Exhalant Using a    Biomimetic Multiplexed eNose Biosensor for COVID 19 Diagnosis\",\n\"This project will develop biomimetic olfaction-based and gustation-based sensors for detecting volatile organic compound (VOC) biomarkers against SARS-CoV-2. We employ a computational design to deduce sets of odorant binding peptide  (OBP) sequences that will selectively bind to exhaled VOCs associated with SARS-CoV-2 from libraries of olfactory proteins. Chimeric peptides\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002945.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Data Science    Approach to Identify and Manage Multisystem Inflammatory Syndrome in    Children (MIS-C) Associated with SARS-CoV-2 Infection and Kawasaki    Disease in Pediatric Patients\",\n\"The primary objective of this study was to design and validate a predictive decision support system for the identification, treatment, and management of SARS-CoV-2 associated with multisystem inflammatory syndrome in  children (MIS-C). To develop this system, we adapted and retrained machine learning algorithms, which we previously trained in patients with Kawasaki\",\n\"Cross-Sectional\",\n\"GRU --- General research use, GRU-IRB --- General research use (irb)\"],\n [\"phs002946.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Identifying and Addressing Critical Social, Ethical, and Behavioral    Factors Associated with COVID-19 Testing and Vaccination Among Spanish    Speakers\",\n\"Latinx communities are disproportionately affected by the COVID-19 pandemic, with Spanish-speaking Latinx communities carrying even heavier burdens of infection, hospitalization, and mortality. Major barriers to COVID-19 testing and vaccination exist,  and a profound need remains to understand and address the social, ethical, and behavioral implications (SEBI) of COVID-19 testing and vaccination\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs002947.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Supporting the Health and Well-Being of Children with Intellectual and    Developmental Disability During the COVID-19 Pandemic\",\n\"Not only are children with intellectual and developmental disabilities (IDD) at dramatically higher risk of becoming infected with Severe Acute Respiratory Syndrome Coronavirus-2 (SARS-CoV-2) and death from COVID-19, they are   of children with IDD and school staff regarding the impact of COVID-19 and the importance of SARS-CoV-2 testing. At the successful completion of this\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs002948.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Motivational Enhancement to Augment Contingency Management for    SARS-CoV-2 Testing and Vaccination Utilization Among Syringe Exchange    Clients\",\n\"People who inject drugs (PWIDs) are highly vulnerable to contracting SARS-CoV-2 and to the effects of the disease caused by SARS-CoV-2, coronavirus disease 2019 (COVID-19). PWIDs experience underlying medical conditions  and unstable housing that put them at increased risk for COVID-19 morbidity and mortality. PWIDs also experience barriers such as a history of\",\n\"Case-Control\",\n\"GRU --- General research use\"],\n [\"phs002964.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Longitudinal at    Home Smell Testing to Detect Infection by SARS-CoV-2\",\n\"Self-report of sudden loss of smell or taste substantially increases the odds of being diagnosed with SARS-CoV-2. However, self-report of smell function is an unreliable predictor of smell loss. We  created a self-administered easy to use \\\"at home\\\" 5-minute objective smell test to uncover alterations in smell function unbeknownst to many\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs003021.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Community Organizations for Natives: COVID-19 Epidemiology, Research,    Testing, and Services (CONCERTS)\",\n\"As the COVID-19 pandemic spreads to every corner of the US, the plight of urban American Indian/Alaska Native (AI/AN) people is often overlooked or ignored. SARS-CoV-2 and the disease it  causes, COVID-19, disproportionately affect AI/ANs, who are at highest risk of any US racial or ethnic group for developing severe disease and dying\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003026.v1.p1\",\n\"Rapid Acceleration of Diagnostics - TECH (RADx-TECH): COVID-19 Test At    Home Efficacy Sub-Study\",\n\"The FDA has published a safety communication regarding At-Home COVID-19 Antigen Tests (https://www.fda.gov/medical-devices/safety-communications/home-covid-19-antigen-tests-take-steps-reduce-your-risk-false-negative-fda-safety-communication). In support of this notice, a preprint, Performance of Screening for SARS-CoV-2 using Rapid Antigen Tests to   sub-study of the COVID-19 Test At Home Study. The objective is to gather data on performance of Quidel QuickVue for detection of SARS-CoV-2. This sub\",\n\"Prospective Longitudinal Cohort\",\n\"GRU --- General research use\"],\n [\"phs003027.v1.p1\",\n\"Rapid Acceleration of Diagnostics - TECH (RADx-TECH): Test US Bank Study\",\n\"This biorepository is being established to obtain, store, and share body fluid samples for SARS-CoV-2 research. Samples will be obtained at the study sites. The overall study duration will be   collects demographic information, delineates the types of samples they are willing to provide, and collects information about experience with COVID-19 (e.g\",\n\"Collection\",\n\"GRU --- General research use\"],\n [\"phs003029.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Southeast Asians in the US: Health Equity and Research to Understand    COVID-19 Stories (SEA US, HEAR US)\",\n\"Unlike other ethnic minority groups, data on the impacts of COVID-19 on Southeast Asian Americans (SEAAs) are limited and oftentimes aggregated with other Asian American groups, thus limiting targeted assistance  , anxiety, and financial hardships resulting from COVID-19 have intensified existing health and mental health issues. Lack of disaggregated data and under\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003030.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    REACH-OUT: Research, Engagement and Action on COVID-19 Health Outcomes    via Testing\",\n\"As the coronavirus disease 2019 (COVID-19) pandemic evolves, equitable access to SARS-CoV testing and contact tracing will be critical to ongoing pandemic surveillance. Even as COVID-19 infection rates, hospitalizations, and  deaths are declining overall, testing will be necessary to identify people who may benefit from COVID-19 therapies, to encourage isolation and\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003031.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Safer at School Early Alert Project HUB\",\n\"Schools serve important community roles beyond academic education. In historically marginalized communities they are trusted providers for a range of support services for families in need. The tradeoff between these   the hardest by COVID-19 primarily due to structural issues. The Safer at School Early Alert (SASEA) program was co-developed by the University of\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003032.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    RADx-UP Supplement-Vaccine Hesitancy and Uptake\",\n\"The Georgia Center for Diabetes Translation Research Center (P30DK111024) proposes to collaborate with our RADx-UP Diabetes rural community partner and utilize locally developed and tested Georgia CEAL (1OT2HL156812-01/16-312-0217571-66105L) COVID-19 communication  materials to enhance COVID-19 testing in addition to vaccine confidence and uptake in underserved populations most severely affected by COVID-19\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003033.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Social and Behavioral Implications for COVID-19 Testing in Delaware's    Underserved Communities\",\n\"The focus of this project is to triangulate data from semi-structured surveys, serological testing, and census tract-linked public health and economic data to better understand social and behavioral factors related  to COVID-19 testing and vaccine uptake in minority communities, and to develop communication strategies to increase acceptance of testing and\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003037.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Maximizing Child Health and Learning Potential: Promoting A School    Culture of Safety in the Era of COVID-19: Feasibility of Strategic    COVID-19 Testing Protocols in At-Risk Urban Schools\",\n\"We will explore the feasibility of strategic COVID-19 testing protocols in at-risk urban school settings with the goal of increasing a sense of safety for parents and staff, and elucidating  information that could assist the district in adopting a testing protocol, which would increase the time students and staff physically spend in\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003049.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Identifying COVID-19 Vaccine Deserts Using Machine Learning and    Geospatial Analyses to Target Community-Engaged Testing for Vulnerable    Rural Populations to Prevent Localized Outbreaks\",\n\"As of June 30, 2021, 23% of West Virginia's (WV) 55 counties were ranked within the top 20% of counties most vulnerable to Covid-19 in the United States. Central to  . Unfortunately, low testing compounds vulnerability to Covid-19 in medically underserved populations where vaccine uptake is low, as they are extremely susceptible\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003050.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Improving Testing for COVID-19 in Skilled Nursing Facilities: IMPACT\",\n\"Purpose: To assess beliefs, attitudes and intentions of skilled nursing facility staff, residents and family members concerning COVID-19 testing and vaccination, via survey and using mixed methods. Design and Methods:  E-surveys were developed and distributed to measure attitudes about SARS-CoV-2 testing and vaccine uptake among skilled nursing facility (SNF\",\n\"Cross-Sectional\",\n\"GRU --- General research use\"],\n [\"phs003051.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Using the Multiphase Optimization Strategy (MOST) to Optimize an    Intervention to Increase COVID-19 Testing for Black and    Latino/Hispanic Frontline Essential Workers\",\n\"Among those at highest risk for exposure to COVID-19 is the large population of frontline essential workers (FEW) in lower status occupations (e.g., retail, in-home health care), among whom Black  and Latino/Hispanic (BLH) persons are over-represented. For those not vaccinated for COVID, regular COVID-19 screening testing is recommended even\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs003056.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Social, Ethical, and Behavioral Factors in the Return to School Among    Underserved Communities in Maryland\",\n\"The goal is of this study is to use surveys to characterize social, ethical, and behavioral factors associated with parents of K-8 students, their willingness to return their children to  in-person school (or to remain in school), and their support for and compliance with COVID-19 infection control and prevention strategies in eight\",\n\"Case Set\",\n\"GRU-NPU --- General research use (npu)\"],\n [\"phs003057.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Evidence-based Interventions - Individual, Community, and Structural    Factors That Predict Lower Rates of Covid-19 Screening Testing in    Underserved and Vulnerable Populations\",\n\"The purpose of this proposal is to identify individual, community (population) and structural factors associated with lower rates of Covid-19 testing in Northern New England (NNE), with a focus on  underserved and vulnerable populations. Our study includes several COVID-19 medically and/or socially vulnerable populations: communities with high\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs003063.v1.p1\",\n\"COVID-19: Post-Hospital Thromboprophylaxis A Multicenter, Adaptive,    Prospective, Randomized Trial Evaluating the Efficacy and Safety of    Antithrombotic Strategies in Patients with COVID-19 Following Hospital    Discharge (ACTIV-4C)\",\n\"This study is an adaptive, prospective, randomized trial designed to compare the effectiveness and safety of antithrombotic therapy with no antithrombotic therapy after hospitalization for 48 hours or longer for  COVID-19. For Stage 1 of this study, participants will be randomized to either prophylactic anticoagulation or no anticoagulant therapy for 30 days\",\n\"Interventional\",\n\"GRU --- General research use\"],\n [\"phs003081.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    COMMIT\",\n\"In the United States underserved and socially vulnerable populations have endured higher rates and disparities of COVID-19 infection, morbidity, and mortality. This disproportionate burden has shone the light on the  root causes of COVID-19 disparities, such as longstanding systemic racial bias in health care delivery, discrimination, and poor social determinants\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003082.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Localized Mobile Health Approach to Boosting COVID-19 Testing and    Vaccine Literacy, Access, and Uptake Among Women with Criminal Legal    System Involvement\",\n\"People with criminal legal system involvement (CLSI) have experienced five times as many COVID-19 infections and have three times the risk of death compared to general population in the U.S.  Heavily impacted by COVID-19 and squarely within NIH health disparities populations, people with CLSI are often poor and disproportionately from\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003083.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    SARS-COV-2 Screening in Dialysis Facilities: Building an Optimal    Strategy to Protect a High Risk Population\",\n\"We plan to offer SARS-CoV-2 screening tests at selected dialysis facilities in order to reduce transmission and detect early cases to prevent serious illness among patients receiving dialysis. Patients receiving  dialysis have an 8-fold higher risk of hospitalization and a 15-fold higher risk of death from COVID-19 compared to the general population. Patients\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003123.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    C2SPARC: Implementing Mobile, POC COVID-19 Testing in Partnership with    a Community-Based Organization to Reach Women Who Use Drugs\",\n\"Through our RADx-UP Phase I \\\"CARE\\\" study, we are examining correlates of point-of-care COVID-19 testing (POCT) uptake at baseline and reuptake at a 3-month follow-up visit among WWUD (N 250)   - homelessness, food insecurity; individual - drug use, race/ ethnicity, health beliefs) and enabling (prior COVID-19 testing, vaccine uptake) factors that predict\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003124.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Exosome-Based    Non-Traditional Technologies Towards Multi-Parametric and Integrated    Approaches for SARS-CoV-2\",\n\"The urgent need to curb the spread of SARS-CoV-2 demands availability of diagnostics that are more rapid, accurate, sensitive and affordable than qPCR and antibody tests. Current qPCR tests are   is similar to Abbott's recently launched BinaxNOW COVID-19 Ag Card, but has the advantage of multiparametric detection of viral RNA (vRNA), including\",\n\"Case Set\",\n\"HMB --- Health/medical/biomedical\"],\n [\"phs003125.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Engaging Church Health Ministries to Decrease SARS-CoV-2 Vaccine    Hesitancy in Underserved Populations in Baton Rouge\",\n\"In the US, COVID-19 unveiled a disproportionate health burden in low income and underserved segments of society. In Louisiana, some of the greatest health and economic consequences are evident in   urgent need to address vaccine hesitancy within the context of the COVID-19 pandemic and the current testing environment. The Louisiana Clinical and\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003126.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Conducting Correctional COVID Research and Implementing Novel,    Ethically Sound, Sustainable Surveillance Systems (CRAINES)\",\n\"Throughout the COVID19 pandemic in the U.S., persons in jails have borne a share of disease 3-fold or greater than that seen in the general population. While many cases have  been imported from surrounding communities, crowding in these congregate settings has led to rapid in-house transmission, especially since testing in\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003127.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    ReOpening Schools Safely and Educating Youth (ROSSEY)\",\n\"While public health measures with school closure helped mitigate the spread of COVID-19, Latino children from rural agricultural communities are experiencing learning gaps due to disproportionate access to resources. Analyzing   COVID-19 throughout the school year. Caregivers of participants randomized to the control group will receive the two videos at the end of the study period\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003128.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Underserved Populations (RADx-UP):    Mobile Health-Targeted SARS-CoV-2 Testing and Community Interventions    to Maximize Migrant Children's School Attendance During the COVID-19    Pandemic\",\n\"Prospective, mixed-methods cohort study of children who are eligible, enrolled, or who previously participated in the Nebraska Migrant Education Program (MEP) and their families who consent to enroll in an  mHealth-targeted, in-home SARS-CoV-2 testing and community aid program. Participant caregivers will perform household-level SARS-CoV-2 infection risk\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003246.v1.p1\",\n\"Multimodal Immune Profiling to Determine Mechanisms of COVID-19 Clinical    Trajectory in Uganda\",\n\"We conducted a prospective cohort study of adults (RESERVE-U-C19) admitted to Entebbe Regional Referral Hospital (ERRH), a national COVID-19 referral center, across three viral variant-driven phases of the pandemic in   whole-blood transcriptomics (N 100) to determine a range of host responses associated with COVID-19 severity, SARS-CoV-2/HIV co-infection, and circulating\",\n\"Prospective Longitudinal Cohort\",\n\"GRU-IRB-COL --- General research use (irb, col)\"],\n [\"phs002315.v1.p1\",\n\"Integrated Analysis of Multimodal Single-Cell Data\",\n\"The simultaneous measurement of multiple modalities, known as multimodal analysis, represents an exciting frontier for single cell genomics and necessitates new computational methods that can define cellular states based on   subpopulations. Moreover, we demonstrate how to leverage this reference to rapidly map new datasets, and to interpret immune responses to vaccination and COVID-19\",\n\"Case Set\",\n\"GRU --- General research use\"],\n [\"phs003086.v1.p1\",\n\"Single Cell Transcriptomic Data from CD4+ T Cells from Children with    MIS-C\",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a severe complication of SARS-CoV-2 infections in the pediatric population characterized by fever, multi-organ involvement and systemic inflammation. Single-cell RNA sequencing was used  to evaluate the transcriptomic signature of CD4+ T cells obtained from the peripheral blood of patients with MIS-C before (n 3) and after (n 5\",\n\"Case-Control\",\n\"DS-AIID --- Disease-specific (autoimmune/inflammatory/immune disease)\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\"]],\n        columns: [[\"string\", \"accession\"], [\"string\", \"name\"], [\"string\", \"description\"], [\"string\", \"Study Design\"], [\"string\", \"Study Consent\"]],\n        columnOptions: [],\n        rowsPerPage: 10,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n    ",
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>accession</th>\n",
       "      <th>name</th>\n",
       "      <th>description</th>\n",
       "      <th>Study Design</th>\n",
       "      <th>Study Consent</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>phs002245.v1.p1</td>\n",
       "      <td>Genetic Determinants of Susceptibility to Seve...</td>\n",
       "      <td>The goal of the project is to identify genetic...</td>\n",
       "      <td>Interventional</td>\n",
       "      <td>GRU --- General research use</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>phs002258.v1.p1</td>\n",
       "      <td>Shotgun Transcriptome and Isothermal Profiling...</td>\n",
       "      <td>In less than nine months, the Severe Acute Res...</td>\n",
       "      <td>Metagenomics</td>\n",
       "      <td>GRU-IRB-PUB-COL --- General research use (irb,...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>phs002299.v1.p1</td>\n",
       "      <td>PETAL Network: Outcomes Related to COVID-19 Tr...</td>\n",
       "      <td>ORCHID was a multicenter, blinded, placebo-con...</td>\n",
       "      <td>Clinical Trial</td>\n",
       "      <td>HMB --- Health/medical/biomedical</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>phs002300.v1.p1</td>\n",
       "      <td>Successful Clinical Response in Pneumonia Ther...</td>\n",
       "      <td>The goal of this study is to iteratively ident...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-IRB --- Health/medical/biomedical (irb)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>phs003128.v1.p1</td>\n",
       "      <td>Rapid Acceleration of Diagnostics - Underserve...</td>\n",
       "      <td>Prospective, mixed-methods cohort study of chi...</td>\n",
       "      <td>Case Set</td>\n",
       "      <td>GRU --- General research use</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>phs003246.v1.p1</td>\n",
       "      <td>Multimodal Immune Profiling to Determine Mecha...</td>\n",
       "      <td>We conducted a prospective cohort study of adu...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>GRU-IRB-COL --- General research use (irb, col)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>phs002315.v1.p1</td>\n",
       "      <td>Integrated Analysis of Multimodal Single-Cell ...</td>\n",
       "      <td>The simultaneous measurement of multiple modal...</td>\n",
       "      <td>Case Set</td>\n",
       "      <td>GRU --- General research use</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>phs003086.v1.p1</td>\n",
       "      <td>Single Cell Transcriptomic Data from CD4+ T Ce...</td>\n",
       "      <td>Multisystem inflammatory syndrome in children ...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>DS-AIID --- Disease-specific (autoimmune/infla...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>175 rows × 5 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "<google.colab.data_table.DataTable object>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#@title Table of studies\n",
    "print(f\"Number of studies for {query}:\", studies.shape[0])\n",
    "data_table.DataTable(studies, include_index=False, num_rows_per_page=10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "Hf23L_FrJPiJ",
   "metadata": {
    "cellView": "form",
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 4383
    },
    "id": "Hf23L_FrJPiJ",
    "outputId": "e8d1138f-ee88-4c95-ddc3-abd70adb7376"
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 25%|██▍       | 43/175 [00:30<01:25,  1.54it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Skipping: phs002577.v1.p1 - no data access through dbGaP.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████| 175/175 [01:56<00:00,  1.50it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "Number of data access requests : 380\n",
      "Number of unique requestors    : 273\n",
      "Number of unique studies       : 70\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    },
    {
     "data": {
      "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/99dac6621f6ae8c4/data_table.js\";\n\n      window.createDataTable({\n        data: [[\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"2023, Mar07\",\n\"phs002522.v1.p1, phs002523.v1.p1, phs002524.v1.p1, phs002525.v1.p1, phs002527.v1.p1, phs002542.v1.p1, phs002543.v1.p1, phs002544.v1.p1, phs002546.v1.p1, phs002549.v1.p1, phs002550.v1.p1, phs002551.v1.p1, phs002553.v1.p1, phs002561.v1.p1, phs002563.v1.p1, phs002565.v1.p1, phs002569.v1.p1, phs002570.v1.p1, phs002572.v1.p1, phs002573.v1.p1, phs002583.v1.p1, phs002585.v1.p1, phs002588.v1.p1, phs002600.v1.p1, phs002602.v1.p1, phs002603.v1.p1, phs002604.v1.p1, phs002609.v1.p1, phs002631.v1.p1, phs002642.v1.p1, phs002657.v1.p1, phs002679.v1.p1, phs002685.v1.p1, phs002689.v1.p1, phs002699.v1.p1, phs002700.v1.p1, phs002702.v1.p1, phs002709.v1.p1, phs002729.v1.p1, phs002744.v1.p1, phs002747.v1.p1, phs002778.v1.p1, phs002781.v1.p1, phs002782.v1.p1, phs002924.v1.p1, phs002945.v1.p1, phs002964.v1.p1, phs003124.v1.p1\",\n{\n            'v': 48,\n            'f': \"48\",\n        }],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"Mar27, Dec19, 2023, Feb07, 2022\",\n\"phs002516.v1.p1, phs002519.v1.p1, phs002523.v1.p1, phs002533.v1.p1, phs002534.v1.p1, phs002535.v1.p1, phs002537.v1.p1, phs002538.v1.p1, phs002539.v1.p1, phs002540.v1.p1, phs002628.v1.p1\",\n{\n            'v': 11,\n            'f': \"11\",\n        }],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer (2)\",\n\"approved\",\n\"I will be conducting testing to confirm that I, as a user of the RADx Data Hub, am only able to access data I have received prior approval for. \",\n\"I am a co-I of the RADx Data Hub program, and seek access to these program datasets to confirm researcher workflow functionality in the Hub is functioning properly. \",\n\"Nov14, 2022\",\n\"phs002516.v1.p1, phs002533.v1.p1, phs002534.v1.p1, phs002537.v1.p1, phs002538.v1.p1, phs002539.v1.p1, phs002540.v1.p1, phs002628.v1.p1\",\n{\n            'v': 8,\n            'f': \"8\",\n        }],\n [\"Miguez, Maria-Jose \",\n\"FLORIDA INTERNATIONAL UNIVERSITY\",\n\"System analysis for COVID humoral response\",\n\"approved\",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Therefore, it is critical to understand who is at risk (e.g. those with allergies). Our long term goal is to improve early diagnosis.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely prepared given our medical and research experience.   \",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Indeed, in a sizable proportion of cases (up to 40%), the patient requires to be admitted to the ICU service. Therefore, it is critical to understand who is at risk (e.g. those with allergic or non-allergic diseases) those with alterations in the IgE network (platelets, eosinophils, specific Th2 or Th17 cytokines). Our long term objective is to develop new strategies for early diagnosis and to identify biomarkers of risk that can be easily deployed globally.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely qualified to do such analyses given the medical background of the PI, our ongoing work in COVID both in humans and with animal models, our unique expertise in IgE, and expertise using machine learning and model-based analyses.   \",\n\"Feb10, 2023\",\n\"phs002781.v1.p1, phs002945.v1.p1\",\n{\n            'v': 2,\n            'f': \"2\",\n        }],\n [\"Chan, Kei Hang \",\n\"BROWN UNIVERSITY\",\n\"NHLBI TOPMed Whole-genome Sequencing Program (TOPMed WGS): Using molecular and genetic epidemiological approach to dissect the mechanism of complex diseases\",\n\"approved\",\n\"We use epidemiological, statistical, and bioinformatics to dissect the molecular mechanism of complex diseases using genetic, clinical and medication information from several well-established cohort studies. We will analyze the data each TOPMed cohort, e.g. WHI, FHS, JHS and CHS separately.\",\n\"Many complex traits such as diabetes and cardiovascular diseases encompass the interplay between genes and the environment. We hypothesize that the so-called \\u00e2\\u0080\\u009cmissing heritability\\u00e2\\u0080\\u009d is explained by the following two main reasons: First, many subtle genetic variations that are well under the genome-wide significance level coordinate with polymorphisms of strong genomic effects to impact diverse biological processes. Secondly, gene-environment (GxE) interactions, clearly demonstrated by the observation that genetically identical individuals often differ extensively in phenotypes, play a fundamentally important role in both obesity and T2D development. We will conduct an integrative genomics, modification assessment, and mendelian randomization studies that leverage genetic, genomic, clinical, medication, and phenotypic data from several large-scale multi-ethnic cohorts in the TOPMed Consortium to dissect the mechanisms of various complex diseases.  Research Objectives: With the requested data from the TOPMed Consortium, we aim to (including but not limited to the following): 1) evaluate the modification of medication use on biomarker-related genetic variants with risk of complex traits such as diabetes and cardiovascular diseases  2) assess the causal association between biomarkers and risk of complex traits such as diabetes and cardiovascular diseases using mendelian randomization approach 3) identify shared and ethnic-specific biological pathways and gene networks that are perturbed by genetic risks of complex traits between ethnic groups  4) identify shared and ethnic-specific gene network x environmental risk factors interactions that contribute to risk of complex traits between ethnic groups 5) identify shared and ethnic-specific biological pathways and gene networks that are perturbed by genetic effect of medication use between ethnic groups. 6) assess the association between potential biomarkers targeted by medication use and risk of complex traits using mendelian randomization approach with medication related genetic loci as instruments. 7) build disease prediction model using demographic data, lifestyle risk factors, biomarkers, and omics data.  Publication Statement: We will ensure that the research conducted using the datasets will only be published according to the regulations specified in the Data Use Certification (DUC).   Phenotypic and genomic data from TOPMed studies will be used for: (a) investigating genotype-phenotype associations; (b) variant summary statistics, (c) common controls in association tests, (d) reference panels to impute into un-sequenced individuals from TOPMed; and (e) population genetic studies.   TOPMed data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Data from an individual with disease-specific consent will not be used in analyses outside of that restriction, including indirect uses such as imputation reference panels, common controls for association studies and variant summary statistics. For example, an individual with consent restricted to lung function studies will not be used to impute data intended for analyses of diseases not clearly related to lung function. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish or otherwise broadly share any findings from this study with the scientific community.   TOPMed Generic Research Use Statement:  This application anticipates collaboration among TOPMed investigators from multiple institutions, who are named in a master list appended to the IRB document uploaded with this application. Within a set of applicants approved for access to the TOPMed Exchange Areas, data may be shared in a cloud environment or by direct transfer, provided that they all agree among themselves to allow such sharing, and that all have approved TOPMed Data Access Requests for the specific data sets to be shared. Otherwise, data will be shared only among Internal Collaborators \\u00e2\\u0080\\u0093 i.e. within an institution.   Data from TOPMed may be combined with non-TOPMed data from the same or other studies (obtained from dbGaP or other sources), to improve the power for novel genetic discoveries, while respecting the consent of all participants. We expect that this activity creates no additional risks to participants.   \",\n\"2022, Dec15\",\n\"phs002299.v1.p1, phs002363.v1.p1\",\n{\n            'v': 2,\n            'f': \"2\",\n        }],\n [\"Ahuja, Sunil \",\n\"UNIVERSITY OF TEXAS HLTH SCIENCE CENTER\",\n\"Markers of immunologic resilience\",\n\"approved\",\n\"People vary in their susceptibility to develop diseases with features of autoinflammation (e.g., cardiovascular disease, Alzheimer\\u00e2\\u0080\\u0099s disease) versus autoimmunity (e.g., multiple sclerosis, lupus). The reasons for highly-variable disease susceptibility and progression are unknown and currently being explored. One possibility is that disease susceptibility relates to interpersonal differences (e.g., genotypes, epigenetics) in risk of a lowering in immunologic resilience in response to common environmental antigenic challenges accrued with age. Potentially, persons that fail to preserve immunologic resilience in response to chronic low-grade or spontaneous high-grade microbial exposures are at higher risk of developing diseases with features of autoinflammation, whereas the opposite is associated with a heightened risk of developing autoimmune diseases as a trade-off. In the proposed study, we will develop methods to assess immunologic resilience by use of genomic and transcriptomic profiling from the GTex, Framingham Heart Study, T2D-GENES, and Normal Twins & Major Depression cohorts.\",\n\"The primary objective of our studies is to develop quantitative measures of immunologic resilience, biomarkers of immunosenescence, and to identify genetic determinants of immunologic health. In ongoing studies, we have evaluated 20,000+ humans and results demonstrate that chronic antigenic stimuli have a negative impact on immunologic integrity. Using the laboratory variables of CD4+ and CD8+ T cell counts, we developed a method to grade immunologic resilience (immunologic health grades). This immune grading system highly correlated with adverse clinical outcomes such as poor vaccine responsiveness, COVID-19 severity, chronic disease progression, risk of cancer development and recurrence, and all-cause mortality.  Next, by using high-dimensional flow cytometry approaches, we identified immunologic traits (e.g., CD28-CD8dim senescent T cells, CD25++CD8bright proliferating T-cells, CD8+NKT-like cells) that associate with immunologic health grades, chronological age, and the interaction of the two. The identification and studies of these traits have contributed to improvement of our routine FACS analyses. In recent studies, we have identified cell subsets associated with IHG (e.g., CD8+ TEMRA, CD8+CD38+), COVID-19 hospitalization (CD4+ Treg suppressive, CD4+ EM), COVID-19 mortality (CD8+HLA-DR+, CD8+ EM anergic), and COVID-19 sVNT %inhibition (IgD- B cells). Additionally, key cytokines (e.g., ITAC/CXCL11, TNFa) were found to be associated with IHG and COVID-19 disease progression. Furthermore, by integrating RNA-seq transcriptomes and these immunologic traits, we developed transcriptomic signatures that highly associate with the immunologic health grades, T cell responsiveness & dysfunction, plasma IL-6 levels, COVID-19 hospitalization, and all-cause mortality. We used these genomic indexes to interrogate 24,000+ NCBI GEO and Array Express datasets and determined associations with different diseases (e.g., HIV, CMV, COVID-19, tuberculosis, lupus, respiratory diseases, aging) and the levels of these signatures in various cell types/tissues. Recently in our COVID-19 study, newly derived transcriptomic signatures associated with COVID-19 disease progression were found to be associated with the immune health grades in several diseases (e.g., COVID-19, lupus, therapy-naive and treated HIV), influenza severity independent of age, 30- and 120-day mortality independent of age and sex in hospitalized COVID-19 patients, and all-cause mortality independent of age and sex in the Framingham Heart Study.  Genetic polymorphisms associated with COVID-19 severity were assessed in our COVID-19 mechanistic cohort (n=1,826), with a prominent association of 3p21.31 rs17713054G>A proximal to the LZTFL1 gene. This marker was associated with a 2-fold excess of COVID-19 all-cause mortality hazard with each 10-year increase in age, with a dosage effect in European-Americans associating with accelerated mortality rates. Gene expression signatures were identified in the upper airways and peripheral blood to proxy the effect of genotype on COVID-19 outcomes, with multiple signatures associating with viral load in independent cohorts. These biomarkers and signatures will be further evaluated in the Normal Twins & Major Depression study (phs000486.v1.p1) and the T2D-GENES study (phs000847.v1.p1) upon renewal.  In addition, we have developed signatures of immune-checkpoint and immune-checkpoint blockade, which we would like to evaluate/validate in different cancer models and also assess the effectiveness of current treatment protocols.  With this updated background, we request access to these dbGaP datasets with exceptionally thorough phenotyping to identify the genetic determinants that associate with previously derived transcriptomic signatures in various cell types/tissues and previously studied genetic biomarkers. The study will test for associations between single nucleotide polymorphisms and our transcriptomic signatures, & then examine the allele-specific effects of each SNP and the individual genes used within each index. We will not be combining the dbGaP datasets with outside datasets and our study does not create any additional risks to participants due to de-identified status of the data. Where available, we are requesting access to BAM files, genotype calls, OMNI SNP arrays, Affymetrix expression data, allele-specific expression, gene expression matrices, sample attributes, subject phenotypes, laboratory measurements, & disease outcomes.\",\n\"Sep16, 2022\",\n\"phs002300.v1.p1, phs002433.v1.p1\",\n{\n            'v': 2,\n            'f': \"2\",\n        }],\n [\"Polyakov, Yuriy \",\n\"DUALITY TECHNOLOGIES, INC.\",\n\"Secure Genome Wide Association Studies using Homomorphic Encryption\",\n\"approved\",\n\"Our goal is to develop the methodology for secure genome wide association studies where all computations are performed on encrypted data. In other words, the genomic data would never be decrypted during computations. We are planning to use homomorphic encryption as the cryptographic tool for secure computations.\",\n\"The goal of our research is to develop a solution (methodology) for efficient secure genome wide association studies using the techniques based on homomorphic encryption. We submitted two prototypes to the IDASH competition track 2 (http://www.humangenomeprivacy.org/2018/competition-tasks.html) and now want to further develop these prototypes to support larger (practical) scales. If we are able to efficiently perform all computations for real datasets in the encrypted domain, we will submit a paper to Nature Biotech. As part of the same study, we are planning to develop a solution for genetic risk prediction. In summary, our goal is to support secure GWAS analysis of encrypted data.  Our research is funded by NIH SBIR (Phase I).  We are planning to analyze these datasets independently.  Update from 11/18/2020.   We published two papers based on the analysis of these data:  Blatt, M., Gusev, A., Polyakov, Y., Goldwasser, S., \\\"Secure Large-Scale Genome-Wide Association Studies using Homomorphic Encryption\\\", Proceedings of the National Academy of Sciences (PNAS), 2020, Vol. 117, No. 21 [https://eprint.iacr.org/2020/563]. Blatt, M., Gusev, A., Polyakov, Y., Rohloff, K., and Vaikuntanathan, V., \\\"Optimized Homomorphic Encryption Solution for Secure Genome-Wide Association Studies\\\", BMC Medical Genomics, 2020, Vol. 13, Suppl. 7, No. 83 [https://eprint.iacr.org/2019/223].   We now also have DARPA funding to extend our solution for privacy-preserving GWAS to a multi-party setting.  Update from 12/9/2020  We plan to apply the same privacy-preserving GWAS approach to the COVID dataset. The methodology will be the same as what we have used for the AMD dataset.\",\n\"Dec16, 2020\",\n\"phs002245.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Pirastu, Nicola \",\n\"UNIVERSITY OF EDINBURGH\",\n\"Understanding the causal determinants of complex traits through the use of two sample mendelian randomization\",\n\"closed\",\n\"We will be using the MVP genome-wide association study results to understand the factors which are causal to complex disease using a technique called Mendelian Randomization.\",\n\"One of the main goals of epidemiology is to understand how to improve people\\u00e2\\u0080\\u0099s health by modifying causal risk factors. Identifying these causal factors has been extremely challenging due to the difficulty in designing proper longitudinal studies or randomised controlled traits. Recently with the increase of strongly powered GWAS studies, an alternative has aroused: Mendelian randomization. In Mendelian randomization (MR), the genetic variants strongly associated with a specific phenotype are used as instrumental variables to assess causal relationships between an exposure trait and an outcome. In order to avoid bias, MR requires that the effects of the SNPs on the exposure trait and on the outcome trait are measured on different sets of samples. Our study broadly aims at understanding the causal relationships amongst traits by using two-sample Mendelian randomization. In particular, MVP represents a very well powered cohort completely independent from UK biobank and thus results from GWAS are particularly well suited to be used in combination with the results coming from UK biobank to be used both as exposures and outcomes. For this goal we need to access full summary statistics from the GWAS published on the MPV on Blood pressure (DBP, SBP and PP) and Blood lipids (HDL, LDL, Total Cholesterol and Triglycerides).  \",\n\"2019, Oct19\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Picardi, Ernesto \",\n\"INSTITUTE OF BIOMEMBRANES/BIOENERGETICS\",\n\"The role of Y chromosome anomalies in Covid-19 severity and outcome\",\n\"rejected\",\n\"The wide range of clinical outcomes following SARS-Cov-2 infection, from asymptomatic up to severe and/or long lasting infections and death, is well known as well as its correlation to the host genetic profile (PMID: 34237774).  Age-related mosaic loss of chromosome Y (MLY) has been associated with a wide spectrum of human diseases including cancer, Alzheimer's disease, and cardiovascular disease as well as to increased mortality rate (PMID:32020362). We plan to extensively investigate the possible correlation of MLY with the clinical outcome of Covid-19 by analysing the genomic profile of a large cohort of infected subjects and controls stratified according to clinical status, age, and other relevant features.\",\n\"The wide range of clinical outcomes following SARS-Cov-2 infection, from asymptomatic up to severe and/or long lasting infections and death, is well known as well as its correlation to the host genetic profile (PMID: 34237774).  Age-related mosaic loss of chromosome Y (MLY) has been associated with a wide spectrum of human diseases including cancer, Alzheimer's disease, and cardiovascular disease as well as to increased mortality rate (PMID:32020362). We plan to extensively investigate the possible correlation of MLY with the clinical outcome of Covid-19 by analysing the genomic profile of a large cohort of infected subjects and controls stratified according to clinical status, age, and other relevant features.\",\n\"2021, Jul30\",\n\"phs002245.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Pers, Tune \",\n\"UNIVERSITY OF COPENHAGEN\",\n\"Identification of Cell Populations Associated with Type 2 Diabetes\",\n\"approved\",\n\"Type 2 Diabetes (T2D) is a growing health concern worldwide. To study the genetic influence on health and disease we will leverage summary statistic data generated from the genome wide association study from the Million Veteran Program and single-cell profiles to identify the tissue and cell populations that are enriched for genetic susceptibility associated with T2D.\",\n\"The objective of this proposal is to leverage large-scale human genetic data and single-cell profiling to identify candidate cell populations that are enriching for genes co-localizing with genetic variants associated with Type 2 diabetes (T2D). We are requesting access to the African American, European, Hispanic American and multi-ethnic type 2 diabetes (T2D) GWAS summary statistics from the Million Veteran Program (MVP). We have developed a computational framework to identify whether a given cell population overall enriches for genes near variants identified in Genome Wide Association Studies (GWAS). We and others have shown that integration of GWAS data with human microarray-based gene expression data and single-cell RNA-sequencing data can help to pinpoint likely cell populations and tissues underlying complex traits and diseases. Towards this aim, we will integrate summary statistics associated with T2D generated from the GWAS conducted within the MVP dataset with publicly available and inhouse single-cell expression datasets to identify candidate cell populations enriching for human genetic variants associated with T2D. For these analyses we are requesting the summary statistics data (phs001672.v4.p1) published in Nature Genetics volume 52, pages 680\\u00e2\\u0080\\u0093691(2020; PMID 32541925). We are not requesting individual level data. There are no planned collaborations, and the data will be exclusively used by our group.\",\n\"2021, Apr01\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Perera, Minoli \",\n\"NORTHWESTERN UNIVERSITY AT CHICAGO\",\n\"Venous Thromboembolism in African Americans \",\n\"closed\",\n\"We plan to use the publicly available GWAS data to first validate genetic mutations we have found to be associated to clot formation in African Americans. We will then use this GWAS data to prioritize genetic mutations that are more likely to change the amount of protein a cell makes. We will then test these mutations for association with clot formation in our clinical data set of African Americans.\",\n\"Venous thromboembolism (VTE) is a complex disease encompassing deep vein thrombosis (DVT), pulmonary embolism (PE), or a combination of both. In the US, each year over 900,000 individuals are hospitalized due to VTE and over 300,000 patients die. African Americans (AAs) have the highest incidence and mortality rates of DVT and PE. Warfarin is used to treat and prevent DVT and/or PE. However, warfarin dose requirement has been shown to be higher among AAs as well as DVT/PE patients. The research objective is to identify genetic variants associated with DVT/PE in African Americans. We have conducted a candidate gene study in African Americans on warfarin to identify SNP associations to DVT/PE. We plan to use the requested GWAS data set to validate our significant findings in an independent data set. We will then use bioinformatic methods to restrict the GWAS finding to only those that are expression quantitative trait loci (eQTLs) in several relevant tissues to enrich for truly functional signals. We will then genotypes these eQTLs in our African American cohort and again test for association to DVT/PE. Results from these studies will published in peer reviewed journals.\",\n\"Sep29, 2021\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Pendergrass, Rion \",\n\"GENENTECH, INC.\",\n\"Using Summary Statistics from Existing Datsets to Seek Replication of Genetic Associations \",\n\"approved\",\n\"In performing Genome Wide Association Studies (GWAS) it is important to seek replication for significant genetic associations within other datasets. It is also critical to do so across GWAS of different ancestries, to determine what is similar and different. The Million Veterans Program (MVP) has performed a series of GWAS across multiple ancestries, with a range of phenotypic outcomes. We intend to use these summary results to compare with GWAS that we are performing for similar phenotypes. \",\n\"We will not combine the requested datsets with other datasets outside of dbGAP  A series of GWAS were completed within the Million Veterans Program (MVP), and association studies are listed here: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/GetListOfAllObjects.cgi?study_id=phs001672.v4.p1&object_type=analysis These studies provide valuable genetic association results that can be used for comparing to genetic association results we have through GWAS of other datasets.  Of particular interest is comparing our GWAS results to the multi-ancestry results of the MVP, as the MVP is more ancestrally diverse than our European-American focused cohorts. Differing linkage disequilibrium and allele frequencies may highlight important differences and similarities between the results of our GWAS and the MVP. We will be comparing summary statistics across these datasets with our other GWAS results, no individual level data is being combined or compared.   While we are comparing association results across study by ancestry, we are not studying population origins or ancestral history.  \",\n\"Oct22, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Pare, Guillaume \",\n\"MCMASTER UNIVERSITY\",\n\"Genetic and Epigenetic Determinants of Cardiometabolic Disease\",\n\"approved\",\n\"Cardiometabolic diseases such as atherosclerosis and diabetes are leading causes of death and disability globally. While the environmental risk factors for these conditions are well understood, less is known about the effect of alterations in DNA on regulating disease risk. The aim of our study is to 1) identify rare genetic variants in the protein coding regions of our genome, which are responsible for regulating essential structural and functional cellular processes, 2) elucidate the role of common genetic variants through polygenic risk score analysis, and 3) characterize the interplay between genetic and epigenetic signals involved in relevant disease processes. In doing so, we can comprehensively assess the genes and pathways involved in cardiometabolic disease onset and leverage our findings to help discriminate an individual\\u00e2\\u0080\\u0099s risk for disease onset. We will compare the frequency of rare protein-coding variants in diseased and reference populations and also examine differences in epigenetic signatures. In order to effectively accomplish our study aims, we are requesting access to individual-level data from pertinent case-control and longitudinal cohort studies. \",\n\"Cardiometabolic diseases such as Atherosclerotic Cardiovascular Disease (ACVD) and Type 2 Diabetes Mellitus (T2DM) represent leading causes of mortality and morbidity worldwide. Large-scale seminal analyses have demonstrated that an individual\\u00e2\\u0080\\u0099s genetic and epigenetic architecture can confer profound risk for the development of both disorders. We propose that assessing the impact of common variants, rare protein-altering variants, and methylation patterns on disease risk can provide invaluable insight into novel molecular pathways underlying cardiometabolic disease etiology. The primary objectives of our analysis will be to 1) identify the burden of rare protein-altering mutations responsible for very early ACVD and T2DM at the exome-wide level, 2) characterize the role of common variants through polygenic risk scores, 3) investigate epigenetic signatures of cardiometabolic conditions through an integration of genotyping and methylation data. To fulfill our primary objectives with sufficient statistical power, we request access to individual-level data from study populations encompassing the MIGen ExS cohorts, the Heart GO cohorts of the NHLBI GO ESP, select cohorts from the NHLBI TOPMed project, select cohorts of the T2D-GENES Project, and study cohorts inclusive of genotype and DNA methylation data.  Case exomes will be incorporated in a case-only association analysis using summary statistics within the ExAC as an external control dataset. Rare variant burden testing will be used to delineate gene-based association signals and effect sizes. These parameters can be used to facilitate gene prioritization in association analyses using locally sequenced cases. For epigenetic analysis, genotype and DNA methylation data will be integrated in order to characterize the epigenetic signatures associated with cardiometabolic disease. Genetic risk scores can then be used to identify causal relationships through Mendelian randomization. Both analyses will be preceded by stringent quality control procedures to prevent spurious associations. We foresee absolutely no risk for study participants within the requested cohorts as data will remain anonymized on secure servers only accessible to study personnel. All individual-level data requested from dbGaP will not be combined with external datasets. \",\n\"2019, Mar01\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Patrick, Matthew \",\n\"UNIVERSITY OF MICHIGAN AT ANN ARBOR\",\n\"Shared Genetic Risk Between Psoriasis and its Comorbidities through the Million Veteran Program\",\n\"closed\",\n\"Psoriasis is an inflammatory skin disease that substantially affects the health and well-being of many patients. Beyond its impact on the skin, psoriasis also has a broader impact on inflammation across the body. As such, patients with psoriasis are also at risk from a wide range of other inflammatory diseases. Understanding which diseases a patient is at risk from is important to identify how best to treat the patient. We therefore intend to use data from the Million Veteran Program (MVP) to identify the risks of different diseases, based on their genetics.\",\n\"We intend to use data from the Million Veteran Program (MVP) to investigate the shared genetic risk factors between psoriasis and its comorbidities. We have conducted a large GWAS meta-analysis for psoriasis, involving over 11,000 psoriasis individuals (Patrick et al. 2018, Nat. Commun.) and have experience investigating shared genetics using GWAS summary statistics. For example, recently we identified IFIH1 as a shared genetic locus between psoriasis and coronary artery disease (Patrick et al. 2021, J. Investig. Dermatol.). The MVP cohort includes relevant traits, such as HDL and LDL cholesterol, peripheral artery disease and venous thromboembolism, that will help us to understand the relationship between psoriasis and cardiovascular disease at a level of finer granularity, as well as reveal important information about the relationship between psoriasis and its other comorbidities.\",\n\"2021, Nov09\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Priest, James \",\n\"STANFORD UNIVERSITY\",\n\"Pre-genotyped Public Datasets for the Study of Cardiovascular Disease\",\n\"closed\",\n\"To understand why different individuals have diseases such as abnormalities of the heart we look at the DNA or genetic code for differences or mutations. To know if a mutation is important or unusual we need to look for similar genetic mutations in other people who do and do not have the disease. The computer software we use for finding mutations has improved significantly over the last number of years. So we would like to use the publicly available genetic information in the NIH dbGAP from patients and perform our own analysis, so that we can use this data as control subjects for the genetic information we have here at Stanford on patients with heart abnormalities.  \",\n\"Ongoing genetic studies of rare inherited cardiovascular disease within the Stanford University School of Medicine require an appropriate control populations for comparison. The rare cardiovascular diseases under study include cardiomyopathies, arrhythmias, and a variety congenital malformations.   For exome or genome sequencing studies performed by next generation sequencing (NGS), the assumptions regarding alignment and variant calling of publicly available datasets are varied and largely unknown. We will obtain raw fastq or bam files from exome data on upwards of 2000 individuals generated from ARIC, SWISS, PCGC,and other projects which have both healthy controls, and also individuals with some of the cardiovascular diseases under investigation. We will perform alignment and genotyping on these dbGAP subjects in the same highly sensitive manner as our study population. This will maximize sensitivity for rare variant detection in a well-phenotyped control population and minimize the systematic error when performing case-control type comparisons.  A second approach using array-based data will test methods for using external control populations for case-control comparisons where only the cases (probands) have been collected. Probands with cardiovascular disease from the pediatric cardiac genomics consortium will be matched with individuals without in the UK Biobank study or the study populations from dbGAP. Individuals will be matched using a principal components analysis (PCA) on the combined sample, computed over common autosomal SNPs which pass variant quality control. Probands which are outliers in any component relative to the distribution of Biobank individuals with self-reported ancestry will be removed, and a second PCA to be used for matching will be computed in this subsample. Matches will be selected by taking the samples nearest each proband by Euclidean distance in the space of the first four components of the new PCA.   The data obtained from dbGAP may be combined with the collected data at Stanford, the UK Biobank study, and the Pediatric Cardiac Genomics Consortium study for the purposes of improving methods for genetic inference. This re-analysis will not pose any additional risk to the subjects because all of the data (dbGAP, Stanford, UK Biobank, Pediatric Cardiac Genomics Consortium) is received in a de-identified fashion, stored and analyzed under dbGAP-compliant security architecture.  The datasets from dbGAP will be used as healthy controls for the purposes of case-control comparison to rare inherited cardiovascular disease. The datasets from PCGC will be used as a replication cohort and \\u00e2\\u0080\\u009cdisease controls\\u00e2\\u0080\\u009d for ongoing studies of congenital heart disease.\",\n\"2019, Sep27\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Palin, Kimmo \",\n\"UNIVERSITY OF HELSINKI\",\n\"Mutational Processes and Genetic Predisposition in Colorectal Cancer and Other Tumor Types\",\n\"approved\",\n\"Each year, approximately 2600 Finns are diagnosed with colorectal cancer and fifth of them will ultimately die for it. Finland is a small country in the north eastern edge of Europe with exceptionally well characterized population history which is well suited for genetic risk studies. We are studying genetic colorectal cancer risks in Finnish population and will use data from dbGAP to evaluate the global effects of the discovered risk loci. Furthermore we study the somatic variations within the tumors to see what happens to a normal cell when it turns cancerous. \",\n\"Each year, approximately 2600 Finns are diagnosed with colorectal cancer and fifth of them will ultimately die for it. CRC risk has a substantial heritable component and total of 20 genetic loci are known to be associated with it in outbred population. We study genetic germline predisposition for CRC and other tumor types of interest in the isolated founder population of Finland in order to find rare (and common) variants associated with tumor predisposition.  In addition to germline predisposition, we will study molecular mechanisms and effects of somatic changes such as mutational patterns in different parts of the genome and signals driving excess proliferation.  All these studies will require development of novel methods which will be developed and tested by using the requested data.\",\n\"2022, Nov29\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Paik, Hyojung \",\n\"KOREA INSTITUTE/SCIENCE/TECH/INFORMATION\",\n\"Single-cell analysis of preeclampsia causing preterm birth\",\n\"closed\",\n\"The most distinctive feature of the placenta is the existence of cells with two or more heterogeneous origins: fetal and maternal cells in a single tissue. Therefore, de-multiplexing analysis according to cell origin is required for the identification of fetal and maternal characteristic differential genetic expression. Preeclampsia is a common obstetric disorder and is one of the causes of preterm birth, the integration of our preterm birth data and phs001886 data will enable the identification of the etiology of preeclampsia causing preterm birth.\",\n\"Preeclampsia is a common obstetric disorder and a leading cause of maternal and perinatal morbidity. Most of the adverse pregnancy outcomes in people with preeclampsia occur before the 34 weeks' gestation. Therefore, it is essential to identify preeclampsia-associated biomarkers in preterm that precede the previously known late-term markers of preeclampsia. We recently generated placental single-cell RNA-seq data from normal full-term pregnancies (37 weeks) and preeclampsia-induced preterm births (32 weeks), and confirmed the differences in immune cell population characteristics. However, since it was difficult to determine whether these results were due to preeclampsia or premature birth, we intend to analyze this by integrating the data of phs001886 with our dataset. To the integrated dataset, we will apply CellRanger-based quality control and Seurat-based downstream pipeline to annotate cell clusters and perform differential expression gene analysis.\",\n\"2022, Jan04\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"PRITCHARD, JONATHAN \",\n\"STANFORD UNIVERSITY\",\n\"Human genetic variation and complex trait risk\",\n\"rejected\",\n\"We are studying the causes of diseases like heart attack, diabetes, and asthma. In this project, we want to look at the role genes play in these diseases, and particularly on how different genes matter in different contexts. We are focusing on how ancestry changes which genes matter, and how much genes matter in general, as our primary goal, and we also want to look at how these common diseases relate to one another. We will first look at genetic factors for each disease, using both our knowledge of people with the disease and its risk factors. Because more people have each risk factor than have the disease, this allows us to build better genetic models. Then we will use these models to predict disease in people with parents from different populations, which is particularly difficult for genetic testing, by finding which genes came from each population. Finally, we will compare the genetic models to what doctors now use from family history and test whether genetic models can improve disease models using these new methods. \",\n\"This project aims to understand the links between genetic variation and risk for complex diseases. In particular we are developing new methods to analyze complex trait associations in European, admixed and non-European individuals. We will focus on complex traits, including biomarkers and cardiometabolic (heart disease, diabetes, obesity, and complications, including metabolic diseases of pregnancy), sleep, and pulmonary (asthma, COPD, etc, risk factors, and complications) traits. Our work will involve both development of novel methods related to estimating polygenic scores in admixed individuals as well as environmental perturbation modeling and technical, environmental, and genetic sources of poor portability of polygenic scores. In some cases we may analyze these data jointly with data from other sources including UK Biobank. We will not attempt to link individuals from different studies and we do not anticipate that this creates any additional risk to participants. At this time the work does not involve collaboration with researchers at other institutions. Some of the data sets provide notes indicating restricted usage, including that come data can only be used to investigate specific types of conditions; in our analytical work we will abide by all usage regulations. We plan to publish our findings in peer-reviewed academic journal articles.  \",\n\"May26, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"POLIMANTI, RENATO \",\n\"YALE UNIVERSITY\",\n\"Investigating the genetic overlap of human traits and diseases\",\n\"approved\",\n\"Many individuals either suffer from or have family members suffering from a major health diagnosis. Much of the distress caused by these diagnoses arises from a lack of understanding of the \\u00e2\\u0080\\u009cwhy\\u00e2\\u0080\\u009d this event happened to them or their family and how this event affects their personal risk for the disorder. Because the genetic risk for complex traits is distributed throughout the healthy and affected population, our study aims to put disease risk into context and provide the community with answers to why certain features of the human phenome (e.g., diseases and other deleterious traits) persist in the population and how they influence other phenotypes. Additionally, we will compare illnesses affecting different age groups to understand how genetic risk changes over the lifetime and in particular in more vulnerable individuals such as children and the elderly.\",\n\"We will investigate how genome-wide pleiotropic variants connect trait domains across the human phenome. This aim will uncover how the polygenic architecture of disease relates to other phenotypes of the human condition to uncover gene and gene-network therapeutic targets. The causes and consequences of pleiotropy will be investigated using sophisticated statistical approaches such as heritability partitioning for various annotations of the genome, effect size distributions, genomic structural equation modeling, bivariate genetic correlation, latent causal variable,  Mendelian randomization, polygenic risk scoring, genotype networking, and multiple tools for drug-repositioning analyses. To understand the dynamics underlying diseases with different ages of onset, we will focus particularly on those related to childhood and elderly age of onset, comparing them to other illnesses. This will permit us to dissect how genetic variants are associated with pathological associated with disorders characterizing different age groups such as dementias in the elderly. Data use will be limited to research and will not be used in any commercial applications or patents. We will publish or otherwise broadly share any findings from their studies with the scientific community. We also will honor all embargo dates associated with any of the publicly available datasets. We do not believe that this creates any additional risks to participants.  \",\n\"2020, Jul22\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"PALMER, ROHAN \",\n\"EMORY UNIVERSITY\",\n\"Systems Genetics of Alcohol and Other Substance Use Disorders \",\n\"rejected\",\n\"This proposal requests assessment and genotype data in dbGAP for training and research purposes. These data will be used to investigate gene networks that influence susceptibility to alcohol and other drug use and misuse. Drug Addiction is a complex disease affecting many Americans. Despite the fact that genetic differences largely account for individual differences in susceptibility to addiction, analyses using dense SNP coverage of the entire human genome only account for a small portion of the liability to disease. The K01 and DP1 grants builds upon the idea of placing more emphasis on biologically relevant genes by using novel systems biology approaches in combination with statistical and algorithmic procedures that capture gene-gene interactions. Moreover, our analytical pipeline leverages the increased power gained in prediction by modeling not just addiction outcomes, but also comorbid psychopathologies, such as ADHD and depression, which have overlapping genetic liabiity with drug addiction. The rich phenotypic, genotypic, and transcriptomic data available in dbGAP and these these data will be used to identify and test gene-networks that confer susceptibility to alcohol, tobacco, cannabis, heroin, and other substances of abuse.\",\n\"Data (e.g., SAGE, COGA, and The Genetic Architecture of Smoking) will be used to facilitate the identification of genetic loci for drug use and to develop novel computational tools for genomic prediction. Specifically, to develop, modify, and apply novel computational techniques that facilitate the modeling of the joint and multiplicative effects of genetic variants. Each dataset affords access to phenotypes not captured in a single study. Sociodemographic, biological, and other phenotypes related to drug addiction will be examined as covariates/moderators. Primary analyses will be conducted in non-overlapping external samples that are sufficiently large (e.g., UKbiobank, Estonian Biobank, Australian, THL Biobank) and deCODE with replication analyses being conducted across the dbGAP samples with available GWAS data (e.g., SAGE, AddHealth, COGA, etc.). When possible (i.e., only when allowed by the Data Use Agreement), dbGAP datasets that have genotype data and relevant phenotypes will be merged (when the Data Use Agreement allows) to increase statistical power after first imputing the genomic data within ancestry using the Michigan Imputation Server. dbGAP samples that are just summary statistics (e.g., PGC summary files) will be used to inform the primary and replication analyses. We will also meta-analyze results across available datasets using summary statistics from within sample GWAS. Transcriptomic evidence from datasets, such as GTEX will also be used to help prioritize genomic regions for analysis. dbGAP samples of various ancestries are requested to aid in the prioritization of genomic regions as recent studies indicate that different populations confer varying power to estimate effects across different regions of the genome. It is important to note that analyses are being conducted within ancestral groups and will only be reported as such; this minimizes any possibility of stigmatization as our focus is on convergent findings across groups. It is hypothesized that: (Hyp 1) neuronal and metabolism pathways and their associated genes will be well represented in identified gene-network models; (Hyp 2) Bayesian and network-based models will be more predictive of than GWAS association methods; (Hyp 3) genes that comprise subnetworks will be consistent across related phenotypes, though  individual variants within genes may be different; (Hyp 4) genetic effects for related traits (e.g., depression, ADHD) will enhance the identification and makeup of the gene network structure. Replication analyses will be limited to independent non-dbGAP sample samples with similar phenotypic information (e.g., the CADD and NESARC). Externally de-identified data from other repositories will not be combined with these data, but the resulting summary statistics from our analyses will be meta-analyzed across relevant data. (Hyp 5a) A subset of loci that contribute to the additive genetic variance of substance use and disorders can be identified by prioritizing information from prior experimental and eQTL studies. (Hyp5b) The \\\"subset of loci\\\" and the integrated data can be leveraged to infer risk in other samples.  Additional Notes: Individualized dbGAP data will not be shared with any external collaborators. These individuals will only have access to summary statistics. If data are needed they will submit a separate application for access. Requests for phenotypes that are phenotypically and genetically correlated with substance abuse is specifically for the purpose of understanding the genetic architecture of these traits (e.g., PGC summary data for schizophrenia, bipolar, ADHD) as their pleitropy can be leveraged to enhance the detection of loci for substance use outcomes. Requests for access to genotypes and phenotypes of cancer-related datasets are to study individual differences in levels of tobacco/nicotine consumption. Lastly, these data will not being used to associate ancestry/race with any health or clinical outcomes. Specifically, data from Europeans, Latino, and African Americans are requested to conduct within-ancestry analyses in order to increase power to detect regional genetic association effects as stratifying analyses by population and identifying convergent findings has proven to offer greater statistical power (see Brick et al., 2019, Addiction Biology for details). \",\n\"2019, Dec05\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"PALMER, ROHAN \",\n\"EMORY UNIVERSITY\",\n\"Evaluation of Polygenic Scores for Opioid Use in European American Populations \",\n\"approved\",\n\"This proposal requests summary statistics from existing GWAS on MVP samples for training and research purposes. These data will be used to investigate how gene sets identified in the parent GWAS are associated with opioid use outcomes in a clinical sample undergoing treatment of opioid use disorder.\",\n\"This project leverages the MVP GWAS summary statistics to address important questions concerning the generalizability and utility of polygenic scores for opioid use and opioid overdose risk. We will investigate how opioid PGSs are associated with opioid outcomes and opioid related behaviors (i.e., depression and risk taking), in a population undergoing methadone maintenance therapy. Moderators of PGS effects will also be examined using sociocultural measures of participant environment (i.e., PGS x environment). All summary statistics provided will be limited to research use only. Resulting publications will be honor all data use requirements and acknowledge relevant MVP related funding sources. No risk to MVP participants are expected. The overarching goal is to understand how genetic risk for opioid phenotypes is associated with problematic use in European Americans.  \",\n\"2021, Sep09\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Posthuma, Danielle \",\n\"VRIJE UNIVERSITEIT\",\n\"Investigating genetic heterogeneity in dimensions and subgroups of alcohol misuse\",\n\"approved\",\n\"Alcohol misuse, consisting of both heavy consumption and pathological patterns of alcohol use, is one of the leading contributors to the global public health burden. Alcohol misuse is heritable, but there are likely different genes and biological processes involved for different individuals and/or different types of alcohol misuse. In this study, we will conduct genetic analyses across these subgroups/subtypes to see what genetic influences are shared or unique. We will compare these results with those from more traditional measures of alcohol misuse (e.g. alcohol use disorder diagnoses) to determine whether a consideration of these different dimensions can lead to improved understanding of alcohol misuse and targeted implementations in precision medicine. \",\n\"Alcohol misuse, consisting of both heavy consumption and pathological patterns of alcohol use, is one of the leading contributors to the global public health burden. Individual risk for alcohol misuse is substantially heritable, yet there has been slow progress in identifying the associated genetic factors and translating them into clinical practice. However, emerging evidence indicates that biological processes differ across dimensions of alcohol misuse (e.g. consumption versus problems) and may also differ across subgroups of individuals (e.g. with or without certain environmental exposures). Better understanding of these processes has the potential to inform precision medicine applications to best target a variety of etiological processes in the development of alcohol misuse.  In this study, we will use biobank data to investigate the genetic etiology of various dimensions and subgroups of alcohol misuse. We will use GWAS and in silico tools to identify associated genetic loci, and compare these with loci identified by more traditional measures of alcohol misuse such as overall consumption and alcohol use disorder (AUD) diagnoses, using publicly available summary statistics. To this end, we aim to include the summary statistics from the Million Veterans Program GWAS studies of alcohol-related measures (AUD, problematic alcohol use [PAU], AUDIT, maximum alcohol use). We will use these summary statistics to compare overlap in loci from our discovery sample, using tools such as functional annotation and (local) genetic correlation. In case of substantial genetic overlap between studies, we will meta-analyze discovery sample and MVP summary statistics (and/or other publicly available summary statistics of alcohol-related GWASs) to increase variant discovery power.  We will use this data in line with the participants\\u00e2\\u0080\\u0099 consent. We will not access any individual level data and will not attempt to identify participants or their ancestries. This research is being conducted for health/biomedical science purposes and the results will be published in scientific journals and shared with the scientific community to advance the understanding of the etiology of alcohol misuse. \",\n\"2021, Dec13\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"RITCHIE, MARYLYN \",\n\"UNIVERSITY OF PENNSYLVANIA\",\n\"Exploring the genetic architecture of complex traits through genome-wide, phenome-wide, and polygenic risk analyses\",\n\"approved\",\n\"Our goal is to identify the role of genetics in complex human disease.  In some diseases, we expect a single gene, or a small set of genes, will be important for disease risk.  In other diseases, we expect that it is a combination of numerous genes across the genome that work together to lead to increased disease risk.  Additionally, because we observe that people often experience multiple diseases throughout their lifetime, often at the same time, we want to improve our understanding of the relationship between diseases at the genetic level.  This means, there may be specific genes or biological pathways that influence multiple parts of the body at the same time and lead to multiple diseases presenting in an individual.  In our study, we want to explore this across people will common diseases, like hypertension, type 2 diabetes, or elevated blood lipids, and improve our understanding of the role that genetics plays in these disease areas.  \",\n\"One of the primary goals in precision medicine research is to identify the underlying genetic architecture of complex traits.  There are a number of study designs and analysis designs that have been developed to conduct research in this domain.  Genome-wide association studies (GWAS), Phenome-wide association studies (PheWAS), and Polygenic Risk Scores (PRS) are three analysis designs that are commonly used.  Integrating the results across these different analyses can be incredibly useful to begin to deconstruct complex trait architecture.  Looking for which genes associate across multiple diseases or disease categories, or does the PRS score for one trait associate with other disease categories \\u00e2\\u0080\\u0093 these are common questions that we are interested in for understanding complex traits.  In this project, we would like to access the MVP GWAS Summary statistics to integrate with summary statistics and data from other datasets that we are currently using for phenome-wide analyses.  First, the Penn Medicine BioBank (PMBB) is a medical biobank from Penn Medicine that includes genetic data and EHR data.  Second, the UK Biobank is a population-based research study from the United Kingdom; around 500,000 participants have genetic data and clinical data available for analysis.  Third, the electronic Medical Records and Genomics network (eMERGE) is an NHGRI funded initiative that combines electronic health record and genetic data from over 125,000 participants across ten health systems in the United States.  In our study, we plan to conduct PheWAS meta-analyses to look for pleiotropy (where one gene or PRS score is associated with multiple diseases), and PRS analyses to look for variability in PRS performance by age, sex, ancestry; and to look for biological pathways that overlap across different disease categories.  The MVP summary statistics will be incredibly useful for meta-analysis with our existing datasets and generating PRS to validate in our datasets where we have access to de-identified, individual level data. \",\n\"2021, Jan25\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"PALMER, ROHAN \",\n\"EMORY UNIVERSITY\",\n\"Characterizing Phenotypic and Etiological Relationships between Alcohol & Cannabis Misuse and Executive Functioning\",\n\"approved\",\n\"Alcohol and cannabis use disorders are persistent in United States due to both genetic and environmental factors. While polygenic scores provide a means to index alcohol and alcohol use disorder risk in the general population, advancements in identifying genes for complex behavioral traits have been slow. In parallel, early impairment of executive functioning has been implicated as a precursor and genetic correlate of substance involvement and may aid in understanding individual differences in misuse; nonetheless, few have examined how polygenic scores for executive functioning increase our ability to predict risk of alcohol and cannabis use disorders. Specifically, new methods have suggested that if we leverage the observed genetic factors commonly shared between behaviors (i.e., pleiotropy) that we will increase the predictive utility of polygenic scores for behavioral outcomes, such as alcohol and cannabis use disorders. The proposed research examines how domains of executive functioning influence developmental patterns of alcohol and cannabis use using a variety of population and genetic approaches. The information obtained from this project will clarify the relationship between domains of executive functioning and the development of substance use, while providing solid groundwork to elucidate novel biologically informed targets which will contribute to more personalized prevention and intervention approaches. \",\n\"The current proposal is the third part of a dissertation study that seeks to demonstrate the utility of executive functioning as a precursor\\u00e2\\u0080\\u0094as well as vulnerability factor\\u00e2\\u0080\\u0094for alcohol and cannabis use and disorders using both sophisticated phenotypic and genetic methods. Specifically, the MVP GWAS findings for AUDIT-C and ICD diagnoses of abuse/dependence will be utilized to develop polygenic scores (PGS) for both alcohol use and alcohol use disorders among adolescents and young adults from 3 separate cohorts across two independent longitudinal samples in the third part of the study at large. Single and multi-trait models containing derived PGS for executive functioning, alcohol/ cannabis use and disorders, and impulsivity will be employed to predict class membership for alcohol and cannabis use trajectories, as well as lifetime alcohol/cannabis use disorders. Underlying this study is the notion that one genetic locus can influence multiple traits (i.e., pleiotropy), suggesting that increases in variance can be gained by including related traits/behaviors in our PGS models more variation among alcohol/cannabis use profiles than single-trait models. Primarily, we will examine how class membership for alcohol and cannabis use trajectories is a mediator between the PGSs and developing alcohol/cannabis use disorder. All summary statistics provided will be limited to research use only. Resulting publications will be honor all data use requirements and acknowledge relevant MVP related funding sources. No risks to MVP participants are expected.  \",\n\"May13, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Rice, John \",\n\"WASHINGTON UNIVERSITY\",\n\"Replication and Polygenic Risk Score Analysis for Alcohol Use and Dependence\",\n\"approved\",\n\"Alcohol use disorders represent a major public health problem.  We will use the results from this this large-scale study to examine replication in various datasets.  Importantly, we will do separate analyses of European (EA) and African American (AA) samples comparing results from this study and COGA.  We will use all the results from this study to construct a genetic risk score separately for EUs and AAs and apply this to the data.\",\n\"We will use data with smoking and alcohol phenotypes including the case-control data from COGA (The Collaborative Study on the Genetics of Alcoholism) to examine replication in European Americans (EA) and African American (AA) for the GWAS significant findings for alcohol consumption and dependence found in this Million Veterans Project (MVP).  We will also generate Polygenic Risk Scores (PRS) for the EU and AA samples to examine proportion of variance explained for each phenotype.  We will compare the PRS derived from the EUs of the MVP to that derived from the AAs when compared in the AA sample.  We will also compare the PRS derived from the UK Biobank with that from the MVP in the EU sample, and do bivariate analyses using both scores.\",\n\"2019, Oct09\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Robinson-Cohen, Cassianne \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Fetuin-A and Diabetes: A Mendelian Randomization Study\",\n\"closed\",\n\"The protein fetuin-A has been linked with risk of diabetes, but despite intensive investigator it's unclear whether this association is one of cause and effect. The proposed study has direct significance for understanding pathologies related to diabetes and may lead to the identification of novel drugs and therapies to prevent this prevalent condition.\",\n\"Observational studies have shown that fetuin-A, a hepatic-origin protein, is associated with risk of type 2 diabetes, but it is unknown whether this association is causal. We propose to study this association using a Mendelian randomization study with single nucleotide polymorphisms within the fetuin-A\\u00e2\\u0080\\u0093encoding AHSG gene, looked up within Million Veteran Program diabetes GWAS. \",\n\"Mar01, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ripke, Stephan \",\n\"CHARITE, UNIVERSITAETSMEDIZIN BERLIN\",\n\"C4 imputation on schizophrenia cohorts of the Psychiatric Genomics Consortium (PGC)\",\n\"approved\",\n\"The complement component 4 (C4) of the Major Histocompatibility Complex (MHC) region in chromosome 6 exists as two functionally distinct genes (isotypes), C4A and C4B; both vary in structure and copy number. C4 variation has been shown to play a role in schizophrenia risk. We will try to get a more precise picture of the C4 related risk using an updated and much larger data collection from schizophrenia patients and healthy individuals.\",\n\"We will use the reference panel to impute the core cohorts of the Psychiatric Genomics Consortium (PGC) schizophrenia working group, replicating the results from Sekar et al. \\\"Schizophrenia risk from complex variation of complement component 4\\\"(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4752392/). This updated collection comprises 90 cohorts of European (EUR) and East Asian (ASN) ancestry from the PGC, totalling 67,390 cases and 94,015 controls.\",\n\"2022, Sep30\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Rietschel, Marcella \",\n\"CENTRAL INSTITUTE OF MENTAL HEALTH\",\n\"Analysis of shared genetic risk factors for psychiatric disorders\",\n\"approved\",\n\"Psychiatric disorders are often comorbid and are thought to have shared genetic underpinnings. We plan to use results generated by the Million Veteran Program to explore the relationship between comorbid disorders such as depression and addiction. \",\n\"Many psychiatric disorders have comorbid presentations and shared features. Major depressive disorder (MDD) is a serious and complex illness with heterogeneous presentation and high polygenicity. Our previous and ongoing research has used genome wide association techniques in major depressive disorder and other psychiatric disorders, examining questions of comorbidity and shared genetic factors between disorders.  We plan to use summary statistics generated by genome-wide association studies (GWASs) from the Million Veteran Program (MVP), to study how MDD relates to different disorders at the genetic level. MDD summary statistics from MVP will be analysed together with other GWASs and genetic data to investigate disease comorbidity (e.g., with Bipolar Disorder and Addiction), using a variety of genomic analyses techniques such as genetic correlation, polygenic risk scoring and other post-hoc genomic techniques.   \",\n\"2021, Sep01\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Rietschel, Marcella \",\n\"CENTRAL INSTITUTE OF MENTAL HEALTH\",\n\"Analysis of shared and specific genetic risk of Post-traumatic Stress Disorder and Borderline Personality Disorder\",\n\"approved\",\n\"Post-traumatic Stress Disorder (PTSD) and Borderline Personality Disorder co-ocour often, and share common risk factors, such as the experience of traumatic events. We will use the results generated by the Million Veteran Program on PTSD to further investigate shared and distinct disorder mechanisms of the two disorders.  \",\n\"Post-traumatic Stress Disorder (PTSD) and Borderline Personality Disorder (BPD) have comorbid presentations and shared features. Molecular genetic analyses indicate both are highly polygenic disorders. Studying the common and distinct genetic risk factors of both disorders, represents a promising approach to understand shared and specific biological and behavioural disorder mechanisms. We have carried out the largest genome-wide association study (GWAS) on BPD. In the proposed project, we will use summary statistics of GWAS of PTSD and its symptom subdomains carried out in the Million Veteran Program (MVP) (Stein et al., 2021, Nature Genetics) to investigate shared (and distinct) genetic factors of PTSD with BPD. PTSD summary statistics from MVP and BPD summary statistics will be analyzed in regard to their association with each other, and with other related GWAS data, e.g. comorbid mental disorders, or relevant risk factors (e.g. early trauma) (e.g. genetic correlations, GenomicSEM). Additionally, additional analysis will be performed to leverage the shared genetic risk to facilitate discovery of (shared) genetic risk factors (e.g. Multi-trait analysis (MTAG), colocalization analysis). \",\n\"2022, Feb24\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Richards, John Brent \",\n\"FIVE PRIME SCIENCES INCORPORATED\",\n\"Identifying genetic causes of common diseases to accelerate development of therapeutics\",\n\"expired\",\n\"To develop new and effective therapies for diseases, it is important to understand what has caused the disease. In some cases, the cause of the disease may be due to the genetic effects. Identifying these specific genes using human genetics can therefore help to identify the targets for medicines that could be used in the future to help treat disease.  We are therefore proposing to use the human genetic datasets that have already been published by the CHARGE consortium and the Million Veterans Program to identify the genetic causes of disease. We are not requesting to have information for each individual person, rather, we are only requesting access to the summarized information for the relationship between genetic variation and risk of disease.  We will use this summarized information to apply three methods to identify genes causing diseases. First we will use Mendelian randomization, second we will use statistical mapping strategies to identify causal genes and third, we will converge results from these published studies with DNA sequencing studies to increase the probability that the identified genes are causal.  Success in this program could help to identify the targets for new therapies for the diseases studied by the CHARGE consortium and the Million Veterans Program. \",\n\"Objectives: We propose to use summary-level genome-wide association test statistics from published genome-wide association studies to identify genes that play a causal role in diseases. We are motivated to do so because genes that play a causal role in diseases are favourable targets for the development of medicines to treat such diseases. Here we propose to be granted access to summary-level results from genome-wide association studies (GWAS) to fulfill this goal. We do not request access to individual-level data. We will also not post these summary-level results in any repository. Thus, the risks to participants due to data handling and analysis are reduced.  Combining datasets: We will not combine datasets. However, we will qualitatively summarize results from separate datasets, including those in this dbGAP application with other types of data. For example, we will use exome sequencing summary-level data from other cohorts and test if the results from the GWAS summary statistics we are requesting in this dbGAP application are consistent across both datasets.   Different hypotheses and outcomes: We are not focusing on outcomes or hypotheses that were not the focus of the original studies.  Study Design: Our study design involves using already generated GWAS summary statistics to identify causal genes at implicated loci by undertaking three different Aims. In our first Aim, we will use Mendelian randomization (MR) analyses. Such analyses test the effect of an exposure, such as a protein, or gene expression product on the etiology of a disease using genetic variants as instrumental variables. The main advantage of using such an approach, is that it can estimate the causal effect of an exposure, such as a particularly protein level, upon a disease outcome.  In our second Aim, we will use a method that we have recently developed to rank the probability of causality of any gene at a GWAS locus, called the Effector Index. This index provides a probability of causality between zero and one for each gene at a GWAS locus, thus allowing for a prioritization of these genes at associated loci.   For our third Aim, we will combine GWAS summary statistics with results from whole-exome sequencing studies to generate synergistic evidence, by combining these two types of datasets, to further increase confidence that a particular gene influences disease incidence.  We will undertake analyses across the outcomes that have been published through the CHARGE Consortium and the Million Veteran\\u00e2\\u0080\\u0099s Program GWASs.  Brief Analysis Plan: Aim 1: MR. We will use human genetics to identify protein and expression markers using MR\\u00e2\\u0080\\u0094an established genetic epidemiology method. In MR, SNPs are used as proxies for modifiable exposures, such as protein and expression biomarkers. MR circumvents inherent limitations of traditional observational studies, such as confounding and reverse causation. As such, MR can provide evidence supporting, or contradicting, the causality of biomarkers in disease susceptibility.  Aim 2: Causal genes at GWAS loci. We will use different methods to identify potentially causal genes at GWAS loci. These include the Effector Index, our recently published algorithm to meet this need, as well as activity by contact, Pops, Locus2Gene mapping and other recently published methods to identify causal gene(s) at GWAS loci. We will then compare these approaches and qualitatively synthesize results. We will also compare these methods to known positive control genes for common diseases, such as Mendelian causes of rare forms of the disease, as well as targets of successfully developed medicines.  Aim 3: Combining results with Exome Sequencing. Last, we will compare results from Aim 1 and Aim 2 to recently published large-scale exome sequencing programs, such as those generated from UK Biobank. When there is convergence of MR results, highly probably genes at GWAS loci and exome sequencing, such targets are likely to be causal for the disease in question.   Summary By using summary-level GWAS test results, we hope to use the above tools to identify high-yield targets for drug development that could render new therapies for the diseases that have been studied by the CHARGE consortium and the Million Veterans Program. Doing so, could help to alleviate the morbidity and mortality caused by these diseases, thereby realizing in part the goals of these research programs. \",\n\"Apr11, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Richards, Brent \",\n\"SIR MORTIMER B. DAVIS JEWISH GEN HOSP\",\n\"Identification of causal genes for disease\",\n\"approved\",\n\"Human genetics can contribute to understanding diseases and providing insights into potential therapeutic targets. Indeed, 66% of new drugs approved by the FDA had evidence from human genetics supporting their efficacy.   Therefore, we propose utilizing the human genetics datasets available from the Million Veterans Program, the CHARGE Consortium and the GTex project to identify the genetic causes of disease and potential therapeutic targets.   We will apply these methods to the summarized information to identify disease-causing genes. First, we will use Mendelian randomization methods; second, we will use a statistical mapping strategy to identify causative genes. Third, we will integrate the results of these published studies with DNA sequencing to increase the probability that the identified gene is the causative gene.  This study is expected to help identify the genetic mechanism of the diseases and targets for new treatments. \",\n\"Objectives: We propose to use summary-level genome-wide association test statistics from published genome-wide association studies and individual-level data from expression studies to identify genes that play a causal role in diseases. We are motivated to do so because genes that play a causal role in diseases are favourable targets for the development of medicines to treat such diseases. Here we propose to be granted access to summary-level results from genome-wide association studies (GWAS) and individual-level data from expression studies to fulfill this goal. We will also not post these summary-level results in any repository. Thus, the risks to participants due to data handling and analysis are reduced.\",\n\"Dec06, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Rice, Kenneth \",\n\"UNIVERSITY OF WASHINGTON\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease.  Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data.\",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry.  Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community.   However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets. \",\n\"2022, Dec05\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Reitz, Christiane \",\n\"COLUMBIA UNIVERSITY HEALTH SCIENCES\",\n\"Shared heritability of cholesterol levels and non-Mendelian early-onst Alzheimer's Disease\",\n\"rejected\",\n\"We are requesting these data on lipid genetics to help us examine the role of cholesterol in Alzheimer's disease.\",\n\"Investigator:  Christiane Reitz, MD PhD Associate Professor of Neurology and Epidemiology Depts. of Neurology, Epidemiology, Sergievsky Center, Taub Institute for Research on the Aging Brain Columbia University 630 W 168th Street, New York, NY 10032  Phone: (212) 305-0865 Fax: (212) 305-2426 Email: cr2101@cumc.columbia.edu  Objective of the Proposed Research Alzheimer disease (AD) is the leading cause of dementia and occurs in all ethnic and racial groups with a wide range of onset age. Early-onset forms of AD (EOAD), characterized by presentation with first symptoms under age 65, represent as much as 10% of all AD. There is mounting evidence that, among EOAD, only 10% is explained by known mutations in the APP, PSEN1 and PSEN2 genes although previous studies have shown that EOAD is significantly enriched for genetic risk factors. Thus, while some genetic variants influencing EOAD have been identified, the majority of EOAD remains unexplained. One hypothesis for the etiology of EOAD is that other risk factors, such as vascular disease or neuropsychiatric phenotypes, increase susceptibility to AD and \\u00e2\\u0080\\u009cpush\\u00e2\\u0080\\u009d the age of onset earlier.    Study Design To begin assessing this hypothesis, the EOAD Working Group of the Alzheimer\\u00e2\\u0080\\u0099s Disease Genetics Consortium (ADGC; EOAD WG leaders: Reitz/Beecham/Cruchaga), will evaluate genetic correlations between vascular traits, neuropsychiatric traits, and EOAD capitalizing on available GWAS array data in over 13,000 meticulously phenotyped subjects with EOAD. While there is strong evidence that cardio- and cerebrovascular disease is associated with late-onset Alzheimer\\u00e2\\u0080\\u0099s disease, there is a paucity of data on their effect on age of onset of AD and contribution to early-onset AD. The analyses are designed to begin to assess the genetic association of EOAD and age at onset of AD with blood lipid levels, a major risk factor for vascular disease.   Aim1.  Evaluation of Genetic Correlations. To dissect the shared and distinct genetic architecture across EOAD subphenotypes (ie. EOAD with different blood lipid levels) we will apply LD score regression (LDSC). LDSC provides a principled framework to partition genetic covariance of complex traits from genome summary statistics while accounting for LD structure from reference populations, and is computationally tractable even at large sample sizes and genome-wide scope. To apply LDSC, we will remove individuals with genetic relatedness coefficient =0.05 and filter out structural variants and variants with missing rate >1%, a MAF < 1% in reference populations (gnomAD; 1000G), strand ambiguous SNPs, variants in regions with exceptionally long-range LD and variants in pericentromeric regions enriched for sequence gaps. We will then estimate genome-wide heritability and pairwise genetic correlations between vascular endpoints and the adjusted summary statistics for these subphenotypes. Results from LDSC will subsequently be further analyzed using GNOVA (GeNetic cOVariance Analyzer).  We will also use PRS, GTCA and MTAG to analyze the overall relationships between these traits.   Aim 2: Joint Analysis of Correlated Traits. If underlying genetic correlations exist between EOAD and blood lipid levels this is evidence that there may be loci that influence multiple traits. To assess this, we will perform a joint analysis of genetically correlated phenotypes employing MTAG (\\u00e2\\u0080\\u009cmulti-trait analysis of genome-wide association summary statistics\\u00e2\\u0080\\u009d) which allows for efficient meta-analysis of related traits, even when samples overlap. We will use MTAG to analyze variation across EOAD and lipid levels to identify variants or genes that are part of the causal pathway of EOAD and dyslipidemia traits. FUSION/TWAS and UTMOST will be employed to infer the overlap in tissue-specific gene expression between these diseases.   External Collaborators Gary Beecham, PhD Associate Professor Director, Division of Research Informatics in the Center for Genetic Epidemiology and Statistical Genetics, John P. Hussman Institute for Human Genomics Email: gbeecham@med.miami.edu  Carlos Cruchaga, PhD Professor of Psychiatry and Neurology  Director NeuroGenomics and Informatics  Knight-ADRC and DIAN Genetics Core leader Email: cruchagac@wustl.edu \",\n\"Apr07, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"RIVAS, MANUEL \",\n\"STANFORD UNIVERSITY\",\n\"Large-scale inference of population cohorts\",\n\"closed\",\n\"We plan to combine summary statistics from MVP cohort with other existing summary statistics to improve power to detect genetic associations, to assess multi-phenotype patterns of genetic associations, and to improve polygenic risk scores. We plan to visualize our association results in Global Biobank Engine (https://biobankengine.stanford.edu). We will use our own compute resources and will follow procedures outlined in NIH communication (NOT-OD-19-023) for using summary statistic data from human genetic studies. \",\n\"We plan to combine summary statistics from MVP cohort with other existing summary statistics to improve power to detect genetic associations, to assess multi-phenotype patterns of genetic associations, and to improve polygenic risk scores. We plan to visualize our association results in Global Biobank Engine (https://biobankengine.stanford.edu). We will use our own compute resources and will follow procedures outlined in NIH communication (NOT-OD-19-023) for using summary statistic data from human genetic studies.   We have developed novel statistical methods for analyzing summary statistics across multiple variants, multiple phenotypes, and multiple studies (https://www.biorxiv.org/content/10.1101/257162v5, https://www.biorxiv.org/content/10.1101/630079v1). These will be applied to the data. \",\n\"2019, Oct28\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Reitz, Christiane \",\n\"COLUMBIA UNIVERSITY HEALTH SCIENCES\",\n\"Shared hereditability of peripheral artery disease and Non Mendelian early onset Alzheimer's disease\",\n\"rejected\",\n\"We are requesting these data on peripheral artery disease genetics to help us examine its role in Alzheimer's disease.\",\n\"Investigator:  Christiane Reitz, MD PhD Associate Professor of Neurology and Epidemiology Depts. of Neurology, Epidemiology, Sergievsky Center, Taub Institute for Research on the Aging Brain Columbia University 630 W 168th Street, New York, NY 10032  Phone: (212) 305-0865 Fax: (212) 305-2426 Email: cr2101@cumc.columbia.edu  Objective of the Proposed Research Alzheimer disease (AD) is the leading cause of dementia and occurs in all ethnic and racial groups with a wide range of onset age. Early-onset forms of AD (EOAD), characterized by presentation with first symptoms under age 65, represent as much as 10% of all AD. There is mounting evidence that, among EOAD, only 10% is explained by known mutations in the APP, PSEN1 and PSEN2 genes although previous studies have shown that EOAD is significantly enriched for genetic risk factors. Thus, while some genetic variants influencing EOAD have been identified, the majority of EOAD remains unexplained. One hypothesis for the etiology of EOAD is that other risk factors, such as vascular disease or neuropsychiatric phenotypes, increase susceptibility to AD and \\u00e2\\u0080\\u009cpush\\u00e2\\u0080\\u009d the age of onset earlier.    Study Design To begin assessing this hypothesis, the EOAD Working Group of the Alzheimer\\u00e2\\u0080\\u0099s Disease Genetics Consortium (ADGC; EOAD WG leaders: Reitz/Beecham/Cruchaga), will evaluate genetic correlations between vascular traits, neuropsychiatric traits, and EOAD capitalizing on available GWAS array data in over 13,000 meticulously phenotyped subjects with EOAD. While there is strong evidence that cardio- and cerebrovascular disease is associated with late-onset Alzheimer\\u00e2\\u0080\\u0099s disease, there is a paucity of data on their effect on age of onset of AD and contribution to early-onset AD. The analyses are designed to begin to assess the genetic association of EOAD and age at onset of AD with peripheral artery disease, and assess whether both traits have shared genetic heritability.   Aim1.  Evaluation of Genetic Correlations. To dissect the shared and distinct genetic architecture across EOAD subphenotypes (ie. EOAD with different blood lipid levels) we will apply LD score regression (LDSC). LDSC provides a principled framework to partition genetic covariance of complex traits from genome summary statistics while accounting for LD structure from reference populations, and is computationally tractable even at large sample sizes and genome-wide scope. To apply LDSC, we will remove individuals with genetic relatedness coefficient =0.05 and filter out structural variants and variants with missing rate >1%, a MAF < 1% in reference populations (gnomAD; 1000G), strand ambiguous SNPs, variants in regions with exceptionally long-range LD and variants in pericentromeric regions enriched for sequence gaps. We will then estimate genome-wide heritability and pairwise genetic correlations between vascular endpoints and the adjusted summary statistics for these subphenotypes. Results from LDSC will subsequently be further analyzed using GNOVA (GeNetic cOVariance Analyzer).  We will also use PRS, GTCA and MTAG to analyze the overall relationships between these traits.   Aim 2: Joint Analysis of Correlated Traits. If underlying genetic correlations exist between EOAD and blood lipid levels this is evidence that there may be loci that influence multiple traits. To assess this, we will perform a joint analysis of genetically correlated phenotypes employing MTAG (\\u00e2\\u0080\\u009cmulti-trait analysis of genome-wide association summary statistics\\u00e2\\u0080\\u009d) which allows for efficient meta-analysis of related traits, even when samples overlap. We will use MTAG to analyze variation across EOAD and lipid levels to identify variants or genes that are part of the causal pathway of EOAD and dyslipidemia traits. FUSION/TWAS and UTMOST will be employed to infer the overlap in tissue-specific gene expression between these diseases.   External Collaborators Gary Beecham, PhD Associate Professor Director, Division of Research Informatics in the Center for Genetic Epidemiology and Statistical Genetics, John P. Hussman Institute for Human Genomics Email: gbeecham@med.miami.edu  Carlos Cruchaga, PhD Professor of Psychiatry and Neurology  Director NeuroGenomics and Informatics  Knight-ADRC and DIAN Genetics Core leader Email: cruchagac@wustl.edu  \",\n\"Nov10, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Reilly, Muredach \",\n\"COLUMBIA UNIVERSITY HEALTH SCIENCES\",\n\"Genetic determinants of cardiovascular diseases across multiple de-identified biobank data sets\",\n\"approved\",\n\"Cardiovascular diseases are a serious and widespread threat to human health. A major goal of biomedicine is to better understand gene-disease associations. Analysis of rare genetic changes, especially potential changes that lead to a loss in function, can help us understand gene function and have implications for the development of medicines. Our groups, both based in the Columbia University Division of Cardiology, largely focus on cardiometabolic diseases such as atherosclerosis and heart attack. The MIGen data provides large-scale, high-quality information about DNA sequence for people who had a heart attack at unusually-young ages, and healthy comparison people who have not had a heart attack. We will use bioinformatic analysis methods to identify changes in DNA, including in sequence and copy number. We will use this analyzed data to look for associations of particular DNA changes or groups of changes with cardiovascular diseases (such as early-onset heart attack). Our existing data is in a South Asian population (Pakistan), and we would like to compare the results with a largely Caucasian population (MIGen) to see if racial and ethnic ancestry changes the strength or direction of effect of genetic changes. \",\n\"A major goal of biomedicine is to better understand gene-disease associations. Phenotypic analysis of rare functional variants, especially putative loss of function variants (PLoF), can help inform gene function and guide drug discovery efforts. Our groups, both based in the Columbia University Irving Medical Center Department of Medicine, Division of Cardiology, largely focus on cardiometabolic diseases such as atherosclerosis and myocardial infarction. Dr. Saleheen also serves as Scientific Director of the Pakistan Genomic Resource (PGR), which is a large-scale biobank (>180,000 participants) with genomic DNA, whole exome sequencing (>80,000 participants), basic biochemistry, anthropometric traits, EKG, and an extensive questionnaire that includes personal and family medical history, diet, tobacco use, alcohol consumption, and physical activity measures. The STAMPEED data provides large-scale, high-quality genotyping in a case-control study of early-onset myocardial infarction cases and matched controls. We will use bioinformatic pipelines to annotate variants of interest, carriers of rare functional variants, and to identify copy number variations. Using the annotations, we will perform genome-wide association tests on variants of interest as well as gene burden associations for rare PLoF, missense, and other functional variants to check for additive and recessive associations.   We will combine and compare the following datasets with those in this application: Pakistan Genomic Resource (PGR), UK Biobank, Biobank Japan, CARDIoGRAMplusC4D, All of Us, and the Munich Vascular Biobank. Together, these studies provide data on ~1.5 million participants (~300,000 cases) for genome-wide common variants. We will also perform ancestry analysis and meta-analysis to compare the effects of variants in the different populations, with two exceptions. We will not perform ancestry analysis for the datasets phs001101.v1.p1 (\\\"Malmo Diet and Cancer Study\\\") and phs001672.v8.p1 (\\\"Veteran's Administration (VA) Million Veteran Program (MVP) Summary Results from Omics Studies\\\") as both of these datasets bar such analyses.   Access to this dataset will aid the discovery and multi-ethnic validation of genetic mutations and copy number variants associated with cardiometabolic diseases. Project personnel using this data will publish or otherwise broadly share any findings from these studies with the scientific community, honoring all possible embargo dates associated with any publicly-available datasets.\",\n\"2022, Oct31\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ray, Debashree \",\n\"JOHNS HOPKINS UNIVERSITY\",\n\"Genetic determinants of diabetes and related traits\",\n\"approved\",\n\"We will use summary information from Million Veteran Program dataset to replicate genetic risk factors of diabetes and other related outcomes.\",\n\"We will use genome-wide association study (GWAS) summary statistics from the Million Veteran Program (MVP) dataset to explore if genetic determinants identified from our in-house data on diabetes and related traits are replicated in MVP. Phenotypes of interest are diabetes and any related trait/outcome. We will not combine the MVP dataset with other datasets outside of dbGaP; we will only combine summary information of MVP with other summary information such as that used for meta-analysis when replicating results. Further, we will use this MVP summary dataset to illustrate novel statistical methodologies based on summary statistics. There is no additional risks to participants.\",\n\"Nov22, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ratan, Aakrosh \",\n\"UNIVERSITY OF VIRGINIA\",\n\"Investigating Heterogeneity of LGL Leukemia\",\n\"approved\",\n\"Large granular lymphocytic (LGL) leukemia is a type of chronic leukemia that affects white blood cells called \\\"lymphocytes\\u00e2\\u0080\\u009d. The disease is heterogeneous both in terms of clinical presentations and the molecular phenotypes. We plan to generate single-cell data from LGL patients and will use publicly available datasets to provide context and understand the various cell types and states that play a role in LGL. \",\n\"LGL leukemia results from a persistent clonal expansion of cytotoxic lymphocytes, including from T-cell or NK-cell lineages. The disease is also associated with neutropenia, anemia and B-cell dyscrasias. The goal of this project is to use multi-modal single-cell sequencing to establish a high-dimensional single-cell reference atlas of LGL leukemia cell types and cell states. We aim to generate multi-modal sequencing datasets from LGL leukemia patients with various clinical presentations and molecular phenotypes. We plan to map new data onto available public reference single-cell and genomic datasets including the dataset generated for study accession phs002315.v1.p1 for contextual interpretation of aberrant cell states. We will then use the supervised annotations to test for differences in cell type abundance or cell type gene expression across the various clinical and molecular phenotypes. In order to control against technical artifacts, we will standardize analysis using the same software pipeline on each dataset, using raw reads as input. After analysis is complete, the generated sequencing data will be deposited in dbGaP, while processed counts data will be deposited to GEO. We intend to broadly share our results with the scientific community by publishing our findings in peer-reviewed journals. \",\n\"2022, Jan13\",\n\"phs002315.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"ROVARIS, DIEGO \",\n\"UNIVERSITY OF SAO PAULO\",\n\"The Biological Relationship between ADHD and Depression in Contrasting Exposomes\",\n\"approved\",\n\"In psychiatry, it is common to diagnose patients with more than one mental health issue. Clinical depression, for example, is often accompanied by ADHD diagnosis (attention deficit hyperactivity disorder). With novel genomic tools, it is possible to find genetic signals that correlate with the presence of mental health issues and may help explain what biological mechanisms are behind them. The environment and society in which these patients are inserted are also of extreme importance since it plays an important role in the well-being and mental health of an individual. In our study, we aim to look deeper into the biology behind the correlation between depression and ADHD looking for genetic signals that are shared between the two, signals unique to depression and signals unique to ADHD. We would like to understand if and how the environmental variables influence these diagnosis as well as which one of these are most relevant. We will compare data from volunteers in our country, Brazil, to data that has been collected by many other countries in the world exploring the differences and similarities between genomics, environment, and diagnosis. \",\n\"This statement has been written to request access to the summary statistics resulting from the Genome Wide Association Study performed by Levey et. al in \\u00e2\\u0080\\u009cBi-ancestral depression GWAS in the Million Veteran Program and meta-analysis in >1.2 million individuals highlight new therapeutic directions\\u00e2\\u0080\\u009d (phs001672.v1.p).  The objective of our proposed research is to better understand the biological association between Major Depressive Disorder (MDD) and Attention Deficit and Hyperactivity Disorder (ADHD) in a clinical ADHD sample collected in Porto Alegre, Brazil, by the Adult ADHD Outpatient Program (PRODAH) in the hospital of the Federal University of Rio Grande of Sul. The sample consists of \\u00cb\\u009c600 ADHD cases and \\u00cb\\u009c600 controls with extensive clinical, sociodemographic, and biological data (e.g., GWAS, EWAS and neuroimaging) additional to longitudinal data in two waves over thirteen years of clinical follow-up.  We plan on using the beforementioned summary statistics to perform 1) polygenic risk score analysis in our sample to explore the biological relationships between ADHD and depression as well as potential synergism between the ADHD and depression PRSs on the severity of ADHD. Additionally, we will 2) explore local correlations between ADHD and depression (using only summary statics data) to find the genomic regions that better explain the comorbidity between ADHD and depression and evaluate them in our clinical sample. Since we request the summary statistics, which does not identify the participants information, our research will be consistent with the data usage limitations.\",\n\"Jun06, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"ROSAND, JONATHAN \",\n\"MASSACHUSETTS GENERAL HOSPITAL\",\n\"Joint Genetic Contribution to Vascular-Related Phenotypes \",\n\"approved\",\n\"Vascular conditions (myocardial infarction and stroke) constitute one of the leading causes of death and disability worldwide. Several recent genetic studies demonstrate that genetic variation has an important role in the occurrence from these diseases.  The main feature of modern genotyping technologies is their impressive ability to generate massive amounts of information related to genetic variation across the genome. While these studies have been very successful in identifying genetic variants that increase the risk of each vascular condition considered separately, interesting opportunities are available to discover genetic variants that contribute to causing vascular conditions in general. The aim of this proposal is to combine data from several studies on different vascular diseases in order to uncover these variants. We seek to add value to already existing resources, leveraging the notion that a single entity, blood vessels, is affected in vascular diseases, regardless of the final organ involved (the heart in myocardial infarction and the brain in stroke).\",\n\"Biologically related conditions share common genetic predisposing factors. This is relevant for vascular conditions, as they share a single anatomical target (blood vessels), final mechanism of injury (occlusion or rupture of vessels) and predisposing non-genetic risk factors (diabetes, hypertension, hypercholesterolemia, smoking, obesity). Large amounts of data are available for vascular conditions, risk factor and related endophentoypes. Likewise, novel informatic and statistical methods designed to analyze this data are available regularly. We intend to publish/broadly share our findings with the scientific community when available.  The aims of this proposal are: 1) to evaluate whether genetic architecture of vascular conditions and their risk factors overlap; 2) develop/test novel analytical techniques designed to assess joint genetic contribution to vascular outcomes and risk factors; 3) identify genetic loci that contribute to these traits; 4) assess if loci identified in #2, (separately/in aggregate) improve prediction models for occurrence, severity and outcome of vascular phenotypes; 5) to develop tools to manipulate/automate genetic and phenotypic data related to vascular phenotypes  We will combine sequencing, genotype + phenotype data related to several of ischemic vascular phenotypes (myocardial infarction, ischemic stroke, acute peripheral occlusion), hemorrhagic vascular phenotypes (hemorrhagic stroke), non-vascular phenotypes with known vascular component (Alzheimer\\u00e2\\u0080\\u0099s Disease), vascular risk factors (diabetes, hypertension, hypercholesterolemia, smoking, obesity) and relevant endophenotypes (white matter hyperintensities). In this combined dataset we plan to conduct single-variant, gene-based, pathway-based, polygenic risk score, and single-trait and joint heritability analyses, as well test newly developed statistical tools aimed at comparing genome-wide data across related phenotypes.   Data is individually/jointly analyze with data outside of dbGaP. External dataset may be meta-analyzed with dbGaP data or serve as replication cohorts. This plan will not create additional risk. Data use is consistent with all set Data Use Limitations and will not be combined with other phenotypes. We intend to publish results and share findings.\",\n\"2020, Dec28\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"ROMANOSKI, CASEY \",\n\"UNIVERSITY OF ARIZONA\",\n\"Fine-Mapping CAD GWAS MVP loci using endohtelial cells molecular data\",\n\"approved\",\n\"In this study, we propose to investigate and identify which variations in DNA pose risk of developing Coronary Artery Disease (CAD) through changing how blood vessel cells work. Our lab has generated a lot of information about the molecular properties of blood vessel cells, under healthy and disease-prone conditions. We will utilize this information together with information from the statistical results of the large population-based study of CAD risk. Together, our goal is to discover specific ways in which blood vessel cells modulate the progression of CAD. \",\n\"The objectives of this project will be to fine-map genetic signals underlying CAD with molecular etiology in vascular endothelial cells. The study design is to intersect summary statistics from the CAD GWAS with genomic loci of interest in endothelial cells to nominate causal SNPs, regulatory elements, and target genes that modulate susceptibility to CAD. Methods used to intersect GWAS and endothelial data will take several forms, including by genomic position as well as using statistical methods like co-localization. Loci of interest from endothelial cells will be defined as those that: 1) demonstrate accessibility in cell-type and activation state-specific patterns; 2) have molecular QTLs for accessibility, TF binding or histone methylation; and 3) have eQTLs. We will not utilize any personally identifiable information in our analysis and will work solely with summary statistics. Therefore, we will not pose any risks to participants. \",\n\"2023, Feb08\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"PALMER, ROHAN \",\n\"EMORY UNIVERSITY\",\n\"Evaluation of Polygenic Scores for Alcohol Use in African American Populations\",\n\"approved\",\n\"Alcohol use and disorders are enduring public health problems in the United States with more than 86.3% of people older than 18 having drank alcohol at some point in their life and 5.8% (i.e., 14.4 million adults) meeting criteria for an alcohol use disorder. Large-scale genomewide associations studies indicate that individual differences in alcohol use and disorder are partly explained by genetic differences. Polygenic scores  provide a means to index alcohol and alcohol use disorder risk in the general population, but few studies have examined their utility in underrepresented groups, such as African Americans. Our study aims to examine the utility of alcohol PGSs in order to understand how they perform within groups of individuals exposed to varying levels of traumatic experiences  lifetime.\",\n\"This project leverages the MVP GWAS summary statistics to address important questions concerning the generalizability and utility of polygenic scores for alcohol use and problems. The provided summary statistics will be used to develop polygenic score in an African American cohort of men and women exposed to trauma. We will investigate how alcohol PGSs are associated with alcohol outcomes and alcohol related behaviors, such as anxiety and depression. Moderators of PGS effects will also be examined using sociocultural measures of participant evivornment (i.e., PGS x environment). All summary statistics provided will be limited to research use only. Resulting publications will be honor all data use requirements and acknowledge relevant MVP related funding sources. No risk to MVP participants are expected. The overarching goal is to understand how genetic risk for alcohol phenotypes is associated with problematic use in African Americans. We will employ recent MVP GWAS findings for AUDIT-C and AUDIT-P. \",\n\"2021, Jun11\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"AGRAWAL, ARPANA \",\n\"WASHINGTON UNIVERSITY\",\n\"Neurobiological bases of psychiatric traits\",\n\"approved\",\n\"Genetic variation contributes to health outcomes. We will use the results generated by the Million Veteran Program to further our understanding of how polygenic liability influences health-related outcomes.\",\n\"Genetic variation contributes to variability in behavior and health outcomes. We will use summary statistics generated by genome-wide association studies (GWASs) conducted within the Million Veteran Program dataset to examine how genetic variation (e.g., polygenic risk scores, trait-related genetically-conferred differences in gene expression) contributes to individual differences in biology, behavior, and health-related outcomes in diverse samples. The overall objective of this study is to use MVP summary statistics independently and alongside other GWAS summary statistics to identify meaningful pathways (e.g., biology, behavior) through which genetic variation contributes to health outcomes. The study design will include use of these summary statistics to inform studies of health outcomes, meta-analysis of these summary statistics with similar summary statistics from other sources, and for a variety of post hoc genomic analyses such as polygenic risk scoring. We do not plan to combine datasets but rather to combine this summary information with other summary information. Phenotypes will include summary statistics for GWAS related to health, including addictions, as well as behavior and biology.   \",\n\"2020, Apr08\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Osburn, William \",\n\"JOHNS HOPKINS UNIVERSITY\",\n\"Role of VTE-associated SNP in the liver\",\n\"approved\",\n\"Every year up to 600,000 individuals suffer from blood clotting disorders such as deep vein thrombosis and/or pulmonary embolism. There is a need for new therapies to treat or prevent these disorders. Several large genetic studies have identified genetic markers that are found more frequently in individuals with blood clotting disorders. While the effect of mutations in a number of these genetic markers is obvious (e.g., marker is within a gene with a known role in blood clotting), these studies have also identified genetic markers whose role in blood clotting disorders is not clear. Therefore, this study will pinpoint specific genetic variants that could be causing increased risk of blood clotting disorders and investigate their role in production of coagulation proteins using human liver cell lines and standard molecular biology experimental approaches.  \",\n\"Venous thromboembolism (VTE) including deep vein thrombosis and/or pulmonary embolism significantly contributes to morbidity and mortality affecting up to 600,000 individuals a year in the United States. New prevention and therapeutic approaches are needed to lessen the burden of VTE-associated disease. Several genome-wide association studies have identified genetic loci that are associated with risk of VTE. While some of the genetic risk loci involve gene products with known activity along coagulation or hemostasis pathways, several novel loci near or within genes having no known role in coagulation were identified, including genes that encode transcription factors, epigenetic proteins and proteins involved in translation. Since most coagulation factors are secreted by the liver and the expression of genes containing or adjacent to the identified risk loci have been documented in human liver and hepatocyte cell lines, we will focus on the effect of the identified single nucleotide polymorphisms (SNP) on transcription, translation, and secretion of proteins in the coagulation pathway in liver. Therefore, we will do fine mapping of the genetic risk loci in those data sets to identify SNP that are potentially causal and study these SNP in vitro using CRISPR-Cas9 approaches in HepG2 cells, a human hepatocellular carcinoma cell line. \",\n\"Nov02, 2021\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Mackey, Scott \",\n\"UNIVERSITY OF VERMONT &amp; ST AGRIC COLLEGE\",\n\"Polygenic Risk Scores for Nicotine Dependence and Problematic Alcohol Use\",\n\"approved\",\n\"Genome wide association studies (GWAS) identify genetic variants associated with a trait such as problematic substance use. We will use two GWAS from dbGAP which targeted nicotine dependence and problematic alcohol use to generate individualized genetic risk scores for these traits in large cohort samples. We will examine whether these risk scores are related to behavior (e.g. age of onset and frequency of substance use, and measures of impulsivity) or brain features detected using MRI, including measures of brain structure (e.g. cortical and subcortical grey matter volume and white matter tract integrity) and function. \",\n\"Research Use Statement: Polygenic risk scores (PRS) derived from GWAS are a powerful tool to examine the genetic liability of a specific trait in a sample not included the GWAS sample. Our aim is to investigate the genetic liability of nicotine and alcohol use in population-based cohorts in relation to behavioral measures and intermediate brain phenotypes associated with problematic alcohol use or nicotine dependence. PRS will be create using the PLINK 2 software based on the results of the Genome Wide Association Studies (GWAS) for nicotine dependence (PMID 33144568 - phs001532.v2.p1) and problematic alcohol use (PMID 32451486 - phs001672.v3.p). These PRS will be regressed onto behavioral phenotypes (e.g. substance use measures including age of onset and frequency of use, and impulsivity measures) and onto MRI brain measures including structure (cortical and subcortical volume measures and white matter diffusivity) and function. This project does not require individual-level data from the mentioned studies and will be consistent with their data use limitations, using the data only by the terms of the model Data Use Certification onto research, and not for profit organizations. \",\n\"2021, Feb17\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Martins Jr, David \",\n\"FUNDACAO UNIVERSIDADE FEDERAL DO ABC - UFABC\",\n\"Impact of COVID-19 during pregnancy on infant neurodevelopment\",\n\"approved\",\n\"COVID-19 during pregnancy increases the risk of negative outcomes, such as maternal death. The in-utero exposure to some other viral infections cause long-lasting neuropsychiatric alterations. We do not know yet if gestational COVID-19 affects the offspring\\u00e2\\u0080\\u0099s neurodevelopment. This project aims at identifying genes related both to gestational COVID-19 and foetal neurodevelopment. First, we have searched the scientific literature for studies about gestational COVID-19 that provide large-scale protein and gene expression. Our search retrieved 12 high-quality, unique studies. We will re-analyse these datasets individually, since different studies use distinct methodologies, and we must assure that they are comparable. After that, for each study, we will have lists of proteins/genes altered in gestational COVID-19. Then, for each organ (cord blood, placenta, or maternal blood), we will integrate the datasets and find the proteins/genes altered in several studies. We will also find factors altered across distinct tissues. Departing from that, we hypothesize that these genes are altered in foetal brain as well. We will study their expression in foetal brain across the gestation. This analysis may help us understand the neurodevelopmental effects of gestational COVID-19.\",\n\"SARS-CoV-2 infection during gestation increases the risk of negative outcomes, such as maternal death and pre-eclampsia. Although transplacental SARS-CoV-2 transmission is very rare, gestational COVID-19 can still affect the foetus through maternal immune activation and other mechanisms. The in-utero exposure to some viral infections, such as the zika virus disease, cause long-lasting neurological and psychiatric alterations. There is no scientific consensus on whether gestational COVID-19 may affect the offspring\\u00e2\\u0080\\u0099s neurodevelopment, and little is known about its long-term effects. This RUS describes a subproject that aims at identifying protein-coding genes related both to gestational COVID-19 and foetal neurodevelopment. The subproject is part of a larger study that combines a clinical study, RNA-sequencing and bioinformatic analyses. First, we have performed a systematic review to identify proteomics and transcriptomics datasets on pregnant people with COVID-19 and their offspring. The inclusion criteria were: 1) The subjects must be human pregnant people, or neonates whose mothers had the investigated condition; 2) The investigated condition should be SARS-CoV-2 infection during gestation; 3) The study must provide original transcriptomics or proteomics data, produced through any methodology. Our search retrieved 194 records. After removal of duplicates, quality assessment and abstract and full-text screening, we remained with 12 records. This includes the dataset requested in this RUS, and other datasets.  Currently, we are in the data extraction phase, which attempts to obtain raw -omics data, or protein/gene frequency matrices. After obtaining the data, we will re-analyse them to identify differentially expressed genes and proteins, and the p-values of these altered factors. For each type of biological sample (cord blood, placenta or maternal peripheral blood), we will integrate the p-values from distinct studies with Liptak\\u00e2\\u0080\\u0099s weighted z-score. Therefore, for each type of sample, we will have a list of factors consistently altered in various studies. After that, we will intersect the lists from different samples and identify factors that may be consistently altered across distinct tissues. Departing from that, we hypothesize that these genes may be potentially altered in foetal brain as well. We will study their expression in foetal neural tissues across the gestation, using public transcriptomics datasets. We will use an algorithm developed by our research group that uses weighted graphs. This analysis may provide an insight on the potential neurodevelopmental effects of gestational COVID-19, and whether it may have an association to foetal sex and trimester of infection.  The re-analysis of the requested dataset, as well as its integration to other datasets, will not result in additional risk to the participants.  The external collaborators of this project are:  - Prof. Dr. Helena Brentani from Institute of Psychiatry - University of S\\u00c3\\u00a3o Paulo - MSc. Maria Laura Gabriel Kuniyoshi from Institute of Psychiatry - University of S\\u00c3\\u00a3o Paulo (PhD student advised by Helena Brentani and coadvised by David Correa Martins Jr) - Prof. Dr. Sergio Nery Sim\\u00c3\\u00b5es from Federal Institute of Esp\\u00c3\\u00adrito Santo  The collaborators aforementioned do not have access to the download dataset, so they do not need to file a request.\",\n\"May09, 2023\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Martin, Hilary \",\n\"SANGER INSTITUTE\",\n\"The genetic architecture and genomic prediction of serum lipid levels and type 2 diabetes in diverse populations\",\n\"approved\",\n\"This research will study the link between a person\\u00e2\\u0080\\u0099s genetic makeup and their lifelong health in South Asian individuals, and compare it with better studied European individuals. By using the large-scale lipid GWAS in MVP, it will contribute to better ways to predicting serum lipid levels and risk of diseases including cardiovascular diseases for which lipids (such as LDL cholesterol) are important risk factors. We plan to use the information gathered from a South Asian cohort to develop polygenic risk scores that work for British Bangladeshi and British Pakistani people, and compare the performance of similar scores constructed for European subjects. This will contribute to better understanding of the genetic differences between the two major populations, proper characterisation of polygenic score portability for various traits, and improvements in health through better and earlier prediction of disease.\",\n\"Large-scale genome-wide association studies (GWASs) have identified numerous genetic variants that are associated with human common complex traits. Polygenic risk scores (PRSs), which are constructed using GWAS results, have been shown to be successful in identifying individuals at high disease risk. Most of the GWASs and genomic prediction studies have been conducted in European samples, and we plan to evaluate the extent to which PRSs derived from these GWASs are useful for predicting complex traits and diseases in a South Asian cohort (East London Genes and Health; ELGH). We will compare PRSs for common diseases (such as T2D, asthma, and coronary atherosclerosis), as well as for quantitative traits (such as serum lipid levels, height, weight, and BMI) in ELGH versus a European cohort. In order to construct powerful polygenic scores for lipids, we are applying for the access to the summary statistics of lipid GWAS performed in the Million Veteran Program (MVP), which has so-far the largest sample size (around 300,000). We are also applying for the access to the summary statistics of the T2D GWAS meta-analysis which includes the MVP cohort. We will construct PRSs for total cholesterol, LDL cholesterol, HDL cholesterol, triglycerides, and T2D in ELGH using various methods, and evaluate their accuracy in this population and in various subgroups thereof.\",\n\"2020, Apr08\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Manning, Alisa \",\n\"BROAD INSTITUTE, INC.\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease.  Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data.\",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry.  Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community.   However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets.  External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility.\",\n\"2022, Nov07\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Manning, Alisa \",\n\"BROAD INSTITUTE, INC.\",\n\"Exploration of the ORCHID randomized controlled trial (RCT) results\",\n\"closed\",\n\"We will use data from the ORCHID trial to reproduce the analyses performed in the original paper by Self, WH. Based on our process of using a Jupyter notebook created by BioData Catalyst collaborators we decide which additional analyses we will perform and we will revise our dbGap application.\",\n\"We will use data from the PETAL Network: Outcomes Related to COVID-19 Treated With Hydroxychloroquine Among Inpatients With Symptomatic Disease (ORCHID) Trial to reproduce and perform new analyses. In a preliminary step, we will reproduce the analyses performed in the original paper by Self, WH,et al., Effect of Hydroxychloroquine on Clinical Status at 14 Days in Hospitalized Patients With COVID-19, to get baseline results enabling further comparisons. This includes every analysis included in the Clinical Trial statistical analysis protocol. The original statistical analysis plan includes an mITT for the primary outcome only. We will leverage a Jupyter notebook created by BioData Catalyst collaborators to replicate the analysis.\",\n\"2021, Feb10\",\n\"phs002299.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Manning, Alisa \",\n\"BROAD INSTITUTE, INC.\",\n\"Exploration and use of MVP Summary Statistics for diabetes genetics research studies\",\n\"approved\",\n\"The purpose of this application is to explore the MVP summary statistics available in dbGAP and extract MVP summary statistics for various regions and various traits to use in on-going diabetes genetics research studies. We generally intend to perform co-localization analyses with MVP summary statistics and our own genetic summary statistics.\",\n\"The purpose of this application is to explore the MVP summary statistics available in dbGAP and extract MVP summary statistics for various regions and various traits to use in on-going diabetes genetics research studies. We generally intend to perform co-localization analyses with MVP summary statistics and our own genetic summary statistics. We will perform these analyses using cloud computing (the BioData Catalyst platform) and will not redistribute any summary statistics.\",\n\"2022, Feb14\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Male, Victoria \",\n\"U OF L IMPERIAL COL OF SCI/TECHNLGY/MED\",\n\"Changes in number and gene expression of decidual immune cells over gestation and in labour\",\n\"approved\",\n\"Preterm birth and complications from it are the leading cause of death in children under 5 worldwide. Approximately a third  of preterm births are classified as idiopathic \\u00e2\\u0080\\u0093 meaning none of the common reasons for preterm labour are present. Therefore, there is a lot to learn about the mechanisms of preterm labour and doing this will help design therapeutic strategies to delay labour.  Immune cells in the lining of the uterus (the decidua) may be involved in initiating labour and those in the muscle of the uterine wall (myometrium) may promote labour once it has begun. We want to take a closer look at the differences in the immune cells present in the decidua and myomedtrium of women in labour at term, compared to those who are not in labour, to learn more about the normal term labour process. We also want to compare term labouring patients to preterm labouring patients to look for differences in the preterm labour process. \",\n\"Our research interests lie in the immune populations present at the maternal-fetal interface and how they change in the term and preterm labour process. Currently we are studying this at a protein level in the decidua basalis, decidua parietalis and myometrium, and wish to access the phs001886.v1.p1 dataset to analyse complementary data at the scRNAseq level. We will use the R package Seurat to analyse the dataset.  Our goal is to reanalyse the data focusing on the maternal immune cell populations, and to determine whether broad immune cell types identified by Pique-Regi et al could be divided into meaningful subsets. This would be compared to immune cell populations identified by scRNAseq in the first trimester (Vento-Tormo, Nature 2018), to define changes in immune populations over the course of pregnancy and determine whether the subset classifications found in first trimester (dNK1, 2, 3 & M1, 2) are relevant at term. No T cell subsets were defined in the first trimester data set, potentially due to T cells being a small population early in pregnancy. This term data set would give us a higher resolution of the T cell population present in the decidua basalis and chorioamniotic membranes. Lastly, data presented in Pique-Regi et al suggested there may be some changes of immune populations in term labour, compared to term not in labour, and preterm labour compared to the term groups. We hope to confirm this, and identify which specific subsets of cells are involved in these. As well as considering the proportions of cell subsets present in each of these cases, we would also look for differentially expressed genes to see how each subset changes from first trimester to term, and between non-labouring and labouring patients.  The proposed reanalysis of the dataset would provide important information about how immune cells in the decidua differ between the first and third trimesters, and how maternal immune cells in the decidua and myometrium may be involved in labour at term and preterm.\",\n\"2020, Sep03\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Maher, Brion \",\n\"JOHNS HOPKINS UNIVERSITY\",\n\"The Role of Polygenic Alcohol Risk in Opioid Use in Multiple Baltimore Cohorts\",\n\"approved\",\n\"Opioid misuse is associated with serious negative consequences, including increased risk for abuse or dependence, compromised physical and mental health, emergency room visits, and overdose deaths. To date, targeting physician overprescribing has been the predominant public health and policy approach to curtailing the prescription opioid epidemic (Compton et al., 2019); however, opioid misuse is a complex phenomenon and effective prevention of this problem requires the consideration of the multiple pathways through which opioid misuse occurs. One individual specific factor that may influence the likelihood of misusing opioids is genetic predispositions. Indeed, it is well established that opioid misuse is genetically influenced, with heritability estimates ranging from .49-.76 (Sun et al., 2019). Given genetic overlap that has been observed across substance use disorders, it is possible that the genetics associated with problematic alcohol use predicts opioid misuse; however, there has been little research conducted in this area. Accordingly, the present study sought to examine whether polygenic risk for problematic alcohol use predicts opioid misuse in two diverse samples. This work has the potential to aid in risk stratification and the identification of at-risk individuals for screening and more effective preventions and treatments for opioid misuse. \",\n\"This research proposal aims to identify whether the genetics of problematic substance use predicts polysubstance use in 2 samples of predominantly low-income, urban African Americans. Based on summary statistics from the MVP (Levey et al., 2020), we plan on creating polygenic risk scores (PRS) for problematic alcohol, tobacco and other drug use using several approaches (e.g., pruning/thresholding and Bayesian methods). The first sample includes three cohorts of participants that were originally recruited in first grade as part of an elementary school-based intervention and followed until adulthood (~ age 30) (Ialongo et al., 1999). We plan to conduct analyses examining whether problematic alcohol use PRS predicts trajectories of heroin use and non-prescription narcotic use (e.g., Oxycodone), lifetime heroin and non-prescription narcotic use, and opioid use disorders (OUD). For the trajectory analyses, we plan on using person-centered approaches (e.g., growth mixture modeling). To examine whether problematic alcohol use PRS predicts lifetime opioid use and OUD status, we will conduct logistic regression analyses. The second sample includes individuals that were recruited to understand the impact of injection drug use on HIV course (Vlahov et al., 1991). We plan on examining whether PRS for problematic alcohol use predict frequency of injection drug use over time. The analytic plan will involve conducting generalized estimation equation models. We will not conduct analyses that are outside of the proposed data use agreement. The analyses conducted will only involve JHU investigators, including Drs. Brion Maher, Nicholas Ialongo, and Jill Rabinowitz.   We are requesting access to phs001672.v3.p1 and phs001532.v2.p1.  We are requesting a continuation of this project to complete our analyses. \",\n\"2020, Dec21\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ma, Jiantao \",\n\"TUFTS UNIVERSITY BOSTON\",\n\"Causal omics biomarkers of nonalcoholic fatty liver disease\",\n\"approved\",\n\"Nonalcoholic fatty liver disease (NAFLD) is the most common chronic liver disease and a significant risk factor of heart disease. Dietary interventions are crucial to the prevention and treatment of NAFLD. Early detection of NAFLD will allow us to identify individuals at high risk of NAFLD; therefore, we can design effective dietary interventions to prevent the development of NAFLD and NAFLD-associated diseases. We will use information derived from genetic association studies of blood molecules (e.g., genes in white cells) and NAFLD to identify causal biomarkers of NAFLD, which may be used as biomarkers for early identification of NAFLD.  \",\n\"Description: We propose to examine the potential causal associations of omics biomarkers with nonalcoholic fatty liver disease (NAFLD) using Mendelian randomization analysis and colocalization analysis. Background: NAFLD is a common chronic liver disease, which is projected to be the first indication of liver transplantation in 2030. NAFLD is also tightly associated with cardiometabolic diseases and risk factors such as obesity and type 2 diabetes; therefore, NAFLD has been called as the liver manifestation of metabolic syndrome. The prevalence of NAFLD is approximately 25% to 30% in the US. Healthy diet is an important lifestyle factor for NAFLD prevention and among the first line treatment of NAFLD. Early detection of NAFLD is crucial to identify individuals at high risk of comorbidities of NAFLD and allow targeted lifestyle medication such as dietary interventions. A large number of omics biomarkers, e.g., transcriptomic and proteomic biomarkers, have been associated with NAFLD. Recently, a large genome-wide association (GWAS), conducted in 218,595 discovery samples and validated in 108,471 individuals, has identified 77 NAFLD-associated genomic loci. Using the summary results from this NAFLD GWAS and GWAS of omics biomarkers, i.e., quantitative trait loci, we can test which omics biomarkers are likely causal to NAFLD. Such analysis will facilitate the identification of omics biomarkers that can be used for early detection of NAFLD.  Aim: The primary objective of this study is to identify potential causal omics biomarkers such as transcriptomic and proteomic biomarkers of NAFLD.  Method: We will fulfill our objective by analyzing existing expression quantitative loci (eQTLs) and protein quantitative loci (pQTLs) in publicly available datasets (e.g., the MR base; https://www.mrbase.org/) and the summary results of the recent NAFLD GWAS (available through dbGAP under accession number phs001672.v7.p1). We will conduct the Mendelian randomization (MR) analysis, implemented using R SampleMR package, and the colocalization analysis, implemented using R HyPrColoc package. In MR analysis, we will perform the primary analysis using the inverse variance weighted (IVW) method and sensitivity analysis using the MR-Egger method. We will test for potential horizontal pleiotropy by examining the MR-Egger intercept P value. For colocalization analysis, we will use a deterministic Bayesian algorithm to calculate the posterior probability of full colocalization (PPFC). Default prior configuration parameters (prior.1=1e-4 and prior.c=0.02) and threshold of 0.7 for PPFC will be used. \",\n\"2023, Jan25\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Olsen, Morten \",\n\"UNIVERSITY OF COPENHAGEN\",\n\"A genome wide association study for identification of genetic loci associated with venous thromboembolism\",\n\"expired\",\n\"This study aims to Identify common genetic variants associated with venous thromboembolism (VTE) as well as develop/calculate polygenic risk score for venous thromboembolism and compare risk estimates in the highest polygenic risk with known risk factors for VTE.\",\n\"The objective is to identify common genetic variants associated with venous thromboembolism (VTE). We will do this by using summary statistics from VTE together with our main study of a danish cohort.   The main study design is a genome-wide association study, where we will use a large number of single nucleotide polymorphisms (SNPs) to identify genetic regions that associate with VTE and the components and complications to it and related diagnoses. We will include patients with a known pathogenic variant as well as patients where the pathogenic variant is unknown. We will use expected genotype counts, and the logistic model will be adjusted for sex, age, principal components.    Using GWAS summary statistics, we will develop/calculate polygenic risk score (PRS) for venous thromboembolism and compare the risk of high PRS (e.g. top 5 %) with the risks associated with known risk factors (e.g. Factor V Leiden, cancer, major surgery, oral contraceptives/hormone re-placement therapy). We will also explore how this PRS associates with known for blood cell and coagulation-markers. For these analyses, we will be using VTE summary statistics from the Million Veteran Program in a meta-analysis with similar summary statistics from other biobank datasets. \",\n\"Apr11, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"MANICHAIKUL, ANI \",\n\"UNIVERSITY OF VIRGINIA\",\n\"Systematic integration of multi-omics data for the study of coronary artery disease and subclinical atherosclerosis\",\n\"approved\",\n\"Large scale genetics studies have identified numerous genetic variants associated with disease. The goal of our project is to combine prior genetic findings with new molecular data in order to identify the molecular mechanisms of disease.\",\n\"Coronary artery disease (CAD) is a leading cause of death and disability worldwide. Large-scale genome-wide association studies (GWAS) of CAD from international consortia, including CARDIoGRAM, C4D and the UK Biobank have identified over 150 CAD-associated genetic loci, but the biological mechanisms and pathways implicated by GWAS are still not fully understood. Multi-omics integration combining GWAS results with molecular data provides a useful avenue to identify biomarkers and elucidate potential causal molecular mechanisms of the disease. Our group has access to molecular \\u00e2\\u0080\\u0098omics data through our work in the NHLBI Trans-omics for Precision Medicine (TOPMed) program. These data include transcriptomics, proteomics, metabolomics and methylomics on circulating samples from the population-based Multi-Ethnic Study of Atherosclerosis [MESA]. Currently, we are applying for MVP and CHARGE GWAS summary statistics from dbGaP with the goal of integrating those GWAS summary data with molecular QTL resources and identifying candidate molecular targets downstream of the GWAS signals. Our work will help to elucidate biological mechanisms of cardiovascular disease traits previously studied by GWAS.\",\n\"2023, Feb24\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Luo, Yiming \",\n\"COLUMBIA UNIVERSITY HEALTH SCIENCES\",\n\"Mendelian randomization of autoimmune and cardiovascular diseases\",\n\"approved\",\n\"Both autoimmune and cardiovascular diseases are significant public health burden. This study is to investigate their casual relationship and shared susceptibility using population genetic data. \",\n\"Both autoimmune and cardiovascular diseases are significant public health burden. Recent studies have suggested that inflammation due to immune dysregulation is a causal factor of cardiovascular diseases. This study is to perform Mendelian randomization and other post-GWAS analyses (such as genetic correlation analysis, colocalization, cross-phenotypic meta-analysis) among autoimmune and cardiovascular diseases. This study will only use summary statistics but not individual-level genotype data. All the analyses will be performed in password-protected desktops/laptops and only study investigators can access the data. \",\n\"Mar14, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Lu, Yingchang \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Cardiometabolic trait associations for variants associated with total body fat percentage and fat-free mass index.\",\n\"approved\",\n\"In order to understand the physiology underlying body fat mass and body fat-free mass regulations, we conducted a genetic study on body fat percentage and fat-free mass index among >400,000 individuals of European ancestry.  We identify 1,137 genetic variants associated with these two traits. We intend to understand the effects of these 1,137 variants on cardiometabolic traits and diseases.\",\n\"Large-scale genome-wide association study (GWAS) of body mass index (BMI) have recently identified 716 independent variants [PMID: 30124842], which together explained 5% of the variance of BMI. Although BMI is a widely used indicator of overweight and obesity, it does not distinguish body fat mass from lean mass. In order to identify novel loci and gain a deeper understanding of genetic regulations of body adiposity and lean mass, we conducted a genome and exome-wide association studies of both body fat percentage and fat-free mass index among >400,000 individuals of European ancestry. In total, we identified 1,137 independent variants associated with these two traits at P < 5E-9. We identified 6 clusters among the 1,137 identified variants, based on their association patterns with 5 standard anthropometric traits, and have found that these clusters show discordant genetic associations with various cardio-metabolic outcomes and continuous traits. We would therefore like to request association results of cardiometabolic traits including lipids and other traits for 1,137 variants analyzed on MVP participants. We are not going to combined requested data with other data. This project will not create any additional risks to participants.\",\n\"2019, Oct09\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Loos, Ruth \",\n\"ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI\",\n\"The genetic susceptibility to type 2 diabetes across diverse ancestries\",\n\"closed\",\n\"The Mount Sinai Biobank, BioMe, is an electronic health record linked biorepository composed of over 60,000 individuals who seek care through the Mount Sinai Medical Center. A majority of the BioMe participants are of non-European ancestry, making us well situated to study traditionally under-represented populations in genetics. The UK Biobank is a large biobank in the United Kingdom that is composed predominantly of individuals of European ancestry, and contains ~5% non-European ancestry individuals. We are requesting access to the Million Veteran Program type 2 diabetes GWAS summary statistics for African Americans, Europeans, Hispanic Americans and the multi-ethnic cohort. Using the summary statistics, we will generate type 2 diabetes polygenic risk scores in an ancestry specific manner in BioMe and the UK Biobank, and use them to test the association for diabetes and related traits.\",\n\"The majority of genome-wide association studies (GWAS) have been done in individuals of European ancestry, hindering our ability to create polygenic risk scores (PRS) in other ancestries. We are requesting access to the African American, European, Hispanic American and multi-ethnic type 2 diabetes (T2D) GWAS summary statistics from the Million Veteran Program (MVP) to generate ancestry-specific PRSs and examine their role in the Mount Sinai Biobank, BioMe, and the UK Biobank (UKBB). BioMe is a multi-ethnic electronic medical record linked biorepository of over 60,000 individuals. Within BioMe, we have > 32,000 whole exome sequencing samples, the majority of which are from individuals of African American and Hispanic ancestry. The UKBB is a large biobank in the United Kingdom that is composed predominantly of individuals of European ancestry and contains ~5% non-European ancestry individuals. Within each biobank, our objectives are:  1. To generate ancestry specific type 2 diabetes PRSs and test their association with T2D in each each ancestry  2. To examine the association of rare variants in known genes known to affect T2D risk and the risk of T2D in each ancestry 3. To investigate the interaction between the PRSs and rare variants risk of T2D and  4. Repeat analyses in 1-2 for diabetes related outcomes, such as retinopathy, neuropathy and nephropathy, as well as diabetes related biomarkers, such as glucose and hemoglobin A1C. 5. Repeat analyses 1-4 using the multi-ethnic summary statistics for comparison with the ancestry specific results. We are not requesting individual level data. We do not expect to collaborate with other institutions.   NOTE: We are only requesting summery statistics related to this paper: Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis. Vujkovic M, Keaton JM, Lynch JA, Miller DR, Zhou J, Tcheandjieu C, Huffman JE, Assimes TL, Lorenz K, Zhu X, Hilliard AT, Judy RL, Huang J, Lee KM, Klarin D, Pyarajan S, Danesh J, Melander O, Rasheed A, Mallick NH, Hameed S, Qureshi IH, Afzal MN, Malik U, Jalal A, Abbas S, Sheng X, Gao L, Kaestner KH, Susztak K, Sun YV, DuVall SL, Cho K, Lee JS, Gaziano JM, Phillips LS, Meigs JB, Reaven PD, Wilson PW, Edwards TL, Rader DJ, Damrauer SM, O'Donnell CJ, Tsao PS; HPAP Consortium; Regeneron Genetics Center; VA Million Veteran Program, Chang KM, Voight BF, Saleheen D.  We are not requesting individual level data. \",\n\"Jul13, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Liu, Jin \",\n\"NATIONAL UNIVERSITY OF SINGAPORE\",\n\"Statistical methods for genetic studies in health-related outcomes\",\n\"closed\",\n\"Genetic studies have been developed rapidly over past twenty years, including genome-wide association studies (GWAS). Rigorous statistical methods will be developed to modulate the relationships among multiple GWAS and a scalable algorithm will be developed to handle these data sets. We plan to use applied data sets in dbGaP to demonstrate the usefulness of our methods. \",\n\"Genetic studies have been developed rapidly over past twenty years, including genome-wide association studies (GWAS).  1. Objective: A rigorous statistical method will be developed to modulate the relationships among multiple health-related traits and a scalable algorithm will be developed to handle admixture of Hispanic population to boost findings of risk variants or factors for health-related traits, and lung disease trait. 2. Study design: We plan to use machine learning techniques following statistical principles to explore the unique property of these data sets, which will potentially produce solid results for interpretation. A statistically rigorous method will be developed to handle large genomic data as current state-of-art for these data set is lack of scalability. We will also explore the integrative analysis using this data set with other external genomic data set to boost our findings. 3. Analysis plan: We plan to analyze multiple health-related traits and lung disease traits in the datasets requested. 4. The analysis of this data set and methodology development will be conducted in my lab.\",\n\"2019, Jan28\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Liu, Dajiang \",\n\"PENNSYLVANIA STATE UNIV HERSHEY MED CTR\",\n\"method development for statistical fine-mapping analysis using multiple traits. \",\n\"approved\",\n\"We will develop software tools to identify genetic variants that elevate the risk for human diseases. Identifying causal variants is of utmost importance to understand the underlying disease biology and gain insights on how to develop therapeutics to reverse the effects of risk variants, which can lead to treatment. Here, we expect to develop methods to jointly analyze multiple health outcomes, and more accurately identify causal variants.  \",\n\"GWAS to date have been extremely successful for identifying genetic regions associated with various complex traits. Yet, due to extensive linkage disequilibrium, the causal variants from most GWAS loci remain unclear. Statistical fine mapping methods are often used to pinpoint causal variants and assign probability to each variant measuring how likely it is causal. Current statistical fine. mapping methods have limited success. Due to shared genetic basis between traits, borrowing strength from multiple traits will improve the estimates of genetic effects and in turn improve fine mapping. For this project, we seek to develop useful methods to improve fine-mapping combining summary statistics from multiple traits. We will release results and distribute software implementing the proposed methods. \",\n\"Feb02, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Linnstaedt, Sarah \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"Assessing genetic overlap and protective factors for common co-morbid posttraumatic sequelae\",\n\"closed\",\n\"Results from the Million Veterans Program will be used to assess the genetic overlap and protective factors influencing vulnerability to posttraumatic stress, chronic pain, and depression using polygenic risk scores in a civilian trauma cohort.\",\n\"Clinical comorbidity between different psychiatric symptoms following trauma is common. The purpose of our proposed research is to investigate the genetic overlap between vulnerability to posttraumatic stress (and relevant symptom clusters), chronic pain, and depression using polygenic risk scores. Polygenic risk scores will be generated using PRSice polygenic risk score software, summary statistics from the Million Veteran Program and other large genome-wide association studies, and individual genetic information from our emergency department cohort. Statistical analyses will be used to determine how well polygenic risk scores predict outcomes and whether protective factors (e.g. social support, socioeconomic status) offset this genetic risk. Datasets will not be combined but will be used in addition to other GWAS summary information.\",\n\"2020, Jun10\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"McCarthy, Mark \",\n\"UNIVERSITY OF OXFORD\",\n\"Deconstructing mechanisms involved in type 2 diabetes predisposition using trait association pattern mapping\",\n\"closed\",\n\"We lead international efforts to use genetics to understand the processes that are involved in the development of type 2 diabetes. One very useful strategy for doing this is to take the DNA sequence differences that we have shown to be associated with type 2 diabetes risk, and examine the effects of those same DNA differences on other diabetes-related measurements such as body mass index, and lipid levels. These analyses are most powerful when the data sets are large and it is for this reason that we request access to the recent data made available by the lipid analysis in the Millions Veterans Project. \",\n\"We are interested in what leads to type 2 diabetes (T2D) and related conditions. We have led international efforts that have identified over 400 type 2 diabetes (T2D) risk loci. Most map to non-coding sequence and mechanisms of action are often unclear. One strategy for defining the physiological processes through which these variants act is to examine their patterns of association to other diabetes-related metabolic and anthropometric traits. Over the past years, we have shown that, by integrating information on quantitative trait associations and tissue-specific regulatory annotation, genetic variants influencing type 2 diabetes (T2D) predisposition can be characterised in terms of the pathophysiological processes through which they operate. This allows a deconstruction of T2D pathophysiology that addresses phenotypic and clinical heterogeneity, promotes mechanistic insights, and reveals novel translational opportunities. We have extended this effort to the T2D-risk variants discovered through our expanded meta-analyses (both European-specific and trans-ethnic) in over a million individuals. For this, we are seeking access to summary statistics for SNPs associated with lipid traits.\",\n\"Nov29, 2018\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"McVean, Gil \",\n\"GENOMICS LIMITED\",\n\"Integrative analysis of genetic association data for individual genome interpretation and translational biological research\",\n\"closed\",\n\"We will combine information from many different studies in dbGaP, and many other studies in our own database, to learn even more about the biology of health and sickness than can be derived from any individual study. We will develop new statistical methods to help do this. We will compare our results with other scientific studies in our database in many different ways, for example by looking at which genes are turned on and off in different cells, to better understand how DNA varies in different people. Ultimately we hope that our work will improve two things about modern medicine. Firstly, we hope that drug companies will be better informed about choosing what to concentrate on when making new medicines. And secondly, that an individual person\\u00e2\\u0080\\u0099s DNA can be used to help in making a diagnosis, or choosing treatment options when they go to the doctor\",\n\"There has been limited translation of important GWA study findings for complex diseases and other phenotypes into healthcare practice. We wish to use certain dbGaP datasets as part of our ongoing efforts to build and integrate a curated database of GWAS and related biological and phenotypic data. As a company, we hope to obtain scientific and commercial value by using insights from the database to develop proprietary analytical tools, and performing analyses of the database to better inform and improve healthcare eg. improvements to drug development and genomics-based precision medicine.   Our planned analyses will rely on systematic processing of dbGaP datasets into summary (non-individual level) statistics, prior to incorporation into our database. We will then interrogate the database to look for association between genetic markers and different combinations and summaries of phenotypic data. Where appropriate, we will perform a GWAS for each phenotype, and a phenotype\\u00c2\\u00ad-wide association study at each genetic variant. Using statistical methods we will summarize the evidence that each genetic variant/locus has an association with available phenotypes. The data generated will be integrated across the genome and phenotypes to build a model showing how genetic variation shapes human biology. We will also use the data to develop, refine, and optimise robust algorithms that form the basis of our analytical tools. As part of this, some of the datasets will be used to compute correlations between genetic markers, especially in an ancestry specific manner- an output typically referred to as linkage disequilibrium maps. These linkage disequilibrium maps can then be used to better conduct our analyses in a manner that properly accounts for variant correlations across each genetic ancestry group. Furthermore, dimensionality reduction techniques such as principal component analysis may be applied to the data in order to summarise the genetic space into a lower dimension map, a process typically used for genetic ancestry assignment. We expect that our tools will be utilised in research laboratories, clinics, and elsewhere. Our tools, used alone or with the database, could be used to generate precision medicine analysis for many traits, aiding interpretation of extant clinical diagnostic information.  Only the collaborators named in our dbGaP application (ie. our staff) will have access to our database. We will absolutely respect all data use certifications and data use limitations when conducting our analyses. In the case of disease-specific limitations, we will limit our analyses accordingly using technical and administrative means. \",\n\"2019, May20\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"McVean, Gil \",\n\"GENOMICS LIMITED\",\n\"Integrative analysis of genetic association data for individual genome interpretation and translational biological research - part 2\",\n\"approved\",\n\"We will combine information from many different studies in dbGaP, and many other studies in our own database, to try to learn even more about the biology of health and sickness than the individual studies can alone. We will develop new statistical methods to help do this. We will compare our results with other scientific studies in our database in many different ways, for example by looking at which genes are turned on and off in different cells, to better understand how DNA varies in different people. Ultimately we hope that our work will improve two things about modern medicine. First that drug companies will be better informed about choosing what to concentrate on when making new medicines. And secondly, that an individual person\\u00e2\\u0080\\u0099s DNA can be used to help in making a diagnosis, or choosing treatment options when they go to the doctor.\",\n\"There has been limited translation of important GWA study findings for complex diseases and other phenotypes into healthcare practice. We wish to use certain dbGaP datasets as part of our ongoing efforts to build and integrate a curated database of GWAS and related biological and phenotypic data. As a company, we hope to obtain scientific and commercial value by using insights from the database to develop proprietary analytical tools, and performing analyses of the database to better inform and improve healthcare eg. improvements to drug development and genomics-based precision medicine.   Our planned analyses will rely on systematic processing of dbGaP datasets into summary (non-individual level) statistics, prior to incorporation into our database. We will then interrogate the database to look for association between genetic markers and different combinations and summaries of phenotypic data. Where appropriate, we will perform a GWAS for each phenotype, and a phenotype\\u00c2\\u00ad-wide association study at each genetic variant. Using statistical methods we will summarize the evidence that each genetic variant/locus has an association with available phenotypes. The data generated will be integrated across the genome and phenotypes to build a model showing how genetic variation shapes human biology. We will also use the data to develop, refine, and optimise robust algorithms that form the basis of our analytical tools. As part of this, some of the datasets will be used to compute correlations between genetic markers, especially in an ancestry specific manner- an output typically referred to as linkage disequilibrium maps. These linkage disequilibrium maps can then be used to better conduct our analyses in a manner that properly accounts for variant correlations across each genetic ancestry group. Furthermore, dimensionality reduction techniques such as principal component analysis may be applied to the data in order to summarise the genetic space into a lower dimension map, a process typically used for genetic ancestry assignment. We expect that our tools will be utilised in research laboratories, clinics, and elsewhere. Our tools, used alone or with the database, could be used to generate precision medicine analysis for many traits, aiding interpretation of extant clinical diagnostic information.  We will share results from various aspects of our research work e.g. via publicly accessible repositories, following the pace of our analyses. Our hope is that this will enable the scientific community ready access to foundational analyses that they can draw insight from and integrate into the body of work of the wider scientific community.  Only the collaborators named in our dbGaP application (ie. our staff) will have access to our database. We will absolutely respect all data use certifications and data use limitations when conducting our analyses. In the case of disease-specific limitations, we will limit our analyses accordingly using technical and administrative means.\",\n\"2021, Sep19\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"McVean, Gil \",\n\"GENOMICS LIMITED\",\n\"Robustness of genetic association and polygenic risk scores to non-European ancestries\",\n\"approved\",\n\"Genetic data has the potential to transform health care, by better predicting diseases, understanding and recommending treatment options and better prevention. This is particularly critical for cardiovascular diseases, for which lifestyle alterations and drugs (e.g. statins) are available for at-risk individuals.  However, much of the data underpinning this revolution have been generated in individuals of European descent. These data cannot be readily used in non-European populations, which limits broad adoption and widens an already existing gap in access to modern genomic technologies. With this project, we are interested in assessing how these genetic predictors can be used in individuals of diverse ancestry. We want to improve the design of these tools so that they become more precise, and usable by a wider range of individuals.\",\n\"There is a growing sense in the genomics community that genetic-based prediction can provide effective information in health care. In particular, polygenic risk scores (PRS) have gathered a lot of attention in the scientific community. PRS are designed to aggregate information from a large number of genetic variants, which, once combined, provide effective predictions on a wide range of traits. We are particularly interested in developing these PRS for diseases that are actionable and/or have ongoing screening programmes which makes diabetes, eye diseases, osteoporosis/fractures, and cardiovascular diseases (lipids studies, hypertensions, myocardial infarction) major foci of this application.   However, opportunities for improved care also exist for a broader range of human traits and diseases, for example, autoimmune conditions for which wrong diagnosis or lack of diagnosis severely impact a large number of individuals. Genome-wide association (GWA) studies provide the data supporting the design of these PRS. However, PRS typically rely on keeping the pattern of linkage disequilibrium (LD) constant. Because the majority of genetic studies have been performed in individuals of European descent, these PRS are not readily exportable to other populations, which limits their widespread adoption.  Given these limitations, we are keen to explore the robustness of PRS and association summary statistics across European and non-European populations. For the common diseases for which genetic prediction is a relevant and practical opportunity, this work will be performed in three stages: We will initially refine the construction of PRS by utilising summary statistics data from studies performed in populations of European descent. We will download data from genome-wide association (GWA) studies with individual-level genotype in European and non-European cohorts and generate PRS data, comparing association statistics between European and non-European data. We will explore alternative PRS generation techniques that leverage individual-level data, such as machine learning techniques (LASSO, neural networks) and understand how these prediction algorithms are impacted by varying ancestries.  This analysis will help us assess the robustness of PRS and improve their design in order to be effective in a broader set of populations. The design of these PRS will combine the novel data accessed as part of this project with the summary statistics that we have previously aggregated for a range of genetic studies. Where permissible in accordance with data use terms, we also combine individual-level data (both from dbGaP and other sources) to understand the interplay between genetics and phenotype with more granularity.  We intend to publish various aspects of our research and share results accordingly. We have already presented some of our prediction work at international conferences (e.g. ASHG 2019) and our paper quantifying the role of genetics for the prediction of cardiovascular disease is currently in press (Validation of an Integrated Risk Tool, Including Polygenic Risk Score, for Atherosclerotic Cardiovascular Disease in Multiple Ethnicities and Ancestries in The American Journal of Cardiology). We will follow a similar process across diseases, for conference and journal submissions, following the pace of our research work.  Only the collaborators named in our dbGaP application (i.e. our staff) will have access to our database. We will absolutely respect all data use certifications and data use limitations when conducting our analyses. In the case of disease-specific limitations, we will limit our analyses accordingly using technical and administrative means.  The IRB certificate is expected to be renewed in August, and the updated certificate will be uploaded as soon as it has been obtained.\",\n\"2021, Jul05\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ross, Elsie \",\n\"STANFORD UNIVERSITY\",\n\"Development and validation of peripheral artery disease polygenic risk scores\",\n\"approved\",\n\"Peripheral artery disease (PAD) affects 8-12 million Americans and has an estimated annual health care cost of $21 billion in the United States. PAD increases risk of heart attack, stroke, and sudden cardiovascular death but often goes undiagnosed. Because prevention measures can significantly improve long-term outcomes in PAD, low rates of diagnosis represent missed opportunities to prevent costly and deadly consequences of disease. Thus, there is a critical need to develop methods to improve PAD risk assessment, diagnosis and treatment. Our overall objective in using the MVP and ARIC data sets is to develop a first ever PAD-specific polygenic risk score and evaluate the utility of incorporating this score into existing clinical data. A polygenic risk score will allow us to combine multiple risk factors identified in the genome into a summary score that can be used to identify who is at high or low risk of PAD.  \",\n\"Peripheral artery disease (PAD) affects 8-12 million Americans and has an estimated annual health care cost of $21 billion in the United States. PAD confers a 2 to 6-fold risk of major adverse cardiovascular events (MACE) but often goes undiagnosed. Because secondary prevention measures can significantly improve long-term outcomes in PAD, low rates of diagnosis represent missed opportunities to prevent costly and deadly consequences of disease. Thus, there is a critical need to develop methods to improve PAD risk assessment, diagnosis and treatment. Our overall objective in using the MVP and ARIC data sets is to develop a first ever PAD-specific polygenic risk score, validate its performance and evaluate the utility of incorporating this score into an EHR-based PAD risk assessment tool.  PAD-specific PRSs have not previously been developed due to difficulty in identifying risk alleles unique to PAD. Recently the MVP GWAS for PAD was published and is of unprecedented size, making it now possible to calculate potentially useful PAD-specific PRSs. Summary statistics for PRS development will be derived from the latest PAD GWAS study, conducted using data from the Million Veterans Program, which includes >31,000 PAD cases and >211,000 controls. Using the MVP GWAS study we will develop PAD PRSs using Bayesian polygenic prediction algorithms. Fit of PRSs will be assessed with AUC and pseudo-R2 metrics. These data will then be combined with our existing electronic health record model to evaluate whether PRS improves PAD risk classification. After developing multiple well-performing PRSs we will validate their performance and utility using data from The Atherosclerosis Risk in Communities (ARIC) Study.\",\n\"Apr06, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Nievergelt, Caroline \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Psychiatric Genomics Consortium for PTSD meta-analysis\",\n\"approved\",\n\" Up to 50% of the variance determining who develops Posttraumatic Stress Disorder following a severe trauma may be genetic; however, most successful analyses to date suggest that risk arises from multiple common genetic variants of small effect size, and that very large sample sizes will be needed for detection. This proposal aims to uncover the genomic architecture of PTSD through large-scale, collaborative genome-wide association studies. Identifying the pathways underlying PTSD will lead to an improved neurobiological understanding, enhanced prevention, and improved treatment of this debilitating and prevalent syndrome.\",\n\"Posttraumatic stress disorder (PTSD) occurs only in vulnerable individuals after exposure to severe traumatic events. This risk is due, in part, to 40-50% heritability of differential vulnerability. Due to increasing collaborations across the field of PTSD genomics and the advent of new analytical tools, it is a very exciting time for PTSD genetic risk discovery. The purpose of this application is to facilitate meta-analyses of genome-wide association study (GWAS) data for symptoms and diagnosis of PTSD. We propose to conduct large-scale meta-analyses through the PTSD group of the Psychiatric Genomics Consortium (PGC). Currently our multi-ethnic data collection includes genotypes from 60 studies with a total N of over 200K combined cases and trauma-exposed controls. We hypothesize that with an increased sample size and deeper phenotype characterization, the PGC-PTSD will accelerate our current understanding of the genetic architecture of PTSD. Our primary Aim is to expand the PGC-PTSD data resource to produce the most powerful GWAS analyses. Addition of the MVP PTS symptoms dataset is a next step. Only those named on the project or appropriate personnel will have access to the data and derivative data for analysis. Only meta-analysis results will be published and shared broadly with the research community. Meta-analyses across GWAS on PTS symptoms and PTSD diagnoses will be performed using the standard PGC pipeline RICOPILI on the LISA server. GWAS downstream analyses will be performed using the FUMA pipeline. Lastly, we will use polygenic risk scores (PRS) generated using PRcise to provide insights into relationships to other traits and advance causal inference. Identifying the genetic pathways underlying PTSD will lead to improved neurobiological understanding, enhanced prevention, and improved treatment of this debilitating and prevalent syndrome. \",\n\"Feb10, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Niarchou, Maria \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"The medical and genetic correlates of short versus long buprenorphine treatment medication in the electronic health record\",\n\"approved\",\n\"Buprenorphine is one of the three medications approved by the FDA for the treatment of Opioid Use Disorder (OUD). Despite the benefits associated with buprenorphine in the treatment for OUD, retention remains poor. It is unknown whether and how genetic risk in the form of polygenic scores (PGS) for OUD is associated with the duration of buprenorphine treatment. Using the summary statistics from the GWAS of OUD (Zhou et al. 2020) we will be able to test whether there is a role for genetic liability of OUD in the duration of buprenorphine treatment. The findings will help us better understand the genetic mechanisms underlying buprenorphine treatment duration. We would like to note here that although genetics can play a role in OUD, the primary cause of OUD is environmental rather than genetic. Moreover, although PGS can be used as a tool to better understand the genetic mechanisms underlying BTD, there currently remain many technical and ethical limitations associated with the use of PGS in the clinical context. We will include these statements in our manuscript.\",\n\"Buprenorphine, which is a partial opioid agonist, is one of three medications approved by the U.S. Food and Drug Administration (FDA) for the treatment of opioid use disorder (OUD). Studies indicate that buprenorphine treatment is associated with reduced opioid use and higher treatment retention rates compared with placebo. Moreover, patients undergoing buprenorphine treatment have lower risk for all cause and overdose mortality and fewer opioid-related emergency department visits. Despite the benefits of buprenorphine, retention in buprenorphine treatment remains poor. Previous research on the impact of co-occurring somatic, psychiatric and other substance use disorders on buprenorphine treatment duration (BTD) has yielded mixed results. For example, some studies have found that, among other factors, psychiatric diagnosis is associated with greater odds of retention. However, other studies indicate that higher retention is associated with lower rates of psychiatric diagnoses, including substance use disorder diagnoses, while other studies have found no association between retention and comorbidity profiles. In order to increase buprenorphine retention rates and treatment efficacy for OUD, further research is needed to obtain a better understanding of the characteristics of patients who stay in buprenorphine treatment. Another factor that may differentiate patients who are retained in buprenorphine treatment is the genetic liability for physical and mental health conditions. There are now large GWAS for OUD, as well as traits closely related to OUD, including chronic pain, problematic alcohol use (PAU), depression, loneliness, and externalizing psychopathology. Taking into account that thousands of genetic variants of small effect sizes contribute to complex traits, the aims of our study were to test the association between somatic and psychiatric comorbidities and BTD, and to determine whether PGS for OUD and other OUD related traits contribute to the variation in BTD. The details of our study are the following:  Design: Case-control study.  Setting: Vanderbilt University Medical Center\\u00e2\\u0080\\u0099s (VUMC) electronic health records (EHR). Participants: N=6,049 individuals with buprenorphine prescription that lasted for less than 6 months (short-BTD) and N=1,919 individuals with buprenorphine prescriptions that lasted for at least 6 months (long-BTD).  Exposures: Phenotypes were based on mapping International Classification of Disease versions 9 and 10 codes.   We would like to request the GWAS summary statistics of OUD from Zhou et al. 2020 to  generate PGS for OUD and apply them to 545 individuals that have been prescribed buprenorphine and have genotype data available from the Vanderbilt Biobank.  Main outcomes: Long versus short BTD.  Our findings will help us better understand the factors that may be involved on whether a patient remains on buprenorphine treatment or not. We would like to note here that although genetics can play a role in OUD, the primary cause of OUD is environmental rather than genetic. Moreover, although PGS can be used as a tool to better understand the genetic mechanisms underlying BTD, there currently remain many technical and ethical limitations associated with the use of PGS in the clinical context. We will include these statements in our manuscript.   \",\n\"2022, Aug24\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ng, Maggie \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Genetic study of cardiometabolic diseases in diverse populations\",\n\"approved\",\n\"Cardiometabolic diseases and associated complications disproportionally affect minority populations in the US. These diseases may be partly contributed by shared genetic risks. Our study will examine genome-wide association data in diverse populations from Million Veteran Program and other studies to identify genetic variants associated with these diseases. We will aggregate the genetic risk scores of these diseases in single or multiple ethnicity to test for their clinical utility in disease prediction and prognosis. We will identify and evaluate the putative functions of genetic variants that are shared among multiple cardiometabolic diseases. \",\n\"Cardiometabolic diseases and related traits, such as type 2 diabetes, hypertension, dyslipidemia, obesity, coronary artery disease, stroke, chronic kidney disease, end-stage renal disease, retinopathy, neuropathy, and non-alcoholic fatty liver disease are highly heritable. Many of these cardiometabolic diseases and traits cluster within individuals and share genetic risks. Racial disparity has been observed in these diseases which disproportionally affect individuals of non-European ancestry. Our research program is interested in the genetic architecture of cardiometabolic diseases and related traits in diverse populations and actively participate in several related genome-wide association study (GWAS) consortia. Our objectives in this project are: 1) to perform ancestry-specific and trans-ancestry GWAS meta-analysis of cardiometabolic diseases and traits in diverse populations; 2) to develop ancestry-specific and trans-ancestry polygenic risk scores of cardiometabolic diseases and traits and evaluate their clinical utility; 3) to perform multi-trait GWAS and variant-trait clustering analyses to identify pleiotropic loci for cardiometabolic diseases and traits in diverse populations. These analyses will combine summary statistics generated from Million Veteran Program (MVP) and other studies, wherever available. This should not pose additional risks to participants. Results will be shared broadly in conference presentations and published manuscripts. \",\n\"2020, Sep03\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Natarajan, Pradeep \",\n\"BROAD INSTITUTE, INC.\",\n\"Analysis of publicly available genomic data and cardiometabolic phenotypes\",\n\"approved\",\n\"The goals of our study is to use genetic and non-genetic factors, in concert with downstream biologic mediators, to characterize the determinants of cardiometabolic disease.\",\n\"We seek to identify the genetic and non-genetic factors for cardiometabolic disease, to identify disease subtypes, and to develop personalized treatment strategies to manage and prevent disease. These studies involve cardiovascular and metabolic traits and outcomes. Exposures include environmental, dietary, lifestyle, and genetic factors. Genetic factors include DNA sequence variation derived from array genotypes, targeted genotyping, targeted sequencing, whole exome sequencing, and whole genome sequencing. We seek to integrate orthogonal \\\"-omics,\\\" such as gene expression / transcriptomics, metabolomics, and proteomics, to better inform genetic analyses and characterize downstream mediators of disease.  We will conduct genotype-phenotype associations, include methods for rare and common variation, using both publicly available annotation sets and information from orthogonal \\\"-omics\\\" datasets generated within the same individual. In addition, genetic data may be used (subject to consent restrictions) for (a) variant summary statistics, (b) common controls in association tests, (c) population genetic studies.. Imputation would only be performed for specific variants of interest for replication of non-sequenced participants so we will not be generating an imputation panel. Datasets from the UK Biobank and FinnGen study will be combined with datasets provided by dbGaP for various analyses to fulfill the aims of this research.  Genetic and non-genetic data will not be used to identify individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Furthermore, when an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish scientific findings and, generally, broadly share any findings from these studies with the scientific community.\",\n\"2021, Apr01\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Mullins, Niamh \",\n\"ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI\",\n\"Genetic and causal relationships between problematic alcohol use, depression and suicidality\",\n\"approved\",\n\"Problematic alcohol use, depression and suicidality commonly co-occur and represent substantial public health problems. This research project aims to leverage genetic analyses and data from the Million Veteran Program and the Collaborative Study on the Genetics of Alcoholism (COGA) cohort, to investigate the genetic, biological and potentially causal relationships between these phenotypes. \",\n\"Objectives: The objectives of the proposed research are to understand the genetic, biological and causal relationships between alcohol, depression and suicide-related phenotypes. Study design: We request approval to access summary statistics from published genome-wide association studies (GWAS) of major depression and problematic alcohol use conducted in the Million Veteran Program study. These GWAS results will be used in the Collaborative Study on the Genetics of Alcoholism (COGA) cohort, to calculate polygenic risk scores and genetic correlations, assess causal relationships using Mendelian randomization, and conduct GWAS meta-analyses. Analysis plan: Phenotypes of interest in the COGA study will be alcohol dependence, alcohol use disorder, problematic alcohol use, depression and suicide-related phenotypes (such as deliberate self-harm, suicidal ideation and suicide attempt). Analyses will be conducted in individuals of European and African-American ancestries, controlling for ancestry, relatedness and other relevant cohort/analysis-specific covariates. Analyses will be performed using the latest statistical genetics software, such as PRS-CS (for polygenic risk scoring), LD score regression (for genetic correlations), GSMR and others (for Mendelian randomization) and METAL (for GWAS meta-analyses). Data will be stored securely and with restricted access on the computing server at the Icahn School of Medicine at Mount Sinai; they will be used for research purposes only, and no attempt will be made to identify participants. This research will be conducted in collaboration with Dr. Peter Barr from SUNY Downstate Medical Center, who has prior dbGaP approval for these data.  \",\n\"2022, Mar28\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Momenan, Abdolreza \",\n\"NIH\",\n\"PRS study of substance use disorders\",\n\"approved\",\n\"Genome wide association studies (GWAS) identify genetic variants associated with a trait such as problematic substance use. We will use two GWAS from dbGAP which targeted nicotine dependence and problematic alcohol use to generate individualized genetic risk scores for these traits in other samples. We will examine whether these risk scores are related to behavior (e.g. age of onset and frequency of substance use, and measures of impulsivity) or brain features detected using MRI, including measures of brain structure (e.g. cortical and subcortical grey matter volume and white matter tract integrity) and function.\",\n\"Polygenic risk scores (PRS) are a powerful tool to examine the genetic liability of a specific trait in another population. Our aim is to investigate the genetic liability to substance use disorders (e.g., alcohol) in population-based cohorts in relation to behavior measures and to identify brain intermediate phenotypes measures associated with greater genetic risk for problematic alcohol use. PRS will be create using the PLINK 2 software based on the results of the Genome Wide Association Studies (GWAS) problematic alcohol use (PMID 32451486 - phs001672.v3.p). These PRS will be regressed onto behavioral phenotypes (e.g., substance use measures including age of onset and frequency of use, and impulsivity measures) and onto brain measures including structure (cortical and subcortical measures and diffusivity) and function. This project does not require individual-level data from the mentioned studies and it is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry.\",\n\"2022, Jun10\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Miller, Melissa \",\n\"PFIZER, INC.-RESEARCH &amp; DEVELOPMENT DIV\",\n\"Evaluating SNP-Phenotype Associations from Million Veteran's Program Summary Statistics to Support Confidence in Rationale for Therapeutic Targets\",\n\"approved\",\n\"Pfizer Research and Development utilize many tools to enable novel target discovery across many different diseases, including human genetics. To that end, we request access to the summary statistics data associated with this dbGAP study in order to understand how single nucleotide polymorphisms, and the genes in which they are contained, are associated to disease to support therapeutic programs, as well as to understand other endpoints associated with our targets that might point to safety or efficacy concerns. Finally, we are interested in understanding the pathways that are linked to disease through the evaluation of genes associated to disease from GWAS studies.\",\n\"The Human Genetics and Computational Biology and Biomedicine groups at Pfizer utilize GWAS and other genetic-disease association data in order to better facilitate novel target discovery and target validation across research and development efforts across our portfolio. To that end, our group has collated various genetics summary statistics datasets in order to understand how traits are correlated genetically, the pathways underlying the genetic risk for disease, and potential causal relationships between certain traits and diseases. We plan to utilize the summary data contained within this study phs001672 to evaluate relationships between the genetics of traits for which we have internal data through such approaches as LD score regression and GCTA, as well as evaluate causal relationships between these traits and circulating proteins/biomarkers through Mendelian Randomization analyses. Additionally, we plan to use these data to perform pheWAS studies on variants which are associated to other disease traits in order to assess potential safety/efficacy issues. Finally, we plan to use these data to test novel algorithms, such as fgwas, for identifying causal genes at GWAS loci through the integration of genomics, transcriptomics, epigenomics, and other various omic's datasets. We request to evaluate all the various phenotypes captured by the Million Veteran's Program to date, as this can provide insight on disease etiology as well as links to secondary indications for targets and mechanisms of interest to our researchers.\",\n\"2020, May11\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Miller, Clint \",\n\"UNIVERSITY OF VIRGINIA\",\n\"Dissecting the causal regulatory mechanisms of coronary artery disease \",\n\"approved\",\n\"Coronary artery disease and heart attacks are the leading causes of death globally. While we have identified a number of modifiable risk factors in the environment that are responsible for heart disease, there are also non-modifiable risk factors that can be identified in our genes. Based on family studies and large-scale population studies we have uncovered genetic risk factors that may contribute to our risk for developing heart disease. These subtle changes in genes may interact with our environment and lead to subtle but substantial changes in risk over our lifetime. Studies that enroll organ donors have been absolutely critical to our understanding of how these changes alter physiological processes in our body and ultimately may inform how to effectively target these pathways to better prevent and treat heart disease.\",\n\"Coronary artery disease (CAD) and myocardial infarction (MI) remain the leading causes of mortality worldwide despite advances in lifestyle modification and therapeutic interventions. Approximately 40% of the risk of developing coronary artery disease may be attributed to genetic risk factors, and large-scale genome-wide association studies (GWAS) have now identified a large proportion of this estimated heritable risk through 95 genome-wide significant loci. These loci are often identified in the form of single nucleotide polymorphisms (SNPs) which are co-inherited withs hundreds of other variants in linkage disequilibrium, presenting a challenge to elucidate causal functional variants responsible for the association. Furthermore, the nearest annotated gene is often not the causal gene, posing additional challenges to understand the mechanisms linking the variant to the disease-relevant phenotype. Expression quantitative trait locus (eQTL) and allele-specific expression (ASE) mapping in relevant cell types and tissues has been critical to prioritize candidate causal variants and genes, through co-localization and cis and trans regulatory network approaches. We have recently identified a number of regulatory variants associated with CAD by integrating genomic, epigenomic, and transcriptomic profiles in coronary artery smooth muscle cells (1). In particular, we have utilized the STARNET dataset to prioritize these loci. This dataset and other CAD and MI GWAS datasets will be further analyzed using a combination of omics datasets and experimental functional follow-up approaches, ultimately to identify novel mechanisms associated with CAD risk amenable to prevention and therapeutic development. We do not anticipate these studies to pose any risk to the study participants.   1. Miller CL, Pjanic M, Wang T, Nguyen T, Cohain A, Lee JD, Perisic L, Hedin U, Kundu RK, Majmudar D, Kim JB, Wang O, Betsholtz C, Ruusalepp A, Franzen O, Assimes TL, Montgomery SB, Schadt EE, Bj\\u00c3\\u00b6rkegren J, Quertermous T. Integrative functional genomics identifies regulatory mechanisms at coronary heart disease loci. Nature Communications. 2016 Jul 8;7:12092.\",\n\"Mar05, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Merkus, Daphne \",\n\"ERASMUS MEDICAL CENTER\",\n\"Placental expression of kynurenine pathway enzymes\",\n\"closed\",\n\"Tryptophan is metabolized through the kynurenine pathway (KP). Indoleamine 2,3-dioxygenase (IDO1) is the first and rate-limiting enzyme of this pathway, and is highly expressed in healthy term placentas. In preeclampsia, placental IDO1 expression is decreased. This impaired metabolism of tryptophan by IDO1, and other KP enzymes might contribute to deficiencies in vascular regulation, immune regulation, and tolerance. However, there is a lot of controversy regarding the exact location of placental IDO1 expression, and we speculate that the function of IDO1 might depend on the cell type it is expressed in. Therefore, we would like to use this data base to explore which placental cells express KP enzymes.\",\n\"Objectives Tryptophan is metabolized through the kynurenine pathway (KP). The first and rate-limiting enzyme is indoleamine 2,3-dioxygenase (IDO1), which is highly expressed in healthy term placentas, but decreased in preeclampsia. Impaired tryptophan metabolism by IDO1, and other KP enzymes might contribute to deficiencies in vascular regulation, immune regulation, and tolerance. We speculate that the function of IDO1 might depend on the cell type it is expressed in. However, the exact location of placental IDO1 expression remains controversial. Based on publicly available data from first trimester placentas by Vento-Tormo et al.1, we have found that IDO1 is mainly expressed in endothelial cells, dendritic cells, monocytes and macrophages. However, IDO1 expression is known to increase throughout pregnancy, and it is therefore essential to confirm these result in term placentas. With the present requested dataset we aim to translate our results from first trimester to term, and preterm delivered placentas. Investigating cell type specific expression of KP enzymes can help us to direct future studies, to investigate cell type specific alterations in placental pathologies (eg preeclampsia), and give a functional meaning to the placental KP. Study design & analysis plan For the requested data set, we only need a count table which shows the expression of all genes per cell. The cells will be sorted based on expression of the enzyme of interest, and all negative cells (0 RNA counts per cell) will be excluded. Next, we will explore which cells are left in the positive fraction. For the phenotyping we would like to use the exact same cell types as described in the study for which we request access (Pique-Regi et al. eLIFE 20192). Adherence to the data use limitations Our study is consistent with the data use limitations as we work from a non-profit organisation. When sharing our data we will acknowledge the original authors of the requested dataset and the funders of their research. Collaborations We do not have any plans on collaboration on our own project. We have used/will use databases from three other studies1, 3, 4 to confirm our results.  References 1. Vento-Tormo R, Efremova M, Botting RA, Turco MY, Vento-Tormo M, Meyer KB, Park J-E, Stephenson E, Polanski K, Goncalves A, Gardner L, Holmqvist S, Henriksson J, Zou A, Sharkey AM, Millar B, Innes B, Wood L, Wilbrey-Clark A, Payne RP, Ivarsson MA, Lisgo S, Filby A, Rowitch DH, Bulmer JN, Wright GJ, Stubbington MJT, Haniffa M, Moffett A, Teichmann SA. Single-cell reconstruction of the early maternal\\u00e2\\u0080\\u0093fetal interface in humans. Nature. 2018;563:347-353 2. Pique-Regi R, Romero R, Tarca AL, Sendler ED, Xu Y, Garcia-Flores V, Leng Y, Luca F, Hassan SS, Gomez-Lopez N. Single cell transcriptional signatures of the human placenta in term and preterm parturition. Elife. 2019;8 3. Tsang JCH, Vong JSL, Ji L, Poon LCY, Jiang P, Lui KO, Ni YB, To KF, Cheng YKY, Chiu RWK, Lo YMD. Integrative single-cell and cell-free plasma rna transcriptomics elucidates placental cellular dynamics. Proc Natl Acad Sci U S A. 2017;114:E7786-E7795 4. Pavlicev M, Wagner GP, Chavan AR, Owens K, Maziarz J, Dunn-Fletcher C, Kallapur SG, Muglia L, Jones H. Single-cell transcriptomics of the human placenta: Inferring the cell communication network of the maternal-fetal interface. Genome Res. 2017;27:349-361\",\n\"2020, Dec30\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Mercader, Josep \",\n\"BROAD INSTITUTE, INC.\",\n\"Discovery and fine mapping of cardiometabolic traits in diverse populations\",\n\"approved\",\n\"Genome-wide association studies have been widely used to identify genetic regions of the genome that modify the risk for a disease. Most of the studies have been performed in European Ancestry participants, but studies in non-European ancestry have been limited. We are planning to extend these studies to diverse populations to better understand the pathophysiology of type 2 diabetes and related traits.\",\n\"We have performed genome-wide association studies in non-European populations. We would like to use additional datasets to replicate and fine-map the the signals identified in our discovery cohorts.  Objectives: The objective of this study is to replicate variants identified in other datasets from diverse ancestries. Study design: We will select variants identified in other studies and perform lookups of the association summary statistics from the MVP data. We will meta-analyze the MVP results with the summary statistics of our study results to assess significance. Analysis Plan: We have performed several meta-analyses of T2D in diverse ancestries. We will select variants with p value < 1e-6 and we will seek for replication in the corresponding ancestry if available within MVP T2D summary statistics. We will consider significant replication if p value is equal to 0.05/number of variants tested. We will consider genome-wide significance when the meta-analysis between the discovery and the replication reaches a p-value below 5e-8. We will perform the meta-analysis using Fixed Effects Invesrse Variance meta-analysis using the software METAL.  We do not plan to combine the requested datasets with other datasets outside of dbGaP. We plan to meta-analyze results obtained from dbGaP datasets with those obtained in our discovery cohorts, but we plan only to meta-analyze summary statistics, and never combine individual-level data. The proposed research is consistent with the Data Use Limitations. We will only use this dataset for biomedical or health research.\",\n\"Nov19, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Menzel, Stephan \",\n\"KING'S COLLEGE LONDON\",\n\"Confirmation of genetic loci for HbF levels and stroke risk in sickle cell disease\",\n\"rejected\",\n\"Whilst sickle cell disease (SCD) is a well-recognised monogenic disorder, the individual patient experience can vary vastly. We believe a certain component of this is due to changes in other genes that influence the complex biology of SCD. We are investigating two aspects of SCD. Firstly, genetic factors that may predispose patients to stroke \\u00e2\\u0080\\u0093 a devastating complication that can affect up to 10% of children with SCD. By looking closely at those who develop a stroke a very young age, in combination with the data available publically, we hope to be able to develop new tests that will help to predict patients most at risk and also, by better understanding key mechanisms involved, develop new methods of treating or preventing stroke in this group. Secondly, we are investigating genetic factors influencing haemoglobin F levels. Higher haemoglobin F levels are known to alleviate disease severity in SCD. We have been researching this for some time and believe the additional data available here will help strengthen our investigations helping us to draw more confident conclusions.\",\n\"The wide phenotypic variation seen in sickle cell disease (SCD) no doubt has significant polygenic influences. Known modifiers of haemoglobin F and alpha-thalassaemia deletions convey a degree of this genetic variability, but do not account for its entirety.  Indeed, known modifiers of haemoglobin F do not even account for all the variability of haemoglobin F levels. Studies investigating such genetic factors are consistently undermined by sample size. At Kings College London, we have established a large database of patients with SCD in whom carefully curated phenotypic data is paired with SNParray data and also a smaller cohort in whom whole exome sequencing (WES) has been performed. We have two facets of investigation for which we believe the dbGaP data will be of great benefit. Firstly, we are investigating genetic factors associated with cerebrovascular disease in SCD. We have performed WES in a cohort of selected patients who have had a stroke at a very young age as we believe genetic influences are likely to be more prominent in this subset. The conclusions drawn from this small sample size will be strengthened by validation in the dbGaP datasets, with those from the SWITCH study providing positive cases and those from the HUSTLE study providing negative cases. Our second interest in the dbGaP dataset relates to common genetic variants controlling stroke risk and Haemoglobin F levels in sickle cell disease. Using our SNParray dataset, we have identified new loci for both traits that we wish to validate in other cohorts. The baseline phenotype data provided contains sufficient information to allow us to utilise the sequencing data for this purpose. Having read the DUC, we are confident that our proposal complies with the data usage limitations, and the anonymity of the participants will not be compromised. We hope this work will improve understanding of two key aspects of SCD, namely cerebrovascular disease and haemoglobin F regulation and provide important new biomarkers in both regards.\",\n\"Sep18, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Melas, Philippe \",\n\"KAROLINSKA INSTITUTE\",\n\"Pharmacogenetic responses to a monoamine stabilizer (OSU6162) in individuals with alcohol use disorder\",\n\"approved\",\n\"Alcohol use disorders (AUD) affect approximately 5% of the world population and impose a major socio-economic burden on the global mental health. Thus, there is a pressing need to develop new medications and identify biomarkers that predict treatment response. Our project aims to enhance our understanding of the molecular and genetic biosignatures that predict treatment response to a novel compound in AUD treatment, called OSU6162. To this end, we use a cohort of AUD individuals who were treated with OSU6162 and calculate polygenic risk scores (PRS), which provide a measure of alcohol use-related disease risk due to their genes. Next, we evaluate whether PRS scores for AUD-related phenotypes correlate with treatment response to OSU6162. Examining whether PRS scores can be used to predict effectiveness in treatment response holds promise for the future since it will allow the prescription of medications to those patients who will have the highest likelihood of responding positively, commonly referred to as personalized medicine.\",\n\"Objectives of the proposed research: Researchers at our institution have previously published a randomized controlled trial evaluating the effects of a monoamine stabilizer (OSU6162) on reducing alcohol-related phenotypes in individuals with alcohol dependence (PMID: 26482910). I am the PI of a follow-up project that uses the same human cohort to conduct pharmacogenetic studies related to personalized medicine. Specifically, our objective is to calculate polygenic risk scores (PRS) for alcohol use-related phenotypes, including comorbid diagnoses (e.g., depression), and examine whether they correlate with therapeutic responses to OSU6162.  Study design: The current study design builds on a completed randomized controlled trial that followed a double blind, placebo-controlled design, where alcohol dependent participants (N=56) were randomized to receive OSU6162 or placebo (PMID: 26482910). The study was approved by the regional ethical review board in Stockholm and the Swedish Medical Products Agency, registered in the European Clinical Trials Database (EudraCT;2011-003133-34), monitored by Karolinska Trial Alliance and conducted in accordance with Good Clinical Practice and the Declaration of Helsinki.  Analysis plan, including the phenotypic characteristics that will be evaluated in association with genetic variants: We have collected genomic data from all study participants using the Illumina Infinium Global Screening Array-24 v2.0. Our PRS analyses, for which we request dbGAP access, will focus on phenotypes including (i) alcohol consumption/AUDIT-C, (ii) alcohol use disorder and (iii) problematic alcohol use, and (iv) depression. Additional PRS analyses, outside dbGAP, will be conducted for Drinks per week (PMID: 36477530). All PRS-related phenotypes will be analyzed independently in relation to OSU6162-response.  Explanation of how the proposed research is consistent with the data use limitations for the requested dataset(s): To calculate the abovementioned PRS scores, we request access to the Veterans Administration (VA) Million Veteran Program (MVP) Summary Results from Omics Studies (dbGAP study accession phs001672.v9.p1). Our proposed research is consistent with the data use limitations and in line with the already published GWAS studies on: (i) alcohol consumption/AUDIT-C and alcohol use disorder (PMID: 30940813), (ii) problematic alcohol use (PMID: 32451486), and (iii) depression (PMID: 34045744). More specifically, the use of the data will be limited to health/medical/biomedical purposes and will not include the study of population origins or ancestry. Moreover, there will be no public posting of genomic summary results, and controlled-access data will only be available to the applicant (Philippe Melas).   Brief description of any planned collaboration with researchers at other institutions, including the name of the collaborator(s) and their institution(s): No such collaborations are planned. \",\n\"2023, Apr18\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Mehta, Divya \",\n\"QUEENSLAND UNIVERSITY OF TECHNOLOGY\",\n\"Multi-omics analysis to identify novel genes for PTSD\",\n\"expired\",\n\"Combining the MVP data with our in-house Australian veteran\\u00e2\\u0080\\u0099s data, we aim to identify novel genes associated with posttraumatic stress disorder (PTSD). This study leverages existing datasets and cutting-edge imputation-based methodologies to uncover robust genes and pathways associated with PTSD through an integrative multi-omics approach. Our team has previously used such a study design to identify novel genes for cancer and we now wish to implement our statistical pipeline on PTSD. These results will be novel and will be submitted to an international peer-review journal. For any publications arising from these analyses, the MVP team will be included as co-authors.\",\n\"Aim: The overarching aim of the proposed study is to develop a bioinformatics pipeline to identify novel PTSD genes through multi-omics integration of secondary and in-house omics data   Objectives:  Objective 1: Identify SNP-regulated genes and methylation sites using validated imputation-based methods - First, using the Million veteran project (MVP) GWAS summary results, we will conduct an imputation-based transcriptome-wide association study (TWAS) using S-PrediXcan (Barbeira et al., 2018) and FUSION (Gusev et al., 2016) to predict the genetically regulated genes that are differentially expressed between PTSD and controls. These genes will be validated by differential gene expression analysis using gene expression measured in our in-house clinically diagnosed Vietnam war combat veterans with and without PTSD (n=300), this cohort is also part of the PGC-PTSD combined cohort (Mehta et al., 2018). Next, using the MVP GWAS summary results and EstiMeth (Freytag et al., 2018) we will perform an imputation-based methylome-wide association study (MWAS) to predict the genetically regulated DNA methylation sites that are differentially methylated between individual with and without PTSD. Validation of these results will again be performed using the DNA methylation measurements from our in-house Vietnam war combat veterans with and without PTSD (Mehta et al., 2017). While the imputation-based approaches identify the genes/methylation sites associated with PTSD, they do not provide information about colocalization of GWAS SNPs and expression quantitative trait loci (eQTLs) or methylation quantitative trait loci (mQTLs). We conduct colocalization genetic analysis using COLOC (Giambartolomei et al., 2014) and EpiCOLOC (Zhou et al., 2020) to identify shared causal variants between GWAS SNPs and eQTLs/mQTLs. We will also conduct mendelian randomization to examine the causal influence of GWAS and colocalized eQTL/mQTL SNPs on PTSD. Lastly, characterisation through enrichment analysis such as tissue specific enrichment, enrichment in other major psychiatric disorders (depression, schizophrenia, bipolar, alcohol use disorder), and pathway analysis will also be conducted for the identified SNPs. Objective 2: Once we have identified functional SNPs that are associated with changes in PTSD gene expression and/or DNA methylation profiles in Objective 1, we will use these SNPs to develop a unique polygenic risk score. The developed model will be validated on Vietnam war veteran dataset (Mehta et al., 2018). Phenotypes will include PTSD and depression to validate if the PRS from PTSD (using only robust functional SNPs from MVP) explain variance in PTSD and/or depressive symptoms. The requested data will be used for health/medical/biomedical purposes, i.e., to identify PTSD-specific biomarkers that can provide further understanding of biological mechanisms underlying PTSD and develop PRS model to identify individuals at risk. The requested data will not be combined with other dbGaP dataset or other sources but rather the results from the independent analysis will be verified in measured gene expression and methylation data sets and tested for enrichment in the summary results of other psychiatric disorders. We will only work with de-identified data and will not attempt to identify any of the participants. Our proposed research does not include the study of population origins or ancestry. The genomic summary results will not be posted publicly. Any findings from the study will be published or otherwise broadly shared with the scientific community. For any publications arising from these analyses, the MVP team will be included as co-authors.  \",\n\"Apr11, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"McVean, Gil \",\n\"GENOMICS LIMITED\",\n\"Robustness of genetic association and polygenic risk scores to non-European ancestries (cancer)\",\n\"closed\",\n\"Genetic data has the potential to transform health care, by better predicting diseases, understanding and recommending treatment options and better prevention. This is particularly critical for cancers, which greatly benefit from early detection, and for cardiovascular diseases, for which lifestyle alterations and drugs (e.g. statins) are available for at risk individuals. However, much of the data  underpinning this revolution have been generated in individuals of European descent. These data cannot be readily used in non-European populations, which limits broad adoption and widens an already existing gap in access to modern genomic technologies. With this project we are interested in assessing how these genetic predictors can be used in individuals of diverse ancestry. We want to improve the design of these tools so that they become more precise, and usable by a wider range of individuals.\",\n\"There is a growing consensus in the genomics community that genetic-based prediction can provide effective information in health care. In particular, polygenic risk scores (PRS) have gathered a lot of attention in the scientific community. PRS are designed to aggregate information from a large number of genetic variants, which, once combined, provide effective predictions for a wide range of traits. We are particularly interested in developing these PRS for diseases that are actionable and/or have ongoing screening programmes, which makes common cancers (breast, prostate, colorectal, ovarian, lung) a major focus of our work.   Genome-wide association (GWA) studies provide the data supporting the design of these PRS. However, PRS typically rely on keeping the pattern of linkage disequilibrium (LD) constant. Because the majority of genetic studies have been performed in individuals of European descent, these PRS are not readily exportable to other populations, which limits their widespread adoption.  Given these limitations, we are keen to explore the robustness of PRS and association summary statistics across European and non-European populations. For the common diseases for which genetic prediction is a relevant and practical opportunity, this work will be performed in three stages: We will initially refine the construction of PRS by utilising summary statistics data from studies performed in populations of European descent. We will download data from genome-wide association (GWA) studies with individual-level genotypes in European and non-European cohorts and generate PRS data, comparing association statistics between European and non-European data. We will explore alternative PRS generation techniques that leverage individual-level data, such as machine learning techniques (LASSO, neural networks) and understand how these prediction algorithms are impacted by varying ancestries.  This analysis will help us assess the robustness of PRS and improve their design in order to be effective in a broader set of populations. The design of these PRS will combine the novel data accessed as part of this project with the summary statistics that we have previously aggregated for a range of genetic studies. Where permissible in accordance with data use terms, we also combine individual-level data (both from dbGaP and other sources) to understand the interplay between genetics and phenotype with more granularity.  We intend to publish various aspects of our research and share results accordingly. We have already presented some of our prediction work at international conferences (e.g. ASHG 2019, 2021) and we are in the process of organising our work for submission in peer-reviewed publications. We will continue to follow a similar process across cancer types, for conference and journal submissions, following the pace of our research work.  Only the collaborators named in our dbGaP application (i.e. our staff) will have access to our database. We will absolutely respect all data use certifications and data use limitations when conducting our analyses. In the case of disease-specific limitations, we will limit our analyses accordingly using technical and administrative means.\",\n\"2019, Oct29\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Okada, Yukinori \",\n\"OSAKA UNIVERSITY\",\n\"The genetic architecture of quantitative traits and their correlation across phenotypes between populations.\",\n\"closed\",\n\"Although association of genotypes with phenotypes is extensively investigated,  the majority of the studies are targeting European ancestry individuals. Here we would like to elucidate the differences between the genetic architecture of quantitative traits between European and East Asian population, and would also like to assess their correlation with other phenotypes. We would like to  evaluate the shared and distinct genetic architecture between populations.\",\n\"GWAS has been clarifying the polygenic architecture of complex human traits. However, the majority of the studies are targeting European ancestry individuals. Here we would like to elucidate the differences between the genetic architecture of quantitative traits between European and East Asian population, and would also like to assess their correlation with other phenotypes. We have been conducting large-scale GWASs of quantitative traits including cholesterol-related traits in Japanese population. By constructing polygenic risk scores and associating them to the end-phenotypes such as late-onset complex diseases or health outcomes in both of the populations, we would like to elucidate shared and distinct effect of the exposure to the outcome. For this purposes we look for the access to the summary statistics of cholesterol-related traits in MVP.\",\n\"2019, Feb21\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"SALEM, RANY \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Genetic Association Analyses of Sequencing Data, Complex Traits and Methods Development Using Biomedical Repositories\",\n\"approved\",\n\"The goal of this project is to better understand how genes influence disease and traits. Improved understanding of how genes influence human biology can help prevent disease, reduce severity, and improve quality of life. We are requesting data for many genetic sequencing studies and will analyze them separately and in combination, using improved statistical methods that we and others develop. We will study body size and shape, related childhood growth traits, the aging process, asthma and lung function, diabetes and metabolic syndrome including related traits and their complications, blood measures (including cholesterol and metabolic products), kidney function, immune disorders and complications, cancer, heart and vascular disease and related traits, and how gene activity differs by tissue. We will ask how genes influence traits over time, use genetics to assess cause and effect, predict disease risk, study how genes influence multiple traits, and analyze genetic factors individually and in combination. Results of this project will be presented at scientific meetings and published in journals.\",\n\"*Research Use Statement (RUS)  In this project, we aim to improve understanding and provide insights into the genetic architecture of diseases and complex traits utilizing whole exome- and whole genome- sequencing from multiple cohorts in dbGaP. The application and improvement in both statistical methodology and study design approaches is an important path to accomplish this goal. Improving insights into the genetic basis of complex traits could help guide new therapies. We believe this project is well aligned with the goals of dbGAP. We are requesting data for a large number of study cohorts, that will be used separately and in combination (integrating genotype, expression, phenotype and sequence data for a specific trait) to achieve these goals. To maximize the utility of these datasets, we propose to explore methodological issues that can inform the association analysis of select phenotypes: anthropometric phenotypes, childhood growth, the aging process, asthma and lung-function, diabetes and glucose metabolism, metabolic syndrome and its components, diabetes and MetSy morbidity, cognitive function, hematological and biochemical measures (including lipid and metabolites), kidney function and disease, rheumatological disorders and musculoskeletal complications, cancer and related disorders, and cardiovascular function and disease, including BP and HTN. Several important genetic association questions will be explored with existing and novel statistical genetics techniques in the following areas: Longitudinal Analysis, Mendelian Randomization, Risk prediction, multi-traits and Pleiotropy, eQTL, and Statistical Genetics. Access to the data will be limited to members of the Salem lab and labs of listed collaborators (all affiliated with UCSD) who have completed both human subjects and data safety training. All users will abide to terms of data use certification restrictions and limit analyses accordingly, and will honor all publication embargo dates. Findings will be disseminated and shared with the scientific community by presenting results at national scientific meetings and publishing in peer-reviewed journals. \",\n\"2022, Jul27\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ruigrok, Ynte \",\n\"UNIVERSITY MEDICAL CENTER UTRECHT\",\n\"Determining the phenotypical effects of proteins associated with abdominal aortic aneurysm\",\n\"closed\",\n\"We request access to the aggregated GWAS data on AAA, which will be used to prioritize promising protein drug targets using in silico experimentation.\",\n\"We will employ the principle of genetically guided drug development to identity and prioritize druggable proteins for an association with abdominal aortic aneurysm (AAA) and related cardio-metabolic traits relevant to determine the benefit/harms profile of on-target perturbation.   Specifically, we propose to leverage in-house developed software and statistical tools for drug target Mendelian randomization (MR), colocalization, and polygenic scores (PGS), to identify drug target (tissue specific splice site, mRNA expression, or protein concentration) effects on the before mentioned outcomes. Targets will be further prioritized by identifying potential repurposing opportunities through linkage with ChEMBL, and the British National Formulary (BNF). Tissue specificity will be assessed sourcing Human Protein Atlas (HPA) data, and target pathways explored by interfacing with REACTOME.   The phenotypical consequences of prioritized drug targets will be shared to inform (de novo) drug development through open access scientific publications, presentations, and through dissemination of a drug target catalogue, concatenating evidence the previously described analyses.\",\n\"2022, Apr07\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Wang, Frances \",\n\"UNIVERSITY OF PITTSBURGH AT PITTSBURGH\",\n\"ENIGMA-Addiction\",\n\"approved\",\n\"Genome wide association studies (GWAS) identify genetic variants associated with a trait such as problematic substance use. We will use two GWAS from dbGAP which targeted nicotine dependence and problematic alcohol use to generate individualized genetic risk scores for these traits in other samples. We will examine whether these risk scores are related to behavior (e.g. age of onset and frequency of substance use, and measures of impulsivity) or brain features detected using MRI, including measures of brain structure (e.g. cortical and subcortical grey matter volume and white matter tract integrity) and function.\",\n\"Polygenic risk scores (PRS) are a powerful tool to examine the genetic liability of a specific trait in another population. Our aim is to investigate the genetic liability to substance use disorders (e.g., alcohol) in population-based cohorts in relation to behavior measures and to identify brain intermediate phenotypes measures associated with greater genetic risk for problematic alcohol use. PRS will be create using the PLINK 2 software based on the results of the Genome Wide Association Studies (GWAS) problematic alcohol use (PMID 32451486 - phs001672.v3.p). These PRS will be regressed onto behavioral phenotypes (e.g., substance use measures including age of onset and frequency of use, and impulsivity measures) and onto brain measures including structure (cortical and subcortical measures and diffusivity) and function. This project does not require individual-level data from the mentioned studies and it is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry.\",\n\"Jul18, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Won, Hong-Hee \",\n\"SUNGKYUNKWAN UNIVERSITY RES/BUS FDN\",\n\"Investigation of association between 14 micronutrients and 22 cancer outcomes using Mendelian randomization analyses\",\n\"approved\",\n\"Numerous observational studies and clinical trials have reported associations between micronutrients, namely vitamins and essential metals, and various health outcomes including cancers. Many people are interested in risk factors associated with various cancers. Many observational studies have reported the benefits of supplementing micronutrients on cancer risks, whereas randomized controlled trials (RCTs) more often reported a null effect. However, observational studies are inherently prone to confounding, which may result in inflated estimates, and RCTs are costly and often insufficiently powered for cancer outcomes requiring long-term follow-up and large cohort size. More evidence is needed to know whether there is actually causal relation. In this study, we will use genomic data to investigate which micronutrients can affect risks for various cancers.\",\n\"Identifying the precise causal risk factors of various cancer is important to reduce social and economic burdens. Numerous observational studies and clinical trials have reported associations between micronutrients, namely vitamins and essential metals, and various health outcomes including cancers. In this study, we will investigate causal relationships between associated risk factors such as micronutrients and cancers. To identify causal associations of micronutrients with cancers, we will use a two-sample Mendelian randomization (MR) approach using genome-wide association study (GWAS) summary statistics. We will not combine requested datasets with other datasets outside of dbGaP and we will analyze these datasets independently.\",\n\"Apr12, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Won, Hong-Hee \",\n\"SUNGKYUNKWAN UNIVERSITY RES/BUS FDN\",\n\"Discovery of additional loci and improvement of disease prediction using the genetic relationship of polygenic traits\",\n\"approved\",\n\"Genetic variants associated with various common diseases are being actively discovered through GWAS. Typically, GWAS is conducted to target a single disease which prohibits the investigation of the complex network between diseases. Furthermore, lack of power limits finding all the true associations between genetic variants and disease. Our goal is to analyze various genetically correlated traits related to metabolic, psychiatric, and neurodegenerative disorders through a multivariate approach, followed by post-GWAS analyses to underpin their biological functions.\",\n\"Genome-wide association studies (GWAS) are fundamental research in genetics. The genetic variants associated with the common disease are being actively discovered. However, GWAS is typically conducted for a single disease which prohibits investigation of the complex interconnection between the diseases. Furthermore, GWAS of many diseases is still limited in terms of a sample size to fully reveal the underlying associations and the fact that causation cannot be determined. Recent methodologies such as multivariate GWAMA and Genomic SEM utilize genetic correlation between multiple diseases to define a common latent phenotype having shared genetic effects. The construction of such genetic architecture is useful to disentangle the relationship between interconnected diseases. Additionally, the statistical power to detect additional genetic variants will increase due to the larger sample size. Besides, it is also possible to discover additional genetic variants for the selected disease by conditioning the genetic effects of closely related diseases using conditional FDR. We aim to apply these approaches to various metabolic, psychiatric, and neurodegenerative disorders. The GWAS result will be followed by thorough post-analysis such as polygenic risk score calculation and evaluating its performance, assessing the shared proportion of genetic variants through MiXeR, detecting causal relationships through mendelian randomization, and so on. Newly discovered genetic variants and their biological relevance are expected to be useful for targeting new biomarkers and drugs. The requested data will be combined with external data through meta-analysis. There would be no harm to participants since it is impossible to identify individuals from summary statistics. All users agree to make results of studies using the data available to the larger scientific community and will abide by terms of data use certification restrictions and limit analyses accordingly and will honor all publication embargo dates. \",\n\"2023, Mar02\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Won, Hong-Hee \",\n\"SUNGKYUNKWAN UNIVERSITY RES/BUS FDN\",\n\"Causal inference between associated risk factors and Alzheimer disease, and identification of novel genetic variants for Alzheimer disease\",\n\"approved\",\n\"Many people are interested in preventing dementia. For example, people are wondering what diseases should be avoided, whether they should exercise, whether they should quit smoking, and whether they should be on a diet. Epidemiological studies on many people have shown that some diseases or lifestyle habits are associated with dementia, but more evidence is needed to know whether there is actually causal relation. In this study, we use epidemiological and genomic data to investigate which risk factors can control Alzheimer dementia.\",\n\"Identifying the precise causal risk factors of Alzheimer\\u00e2\\u0080\\u0099s disease (AD) is important to reduce social and economic burden. Candidate associated factors suggested by traditional epidemiological studies can be biased by reverse causation, confounding factors, and measurement errors. To overcome these problems and identify causal risk factors, the Mendelian randomization (MR) method has been proposed and is currently widely used. MR analysis is less likely to be influenced by social and economic bias because germline mutations are not affected by confounders.In this study, we will investigate causal inference between associated risk factors and AD, and identify novel genetic variants for AD. First, we will identify associated risk factors for AD from independent healthcare datasets (Elderly cohort in South Korea, UK biobank, and Korean genome and epidemiology study). We will employ the time-varying Cox hazard model with outcome of overall survival free of AD. All candidate associated factors for AD will be tested as independent variables of interest. This step will be performed independently of the NIAGADS data. Second, we will identify causality of these associated factors for AD using one-sample Mendelian randomization (OSMR). The OSMR estimates a polygenic risk score (PRS) related to the significant associated factors for each sample using the PLINK software. Summary statistics of genetic association studies of associated factors from LD hub (http://ldsc.broadinstitute.org/ldhub/) will be used to estimate the PRS. Next, we will test for the association of the PRS with incident AD using a logistic-regression analysis adjusted for covariates such as age and sex in the NIAGADS data. Third, we will investigate biological pathways shared by the causally-related phenotypes. The DEPICT tool can be used to prioritize and suggest associated biological pathways by identifying the most likely causal genes of identified loci using various databases.Our study will provide evidence for the causal risk factors for AD.  Use of this data is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry. All users will abide to terms of data use certification restrictions and limit analyses accordingly and will honor all publication embargo dates. Findings will be disseminated and shared with the scientific community by presenting results at national scientific meetings and publishing in peer-reviewed journals. This genetic study will conduct to learn about, prevent, or treat health problems; will not use at profit purpose. The aims of study will limit to health research on dementia, especially genetic basis. This study is related to AD, a neurodegenerative disease. This study will be conducted in the non-profit organization (SungKyunKwan Universty).\",\n\"Jun22, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Wingo, Thomas \",\n\"EMORY UNIVERSITY\",\n\"Proteome-wide association study of Psychiatric Diseases in MVP\",\n\"approved\",\n\"We will use the results of the genome-wide association study of psychiatric disorders (e.g. anxiety, substance abuse) with the protein data to investigate the proteins in the brain that contribute to the risk of these diseases and facilitate findings for new treatments.  \",\n\"Research Objective.: Genetic variants that alter brain protein abundance may contribute to the development of different psychiatric diseases, such as depression, anxiety, and addiction to substance abuse (including opioids, cannabis, and alcohol). These disorders are debilitating and treatments are ineffective in a large subset of the patients. We propose to impute protein expression levels using summary statistics from the summary GWAS from MVP (e.g. Levy et al. 2020, AmJ Psychiatry; Zhou et al. 2020, Jama Psychiatry; Zhou et al 2020, Nature Neuroscience), to perform a proteome-wide association study (PWAS) of psychiatric diseases.   Our goal is to identify brain proteins contributing to these disorders to facilitate effort in finding novel treatments.  Study design: We will adapt a published pipeline (FUSION by Gusev et al 2016) that integrates the genome-wide association study (GWAS) summary statistics from MVP and assayed protein expression data to perform an imputed protein study of psychiatric disorders (e.g. anxiety and substance abuse).   Analysis plan: We will use as reference a brain proteome from post-mortem brain samples, already generated from our group. We will estimate brain protein expression weights from the brain proteomes and their corresponding genome-wide genotyping using Gusev et al 2016 FUSION method. Next, we will integrate the requested GWAS summary statistics with the protein weights to perform a proteomic analysis of psychiatric disorders following FUSION pipeline. We may also compare these results with the one obtained from other disorders (e.g. neurodegenerative diseases) or traits (personality, cognition, anthropometric) to identify overlapping and unique genes/pathways responsible for those disorders.   \",\n\"2020, Sep03\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Wingo, Aliza \",\n\"EMORY UNIVERSITY\",\n\"PWAS of PTSD in Veterans\",\n\"closed\",\n\"We will use the results of the genome-wide association study of PTSD with the protein data we generated to investigate the proteins in the brain that contribute to risk of PTSD.\",\n\"Objective: PTSD is a debilitating disorder and its treatment is ineffective in a large subset of the patients. Our goal is to identify brain proteins contributing to PTSD to facilitate effort in finding novel treatment for PTSD. Study design: We hypothesized that genetic variants lead to altered protein abundance in the brain, which in turn leads to susceptibility for PTSD after trauma exposure. To test this hypothesis, we will adapt a published pipeline (FUSION by Gusev et al 2016) that integrate the PTSD genome-wide association study (GWAS) summary statistics and assayed protein expression data to perform an imputed protein study of PTSD.  Analysis plan: Our group has generated brain proteomes from post-mortem brain samples. We will estimate brain protein expression weights from the brain proteomes and their corresponding genome-wide genotyping using Gusev et al 2016 FUSION method. Next, we will integrate the requested GWAS summary statistics with the protein weights to perform a proteomic analysis of PTSD following FUSION pipeline.\",\n\"2020, Apr08\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Wareham, Nicholas \",\n\"UNIVERSITY OF CAMBRIDGE\",\n\"Molecular basis of human metabolism and relevance to disease outcomes\",\n\"rejected\",\n\"Disorders of human metabolism such as type 2 diabetes, its risk factors and complications (e.g. heart attacks) affect millions of people worldwide. A key limiting factor in the development of preventive and therapeutic strategies is an incomplete understanding of the disease mechanisms. With this proposal we plan to combine information about the genetic makeup and the environmental risk factors to understand the causes of these metabolic diseases and help the development of strategies to prevent and treat these conditions.  \",\n\"Our research aims to better understand the genetic and molecular determinants of human metabolism, their interplay with the environment in shaping the risk of health outcomes in order to enable the development of safe and effective preventive or therapeutic strategies. To gain these insights, our approach is to integrate data from large-scale population-based studies, including genomics, multi-\\u00e2\\u0080\\u0098omics\\u00e2\\u0080\\u0099 measurements (e.g. metabolomics, proteomics, transcriptomics, etc.), detailed anthropometry, growth and development-related measurements,  nutritional measures and biomarkers, physical activity and other behavioural variables, environmental exposures, electronic health records, functional characterisation of genetic variants or genes in experimental models. Our team have had a leading role in large-scale studies identifying genetic determinants of metabolic traits (e.g. adiposity, insulin resistance, glycaemic measures) and diseases (e.g. type 2 diabetes) and the understanding of their role in human metabolism via the integration of genetic data and with phenotypic measurements. The aims of this proposal are: (a) discovery and characterization of genetic determinants of metabolic diseases and related conditions (as defined by the GERA as disease outcomes e.g. type 2 diabetes and cardiovascular disease) via genome-wide association studies or other relevant approaches; (b) observational and genetic epidemiology (e.g. Mendelian randomization) studies of the aetiologic relevance of risk factors to these diseases; (c) studies of naturally-occurring genetic variants known or suspected to affect the function of gene products in order to identify or validate drug targets for these conditions; (d) identification of causal genes and mechanisms for these conditions via fine-mapping, study of the co-localisation of quantitative trait loci or other integrative approach and (e) development and use of novel methods in molecular epidemiology.  We will combine the requested data with existing publicly available data in a meta-analysis of results. We will not meta-analyse individual level data across studies. There are no additional risks to participants.\",\n\"2021, Sep20\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Wang, Kevin \",\n\"STANFORD UNIVERSITY\",\n\"long noncoding RNA (lncRNA) regulation of human preterm labor \",\n\"approved\",\n\"The mechanisms that trigger preterm labor have remained largely unknown, leading to a limited number of treatment options. Molecules called cytokines that are produced by the body\\u00e2\\u0080\\u0099s immune system in response to stress during pregnancy are thought to play important roles to jumpstart the uterus and the contraction process, leading to preterm labor. Understanding how cytokine release and control are regulated is critical to the development of better therapies. We have discovered a new class of molecules called long noncoding RNAs (lncRNAs) that play critical roles in the regulation of cytokine signals in humans. We will leverage the dbGAP data to identify and validate the existence and abnormal expression of these lncRNAs and begin to characterize how they affect the immune response.\",\n\"We are interested in analyzing preterm labor (PTL) datasets generated from different cell types like Whole blood, Monocytes, and chorioamniotic membrane to understand the cell type specific immune regulated changes happening in PTL condition and try to uncover long noncoding RNAs (lncRNAs) that change in expression through their interactions with epigenetically regulated immune genes in 3-dimensional chromatin space. We would like to reprocess this scRNA data to examine cluster specific differences. We will also compare and correlate the Monocyte cluster with the bulk RNAseq data from the Monocytes in order to identify Monocyte specific lncRNAs that are differentially expressed and regulated along with the differential immune genes.\",\n\"Feb01, 2022\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"WRAY, Naomi \",\n\"UNIVERSITY OF QUEENSLAND\",\n\"The effect of blood pressure drugs on PTSD - a Mendelian randomisation analysis\",\n\"closed\",\n\"We aim to use genetic information to try and predict the effect of different classes of blood pressure drugs on PTSD risk and PTSD symptoms\",\n\"Observational studies suggest that some blood pressure drugs may have beneficial effects on PTSD symptoms. This project will use GWAS summary data from the Million Veteran Program as well as expression quantitative trait loci summary data from GTEX, PsychENCODE and eQTLGen to conduct a two-sample summary-based Mendelian randomisation analysis to predict the potential effect of different classes of blood pressure drugs on PTSD risk and risk of re-experiencing PTSD symptoms.\",\n\"2020, Oct05\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Li, Yun \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease. Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data.\",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry. Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community. However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets.\",\n\"Nov22, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"WRAY, Naomi \",\n\"UNIVERSITY OF QUEENSLAND\",\n\"Study of the genetic contribution to aortic aneurysm\",\n\"approved\",\n\"Aortic aneurysm has a strong genetic contribution but due to the inadequate sample sizes, our understanding of AA is lacking. Meta-analysis of the summary statistics from the Million Veteran Program with data from other programs will increase our ability to understand the genetic architecture of AA.\",\n\"Aortic aneurysm is a common cardiovascular disease that could be lethal if managed inappropriately. The ability to predict the onset and progression of AA is important as the management strategies for different stages of AA is different. Taking genetic factors into the prediction is meaningful as evidence showed that there is a strong genetic component in AA \\u00e2\\u0080\\u0093 twin studies showed heritability for AA could be as high as 70%. However, only ~25 loci have shown to be related to AA so far, potentially attributable to inadequate sample size to detect the difference. In our study, we are planning to meta-analyse all available GWAS sum statistics with our own data(N ~ 7500). Incorporating summary data from MVP would be invaluable to our research because of its large sample size. With MVP data and other sumstat from FinnGenn, Biobank Japan, and our own research, the total sample size will increase to N=25,000. We hope this will increase our power to detect significant SNPs related to AA.\",\n\"2022, May11\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"WON, HYEJUNG \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"Elucidating neurobiological mechanisms underlying alcohol dependence \",\n\"closed\",\n\"Substance use disorders remain as a public health crisis. Genetic studies have elucidated genomic regions that are associated with alcohol dependence, while the functional impact of those regions remains to be studied. With our analyses, we hope to identify alcohol dependence risk genes from the genomic regions. Characterization of alcohol dependence risk genes will bring us closer to understanding the core biological mechanisms underlying alcohol dependence, and provide a foundation for developing treatment strategies for alcohol dependence. \",\n\"Genome-wide significant association studies (GWAS) have identified a plethora of genetic variants associated with complex traits such as substance use disorders (SUDs). However, most genetic risk variants reside in non-coding regions of the genome. Several gene mapping tools including MAGMA have been developed to assign GWAS SNPs to their cognate genes. We recently developed Hi-C coupled MAGMA (H-MAGMA), which harvests the capacity of MAGMA while leveraging chromatin interactions to assign non-coding SNPs to their cognate genes. In our previous study in which we applied H-MAGMA to psychiatric disorders, we have shown that H-MAGMA can dramatically increase the power of MAGMA by identifying more genes based on functional genomic evidence (Sey et al., bioRxiv 2019). Here, we plan to apply our H-MAGMA framework to identify genes associated with alcohol dependence. Once SNPs are mapped onto the genes, the subsequent functional analyses on the alcohol dependence-associated genes can elucidate the neurobiological mechanisms underlying alcohol dependence. Data from MVP will be used for research purposes only and no personal identifiers will be included in any of our analyses. \",\n\"2020, Jan13\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"WENDT, FRANK \",\n\"UNIVERSITY OF TORONTO\",\n\"Genetic and Environmental Factors Associated with Internalizing Psychopathologies\",\n\"approved\",\n\"Many environmental and genetic factors associate with whether a person has a diagnosis of depression, anxiety, posttraumatic stress, and related disorders or diseases. This study is designed to investigate these various factors using sophisticated statistical approaches applied to large-scale aggregated data from dbGaP and external cohorts. The analytic approach investigates different types of genetic markers associated with the above traits, their interaction with one another and with environmental variables, and the ways in which these relationships may be used to mitigate disorder symptoms. Our goal is to deliver the scientific community with a collection of genetic positions and environmental variables with large effects on diagnosis. \",\n\"The internalizing spectrum of psychiatric disorders includes major depression, generalized anxiety, and posttraumatic stress disorders including their transdiagnostic features like neuroticisms and suicidal thoughts and behaviors. Large genetic studies support a heritable component of these traits. This project has three goals: (i) develop methods using genetic data to identify participants at high risk for disorder, (ii) uncover symptom heterogeneity in diverse populations, (iii) identify relevant brain mechanisms of psychopathology across the lifespan using large genome-wide association studies. Upon completion of our studies, we will have a better understanding of the various types of genetic loci that contribute to internalizing psychopathologies, including the potential for disseminating large-effect genetic variants and environments to the scientific community. In combination, these results may inform one path towards pharmacological and non-pharmacological intervention strategies.  Our study design relies of the use of genotype-phenotype association, imputation of various marker types, polygenic scoring, gene-by-environment genome-wide interaction testing, predicted gene expression, etc. For datasets that contain whole-exome and/or whole-genome sequencing data, we also plan to extract genotypes for multi-allelic loci across the genome using various read alignment strategies. All associations between phenotype~genotype and phenotype~environment will include baseline covariates like age and sex. Genetic analyses will consider genetic ancestry as an additional covariate. Because many of the requested datasets were not created to study internalizing traits, we also will include study ID as a covariate to mitigate the effects of targeted ascertainment in the diverse list of requested studies. Quality control protocols include detection of related individuals using genetic data, population stratification to inform genotype-phenotype relationships among diverse populations, and imputation quality thresholding for various genomic loci. Data from this dbGaP request may be either combined or meta-analyzed with other datasets including, but not limited to, UK Biobank, the Million Veteran Program, Biobank Japan, FinnGen, and internally curated genetic and psychiatric studies. We are requesting several datasets with the goal of merging and/or meta-analyzing across cohorts and with the external cohorts listed above, where appropriate, to improve statistical power of association. We anticipate no risk to participants as a consequence of meta-analysis.  \",\n\"2022, Dec20\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Vrieze, Scott \",\n\"UNIVERSITY OF MINNESOTA\",\n\"GWAS & Sequencing Consortium of Alcohol and Nicotine\",\n\"approved\",\n\"Our goals are to identify genetic risk (and protective) factors for alcohol and nicotine use, to identify genetic risk factors that co-occur across multiple substance use behaviors, and to identify biological targets for future functional experiments or drug development relating to addiction or substance use.  To do this, we are aggregating data across dozens of studies to accumulate sample sizes of >1 million individuals to discover genes and other aspects of our DNA that are related to smoking and alcohol use behaviors. \",\n\"Genome-wide association meta-analyses have detected a handful of loci associated with heaviness of smoking, smoking initiation and cessation, alcohol use, and alcohol dependence, but many associated variants remain to be discovered. We are requesting access to studies from dbGaP to conduct genome-wide meta-analyses to better characterize the genetic etiology of nicotine and alcohol use. Phenotypic and genomic data from dbGaP studies will be used to: a) discover new loci associated with smoking and drinking phenotypes, b) detect shared genetic etiology across smoking and drinking phenotypes, and c) implicate novel gene sets and biological systems in the pathophysiology of addiction and use. Phenotypic data will include subject responses to questions about age of initiation of smoking, smoking cessation, heaviness of smoking, whether an individual has ever smoked regularly, and alcohol consumption (e.g., drinks per week, binge drinking). These studies will include individuals from diverse populations, including those historically under-represented in health research. We plan to publish all results in a peer-reviewed journal and share summary statistics from any meta-analyses with the broader scientific community.  Data from dbGaP may be meta-analyzed with data from other studies (obtained from dbGaP and other sources) to improve the power for novel genetic discoveries, while respecting the consent of all participants. However, each dataset from dbGaP will be analyzed independently prior to the meta-analysis. We expect that this activity creates no additional risks to participants. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues.\",\n\"2020, Jul27\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Voight, Benjamin \",\n\"UNIVERSITY OF PENNSYLVANIA\",\n\"Post-GWAS analyses for Million Veteran Program Summary Data\",\n\"approved\",\n\"The Million Veteran Program is a large-scale genetic biobank with over 500,000 participants with genotype and phenotype data available from diverse ancestries. Summary results from genome-wide association scans (GWAS) performed by the Program are available (dbGAP). These summary data are not human subjects data, are de-identified, and carry no personal health identifying information. We want to utilize the summary data to perform additional genetic analyses, to better characterize the genetic basis and architecture of traits analyzed by MVP, in order to identify causal variants, genes, tissues, and risk factors for these traits. \",\n\"We would like to use MVP summary data across the range of traits analyzed to date to perform a battery of computational post-GWAS analyses. These analyses are designed, broadly, to identify additional novel loci, or to triangulate on the causal variant, gene, and tissues that underlying the observed associations. These analyses also have the objective to characterize trait heritability and genetic correlation with additional traits, but also to perform causal inference studies with other, publicly available data sets to generate causal evidence between traits and disease endpoints.  The batter of analyses includes: causal inference studies via Mendelian Randomization, multivariate GWAS, conditional analysis, genetic fine-mapping, genetic co-localization analysis with GTEx and other QTL mapping data sets, and genetic correlation and heritability studies. At the time of this writing, summary data includes, but is not limited to: type 2 diabetes, lipid levels, blood pressure, eGFR, alcohol and substance use/abuse, PAD, and VTE.\",\n\"2020, Dec21\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Vlachos, Ioannis \",\n\"BETH ISRAEL DEACONESS MEDICAL CENTER\",\n\"Effect of genetic, genomic, and microbiomic variation on complex traits\",\n\"approved\",\n\"Information is often lost in extensive cohort studies that can prove crucial to our understanding of the effects of the genotype on numerous quantitative traits and to complex disease phenotypes. We will use established and novel techniques that will enable the characterization of processes and traits across cohorts, uncover their in-between interactions and their associations with complex phenotypes\",\n\"In this project we aim to identify the effects of human variation on genetic, genomic and microbiomic level on complex phenotypes. We will perform this study by employing established techniques to assess the effects of genotype on different quantitative traits or develop new methods that can quantify and assess the associations between genotype and traits, as well as between the quantitative traits and the complex phenotype. We are requesting data from different study cohorts that will enable us to design, validate, and test our methods across phenotypes and populations.  The datasets will be used independently, combined and/or integrated. We will focus mostly on traits derived by analyzing RNA processing events, non-coding regulation, and microbiome profiling. The main research questions are:  a) Which are the genetic variants controlling quantitative traits that are until now lost in most analyses? How are they connected with the phenotype? b) Can we use common variants to reduce the search space for coding and non-coding functional events or loci? c) Can we identify interactions between different quantitative traits in order to maximize our understanding of the underlying complex phenotypes?  Access to the data will be limited only to members of the Vlachos Lab and collaborators who have completed human subjects and data safety training. All data users will agree to terms of data use certification restrictions and limit analyses accordingly and honor all publication embargo dates. Findings will be disseminated widely and shared with the scientific community by presenting results at national scientific meetings and publishing in peer-reviewed journals. \",\n\"2023, Mar20\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"VAN DEN OORD, EDWIN \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"Integrating methylomic and genomic summary statistics to better characterize alcohol-related DNA methylation\",\n\"closed\",\n\"Despite extensive anti-drinking efforts, ~6.2% of US adults qualify for an alcohol use disorder and many more engage in habitual or problematic drinking. Mounting evidence suggests that DNA methylation may play a role in drinking behaviors. However, many DNA methylation sites are regulated by common genetic variation. The research objective is to utilize summary statistics from alcohol-related genome-wide association studies (GWASs) to determine if alcohol-related methylation is simply picking up on genetic signal or represents a unique biological signature separate from common genetic variation.   This project will analytically combine the summary statistics from alcohol GWAS and summary statistics from alcohol methylation-wide association studies (MWASs). The combination of the GWAS and MWAS results does not create any additional risk to participants as we will only be using summary statics and do not need any participant-level information to conduct the project. Specifically, we will test whether the top findings from the alcohol GWAS occur more frequently among the top findings of our MWAS (i.e., are enriched) than what is expected by chance.  \",\n\"Despite extensive anti-drinking efforts, ~6.2% of US adults qualify for an alcohol use disorder and many more engage in habitual or problematic drinking. Mounting evidence suggests that DNA methylation may play a role in drinking behaviors. However, many DNA methylation sites are regulated by common genetic variation. The research objective is to utilize summary statistics from alcohol-related genome-wide association studies (GWASs) to determine if alcohol-related methylation is simply picking up on genetic signal or represents a unique biological signature separate from common genetic variation.   This project will analytically combine the summary statistics from alcohol GWAS and summary statistics from alcohol methylation-wide association studies (MWASs). The combination of the GWAS and MWAS results does not create any additional risk to participants as we will only be using summary statics and do not need any participant-level information to conduct the project. Specifically, we will test whether the top findings from the alcohol GWAS occur more frequently among the top findings of our MWAS (i.e., are enriched) than what is expected by chance.  \",\n\"2020, May11\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Worthey, Elizabeth \",\n\"UNIVERSITY OF ALABAMA AT BIRMINGHAM\",\n\"Understanding the molecular underpinnings of ME/CFS - Long Covid\",\n\"approved\",\n\"Myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS) is a disabling and complex illness. There is significant overlap between ME/CFS and Long Covid and many researchers and clinicians with expertise in these diseases believe they are one and the same. ME/CFS/Long Covid ranges from mild to severe, and can cause more symptoms in some people than in others. It can affect many organs and systems and in addition to chronic fatigue is associated with tender lymph nodes in the neck or armpits, frequent sore throats, digestive issues, like irritable bowel syndrome, chills and night sweats, allergies and sensitivities to foods, odors, chemicals, light, or noise, muscle weakness, shortness of breath, and irregular heartbeat. Although ME/CFS/Long Covid is a relatively common disorder there are few approved and available treatments. We will combine dbGAP ME/CFS and related datasets (e.g. long covid and ehlers danlos) with internally generated data to better understand the mechanisms underlying development and progression of disease. We will also use these datasets to predict drugs that may can address the impact of variation in ME/CSF/Long Covid with the goal of identifying therapeutic candidates for testing in in vitro and in vivo ME/CFS/Long Covid-relevant systems.\",\n\"Myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS) is a disabling and complex illness. ME/CFS ranges from mild to severe, and can cause more symptoms in some people than in others. There is notable overlap is symptoms and disease course/outcomes between these diseases. Symptoms can affect many organs and systems and in addition to chronic fatigue is associated with tender lymph nodes in the neck or armpits, frequent sore throats, digestive issues, like irritable bowel syndrome, chills and night sweats, allergies and sensitivities to foods, odors, chemicals, light, or noise, muscle weakness, shortness of breath, and irregular heartbeat. Although ME/CFS is a relatively common disorder there are few approved and available treatments. Recently the dramatic increase in numbers of patients seen with this disorder due to covid infection has provided an impetus to data sharing that seems likely to finally support a better understanding of disease pathology through comprehensive clinical data/omic/other dataset based studies into causation/outcomes/therapeutics. We will combine dbGAP ME/CFS and related datasets (e.g. long covid and ehlers danlos) with internally generated RNA-Seq, WES, and WGS datasets and apply computational biology and machine/deep learning approaches to better understand the mechanisms underlying development and progression of disease. We have a well phenotyped ME/CFS cohort at UAB that also includes a variety of metabolic, immune, and imaging datasets. We will use this data to develop a model that can classify patients based on attributes, which will be used to look for differing underlying molecular causes of disease or risk factors in the different groups. We will also use these datasets to perform signal inversion in expression datasets to predict drugs with inverted expression profiles that may can address the impact of variation in ME/CSF with the goal of identifying therapeutic candidates for testing in in vitro and in vivo NF1-relevant systems.  It is our goal to apply computational biology and AI based approaches to existing genomic and transcriptomic datasets in combination with in-house derived data to benefit patients by defining molecular signatures of ME/CFS. No individual participants will be identifiable from the analysis that we perform. Our project will comply with all data use limitations for dbGaP studies. The dbGaP datasets will be used in accordance with the user specifications for the de-identified data. We will not seek or pursue findings that are beyond the respective scope of each data set. The data will be used for research purposes only. Any results of the study will be disseminated to the scientific community through publications and presentations, with the requisite acknowledgment of the NIH-designated data repositories and the specific dataset accession numbers.\",\n\"2023, Apr14\",\n\"phs002245.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Worthey, Elizabeth \",\n\"UNIVERSITY OF ALABAMA AT BIRMINGHAM\",\n\"Understanding the molecular underpinnings of chorangiomas of the placenta.\",\n\"approved\",\n\"Chorangiomas are benign placental capillary lesions associated with significant alteration of the placenta when they become extensive increasing risk of polyhydramnios, heart failure, growth restriction of the fetus, and/or preeclampsia in the mother. Despite the clear importance of these lesions, their molecular underpinnings are not understood. The relationships between development or expansion of chorangiomas, and adverse outcomes for mother or child remain unknown and are poorly studied. Only a single study of large chorangiomas has been performed and it identified no characteristic molecular variation. There are no early and accurate detection methods and treatments of these lesions are limited and varied in effect. The main objective of the research proposed is to use these dbGAP datasets as part of similar studies we are performing with internal cases to study the molecular underpinnings of chorangiomas. We use a multi-omics approach to identify and characterize deleterious molecular changes associated with development, vascularization, and growth in chorangiomas. We believe that our findings may also shed some light on mechanisms in preeclampsia and potentially other vascular tumors as these occurrences share overlap with chorangioma development.\",\n\"The main objective of the research proposed is to use these dbGAP datasets as part of similar studies we are performing with internal data. We are studying examples of chorangiomas identified within our affiliated women\\u00e2\\u0080\\u0099s and children\\u00e2\\u0080\\u0099s hospital referred to a collaborating pathologist. Our research focuses on analysis of these samples to identify molecular alterations and mechanism characteristic in chorangiomas. These generally benign placental capillary lesions are associated with significant hemodynamics alteration of the placenta when they become extensive increasing risk of polyhydramnios, heart failure, growth restriction of the fetus, and/or preeclampsia in the mother. Despite the clear importance of these lesions, their molecular underpinnings are poorly understood. We use a multi-omics approach to identify and characterize deleterious molecular alterations. We hypothesize that replication, vascularization, and growth pathways are altered in chorangiomas and share overlap with preeclampsia and potentially other vascular tumors.  We have already identified a number of variants and mutational signatures that we believe to be chorangioma specific and wish to use these external dbGAP datasets to validate and add to our findings. For instance, we will make use of transcriptome data requested to better understand the non-chorangioma expression landscape of the placenta as a control for our in-house generated data. We will also make use of the dbGAP expression data to compare and contrast regulatory networks from non chorangioma data with the networks already identified from our internal chorangioma and unaffected tissue. Utilization of the dbGAP methylation datasets will be used to identify associations between altered methylation sites and loci identified from internal and external genomic and expression data. We believe that by better understanding these distinct mutational signatures, the regulatory and others networks present in health and affected tissue, and the differences between these samples we will be able to advance our understanding of both healthy and chorangioma associated diseases processes in this critical and understudied organ. No individual participants will be identifiable from the analysis that we perform. We do not believe our project represents additional risks to participants. Our project will comply with all data use limitations for dbGaP studies. We will not combine datasets, but rather use one to validate findings from the other. Genetic ancestry will be considered in cleaning of the data and checking for sample relationships, but will not be a focus or primary outcome of our study. We will not seek or pursue findings that are beyond the respective scope of each data set. \",\n\"2023, Mar20\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Wu, Lang \",\n\"UNIVERSITY OF HAWAII AT MANOA\",\n\"Identifying methylation and protein biomarkers of Type 2 diabetes using genetic instruments\",\n\"approved\",\n\"Type 2 diabetes brings huge public health burden. We propose to conduct studies using genetic instruments to better understand methylation and protein biomarkers influencing risk of Type 2 diabetes.\",\n\"It remains largely unknown for which DNA methylation and protein biomarkers may influence risk of Type 2 diabetes (T2D). Recently, there has been a new approach of using genetic instruments to evaluate associations between exposures and diseases. For example, novel disease susceptibility genes may be identified by evaluating associations of genetically-determined gene expression with disease risk (transcriptome wide association studies (TWAS) design). To identify novel DNA methylation and protein biomarkers associated with T2D, we plan to perform research using relevant T2D GWAS summary statistics data and data from several studies (e.g., MESA etc) containing relevant genetic and molecular information well suited for our research goals. In brief, we will develop molecular prediction models to represent the genetically-determined exposure of interest, and assess their associations with T2D. We hope our work will generate new knowledge for T2D. We plan to broadly share our results with the scientific community by publishing the findings. Data from the requested datasets will only be used in research consistent with the data use limitations of the corresponding datasets and will not be combined with other datasets of other phenotype.\",\n\"2022, Jun10\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Wuest, Stefan \",\n\"UNIVERSITY OF REGENSBURG\",\n\"LawStress: Psychological, endocrine, central nervous and genetic predictors of chronic stress responses in daily life \\u00e2\\u0080\\u0093 a prospective-longitudinal quasi-experiment\",\n\"approved\",\n\"Chronic stress increases the risk for several disorders, but the mechanisms mediating this effect are not fully understood. Our project aims at an exploratory analysis of the predictive value of a polygenic score (PGS) regarding phenotypes related to chronic stress responses in daily life. In a longitudinal study with a sample of healthy law students, we investigated a stress group experiencing a 13 months stress phase (preparation for a major exam) and a control group. At baseline, several predictor variables have been assessed, including the neural response to a stress induction in a MRI scanner. Subsequently, perceived stress levels, anxiety and depression symptoms as well as indicators of the regulation of the hormone cortisol have been assessed repeatedly over 13 months. With the planned polygenic score modelling we would like to study if the genetic liability to depression is related to these variables in individuals experiencing two different environments (with and without particular stress exposure).    \",\n\"There is ample evidence for an association between chronic stress and several disorders, including depression, anxiety, cardiovascular diseases as well as immune function related pathologies. However, biopsychological mechanisms mediating this link are not fully understood. Our project aims at investigating predictors of chronic stress responses in daily life. In a prospective-longitudinal study with a quasi-experimental design and a sample of healthy law students, we investigated a stress group experiencing a 13 months stress phase (preparation for the first state examination for law students) and a control group. We will use summary statistics generated by genome-wide association studies (GWASs) conducted within the Million Veteran Program dataset to investigate the combined predictive value of a polygenic score (PGS) and the environmental stressor (state examination) on changes in depression symptoms and other stress related phenotypes (e.g. momentary perceived stress levels, markers of hypothalamic-pituitary-adrenal axis regulation and responses to a standardized fMRI stress induction paradigm).  These exploratory analyses will be conducted in a multilevel framework to control for individual differences in intercept and slopes of the dependent variables. We are not requesting individual level data.  \",\n\"2022, May23\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"van der Sluis, Sophie \",\n\"VRIJE UNIVERSITEIT\",\n\"Genetic relationship between 13 psychiatric disorders\",\n\"closed\",\n\"This project concerns a study on the genetic relatedness and distinctness of 13 psychiatric traits through an extensive series of analyses, focusing not only on the genetic overlap between these traits but also on their genetic uniqueness.\",\n\"This project concerns a study on the genetic relatedness and distinctness of 13 psychiatric traits through an extensive series of genetic analyses. Contrary to related projects, we will not only focus on the genetic overlap between these traits but also on their genetic uniqueness. The main objective of the project is to identify genes or gene sets that explain comorbidity between psychiatric disorders, and to identify genes or gene sets that are unique to these disorders. This study concern the analyses of GWAS summary statistics only. We will calculate (partial and local) genetic correlations, conduct genomic structural equation modelling and Mendelian randomization to scrutinize the genetic relatedness between the included disorders. If we have access to multiple GWAS summary statistics on the same trait in non-overlapping samples, then we will meta-analyze these sum stats first and then continue with the summary statistics from the meta-analysis.  For this project, we have obtained access to GWAS summary statistics of major psychiatric disorders such as schizophrenia, depression, anxiety, PTSD, anorexia, OCD, ADHD and Tourette syndrome. We wish to include alcohol use as a trait in this project as well because it is highly comorbid with many other psychiatric disorders, which makes studying the genetic overlap and uniqueness, and direction of causation, specifically interesting. The summary statistics that we here ask permission to are very valuable as these concern data on a large and well described sample, which would greatly improve our statistical power to detect overlapping and unique genes/gene sets for this disorder. All analyses in this project will be conducted by people from the CTG department at the VU (post doc Mats Nagel in particular), but we will collaborate with prof Murray Stein and prof Joel Gelernter, via whom we gained access to the latest GWAS summary statistics on PTSD from the VA Million Veteran Program.  The requested data will only be used for the purpose described here, i.e., to gain understanding on the (shared and unshared) genetics of mental health disorders. The data will not be used to study population origins or ancestry. \",\n\"Nov10, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"van Rijn, Bas \",\n\"ERASMUS MEDICAL CENTER\",\n\"Placental expression of trophoblast subtypes\",\n\"approved\",\n\"Most children are born with a normal birth weight and have a healthy start to life. However, each year about 5% of children have not grown very well during pregnancy and are born too small and often too early. This is due to a problem with an organ called the placenta. The placenta is the organ by which the unborn baby makes contact with the mother and can extract food components and oxygen. In children that are born too small, the placenta has not developed very well. However, we do not fully understand how the cells in the placenta are formed in normal placentas and abnormal placentas. In this project, we compare data from analysis of cells that have been harvested from the placenta to figure out how these cells are formed during pregnancy. \",\n\"Regenerative capacity of the placenta relies on coordinated differentiation and growth of trophoblast subtypes. For this project we would like to investigate the proportions of trophoblast subtypes during gestation. The requested dataset allows for analysis of trophoblast subtypes which we will compare with our in-house sequencing data of first trimester placenta.  \",\n\"Sep12, 2022\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"ming, li \",\n\"KUNMING INSTITUTE OF ZOOLOGY\",\n\"The relationship between human behavior and lifespan\",\n\"approved\",\n\"We aimed to explore the relationship between human behavior and lifespan by leveraging genetics data. Research result will help to increase awareness of human behavior and lifespan.\",\n\"We plan to study the relationship between human behavior, like depression, drinking, smoking and lifespan by analyzing the GWAS summary statistics data. We will conduct a series of genetic analyses including linkage disequilibrium score regression (LDSC), Mendelian randomization, and polygenic overlap quantification and identification to understand the genetic architectures of human behavior and lifespan. The applied dbGap data will be analyzed independently and will not be combined with other datasets.\",\n\"Jan18, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"da Silva Couto Alves, Alexessander \",\n\"UNIVERSITY OF SURREY\",\n\"Mendelian Randomization of Birthweight on Adult Blood Lipids\",\n\"closed\",\n\"Cardiovascular diseases are complex caused by a combination of genetics and environmental factors. The identification of environmental causal factors contributing to risk of cardiovascular disease is hampered by confounding factors.  Mendelian randomization is a statistical approach that is less prone to confounding. To conduct mendelian randomization only summary statistics are required. No need for individual-level genetic information to be shared, thus volunteer genetic data is never put at risk. We request the adult lipid summary statistics corresponding to the 20 analyses listed below: pha004662.1 pha004663.1 pha004664.1 pha004665.1 pha004827.1 pha004828.1 pha004829.1 pha004830.1 pha004831.1 pha004832.1 pha004833.1 pha004834.1 pha004835.1 pha004836.1 pha004837.1 pha004838.1  \",\n\"I would like to access the GWAS summary statistics reported on the paper (Klarin 2018 NatGen) to conduct mendelian randomization analyses. We aim to understand the causal effect of developmental phenotypes such as birthweight on the risk of cardiovascular markers and risk factors such as adult blood lipids. The focus of this study is birth weight and adult lipids. References Klarin 2018. Genetics of blood lipids among ~300,000 multi-ethnic participants of the Million Veteran Program. Nature Genetics.\",\n\"Jul06, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Zhang, Zhongyang \",\n\"ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI\",\n\"Association study on placenta cell composition and gestational age\",\n\"approved\",\n\"Preterm birth (PTB) is the birth of a baby before 37 weeks of pregnancy. In 2018, preterm birth affected 1 of every 10 infants born in the United States, and the preterm birth rate rose for the fourth straight year. Additionally, racial and ethnic differences in preterm birth rates remain. The placenta is a complex organ relevant to PTB. However, little is known about the cell composition of placenta and pathologic process underlying PTB. We will use RNAseq data to investigate the cell composition of placenta and how it is relevant to PTB at the cellular and molecular level. This study has great public health significance especially in addressing the reproductive health disparities in the U.S. \",\n\"Preterm birth (PTB) is when a baby is born too early, before 37 weeks of pregnancy have been completed. In 2018, preterm birth affected 1 of every 10 infants born in the United States. The preterm birth rate rose for the fourth straight year in 2018. Additionally, racial and ethnic differences in preterm birth rates remain. For example, in 2018, the rate of preterm birth among African-American women (14%) was about 50 percent higher than the rate of preterm birth among white women (9%).  The cause of PTB molecular underpinnings of human parturition in gestational tissues, and in particular the placenta, are still poorly understood. The placenta is a complex heterogeneous organ consisting of both maternal and fetal cells. Little is known about the cell type composition and transcriptome of placenta and pathologic process underlying PTB at the cellular and molecular level. We will apply for NIH dbGaP single cell RNAseq data (accession number: phs001886) and plan to integrate it with bulk RNAseq data of placenta collected from term birth and from birth of short gestational age (SGA). We will deconvolve the bulk RNAseq data and infer their cell composition, and further associate them with SGA status and factors important in reproductive health. Potentially this proposed study will fill in the knowledge gap of PTB etiology at the cellular and molecular level. This study has great public health significance especially in addressing the reproductive health disparities in the U.S.\",\n\"2020, Sep03\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Zhang, Zhengdong \",\n\"ALBERT EINSTEIN COLLEGE OF MEDICINE\",\n\"Evolutionary analysis of aging and age-related disorders in a large multi-ethnic cohort\",\n\"rejected\",\n\"Identification of genetic factors underlying longevity is of paramount importance for the healthcare of the ageing population. It is believed that the specific pattern of disease coincidence in the ageing population is partly influenced by the genetic makeup of the respective population. Adaptation over the several thousand years has shaped the population gene pool and resulted in signatures of selection to be preserved in the genome of populations. Through analysis of signatures of selection across different age bins, we aim to capture the impact of ongoing selection on human longevity and assess the survival burden of genetic variants contributing to human quantitative traits. The ultimate goal of our project is to identify pro-longevity variants that enable survival to older ages and disentangle variants that increase the chance of mortality.\",\n\"Identification of ethnic-specific comorbidity patterns and evolutionary processes underlying age-specific mortality is of practical interest for the healthcare system. Evidence in favor of hard selective sweeps across the human genome, and often, evolutionary scans of the genome for identification of selective sweeps is confounded by population history.  Through the application of diverse methods for identification of selective sweeps across the genome of the ageing population we envisage identification of loci that are under directional natural selection across the genome of the healthy ageing population.  By aggregating data from our large in-house cohort of centenarian Ashkenazi Jewish population we plan to 1) investigate patterns of allele frequency changes across different age groups while controlling for confounding factors such as ethnicity and population structure; 2) we will assess the degree of epistasis between many pairs of SNPs in the context of longevity through composite likelihood modeling, and; 3) investigate the burden of recessive and loss of function mutations in advanced age through the development of robust statistical methods  Through investigation of ethnic-specific comorbidity patterns we aim to discover covariance between age-related traits and identify pleiotropic effects that are shared between anthropomorphic traits that correlate with human life span.  Ultimately by extending the statistical method recently developed by Jacobs et al. we aim to scan the entire genome for known and novel signatures of selection. By utilizing the GERA, the Framingham, and the Million Veteran cohorts, we will comprehensively and systematically analyze genetic variants influencing human survival in the extant human population.  Confidentiality of research participants is ensured by using anonymized data. The requested data will be deposited to the firewall-secured institutional storage server and will be accessible to the authorized applicants through the secured UNIX shell environment. \",\n\"2021, Feb10\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Zhang, Shijie \",\n\"TIANJIN MEDICAL UNIVERSITY\",\n\"Identification and prioritization of causal regulatory variants for cardiovascular disease based on GWAS data\",\n\"approved\",\n\"GWAS have successfully identified many disease/trait associated variants, but the reported leading associated variants are unlikely to be the actual causal variants. Associated loci may contain hundreds of variants spanning large regions and encompass several genes, making the biological interpretation of GWAS loci more difficult. We plan to combine eQTL data, GWAS data and epigenomic data to identify and prioritize functional eQTLs and causal variants that explain the CAD pathogenesis. Using experimental validation, we will investigate the biological mechanism of functional variants and uncover how these genetic variants affect gene expression. The study could provide a comprehensive resource of casual regulatory variants in CAD development. \",\n\"Over a hundred of coronary artery disease (CAD) genetic risk loci have been identified in the last decade through genome wide association studies (GWASs). A number of studies have experimentally validated regulatory effects of risk allele for complex diseases at individual loci. Yet for most of these associations the true causal variants remain unknown, especially for the associated variants located in the noncoding genomic region. Million Veteran Program (MVP), a nationwide cohort drawn from an integrated healthcare system serving a diverse population including many Black and Hispanic people. Previously, we have developed several computational methods to predict and prioritize regulatory variants in context-dependent manner (Li MJ et.al. Genome Biology. 2017 16;18(1):52). We plan to (1) collect public eQTL and GWAS data. (2) perform colocalization analysis based on eQTLs and the result of GWAS meta-analysis of CAD, with additional information of chromatin states, epigenetic markers as well as chromosome interactions, to identify and prioritize functional eQTL and causal variants. (3) Perform experimental validations to investigate the regulatory effect and potential mechanism of several identified causal regulatory variants for CAD susceptibility.. We expect that our study could provide new insights into the genetic basis and pathogenesis of CAD. \",\n\"Oct17, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Zhang, Liye \",\n\"SHANGHAITECH UNIVERSITY\",\n\"Explore human placenta in term and preterm parturition.\",\n\"closed\",\n\"Parturition is essential for the reproductive success of viviparous species, yet,the mechanisms responsible for the onset of labor remain to be elucidated. Clinically,preterm is common and may post danger to the newborn. Therefor, this project mainly wants to explore the molecular mechanism of preterm on the level of single cells and aims to find some molecular markers for the clinical diagnosis. We are also interested in comparing the difference in preterm birth between Asian and Caucasian groups. \",\n\"Research Objectives: Explore the mechanism of preterm birth and compare the single cell population of Asian and Caucasian population of preterm birth. Given the difficulties in obtaining the placenta samples, we want to reanalyze the data generated in this dataset and compare it with our small cohort on preterm birth in China.   Study design and analysis plan 1. Intergrate data with our own single cell data and validate whether some pattern we observed in our data is also observed in this dataset. In our datasets, we observe distinct population structure in different preterm birth cases. We want to check whether some of the patttern we observed in our sample is also observed in this dataset. For example, if we observed a increase in cell type X in our dataset in our preterm patient. We want to check whether such pattern can be observed at any patient in this dbgap dataset.   2. Compare aneuploidy of placenta between preterm and in-term. Recent study in Nature suggest common aneuploidy in human placenta. Given it is possible to infer aneuploidy based on the results in cancer, we want to analyze whether the aneuploidy is associated with preterm and interm birth.   3. explore the epigenetic dysregulation based on the eRNA captured in the single cell RNA-Seq.  We have established method to uncover eRNA from single cell RNA-Seq, we want to see whether the enhancer profiles predicted from eRNAs of preterm birth differs from the in term birth. \",\n\"2021, Apr16\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Zhang, Haitao \",\n\"NIH\",\n\"Genetic association and polygenic risk scores of substance use disorders and related psychiatric diseases\",\n\"approved\",\n\"The proposal will utilize the large, well-characterized dbGaP summary results and datasets to derive Polygenic risk scores (PRS) as genetic components in epidemiologic studies and prediction model in substance use disorders and related psychiatric diseases. We will also validate the findings of our own genetic association studies on various psychiatric disorders and substance use disorders, such as mood disorder, anxiety disorder, Posttraumatic stress disorder, personality disorder, eating disorder, alcohol use disorder, nicotine use disorder, and drug use disorders. \",\n\"Polygenic risk scores (PRS) derived from dbGaP summary statistics will be used as genetic components in epidemiologic studies and prediction model in substance use disorders and related psychiatric diseases. In addition, we will utilize dbGaP summary results to further replicate/elucidate susceptibility genes identified in our genetic epidemiology research. We will combine the dbGaP summary results with our dataset in the process of analysis. Because we only use dbGaP summary results in population level in our research analysis, it will NOT create any additional risks to participants in dbGaP projects. The genetic data in our lab are generated from the genotyping array of Saliva DNA, with the aim of discovering potential genetic variants in various psychiatric disorders and substance use disorders, such as mood disorder, anxiety disorder, Posttraumatic stress disorder, personality disorder, eating disorder, alcohol use disorder, nicotine use disorder and drug use disorders. We intend to request dbGaP datasets to further validate the genetic variants identified from our own genetic studies. To address the issue of the different genotyping array between dbGaP datasets and our own studies, imputation approach will be used to get fully shared genetic data between different GWA studies. Meta-analysis will be primarily performed to identify genes contributing to these mental disorders. Since psychiatric disorders may have some shared as well as specific genetic effects, we plan to carry out an additional meta-analysis to identify genetic factors contributing to shared genetic components between two or more disorders by combining the genetic data from GWA studies on different traits. These meta-analyses will utilize large samples and thus gain more power to detect genes predisposed to complex disorders. Despite the important roles of copy number variations (CNVs) in causing many human complex diseases, especially psychiatric diseases, CNVs were not well studied in published GWA studies. The CNVs will be called using combined methods in our study plan and the association of CNVs with psychiatric disorders will be further tested. In addition, gene x gene interactions and gene x environmental interactions may be involved in the development of complex disorders and thus these interactions will be also evaluated. Given that susceptibility genes are likely to have small effects on the complex diseases, we will also test the effects of biological pathway and gene network on psychiatric disorders. Here we are more interested in potential common biological pathway shared among different psychiatric disorders. In addition, we will evaluate the genetic correlation between the genetic values for two or more diseases or disorders. We will also integrate data from GWAS and eQTL studies to identify genes whose expression level is associated with psychiatric disorders. Combining datasets and performing comprehensive genetic analysis from multiple genetic studies hold great promise in a better understanding of the genetic causes of psychiatric disorders and their comorbid disorders and thereby lead to better ways of prevention, diagnosis, and treatment. These dbGaP data will be used only for research purposes. \",\n\"2021, Jan19\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"ZHU, XIAOFENG \",\n\"CASE WESTERN RESERVE UNIVERSITY\",\n\"Statistical analysis of high-throughput genotype and sequencing data\",\n\"approved\",\n\"There is pressing need for new analytical tools to extract meaningful information from the existing large amounts of genomic data. We plan to apply our developed novel statistical methods to analyze biological mechanisms using existing genomics data and for detecting rare and common genetic variants of associated with Alzheimer's disease. \",\n\"Alzheimer's disease related Phenotypic and genomic data from the dbgap data will be used for investigating genotype-phenotype associations to detect genetic variants associated with the disease; and (b) test new statistical methods for genetic association analysis. Data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses.  We will not make the data available to anyone outside of the named collaborators on the DAR.   We will report any findings in scientific journals.  \",\n\"Jul30, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"ZHU, XIAOFENG \",\n\"CASE WESTERN RESERVE UNIVERSITY\",\n\"Genetic analyses of sleep apnea and hypertension related traits\",\n\"approved\",\n\"Obstructive sleep apnea (OSA) is a common disease which consists of repeated awakenings while gasping for air. OSA is an important contributor to several diseases, including cardiovascular disease. High blood pressure (BP) is a common condition associated with heart, brain, and kidney diseases.  We have established that there is a strong genetic component contributing to OSA and blood pressure, independent of obesity. By combining data from our Cleveland Family Study with other studies, we hope to assemble the largest whole genome sequencing data of OSA and blood pressure performed to date. We will combine family-based genetic signals and cutting-edge tools developed by our group to improve the chances of finding genetic mutations contributing to OSA and blood pressure. The combination of datasets outside dbGaP, which will be analyzed both independently and together with dbGaP datasets in meta-analysis, will not create any additional risks to participants.\",\n\"The extent and nature of sleep is an important determinant for both physical and mental health. Sleep disorders and disruptions of normal sleep architecture and duration have adverse health outcomes, increasing propensity to hypertension, cardiovascular disease, diabetes, stroke, depression and other conditions. The most common sleep disorders are Obstructive Sleep Apnea (OSA) and insomnia. OSA is characterized by recurrent episodes of pharyngeal obstruction during sleep, exposing the affected individual to severe nightly cardiovascular stress. Insomnia characterized by difficulties falling or staying asleep, and is associated with perturbations in physiological systems. Disturbed sleep architecture, from OSA, insomnia or other etiologies, may similarly contribute to health problems. OSA, insomnia and physiological disturbances in sleep may thus contribute to a wide range of health effects, supporting the need to identify the genetic bases for these interrelated disorders and traits. We aim to identify genes/variants underlying OSA traits.  High blood pressure (BP) is a common condition associated with heart, brain, and kidney diseases. The BP heritability estimated from family data ranges from 30 to 60%.  Recent genome-wide association studies (GWAS) have identified over 100 common BP variants that together explain ~3.5% of BP variation. Thus, the heritability of BP remains missing and most of the genetic variants remain to be discovered. It is expected that low frequency and rare variants may be able to explain a portion of the BP \\u00e2\\u0080\\u009cmissing heritability\\u00e2\\u0080\\u009d. Because of lower sensitivity to genetic heterogeneity, linkage analysis of family data can be used to identify low frequency and rare variants. Thus, we aim to identify novel low-frequency and rare variants associated with BP traits using TOPMed whole genome sequencing data.  The research will be published in peer-reviewed journals.\",\n\"Jul16, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Yazdi, Puya \",\n\"GENIUS LABS COMPANY\",\n\"Validation of novel robust relative risk score models for common diseases across multiple populations\",\n\"approved\",\n\"Disease risk is determined by the combined influences of genetics and environment. The genetic component of disease risk can be estimated through the use of robust polygenic risk score (PRS) models, which estimate the risk of developing a certain disease, relative to the general population.Through this project, we aim to test and improve the risk models we have previously developed for multiple common diseases and biomarkers--measurements used to infer health status and risk. We are assessing these models for robustness, accuracy, and sensitivity across multiple populations. We are primarily concerned with models of relative risk for cardiovascular diseases, diabetes, hypercholesterolemia, dementia, stroke, and the various biomarkers associated with these disease states. This work will further our understanding of how genomic differences can influence individuals\\u00e2\\u0080\\u0099 relative risks of various diseases.  \",\n\"We have developed several methods of generating PRS models: from GWAS summary statistics alone, summary statistics combined with individual level data in a hybrid model, and a novel absolute risk model which incorporates levels of various biomarkers as priors. Validation of these PRS models against simulated data  indicates the models are robust and potentially applicable to populations of EUR and non-EUR ancestry. However, without testing against real datasets unassociated with the training and validation sets, we cannot be confident in these claims. We propose to use the genotype and phenotype information included in these dbGAP datasets to test our PRS models. We aim to use the information about the models\\u00e2\\u0080\\u0099 limitations gained from this testing phase to improve these models to make them more broadly applicable.   We further propose to utilize a new linear mixed model we have developed for association testing, to identify additional disease associated variants from these datasets.   We may compare the SNP frequencies and other statistics with outside databases, but we will use the data provided in accordance with the Data Use Limitations and Certification of each study.   We plan to publish the software associated with these PRS models under an open-source license, to contribute to the improved discovery of disease-significant genetic variants, and greater understanding of the differences in genetic architecture between populations.   \",\n\"Apr19, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Yang, Qiong \",\n\"BOSTON UNIVERSITY MEDICAL CAMPUS\",\n\"Mendelian randomization study of causal relationships between serum proteins and cardiovascular phenotypes\",\n\"closed\",\n\"We will correlate the genetic association results of proteins and cardiovascular disease risk factors (lipid levels and blood pressure) to uncover new biological pathways.\",\n\"Recent advances in aptamer-based proteomic technologies provide high sample throughput, rendering the profiling of low abundance analytes in epidemiological cohorts far more feasible.  We have recently completed genome-wide association studies of ~1300 aptamers in about 3,000 individuals , which provides opportunity to study the potential causal relationships between proteins and  other cardiovascular phenotypes using the Mendelian randomization methods. In this study, we will interrogate the genetic association results of protein profiles in our study and those of lipid levels and blood pressure phenotypes in Million Veteran Program (MVP) to infer causal relationships using various two sample Mendelian randomization techniques.\",\n\"2019, Aug09\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"YANG, BAO-ZHU \",\n\"YALE UNIVERSITY\",\n\"Post-GWAS analyses on the genetics of addictions, medical and psychiatric disorders, and behaviors\",\n\"approved\",\n\"Differences in genetic makeup contribute to variations in behavior, health, and diseases. We will use the summary information derived from the Million Veteran Program\\u00e2\\u0080\\u0099s genome-wide association studies (GWASs) to help understand the genetic basis of addictions, medical and psychiatric disorders, and behaviors.\",\n\"Differences in genetic makeup contribute to variations in behavior, health, and diseases. The Million Veteran Program (MVP) has generated rich data, including genome-wide association studies (GWASs)\\u00e2\\u0080\\u0099 summary statistics. We will use MVP GWAS summary statistics to conduct post-GWAS analyses, such as (but not limited to) computing genetic correlations, generating polygenic scores, converting to the gene-based association, conducting Mendelian randomization, carrying out meta-analyses, or performing pathway or network analysis to examine the genetic basis of behavior, health, and diseases, including addictions, medical and psychiatric disorders, and behaviors. We will analyze MVP GWAS summary statistics independently, combine them with GWAS summary statistics for traits from other sources, or link to gene expression data such as the Genotype-Tissue Expression (GTEx) data. We will not combine datasets but integrate the MVP GWAS summary statistics with summary information for other traits. Our overarching objective is to decipher the biology or pathways of behavior, health, and diseases through genetic summary information and gene expression data. \",\n\"2022, Dec05\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Xu, Lin \",\n\"UT SOUTHWESTERN MEDICAL CENTER\",\n\"Leveraging new deep learning algorithms to predict functional noncoding variants\",\n\"approved\",\n\"Artificial intelligence (AI) is revolutionizing and reshaping the health-care systems by enabling new algorithms and computer-based decision-support systems to increase diagnostic accuracy, design novel therapeutic strategies with better effectiveness, improve clinical workflow, decrease human resource costs and optimize treatment choices. My goal is to integrate cutting-edge artificial intelligence technology and preclinical drug tests as an experimental validation step, in order to establish a new framework for developing \\u00e2\\u0080\\u009cgame-changing\\u00e2\\u0080\\u009d strategy to identify previously known functional noncoding variants in various human diseases, in order to identify new biomarkers for disease diagnosis and explore new mechanisms of disease development through mutations in noncoding regions.\",\n\"In this study we will develop algorithms to predict gene and variant functions in a variety of human diseases. With the access to the dbGaP sequencing datasets, we will leverage our algorithms to identify potential functional variants. My lab will release our results to the public after publications are accepted. Our scripts used for this study will be also be released on GitHub. My lab will not combine requested datasets with other datasets outside of dbGaP. Data downloaded from this resource will be stored safely and avoid to be accessed by others without permission of PI. Use of the data will be limited to not-for-profit organizations. phs000259, phs000735, phs000970, phs000260 are related to the human microbiome. Sequence data generated from microbiome samples will be used only for the purpose of studying the microbial components of the microbiome and will not be used to make inferences about the human host DNA itself. phs002245 is for COVID-19 infection. We will follow these restriction for data usage and storage.\",\n\"2020, Dec04\",\n\"phs002245.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ursini, Gianluca \",\n\"LIEBER INSTITUTE, INC.\",\n\"Diabetes GRS and completed suicide\",\n\"rejected\",\n\"Our data on gene-expression in completed suicide brains point to energy metabolism as a contributor to the suicidal phenotype at the molecular level; additionally, our cases of suicide appear to have a significantly lower BMI than the other individuals. The proposed project plans to employ the genetic variants associated with risk for diabetes to construct a diabetes genetic score of risk and investigate its association, as a proxy of altered energy metabolism, with completed suicide in our sample of post-mortem human brains. The results of this study may help to identify biomarkers of suicide risk that could inform prevention.\",\n\"Research on genetic components of suicide relies mostly on hypothesis-driven approaches, aimed at suicide attempters. Our data on gene-expression in completed suicide brains point to energy metabolism as a contributor to the suicidal phenotype at the molecular level; additionally, our cases of suicide appear to have a significantly lower BMI than the other individuals. This project intends to leverage summary statistics from the MPV study to construct a diabetes Genomic Risk score (GRS) using the alleles associated with diabetes (European ancestry cohort) at different threshold of significance, to investigate whether the aggregate effect of GWAS-derived diabetes variants, as a proxy of altered energy metabolism, contributes to the risk for completed suicide in our sample of post-mortem human brains of European ancestry, including suicides. We hypothesize that diabetes GRS contributes to the prediction of case-control status in our cohort.\",\n\"2021, Jun02\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Uher, Rudolf \",\n\"DALHOUSIE UNIVERSITY\",\n\"Genomic predictors of developmental psychopathology\",\n\"approved\",\n\"Mental illness often develops in adolescence or in young adulthood and continues to affect people through their life. To effectively prevent the development of mental illness, we need to predict which young people may be at risk. We proposing to use the Million Veterans Project as a source of genetic predictors that we will apply to a new group of young people with information on mental disorders. This results will evalute if a genetic test may help predict which young person is at risk and who may benefit form early preventive interventions.\",\n\"The aim of this project is to predict which young people are at high risk for developing mental disorders, such as major depressive disorder, bipolar disorder, and psychosis. We will test a set of polygenic scores as predictors of developmental psychopathology in a cohort of youth with prospectively determined diagnoses of mental disorders. We would like to use genetic data from the Million Veterans Program to derive the polygenic scores for anxiety and for substance use. Although he genome-wide analyses of these traits have been published, we are required to obtain access to the dataset in order to obtain the summary statistics. We will apply the resulting polygenic risk scores to a cohort of over 200 youth to help predict anxiety, ADHD, depression, bipolar disorder, and substance use disorders. \",\n\"2023, Jan25\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Tripathy, Shreejoy \",\n\"CENTRE FOR ADDICTION AND MENTAL HEALTH\",\n\"Studying common genetic liability to substance use disorders across disparate substances\",\n\"approved\",\n\"Addiction and substance use disorders have a partially genetic basis. Genetic studies of substance use disorders have traditionally focused on a single substance at a time, for instance in studies of alcohol dependence, nicotine dependence, cannabis use disorder or opioid use disorder. However, recent research has shown that many of the same genetic factors may predispose people to all of these different substance use disorders. We plan to combine the results of multiple prior genetic studies of substance use disorders across different substances to try to find genetic variants and genes that influence substance use in general. Along the way, we hope to learn more about how substance use disorders influence and are influenced by other psychiatric and non-psychiatric disorders.\",\n\"Addiction and substance use disorders have a partially genetic basis. Genetic studies of substance use disorders have traditionally focused on a single substance at a time, for instance in studies of alcohol dependence, nicotine dependence, cannabis use disorder or opioid use disorder. However, recent research has shown that predisposition to these disparate substance use disorders can be accurately modelled by a single underlying genetic factor. We plan to combine the requested dbGAP summary statistics with other public summary statistics from outside dbGAP (e.g. from the Psychiatric Genomics Consortium) to perform a genome-wide association study of this underlying factor, a process which should not create any additional risks to participants as it preserves patient confidentiality. Through genetic correlation, Mendelian Randomization and post-GWAS gene prioritization analyses, we further plan to explore the interplay of substance use with other psychiatric and non-psychiatric traits, and uncover candidate causal genes involved in genetic liability to addiction and substance use disorders.\",\n\"2021, Dec03\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Speed, Doug \",\n\"UNIVERSITY OF AARHUS\",\n\"Using MVP summary statistics to construct polygenic risk scores and perform gene-based association testing\",\n\"approved\",\n\"Given an individual's DNA, it should be possible to acccurately predict their risk of developing different diseases, and decide the best course of medication should they become ill. However, for this to be possible, we must better understand the genetic factors that influence disease risk. The Million Veterans Project has performed genetic studies for a number of traits (e.g., cholesteral and coronary artery diseases). We will use the results from these studies to improve our understanding of the genetic basis of complex diseases.\",\n\"We would like access to the summary statistics produced by recent GWAS by the Million Veterans Project (MVP). Specifically, we are interested in the summary statistics for cholesterol (LDL and HDL), type 2 diabetes and when available coronary artery disease. We have two main uses for these summary statistics.   1. Construct polygenic risk scores (PRS) for these traits. Based on data from UK Biobank, we believe our software LDAK (www.ldak.org) is able to produce the most accurate PRS. We would like to test our software on MVP summary statistics. We will use these summary statistics to construct PRS, then test the accuracy of these on UK Biobank data (i.e., measure how well the PRS constructed from MVP can be used to predict cholesterol levels, type 2 diabetes and coronary artery disease in UK Biobank data).  2 - Perform gene-based association testing. It is normal to test SNPs (genetic variants) individually for association with the trait. However, LDAK includes a tool for testing groups of SNPs jointly (e.g., SNPs corresponding to a gene). We would like to test the performance of this tool on MVP summary statisitics. In particular, see whether it is possible to detect causal loci using our gene-based test that are not detected from single-SNP analysis.  We are only requesting summary statistics (not individual-level data) so there are minimal risks (e.g., it is almost impossible to identify individuals, and we will not try to). In order to run our two tools, we require summary statistics and a reference panel (the latter is used to estimate SNP-SNP correlations). For this we will use genetic data from the 1000 Genome Project or UK Biobank. Once we have constructed PRS, we will test their performance using data from UK Biobank (i.e, the PRS allow us to predict traits for UK Biobank individuals, then we can compare these predicted trait values to their actual values).\",\n\"Nov02, 2021\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Sofer, Tamar \",\n\"BRIGHAM AND WOMEN'S HOSPITAL\",\n\"Polygenic scores for a range of phenotypes across population background groups. \",\n\"approved\",\n\"This research will use results from genetic association analysis performed in a diverse population from the Million Veterans Program, to generate \\u00e2\\u0080\\u009cinstructions\\u00e2\\u0080\\u009d to create summaries of genetic data that explain and predict various phenotypes, such as blood pressure, lipids, and kidney function. These gene-based prediction equations will be adapted to and evaluated across different population groups. In the long term, this work will help increase the availability of genetic risk prediction tools to underrepresented minorities.\",\n\"The goal of the proposed research is to develop polygenic risk scores (PRSs) based on different background groups and combined (African Americans, Hispanic Americans, etc) that are associated with a range of phenotypes, such as blood pressure, kidney function, and lipids, and study their usefulness for explaining these phenotypes and for prediction such phenotypes across the different background groups.   For this purpose, we will use summary statistics from the GWAS of performed in MVP, including GWAS of blood pressure, and other phenotypes, to select genetic variants and estimates weights for the PRSs. To select the genetic variants, we will use methods that take into account the summary statistics from testing each genetic variant, and also how the various genetic variants correlate with one another (linkage disequilibrium). Linkage disequilibrium will be assessed from different, independent datasets, that will serve as \\u00e2\\u0080\\u009creference panels\\u00e2\\u0080\\u009d.  We will evaluate the PRSs association with these traits in independent datasets.  This research will help determine how to construct PRSs that one day may be used in clinic for risk prediction, while benefitting populations that are underrepresented in current genetic medical research. This research purpose is consistent with the data use agreement, because this is a biomedical research that will not be used to study the population origin or ancestry.   The summary statistics (the data that we will download) will be used only in the PI\\u00e2\\u0080\\u0099s institution. \",\n\"2019, May22\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Sofer, Tamar \",\n\"BETH ISRAEL DEACONESS MEDICAL CENTER\",\n\"Polygenic scores for a range of phenotypes across population background groups. \",\n\"approved\",\n\"This research will use results from genetic association analysis performed in a diverse population from the Million Veterans Program, to generate \\u00e2\\u0080\\u009cinstructions\\u00e2\\u0080\\u009d to create summaries of genetic data that explain and predict various phenotypes, such as blood pressure, lipids, and kidney function. These gene-based prediction equations will be adapted to and evaluated across different population groups. In the long term, this work will help increase the availability of genetic risk prediction tools to underrepresented minorities.\",\n\"The goal of the proposed research is to develop polygenic risk scores (PRSs) based on different background groups and combined (African Americans, Hispanic Americans, etc) that are associated with a range of phenotypes, such as blood pressure, kidney function, and lipids, and study their usefulness for explaining these phenotypes and for prediction such phenotypes across the different background groups.   For this purpose, we will use summary statistics from the GWAS of performed in MVP, including GWAS of blood pressure, and other phenotypes, to select genetic variants and estimates weights for the PRSs. To select the genetic variants, we will use methods that take into account the summary statistics from testing each genetic variant, and also how the various genetic variants correlate with one another (linkage disequilibrium). Linkage disequilibrium will be assessed from different, independent datasets, that will serve as \\u00e2\\u0080\\u009creference panels\\u00e2\\u0080\\u009d.  We will evaluate the PRSs association with these traits in independent datasets.  This research will help determine how to construct PRSs that one day may be used in clinic for risk prediction, while benefitting populations that are underrepresented in current genetic medical research. This research purpose is consistent with the data use agreement, because this is a biomedical research that will not be used to study the population origin or ancestry.   The summary statistics (the data that we will download) will be used only in the PI\\u00e2\\u0080\\u0099s institution. \",\n\"May05, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Silveira, Patricia \",\n\"MCGILL UNIVERSITY\",\n\"Validation of a polygenic score of insulin receptor associated gene networks\",\n\"approved\",\n\"Complex traits such as mental health outcomes are polygenic.  This idea has led to the use of a genomic risk profile that captures the genetic burden as reflected by a set of \\u00e2\\u0080\\u0098risk\\u00e2\\u0080\\u0099 alleles for specific psychiatric disorders. In the current project we want to test developed a methodology for a computing total genetic risk based on gene networks of biological relevance.\",\n\"An individual\\u00e2\\u0080\\u0099s cumulative genetic risk, for a trait, can be summarized by the polygenic risk score (PRS); by multiplying the measured number of risk alleles at a locus by the effect size of the association between that genotype and outcome and summing over all SNPs at a certain significance threshold. Despite holding much promise the caveat of the PRS is that they do not provide information about the underlying network in which these genes operate nor the neurobiology of the disease. To overcome this we have developed a methodology for a \\u00e2\\u0080\\u0098refined\\u00e2\\u0080\\u0099 polygenic risk scores (PRS) of biological relevance. Insulin modulates synaptic plasticity, especially in dopaminergic neurons, influencing food intake, mood, impulsivity/inhibitory control, cognition and memeory. We extracted insulin receptor (IR) related gene network based on genes co-expressed with the IR in key brain regions such as the PFC or hippocampus. Based on the gene networks extracted we created two refined PRS-IRs (termed PRS-IRPFC & PRS-IRhippo). These were associated with the functional readouts of the specific brain regions- impulsivity for the PFC and memory related tasks for the hippocampus- in children. The Maternal Adversity, Vulnerability And Neurodevelopment cohort-a longitudinal study of Canadian mother-child dyads-was used for this analysis. In the current project we will validate these findings in an independent adult cohort.   The aim of this study is twofold: Validate the refined PRS-IRs in an independant dataset. Use an adult cohort to access whether PRS-IRs capture the genetic risk for relevant psychopathology(substance use for impulsivity associated PRS-IRPFC,Alzheimer\\u00e2\\u0080\\u0099s disease for memory associated PRS-IRhippo).  Datasets selected 1.GenADA-These which will be used to validate of the PRS-IRhippo. 2. Study of Addiction: Genetics and Environment (SAGE)-This will be used to validate of the PRS-IRPFC.  Study Plan: SNP genotypes from the GenADA and SAGE datasets will be used to compute the PRS-IRhippo and PRS-IRPFC respectively.The association between the PRS-IRs and appropriate affection status will be assessed. Confounders such a gender and study design will be controlled for.\",\n\"Feb10, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Sherva, Richard \",\n\"BOSTON UNIVERSITY MEDICAL CAMPUS\",\n\"Genome-wide association study of substance dependence traits\",\n\"closed\",\n\"The risk for substance use disorders has a known genetic component. A genome-wide association study, in which millions of single base pair mutations throughout the human genome are tested for association with a trait, are one way to identify the specific genes responsible for this genetic risk. In this study, we will attempt to identify genes associated with the risk of alcohol, cannabis, opioids, and cocaine dependence.  \",\n\"Our research objectives are to identify susceptibility genes for substance use disorders, including alcohol, cannabis, opioids, and cocaine dependence through genetic association. The data requested will serve as a replication sample to validate associations observed in two samples: the Yale-Penn sample and SAGE (both already in dbGaP). Our primary analyses will use the DSM-IV criteria counts for the substances listed as ordinal outcome measures. We will use SNP minor allele dosage as a predictor variable. Yale-Penn and SAGE have been combined as a discovery sample and we will first test a small number of regions in the dataset we are requesting in an attempt to replicate these findings. We will also conduct a full GWAS in the requested data. Results will be combined with the discovery sample via meta-analysis. We will impute the requested data to the current 1000 Genomes reference panel.\",\n\"Jul13, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Shah, Sonia \",\n\"UNIVERSITY OF QUEENSLAND\",\n\"Using statistical genomics approaches to identify potential therapeutic compounds for treatment of Abdominal Aortic Aneurysm\",\n\"approved\",\n\"A genome-wide association studies (GWAS) aims to identify the genes that are relevant to the development of disease, which gives us better understanding of disease biology. Many cutting-edge statistical genomics methods have been developed that can be applied to data generated from GWAS for various purposes, including identifying drug targets for the disease as well as identifying existing drugs that could be repurposed for the disease in questions. This study aims to use such methods to identify potential drug targets and therapeutic compounds for the treatment of AAA. \",\n\"Objective: Using statistical genomics approaches to identify potential therapeutic compounds for treatment of Abdominal Aortic Aneurysm  Analysis Plan: Using methods such as MetaXScan that integrate expression quantitative trait loci (eQTL) data with AAA GWAS summary data that predict gene expression changes between cases and controls, we will first generate a gene expression signature for AAA using blood eQTL and GTEx aorta data. Next, we will use this gene expression signature query the connectivity map, a database of compound gene expression signatures, to identify compounds that are most likely to reverse AAA disease gene expression signatures, and may therefore show potential for treatment of AAA. We will also use Mendelian randomisation approaches to identify potential drug targets for AAA.  All analyses only require GWAS summary data, and no individual level data is requested. \",\n\"2022, Jul06\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Schwarz, Emanuel \",\n\"CENTRAL INSTITUTE OF MENTAL HEALTH\",\n\"Identification of Genetic Schizophrenia Subgroups\",\n\"approved\",\n\"Psychotic disorders are among the most serious mental illnesses and major causes of disability, absence from work and early retirement. A major problem is that current diagnostic criteria are based on symptom and clinical course criteria, which, almost by definition, do not map to the underlying biology. As a consequence, genetic illness associations regularly cross diagnostic boundaries and are hallmarked by small effect sizes due to the polygenic disease architecture and heterogeneity of diagnostic constructs. In this project, we will use the dbGaP data to identify and validate biological subgroups of schizophrenia patients. We will investigate the polygenic nature of such subgroups and explore their relevance for brain function using brain imaging data. This study will yield important insights into the biology of schizophrenia and related mental illness, and lay the groundwork for larger-scale follow-up experiments.\",\n\"A major problem for schizophrenia research is that current diagnostic criteria are based on symptom and clinical course criteria, which, almost by definition, do not map to the underlying biology. As a consequence, genetic illness associations regularly cross diagnostic boundaries and are hallmarked by small effect sizes due to the polygenic disease architecture and heterogeneity of diagnostic constructs. Our group is developing machine learning approaches to identify accurate classifiers using data from individual, as well as multiple modalities of high-dimensional data. We are integrating external information sources on biological systems-knowledge (such as protein-protein interaction, co-expression information, as well as expression data from the BrainCloud or GTX datasets) to stratify the high-dimensional data space. We have already shown that using this strategy, developed classifiers outperform conventional machine learning strategies on genome-wide methylation data and are now in the process of adapting these approaches to genome-wide association data. For this, we will apply algorithms on dbGaP data (such as the Molecular Genetics of Schizophrenia datasets). We will furthermore integrate these datasets with other datasets (i.e. methylation or gene expression) to derive multi-modal classifiers or perform cross-modality prediction. Profiles will further be mapped to differences in brain structure or function in an in-house MRI dataset. These integrative analyses will not create any additional risks to participants. We will employ graph theoretic and machine learning methods to identify subgroups and their polygenic profiles. We will further test developed methods on other (related) mental illness, such as alcohol use disorders, as a basis for the identification of illness-specific and overlapping biological hallmarks. This research may yield important insights into the biology of schizophrenia related mental illness.\",\n\"Feb09, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Schwartz, Robert \",\n\"WEILL MEDICAL COLL OF CORNELL UNIV\",\n\"Analysis of cell types in the placenta.\",\n\"approved\",\n\"The placenta is important and the cell constituents that give rise to the placenta are important to study. Several diseases impact placental formation, cell composition, and function. We plan to examine the placenta for changes in cell composition and gene expression in several disease processes.\",\n\"The placenta is of great significance for human parturition, while the cell types in the placenta are still relatively understudied. We are currently examinging and performing scRNAseq on the placenta and want to reeveluate and confirm our findings with an established dataset. CMV is a common viral infection, especially among young children. Congenital CMV occurs when the infection is passed across the placenta from a pregnant woman to her developing baby. Some babies with congenital CMV infection show signs of disability at birth, while others are born healthy. Other babies who do not have evidence of congenital CMV infection but do have evidence of placental abnormalities or malformations. Our proposed research requires using the dataset of healthy (normal) placenta to identify the heterogeneous cell types present and present a normal baseline gene expression for each cell type.  Once we have single cell RNA sequenced CMV-infected placenta, we intend to use markers gained from identifying cell types in this dataset to annotate the cell types in our own infected single cell data and thus once this is achieved also compare and produce differential gene expression in each cell type between infected and non-infected cells and also the cell type composition over time points pre-birth. The main data we will be using will be gene count matrices which will be analyzed in Seurat R package along with our own counts data. Processed data in terms of differentially expressed genes, UMAP plots and cluster markers will be accessible only to internal collaborators. All internal WCM collaborators are listed on this application. There are no external collaborators.\",\n\"Sep18, 2020\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Schork, Andrew \",\n\"REGION HOVEDSTADEN\",\n\"Characterizing pleiotropy between cardio-metabolic and psychiatric phenotypes\",\n\"approved\",\n\"Psychiatric patients are known to exhibit excessive cardio-metabolic comorbidity which can lead to premature mortality and exacerbate poor health outcomes.  It is critical to understand the etiology of these comorbidities as this could improve care of an already at-risk population.  We will integrate results of large genetic studies of cardio-metabolic and psychiatric traits in the MVP study with a prospective longitudinal cohort of population controls and psychiatric patients to better understand the genetic contributions to cardio-metabolic health of psychiatric patients.\",\n\"We aim to systematically and comprehensively describe the shared genetic etiology between psychiatric and cardio-metabolic phenotypes.  We will use state-of-the-field and novel methods for assessing the overall genetic sharing (i.e. genetic correlation), testing the cross-predictive implications for (i.e., polygenic risk score analysis), and mapping individually pleiotropic loci (i.e., GWAS-like approaches) for cardio-metabolic and psychiatric phenotypes.   We will also look at predictors and mediators of longitudinal psychiatric and cardiovascular outcomes of individual population controls and psychiatric patients from the iPSYCH case-cohort study.  This study will integrate an extensive, broad catalog of GWAS summary statistics that include contributions from the \\u00e2\\u0080\\u009cVeterans Administration (VA) Million Veteran Program (MVP) Summary Results from Omics Studies\\u00e2\\u0080\\u009d project (phs001672.v4.p1), additional public GWAS data sources, and in-house data generated as part of the iPSYCH case-cohort study (PMID: 28924187).  We will pursue purely summary statistics comparison-based analysis to describe broad genetic sharing among these and other cardio-metabolic and psychiatric GWAS summary results.  We will also use the data obtained here to calculate polygenic risk assessments for the iPSYCH data cohort to study determinants of and predictors of heterogeneity for longitudinal psychiatric and cardio-metabolic outcomes.  This study will push forward our understanding of the etiology of excessive cardio-metabolic co-morbidities among psychiatric patients, extending the biobank\\u00e2\\u0080\\u0099s aims towards improved understanding health and disease in veterans to other populations, consistent with the aims of this resource..  This research will be conducted in-house as described above, with no plans to share or use data more broadly, and is in accordance with the Use Restrictions for this resource and will follow all regulations regarding data security, IP, and publication.\",\n\"Oct22, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Schmitz, Lauren \",\n\"UNIVERSITY OF WISCONSIN-MADISON\",\n\"The Impact of Socioeconomic Environments and Polygenic Risk on Alcohol Use\",\n\"closed\",\n\"Although there are very few genetic differences between people, researchers have been able calculate which of these small differences in people\\u00e2\\u0080\\u0099s DNA are related to certain health behaviors like drinking. When aggregated, these small differences form a polygenic risk score (PGS). PGSs can help researchers understand whether people are predicted to be at a greater risk for alcohol abuse based on their DNA. Our study will use these PGSs to understand how they interact with social environments and experiences (e.g., job loss) to shape alcohol consumption or abuse. For example, the degree to which job loss affects alcohol use may vary by an individual\\u00e2\\u0080\\u0099s underlying genetic predisposition. \",\n\"Social science research has rapidly incorporated genetic data into quantitative analyses to better understand how the small DNA variations between people can affect health. For many health behaviors, researchers have found null results of the effects of major life events (e.g., job loss, divorce, retirement) on health behaviors, such as drinking. However, with the advent of polygenic risk scores, researchers can examine if there is variability in drinking behaviors based on genetic risk.   This study will utilize gene by environment (GxE) methods to assess how different environmental exposures related to socioeconomic status affect drinking behaviors when interacted with an alcohol consumption polygenic score (PGS). The GWAS weights from the Million Veterans Project for alcohol use will be utilized to build whole-genome PGSs and incorporated into GxE statistical frameworks to examine whether drinking behaviors among young adults in the Add Health study and older adults in the Health and Retirement Study (HRS) vary by underlying genetic predisposition. The environmental exposures will be related to socioeconomic status and life events, while the phenotypes will be alcohol consumption or an indicator for alcohol use disorder (AUD).  \",\n\"Apr07, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Sanchez Roige, Sandra \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Genetic studies of substance use disorders\",\n\"approved\",\n\"We aim to investigate the genetics of substance use disorders (SUD) and its overlap with other comorbid conditions. We will use the results generated by the Million Veteran Program and the iNDiGO Consortium to further our understanding of how polygenic liability influences different substance use disorder phenotypes measured across cohorts that differ in their ascertainment (population- and hospital-based cohorts), and how it overlaps with other known comorbid conditions, such as impulsivity, externalizing, chronic pain. This work will inform genetic studies of substance use disorders and the complex web of medical comorbidities associated with such a debilitating condition.\",\n\"We are requesting to use GWAS summary statistics from various sources (the Million Veterans Program, MVP, and the iNDiGO consortium) to investigate the genetic signal of many substance use disorders (SUDs), and its correlation with other comorbid traits. We have assembled multiple substance use phenotypes, ranging from exposure to chronic use, dependence and misuse, in several population- and hospital-based cohorts, and performed genome-wide association studies (GWAS) of each of the measured phenotypes (not available in dbGaP). We will use standard statistical tools for polygenic analyses (e.g. polygenic risk scores, LDSC genetic correlations) to calculate the amount of genetic overlap across the different SUD definitions and other comorbid traits, such as chronic pain. We will also seek to perform meta-analyses of the correlated traits. We do not plan to combine datasets but rather use the provided summary statistics alongside our summary information. Phenotypes will include summary statistics for substance use disorders (opioids, alcohol, tobacco, cannabis, cocaine) and comorbid traits, such as impulsivity, gambling, externalizing. The use of this data is requested solely for health, medical and biomedical purposes and will not involve any commercial applications or patents. We will only be downloading the summary statistics and thus do not anticipate any additional risk to participants. The summary statistics will be used only within the PI\\u00e2\\u0080\\u0099s institution. We will publish or broadly share any findings with the scientific community. The proposed research is consistent with the Use Restrictions for the requested dataset, and we will follow all regulations therein. This work will validate alternatives to rigorous clinical ascertainment for genetic studies of substance use disorders, which could serve to achieve large sample sizes in a cost-efficient manner, advancing our understanding of substance use disorders. This work will also examine the genetic overlap between substance use disorders and other comorbid conditions, which could have significant clinical implications. \",\n\"Jun29, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Sanchez Roige, Sandra \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Genetic studies of opioid use disorders\",\n\"closed\",\n\"Genetic factors influence opioid use disorders, but genetic studies of opioid use disorders have been severely limited due to the complexity of assessing opioid misuse. We will use the results generated by the Million Veteran Program to further our understanding of how polygenic liability influences different opioid use phenotypes measured across cohorts that differ in their ascertainment (population- and hospital-based cohorts). This work will inform genetic studies of opioid use disorders that use questionnaire-based or hospital-based ascertainment as a complementary alternative to traditional clinical ascertainment.\",\n\"Genetic factors contribute to variability in opioid use and abuse, but studies of opioid use disorders have been severely limited due to insufficiently powered samples, high phenotypic heterogeneity, and controls with undocumented exposure history. We have assessed multiple opioid use phenotypes, ranging from exposure to chronic use, dependence and abuse, in several population- and hospital-based cohorts, and performed genome-wide association studies (GWAS) of each of the measured phenotypes (not available in dbGaP). We are requesting to use GWAS summary statistics from the Million Veterans Program (MVP) to determine how genetic variation captured by our different opioid use definitions contributes to individual differences in opioid use disorders as defined by the MVP, which will serve as the \\u00e2\\u0080\\u009cgold standard\\u00e2\\u0080\\u009d. The objectives of this project are two-fold. First, we will use standard statistical tools for polygenic analyses (e.g. polygenic risk scores, LDSC genetic correlations) to calculate the amount of genetic overlap across the different opioid definitions. Next, we will use the MVP data to investigate whether the findings derived from our samples replicate in the MVP cohort. We do not plan to combine datasets but rather use the provided summary statistics alongside our summary information. Phenotypes will include summary statistics for opioids and comorbid traits, such as impulsivity, gambling, externalizing. The use of this data is requested solely for health, medical and biomedical purposes and will not involve any commercial applications or patents. We will only be downloading the summary statistics and thus do not anticipate any additional risk to participants. The summary statistics will be used only within the PI\\u00e2\\u0080\\u0099s institution. We will publish or broadly share any findings with the scientific community. The proposed research is consistent with the Use Restrictions for the requested dataset, and we will follow all regulations therein. This work will validate alternatives to rigorous clinical ascertainment for genetic studies of opioid use disorders, which could serve to achieve large sample sizes in a cost-efficient manner, advancing our understanding of opioid use disorders.\",\n\"Dec09, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Salomonis, Nathan \",\n\"CINCINNATI CHILDRENS HOSP MED CTR\",\n\"Understanding the mechanisms of preterm labor\",\n\"approved\",\n\"To understand the precise mechanisms of preterm birth we have developed mouse models of preterm labor which nominate new pharmacological targets for therapy. These analyses will help verify their predicted importance and relevance to human biology.\",\n\"We wish to simply use the non-identifiable count-matrix level data and previously produced cell-to-cluster associations to validate predictions from a mouse model of preterm label, to validate human relevance relative to bulk RNA-Seq (wild-type and KO). We will perform differential expression comparing term and preterm populations in the human samples and evaluate overlapping changes in mouse.\",\n\"2021, Sep22\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"SUGIURA, AYUMU \",\n\"TAKEDA PHARMACEUTICAL COMPANY LTD\",\n\"Integrative analysis of PTSD and multiple neurological disorders\",\n\"approved\",\n\"Understanding genetic background of patients with Posttraumatic stress disorder (PTSD) is a critical step to elucidate the molecular mechanism of the disease. We would like to use dbGaP data to identify genetic markers, sub-populations of PTSD, and associations with other neurological disorders. It could help us to understand genetic background contributing crucial biological processes and potential roles in disease phenotypes. In the future, the obtained knowledge will be used to develop more effective and safer treatment for PTSD and other neurological disorders.\",\n\"Understanding genetic background of patients with Posttraumatic stress disorder (PTSD) is a critical step to elucidate the molecular mechanism of the disease. We will perform GWAS of PTSD to identify genetic markers, and conduct post-GWAS analysis using some tools and public resources such as eQTL and pQTL databases. Moreover, we will apply several analytical methods such as co-localization and Mendelian randomization to PTSD GWAS for identifying genetic markers especially associate with other neurological disorders. Potentially, a genetic marker associates with sub-populations could be identified. These genetic markers should be very helpful to clarify crucial biological processes of the sub-populations and detect sub-populations having specific genetic background. These analyses might be able to reveal the biological basis of PTSD and relationships with other neurological disorders. Also, we could superimpose in-house data on these results, and it potentially helps us to investigate responder/non-responder or safety markers associate with the sub-populations of PTSD. It could also help us to identify potential drug targets for PTSD patients especially belonging to the sub-populations. Furthermore, an integration of these results with other knowledge and genomics data such as epigenome and gene expression might help us to obtain better understanding of PTSD at the molecular level. We will strictly conduct these studies according to Data Use Restriction. No violation of Use Restriction is expected.\",\n\"2019, Oct23\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"SO, Hon-Cheong \",\n\"CHINESE UNIVERSITY OF HONG KONG\",\n\"Dissecting the genetic basis of complex diseases and phenotype prediction\",\n\"approved\",\n\"The advance of genotyping technologies enable us to interrogate a large number of genetic variants at the same time. Individual genetic variants have limited prediction power, but by combining information from multiple variant, predictive accuracy may be improved. In this project we aim to test various statistical and machine learning strategies to improve risk prediction in diseases by using genomics data, and other omics or clinical data if available. We will also attempt to discover new variants for diseases and study the interaction of genetic variants. \",\n\"With the advance of genotyping technologies and large-scale genomic studies, an increasing number of susceptibility variants have been found for complex diseases. However, individual genetic variants have limited prediction power and a multi-locus approach is often used for risk or disease phenotype prediction. In this project we aim to test different supervised and unsupervised statistical and machine learning methodologies, such as regularized regression, decision trees and support vector machines, in individualized disease risk prediction. The prediction will be based on genomics profiles, and other \\\"omics\\\" (e.g. transcriptomics, methylomics) or clinical data, if such data is available. The performance of different predictive models will be assessed and compared. We also aim to employ statistical learning methods to uncover new susceptibility variants and genetic interactions, and to understand the genetic architecture of complex diseases.   In this project we aim to request data access to a range of complex diseases. For studies that only allow disease-specific research, the data will only be used to study the genetic basis for the specific disease/trait of interest and develop predictive models for that disease.    The data accessed will be used only for biomedical research purposes. We will not attempt to reveal the individual identity/ancestry of any person or of their relationship to another person. Each dataset will only be used in line with the consent of the participants. Findings from our study will be published and shared with the scientific community. \",\n\"2019, Feb08\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"SMOLLER, JORDAN \",\n\"BROAD INSTITUTE, INC.\",\n\"Psychiatric Genomics Consortium Cross-Disorder Group v3 (PGC-CDG3): Probing multivariate genetic architecture across psychiatric traits\",\n\"rejected\",\n\"Psychiatric disorders overlap substantially at the genetic level. Cross-disorder genomic projects that aim to better characterize this genetic overlap then have the potential to produce key insights into the underpinnings of rampant psychiatric comorbidity. To this end, the current proposal aims to analyze the most up-to-date summary statistics across 14 major psychiatric disorders using four cutting-edge genomic methods: Genomic SEM, CC-GWAS, MiXER, and LAVA. These methods can be used to examine genetic overlap in aggregate across the genome, for specific classes of genes (e.g., genes that are expressed early in life), and at the level of individual genetic variants. By applying different methodoligcal approaches we aim to identify whether a convergent narrative emerges about the patterns of genetic overlap across disorders at varying levels of analysis. With respect to the number of disorders and utilized methods, this project aims to perform the comprehensive, cross-disorder psychiatric analysis to-date. We are specifically requesting permission to use the summary statistics that include the MVP cohort for: alcohol use disorder, opioid use disorder, anxiety disorder, major depressive disorder, and post-traumatic stress disorder.\",\n\"Building on the work of the PGC Cross-Disorder Working Group (PGC CDG2) in clarifying the genetic structure of psychopathology, we now aim to leverage multiple, multivariate methods to better define the shared and disorder-specific genomic underpinnings across 14 major psychiatric disorders. At the genome-wide level, we will specifically apply MiXER, Genomic SEM, and CC-GWAS. Using Genomic SEM, we will produce a factor model based on the genetic correlations across the 14 disorders with the aim of identifying subgroups of disorders that more strongly cluster together. MiXER then takes an alternative approach to quantify polygenic overlap by estimating the total number of shared and trait-specific causal variants, while CC-GWAS calculates the overall \\u00e2\\u0080\\u009cgenetic distance\\u00e2\\u0080\\u009d across two traits based on differences in allele frequencies for cases. Collectively, these results will help us determine whether a convergent narrative emerges about the overall pattern of genetic overlap across these disorders.  At the functional level, we will apply two new methods: Stratified Genomic SEM and LAVA. Stratified Genomic SEM builds upon the findings from Genomic SEM to examine multivariate enrichment at the level of the psychiatric factors. LAVA then examines local genetic correlations at the level of individual loci. In conjunction, these two methods allow us to identify specific categories of genes and loci that drive genetic overlap across disorders. Finally, at the SNP-level of analysis, we will use CC-GWAS and multivariate GWAS in Genomic SEM to identify SNPs that are both highly pleiotropic and disorder-specific. The cumulative set of findings will offer critical insights into the multivariate architecture of psychiatric disorders utilizing cutting-edge methods at three levels of analysis (genome-wide -> functional -> SNP). Indeed, the proposed project reflects the most comprehensive cross-disorder, psychiatric analysis to-date. We are specifically requesting permission to use the summary statistics that include the MVP cohort for: alcohol use disorder, opioid use disorder, anxiety disorder, major depressive disorder, and post-traumatic stress disorder.\",\n\"2021, Dec13\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Sorensen, Peter \",\n\"UNIVERSITY OF AARHUS\",\n\"Using correlated trait information for improved gene mapping and prediction of complex traits and diseases\",\n\"approved\",\n\"Current approaches for estimating an individual's genetic disease predisposition only explain a small proportion of the disease risk. We will increase mapping power and prediction accuracy for complex diseases such as Type 2 Diabetes (T2D) and Coronary Artery Disease (CAD) by developing statistical and bioinformatic approaches that take advantage of the rapid accumulation of functional genomic information in biological databases, and availability of large independently collected genotype and phenotype datasets for multiple complex diseases. Our project will provide genetic markers and functional marker groups associated with T2D and CAD disease risk that points towards biological important processes underlying the disease. This information can be used to identify potential drug targets in the early stages of the drug discovery process and can help priorities which of these drug targets should be further investigated. Furthermore utilizing genomic information in precision medicine for stratifying individuals based on their inherent genetic disease predisposition has great potential to identify high-risk patients enabling early-lifestyle and/or pharmacological interventions.\",\n\"The aim is to increase power of detecting genetic variants underlying disease risk by developing bioinformatics and statistical modeling approaches that 1) combines information on multiple correlated traits across ancestries, and 2) use information on functional marker groups to capture weak genetic signals within and across traits. We will apply novel statistical methodology to well-established and high-quality human genetic databases for T2D and CAD and we therefore believe that the project could generate significant new knowledge and potentially uncover novel therapeutic mechanisms and/or drug targets and more accurate genomic predictions for cardiometabolic diseases.  The discovery of novel genetic variants associated with complex diseases can help identify biomarkers and potential drug targets for the disease including 1) genes and regulatory elements located nearby disease-associated variants or biological pathway, and 2) biomarkers (transcriptome, metabolome and proteome) whose association signals are linked to disease associated signals.  \",\n\"2022, Nov23\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Speliotes, Elizabeth \",\n\"UNIVERSITY OF MICHIGAN\",\n\"Analysis of Nonalcoholic Fatty Liver Disease(NAFLD) and Cardiometabolic Traits in dbGAP Datasets\",\n\"approved\",\n\"With increasing obesity, Nonalcoholic Fatty Liver Disease (NAFLD) has reached epidemic proportions. Genetic variants have been found to associate with both NAFLD and cardiometabolic traits/diseases. We will use dbGAP datasets with measures of NAFLD to 1. identify new common genetic variants that associate with NAFLD 2. fine map and characterize NAFLD associated variants by performing larger more powered GWAS analyses in diverse ancestries with measures of NAFLD 3. test variants that have been found to reproducibly associate with cardiometabolic traits/diseases for effects on NAFLD and variants that associate with NAFLD for effects on cardiometabolic traits/diseases and 4. determine whether the effects of NAFLD-associated SNPs are altered by environmental variables. This work will define disease risk categories and environmental triggers of disease development, gene targets for therapeutic intervention, and equalize the benefits of genetic discoveries to diverse ancestries. \",\n\"Nonalcoholic Fatty Liver Disease (NAFLD) is a spectrum of disease that includes steatosis, steatohepatitis, and fibrosis/cirrhosis in the liver. Genetic variants have been found to associate with both NAFLD and cardiometabolic traits/diseases. These will be cross sectional analyses that will use dbGAP datasets where we have CT-measured hepatic steatosis (Amish, AGES, Family Heart Study, Framingham Heart Study, IRAS, Cardia, GENOA, MESA, Dutch Nelson, Jackson Heart Study, and COPD Gene) and genome wide data available (on over 22,000 individuals) to identify new common variants that associate with CT measured NAFLD using regression analyses controlling for confounders (age, gender, alcoholic drinks, principal components) in the individual studies. We will carry out fixed effect meta analysis across groups and replicate promising associations in histology diagnosed NAFLD in cases from the FLIP consortium and the Geisinger bariatric sample matched to publically available controls. We will also see whether these variants reproduce for effects on NAFLD related diagnoses in UKBiobank and Michigan Genomics Initiative (MGI) datasets that we have now also obtained. We will also meta analyze our data with association data from UKBiobank and MGI with dbGAP datasets to increase our power to identify reproducible associations.  Variants that reach genome wide significance or that replicate will be considered real associations. We will use cross ancestry meta analysis and  conditional analyses to help identify causal variants at associated loci. We will test variants that have been found to reproducibly associate with cardiometabolic traits/diseases for effects on NAFLD  and also test variants that have been found to reproducibly associate with NAFLD for effects on cardiometabolic traits/diseases in dbGAP datasets, histology datasets, and the UKbiobank and MGI datasets. We will determine whether the effects of NAFLD-associated SNPs are altered by environmental variables including anthropometric, serum laboratory, and dietary variables. We will expand our analyses of BMI and NAFLD to include analyses of the genetic and epidemiological contributions to all continuous and dichotomous human traits and diseases using regression and data mining methods.  We will examine the effects of associated variants on affecting gene expression in STARNET and other eQTL datasets as they become available. This work will define disease risk categories and gene targets for therapeutic intervention.  Please see 2014 approval for method details.\",\n\"Dec19, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Torkamani, Ali \",\n\"SCRIPPS RESEARCH INSTITUTE\",\n\"Identification of Genomic Risk Factors and Prediction of Cardiovascular Disease through Integrative Network Analysis and Machine Learning\",\n\"approved\",\n\"Our overarching goal is to pursue integrative multi-omics and systems level analyses of whole genome sequencing data from deeply phenotyped cohorts to define novel coronary artery disease biology, and to develop coronary artery disease risk prediction models that combine these discoveries with other polygenic and clinical features. While prior studies have shown basic coronary artery disease prediction models provide modest clinical utility, improved, comprehensive models can make this a reality of clinical practice. This information can be used to help guide therapy decisions and provide an impetus for optimizing lifestyle modifications, thereby improving health outcomes and clinical efficiency.\",\n\"Genome-wide association studies (GWAS) and other genetic studies have identified hundreds of genetic variants associated with coronary artery disease (CAD). Polygenic risk scores (PRS) limited to known CAD loci have been shown to be useful for the identification of high-risk individuals who could benefit most from initiation of statin therapy or lifestyle changes. However, currently developed PRSs have some limitations. These models cannot accurately predict which individuals will develop disease, and fail to: (1) incorporate rare pathogenic genetic variants, (2) consider sub-genome-wide significant genetic variants implicated in CAD pathogenesis through integrative analysis, (3) incorporate traditional clinical risk factors, (4) consider higher level interactions between loci, and (5) evaluate the utility of these models in non-European individuals. These limitations, individually and in combination, limit the clinical utility of PRSs in individual risk prediction and can only be overcome by the development of more sophisticated risk models. The goals of this proposal are: (1) to combined polygenic, rare variant, and clinical risk factor score for multi-ethnic prediction of coronary artery disease and myocardial infarction. (2) To identify novel genomic associations for coronary artery disease and myocardial infarction through multi-omic network enrichment analyses. (3) To integrate these novel genomic associations, and known genetic and non-genetic risk factors for prediction of coronary artery disease, myocardial infarction, and therapeutic intervention outcomes into standard PRS, regularized regression, and machine learning frameworks.  Analyses may include the Wellcomme Trust Case Control Consortium, CARDIoGRAMplusC4D, UKBiobank and other non-dbGAP data. No attempt will be made to cross reference individuals across databases.  Results of this research will be published and distributed broadly.  Note(1): general research use datasets will be used for imputation and to train other genomic encoding approaches.  Note(2): Our previous Research Use Statement and personnel included collaborators from Intel Corp, however, they have not been able to navigate the process to acquire ERA Commons + dbGAP accounts leading to rejection of our data request. Intel Corp is removed from this request and access to data.\",\n\"Oct09, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Spracklen, Cassandra \",\n\"UNIVERSITY OF MASSACHUSETTS AMHERST\",\n\"Overlapping MVP T2D Summary Statistics with caQTL and eQTL results in Pancreatic Islets\",\n\"approved\",\n\"I would like to use the summary results files so that I can see the full set of results for all of the DNA variants tested for association with T2D (not just the most significant variants reported in the main publication). I will also use the summary data to generate graphs/plots of the results.\",\n\"I will use the T2D summary statistics from the trans-ancestry and single-ancestry analyses to look-up single-variant results from my current work overlapping T2D GWAS hits with caQTL and eQTL results from human pancreatic islets. I also intend to use the summary statistic files to generate Locus Plots for visualization of the GWAS signal.\",\n\"2021, Nov16\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Thorgeirsson, Thorgeir \",\n\"DECODE GENETICS, EHF\",\n\"Genetic Risk Factors For Addiction\",\n\"rejected\",\n\"We aim to understand the genetic factors conferring risk of nicotine dependence, alcoholism, and addiction in general. We plan to accomplish this by combining data sets available from dbGAP with our own data sets. We will then search for variants associated with various kinds of addiction in the combined data sets. Both common and rare variants, and both single base changes in the genetic code and structural polymorphisms such as duplications and deletions will be examined.\",\n\"Our project aims to use genetic information to aid in understanding the biological processes involved in addiction. To do this, we plan to identify genetic risk factors for addiction by using the large data sets resulting from the combination of information from the requested data sets and from Iceland. Phenotypes to be studied include nicotine (tobacco) addiction, alcohol and substance use disorders. A combined addiction phenotype will also be examined, using data sets permitted to be used for this more general phenotype. Both the requested data sets and the Icelandic data sets will be imputed using reference sets from sequencing projects such as the 1000-genome project; association analyses will be conducted; and the results from the different data sets will be combined using standard meta-analyses techniques. The proposed research is consistent with Use Restrictions for the requested data sets, and we will follow all regulations therein.\",\n\"2019, May17\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Thom, Christopher \",\n\"CHILDREN'S HOSP OF PHILADELPHIA\",\n\"Genetic interactions between blood traits and complex disease. \",\n\"approved\",\n\"Blood cell biology is incredibly important yet not well understood. Complex physiology impacts blood cell formation and function in vivo, but is rarely considered in the context of blood trait genetics. The goals of this project are to genetically define how blood traits and complex disease traits are related, including a directional estimation to determine if disease traits impact blood traits, or vice versa. After defining trait associations, we will use statistical genetics approaches to define loci and genes that can account for the interrelatedness of traits in order to understand underlying genetic mechanisms. Further, we will use established data sets to test gene-specific hypotheses based on our model system work. In sum, this project will better define blood cell-intrinsic and systemic mechanisms that contribute to genetic inheritance of blood traits and related complex disease. Our findings will be relevant for understanding blood cell biology, complex disease physiology, and related therapeutic opportunities.  \",\n\"Mechanisms governing blood cell formation and function are critical yet incompletely understood. In vivo, blood cell formation (hematopoiesis) is influenced by myriad systemic and environmental factors. Blood cell biology also influences complex disease development, severity and physiology. Genome wide association studies (GWAS), patient studies, and model systems interrogating have defined thousands of genomic loci that impact blood traits. However, molecular mechanisms and trait interactions are not well defined. The goal of this project is to use statistical genetics approaches to discover novel trait associations and integrated multi-omics methods to define processes that influence, or are influenced by, blood cell physiology.   Prior work based on publicly available GWAS defined novel functional loci and related target genes, which were subsequently validated using murine and stem cell model systems. We have used statistical genetics approaches, including colocalization and Mendelian randomization (MR) methods, to define novel links between blood, cardiovascular, metabolic, autoimmune, psychiatric, oncologic, and behavioral (e.g., tobacco and alcohol use) traits. The project defined herein will expand on these prior studies and related hypotheses.    Using the MR framework, we will comprehensively define associations between quantitative blood traits (e.g., blood cell counts and percentages), blood cell function (platelet reactivity), and disease outcomes. These disease outcomes will include cardiovascular (coronary artery disease, heart failure, stroke, pulmonary embolism, peripheral artery disease, atrial fibrillation), metabolic (type 2 diabetes, lipid and cholesterol levels), immune/autoimmune (asthma, eczema, rheumatoid arthritis, lupus), psychiatric (schizophrenia, bipolar disorder, neurodegenerative), and oncologic disease predisposition, as well as anthropometric (height, body mass index, waist-to-hip ratio) and behavioral/lifestyle (tobacco use, alcohol use, circadian rhythms, socioeconomic status, education level) traits. After establishing causal inferences between traits, we will define relevant organ systems, cell types, genes and genomic loci using genetic colocalization and statistical fine-mapping approaches.    Prior GWAS and our studies have also prioritized genes that we anticipate directly influence blood cell formation and/or function, including but not limited to Tropomyosin 1 (TPM1). We will specifically query for coding mutations in this and other high priority target genes among whole exome/whole genome sequencing cohorts to identify naturally occuring mutations and related phenotypes that can help explain gene functions to hematopoiesis and other systems.  \",\n\"Feb23, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Tcheandjieu, Catherine \",\n\"J. DAVID GLADSTONE INSTITUTES\",\n\"genetic determinant for ASCVD\",\n\"approved\",\n\"my proposal require access to genetic results published by the million Veteran to create methods for disease risk prediction that is applicable and beneficial for people from all ancestral background. \",\n\"Despite decades of research, Atherosclerosis cardiovascular diseases (ASCVDs) remain the leading cause of death worldwide. The lack of progress is at least partially related to the disproportionally higher incidence of ASCVD in minority groups. A higher prevalence of non-genetic ASCVD risk factors in some minority groups is a major contributor to this disparity and requires substantial attention. However, an advanced understanding of the genetic susceptibility to ASCVD in minority groups will also help to achieve progress. Unfortunately, while genetic determinants of ASCVD have been identified among individuals of European ancestry including the development of polygenic risk scores (PRS), very little is known about the genetic risk of ASCVD for other populations. This is consequential, because current PRSs, as shown in my previous work as well as others, do not have transferability across ancestry groups. Thus, although PRSs show promise as a precision medicine tool to advance diagnosis, prevention, and treatment of ASCVD, they also have a strong potential in their current state to exacerbate preexisting ASCVD disparities. Based on my previous study, I hypothesize that studying populations with diverse genetic backgrounds will not only reduce inequity in the applicability of genomic precision medicine but will also substantially improve our understanding of the underlying ASCVD biology. My goal is to leverage genetic diversity across global populations to (1) better understand the biology of ASCVD and (2) develop novel approaches for ASCVD precision medicine. I would like to access the summary statistic of various phenotype from the multi-ancestry cohort of the Million veteran project to create Polygenic risk score for ASCD that also include their risk factors. \",\n\"Jan18, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Tang, Hua \",\n\"STANFORD UNIVERSITY\",\n\"Learning Genetic Architecture of Complex Traits Across Populations\",\n\"approved\",\n\"Genome-wide association studies (GWAS) have become a primary tool for understanding the genetic basis of complex traits and diseases.  The goal of this research is to advance our understanding of cardiovascular diseases in minority populations, so that GWAS can benefit all individuals. Specifically, by leveraging information across diverse populations, our analyses aim to identify minority-relevant genetic variants and to build polygenic risk scores (PRS) for minority individuals.  \",\n\"Objective. Understanding the genetic factors influencing complex traits in minority populations is challenging because the sample sizes areoften much smaller in Genome-wide association Studies (GWAS) of minority populations, compared to GWAS in European populations. As described below, we have previously developed minority-centric association mapping and genetic risk prediction approaches. The key insight underlying these approaches is to leveragesmulti-ethnic and trans-ethnic information, while acknowledging heterogeneity across populations. The objective of this dbGaP application is to apply and extend these methods to identify identifying minority-relevant genetic factors underlying plasma lipids, stroke and VTE in African Americans and Hispanics. We will also build population-specific polygenic risk score (PRS) for lipids and CAD (stroke and VTE).    Study Design. For association mapping, we will apply an empirical Bayes approach developed by Dr. Tang\\u00e2\\u0080\\u0099s group, XPEB (Coram et al.2015; \\u00c2\\u00a0PMID: 25892113). This method considers a study design, in which summary-level statistics from two GWAS are available: one in the minority population of primary interests (referred to as the focal population), and the other in a reference population of much larger sample size(referred to as the base population). As an example, to identify genetic loci associated with low-density lipid cholesterol (LDL) in African Americans, we will generate the focal GWAS summary-statistics by meta-analyzing an African American LDL study performed in the Women\\u00e2\\u0080\\u0099s Health Initiative (WHI) with the African American LDL analysis in the Million Veterans Program. (All investigators on this, and the collaborating, applications have access to WHI).For base GWAS, we will use MVP\\u00e2\\u0080\\u0099s European-stratified LDL summary statistics. For PRS, we will apply a mixed-effects model approach developed by Dr. Tang\\u00e2\\u0080\\u0099s group, XP-BLUP (Coram et al. 2017; PMID:\\u00c2\\u00a028757202).This method considers a study design, in which individual-level genotype-phenotype data is available in a minority cohort (the focal GWAS), while only summary-level GWAS statistics are available in the base GWAS. In the proposed study, WHI individual-level African Americans and Hispanic American GWAS data are available to all investigators (and through dbGaP).    Analysis plan. For association mapping, we will apply XPEB(implemented in an open source R package) \\u00c2\\u00a0for each of the lipids and CAD traits, andusing African Americans or Hispanics as the focal population. \\u00c2\\u00a0For PRS, XP-BLUP program, implementing the mixed-effects model, is available on github. Specifically, we will include age and genetic-PCas covariate for all analysis. In addition, trait-specific covariates will be adjusted as needed. \\u00c2\\u00a0Validation/Replication: For minority-relevant loci identified using XPEB, we will seek replication in the appropriate minority-populations. Because we anticipate some loci identified by XPEB feature moderate/weak effect sizes, other minority-cohorts are under powered.As an alternative, we will examine consistency in allelic effects and assess the improvement in disease risk prediction. For minority-specific PRS, we evaluate the predictive performance on an independent cohort of matching minority population (African Americans or Hispanics). We will compare PRS built byXP-BLUP with PRS directly transferred from European cohorts.    The DUL states \\u00e2\\u0080\\u009cuse of this data is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry. Use of the data includes methods development research(e.g., development of software or algorithms).\\u00e2\\u0080\\u009d Our analysis will use the population-specific GWAS results produced by the MVP; no analysis is proposed to study the population origin or ancestry. \\u00c2\\u00a0Therefore, this application is consistent with the DUL.   Collaboration with researchers at other institutions, if appropriate.The proposed research will be performed in collaboration with Drs. Charles Kooperberg and Alex Reiner\\u00e2\\u0080\\u0099s groups at the Fred Hutchinson Cancer Research Center (FHCRC). \\u00c2\\u00a0Drs.Kooperberg/Reiner are submitting a dbGaP application through FHCRC.\\u00c2\\u00a0\",\n\"Apr29, 2021\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"TUSCHL, THOMAS \",\n\"ROCKEFELLER UNIVERSITY\",\n\"Reanalyze single-cell data to better resolve cell types of the placenta\",\n\"closed\",\n\"Reanalysis of cell composition of placenta at different stages of pregnancy in preparation to evaluate if 2nd trimester intermediate state deserves future experimental analysis\",\n\"We wish to reanalyze single-cell and RNA-seq placenta data of the 3rd trimester for comparison to data we have generated for 1st and 2nd trimester. To best compare scRNA datasets they need to run through the same pipeline or be combined and then run through the same analysis. The statement above was requested to be expanded upon 1st review. In order to compare two single cell studies performed by 2 investigators at different sites, it is important to use the same bioinformatic pipeline for mapping sequence reads as well as setting thresholds to identify cell types and subtypes, e.g. separating cytotrophoblasts from syncytiotrophoblasts. also, we developed a analysis process, which requires multiple reads to support the unique molecular identifier (UMI) when counting transcripts per cell. in short, when comparing 2 datasets, both of which are published, access to primary sequencing data is necessary. we were planning to conduct a single-cell sequencing study using 2nd trimester placenta, and first determining if there were differences between 1st and 3 rd trimester placenta cell types is a critical step required to be shown as preliminary data in a NIH grant application for study of 2nd trimester placenta.\",\n\"2020, Jul27\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"TUSCHL, THOMAS \",\n\"ROCKEFELLER UNIVERSITY\",\n\"Curated Human Transcriptome\",\n\"approved\",\n\"Many issues currently confound the quantification of Gene Expression. As yet,  reference transcript sets are not completely non-redundant (there are many issues of multi-mapping), they do not have a representative one transcript per gene (multiple transcripts are proposed to cater for alternate splicing) and variant nucleotide positions are computationally predicted and so tend to be incorrect fairly frequently. Accurate and unique mapping of sequence fragments from RNA sequencing datasets uniquely to transcripts is absolutely critical. And towards that goal, it is essential to have a well-curated and well-annotated set of transcript sequences and all their variants. We propose to create such a Curated, Annotated, Comprehensive, Non-Redundant, Coding and Non-Coding Human Reference Transcriptome. Accurate Gene Expression reports (and normal Gene Expression ranges of all genes across all known samples) and displays of alignments and resolving any multimapping is absolutely critical for making any biological hypotheses\",\n\"Mine datasets in the public domain to curate and annotate a Human Reference Transcriptome: As more diverse tissues and cell types are sequenced, genes that are expressed in rare conditions are available for curation and mining. Based on read evidence, allelic variations and junctions are created using our software platform, UTRs and ORF are annotated for coding genes. Precursors, mature non-coding sequences and special features like loop structures and other secondary structures are annotated on the sequence. We would be sequentially mining each dataset in our request separately for alternate splicing and allelic variations to enrich our reference transcriptome by extending our catalog of genetic variation and annotated transcripts. The datasets will be analyzed separately but kept together in one RU-located database. We are mining common (non-pathogenic) sequence variation and splice variation to complete a curated reference genome (and do not intend to discover genetic diseases outside dataset specific user agreements). The RU-located database is kept in a secure supercomputing cluster at RU. This repository contains primary sequence reads from controlled access RNA-seq datasets that have undergone minor processing. The processed and trimmed reads from the dbGAP datasets are made available for viewing one gene at a time using a small program; complete datasets are never exposed. This repository complies with NIH security guidelines. There are no collaborators.\",\n\"Aug17, 2020\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"TAYLOR, KENT \",\n\"LA BIOMED RES INST/ HARBOR UCLA MED CTR\",\n\"TOPMED Data Analyses\",\n\"approved\",\n\"Data will be used to develop a genetic test to predict cardiovascular disease in woman and African Americans.\",\n\"Background We have recently demonstrated that genomics risk scores for cardiovascular disease, based on the results of GWAS, do not predict cardiovascular phenotypes in women from African and European descent (Hajek et al, 2018. Circ Genom Precis Med. 2018 Oct;11(10):e002324.  doi: 10.1161/CIRCGEN.118.002324.  Objectives for Project * Development of regularized matrix decomposition methods for analyses of multiple phenotype and omic datasets * Identification of genomic variants for prediction of cardiovascular outcomes in the MESA cohort with focus on women from African and European descent * Perform confirmation studies of matrix results using women from the Framingham Heart Study, the WHI, and the Jackson Heart Study  Study Design Case control omic analyses of epidemiological studies  Analysis Plan In all analyses, covariates are age and BMI. Gender and race are not included because the we will be analyze women by African or European descent separately. 1. regularized matrix decomposition will be applied to the MESA cohort (R package glmnet). This analysis perform a 10-fold cross validation of prediction error in order the determine the penalty to be used in the final analysis to determine genetic risk score. 2. The resulting genetic risk score will be applied to the FHS and JHS cohorts validation using logistic regression.  I recognize that the development of a genetic risk score will be of value in the clinical setting. However, I have every intent to publish the results of the study described here and make results available to my colleagues and to the public. The datasets additional to TOPMed are for confirmation studies in two-stage analytic methods. \",\n\"2021, Aug24\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"TAYLOR, KENT \",\n\"LA BIOMED RES INST/ HARBOR UCLA MED CTR\",\n\"NHLBI TOPMed Whole-genome Sequencing Program (TOPMed WGS)\",\n\"approved\",\n\"The goals of the TOPMed Whole Genome Sequencing program are to identify genetic risk (and protective) factors for disease, to identify subtypes of disease, and to develop targeted and personalized treatments for disease.  The program consists of multiple studies addressing these goals for phenotypes focused on heart, lung, blood and sleep disorders, which have a high impact on public health.  These studies will include individuals from diverse populations, including those historically under-represented in health research.\",\n\"TOPMed Generic Research Use Statement  NOTE TO REVIEWER OF THIS APPLICATION: I AM COLLABORATING WITH MULTIPLE WORKING GROUPS IN MESA, TOPMED, AND CHARGE. I CERTIFY THAT AT NO TIME WILL I SHARE DATA DOWNLOADED FROM DBGAP WITH OTHER INVESTIGATORS NOT LISTED IN THIS APPLICATION. IN THESE GROUPS WE PERFORM ALL OF OUR ANALYSES BY SHARING SUMMARY DATA ONLY AND PERFORMING META-ANALYSES.  Revised 12/21/17  Phenotypic and genomic data from TOPMed studies will be used for: (a) investigating genotype-phenotype associations; (b) variant summary statistics, (c) common controls in association tests, (d) reference panels to impute into un-sequenced individuals from TOPMed; and (e) population genetic studies.  TOPMed data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Data from an individual with disease-specific consent will not be used in analyses outside of that restriction, including indirect uses such as imputation reference panels, common controls for association studies and variant summary statistics. For example, an individual with consent restricted to lung function studies will not be used to impute data intended for analyses of diseases not clearly related to lung function.  When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish or otherwise broadly share any findings from this study with the scientific community.  This application anticipates collaboration among TOPMed investigators from multiple institutions, who are named in a master list appended to the IRB document uploaded with this application. Within a set of applicants approved for access to the TOPMed Exchange Areas, data may be shared in a cloud environment or by direct transfer, provided that they all agree among themselves to allow such sharing, and that all have approved TOPMed Data Access Requests for the specific data sets to be shared. Otherwise, data will be shared only among Internal Collaborators \\u00e2\\u0080\\u0093 i.e. within an institution.  Data from TOPMed may be combined with non-TOPMed data from the same or other studies (obtained from dbGaP or other sources), to improve the power for novel genetic discoveries, while respecting the consent of all participants. We expect that this activity creates no additional risks to participants. \",\n\"2021, Jun07\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"TALL, ALAN \",\n\"COLUMBIA UNIVERSITY HEALTH SCIENCES\",\n\"Causal Analysis of Lipid Traits and Other Metabolic Syndrome Traits in Coronary Artery Disease\",\n\"closed\",\n\"Coronary artery disease is the leading cause of death in the United States, accounting for 28% of overall mortality. Low-density lipoprotein cholesterol, high-density lipoprotein cholesterol, and triglycerides have been identified as risk factors for coronary artery disease. Therapies targeting high-density lipoprotein cholesterol and triglycerides have so far failed to improve coronary artery disease outcomes. It is unclear whether individuals with low high-density lipoprotein cholesterol and high triglycerides cause coronary artery disease, or whether correlated metabolic traits such as obesity, blood pressure, and type 2 diabetes mediate these effects. We will use genetic variants that modify high-density lipoprotein cholesterol and triglycerides identified in the Million Veterans Program study to assess whether these traits cause coronary artery disease. This study will determine whether high-density lipoprotein cholesterol or triglycerides represent suitable targets for the next generation of preventative therapies in coronary artery disease. \",\n\"Coronary artery disease is a significant cause of morbidity and mortality in the United States, particularly in individuals with high low-density lipoprotein cholesterol (LDL-C), low high-density lipoprotein cholesterol (HDL-C), and high triglycerides (TG). The objective of this research is to determine the causal effects of variation in HDL-C and TG on coronary artery disease risk using Mendelian randomization. Previous studies have used genetic variants in LDL-C, HDL-C, and TG to model causal effects of these traits and reported strong evidence for a causal role of LDL-C, moderate evidence for a causal role of TG, and no evidence for a causal role of HDL-C in coronary artery disease. However, we have found that more than 20% of genetic variants affecting HDL-C and TG also modify other metabolic traits that influence coronary artery disease risk, such as body mass index (BMI), systolic blood pressure (SBP), and type 2 diabetes risk (T2D). We have acquired external datasets for BMI, SBP, and T2D genetic associations.   We will develop new multivariable Mendelian randomization methods that account for these pleiotropic effects by regressing coronary artery disease risk on LDL-C, HDL-C, TG, BMI, SBP, and T2D in a multivariate model using MR-inverse variance weighted and MR-Egger methods. We will also perform sensitivity analyses in which variants with pleiotropic effects on specific traits are excluded. Analyses will be performed in R 3.6.3 and will not require the use of cloud computing. We will use the Million Veterans Project genomic association summary results for blood lipids (LDL-C, HDL-C, and TG) to identify a sufficiently large set of lipid trait variants to model coronary artery disease risk effects of each trait after adjustment for pleiotropy. Use of this specific disease dataset will clarify the causal role of HDL-C and TG in coronary artery disease and establish one or more of these traits as targets for therapeutic development in coronary artery disease. The use of this data for methods development is consistent with the allowed use specified in the Data Use Restrictions. We will not perform research on population origins or ancestry. In accordance with the specifications of the Data Use Certification Agreement, we are not requesting any individual-level data. Results will be widely shared in published manuscripts. We will not collaborate with any external collaborators on this project.  \",\n\"Apr16, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Sundstr\\u00c3\\u00b6m, Johan \",\n\"UPPSALA UNIVERSITY\",\n\"Investigating the causal relationship between lung function and atherosclerosis\",\n\"approved\",\n\"Individuals with impaired lung function are at increased risk of CVD. The relationship between lung function and CVD persists even after adjustments for shared risk factors such as smoking and inflammation, and has also been observed in studies restricted to never-smokers. However, whether impaired lung function leads to CVD or vice versa is currently not known. We will investigate this causal relationship, which might reveal the underlying pathophysiology required for the development of therapeutic interventions, using summary statistics from published large scale genome-wide association studies.\",\n\"Individuals with impaired lung function are at increased risk of CVD. The relationship between lung function and CVD persists even after adjustments for shared risk factors such as smoking and inflammation, and has also been observed in studies restricted to never-smokers. However, whether impaired lung function leads to CVD or vice versa is currently not known. Investigating the causal relationship might reveal the underlying pathophysiology that could be useful for the development of therapeutic interventions. To investigate the causal relationship between lung function (forced vital capacity and forced expired volume in 1 second) and atherosclerosis (carotid artery internal media thickness, presence of carotid plaque) and peripheral artery disease, we will perform a two-sample Mendelian randomization analysis. We will use summary statistics from published large scale genome-wide association studies to study this relationship bidirectionally, i.e. we will use genetic variants associated with lung function to assess the effect of lung function on atherosclerosis or peripheral artery disease and genetic variants associated with atherosclerosis to assess the effect of atherosclerosis or peripheral artery disease on lung function. Adjustment for smoking behavior will be performed using multivariable Mendelian randomization and, depending on access to external data, using subgroup analysis in never-smokers. \",\n\"Apr16, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Sumner, Jennifer \",\n\"UNIVERSITY OF CALIFORNIA LOS ANGELES\",\n\"Joint Contributions of PTSD and Polygenic Risk for High Blood Pressure to Blood Pressure and Hypertension\",\n\"closed\",\n\"PTSD has been associated with elevated blood pressure, but the role of genetic factors is unclear. This study will examine whether the link between PTSD and elevated blood pressure is greater for individuals with greater genetic vulnerability to high blood pressure.\",\n\"Posttraumatic stress disorder (PTSD) is associated with increased risk of elevated blood pressure (BP) and hypertension (e.g., Buckley & Kaloupek, 2001; Burg et al., 2017; Sumner et al., 2016). However, the role of genetic factors is unclear. Polygenic risk scores for BP have been studied in both European ancestry (EA) and trans-ancestry genome-wide association studies (GWASs) (Smith et al., 2015). These polygenic risk scores have been found to explain a small but significant percent of the variance in systolic BP, diastolic BP, and hypertension, above and beyond traditional risk factors (Fava et al., 2013; Franceschini et al., 2013; Giri et al., 2018; Warren et al., 2017). The aims of this PGC-PTSD secondary analysis proposal are to examine the interaction of PTSD with polygenic BP risk to BP and hypertension outcomes. We hypothesize that associations of PTSD with elevated BP and hypertension will be stronger for those with greater polygenic BP risk.  Studies from the PGC-PTSD working group with relevant data will contribute to this project. We are requesting access to the MVP trans-ethnic BP GWAS summary statistics so that we can compute polygenic risk scores for systolic and diastolic BP. Some of the contributing studies include individuals of European ancestry, but others are more diverse (e.g., individuals of African or Hispanic ancestry). Having access to the summary statistics from the MVP trans-ethnic BP GWAS will be useful for studying these diverse ancestry samples.   In each study with relevant data, we will calculate polygenic risk scores for BP. Then we will use hierarchical linear regression to examine the main effects and interaction of PTSD and BP polygenic risk score on BP (separate regression models will be conducted for systolic and diastolic BP). Population substructure (PCs), age, sex, PTSD, and BP polygenic risk score will be entered in the first step of the regression model. The PTSD x BP polygenic risk score interaction will then be entered in the second step of the regression model. We will also consider including PC x BP polygenic risk score and PC x PTSD interactions to properly control for potential confounders. We will then meta-analyze results across the different studies. \",\n\"Apr16, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Su, Jinni \",\n\"ARIZONA STATE UNIVERSITY-TEMPE CAMPUS\",\n\"Gene-Environment Interactions and Alcohol Use and Related Outcomes \",\n\"approved\",\n\"Alcohol use disorders are prevalent and associated with significant personal, interpersonal, and societal costs. Understanding the risk and protective factors associated with alcohol use and related outcomes is important to inform prevention and intervention efforts for those who are at risk. Some people are genetically predisposed to have higher risk of developing alcohol use disorders. This research aims to identify malleable environmental factors across multiple levels (e.g., family, peer, school, neighborhood, culture) that may exacerbate or buffer genetic risk. \",\n\"Alcohol use disorders (AUD) are prevalent and associated with significant personal, interpersonal, and societal costs. Understanding how genetic and environmental influences interact (i.e., gene-environment interaction) to contribute to the development of AUD among diverse populations is critical for developing prevention and interventions. The overarching goal of this research is to understand how genetic risk of AUD interact with environmental factors to influence alcohol use and related outcomes (e.g., depression, anxiety, externalizing behaviors, etc.) in adolescents of racially/ethnically diverse backgrounds. We will conduct secondary data analysis of the Adolescent Brain Cognitive Development Study, which enrolled a large racially/ethnically diverse sample of children (N = 11,875; 52.1% White, 15.0% Black, 20.3% Hispanic, 2.1% Asian, and 10.5% other [e.g., biracial]; 47.8% female; aged 9-10 at baseline assessment) and follows them up over time. We will conduct multiple regression analysis and structural equation models to examine how genetic risk of AUD(characterized by genome-wide polygenic scores using summary statistics from genome-wide association study of AUD) interacts with environmental factors across multiple levels (e.g., parenting, peer deviance, school belonging, neighborhood disadvantage, cultural values) in predicting alcohol use and related behavioral and emotional health outcomes in adolescents. We are requesting access to the summary statistics from genome-wide association study of alcohol phenotypes with the Million Veteran Program study, which will be used to calculate polygenic scores for our project.\",\n\"Jul19, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"St\\u00c3\\u00a5hl, Patrik \",\n\"KTH-ROYAL INSTITUTE OF TECHNOLOGY\",\n\"Spatial analysis of human placenta in pre-eclampsia\",\n\"closed\",\n\"Preeclampsia complicates about one in twenty pregnancies. This disease is associated with an increased risk of morbidity and mortality for both mother and baby. At present, we do not know what is causing preeclampsia. It is believed that poor placentation is a contributing factor behind the disease. What is known is that there are several changes in the placenta with preeclampsia. The placenta itself is a facinating organ with many internal structures. We have investigated how genes are expressed in different parts of these structures and will compare with the single-cell data to learn more of how the placenta is affected by preeclampsia.\",\n\"Preeclampsia (PE) complicates about 2-8% of all pregnancies worldwide. As such it is a major obstetric problem, with a high prevalence of maternal and perinatal mortality/morbidity. Although the underlying cause of preeclampsia is unknown, poor placentation has been suggested as an important contributing factor, especially in early-onset cases. Several alterations in the placenta has been reported with preeclampsia. Much of these results are based on histology or tissue homogenates, potentially missing alterations present in different parts of the villous tissue. Recently a method to determine spatial gene expression in tissues has been developed at KTH Royal Institute of Technology (Sweden). We have utilized this method on placental tissue from healthy pregnancies and pregnancies complicated by preeclampsia. We would like to request access to this dataset to compare single cell gene expression with our data on spatial transcriptomic gene expression, to validate/identify cell types detected in our data set.\",\n\"Nov10, 2020\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Straughen, Jennifer \",\n\"HENRY FORD HEALTH SYSTEM\",\n\"Characterize DNA Methylation alterations in placenta and their associations with gestational age\",\n\"approved\",\n\"The genomic and molecular characterization of placenta tissues is crucial to understand how placenta functions are related to childbirth outcomes. The traditional bulk mRNAseq measures gene expression of a mixture cells and placentas contains various types of cells. It is critical to characterize cell type specific expressions to enhance our understanding of placenta tissues. We will use single cell sequencing data from phs001886 as references to estimate cell populations in bulk RNA sequencing through mathematical algorithms. \",\n\"We are interested in characterizing molecular and genomic changes in placenta tissue with respect to gestational age and environmental exposures. Specifically, we have a project to study how placental changes in DNA methylation may mediate the effect of environmental exposures on gestational age at delivery. This is a recent field of study, and while it is appreciated from other areas of research that cellular heterogeneity can impact genomic studies of particular tissue types, very little information on this effect on genomic studies of placenta is available, despite the extreme heterogeneity in this vital developmental organ. In combination with another dbGaP dataset with bulk placental methylation and RNA-Seq data (phs001717), we intend to use the data from phs001886 to gain insight into the heterogeneous nature of placental tissue. Specifically, we will use this data to achieve two scientific objectives. First, using methods including CIBERSORT and MuSIC, we will use the placental single-cell RNA-Seq data from phs001886 as a reference to deconvolute the bulk placental RNA-Seq data from phs001717 and identify the individual cell types and their corresponding percentages in that data. Second, we will utilize these results to inform a deconvolution of our placental methylation data to explore how the heterogeneity in placental cellular subtypes may influence association analyses of methylation alterations and gestational age at delivery.  It is our intention to present all our analyses and findings of our data and the data from phs001886 to the scientific community. We will present these results in both national/international conferences and peer-reviewed scientific journal publications. This work will greatly boost our understanding of how biological changes in placenta impact preterm birth. \",\n\"Dec16, 2020\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Stallings, Michael \",\n\"UNIVERSITY OF COLORADO\",\n\"Comparing shared genetic risk of substance use disorder across associated phenotypes\",\n\"approved\",\n\"Substance use disorders are common and can occur in many different types of people with different diagnostic and personality profiles. We are interested in investigating how these different profiles overlap and/or diverge with respect to genetically-informed risk for addiction to substances.\",\n\"Substance use disorders (SUDs) are both phenotypically and genetically correlated with several pathological and behavioral phenotypes. However, it is not clear to what extent genetic liability for SUDs is common across individuals who share a high predisposition for SUDs but who differ with respect to other risk factors. The objective of the proposed study is to use Genomic Structural Equation Modeling (Genomic SEM) to demonstrate potential genetic heterogeneity in liability to SUDs. To create our models, we intend to use summary SUD and mental health data from other consortia alongside summary SUD and mental health data from MVP. \",\n\"2022, Oct04\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Linial, Michal \",\n\"HEBREW UNIVERSITY OF JERUSALEM\",\n\"Polygenic Risk Prediction for complex disease by gene-based association studies\",\n\"closed\",\n\"We will search for genes and protein mutations that cause depression and other diseases (e.g. chronic pain, risk of addiction, or chronic diseases such as diabetes) using the genetic data in the VHA omics dataset. We will look not just at changes in the genome, but also if these mutations affect the created proteins significantly, as a new way of finding more important factors.\",\n\"Our lab focuses on genomic and proteomic aspects of biology. We will use the VA dataset to search for genes and their associated protein level mutations that associate with diseases and other physiological troubles - we will focus on the target of the original study (depression), but will also examine other included covariates if available (e.g. chronic pain, schizophrenia, bipolar disorder) using the genetic/omics data in the VHA omics dataset. We aim to combine multiple biological risk factors to find novel risk factors for complex diseases. Use will be entirely computational, to find associations between the recorded data and recorded diseases.\",\n\"2021, Sep16\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Li, Wei \",\n\"UNIVERSITY OF CALIFORNIA-IRVINE\",\n\"Characterizing the role of alternative polyadenylation in human immune cells\",\n\"approved\",\n\"Approximately 70% of human gene transcripts have utilized different polyA sites to generate alternative 3\\u00e2\\u0080\\u0099UTR region. Recently, it has been shown that marked 3\\u00e2\\u0080\\u0099UTR shortening event were observed upon innate immune response. We have developed a novel bioinformatics tool to identify 3\\u00e2\\u0080\\u0099 untranslated region (3\\u00e2\\u0080\\u0099 UTR) from RNA-seq. With access to the requested RNA-Seq and genotyping data from Immunological Variation Consortium, we will be able to provide a comprehensive landscape of 3\\u00e2\\u0080\\u0099 UTR usage across different immune cells, in addition, we will also investigate how genetic variations are associated with these 3\\u00e2\\u0080\\u0099UTR usage.\",\n\"For more than 70% of human genes, their 3' UTR can be shortened or lengthened under diverse physiological conditions. Since 3\\u00e2\\u0080\\u0099 UTRs contain many important cis-regulatory elements, such as miRNA binding sites, gene with shorter 3\\u00e2\\u0080\\u0099 UTRs will no longer be repressed by miRNA, leading to higher expression. In general, widespread 3\\u00e2\\u0080\\u0099UTR shortening have been observed upon active innate immune response. However, the critical target genes subject to differential 3\\u00e2\\u0080\\u0099 UTR usages in other immune cell types and their underlying genetic mechanism regulating dynamic 3\\u00e2\\u0080\\u0099 UTR usages during immune activation remain poorly understood. Therefore, we previously developed a novel bioinformatics, termed DaPars, for dynamic analysis of alternative polyadenylation from RNA-seq (Xia, Nature Commun., 2014), which can effectively profile 3\\u00e2\\u0080\\u0099UTR usages from RNA-seq data. The objective of this project is to apply our new developed DaPars2 algorithm to various RNA-seq data to provide a comprehensive landscape of dynamic 3\\u00e2\\u0080\\u0099UTR usages across human immune cells and to infer the potential upstream genetic variations modulating 3\\u00e2\\u0080\\u0099 UTR usage. Together, these differential 3\\u00e2\\u0080\\u0099 UTR usage and their associated genetic variants represent an illustrative case of genomic \\u00e2\\u0080\\u009cdark matter\\u00e2\\u0080\\u009d beyond coding regions, and thus may provide a new layer for understanding autoimmune or inflammatory diseases.\",\n\"2021, Jun23\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Li, Xin \",\n\"SHANGHAI INSTITUTES FOR BIOLOGICAL SCIS\",\n\"Statistical methods for decoding the genetic architecture of rare disease\",\n\"expired\",\n\"We seek to uncover the role of rare non-coding variation in gene regulation and complex traits. The mechanism how regulatory variants cause human disease is largely unknown. We plan to utilize modern machine learning techniques to train a trans-omics model, with the goal of better interpreting the function of the non-coding genome. \",\n\"Most verified disease-causing mutations have been limited to protein coding regions, with the function of non-coding variation remains largely unknown. The efficient interpretation of non-coding variants among human populations holds a substantial potential for furthering our understanding of disease etiology and finding new therapeutic targets. However, investigation of noncoding variation has been significantly barred by the lack of explicit genetic code, the requirement of extra-large sample sizes and deep whole genome sequencing. UDN and other rare disease cohorts with whole genome sequencing provides us with the unique opportunity to unveil a comprehensive landscape of the functional impact of rare non-coding variants.   In this study we set out to develop a new machine learning model to characterize the roles of many types of rare non-coding variants across multiple regulatory cascades. Utilizing solved and unresolved cases in the cohort, we seek to train the model over a variety of genomic and epigenomic features, aiming at identifying potential rare non-coding variants that are more prone to regulatory modification and pathogenesis. A successful outcome of this study will build a detailed catalog of all types of rare non-coding variants and their regulatory effects, forming the basis for prioritizing rare variants for further phenotypic characterization. \",\n\"2021, Jan19\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Coleman, Jonathan \",\n\"KING'S COLLEGE LONDON\",\n\"Use of genomic summary statistics for neuropsychiatric genetic epidemiology\",\n\"approved\",\n\"Research has identified many places in the human genome where certain changes in the DNA are found more often in people with brain-related conditions (like depression and Alzheimer\\u00e2\\u0080\\u0099s disorder) compared to people who do not have the condition. The results of research like this from different groups of scientists can be used to examine how similar the genetic effects on one condition are to the genetic effects on another. We will use results from the Million Veteran Program to examine how similar genetic effects on various conditions studies by the Million Veteran Program scientists are to the brain-related conditions we study.\",\n\"Genome-wide association studies (GWAS) have implicated numerous regions of the genome as associated with complex brain-related phenotypes, including normative behaviour, psychiatric illness, and neurological degeneration and dysfunction. Studies of the genetic overlap between traits have benefited from statistical methods that use GWAS summary statistics. We will use summary statistics from GWAS conducted by the Million Veteran Program to study how genetic variation contributes to brain-related traits, including normal behaviours such as personality, psychiatric disorders such as depression and anxiety disorders, and neurological dysfunction, including Alzheimer\\u00e2\\u0080\\u0099s disorder. Ultimately, we aim to understand better both the prediction of such traits, and their biology. We will examine the role of genetic influences in isolation and also within the context of environmental risk factors. We intend to use the summary statistics obtained in this request (alongside other summary statistics from other sources) to create polygenic risk scores within the individual level genetic datasets available to our research groups. We will also use summary statistics to conduct between-trait analyses at the summary level, through methods including genetic correlation analysis (e.g. in LDSC), genomic mixture analysis (e.g. in MiXer), and multivariate GWAS and genomic structural equation modelling (e.g. in GenomicSEM). \",\n\"Jan03, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"DICK, DANIELLE \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"Prediction of Alcohol Use Disorders across Ancestral Populations\",\n\"closed\",\n\"Polygenic scores are becoming of interest for medical practitioners and clinicians. The current analysis will examine the clinical utility of polygenic risk scores for identifying those at extreme risk for substance use disorders. We will focus on individuals of both European and African ancestry to address the diversity gap in genomic research.\",\n\"Polygenic scores derived from GWAS summary statistics are becoming increasingly useful in medical research and there is hope that these will provide actionable information for clinical intervention in regards to substance use disorders. However, the majority of the studies focus on individuals of European ancestry. Here, we plan on using summary statistics from large scale GWAS of alcohol use disorders (AUD) and alcohol consumptions in both European and African ancestry samples in the Million Veterans Program (MVP) to create polygenic risk scores in a diverse sample of individuals deeply affected with AUD. The goal of this research will focus on whether the extreme ends of the polygenic score distribution for alcohol phenotypes provide any clinically relevant information in individuals of European or African descent. We will focus on a variety of substance use disorders including alcohol, cannabis, cocaine, nicotine, and others. Data will be used only for research purposes and we will make no attempt to reveal the individual participants. The proposed research is consistent with the data use limitation noted in the Data Use Certification Agreement for MVP. \",\n\"2019, Aug08\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"DICK, DANIELLE \",\n\"RBHS-ROBERT WOOD JOHNSON MEDICAL SCHOOL\",\n\"Prediction of Alcohol Use Disorders across Ancestral Populations\",\n\"approved\",\n\"Polygenic scores are becoming of interest for medical practitioners and clinicians. The current analysis will examine the clinical utility of polygenic risk scores for identifying those at extreme risk for substance use disorders. We will focus on individuals of both European and African ancestry to address the diversity gap in genomic research.\",\n\"Polygenic scores derived from GWAS summary statistics are becoming increasingly useful in medical research and there is hope that these will provide actionable information for clinical intervention in regards to substance use disorders. However, the majority of the studies focus on individuals of European ancestry. Here, we plan on using summary statistics from large scale GWAS of alcohol use disorders (AUD) and alcohol consumptions in both European and African ancestry samples in the Million Veterans Program (MVP) to create polygenic risk scores in a diverse sample of individuals deeply affected with AUD. The goal of this research will focus on whether the extreme ends of the polygenic score distribution for alcohol phenotypes provide any clinically relevant information in individuals of European or African descent. We will focus on a variety of substance use disorders including alcohol, cannabis, cocaine, nicotine, and others. Data will be used only for research purposes and we will make no attempt to reveal the individual participants. The proposed research is consistent with the data use limitation noted in the Data Use Certification Agreement for MVP.\",\n\"Feb06, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"DE VRIES, PAUL \",\n\"UNIVERSITY OF TEXAS HLTH SCI CTR HOUSTON\",\n\"Mendelian randomization to assess the effect of hemostatic factors on disease phenotypes\",\n\"approved\",\n\"The proteins that affect blood clotting may affect our risk of certain diseases, such as heart disease and stroke. Learning more about whether these proteins are disease risk factors could affect the way we try to prevent and treat disease. We can use genetics to research the effect of these clotting proteins on diseases. This approach is called \\\"Mendelian randomization\\\", and is done by comparing the effect of genetic variants on blood levels of a clotting protein to the effect of those genetic variants on disease risk. If genetic variants that increase blood levels of the clotting protein also increase the risk of a disease, then that tells us that the clotting protein might be influencing the risk of disease. To carry out this research, we request access to results from the MVP study. The results from MVP can tell us about the effects of genetic variants on disease, which we can then use to do Mendelian randomization.\",\n\"Circulating levels of hemostatic factors such as fibrinogen, Factor VII, Factor VIII, and von Willebrand factor are hypothesized to influence the risk of venous thromboembolism as well as the risk of diseases characterized by arterial thrombosis such as coronary artery disease, ischemic stroke, and peripheral artery disease. Many hemostatic factors are also thought to influence the risk of bleeding disorders such as hemorrhagic stroke. It is also hypothesized that specific hemostatic proteins may have effects on disease that are not characterized by thrombosis or bleeding, such as type 2 diabetes or chronic kidney disease. Mendelian randomization is an approach that uses genetic variants associated with an exposure as instrumental variables to test the causal association of that exposure with one or more disease phenotypes. Mendelian randomization can be conducted using summary statistics from genome-wide association studies (GWAS). Within the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE)consortium we already used this approach to investigate the causal effect of several hemostatic factors on venous and arterial thrombosis. We now propose to use summary statistics from MVP GWAS to conduct Mendelian randomization analyses of hemostatic factors on a variety of disease outcomes. When appropriate we will meta-analyze summary statistics from MVP with summary statistics from our GWAS. From the disease phenotypes that currently have MVP summary statistics available on dbGaP we propose to use venous thromboembolism, peripheral artery disease, eGFR, and type 2 diabetes. We are also interested in using summary statistics for coronary artery disease and stroke, if these become available. We also intend to use the MVP summary statistics to create polygenic scores for blood pressure and lipid levels, and examine how those scores associate with longitudinal trajectories of blood pressure and lipid levels, as well as with cardiovascular outcomes. Given that we will be working only with summary statistics, this proposed research poses no additional risks to MVP participants. \",\n\"Nov10, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Czamara, Darina \",\n\"MAX PLANCK INSTITUTE OF PSYCHIATRY\",\n\"Neurobiological correlates of PTSD polygenic risk scores in dimensional samples of stress-related psychiatric disorders\",\n\"closed\",\n\"Genome wide association studies (GWAS) have identified many genes associated with psychiatric diseases. However, it is not known how the identified risk genes contribute to maladaptive physiological responses, brain functioning and behaviour. In our analyses, we will GWAS data for post-traumatic stress disorder to compute polygenic risk scores. These scores reflect a small part of our study participant\\u00c2\\u00b4s risk for having a psychiatric disorder. We will then analyze whether the polygenic risk scores are associated with physiological responses (for example stronger pupil dilations to threatening stimuli), brain functioning (for example activation of rewarding brain regions in response to rewarding stimuli) and behavior (for example having trouble to get out of bed in the morning). This will help us understand how the genetic predisposition shapes mental health. \",\n\"The GWAS data will be used for the calculation of polygenic risk scores, mainly in the ongoing BeCOME study (registered on ClinicalTrials.gov, TRN: NCT03984084) that is designed to identify biology-based classes of stress, affective and anxiety related mental disorders and recruits healthy controls and patients with different degrees and varying comorbidities of stress-related psychiatric disorders. Polygenic risk scores will be used to predict pathology and task performance in the BeCOME study, either as a main predictor or in interaction with other variables of interest, such as childhood trauma. The planned analyses do not convey additional risks to the participants. \",\n\"2021, Jul28\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Crowley, James \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"Genetic study of trichotillomania\",\n\"expired\",\n\"We are performing a genetic study of compulsive hair pulling, also known as trichotillomania (TTM). This study has both a rare variant component and common variant component. For the rare variant component, we have access to whole exome sequencing data for 100 TTM cases and are searching for an appropriate control group of neurologically normal subjects with whole exome sequencing data. The dataset we apply for here appears to be an ideal control group for case-control analyses. We will test whether subjects with TTM have a greater burden of rare, deleterious coding variants relative to controls.   For the common variant component, we have access to genome wide SNP data for these same 100 cases. We would like to test whether these TTM cases have an elevation in polygenic risk scores for anxiety and other psychiatric traits. The Million Veteran Program (MVP) dataset provides a well-powered GWAS on anxiety and we intend to use summary statistics to compute polygenic risk scores for the TTM cases. We will test whether subjects with TTM have a higher burden of common alleles for anxiety.\",\n\"We are performing a genetic study of compulsive hair pulling, also known as trichotillomania (TTM). This study has both a rare variant component and common variant component. For the rare variant component, we have access to whole exome sequencing data for 100 TTM cases and are searching for an appropriate control group of neurologically normal subjects with whole exome sequencing data. The dataset we apply for here appears to be an ideal control group for case-control analyses. We will test whether subjects with TTM have a greater burden of rare, deleterious coding variants relative to controls.   For the common variant component, we have access to genome wide SNP data for these same 100 cases. We would like to test whether these TTM cases have an elevation in polygenic risk scores for anxiety and other psychiatric traits. The Million Veteran Program (MVP) dataset provides a well-powered GWAS on anxiety and we intend to use summary statistics to compute polygenic risk scores for the TTM cases. We will test whether subjects with TTM have a higher burden of common alleles for anxiety.\",\n\"2021, Mar06\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Corces, Michael \",\n\"J. DAVID GLADSTONE INSTITUTES\",\n\"Investigating the role of noncoding SNPs in neuropsychiatric disorders\",\n\"approved\",\n\"Small variations in non-coding sequences of the human genome can increase susceptibility to disease. We will use data from the Million Veteran Program to highlight specific changes that are relevant to neuropsychiatric disorders, and further our understanding of the biological mechanisms connecting these variants to disease symptoms. We hope to identify new therapeutic targets that will aid in combatting these personally and socially impactful disorders.\",\n\"The broad research approach of our laboratory is to use insights from the epigenome to inform our understanding of human diseases, with the goal of identifying novel avenues for therapeutic intervention. We are interested in how the genome, transcriptome, and epigenome intersect and how inherited genetic variation interfaces with lived experiences and environmental factors to produce durable and functional gene regulatory alterations that lead to disease. We are particularly interested in neurological and neuropsychiatric diseases.  A conceptual tenet of our group is that single nucleotide polymorphisms (SNPs) associated with diseases can lie in regions of the genome that regulate gene expression, driving cellular and ultimately organismal phenotypes from the molecular level. To apply this idea toward understanding human disease, we will use summary statistics from genome-wide association studies (GWASs) conducted as part of the Million Veteran Program to identify candidate noncoding SNPs that may play a role in the pathogenesis of complex neuropsychiatric disorders such as Major Depressive Disorder (MDD), Generalized Anxiety Disorder (GAD) and Post-Traumatic Stress Disorder (PTSD).  We will overlay these summary data onto our own single-cell ATAC-seq, bulk ATAC-seq, and three-dimensional chromatin conformation data from primary brain samples, allowing us to identify noncoding SNPs that fall in cell-type and cell-state specific regions of accessible chromatin and interact with other known regulatory elements. Highlighting these putative functional SNPs will allow us to derive hypotheses about the biological mechanisms mediating associations between genetic variants and symptom phenotypes. The proposed combination of the requested data and our own data will not create any additional risk to participants. \",\n\"Nov02, 2021\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Coombes, Brandon \",\n\"MAYO CLINIC ROCHESTER\",\n\"Polygenic risk prediction of psychiatric, addiction, and treatment response outcomes   \",\n\"approved\",\n\"Although individual genetic variations are unlikely to have large effects on complex traits such as substance use disorders, the collective effects of relevant genetic variations may be quite important. We will use results from the Million Veteran Program studies to examine how genetic variation/genetic risk contributes to individual differences in biology, behavior, and health-related outcomes in studies of psychiatric and addiction outcomes. \",\n\"Although individual genetic variations are unlikely to have large effects on complex traits such as substance use disorders, the collective effects of relevant genetic variations may be quite important. We will use summary statistics generated by genome-wide association studies (GWASs) conducted within the Million Veteran Program dataset to examine how genetic variation/genetic risk contributes to individual differences in biology, behavior, and health-related outcomes in studies of psychiatric, addiction, and treatment response outcomes. The overall objective of this study is to use MVP summary statistics to create polygenic risk scores in independent samples to further our understanding of psychiatric and addiction outcomes. We will also meta-analyze these summary statistics with independent GWASs of similar outcomes. Phenotypes will include summary statistics for GWAS related to addiction (e.g. problematic alcohol use, opioid use disorder, etc.), psychiatric outcomes (PTSD, resilience, depression, etc.), as well as outcomes for health, behavior and biology.\",\n\"2020, Oct30\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Claypool, Kajal \",\n\"MIT LINCOLN LABORATORY\",\n\"MIT LL Test Project\",\n\"approved\",\n\"This is a test request for the study to conduct a test for approval and notification to RAPIDS\",\n\"This is a test request for the study to conduct a test for approval and notification to RAPIDS\",\n\"2022, Oct13\",\n\"phs002516.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Evans, Luke \",\n\"UNIVERSITY OF COLORADO\",\n\"Association mapping and the genetic architecture of complex traits.\",\n\"approved\",\n\"I am requesting the genotype and phenotype data of the datasets to perform analyses on the genetics of biometric measures (e.g., height, BMI), and behavior and substance use such as nicotine and alcohol dependence and anxiety. Specifically, I will use a range of methods to assess the overall genetic contribution to these complex traits (i.e., their heritability), identify variants and genes that influence these traits (through GWAS and TWAS), the genetic relationships among traits (their genetic correlations), and assess functional enrichment of associations by leveraging external annotation and expression data. The analyses will lead to a greater understanding of the genetics of complex traits, including the genetic etiology of nicotine and alcohol use and internalizing disorders such as depression and anxiety.\",\n\"The data will be used for health/medical/biomedical purposes, including studies of the genetic basis of biometric and health measurements, focusing on behavior and substance use phenotypes, in particular depression, anxiety, alcohol use and nicotine use. The first study is a study on the genetics of smoking behaviors. I will use the data for GWAS as well as transcriptome-wide association studies (TWAS). I will combine the GWAS and TWAS summary statistics from these datasets with other approved dbGaP datasets such as NESARC-III, HRS, and Add Health, along with internal datasets here at CU Boulder and the UK Biobank via meta-analysis of the SNP- and gene-level effect sizes and p-values estimated within each study separately. TWAS will be performed using dbGaP-approved access GTEx, HBCC, and NABEC expression data to obtain SNP effects on expression, and these estimated SNP effects will be used with the individual studies to estimate gene expression-trait associations within each study. I will combine the resulting summary statistics from each dataset via meta-analysis.  Analyses will include GWAS and TWAS, association enrichment of functional annotations, genetic correlation of traits, and SNP-heritability estimation. The proposed research falls within the General Research Use consent of the requested data. All data are de-identified, and no attempt will be made to re-identify individual participants. I have received approval following an expedited review of my proposed research by the CU IRB (document attached, full proposal available upon request). The data will be stored at the CU Research Computing, a supercomputing facility with robust security here at CU Boulder and with dedicated staff to maintain it.  We intend to publish any findings, and intend to share our findings with the scientific community via presentations at scientific conferences (e.g., WCPG, ASHG, BGA, SRNT). \",\n\"Sep18, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Claussnitzer, Melina \",\n\"BROAD INSTITUTE, INC.\",\n\"Multi-phenotype mapping for cardiometabolic phenotypes\",\n\"expired\",\n\"Cardiometabolic diseases such as type 2 diabetes, obesity and coronary artery disease appear to have shared etiology, but the underlying genetic factors remain largely unknown. To identify genetic loci that may have pleiotropic effects on cardiovascular and metabolic traits, we seek to study GWAS summary statistics to learn about common disease aetiologies.\",\n\"We plan to combine summary statistics from the MVP cohort with other existing summary statistics to improve power to assess multi-phenotype patterns of genetic associations with metabolic diseases and traits, and inform priors for cell types and mechanisms for individual GWAS loci. We will use multi-trait analysis of GWAS (MTAG), a method for joint analysis of summary statistics from genome-wide association studies (GWAS) of different traits, to gain insight into individual GWAS loci and globally. We will use our own compute resources and will follow procedures outlined in NIH communication (NOT-OD-19-023) for using summary statistic data from human genetic studies.\",\n\"Apr07, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Clark, Shaunna \",\n\"TEXAS A&amp;M UNIVERSITY HEALTH SCIENCE CTR\",\n\"Integrating methylomic and genomic summary statistics to better characterize alcohol-related DNA methylation\",\n\"approved\",\n\"Despite extensive anti-drinking efforts, ~6.2% of US adults qualify for an alcohol use disorder and many more engage in habitual or problematic drinking. Mounting evidence suggests that DNA methylation may play a role in drinking behaviors. However, many DNA methylation may be impacted by variation that occurs on the genetic level. We propose to use summary statistics from alcohol genetic studies in order to determine if our alcohol-related DNA methylation findings are simply picking up on genetic signal or if they represent a unique biological signature not related to underlying genetics. \",\n\"Despite extensive anti-drinking efforts, ~6.2% of US adults qualify for an alcohol use disorder and many more engage in habitual or problematic drinking. Mounting evidence suggests that DNA methylation may play a role in drinking behaviors. However, many DNA methylation sites are regulated by common genetic variation. The research objective is to utilize summary statistics from alcohol-related genome-wide association studies (GWASs) to determine if alcohol-related methylation is simply picking up on genetic signal or represents a unique biological signature separate from common genetic variation.   This project will analytically combine the summary statistics from alcohol GWAS and summary statistics from alcohol methylation-wide association studies (MWASs). The combination of the GWAS and MWAS results does not create any additional risk to participants as we will only be using summary statics and do not need any participant-level information to conduct the project. Specifically, we will test whether the top findings from the alcohol GWAS occur more frequently among the top findings of our MWAS (i.e., are enriched) than what is expected by chance. \",\n\"2020, Mar17\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Testing Data Access for RADx developer (2)\",\n\"approved\",\n\"I will be confirming data access using the Hub.    \",\n\"I am testing data access for select RADx program data. I will be using the Data Hub to access the data. \",\n\"Dec12, 2022\",\n\"phs002525.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Testing Data Access for RADx developer\",\n\"approved\",\n\"I will be viewing data and loading it into a Jupyter Notebook to confirm functionality works. \",\n\"I am testing data access for select RADx program data. I will be using the Data Hub to access the data.\",\n\"Dec12, 2022\",\n\"phs003027.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer\",\n\"approved\",\n\"I will be testing whether I can see data in RAPIDS, download data from RAPIDS, open the data in the Data Hub and use cloud tools to analyze this information, and use this information to confirm Data Hub functionality, identify areas for improvements, and improve training materials for researchers. \",\n\"This request is to serve testing purposes of Data Hub functionality. \",\n\"2022, Nov09\",\n\"phs002519.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Churilla, James \",\n\"UNIVERSITY OF NORTH FLORIDA\",\n\"Sedentary time and heart failure and heart failure sub-type\",\n\"approved\",\n\"This study plans to investigate the potential causal pathway between sedentary time and various types of heart failure. This study also plans to investigate whether physical activity plays a role in this relationship. This study plans to utilize data from several data sources and combine the necessary data for analyses. Adults 40 years of age and older with data for all variables needed for the analyses will be included. This study will illustrate whether sedentary behavior is a causal component in heart failure and if physical activity plays a role in this relationship.\",\n\"Faculty from the Kinesiology and Lifestyle Medicine Program at the University of North Florida (UNF) possess a history of physical activity, cardiometabolic, and heart failure research. Previous work out of UNF has shown a significant relationship between sedentary time and incident heart failure with preserved ejection fraction (HFpEF) utilizing Multi Ethnic Study of Atherosclerosis (MESA) data. Other studies have identified one locus for HFpEF and 13 loci for heart failure with reduced ejection fraction (HFrEF). For this study, our team aims to investigate the potential causal pathway between sedentary time and HFrEF and HFpEF through a Mendelian randomization analysis. A secondary aim will be to investigate physical activity as a potential mediator or moderator. The Mendelian randomization will utilize data from three separate genome wide association studies (GWAS): 1) Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention; 2) Genome-wide association and Mendelian randomization analysis provide insights into the pathogenesis of heart failure; 3) Genetic architecture of heart failure with preserved versus reduced ejection fraction. (data being requested). Inclusion criteria will include adults >40 years of age with valid data for all variable in the analyses. Primary independent variables will be sedentary time and physical activity. Our dependent variable will be heart failure sub-types. We will utilize Million Veterans Project genomic association summary results for ejection fraction to identify heart failure sub-types. Summary statistics and risk estimates will be produced utilizing frequency and regression procedures in R. Our team will utilize genetic and phenotype data to add to the current evidence in the area of sedentary behavior and heart failure. Our team will publish our findings and abide by all embargo timelines associated with all datasets.\",\n\"Apr12, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Choi, Murim \",\n\"SEOUL NATIONAL UNIVERSITY\",\n\"Colocalization analysis of GWAS and eQTL in NAFLD\",\n\"approved\",\n\"Non-alcoholic fatty liver disease (NAFLD) is a complex disorder that are caused by multiple factors. Many genes, variants, diet, and other lifelog history are known to contribute to the development of NAFLD in a complicated manner. Nevertheless, it is crucial to understand genetic cause of NAFLD to precisely predict differential susceptibility to the disease across different individuals. To achieve this goal, we established a bioinformatic pipeline called NAFLD-specific expression quantitative trait loci (eQTL) pipeline and proved its utility in identifying potential drug targets for certain individuals with specific genetic variants (Yoo et al., J Hepatol 2021). Using the NAFLD GWAS data available from this data set, we plan to compare GWAS signals with those from our eQTL screening. The result will help us to understand genetic causation of NAFLD that may lead to the personalized disease prediction and therapy.\",\n\"Non-alcoholic fatty liver disease (NAFLD) is a complex disorder involving function of many genes and variants in addition to contribution from environmental and other complicating clinical factors. Nevertheless, it is crucial to understand genetic etiology of NAFLD to precisely predict disease susceptibility and individual-level variation. To achieve this goal, we established a NAFLD-specific expression quantitative trait loci (eQTL) pipeline and proved its utility in identifying potential drug targets that may behave differentially among individuals of different genotype (Yoo et al., J Hepatol 2021). Using the NAFLD GWAS data available from this data set, we plan to compare GWAS signals with those from our eQTL screening. The result will help us to understand genetic causation of NAFLD that may lead to the personalized disease prediction and therapy.\",\n\"2022, Oct04\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"DRACHEVA, STELLA \",\n\"JAMES J PETERS VA  MEDICAL CENTER\",\n\"Integration of the opioid use disorder GWAS dataset with brain cell type-specific ATAC-seq and H3K27ac ChIP-seq data \",\n\"approved\",\n\"Opioid Use Disorder (OUD) is a problematic pattern of opioid use that leads to substantial impairment and disability. A tenfold increase in opioid analgesic prescriptions between 1990 and 2010 in the U.S. contributed to an epidemic of opioid abuse and overdose deaths. A recent large-scale genome-wide association study (GWAS) identified many gene regions (loci) that are associated with OUD. As in other complex disorders, most of the OUD risk loci reside within non-coding regions of the human genome where they are predicted to alter the activity of gene regulatory elements (GREs), which regulate the expression of nearby genes.  The effect of a GRE on gene expression is dependent on the cell and tissue type, making it difficult to understand the functional impact of GWAS risk loci.  Linking the disease risk variants with epigenetic maps of GREs in relevant tissues/cell types yielded formidable results in other psychiatric disorders  and holds promise for OUD.  Here we propose to integrate the OUD GWAS with cell-type-specific epigenetic data that were generated in our laboratory. The proposed studies should lead to the discovery of the OUD risk variants and the affected genes that are likely to be causal for OUD and act in specific brain cell types.\",\n\"We are requesting summary-level association data from the study by Kember et al. 2022. The goal of the proposed research is to understand if cell-type-specific non-coding regulatory elements (e.g., enhancers, promoters) in the orbital frontal cortex (OFC; the brain region that has been implicated in drug addiction) are enriched for opioid use disorder (OUD) risk SNPs.   Most functional noncoding SNPs are predicted to exert their effects through the alteration of gene expression via perturbation of transcription factor binding and/or regulatory element function. Such regulatory elements are highly cell type specific, suggesting that the resultant effects of noncoding SNPs (including risk SNPs) would be equally cell type specific.  As a part of our NIH-funded study (R01DA04324; NIH/NIDA; Cell Specificity of the Human Heroin Epigenome), we obtained transcriptional (RNA-seq) and epigenetic (ATAC-seq and H3K27ac ChIP-seq) data in the OFC samples from subjects who died of heroin overdose and control subjects (N=30 per group). The data were obtained in the nuclei from three major brain cell types\\u00e2\\u0080\\u0094inhibitory GABAergic neurons (GABA), excitatory glutamatergic neurons (Glu), and oligodendrocytes (OLIG)\\u00e2\\u0080\\u0094that were separated from postmortem brains using fluorescent-activated-nuclear sorting.   Our cell-type-specific ATAC-seq and H3K27ac ChIP-seq data, which are indicative of open chromatin and active enhancer/promoter regions, respectively, allowed us a comprehensive nomination of the putative functional regulatory elements that are active in each of the major brain cell types (GABA, Glu, OLIG) in OFC. These critical data will be used to test the functional importance of genetic risk loci in OUD.  To this end, we aim to perform linkage disequilibrium score regression (LDSR) analysis using summary-level GWAS data from Kember et al. 2022, as well as publicly available GWAS data from other brain disorders and traits that are not directly related to brain cell types. In addition to testing whether the cell-type-specific regulatory elements in OFC are enriched for OUD risk SNPs, our multiomic data set should allow for the nomination of specific functional OUD risk SNPs, which could be tested in the future studies.  \",\n\"Apr10, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Daly, Mark \",\n\"BROAD INSTITUTE, INC.\",\n\"VA-MVP Cohorts for Replication Efforts in Finnish T2D Analyses\",\n\"closed\",\n\"We aim to use the summary statistics from the Veterans Administration (VA) Million Veteran Program (MVP) cohorts from their recent meta-analysis for replication for our T2D association study within the FinnGen Study.\",\n\"We have been pursuing our Type 2 diabetes (T2D) project to use genome-wide association techniques to discover variants enriched through the Finnish founding bottleneck (eligible to have stronger biological effects than their frequency would generally indicate). We have already completed over 29,000 Finnish T2D cases, and are comparing these to physiologic results from the Botnia Study which has compiled physiologic results after oral glucose tolerance testing on over 3000 Finns without T2D. We propose the use of the case control data from the recent international Type 2 diabetes meta analysis (~228,000/~1,180,000) to replicate potential risk and protective variants that are found to be associated in this Finnish population sample. While we expect many of these to be much rarer in either the either European or international samples, they will often exist and the case-control distribution can be quite informative as replication given the sample numbers involved.  The specific samples of interest are from analyses pha004943.1, pha004944.1,  pha004945.1 , pha004946.1, and pha004947.1.  (Vujkovic M, Keaton JM, Lynch JA, et al. Nat Genet. 2020;52(7):680-691.)\",\n\"2020, Oct05\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Damrauer, Scott \",\n\"UNIVERSITY OF PENNSYLVANIA\",\n\"Cross trait genetics of cardiometabolic diseases, their causes and consequence\",\n\"approved\",\n\"We aim to is to analyze the genetic architecture of a wide variety of cardiometabolic diseases and their risk factors. We are interested in utilizing numerous data sources to identify DNA variants associated with our diseases and risk factors of interest to better understand how genes confer disease risk. We will conduct genome-wide association studies to link genetic sequence variants with risk of cardiometabolic disease and risk factors and integrate across datatypes mapping to identify novel variant associations with gene expression through a method known as expression quantitative trail locus (eQTL) mapping. Similar methods will be applied to link genetic variants to other intermediate traits including epigenetic markers, protein levels, and metabolite levels, which may mediate cardiometabolic disease and its risk factors. These integrative approaches will enable us to elaborate powerful datasets for downstream causal inference experiments to evaluate the casual links between risk factors, intermediate phenotypes/traits, and cardiometabolic diseases.\",\n\"The purpose of the proposed activity is to analyze the genetic architecture of cardiometabolic diseases including but not limited to: CAD, PAD, cIMT, CAC, AAC, VTE, AAA, TAA, CHF, stroke, carotid stenosis, and carotid plaque, as well as the risk factors for these diseases including but not limited to diabetes, dyslipidemia, hypertension, smoking, obesity, alcohol use, and psychiatric disorders. We are interested in identifying DNA sequence variants that associate with these diseases, understanding the molecular consequences of these variants, and identifying how these variants influence the relationship between risk factors and disease. We will leverage publicly available, deidentified individual-level, and summary statistics data sources for the above diseases and risk factors as well as all of MVP and GTEx. We will apply computational methods to:  1. identify the relationships between DNA sequence variants and cardiometabolic diseases/risk factors 2. integrate these datasets with other summary-level data that are both internally generated, as well as existing in the public domain 3. apply causal inference methods to understand the relationships and mechanisms that link DNA sequence variation with disease First, we will conduct genome-wide association study (GWAS) meta-analyses to link genetic sequence variants with risk of cardiometabolic disease and its risk factors. Next, we will integrate across datatypes, for example linking publicly available genotyping and/or genome sequencing data with gene expression data to perform expression quantitative trait locus (eQTL) mapping, enabling the identification of novel variants associated with gene expression. As cell-type specific eQTL data has been shown to be more powerful for discovery, we will integrate individual level GTEx data with single cell transcriptomics to compute interaction eQTLs. Similar methods will be applied to link genetic variants to other intermediate traits, including epigenetic markers, protein levels, and metabolite levels, which may mediate cardiometabolic disease and its risk factors. These integrative approaches will enable us to elaborate powerful datasets for causal inference experiments. Finally, we will apply causal inference methods including but not limited to Mendelian Randomization to evaluate the casual links between risk factors, intermediate phenotypes/traits, and cardiometabolic diseases.  \",\n\"2022, Nov23\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Daskalakis, Nikolaos \",\n\"MCLEAN HOSPITAL\",\n\"Integrating genetics of neuroimaging into functional characterization of neuropsychiatric GWAS\",\n\"approved\",\n\"We will study the genetic risk architecture of mental disorders, with a focus on the portion of the human genome influencing imaging of the brain. \",\n\"We will integrate neuroimaging genetics with neuropsychiatric risk by applying our neuroimaging models to the GWAS from neuropsychiatric traits. Our polygenic models of neuroimaging phenotypes have specific SNP weights that can be applied to neuropsychiatric GWAS summary statistics to aggregate the SNP association statistics to neuroimaging phenotype association statistics. Using causal mediation analysis, we will distinguish causal associations from false positives. \",\n\"2020, Oct05\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Erdos, Michael Reynolds\",\n\"NIH\",\n\"Genetic drivers of type 2 diabetes in diverse ancestries\",\n\"approved\",\n\"Type 2 diabetes (T2D) is a leading contributor to global morbidity and mortality. Previous genetic studies have demonstrated the role of genetic predisposition in T2D risk and successfully identified >240 loci associated with T2D. However, most lie in non-coding regions of the genome, masking the underlying \\u00e2\\u0080\\u009ceffector gene\\u00e2\\u0080\\u009d. Additionally, these studies were primarily performed in individuals of European ancestry, threatening to exacerbate existing health disparities among non-European communities disproportionately suffering from T2D as genetic findings are integrated into clinical interventions. This study aims to expand our genetic understanding of T2D in non-European ancestry groups by combining data from diverse cohorts to perform a trans-ancestry genetic analysis of T2D. Such a study has the potential to make substantial progress towards eliminating existing health disparities by proposing genetically-derived preventative interventions (e.g., polygenic scores) while simultaneously advancing our understanding of the molecular drivers of T2D by nominating likely-causal variants.\",\n\"Type 2 diabetes (T2D) is a leading contributor to global morbidity and mortality. T2D is characterized by hyperglycemia resulting from \\u00c3\\u009f-cell dysfunction and impaired insulin-stimulated response, but the molecular and cellular pathways driving these observations remain elusive. Previous genetic studies have demonstrated the role of genetic predisposition in T2D risk and successfully identified >240 loci associated with disease. However, most lie in non-coding regions of the genome, masking the underlying \\u00e2\\u0080\\u009ceffector gene\\u00e2\\u0080\\u009d. Additionally, these studies were primarily performed in individuals of European ancestry, threatening to exacerbate existing health disparities among non-European communities disproportionately suffering from T2D-related health complications as genetic findings are integrated into clinical interventions. Expanding our genetic understanding into non-European ancestries is essential as these studies will mitigate such health disparities and improve our ability to identify causal loci and mechanisms.  This study will combine data from diverse cohorts, with an emphasis on non-European ancestry groups, to perform a trans-ancestry common variant genome-wide association study of T2D, identifying trans-ancestry and ancestry-specific T2D-associated loci. We will fine-map these loci to identify likely causal variants and perform experimental validation of the strongest candidates using genome engineering techniques. Finally, we will generate and validate polygenic scores in each ancestry major group. Together, this study has the potential to make substantial progress towards eliminating existing health disparities by proposing genetically-derived preventative interventions while simultaneously advancing our understanding of the molecular drivers of T2D.  We are requesting the T2D summary statistics from the Million Veterans Program (MVP) to include in our meta-analysis. Due to the diversity represented in MVP, this dataset will be crucial to maximize our sample size of non-European ancestries. All data will be stored on the NHGRI Trek cluster, which is secured behind the NIH firewall. All approved users have received data security training. \",\n\"2021, Dec01\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Elo, Laura \",\n\"UNIVERSITY OF TURKU\",\n\"Computational methods for complex disease characterization \",\n\"closed\",\n\"With modern high-throughput biotechnologies it is possible to study biological systems in unprecedented insight and resolution. Specifically interesting is how the data will best be utilised in clinical practice and individualized prediction of diagnosis, prognosis or treatment of complex disorders such as immune-related diseases, neurodevelopmental disorders, and obesity. We plan to focus on developing and optimising computational data analysis tools and methods using large-scale data such as genotyping and next generation sequencing data with the aim of enabling robust and reproducible interpretation of the data. By applying mathematical, statistical and machine learning methods we aim to identify key features and patterns characterizing and revealing new insights into complex disorders.\",\n\"The objective of our research is to develop effective computational data analysis tools and mathematical modelling methods for biomedical research. A specific focus is on analyzing and interpreting data generated by modern high-throughput biotechnologies and on assessing their utility in clinical practice. Building on our previous computational, statistical and network-based studies, we aim at establishing a computational framework that allows optimized integration and analysis of large- scale data as well as heterogeneity between individuals to enable robust and reproducible interpretation of the data. The eventual goal is to improve the diagnosis, prognosis and treatment of complex diseases.  The objective of our research is to develop effective computational tools and mathematical models towards inferring optimal solutions for clinical problems at molecular level. In particular, we aim at enabling robust and reliable analysis and interpretation of data generated by modern high-throughput biotechnologies. A specific focus is on assessing the utility of next generation sequencing data in clinical practice and individualized prediction of diagnosis, prognosis or treatment of complex disorders such as immune-related diseases, neurodevelopmental disorders, and obesity. The data will be used for developing generally useful computational tools, specific models as well as for assessing a general background. We will develop and apply mathematical, statistical and machine learning methods to identify key features and patterns characterizing the particular disorder at multiple levels. An important prerequisite for developing generally useful computational solutions for biomedical applications is accessing large-scale well-designed datasets. Therefore, to ensure generally useful solutions that are unbiased and reliable, we would like to apply for the access to the comprehensive datasets featuring genotyping and next generation sequencing data. Such datasets fall within our requirements for the quality and the number of samples. Besides optimization of the data analysis procedures, consideration of multiple datasets holds potential to reveal also new insights into genetic abnormalities, genetic correlation between traits and heritability.  We do not reveal any information on any individual and the proposed research is consistent with the Use Restrictions. \",\n\"Mar13, 2020\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ellinor, Patrick \",\n\"BROAD INSTITUTE, INC.\",\n\"Genetics of atrial fibrillation\",\n\"approved\",\n\"Atrial fibrillation (AF) is a cardiac arrhythmia affecting 3 Million Americans. The causes of the disease are not clear, and current treatment methods are only partially successful.  Our research group has studied the genetic causes of AF over the last decade, and genetic screening has helped us identify mutations causing the disease in families with AF, as well as more common genetic variants linked with this disease. Recently, whole genome- and exome sequencing have made it possible to sequence large regions of the genome, and these types of studies increase our power to detect genetic variants associated with AF. dbGaP is a unique source of genetic and phenotypic data. We propse to analyze available exome sequencing, genome sequencing, exome chip and genome wide genotyping data in individuals with and without AF. We hope that our work will allow us to identify new genes and pathways, that will eventually lead to identification of new molecular targets for the treatment of AF.\",\n\"In this project we seek access to exome sequencing data, whole genome sequence data, exome chip, and genome wide genotyping data from studies with data on atrial fibrillation (AF) that are registered in dbGaP. The primary goal of the current proposal will be to identify novel genetic variants associated with atrial fibrillation by integrating exome sequence data, exome chip and genome wide genotyping data. We plan to combine the raw sequence data available in individuals with exome sequence data and AF and plan on a joint recalling of these datasets. When available, exome chip and genome wide genotyping data will be used to facilitate quality control metrics in each study. Cases will consist of individuals with AF in the requested cohorts using currently available phenotypic data in dbGap; referents will consist of individuals without AF in each cohort.  We do not believe that the combination of these genetic data will impose any new risks on the participant in the cohorts accessed through dbGap. The data obtained from dpGaP will not be shared with any additional collaborators. We request de-identified data, and will not attempt to decode any personal information using the requested data. The obtained data will be used for genetic research that facilitates discovery of variation related to cardiovascular disease (CVD), and for no other purposes. We plan to include cohorts from other continents in the combined genetic analyses, but the data included from the dbGaP will not be shared with other collaborators, and all analyses will be performed by our research group led by the PI. \",\n\"2022, Aug09\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Elam, Kit \",\n\"INDIANA UNIVERSITY BLOOMINGTON\",\n\"Examining Polygenic Effects on Longitudinal Depression Symptoms\",\n\"approved\",\n\"We plan to examine genetic influences on longitudinal depression symptoms in two independent samples. We will investigate whether these influences are related to individual characteristics, substance use, and family factors, or possibly buffered by a family-centered intervention. We will examine this in African American and European American participants. \",\n\"We request access to aggregate summary statistics (not individual-level data) from the bi-ancestral GWAS of depression in the Million Veteran Program sample (MVP; Levey et al., 2021; phs001672). We will not combine the requested data with other datasets outside dbGaP. Aggregate summary statistics will be used to create polygenic scores (PGS) for depression in African American (AA) and European American (EA) participants using PRS-CSx in two independent datasets. Participants in the Early Steps and Project Alliance 1 (PAL1) samples were randomized to a family-based intervention or control condition and assessed over time. Both samples primarily consist of African American (AA) and European American (EA) participants.   Our primary objective is to examine PGS prediction of longitudinal depression symptoms in two independent samples (Early Steps: age 2-16; PAL1 age 16-31). Guided by preliminary analyses, a secondary objective is to examine for moderation and/or mediation of genetic effects on depression by individual, peer/familial, and sociodemographic characteristics. We will examine for the moderating role of substance use as well as an outcome of depression. We will also examine for buffering effects of the intervention condition. Our final objective is to examine if pathways differ across AA/EA subgroups. The proposed research is consistent with the data use limitations as we plan to leverage aggregate GWAS data to examine effects on longitudinal health outcomes but do not plan to study population origins or ancestry.  \",\n\"May13, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Duncan, Laramie \",\n\"STANFORD UNIVERSITY\",\n\"Investigating a Male Protective Effect in PTSD Using an Ancestry-adjusted Polygenic Risk Score Meta-analysis\",\n\"approved\",\n\"Many psychiatric diseases show differences in prevalence by sex. For example, autism has a female to male prevalence ratio of ~1:4. Significant research has gone into understanding why these prevalence differences exist. For autism, there is evidence of a Female Protective Effect (FPE), meaning that more genetic risk factors are required for females to develop autism as compared to males. PTSD has a female to male sex prevalence ratio of 2:1. The aim of this project is to determine if there is a Male Protective Effect (MPE) in PTSD; that is, if males require more genetic risk factors to develop PTSD than females. To do so, we will use datasets from the Psychiatric Genetics Consortium PTSD group. In order to assess genetic risk for PTSD of each individual, we will use a technique called polygenic scoring. Polygenic scoring yields a single number\\u00e2\\u0080\\u0094a genetic score\\u00e2\\u0080\\u0094that quantifies an individual\\u00e2\\u0080\\u0099s genetic propensity for a given phenotype (e.g. PTSD). We can compare the polygenic scores of the males with PTSD to the females with PTSD in our datasets to determine if an MPE exists.\",\n\"Many psychiatric diseases show differences in prevalence by sex. For example, autism has a female to male prevalence ratio of ~1:4. Significant research has gone into understanding why these prevalence differences exist. For autism, there is evidence of a Female Protective Effect (FPE), meaning that more genetic risk factors are required for a female to develop autism as compared to a male. PTSD has a female to male sex prevalence ratio of 2:1. The aim of this project is to determine if there is a Male Protective Effect (MPE) in PTSD; that is, if males require more genetic risk factors to develop PTSD than females. In order to assess genetic risk for PTSD of each individual, we will use a technique called polygenic scoring. Polygenic scoring yields a single number\\u00e2\\u0080\\u0094a genetic score\\u00e2\\u0080\\u0094that quantifies an individual\\u00e2\\u0080\\u0099s genetic propensity for a given phenotype (e.g. PTSD). The predictive power of psychiatric polygenic risk scores has dramatically increased in the last several years, meaning we can now more accurately assess genetic risk for PTSD across males and females.  In the proposed analysis we will compare polygenic risk scores in female cases to male cases.  To adjust  for ancestry, we will regress out 10 principal components from the polygenic risk scores of each participant on a sample-by-sample basis.  This will leave us with ancestry-adjusted residuals of the polygenic risk scores, which will be used to determine the presence of an MPE in each sample.  We will then conduct a meta-analysis across the individual datasets which range in sample size from 208 to 6,565 participants. We will also use the PTSD polygenic risk scores in the typical manner (i.e. for prediction of PTSD and related phenotypes). Our ability to determine the presence of an MPE in PTSD  greatly depends on the quality of the polygenic risk score used. Thus, we are requesting access to the most recent GWAS of PTSD, that from the Million Veteran Program of over 250,000 participants, with which we can derive the most powerful polygenic risk score for PTSD.  \",\n\"Dec22, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Dubin, Ruth \",\n\"UNIVERSITY OF CALIFORNIA, SAN FRANCISCO\",\n\"The Impact of Renal Dysfunction on Biological Mechanisms of Cardiovascular Disease\",\n\"expired\",\n\"Over 20 million people in the United States have chronic kidney disease, and all of them are at high risk for development or progression of heart disease. Unfortunately, we have few effective therapies for this group of patients, because we have limited understanding of the biology of their disease. In this project, we will utilize a unique dataset of 1100 proteins that have been measured on about 1000 patients with heart disease; after following these 1000 patients for 10 years, we know which ones had a heart attack or were hospitalized for heart failure. We will select proteins that are associated with a risk of either of these clinical events. We anticipate that we will find new proteins that cause cardiovascular disease in patients with kidney disease. The level of such a protein is likely to vary person to person, and some of this variation depends on the person's genes. We will confirm the importance of any new protein using genetic information collected from thousands of patients and available publically on research sites.\",\n\"    Studies in both community-based and high-risk populations have confirmed that chronic kidney disease(CKD)confers at least a two-fold increase in cardiovascular (CV) risk. Approximately 10% of the $200 billion 2008 United States Medicare budget was devoted to patients with concomitant CKD and cardiovascular disease (CVD).Despite this enormous public health impact, we have a limited understanding of the biology fueling accelerated CVD phenotypes in patients with CKD.      The question we wish to answer is \\u00e2\\u0080\\u009cWhich biomarkers/pathways explain the risk of cardiovascular disease in patients with chronic kidney disease?\\u00e2\\u0080\\u009d Previous investigations typically focus on a single biomarker\\u00e2\\u0080\\u0099s association with adverse CV outcomes, and use multivariable adjustment to clarify the role of renal dysfunction in this association. This approach is limited, not only due to its myopic single-marker focus, but also because confounders cannot be distinguished from mediators by multivariable adjustment alone.      Our group has recently published the first large-scale aptamer proteomics study of CV risk prediction, in partnership with Somalogic (Boulder CO).The Heart and Soul Cohort includes 1024 participants with stable coronary artery disease, of whom 978 had proteomics (1124 proteins) measured at baseline. Approximately 30% had CKD (defined as estimated glomerular filtration rate (eGFR) <60ml/min/1.73m2 or albuminuria >30mg/g), and prior work has shown that cystatin C, a sensitive measure of renal dysfunction, is independently associated with atherosclerotic CVD and heart failure (HF) in the cohort. In the current project, rather than risk prediction, we will investigate novel associations between proteins and either atherosclerotic CVD or HF in the Heart and Soul Cohort, and then corroborate findings with GWAS analyses performed for baseline and longitudinal CVD in the Framingham cohort.\",\n\"Jan15, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Duan, Rui \",\n\"HARVARD SCHOOL OF PUBLIC HEALTH\",\n\"A federated transfer learning framework for improving genetic risk prediction in ancestry-diverse populations\",\n\"approved\",\n\"The limited representation of minorities and disadvantaged populations in large-scale clinical and genomics research has become a barrier to translating precision medicine research into practice, which may further exacerbate known health disparities, and there is an urgent need for methods in the field of statistics and ML to address biases and unfairness caused by such lack of representation. In this project, we aim to improve knowledge discovery and risk prediction in underrepresented populations through two strategies: (1) leverage the shared knowledge across populations through novel transfer learning approaches that allow rapid evolution of predictive models as they are \\u00e2\\u0080\\u009ctransferred\\u00e2\\u0080\\u009d between data sets to jumpstart the modeling process; (2) develop efficient and safe information-sharing strategies to facilitate biomedical data integration, improve the diversity, and reduce the potential for model overfitting. \",\n\"Polygenic risk scores (PRS) have shown promising potentials for early disease detection, prevention and intervention. However, as existing genome-wide studies were predominantly conducted in European-ancestry (EA) populations, the performance of PRS is much poor in non-EA populations compared with EA populations, which may exacerbate existing health disparities. The goal of our proposed work is to develop novel transfer and federated learning methods to improve the performance of PRS in ancestry-diverse populations. We will (1) develop a transfer learning framework based on penalized high-dimensional regressions, which transfer the shared genetic architectures learned from individual-level datasets and available GWAS summary statistics to a target population while accounting for the heterogeneity across populations; (2) develop communication-efficient federated learning algorithms to increase the training sample size of the target population by incorporating multiple study cohorts with communication efficiency and individual-level privacy protection. Completion of this project will lead to novel tools that produce high quality PRS with better predictive performance for underrepresented non-EA populations, which can ultimately help advance precision medicine and reduce the disparity in PRS-based research.   We will apply the proposed methods to combine summary statistics obtained from the applied datasets as well as summary statistics obtained from datasets outside of dbGAP, including the UK biobank, and construct PRS for both quantitative traits and binary traits including cancers, cardiovascular diseases, Alzheimer's disease, and mental disorders, targeting underrepresented populations. We will use metrics such as R squared and AUC to evaluate the prediction models for quantitative traits and binary traits respectively.   We intend to publish methodology papers in the field of statistical genetics and machine learning to share our methods to the scientific community. We will also publish our study results of PRS models to genetic study community and precision medicine community. We will publish open-source software packages in github and cran. Our proposed methods can be generalized to other areas in health and biomedical research to facilitate knowledge discovery and risk prediction in a minority sub-population, and can help reduce health disparities and advance precision medicine. \",\n\"May31, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Dewey, Frederick \",\n\"FORESITE LABS, LLC\",\n\"Systematic analysis of the relationship between genetic drug target modulation and clinical outcomes\",\n\"approved\",\n\"We will combine aggregated information from many different studies from dbGAP and internal and external databases to gain a better understanding of the relationship between genetics and outcomes such as disease and its response to treatment. We plan to use the data to improve inferences made with genetic information, leading to more reliable risk prediction and more efficient drug development. We also aim to reduce harm and improve outcomes when developing new treatments. A secondary benefit is that treatments developed in this manner might be more targeted and therefore more cost-effective. Thirdly, we aim to replicate, improve and develop robust methods for understanding the relationship between genetic targets, disease biology, and health-related outcomes.\",\n\"This application will utilize the MVP genotype-phenotype association summary statistics to support ongoing research in discovery and validation of therapeutic targets and biomarkers. These include co-localization, Mendelian randomization analyses, and other summary statistic analyses in combination with data from internal and external databases, and development and validation of polygenic risk score (PRS) approaches applicable to multiple ancestries.  All analyses derived from the requested datasets will only be used in aggregated form and will abide by dataset-specific limitations. Access to the data will be limited to the collaborators named in our application (i.e. our staff and Dr. Connor Emdin of FL2022-001) and will abide by all applicable data use certifications. The proposed research does not involve patient identification and is consistent with the data use agreement and consent requirements. \",\n\"Oct06, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Dewan, Andrew \",\n\"YALE UNIVERSITY\",\n\"Genetic studies on peripheral artery disease (PAD)\",\n\"approved\",\n\"Peripheral artery disease (PAD) is a vascular disease with severe morbidity and mortality. Additional data to validate our findings of genetic risk variants for PAD could improve our understanding of the etiology. We will use the summary statistics being requested and summary statistics generated from the PAD case-control data being requeested as validation for our findings from an independent dataset and to construct new genetic models of PAD that incorporate information from multiple genetic variants.  \",\n\"Our objectives for this research is the discovery and validation of genetic risk factors for peripheral artery disease (PAD). We currently have genotype and phenotype data from the UK Biobank. We plan to use PAD summary statistic data from dbGAP in two ways: 1) validate findings from the analysis of UK Biobank; and 2) conduct polygenic risk score (PRS) analysis by using the GWAS summary statistics from million veteran program to construct models for us to then apply to the UK Biobank data (not a part of this application) and the other PAD case-control phenotype and genotype data being requested from dbGaP. No individual genotype or phenotype data from any of the dbGaP datasets we are requesting will be combined with the UK Biobank data or any other external data. The only combination of data will potentially the combination of summary statistics from dbGaP datasets and UK Biobank summary statistics for PAD that we generate in a meta-analysis framework. This does not increase risk or create new risks to the participants in any of the dbGaP studies we are requesting. Results from this study will be published in a peer-reviewed journal. \",\n\"2022, Aug26\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Derks, Eske \",\n\"QUEENSLAND INSTITUTE OF MEDICAL RESEARCH\",\n\"Incorporating depression PRS into network analyses of depressive symptoms\",\n\"expired\",\n\"Depression is a highly prevalent psychiatric disorder that causes substantial burden worldwide. Patients with depression present in many different ways, with a large number of different symptom profiles being observed. In this project we seek to investigate how genetic risk for depression is related to different patterns and networks of depression symptoms.\",\n\"Depression is a clinically heterogeneous disorder. The network approach to psychopathology, which conceptualises mental disorders as a system of causal interactions between symptoms, embraces the large heterogeneity observed within psychiatric disorders. In this project, we will incorporate depression PRS into a network analysis of depressive symptoms in order to assess how PRS is associated with particular symptoms while accounting for between symptom relationships.   We will construct a depression PRS from large-scale depression GWAS data, and conduct a phenotypic network analysis of depressive symptoms in the UK Biobank. We will explore how the network structure of symptoms differs between individuals with high vs. low PRS. We will not combine GWAS data of the Million Veterans Program with other datasets (e.g., PGC) ourselves. We do request permission to use the meta-analyzed datasets (i.e., MVP and PGC) that are currently being generated in a project jointly led by the MVP and PGC. We do not believe this creates any additional risks to participants as the meta-analysis is restricted to summary statistics only).\",\n\"Apr11, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Demontis, Ditte \",\n\"UNIVERSITY OF AARHUS\",\n\"Polygenic effects on psychiatric disorders, comorbidity risk and severe outcomes \",\n\"approved\",\n\"Common genetic variation contributes considerably to human behavior and complex diseases like psychiatric disorders. We will use the results generated from genome-wide association studies (GWAS) conducted in the Million Veteran Program (MVP) to advance our understanding of how the polygenic architecture affects the risk of psychiatric disorders and comorbidities in the Danish iPSYCH cohort.\",\n\"Common genetic variation contributes to variability in behavior and disease risk. We will use the summary statistics generated by genome-wide association studies (GWASs) of individuals in the Million Veteran Program (and meta-analyses with other cohorts)  to examine how the polygenic architecture contributes to individual differences in biology, comorbidity patterns and disease outcomes in the Danish iPSYCH cohort consisting of 130K genotyped samples with major psychiatric disorders (80K individuals) and population-based controls (50K individuals). The MVP data will be used as basis for generating polygenic risk scores for the individuals in the iPSYCH cohort for subsequent analyses of how the polygenic risk load associates with overall risk for various health outcomes and comorbidities as well as analyses stratifying cases by their polygenic risk load to evaluate risks within polygenic risk score bins. Additionally, the MVP data will be used for Mendelian randomization analyses and mediation analyses to test for causality and direct and indirect effects of one phenotype on another. The described analyses will be used to evaluate the genetic contribution to risk focusing on psychiatric disorders (including substance use disorder phenotypes). \",\n\"2022, Oct25\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Deloukas, Panagiotis \",\n\"UNIV/LONDON-QUEEN MARY&amp; WESTFIELD COLL\",\n\"Investigating the casual relationship between thyroid function and serum lipid levels.\",\n\"rejected\",\n\"In this study we aim to investigate the causal relationships between thyroid function and blood lipid levels using the Mendelian Randomization approach.  \",\n\"Observational studies reported an association between thyroid function and serum lipid levels. This study will use a two-sample Mendelian Randomization (MR) approach to examine whether a causal relationship underlies this association. MR uses genetic variants as proxies to evaluate the casual effect of an exposure on the outcome of interest. In this two-sample MR we will use summary-level statistics from the largest genome-wide association studies (GWAS) on thyroid function traits and blood lipid levels. Causal estimates will be assessed using inverse-variance weighted method, followed by sensitivity analyses. No participant-level data will be used in this project.   \",\n\"2019, May06\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Del-Aguila, Jorge \",\n\"MERCK AND COMPANY\",\n\"Target identification using human genetics for peripheral artery disease (PAD)\",\n\"closed\",\n\"Peripheral artery disease (PAD), a condition where fatty deposits build up in the arteries and restrict blood flow.  When untreated, PAD can lead to amputation, additional cardiovascular disease and, in the worst scenario, death.  We aim to use the human genetic analysis of PAD from the Million Veterans Program (MVP) to identify dysfunctional genes that lead to PAD using a standard analytical pipeline in the field of human genetics.\",\n\"Peripheral artery disease (PAD) is a leading cause of cardiovascular morbidity and mortality, however current medical treatments mostly target risk factors for PAD, such as diabetes, hypercholesterolemia and high blood pressure and not PAD itself, or involve invasive surgical procedures.  As part of Merck's efforts to identify novel drug targets to treat disease, we aim to identify promising targets for PAD by analyzing the GWAS PAD summary statistics from the Million Veteran Program (MVP) recently published by Klarin et al (Nature Genetics 2019).  This effort is based on the observation that human genetic evidence linking a target to disease increases the probability of success to advance a drug through development by Nelson et al (Nature Genetics 2015).  Our analysis pipeline aims to identify the most likely causal gene at each of the genome-wide significant loci (P<5e-8).  The pipeline contains statistical analyses to perform 1) fine-mapping of the 95% credible set to identify most likely causal variants, 2) conditional analysis to detect if multiple independent associations exist, 3) colocalization analysis with summary eQTL data from GTEx and summary pQTL data from Sun 2018 to identify putative causal genes.  \",\n\"Apr09, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Davis-Dusenbery, Brandi \",\n\"SEVEN BRIDGES GENOMICS, INC.\",\n\"RADx Data Hub Infrastructure Testing\",\n\"approved\",\n\"The RADx Data Hub provides a critical repository and ecosystem to support data access and analysis. This data request will allow testing and continued development of the RADx Data Hub in order to support the long term research objectives of RADx investigators. \",\n\"This data will not be used for research purposes, but rather to test the RADx Data Hub for accessing files from controlled datasets and uploading them to a Data Hub workbook.  This test will show that only authorized and authenticated researchers can access controlled data on the RADx Data Hub\",\n\"Nov14, 2022\",\n\"phs002519.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Davis, Lea \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Multi-ethnic and ancestry-specific polygenic scores in blood lipids and other laboratory test results\",\n\"approved\",\n\"Electronic health records linked to genetic data hold great promise for improving human health.  Our research in the Vanderbilt University Medical Center biobank, BioVU, harnesses the hundreds of laboratory test results collected in routine clinical care to discover novel gene-biomarker associations.  A major focus of our project is to include patients of diverse ancestries, a population largely overlooked by the last decade of genetic studies.  In an initial experiment, we are requesting results from a new multi-ethnic genetic study of blood lipids in the Million Veteran Program participants.  We will use these results to quantify genetic risk factors for blood lipids across BioVU patients (~20% of non-European ancestry), and to test these for ancestry-specific associations with lipids and other biomarkers in the biobank.  We are additionally requesting results from a study of problematic alcohol use in the Million Veteran Program participants. We will use these results to quantify genetic risk factors for problematic alcohol use across BioVU patients (~20% of non-European ancestry), and to test these for ancestry-specific associations with buprenorphine treatment duration in opioid use disorder patients in the biobank.\",\n\"Genome-wide association studies (GWAS) over the past decade have revealed thousands of variants associated with common complex diseases.  These studies have powered analyses with polygenic scores, which summarize the effects of multiple genetic variants into a single measure, and which can be tested for association with the same, or a different trait in any genotyped sample, using only summary statistics from an external GWAS.  Previous GWAS, however, have overwhelmingly been in European-ancestry individuals, hampering our ability to generate valid polygenic scores in people of other ancestries.  We are requesting access to blood lipid GWAS summary statistics from the Million Veteran Program (MVP) of ~300,000 multi-ethnic participants.  Our objectives are:  1) To develop multi-ethnic and ancestry-specific polygenic scores for blood lipids from the MVP summary statistics; and 2) To test these polygenic scores for association with blood lipids and other laboratory test results in patients of different ancestries from the Vanderbilt University Medical Center biobank, BioVU. 3) To develop multi-ethnic and ancestry-specific polygenic scores for Problematic Alcohol Use from the MVP summary statistics; and 4) To test these polygenic scores for association with buprenorphine treatment duration (BTD) in Opioid Use Disorder (OUD) patients of different ancestries from the Vanderbilt University Medical Center biobank, BioVU   BioVU includes >285,000 patients (~20% of non-European ancestry) with DNA samples linked to electronic health records (EHR) that span 10 years, on average.  Genotypes were measured on the Illumina MEGAEX array of >2 million SNP markers, and ethnicity was assigned by the principal components of ancestry.  Using the MPV summary statistics, our next step is to derive multi-ethnic and ancestry-specific polygenic scores (e.g., using the software PRSice, LDpred, and/or PRS-CS), and to test these scores for association with blood lipids and other laboratory test results (e.g., C-reactive protein) in the EHR of BioVU patients, stratified by ethnicity.  Results will be shared broadly in conference presentations and published manuscripts.  Despite the benefits associated with buprenorphine in the treatment for Opioid Use Disorder (OUD), retention remains poor. Research on the impact of co-occurring medical issues on buprenorphine treatment duration (BTD) has yielded mixed results. It is also unknown whether genetic risk in the form of polygenic scores (PGS) for OUD are associated with BTD. The objectives of our research are to test the association between somatic and psychiatric comorbidities and BTD, and to determine whether PGS for OUD and several other OUD-related traits are associated with BTD. We would like to request the GWAS summary statistics from Zhou et al. (2020), to generate PGS of Problematic Alcohol Use and apply them to individuals with genotype data from Vanderbilt University Medical Center's (VUMC) biobank.  In accordance with the data use limitations for the dataset, we are not requesting any individual-level data.  We do not anticipate collaborating with other institutions on this project. \",\n\"2019, May17\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Chimusa, Rugamika \",\n\"UNIVERSITY OF CAPE TOWN\",\n\"African Genomic Risk Assessment\",\n\"closed\",\n\"Various models for genetic risk prediction have been possible due to numerous genome-wide association studies mostly of European-ancestry with some studies reaching up to hundreds of thousands study subjects. Clinical utility of disease risk prediction has largely been explored in populations of European-ancestry. While it is well-known that the prediction varies according to genetic ancestry of the population, current genetic risk prediction models have not been evaluated in populations of non-European ancestry. This raises questions as to whether the clinical utility within the predictive power of these methods could be equitable across populations and diseases, it is currently unclear how to accurately predict health and disease risk in populations of non-European ancestry. Here, we will independently assess current polygenic risk score tools through various human phenotypes including cancers, Chronic obstructive pulmonary disease, Autism Spectrum and Cardiovascular, Heart, Lung and heart diseases) from multi-ethics and simulated data; will develop a Bayesian posterior inference that uses a combination of (a) likelihood function that jointly sampling from training large reference data (General Research use data sets) and targeted training data from a given disease data.\",\n\"Various models for genetic risk prediction have been possible due to numerous genome-wide association studies mostly of European-ancestry with some studies reaching up to hundreds of thousands study subjects. Clinical utility of disease risk prediction has largely been explored in populations of European-ancestry. While it is well-known that the prediction varies according to genetic ancestry of the population, current genetic risk prediction models have not been evaluated in populations of non-European ancestry. This raises questions as to whether the clinical utility within the predictive power of these methods could be equitable across populations and, it is currently unclear how to accurately predict health and disease risk in populations of non-European ancestry.  This project aims to:  a. Assess current polygenic risk score (PRS) tools. We aim at running various PRS tools on (1) various simulated data mimicking various population genetics model (we will use datasets from General Research ); (2) various type of Cancer phenotypes; (3) Chronic obstructive pulmonary disease (COPD); (4) Autism Spectrum Disorder; (5) Cardiovascular Disease, Heart, Lung and heart diseases. To evaluate the accuracy and predictive power of current PRS tools on accessed data sets with differing sample size, phenotype from same and/or different ethnic groups, each accessed data set with specific phenotype from specific a particular population will be assessed independently. The data set with phenotype information will NOT be merged, however accessed large-scale (only General Research use) data sets based on similar ethnic groups as for data set with specific phenotype will be be used as reference training for some PRS tool. The General Research Use data sets with 1000 Genomes data will merged after rephasing and imputation processes and the final merged data set will be used as training reference panel in PRS analysis and also from  developed new tool in each specific phenotypic data set (each cancer type , Chronic obstructive pulmonary disease (COPD); Autism Spectrum Disorder;  Cardiovascular Disease, Heart, Lung and heart diseases, respectively).  b. Develop a Bayesian posterior inference that uses a combination of (1) a likelihood function that jointly sample from training large reference data and training data from the target population by modelling linkage disequilibrium and (2) a finite mixture of normal distributions prior on the marker\\u00e2\\u0080\\u0099s effects, local-specific ancestry and individual non-genetic risk factors.  c. To improve the power for risk prediction and independently apply the developed methods to various type of phenotype data sets from diverse ethics, We will develop an approach for integrating large-scale (at least 50 k samples) data sets with general purpose (Not data with phenotype information to avoid cross phenotype merge and potential risks to participants) across diverse and multi-ethics to serve as training data set in the model. Each cancer type data data set, Chronic obstructive pulmonary disease (COPD) data set; Autism Spectrum Disorder data set; and Cardiovascular Disease, Heart, Lung, Blood diseases and heart disease data set will be independently analysed (as targeted training data set within GWAS summary statistics, respectively) using a separate large-scale multi-ethnic training reference data (as described above). We aim to dissimulate broadly our findings with respect to each phenotype and publish our work to the scientific community through high impact factor journal.\",\n\"Jul20, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Chiang, Charleston \",\n\"UNIVERSITY OF SOUTHERN CALIFORNIA\",\n\"Leveraging the Evolutionary History to Improve Identification of Trait-Associated Alleles and Risk Stratification Models of Cardiometabolic Traits in Diverse Human Populations\",\n\"approved\",\n\"It is imperative to extend human genetic studies of metabolic traits and diseases to include diverse populations around the world. In this study we will gather diverse populations not standardly included in existing GWAS studies (i.e. those outside of the European and East Asian ancestry populations, and those from the major ethnic minority populations such as the African Americans and Latino Americans) to perform a genetic study. We will evaluate the currently available genomic resource to see if it is helpful in genetic studies of understudied populations, explore alternative approaches that could overcome the (presumed) lack of genomic resources for these populations, and combine data from multiple populations to identify genes associated with diseases like type-2 diabetes.\",\n\"In order to personalize medicine for everyone, we need to combine an improved understanding of environmental and lifestyle contributions to disparity with a better understanding of the genetic etiology of complex diseases in diverse, often understudied, populations. For metabolic diseases such as type-2 diabetes and obesity, they are prevalent among Oceanian and Asian populations. While there are some large-scale efforts to study these traits and related diseases in East Asia, other parts of the world (such as Southeast Asia or the Pacific) have been ignored. Because each of these individual populations are understudied and cohort recruitments have been difficult to extend to large scale, it is necessary to aggregate multiple cohorts to obtain substantial statistical power. Furthermore, genetic studies of these understudied populations brings the opportunity to identify population- or region-specific variants of large effect, such as the variant found in CREBRF gene (Minster et al. Nat Genet 2016). Identification of these variants would ideally require sequencing data, which is also rare for understudied population. We will thus also evaluate the imputation accuracy for these population using the best available resources currently available, including TOPMED and GenomicAsia 100K reference panels. We will then apply imputation-based GWAS meta-analysis design to identify novel variant association to quantitative traits such as fasting glucose and insulin levels, BMI, Waist-to-hip ratio, and binary traits such as type-2 diabetes and obesity. We will also evaluate the efficacy of risk stratification models based on polygenic risk scores in different populations, and attempt to create an efficacious model across continental or subcontinental populations. We will further investigate the population structure in these cohorts, and explore its impact in confounding or empowering genetic mapping studies. Our study design has been approved by an expedited IRB review and all of our findings will be published to share with the larger scientific community and the public.\",\n\"May17, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Cheung, Ching Lung \",\n\"UNIVERSITY OF HONG KONG\",\n\"Mendelian Randomization of Lipids\",\n\"expired\",\n\"Circulating lipid plays an important role in multiple chronic diseases. The current study will elucidate whether circulating lipids play a primary role in disease development or as a marker of chronic diseases.\",\n\"Circulating lipid is an important risk factor and mediator of multiple chronic diseases. Thus, it is important to evaluate if circulating lipid plays an important role in contributing to the pathogenesis of chronic diseases. We aim to evaluate the role of circulating lipid in multiple chronic diseases via Mendelian Randomization (MR) approach. Summary statistics of GWAS of lipids from the MVP programme will be used in the MR study. Summary statistics of circulating lipids will either be used as the primary variables or confounding factors that will be adjusted in the multivariable analysis. The findings will be important to elucidate if lipids play a role in pathogenesis of different chronic diseases, and whether lipids are mediated in the relationship between other risk factors and chronic diseases.\",\n\"Apr23, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"BACANU, SILVIU-ALIN \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"Cross species alcohol trait analyses\",\n\"rejected\",\n\"We intend to analyze this human study together with our animal ones to discover genes that regulate alcohol traits in humans and animals.\",\n\"We will use summary statistics from this study to impute agnostic and transcriptomic gene statistics for  this study.  Subsequently we will used a weighted FDR approach to combine these human results with the animal model results from Virginia Commonwealth University Alcohol Research Center. Each animal model will use the summary stats from the other animal models to prioritize genes.\",\n\"2020, Aug07\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Avillach, Paul \",\n\"HARVARD MEDICAL SCHOOL\",\n\"Exploration of the ORCHID randomized controlled trial (RCT) results\",\n\"closed\",\n\" In a preliminary step, we will use data from the  ORCHID trial to reproduce the analyses performed in the original paper by Self, WH to get  baseline results enabling further comparisons. Additionally we will analyze the primary and secondary outcomes using other treatments as treatment groups (ie information available regarding premedication and in-hospital use of corticosteroids, tocilizumab, sarilumab, interferon, immunomodulators, azithromycin).  We will also compare the lab trajectories between the different treatment groups (labs information available up to the 5th day), and correlation between lab trajectories and the main outcome. We will study the secondary outcomes and safety events with a modified Intention-To-Treat approach (mITT), that is constituting the treatment groups looking at the treatment that has actually been received by the individuals as opposed to the treatment originally planned to be given (i.e. differences arising from individuals that dropped from the study or that did not received the complete treatment regimen as planned in the initial protocol). \",\n\"We will use data from the PETAL Network: Outcomes Related to COVID-19 Treated With Hydroxychloroquine Among Inpatients With Symptomatic Disease (ORCHID) Trial to reproduce and perform new analyses. In a preliminary step, we will reproduce the analyses performed in the original paper by Self, WH,et al.,  Effect of Hydroxychloroquine on Clinical Status at 14 Days in Hospitalized Patients With COVID-19, to get baseline results enabling further comparisons. This includes every analysis included in the Clinical Trial statistical analysis protocol. Additionally, we will perform several new analyses based on the data collected for this trial, as ancillary studies. 1) We will use the information regarding the concomitantly administered treatments received by the patients during the trial to look for an association between the administration of these treatments and any of the outcomes (i.e. using information available regarding premedication and in-hospital use of corticosteroids, tocilizumab, sarilumab, interferon, immunomodulators, azithromycin). Since these treatments were prescribed based on the actual health status of the patients, as opposed to the random allocation of hydroxychloroquine/placebo, the analyses will need to be adjusted on clinical and biological characteristics of these patients to take into account confounding factors. This adjustment could be made using multivariate models or propensity score matching, which allows to study the average treatment effect. Moreover, we will look for interaction between the concomitant drugs and the treatment groups, regarding both the clinical outcomes and the safety events. 2) We will compare the lab trajectories between the different treatment groups (labs information available up to the 5th day), and the correlation of these trajectories with the clinical outcomes. We will be able to compare these trajectories with external work in the literature studying the lab trajectories for COVID-19 patients, namely one performed by the 4CE consortium. 3) We will study the secondary outcomes and safety events with a modified Intention-To-Treat approach (mITT), that is constituting the treatment groups looking at the treatment that has actually been received by the individuals as opposed to the treatment originally planned to be given (i.e. differences arising from individuals that dropped from the study or that did not received the complete treatment regimen as planned in the initial protocol). The original statistical analysis plan includes an mITT for the primary outcome only. The mITT is especially suited for the safety outcomes analyses. Studying the subpopulation of patients which were excluded of the study after randomization, for worsening of their condition. We plan to look for specific characteristics of this population regarding the outcomes, the safety events and the concomitant treatment used.\",\n\"2021, Jan04\",\n\"phs002299.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Auton, Adam \",\n\"23ANDME, INC.\",\n\"Discovery & Development of New Medicines & Vaccines using GWAS Summary Statistics\",\n\"approved\",\n\"Genome-wide association studies (GWAS) identify genetic variants associated with complex diseases which can be useful in identifying new therapeutics and optimizing the development of existing therapeutics.  We will combine results from multiple GWAS studies to create well-powered studies to conduct this research.  Scientists with expertise in human genetics will interpret the results.   \",\n\"We will use the GWAS summary statistics for the purposes of research focused on identifying genes associated with patient outcomes which can be used to discover or optimize development of new medicines and vaccines.  This research is conducted by performing meta-analyses of multiple genetic association studies to create a well-powered study of the trait of interest and colocalization with expression QTL results. These GWAS summary statistics will be meta-analyzed with GWAS summary statistics derived from analysis of UK Biobank, FinnGen, 23andMe and other publicly available data sets and colocalization with expression QTL summary statistics including Blueprint and GTEX and other publicly available datasets.     We also plan to use the GWAS summary statistics to evaluate different strategies for fine and gene mapping, e.g. via colocalization to expression, splicing and other QTL datasets.  This research may include use of results in research collaborations, but the GWAS summary statistics will not be passed to any third party. Any publication relating to this work would include appropriate attribution/credit. \",\n\"2021, Jul05\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Auton, Adam \",\n\"23ANDME, INC.\",\n\" Study of blood lipid levels in a multi-ethnic cohort\",\n\"closed\",\n\"The goal of this project is to explore the genetics of blood lipid levels in more detail by combining them with results from the 23andMe research cohort. Combining these summary statistics will lead to increased power to find new associations and accurately pinpoint disease-causing genes. \",\n\"This proposal aims to use MVPLipids data as an independent source of GWAS summary statistics for blood lipid levels to further explore the genetics of blood lipid levels in the multi-ethnic research cohort.   We will use the provided summary statistics in combination with association statistics from GWASes on the 23andMe cohort to perform meta-analysis and co-localization. Since the power of association and fine-mapping tests increases with sample size, we expect that combining the summary statistics may lead to the discovery of new associations. We will also use the summary statistics to predict lipid levels in the multi-ethnic cohort to examine if the associations have predictive utility.  The use of this data is requested solely for health/medical/biomedical purposes, and does not include any study of population origins or ancestry of the MVPLipids data. \",\n\"Sep14, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Auer, Paul \",\n\"MEDICAL COLLEGE OF WISCONSIN\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease. Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data.\",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry. Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community. However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets.\",\n\"2023, Mar29\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Andreassen, Ole \",\n\"UNIVERSITY OF OSLO\",\n\"TOP study\",\n\"approved\",\n\"Severe mental disorders such as schizophrenia, bipolar disorders and major depression are severe mental illnesses with high degree of heritability. Still, the disease mechanisms are not yet known. The study will combine genetic data from samples from several populations, to investigate genetic risk factors. Further, the study will investigate overlapping genetic factors among several psychiatric disorders, including schizophrenia, bipolar disorder, major depression, autism and ADHD, and with important comorbid mental disorders such as substans use disorders, somatic disease such as cancer, and cardiovascular disease, and link to mental traits, such as sleep patterns. \",\n\"The overall research objective is to gain more knowledge of genetic factors involved in development of severe mental disorders (eg  neurodevelopmental disorders, schizophrenia, bipolar and depression disorder and substance use disorders), and overlapping phenotypes - to learn more about the pathophysiology and improve clinical treatment. The objective is to identify genes related to these disorders, and potential overlapping phenotypes and shared genetic architecture and pleiotropy mechanisms:  1) Focus on gene discovery with association studies, including newly identified genes for neuronal migration, neurotransmission, synapse function, and immunological factors. Here we will use use gene variants recently identified in association studies and investigate if these findings are replicated in independent samples. We will use diagnoses together with phenotype characteristics. 2) Focus on overlapping genetic architecture among psychiatric disorders and mental traits. There are indications that the same genetic factors could lead to different clinical phenotypes. We wish to investigate if the genetic risk factors are overlapping between schizophrenia, bipolar disorders and major depressive disorder, and between other mental disorders such as substance use disorders, eating disorders, anxiety, autism, OCD, PTSD, ADHD or with dementia, another brain disorder with comorbid psychiatric symptoms, and a range of mental traits such as cognition, mood and sleep characteristics. 3) Focus on comorbid substance use, including smoking. Both in schizophrenia and bipolar disorder, there is a 2-3 fold higher prevalence of smoking and 50% comorbidity with substance use disorders in general, including alcohol. We will investigate potential overlapping genes between substance use traits and disorders  and severe mental illness, both in terms of overlapping genetic architecture, and specific overlapping gene loci. We will also investigate if these risk genes individually or polygenic effects, lead to higher risk of secondary diseases, such as cancer or other somatic disorders, which from epidemiological evidence seems to be relatively high in severe mental illness.  4) Focus on somatic comorbidity, including cardiovascular disease. There is a severely increased mortality in mental illness, especially schizophrenia and bipolar disorder, but also major depression. We will investigate potential overlapping genes between cardiovascular disease and severe mental illness. We will also investigate if such genes are enriched in patients with comorbid heart diseases. There is some epidemiological evidence for overlapping pathophysiology, and unmedicated early phase patients seem to have indicated abnormal heart conditions in severe mental illness.   For these approaches it is necessary to have a good estimates of the genetic variation in the general population.  We will focus on diseases involving adults (comorbidity etc.) but also compare with ADHD and autism since these are neurodevelopmental disorders which can lead to later adult psychopathology.   We will use standard statistical tools for analysis of gene variants associated with diseases and phenotypes, and use novel statistical tools for pleiotropy based on annotation and enrichment methods, and mathematical modeling including MiXeR, condFDR and multivariate approaches, such as MOSTest and new prediction tools, including PHS.   The proposed research is consistent with Use Restrictions for the requested dataset, since the data will be used for gaining more knowledge about disease mechanisms of the severe mental disorders, and strictly for the purpose outlined in this application, and we will follow the regulations about data security, publication and IP regulations.  Will not share data with other persons than the listed personnel from Univ of Oslo.  \",\n\"Apr21, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Andreassen, Ole \",\n\"UNIVERSITY OF OSLO\",\n\"Enrichment methods to improve gene discovery in neuropsychiatric disorders\",\n\"approved\",\n\"We will develop and apply novel statistical genetics methods to improve gene discovery in neuropsychiatric disorders and comorbidities, and improve risk prediction / stratification based on genetic data. This will provide more knowledge about genetic factors related to neuropsychiatric diseases, including functional understanding and risk prediction may have implications for clinical use, including stratification. All the results of the project will be made available in accordance with IRB and data protection regulations\",\n\"Neuropsychiatric disorders are highly heritable, but few genes have yet been detected. We propose to leverage a new statistical framework based on annotation and pleiotropy enrichment, to improve gene discovery in genome-wide association studies (GWAS), reveal genetic architecture and improve risk prediction in neuropsychiatric disorders.   There are certain regions of the genome that are more likely to have an effect (genic annotation), and we will use this information to improve statistical methods. Further, there are overlapping phenotypes in neuropsychiatric disorders, with similar symptoms profiles in several diseases. Further, there is a comorbidity with other disorders. It is likely that such overlap can be explained by common underlying genetic factors (genetic pleiotropy).  Our methodology has the potential to improve the power of traditional GWAS, increase the likelihood of identifying replicable gene variants and improve phenotypic prediction from SNP data (generalization performance). We will now apply our statistical approach to psychiatric disorders.  The overall aim is to discover new genes in schizophrenia, bipolar disorder, major depression, ADHD, autism, anxiety, dementia and related neurodegenerative disorders, substance use disorders, including alcohol use with the following sub-aims:  1) Assess the validity of our statistical approach using completed GWAS p-values in psychiatric disorders 2) Leverage genomic annotation information to improve gene discovery in psychiatric disorders 3) Leverage associated phenotypes (genetic pleiotropy) to improve gene discovery in psychiatric disorders 4) Use the information obtained in 1-3 to improve SNP-based prediction (generalization performance) in neuropsychiatric disorders, and identify functional pathways.  Methods:  We will use in-house pipeline for annotation of different genetic markers. We will leverage overlapping phenotypes for pleiotropy enrichment (immune-, cardiovascular- and neurological diseases, cancer and anthropometry), and diverse genetic categories for annotation enrichment. We will use a statistical framework based on bayesian approach, including Q-Q plots, FDR and replication rate, and  mixture model (CM3, BGMG, univariate causative mixture models (MiXeR), bivariate MiXeR, Finemap and GSA MiXeR, the recent MOSTest multivariate method and extensions, including functional information and mediation (mendelian randomization) approaches.  All analyses are done at Univ. of Oslo. No data are shared with third parties. We will provide summary statistics (anonymous data) as part of publications, if the IRB approvals allow. \",\n\"May25, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Anchang, Benedict Nchang\",\n\"NIH\",\n\"The Materno-Fetal System during Pregnancy at Single-cell resolution\",\n\"approved\",\n\"This study aims to identify and characterize major cell types and their interactions representing placenta development across various ethnic groups and benchmarking the results with the Human Cell Atlas data and other environmental related databases. The tools and resources to be developed will be made publicly available to the research community as well as support innovative data generation and methods to reduce health disparities in the field of reproductive health.\",\n\"Prenatal environmental chemical exposures such endocrine disruptors and maternal infections (e.g., HIV and malaria) and morbidities (e.g., hypertensive disorders of pregnancy) are the leading causes of global perinatal (fetal and neonatal) and maternal morbidity and mortality. In addition, the exposure of these factors at different levels in different regions and ethnic groups results in global disparity of reproductive health outcomes. For example, Nigeria contributes the largest portion about 20% of the burden of maternal mortality worldwide, despite having just 2% of the global population. To improve understanding of how prenatal exposures affect fetal growth, maternal health and ultimately lifelong health outcomes, cellular reference maps of normal biological processes need to be developed during pregnancy. As a PI and part of the Ancestry network for Human cell atlas funded by the Chan Zuckerberg Initiative, the goal of this study is to provide a reference reproductive and perinatal developmental cell map of African and non-African ethnic groups that can be used to study early and long-term effects during normal and perturbed pregnancy from environmental and socioeconomic factors.  To account for environmental variations due to geography and ethnicity we plan to integrate single-cell RNA-sequencing placenta data from 3 Nigeria ethnic groups; Fulani, Ibo and Yoruba with other ancestry groups e.g., European ancestry. For example, the dbGaP single-cell RNA-sequencing data from the phs001886.v4.p1 study of human myometrium during spontaneous term labor include 113 placenta samples of various placenta regions: the placental villous tree, basal plate, and chorioamniotic membranes of women across a range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19. Integrating the control data from non-African ancestry with the African study data will increase sample size and diversity as well help generate a robust reference placenta cell atlas.  I will be responsible for accessing the controlled access data. The research use of the data is limited only by the terms of the model Data Use Certification.  Also, this research study is independent of collaboration with researchers at other institutions.    \",\n\"2023, Mar20\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Almasy, Laura \",\n\"CHILDREN'S HOSP OF PHILADELPHIA\",\n\"Polygenic effects on complex traits across development\",\n\"approved\",\n\"Genetic markers are an important source of variation in human health and behavior. We will use the results generated from genome-wide association studies (GWASs) conducted in the Million Veteran Program (MVP) to advance our understanding of how multiple genes may influence medical, psychiatric, and substance use outcomes in children and adults.\",\n\"Summary statistics from genome-wide association studies (GWAS) conducted within the Million Veteran Program (MVP) will be used to examine the genetic architecture of complex traits across development. We will use the MVP GWAS summary statistics to compute genetic correlations, create polygenic scores, and employ other statistical techniques, i.e., Mendelian randomization, to examine the genetic contribution to an array of phenotypes, including medical, psychiatric, and substance use symptoms and disorders. The findings will then be tested in independent data sets that include the same medical, psychiatric, and substance use phenotypes in samples that include a broad age span. \",\n\"Jun22, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Adibi, Jennifer \",\n\"UNIVERSITY OF PITTSBURGH AT PITTSBURGH\",\n\"Tissue type differences in placental transcriptome\",\n\"expired\",\n\"Maternal exposures including pre-pregnancy obesity predispose women to adverse pregnancy outcomes possibly caused by molecular changes in the placenta. Obesity may cause changes in placental cell types or through modifying multiple genetic pathways. This study will help characterize placental changes in gene expression and cell types that may explain associations between maternal obesity and gestational age at delivery. \",\n\"Our research interest is characterizing how maternal exposures predispose women to adverse pregnancy outcomes through changes in the placental transcriptome. The placenta is a heterogenous organ with multiple tissue and cell types that may be modified by maternal exposures differently. We have collected RNA sequencing data on three placental tissues including chorion smooth, chorion frondousm, and basal plate from a de-identified cohort of pregnant women. In this project, we are interested in identifying genes differentially expressed in the placenta that mediate the effects of maternal obesity on gestational age using high-dimensional mediation analyses (HIMA in R). It is possible that differential gene expression in bulk placental tissue may reflect heterogeneity in cell composition. To complement the mediation analysis, we aim to quantify differences in cell composition by cell deconvolution (EpiDish in R) using scRNA-seq datasets of placental and decidual tissues from Pique-Regi et al. (2019), Vento-Tormo et al. (2018), Suryawanshi et al (2018), and Tsang et al. (2017). Cell deconvolution of bulk placental tissue will additionally help identify cell-specific markers for future transcriptomic analyses of maternal exposures in our pregnancy cohort. Therefore, we are requesting the dbGap dataset (phs001886.v1.p1) of villous tissue, basal plate, and chorioamnionitic membranes in term and preterm deliveries to understand placental cell-type patterns associated with maternal exposures such as obesity. \",\n\"2021, Dec14\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Adebamowo, Sally \",\n\"UNIVERSITY OF MARYLAND BALTIMORE\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease. Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data. \",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry. Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community. However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets.  \",\n\"Dec14, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Aday, Aaron \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Peripheral Artery Disease and Major Depressive Disorder\",\n\"approved\",\n\"People with depression are at higher risk of developing coronary artery disease, or cholesterol buildup in the heart arteries. Prior studies suggest that some changes in our DNA they make us more likely to develop depression are the same as some changes that make us more likely to develop coronary artery disease. Cholesterol buildup can affect other arteries, including the leg arteries. This disease is called peripheral artery disease. We do not know if the changes in our DNA that make us more likely to develop depression also make us more likely to develop peripheral artery disease. The goal of this proposed research is to answer this question.\",\n\"Overview Major depressive disorder (MDD) is phenotypically and genetically correlated with coronary artery disease (CAD), but there are limited data suggesting these correlations extend to peripheral artery disease (PAD). We propose to (1) test the association of a PAD polygenic risk score (PRS) with MDD and related biomarkers, (2) stratify such analyses by covariates important in PAD pathogenesis, including sex and diabetes, and (3) use previously published meta-GWAS summary statistics to identify genes and pathways involved in comorbid MDD and PAD.  Study Population The VUMC biobank, BioVU, will serve as the discovery population.  Genotyping and quality control  A subset of BioVU patients (n = 24,262) was genotyped as part of various institutional and investigator-initiated projects on the Illumina MEGAEX platform, which contains more than two million markers. Genotypes were imputed using SHAPEIT/IMPUTE4 with the 1000 genomes phase I reference panel, and variants with INFO < 0.3 were excluded. A subset of SNPs in linkage disequilibrium was used to calculate relatedness and principal components of ancestry using multidimensional scaling in PLINK v1.9. We restricted to a homogenous population of European descent defined by principal components of ancestry to avoid population stratification effects, leaving 18,385 individuals for analyses. Samples were genotyped in five batches, and variants were removed if allele frequencies differed significantly (P < 5 \\u00c3\\u0097 10-5) between any batch and the rest of the sample. Finally, we filtered multiallelic and structural variants, converted dosage data to hard genotype calls, and excluded variants with certainty <0.9 or INFO < 0.95, resulting in 5,218,407 high quality SNPs across the autosomes.  Phenotype data  To facilitate the investigation of comorbidity patterns across the medical phenome, we will map ICD-9 billing codes in the EHR to phecodes. ICD-9 codes will be mapped to 1814 phecode categories according to the Phecode Map v1.2. Patients will be assigned to the case group for a given phecode if they have at least two different ICD-9 codes that mapped to a given phecode, or if they had at least two separate occurrences of a single ICD-9 code that mapped to the given phecode.  PAD is focus of our analysis. Age in cases will be defined by the age of first PAD-defining feature in the EHR, while in controls, age will be defined by the age at last encounter. Data on PAD risk factors were also extracted from the EHR either from structured data or via text-mining algorithms. PAD risk factors included body mass index (BMI), hypertension, type 2 diabetes diagnosis, pre-medication blood levels of high- and low-density lipoprotein cholesterol (HDL and LDL) and of triglycerides, smoking history, and socio-economic status.  We will define MDD by the phecode 296.22 \\u00e2\\u0080\\u009cMajor depressive disorder.\\u00e2\\u0080\\u009d We will also define broad categories of mental illness, to adjust for the possibility that any degree of mental illness could increase PAD risk simply due to lifestyle factors that may be correlated with having the mental illness itself. We will define a milder depressive symptoms phenotype by one or more depression ICD-9 codes (296.2 and 296.3, including all fifth digit subclassifications, and 311), and as a final test of the robustness of associations, we will exclude patients with any psychiatric symptoms. Statistical analyses  Polygenic scores for MDD will be computed using the PRSice software package. In targeted analyses of PAD, we will select the P threshold at which the polygenic score is most strongly associated with PAD risk (i.e., at an P threshold different than 0.05). Risk ratios (odds or hazard) will reflect the risk increase per 1-SD increase in the polygenic score.  Phenome-wide association studies will be conducted with the PheWAS R package v0.12. We will require phecodes to have at least 100 cases, and we will include covariates for sex, median age across the EHR, genotyping batch, and the first 10 principal components of ancestry.  Targeted analyses of PAD in BioVU will employ logistic regression. Minimally adjusted models will include covariates for sex, age, genotype batch, and the first ten principal components of ancestry. Fully adjusted models will additionally include covariates for BMI, hypertension, smoking, type 2 diabetes, blood measurements of HDL, LDL, and triglycerides, highest level of education, a polygenic score for PAD, and depressive symptoms.  \",\n\"Mar03, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Adams, Erin \",\n\"UNIVERSITY OF CHICAGO\",\n\"Immune gene regulation at maternal-fetal interface\",\n\"closed\",\n\"The molecular abundance of a gene, or its gene expression level, can give us a snapshot of this gene's reaction to specific conditions. Specific conditions can include things like getting sick, intense exercise, or becoming pregnant. It has been especially difficult to obtain single snapshots, let alone multiple snapshots over time, of gene expression levels in the uterus during pregnancy for ethical, medical, and technological reasons. However, recent advances have provided us with exactly that. By using the data published herein, we hope to generate a data set that will allow us to look at three different gene expression level snapshots\\u00e2\\u0080\\u0093one during the first, one during the second, and one during the third trimester of pregnancy\\u00e2\\u0080\\u0093to study how genes react during a developing human pregnancy to learn more about which genes are important during healthy human development. \",\n\"We will combine these data with similar published data generated from women during the first trimester of pregnancy to generate a pregnancy transcript expression time-course data set in order to investigate the expression patterns of immune-related genes of interest within single cells. The same published methodology will be used to generate count matrices and perform additional downstream normalization for all samples. There are no plans to perform any type of genetic association analyses. There are no plans to associate expression patterns with phenotypic characteristics. The goal is to determine gene expression patterns over time during healthy human pregnancy. \",\n\"Nov10, 2020\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Aagaard, Kjersti \",\n\"BAYLOR COLLEGE OF MEDICINE\",\n\"Placental immunity at single-cell resolution\",\n\"approved\",\n\"The identification of microbes at the maternal-fetal interface in the placenta raises the question of whether they are transient, constant, or serve a function. We hypothesize these microbial challenges train the fetal immune system how to deal with beneficial or harmful pathogens. Access to this dataset will provide (1) cross-validation of our uninfected datasets and (2) allow for direct comparisons of many uninfected and infected cell types. The overarching objective of this proposed work is to determine unique transcription programs in the placenta that restrict (or permit) vertical transmission of microbes.\",\n\"The objective of this proposed work is to determine unique transcription programs in the placenta that restrict (or permit) vertical transmission of microbes. We plan to integrate the Pique-Rengi et al. (2019) and Pique-Rengi et al. (2020) scRNA-seq datasets from term and preterm chorioamniotic membranes, placental villi, and basal plate samples with our own scRNA-seq datasets. Our datasets are from patients with congenital viral infections, and we are aligning the raw data to custom human+virus transcriptomes. Thus, we need the raw data from the Pique-Rengi publications to make sure our viral transcripts are not seen in (presumably) uninfected samples. We will apply the same pre-processing (CellRanger) and downstream (Seurat in R) analyses to annotate our cell clusters and to do differential expression analyses. We anticipate finding infected and activated immune cells with distinct viral and host transcription responses. This proposed research will be consistent with the data use limitations and all analyses will be done in my lab and will not be shared in collaborations.\",\n\"2021, Sep22\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"ASSIMES, THEMISTOCLES \",\n\"STANFORD UNIVERSITY\",\n\"Mendelian Randomization Studies of Atherosclerosis Related Traits and Chronic Diseases\",\n\"approved\",\n\"Some human traits (risk factors) that predispose to heart attacks and strokes have been found to be related not only to these atherosclerosis related conditions but also a variety of other diseases or traits.  It is often not clear if these risk factors cause these conditions or are a consequence of these conditions.  In this study, we will use genetics to helo determine whether all, some, none of these relationships are causal. We will take advantage of the wide range of chronic disease outcomes available in dbGAP to examine these relationships for several risk factors of atherosclerosis. \",\n\"Some CVD risk factors have not only been linked to CVD but also to a number of other chronic complex diseases.  The objective of this study is to implement a Mendelian Randomization (MR) approach to identify causal associations between any known or purported atherosclerosis related trait (e.g. lipid, glucose, and insulin levels, age at natural menopause, etc.) and any other chronic disease that has been linked to that same trait (e.g. atherosclerosis related outcomes, other CV diseases, cancers, fractures and osteoporosis, neurological disorders, etc.).  Established genetic variants for exposures of interest will serve as MR instruments.  We will then test for association between these variants and a set of chronic diseases. For each study, the set of diseases will be defined by previous publications or our own novel hypotheses.  For example, one hypothesis could be to test for causality using MR between LDL levels and not only multiple cancers (lung, breast,endometrial), but also CHD, stroke, and Alzheimer's disease.  We may test several hypotheses in this fashion separately with each study providing us with insights on the pleiotropic link between a given atherosclerosis related exposure and a set of diseases.  Individual dbGAP datasets will ALWAYS be used in compliance with their data use certificates and their data use limitation to only examine causal (genetic) associations between an exposure of interest and the phenotype(s) listed in the data use limitation for that specific dataset.  The data requested in the application will be used only for genetic studies to learn about, prevent, or treat chronic diseases in general. Data use will be limited to research and will not be used in any commercial applications or patents.  The data requested in this proposal may be combined with non dbGAP datasets but these non dbGAP datasets will always be de-identified datasets of IRB approved research studies on chronic diseases that have been performed by other institutions or organizations which do not deposit to dbGAP (e.g. UK Biobank).   We do not believe that this creates any additional risks to participants. \",\n\"Apr03, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"ASSIMES, THEMISTOCLES \",\n\"STANFORD UNIVERSITY\",\n\"Genetic Determinants of atherosclerosis and risk factors for atherosclerosis, addition of deidentified data from other studies\",\n\"approved\",\n\"The Division of Cardiovascular Medicine at Stanford University has a long history of involvement in human genetic studies related to atherosclerosis. Atherosclerosis, or hardening of the arteries, is responsible for many serious health conditions including heart attacks, strokes, and leg amputations.  The inherited basis of all of these rather common conditions is very complex with multiple DNA mutations each appearing to have subtle effect on risk.  In this context, the challenge of identifying mutations that predispose to these conditions can only be overcome through the analysis of multiple large data-sets including those available through dbGAP.  The development of novel methods to analyze genetic data is also important.  Members plan to responsibly use the data available on dbGAP to support the evidence for association between one or more mutations and conditions related to atherosclerosis in other studies they are involved in.  This exercise will allow them to plan the next phase of population genetic studies. They will also use the data to help plan basic science experiments that will shed light on why some mutations predispose to atherosclerosis and to test novel statistical algorithms developed by Stanford faculty. Individuals using this data will publish or otherwise broadly share any findings from their studies with the scientific community.  They also will honor all embargo dates associated with any of the publicly available datasets.\",\n\"The Division of Cardiovascular Medicine at Stanford University has a long history of involvement in human genetic studies related to atherosclerosis. Faculty in the division have served as PIs or co-investigators in multiple NIH grants with a primary aim of identifying genetic determinants of cardiovascular medicine related traits including but not limited to hypertension, high cholesterol, high blood pressure, diabetes, obesity, smoking, insulin resistance, coronary atherosclerosis, peripheral arterial disease including aortic aneurysms, and cardiomyopathies. Members of the division are currently active in several related GWAS consortia. Members are also collaborating with bio-statisticians at Stanford who are interested in developing novel methods of analyzing genetic data but need individual level data in large numbers to carry out such projects effectively.  Finally, faculty members in the division also work with molecular biologists at Stanford who are trying to unravel the mechanism behind some of the most intriguing genetic associations discovered to date related to CVD .  Project members will use the genetic and phenotype data to augment evidence for association with a CVD related phenotypes derived from other human genetic studies.  They will also use the multi-ethnic data to better understand the haplotype structure in established regions of association with CVD The data will also be used to test novel statistical methodologies or applications of existing methodologies developed by Stanford faculty and to plan future genetic studies including studies that will test the predictive value of multi locus genetic risk scores of CVD.  Individuals using this data will publish or otherwise broadly share any findings from their studies with the scientific community.  They also will honor all embargo dates associated with any of the publicly available datasets.\",\n\"Jul30, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Avillach, Paul \",\n\"HARVARD MEDICAL SCHOOL\",\n\"NHLBI BioData Catalyst (BDC) \\u00e2\\u0080\\u0093 Infrastructure and Methods Development\",\n\"approved\",\n\"The NHLBI BioData Catalyst is a cloud-based platform, or technical framework, for tools, applications, and workflows. The NHLBI BioData Catalyst is being developed to provide secure workspaces for researchers to share, store, cross-link, and compute large sets of data generated from biomedical research. In this project, not only will approved investigators use controlled-access datasets from GTEx, NHLBI, and NHLBI TOPMed studies be utilized but an investigator will also be able to upload their own data into the system and will perform the following activities: Developing and testing cloud-based platforms and user interfaces to store, manage and interact with biomedical data and analysis tools; Establishing community-endorsed guidelines for Findable, Accessible, Interoperable and Reusable (FAIR) digital objects (data, software tools, etc.); Developing the ability to access controlled-access data through novel user authorization and authentication approaches; Developing analysis workflows that are portable across multiple cloud service providers; Agile software development; and Analyzing and evaluating the products and processes of the NHLBI BioData Catalyst for cost, utility, efficiency, usability, and adherence to FAIR guidelines. \",\n\"This Project is submitted by investigators involved in the NHLBI-supported BioData Catalyst  (https://biodatacatalyst.nhlbi.nih.gov/), a cloud-based infrastructure where heart, lung, blood, and sleep (HLBS) researchers can go to find, search, access, share, cross-link, and compute on large scale datasets. The NHLBI BioData Catalyst will provide tools, applications, and workflows to enable those capabilities in secure workspaces. In this project, not only controlled-access datasets from GTEx, NHLBI, and NHLBI TOPMed studies  will be utilized but an investigator will also be able to upload their own data into the system, and the following activities will be performed:  \\u00e2\\u0080\\u00a2 Developing and testing cloud-based platforms and user interfaces to store, manage and interact with biomedical data and analysis tools;  \\u00e2\\u0080\\u00a2 Establishing community-endorsed guidelines for Findable, Accessible, Interoperable and Reusable (FAIR) digital objects (data, software tools, publications, etc.);  \\u00e2\\u0080\\u00a2 Developing the ability to access controlled-access data through novel user authorization and authentication approaches;  \\u00e2\\u0080\\u00a2 Developing analysis workflows that are portable across multiple cloud service providers;  \\u00e2\\u0080\\u00a2 Agile software development; and  \\u00e2\\u0080\\u00a2 Analyzing and evaluating the products and processes of the NHLBI BioData Catalyst for cost, utility, efficiency, usability, and adherence to FAIR guidelines.   This list is not comprehensive and is subject to changes as the development of this cloud-based data ecosystem for storage and computing on biomedical digital objects evolves over the development and production phases of deployment. If there are significant modifications or changes to the scope of the project, this Research Use Statement must be amended and resubmitted for NHLBI Data Access Committee (DAC) review.   The investigators understand that any of the requested controlled-access datasets have Data Use Limitations (DULs) based on the terms of the informed consent of the study participants and/or stipulated by the submitting institution.  All approved investigators will abide by the DULs of each selected dataset, the Data Use Certification Agreement, NIH Genomic Data Sharing Policy (https://grants.nih.gov/grants/guide/notice-files/NOT-OD-14-124.html), Genomic Data User Code of Conduct (https://osp.od.nih.gov/wp-content/uploads/Genomic_Data_User_Code_of_Conduct.pdf), the NIH Security Best Practices for Controlled-Access Data Subject to the NIH Genomic Data Sharing (GDS) Policy (https://osp.od.nih.gov/wp-content/uploads/NIH_Best_Practices_for_Controlled-Access_Data_Subject_to_the_NIH_GDS_Policy.pdf), and the NIH Notice for Use of Cloud Computing Services for Storage and Analysis of Controlled-Access Data Subject to the NIH Genomic Data Sharing Policy (https://grants.nih.gov/grants/guide/notice-files/NOT-OD-15-086.html).   Under no circumstances will investigators listed in this Project seek to perform hypothesis-driven biomedical research, or develop methods to identify statistical patterns and assist in their own interpretation of the requested datasets or of any derived data for research purposes. Should the need arise to use the requested data for scientific, use-case driven biomedical research purposes, investigators will submit a separate dbGaP project application. At no time shall the Institution, Investigator, or Approved Users conduct Data Transfer or Data Access of the data and any derived data out of the approved cloud environments; nor shall the Institution, PI, or Approved Users attempt to recreate or replicate the data and any derived data for use outside of the approved cloud environments; this includes, but is not limited to, capturing screenshots or videos of data or derived data, or locally transferring software results or runs that may enable recapitulation of the data or derived data. At no time shall the Institution, PI, or Approved Users use the data or derived data for any other purpose outside of or beyond the scope of the work they are doing for the NHLBI BioData Catalyst. No ownership rights of the data (including derived or derivative data) are granted to the Institution, Investigator, Approved Users, or the cloud service providers in the development of the NHLBI BioData Catalyst. \",\n\"2021, Mar25\",\n\"phs002299.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"BACANU, SILVIU-ALIN \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"Opiate use disorders\",\n\"approved\",\n\"We plan to use the result from present study to i) run secondary analyses, e.g. to uncover genes, proteins associated with OUD and ii) replicate variant/gene/protein/pathways results found by other means.\",\n\"We plan to use the summary statistics from opiate use disorder (OUD) study to i) run secondary analyses, e.g. to uncover genes, proteins associated with OUD and ii) replicate variant/gene/protein/pathways results found by other means. The preliminary plan of analyses is, for now, 2-step. First, we want to run new TWAS/PWAS gene/protein and pathway analyses using only summary statistics from this opiate use disorder (OUD) study. Second, replicate the findings obtained by imputing OUD phenotypically in UK Biobank.\",\n\"Jul13, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Chen, Weijie \",\n\"U.S. FOOD AND DRUG ADMINISTRATION\",\n\"Investigation of image-based biomarkers for radiogenomics of COVID-19\",\n\"approved\",\n\"Effective COVID-19 diagnosis and treatment rely on the information from multiple patient tests involving clinical, molecular, imaging, and genomic data. Radiomics, an expansion of computer-aided diagnosis, is a growing effort that involves these computerized image analyses in attempt to further relate quantitative image data to other \\u00e2\\u0080\\u009c-omic\\u00e2\\u0080\\u009d data such as clinical, pathologic, and genomic data. The integration of imaging data with genomic data allows for the study of associations between the imaging findings and the genomic measurements of the same disease site. Using the MIDC datasets of COVID-19, we will investigate relationships between computer-extracted quantitative radiomic COVID-19 features and various clinical, molecular, and genomics markers of prognosis and risk of recurrence, including gene expression profiles. We will assess the predictive ability of the quantitative COVID-19 CT/CXR features relative to multiple tasks: clinical stage, risk of recurrence, and genomics.\",\n\"Effective COVID-19 diagnosis and treatment rely on the information from multiple patient tests involving clinical, molecular, imaging, and genomic data. Radiomics, an expansion of computer-aided diagnosis, is a growing effort that involves these computerized image analyses in attempt to further relate quantitative image data to other \\u00e2\\u0080\\u009c-omic\\u00e2\\u0080\\u009d data such as clinical, pathologic, and genomic data [Zhu 2015]. The integration of imaging data (radiomics) with genomic data, referred to as \\u00e2\\u0080\\u009cimaging-genomics\\u00e2\\u0080\\u009d or \\u00e2\\u0080\\u009cradiogenomics\\u00e2\\u0080\\u009d, allows for the study of associations between the radiomic phenotypes and the genomic measurements of the same disease site. Radiomic phenotypes that are highly correlated with important clinical, molecular, or genomic biomarkers can potentially serve as diagnostic or prognostic tools for patient monitoring and assessing therapeutic response, and thus augment the utility of medical imaging as a non-invasive technology for COVID-19 care, like a \\u00e2\\u0080\\u009cvirtual digital biopsy\\u00e2\\u0080\\u009d. Using the MIDC datasets of COVID-19, we will investigate relationships between computer-extracted quantitative radiomic COVID-19 features and various clinical, molecular, and genomics markers of prognosis and risk of recurrence, including gene expression profiles. We will assess the predictive ability of the quantitative COVID-19 CT/CXR features relative to multiple tasks: clinical stage, risk of recurrence, and genomics. Aims: 1. To create a multi-omics dataset of cases, including radiomics of COVID-19 and non-COVID-19 patients from chest x-rays and/or CTs along with their molecular and genomic markers, which will be made openly available for discovery by computational biologists and machine learning experts. MIDRC has the chest x-ray image data corresponding to the EHR data in RED CORAL: PETAL Repository of Electronic Data COVID-19 Observational Study. Our first goal is to test the interoperability of MIDRC and the Biodata CATALYST. We will link the imaging data with the EHR data and investigate methods for use of data from both sources for machine learning development and assessment. 2. To determine if these quantitative radiomics (virtual biopsy) of COVID-19 CT/CXR can serve as a surrogate biomarker for assessing the molecular environment, enabling translation of virtual biopies. Push/Pull Relationships: The multi-institutional chest radiograph and CT datasets of the MIDC will enable the rapid investigation of radiogenomics for prognosis, prediction, and monitoring of response of the COVID patient. This project will deliver discovery findings for COVID-19 lungs to expedite the understanding and development of machine intelligence as well as a clinical tool. \",\n\"Sep07, 2022\",\n\"phs002363.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Bakulski, Kelly \",\n\"UNIVERSITY OF MICHIGAN AT ANN ARBOR\",\n\"Bulk villous tissue deconvolution in the term placenta: parallel single-cell and sorted cell type-specific gene expression\",\n\"approved\",\n\"The public health burden of negative pregnancy outcomes is substantial. The functional tissue of the placenta has been studied to better understand how the placenta works and leads to negative pregnancy outcomes. Tissue-level measures of gene activity represent a mixture of measurements from individual cell types that make up the tissue. Each cell type has a unique gene activity pattern. Failure to account for each cell type\\u00e2\\u0080\\u0099s pattern can cloud study results and limit our understanding of the placenta. Critically, we have a limited understanding of cell type gene activity patterns in the placenta. The overall goal of our study is to measure these patterns to strengthen studies of the placenta. We would like to compare our results and findings to a previous study. In addition, we would like to combine our datasets and reanalyze our results and findings in a combined dataset. \",\n\"The public health burden of adverse pregnancy outcomes is substantial. Villous tissue, the parenchyma of the placenta, has been characterized at term through gene expression to better understand placental function. However, tissue-level expression measurements represent a convolution of signals from individual cell types that compose the tissue. Cell types exhibit cell type-specific gene expression patterns. Failure to account for cell type heterogeneity could bias study results and limits interpretability. Critically, placental cell type-specific RNA references are limited or unavailable to deconvolve bulk placenta villous tissue measures. This research integrates RNA sequencing of seven flow-sorted placental cell types and single-cell RNA sequencing of term placental villous tissue to robustly generate cell type-specific gene expression profiles. Cell type-specific gene expression references will be publicly disseminated for bulk tissue deconvolution to advance molecular epidemiology studies of the placenta.  We request access to the phs001886.v1.p1 single-cell RNA sequencing dataset to compare our results to previously published findings and integrate the phs001886.v1.p1 dataset with primary single-cell RNA sequencing data to improve power in identifying cell type-specific gene expression, if appropriate and accounting for batch effects. The study by Pique-Regi et al. examined the transcriptional landscape of major placental compartments among preterm births, non-laboring term birth, and laboring term birth. Access to their findings would provide an opportunity to test for biological replication and integrate additional data to identify cell type-specific gene expression patterns. \",\n\"2020, Jul27\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Chen, Chia-Yen \",\n\"BIOGEN IDEC U S CORPORATION\",\n\"Drug target identification using cross-trait meta-analysis\",\n\"expired\",\n\"We seek to combine the results of large-scale GWAS from MVP cohort and additional publicly available GWAS summary statistics through meta-analyses and downstream bioinformatic analyses to identify potential new targets. To do so, we will perform meta-analysis using different statistical methods and evaluate results for diseases and traits across different categories. We expect these meta-analysis results will inform us on novel disease loci with different effects across diseases, which will potentially lead to new drug targets. \",\n\"Large-scale genome-wide association studies (GWAS) have provided unprecedented opportunities for new drug target identification through unbiased, hypothesis-free genome-wide search. We plan to leverage the summary statistics of large-scale GWAS from MVP cohort and additional publicly available GWAS summary statistics, of the same traits and other genetically correlated traits, to identify potential new targets. We will evaluate the results from 3 different classes of meta-analysis methodologies, including 1) conventional inverse-variance weighted meta-analysis, designed for GWAS of the same traits; 2) multi-trait analysis of GWAS (MTAG; PMID:29292387), designed for combining GWAS of genetically correlated traits; 3) Association analysis based on subsets (ASSET; PMID:22560090) for dissecting concordant/discordant genetic effects across different traits. We will test these methods on diseases and traits across different disease categories and genetic architectures to evaluate the robustness of the methods when applied on real data. For the validated genome-wide meta-analyses, we will perform downstream bioinformatic analyses to gain insight into the molecular basis of the novel loci, including transcriptome-wide association analysis (TWAS), colocalization, Mendelian randomization, gene-based and pathway-based analysis, and drug database annotation. We will also evaluate if these meta-analyses can improve polygenic score prediction. We expect these results will inform us on novel disease loci and shared loci between diseases/traits with concordant/discordant effects, which will potentially lead to new drug target identification.\",\n\"2020, Jun10\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Chasman, Daniel \",\n\"BRIGHAM AND WOMEN'S HOSPITAL\",\n\"Causality and/or shared physiology linking plasma lipids to migraine\",\n\"closed\",\n\"Many studies have observed correlations of elevated levels of circulating LDL-cholesterol (LDL-C) and triglycerides with migraine. However, whether these lipids are causal for migraine is not known. The proposed research would use genetic methods to understand whether the correlation is due to causality.  The outcome of the research is expected to be improved understanding of the biological mechanisms underlying migraine. If lipids are determined to contribute to the cause of migraine, the results would suggest that existing drugs that lower LDL-C and triglycerides may be part of an effective therapeutic strategy for migraine. \",\n\"The comorbidity of migraine and incident cardiovascular disease (CVD) has motivated a search to understand potential shared pathophysiologic mechanisms.  Some of these explorations have focused on one of the most important classes of CVD risk factors, plasma lipid levels for which associations between migraine with lipids have been consistent.  These associations implicate elevated LDL cholesterol (LDLC) and triglycerides but not HDL cholesterol in migraine. Within this context, a recent randomized trial among migraineurs found that the combination of simvastatin, an LDLC lowering agent, and vitamin D significantly reduced migraine frequency compared to placebo. These results raise two non-exclusive possibilities for potential links between the plasma lipids and migraine: there may be a causal (or reverse causal) relationships and/or there may be shared underlying biology. We will implement recent analytic approaches leveraging genetics to evaluate the extent to which each of these two scenarios may be relevant.  The first analytic approach termed \\u00e2\\u0080\\u009cMendelian Randomization\\u00e2\\u0080\\u009d, exploits the randomized and irreversible meiotic assortment of alleles influencing lipid levels to assess whether lifelong exposure to a genetically determined risk factor may be associated with migraine, thereby implying causality.  Using individual level information for migraine, lipid levels, and genome-wide genotype data from the Women\\u00e2\\u0080\\u0099s Genome Health Study (5,122 migraineurs/23,294 total), the UK Biobank (14,392 migraineurs/~500,000 total), we will perform MR, including tests of reverse causality, to compare observational effects with genetically inferred causal effects, which would be further characterized by mediation analysis.  These analyses will be complemented by MR analysis using summary statistics from extremely well-powered, genome-wide association studies of migraine (>56,000 migraineurs, Gormley et al. Nat. Gen. 2016v48p856) and lipid levels (~350,000 individuals Klarin et al. Nat. Gen. 2018v50p1514).  In the second analytic approach, we will estimate the extent of shared genetic influence, or equivalently \\u00e2\\u0080\\u009cshared heritability\\u00e2\\u0080\\u009d or \\u00e2\\u0080\\u009cgenetic covariance\\u00e2\\u0080\\u009d, between each lipid risk factor and migraine in these same data sets while also identifying specific genetic variants with shared associations.  Moreover, we will use computational approaches to infer which tissue types and/or biological pathways may be most relevant to the shared genetic effects. Due to the robustness of the analytic approaches and the statistical power of available data, the proposed research is expected to reach definitive conclusions about relationships between migraine and lipids, and is likely to advance understanding of migraine pathophysiology if not also to suggest novel therapeutic strategies for migraine.\",\n\"2019, May17\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Cebola, Ines \",\n\"IMPERIAL COLLEGE OF SCIENCE, TECHNOLOGY AND MEDICINE\",\n\"The contribution of human cell types to NAFLD\",\n\"approved\",\n\"Non-alcoholic fatty liver disease (NAFLD) is a progressive disease estimated to affect up to a third of the World's population that arises from a combination of genetic and life style factors. Despite being a significant contributor to higher mortality and a leading cause of liver transplantation and liver cancer in its more advanced cases, many of the molecular mechanisms underlying NAFLD are still not known. This lack of knowledge is explained, at least in part, by the high cellular heterogeneity of the liver, which is comprised of many different cell types but where about 80% of the cell mass is comprised of a single cell type - hepatocytes. As a result of this, many of the current and previous studies were only able to investigate molecular mechanisms of NAFLD in hepatocytes. This gap in knowledge is particularly acute in the investigation of genetic mechanisms.   In this project, we are using isolated liver cells to develop maps of active genes and other regions of the genome that act as 'DNA switches' to turn genes on. We envision that by looking at individual and less explored types of cells from the human liver in combination with DNA variants that change our risk of developing NAFLD, we will understand better why some people are particularly susceptible to develop NAFLD.\",\n\"Our study seeks to address distinct research hypotheses that were not the focus of the primary studies. Specifically, we seek to investigate what is the contribution of different human cell populations to nonalcoholic fatty liver disease (NAFLD) genetic risk, including the 'NAFLD proxy traits'. To this end, we request access to the full summary-level association data from the multiancestry, EA, AA, HISP and ASN analyses for unexplained chronic ALT elevation as published by Marijana et al. (https://doi.org/10.1038/s41588-022-01078-z), which is to date the largest GWAS dataset for a NAFLD proxy trait. The association data will be used to test the enrichment of associated variants at specific categories of cis-regulatory elements, which are currently being mapped by my team using different experimental models and sets of primary samples. These datasets include ATAC-seq and ChIP-seq/CUT&RUN/CUT&Tag from primary hepatocytes, liver sinusoidal endothelial cells, and human pancreatic islets, and single-nuclei ATAC-seq/CUT&Tag from human liver samples. We will use publicly available ENCODE datasets for other tissues and cell types as controls. With these analyses, we will identify genomic features and cell types relevant to NAFLD and NAFLD-related traits.   Enrichment of variants at specific cis-regulatory element categories will be tested using fgwas (https://github.com/joepickrell/fgwas), GREGOR (https://genome.sph.umich.edu/wiki/GREGOR), GARFIELD (https://www.ebi.ac.uk/birney-srv/GARFIELD/), and LD-score regression (https://github.com/bulik/ldsc).  The data will be tested separately from other genetic association datasets and all data analysis will be carried out in our secure high-throughput computing facility (https://www.imperial.ac.uk/admin-services/ict/self-service/research-support/rcs/), we will not use cloud computing for this project.\",\n\"Mar26, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"CULE, MADELEINE \",\n\"CALICO LIFE SCIENCES, LLC\",\n\"Genetic risk factors of thoracic and abdominal aneurysm\",\n\"approved\",\n\"We are studying the genetic basis of risk factors for cardiovascular disease, and have conducted studies which identified risk factors for aneurysm in a large British cohort. We wish to compare our findings in a large British cohort with similar findings in the Million Veteran\\u00e2\\u0080\\u0099s cohort\",\n\"We are investigating the etiology of cardiovascular disease, including thoracic and abdominal aortic aneurysm, using the features derived from MRI scans in the UK Biobank resource. We have conducted genetic studies of these risk factors and plan to investigate the genetic correlation between these traits and aneurysm outcomes in the MVP (as described in Klarin et al, Circulation, https://doi.org/10.1161/CIRCULATIONAHA.120.047544) . We also plan to investigate whether the genetic signals found in the two traits colocalize to implicate the same genes/variants. No attempt will be made to identify individual participants.\",\n\"2022, Jul06\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"CHANG, SUHUA \",\n\"PEKING UNIVERSITY\",\n\"Association between sleep traits and the risk of posttraumatic stress disorder: a mendelian randomization study\",\n\"closed\",\n\"Increasing evidence shows that pretrauma factors, such as sleep traits, are associated with increased risk for PTSD following traumatic exposure, pointing to a potential modifiable target for prevention. However, the causality and direction of these associations remains unclear. The 2-sample MR was performed to examine whether sleep traits have a causal effect on risk of PTSD, and vice versa. As genotypes are randomly allocated at conception, they are largely free from confoundings that could bias the typical observational findings. In addition, the MR method reduces the potential bias that are common in conventional observational studies such as confounding, reverse causation and measurement error, provided certain assumptions are met. Summary statistics for genetic variants were identified from the largest available, nonoverlapping GWAS of sleep traits (n = up to 1331010 individuals) and PTSD reexperiencing symptoms from the genome-wide association meta-analysis conducted in the US Million Veteran Program from predominantly European descent.\",\n\"Research objectives and study design: Increasing evidence shows that pretrauma factors, such as sleep traits, are associated with increased risk for posttraumatic stress disorder (PTSD) following traumatic exposure, pointing to a potential modifiable target for prevention. However, the causality and direction of these associations remains unclear. The global lifetime prevalence of PTSD is approximately 1.3 to 12.2%, with a 12-month prevalence ranging from 0.2 to 3.8%. Evidence from epidemiologic studies, which showed that PTSD was not an inevitable consequence of trauma, also suggested that many variables previously considered to be the outcomes of trauma are likely to be pretrauma risk factors associated with PTSD. Although much of the literature on PTSD risk has focused on the effects of cognitive ability and educational attainment (years of schooling), less studies investigate the potential adverse effects of sleep disturbance and traits such as sleep duration, chronotype (morning or evening preference) and insomnia. The 2-sample mendelian randomization (MR) were performed to examine whether sleep traits have a causal effect on risk of PTSD, and vice versa. MR analyses use genetic variants robustly associated with potentially modifiable risk factors (e.g., sleep traits) as proxies to explore causal effects on diseases (e.g., PTSD). As genotypes are randomly allocated at conception, they are largely free from confoundings that could bias the typical observational findings. This, as well as the invariant nature of genetic variants, provides an analogy to randomized clinical trail, in which exposure is allocated randomly and is not affected by subsequent disease. The MR method reduces the potential bias that are common in conventional observational studies such as confounding, reverse causation and measurement error, provided certain assumptions are met. We retrieved summary statistics for the association between genetic instruments and self-reported sleep traits from the respective genome-wide association studies (GWAS) conducted among individuals of European ancestry. We tend to retrieved summary statistics for PTSD re-experiencing symptoms from the genome-wide association meta-analysis conducted in the US Million Veteran Program (146,660 European Americans and 19,983 African Americans). Analysis plan: Two sample MR analyses were conducted using the R package TwoSampleMR, which harmonizes exposure and outcome data sources containing information on selected instrument SNPs (effect alleles, effect sizes (\\u00c3\\u009f statistics), standard errors, P values and effect alleles frequencies). For each direction of potential effect, we used an random-effects inverse variance weighted (IVW) meta-analysis as main analysis method to combined the Wald estimates for each SNPs (SNP outcome estimates divided by SNP exposure estimate where the intercept is constrained to zero). IVW method can provided an unbiased estimate in the absence of horizontal pleiotropy or when horizontal pleiotropy is uncorrelated with SNP-confounder associations and is balanced. To account for horizontal pleiotropy, we performed complementary analyses using the weighted median and MR-egger regression methods which are relatively robust under different assumptions about pleiotropy, although at the cost of statistical power. To further detect and correct the obtained causal estimates, we performed MR-Pleiotropy Residual Sum and Outlier (MR-PRESSO) test to identify and remove any outliers reflecting likely pleiotropic biases. We also conducted tests for significant results using meta-analytic methods to detect heterogeneous outcomes, including leave-one-out analyses, Cochran\\u00e2\\u0080\\u0099s Q statistics and the MR Egger intercept test of deviation from null. To assess the robustness of causal estimates obtained using SNPs associated with self-reported sleep traits, we conducted MR analyses using genetic variants associated with objective accelerometer-based sleep traits from GWAS in UK biobank (up to 85830), including 11 independent SNPs associated with nocturnal sleep duration (to be compared with self-reported sleep duration), 6 SNPs associated with timing of the least active 5 h (L5 timing; to be compared with self-reported chronotype of morning preference) and 21 SNPs associated with number of nocturnal sleep episodes (to be compared with self-reported insomnia symptoms). All analyses were performed using R version 3.4.3 (The R Foundation for Statistical Computing). \",\n\"2019, Dec30\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Brumpton, Ben \",\n\"NORWEGIAN UNIV OF SCIENCE &amp; TECHNOLOGY\",\n\"Investigating the causal effects of childhood and adulthood adiposity on later life mental and sleep health outcomes: a mendelian randomisation study\",\n\"approved\",\n\"We want to see the effect of childhood and adulthood adiposity on later life depression, anxiety and insomnia. As anxiety is one of our outcome we will use anxiety data to see whether childhood adiposity has its direct effect on later life anxiety or its effect goes through adulthood adiposity. For our research purpose we need genetic data of anxiety as it is not freely available.\",\n\"Anxiety is one of our outcome. We will use anxiety summary statistics in our two sample mendelian randomisation study. We are not going to combine the data with any other data set. Objective: To investigate the causal effects of childhood adiposity on later life depression, anxiety and insomnia independent of adulthood adiposity. Method: Two-sampled univariable and multivariable Mendelian randomization (MR). Analysis plan: Investigate the total effect of childhood body size and direct effect after accounting for adulthood body size on depression, anxiety and insomnia. We are using summary statistics of childhood and adulthood adiposity, depression, insomnia from different large GWASH consortium. As anxiety data is not freely available so, we want to request for summary statistic of anxiety.\",\n\"2022, Dec05\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Bountress, Kaitlin \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"Investigating Shared Risk Factors for Comorbidity \",\n\"approved\",\n\"Traumatic events are common, and increase risk for adverse outcomes. Increased alcohol use/consumption (AC), alcohol use disorder (AUD), recreational cannabis use (RCU), cannabis use disorder (CUD), opioid use disorder (OUD), cigarettes per day (Cpd) current and past, smoking initiation (SmkInit), trajectories, and cessation (SmkQuit), and posttraumatic stress disorder (PTSD) may occur following trauma, and may co-occur. Additionally, there is evidence suggesting that shared genetic factors may partially account for this co-occurrence. This project seeks to use summary statistics for PTSD,  and AUD, RCU, CUD, OUD, Cpd current and past,  SmkInit, smoking trajectories, and SmkQuit as well as PTSD and RCU, from the Million Veteran Program (MVP) in order to understand the potential shared molecular genetic underpinnings of the PTSD-AC/AUD and PTSD-RCU comorbidity. This research will help to move forward the literature on potential shared risk factors for this debilitating comorbidity.\",\n\"The focus of this project is on understanding the etiology of comorbid conditions, specifically Posttraumatic Stress Disorder (PTSD), alcohol and substance  phenotypes and positive phenotypes (e.g., positive subjective well-being, life satisfaction, openness to experience).   Trauma is common, and associated with a host of deleterious outcomes. Posttraumatic Stress Disorder (PTSD) is one of the most common disorders resulting from trauma, tending to co-occur with alcohol consumption (AC), alcohol use disorder (AUD), and recreational cannabis use (RCU), cannabis use disorder (CUD), opioid use disorder (OUD) and cigarette consumption (e.g., cigarettes per day). Much of the research on the comorbidity of PTSD and alcohol phenotypes has focused on PTSD-AUD. As increased consumption of alcohol is associated with AUD, research is needed to determine whether the same etiologic processes underlying PTSD-AUD comorbidity are those underlying PTSD and AC. Similarly, there is limited research focused on the etiology of PTSD and RCU, CUD and cigarette smoking. .  Thus, the proposed study will use large-scale de-identified data and employ cutting-edge methods to broadly investigate the genetic architecture of PTSD, AUD-related phenotypes, AC, RCU, CUD, OUD, current and past cigarettes per day (Cpd), smoking (SmkInit), trajectories, and cessation (SmkQuit),  and their comorbidity. Specifically, this project proposes to use GWAS summary statistics for PTSD, AUD, maximum alcohol intake, AUDIT-C (consumption), and lifetime RCU, lifetime OUD, current and past cigarettes per day (Cpd), smoking initiation (SmkInit), trajectories, and cessation (SmkQuit)  from a number of large consortia datasets, including Million Veteran Program (MVP) in order to better understand: 1) the genetic correlations between PTSD-AUD, PTSD-AC, and PTSD-RCU, PTSD-CUD, PTSD-OUD, PTSD-Cpd current and past, PTSD-SmkInit, PTSD-smoking trajectories, and PTSD-SmkQuit, 2) whether aggregate genetic risk for AUD/AC predict PTSD, whether aggregate genetic risk for PTSD predicts AUD/AC, whether aggregate genetic risk for RCU predicts PTSD, and whether aggregate genetic risk for PTSD predicts RCU, whether aggregate genetic risk for PTSD predicts CUD, whether aggregate genetic risk for PTSD predicts OUD, whether aggregate genetic risk for PTSD predicts past and current Cpd, whether aggregate genetic risk for PTSD predicts SmkInit, whether aggregate genetic risk for PTSD predicts smoking trajectories, and whether aggregate genetic risk for PTSD predicts SmkQuit, 3) if PTSD and AUD/AC are related to one another in a causal manner,  and if PTSD and RCU are related to one another in a causal manner, if PTSD and CUD are related to one another in a causal manner, if PTSD and OUD are related to one another in a causal manner, if PTSD and current and past Cpd  are related to one another in a causal manner, if PTSD and SmkInit are related to one another in a causal manner, if PTSD and smoking trajectories are related to one another in a causal manner, if PTSD and SmkQuit  are related to one another in a causal manner and 4) the genetic factor structure of PTSD, alcohol, and cannabis, opioid, and cigarette smoking phenotypes. In order to accomplish these goals, we will employ the use of linkage disequilibrium score regression (LDSR; 1), polygenic risk scores PRS (2), mendelian randomization (MR; 3), and genomic structural equation modeling (gSEM; 4). A combination of Unix and R/RStudio will be utilized in order to conduct the proposed analyses. Only summary statistics will be utilized. Analyses will be split by ancestry, specifically for European Ancestry and African Ancestry separately, as well as split by sex. The proposed study represents an important step forward in clarifying risk factors for this co-occurrence.  These analyses will be conducted by researchers at Virginia Commonwealth University. Dr. Kaitlin Bountress is the PI for the proposed study. Dr. Bountress, with collaboration from colleagues (including Drs. Ananda Amstadter, Todd Webb, Kenneth Kendler, Silviu Bacanu, Nathan Gillespie) and graduate students will complete these analyses.   **Update to application: we will be extending these analyses to include other substance outcomes; we are requesting summary stats for: recreational cannabis use (RCU), problematic alcohol use (PAU), opioid use disorder(OUD), cannabis use disorder (CUD), cigarettes per day (CPD; current and past), SmkTrajectory, SmkInit, SmkQuit.\",\n\"2020, Aug11\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Blelloch, Robert \",\n\"UNIVERSITY OF CALIFORNIA, SAN FRANCISCO\",\n\"Trophoblast gene expression programs in the chorionic villi and smooth chorion\",\n\"approved\",\n\"The human placenta contains two specialized regions: the villous chorion and the smooth chorion. In this study, we use single cell RNA sequencing to compare the cell types and molecular programs between these two regions in the second trimester human placenta. Access to these data will allow comparison 1) between the cell types we identify and those identified in these studies 2) comparison between cell types identified in the second trimester and term. Analyses of these data will improve analysis and definition of trophoblast subtypes and expand analysis into changes related to gestational age.\",\n\"We are using single cell RNA sequencing to compare the cell types and molecular programs between trophoblasts isolated from the chorionic villi and the smooth chorion regions of the second trimester human placenta. We will integrate our scRNA-seq data from these regions with the data in Pique-Regi et al., 2019 and 2020 from the healthy chorioamniotic membranes, placental villi, and basal plate samples collected at term. Our goal is to compare the cell types identified in our analysis of the second trimester to those found in these publications. To do so, we require access to the raw data from these studies so that they may be processed using matching methodologies to our data, which will minimize batch effects and improve integration of all datasets. Our proposed research plans are consistent with data use limitations and not be shared in other collaborations. No genetic variant association analyses will be performed. \",\n\"Jun27, 2022\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Bick, Alexander \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"NHLBI TOPMed Whole-genome Sequencing Program (TOPMed WGS)\",\n\"approved\",\n\"The goals of the TOPMed Whole Genome Sequencing program are to identify genetic risk (and protective) factors for disease, to identify subtypes of disease, and to develop targeted and personalized treatments for disease.  The program consists of multiple studies addressing these goals for phenotypes focused on heart, lung, blood and sleep disorders, which have a high impact on public health.  These studies will include individuals from diverse populations, including those historically under-represented in health research.\",\n\"Phenotypic and genomic data from TOPMed studies will be used for: (a) investigating genotype-phenotype associations; (b) variant summary statistics, (c) common controls in association tests, (d) reference panels to impute into un-sequenced individuals from TOPMed; and (e) population genetic studies.  TOPMed data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Data from an individual with disease-specific consent will not be used in analyses outside of that restriction, including indirect uses such as imputation reference panels, common controls for association studies and variant summary statistics. For example, an individual with consent restricted to lung function studies will not be used to impute data intended for analyses of diseases not clearly related to lung function.  When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish or otherwise broadly share any findings from this study with the scientific community.  This application anticipates collaboration among TOPMed investigators from multiple institutions, who are named in a master list appended to the IRB document uploaded with this application. Within a set of applicants approved for access to the TOPMed Exchange Areas, data may be shared in a cloud environment or by direct transfer, provided that they all agree among themselves to allow such sharing, and that all have approved TOPMed Data Access Requests for the specific data sets to be shared. Otherwise, data will be shared only among Internal Collaborators \\u00e2\\u0080\\u0093 i.e. within an institution.  Data from TOPMed may be combined with non-TOPMed data from the same or other studies (obtained from dbGaP or other sources), to improve the power for novel genetic discoveries, while respecting the consent of all participants. We expect that this activity creates no additional risks to participants.\",\n\"2021, Mar10\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Bhatt, Ishan \",\n\"NORTHERN ARIZONA UNIVERSITY\",\n\"Investigating the genetic basis of tinnitus and hearing disorders\",\n\"closed\",\n\"Tinnitus, the phantom perception of sound in the absence of an external sound source, is a prevalent hearing disorder. The primary goal of this study is to identify detailed phenotypic and genotypic profiles of tinnitus and related hearing health concerns (e.g., noise-induced hearing loss). This research will be useful to construct a genetic risk profile that can be used by health-care providers and educators to identify individuals genetically at risk for tinnitus. \",\n\"Title: Investigating the genetic basis of tinnitus and hearing disorders Objectives of the proposed project: The primary objective of the proposed project is to identify the genetic factors underlying tinnitus, commonly known as ringing in ears/head and tinnitus-related distress. This project will also investigate the genetic factors underlying other hearing disorders that are common in individuals exposed to occupational noise (e.g., noise-induced hearing loss). Study design: Over the past decade, genome-wide association studies (GWAS) have identified numerous genetic variants associated with common diseases and traits [1]. The small fraction of disease heritability explained, thus far, by GWAS findings leads to prediction models that generally fail to accurately and comprehensively predict which individuals will develop the disease in the future [2]. However, it is projected that comprehensive polygenic risk prediction models will provide clinically useful predictions of disease risk if the limitations of current GWAS can be overcome [3]. The present study will evaluate the genotype-phenotype relationship for tinnitus, tinnitus-related distress, and hearing disorders that are common in individuals exposed to occupational noise (e.g., noise-induced hearing loss). We will employ machine learning strategies to identify the genetic association with hearing disorders. The primary dataset we will evaluate this approach is on the Veterans Administration (VA) Million Veteran Program (phs001672.v3.p1). Analysis plan: We will extract SNP genotypes from complete genomes by using Plink, a whole-genome association analysis toolset designed to perform a range of large-scale analyses. We will extract the phenotype variables related to tinnitus, tinnitus-related distress, and other hearing measures (e.g., hearing sensitivity). We will identify individuals with and without tinnitus. We will start by performing odds ratios, regression, and principal component analyses to detect an initial set of SNPs associated with tinnitus groups. Using the SNPs as predictor variables and tinnitus groups as a response, SNPs exhibiting a p-value<0.05 for the ORs, principal component correlations, or regression t-tests will be regarded as significantly associated to hearing loss. P-values between 0.05 and 0.07 will be regarded as marginally significant. After identifying the possible associated SNPs, with either significant or marginally significant associations, we will build mixed models for additional regression analyses in order to identify possible confounding factors that may amplify or cancel the effect of a given SNP over the prediction power of the statistical models. Finally, we will identify possible cooperation effects among SNPs by combining them as interaction terms in the mixed models. We will apply machine learning techniques such as random multi-model inference, parameter exchange, and model selection in order to test all possible combinations and interactions among all SNPs analyzed in a feasible computational time. For comparing the quality and accuracy of the statistical models, we will use Aikaike Information Criterion (AIC), Bayesian information criterion (BIC), Area Under the Curve (AUC), ANOVA, and likelihood-ratio tests. We will employ other appropriate methods to identify a confluence of genetic and other health-related factors for tinnitus and related hearing-health concerns (e.g., tinnitus-related distress and noise-induced hearing loss). SNPs with the p-value <5\\u00c3\\u009710-8 in both samples will be regarded as a statistically significant association with the phenotypes. SNPs with the p-value <0.05 and >5\\u00c3\\u009710-8 in both samples will be regarded as a potential association with the phenotypes.  References: 1. Welter, D., et al., The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Res, 2014. 42(Database issue): p. D1001-6.  2. Wray, N.R., et al., Pitfalls of predicting complex traits from SNPs. Nat Rev Genet, 2013. 14(7): p. 507- 15.  3. Chatterjee, N., et al., Projecting the performance of risk prediction based on polygenic analyses of genome-wide association studies. Nat Genet, 2013. 45(4): p. 400-5, 405e1-3. \",\n\"Jun29, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Beulens, Joline \",\n\"VRIJE UNIVERSITEIT\",\n\"The causal association of lifestyle factors associated with peripheral artery disease: a Mendelian Randomization approach\",\n\"closed\",\n\"Peripheral artery disease (PAD) is one of the most common types of cardiovascular disease. It is a multifactorial disease, meaning it is caused by a combination of genetic and lifestyle factors. Even though many factors are known to play a role in the onset of PAD, classic epidemiological study designs lack the ability of determining causal relationships between risk factors and PAD since they are limited by bias. Therefore, we aim to evaluate the causal relationship between lifestyle related risk factors and the development of PAD via a Mendelian Randomization (MR) approach. Summary statistics of the lifestyle related risk factors type 2 diabetes, lipid metabolism, obesity, smoking, alcohol use, coffee consumption, physical activity, sleep and educational level will be used as instruments, information will be extracted from publically available GWASs. Summary statistics of GWAS of PAD from the MVP program will be used in the MR study as a outcome variable. By determining the causal relationships between lifestyle factors and PAD, a solid basis for targeted disease prevention strategies can be created.\",\n\"Peripheral artery disease (PAD) is one of the most common types of cardiovascular disease, and is estimated to affect over 200 million people worldwide with a prevalence rising sharply with age. The epidemiology of PAD has been studied comprehensively, and a plethora of genetic and lifestyle factors are linked to the development of PAD. However, since PAD can long exist in an asymptomatic state and the severity of symptoms can vary significantly between individuals, it is difficult to accurately define a precise moment of disease onset. For that reason, many studies investigating PAD risk factors are based on cross-sectional associations. Although such studies are potentially informative, bias such as confounding and reverse causation are introduced, which prevents researchers from drawing causal conclusions. To overcome this limitation, we aim to investigate type 2 diabetes, lipid metabolism, obesity, smoking, alcohol use, coffee consumption, physical activity, sleep and educational level as lifestyle related risk factors for PAD by using a mendelian randomization (MR) design, that will enable us to investigate causal relationships between PAD and putative risk factors.  A 2-sample MR design will be used to investigate the relationship between lifestyle factors and PAD. In a MR design, single nucleotide polymorphisms (SNPs), genetic variants that are robustly associated with potentially modifiable risk factors, are used as an instrumental variable. Since SNPs are randomly allocated at meiosis, they are largely free of the classic forms of bias that are observed in observational research (e.g. confounding, reverse causation, measurement error) and can therefore be used as proxies to explore causal effects on an outcome, providing certain assumptions are met.  We will conduct an MR study by using summary statistics from genome wide association studies (GWAS). Data from publically available GWASs will be used to select instrumental SNPs. An SNP will be used as an instrument if associated with the risk factor at the genome wide significance level (P<5x10-8). When linkage disequilibrium (r2>0.1) occurs, the SNP with the strongest correlation with the risk factor will be used. SNPs correlated with type 2 diabetes, lipid metabolism, obesity, smoking, alcohol use, coffee consumption, physical activity, sleep and educational level will be used as instruments. The summary statistics from PAD as analyzed in the MVP GWAS will be used as an outcome variable.  The random-effects inverse variance weighted (IVW) method will be used as main analysis method to combine the Wald estimates for each SNPs (SNP outcome estimates divided by SNP exposure estimate where the intercept is constrained to zero) into one causal estimate of each risk factor. The IVW method will provide an unbiased estimate in the absence of horizontal pleiotropy or when horizontal pleiotropy is uncorrelated with SNP-confounder associations and is balanced. Sensitivity analyses will be performed to assess the validity and robustness of the IVW by assessing the presence of possible pleiotropy. Firstly, we will perform complementary analyses using the weighted median and MR-egger regression methods which are relatively robust under different assumptions about pleiotropy, although at the cost of statistical power. Secondly, the MR pleiotropy residual sum and outlier test will be used to identify outlying SNPs reflecting likely pleotropic biases, in which it will be checked whether exclusion of the outlying SNP changes the causal estimate.  All analyses will be performed in RStudio (The R Foundation for Statistical Computing). Results will be reported as odds ratios (OR) with corresponding 95% confidence intervals. Since only summary statistics and no individual-level data will be used in these combined analyses, participants are not exposed to any additional risks.  \",\n\"2021, Mar15\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Bendall, Matthew \",\n\"WEILL MEDICAL COLL OF CORNELL UNIV\",\n\"Investigating retroelement expression in cancer for biomarker discovery\",\n\"approved\",\n\"In human cancers, HERVs (fossil viruses which live in all of our DNA) have been suspected to be involved in pathogenesis of some cancers, with the detection of HERV expression in germ cell tumors, prostate and breast cancers, melanoma, renal cell carcinoma. However, HERV protein, and particle expression during cancers does not ascribe a definitive role for HERV activity in contributing to cancer etiology in humans. Whether or not HERVs play a causative role, HERV expression is an overlooked source of neoantigenic targets or biomarkers of tumor responsiveness. We wish to understand whether targeting specific HERV insertions could be used therapeutically in cancers.\",\n\"The goal of this project is to investigate how transcriptional activity of retroelements, including Human Endogenous Retroviruses (HERVs) and Long Interspersed Nuclear Elements (LINEs), may be involved in disease pathogenesis and prognosis. We have developed an analysis tool, Telescope, that can accurately quantify transposable element expression at a locus specific level. We will use Telescope to determine which HERVs and LINEs are expressed in several different cancers. This project extends our initial work in prostate, breast and colon cancers to also focus on cutaneous and uveal melanomas, lymphomas, and head and neck cancer. We intend to broadly share our results with the scientific community by publishing our findings in peer-reviewed journals.\",\n\"Aug27, 2021\",\n\"phs002315.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Below, Jennifer \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Meta-analyses of Type 2 Diabetes and Lipid Traits in Hispanic/Latino Populations\",\n\"approved\",\n\"Genetic studies of type 2 diabetes and blood lipid traits have focused primarily on European ancestry populations, though Hispanic/Latino populations have higher rates of these diseases. We are performing a study that will aggregate data from many smaller studies of diabetes and blood lipids in Hispanic/Latino populations to increase our ability to understand genetic factors that contribute to disease.\",\n\"Genetic studies of type 2 diabetes (T2D) and lipid traits in populations of European ancestry have uncovered novel, biologically relevant pathways. However, the predominant focus on Non-Hispanic white ancestral populations limits the global relevance and therefore translational potential of these large-scale genome-wide association study (GWAS) efforts, particularly in populations overlooked by many ongoing GWAS initiatives, but most burdened by T2D and dyslipidemia and their downstream consequences, i.e. Hispanic/Latinos. Therefore, we have assembled the DIAMANTE Hispanic/Latino ancestry group and the Global Hispanic Lipids Consortium to perform large-scale meta-analyses and functionally oriented TWAS analyses of T2D and lipid traits in H/L populations.  We currently have summary statistics from nineteen contributing GWAS of T2D, HDL cholesterol, LDL cholesterol, total cholesterol, and/or triglycerides. We are requesting summary statistics for the T2D and lipids analyses in MVP to incorporate into our meta-analyses. This will involve combining summary statistics generated from MVP data with summary statistics generated in other datasets, however this should pose no additional risk to any of the participants. To examine the meta-regression results in a functionally oriented context, we will then use genetically regulated expression prediction models to determine association of each phenotype with predicted gene expression levels.\",\n\"Jul13, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Batini, Chiara \",\n\"UNIVERSITY OF LEICESTER\",\n\"The genetics of smoking and drinking behaviours in underrepresented populations \",\n\"approved\",\n\"Smoking is a leading risk factor for many of the top ten causes of death world-wide, and  smoking prevalence varies among world regions, with 80% of the 1.3 billion smokers worldwide living in low and middle-income countries. Similarly to smoking, alcohol use is a very important risk factor for disease and disability. Both behaviours have shown a genetic background but most studies have been performed in European individuals. We are now conducting studies in African and admixed individuals and the MVP data will be used to verify the robustness of our findings. \",\n\"Smoking is a leading risk factor for many of the top ten causes of death world-wide, and  smoking prevalence varies among world regions, with 80% of the 1.3 billion smokers worldwide living in low and middle-income countries. A reduction in smoking prevalence is expected in most areas by 2025, with the African and the Eastern Mediterranean regions showing an opposite trend. To date, the largest genome-wide association study (GWAS) of smoking behaviour traits includes 1.2 million individuals and highlights genes coding for proteins involved in neurotransmission of nicotine, dopamine and glutamate. However, as in most GWASs, it only includes individuals of European origin failing on the representation of human diversity worldwide. Similarly to smoking, alcohol use is a very important risk factor for disease and disability. Evidence from large GWASs, mostly in European populations, suggest that genetic susceptibility underpins alcohol use and high genetic correlations have been observed between alcohol and smoking behaviours.   In order to disentangle the role of smoking and drinking behaviours on disease susceptibility we are studying the genetic architecture of both traits in different international cohorts (AWI-Gen, Uganda Genome Resource and Mexican Biobank) as well as in the different ancestral backgrounds represented in UK Biobank, with a special interest in individuals of African and admixed ancestry, by far the least represented in genomic studies.   We are now looking for datasets to perform replication analyses for smoking and drinking behaviour traits in individuals of African and admixed ancestries. The results will be compared with those for European individuals to understand the shared genetic architecture among populations and among traits. The traits of interest include smoking initiation and cessation, alcohol use disorder diagnosis and AUDIT-C score, and any trait closely related to these. To date, MVP is the only available cohort with the ancestries of interest represented and with comparable phenotypes, and with sample sizes that would allow a robust replication analysis for our study.  \",\n\"2021, Jul28\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Barr, Peter \",\n\"SUNY DOWNSTATE MEDICAL CENTER\",\n\"Prediction of Alcohol Use Disorders across Ancestral Populations\",\n\"approved\",\n\"Polygenic scores are becoming of interest for medical practitioners and clinicians. The current analysis will examine the clinical utility of polygenic risk scores for identifying those at extreme risk for substance use disorders. We will focus on individuals of both European and African ancestry to address the diversity gap in genomic research.\",\n\"Polygenic scores derived from GWAS summary statistics are becoming increasingly useful in medical research and there is hope that these will provide actionable information for clinical intervention in regards to substance use disorders. However, the majority of the studies focus on individuals of European ancestry. Here, we plan on using summary statistics from large scale GWAS of alcohol use disorders (AUD), alcohol consumption, and correlated phenotypes (e.g., major depression, other substance use disorders) in both European and African ancestry samples in the Million Veterans Program (MVP) to create polygenic risk scores in  holdout samples with available phenotypic information on substance use disorders. Where possible, we will meta-analyze MVP results with other available summary statistics to increase the predictive power of polygenic scores. The goal of this research will focus on whether the extreme ends of the polygenic score distribution for alcohol-related phenotypes provide any clinically relevant information in individuals of European or African ancestries. We will focus on a variety of substance use disorders including alcohol, cannabis, cocaine, nicotine, and others. Data will be used only for research purposes and we will make no attempt to reveal the individual participants. The proposed research is consistent with the data use limitation noted in the Data Use Certification Agreement for MVP. This project will be in collaboration with Dr. Danielle Dick at Rutgers University as part of ongoing efforts to better understand genetic risk in diverse populations (project ID: 21833).\",\n\"2021, Nov09\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Erlebacher, Adrian \",\n\"UNIVERSITY OF CALIFORNIA, SAN FRANCISCO\",\n\"Transcriptional regulation of human decidual stromal cells\",\n\"closed\",\n\"Successful pregnancy involves the coordinated interactions between multiple cell types at the interface between the mother\\u00e2\\u0080\\u0099s uterus and the fetus. It is currently thought that these interactions, when aberrantly controlled, cause a number of pregnancy complications, including preterm birth. The long-term goal of our work is to understand how gene expression within a key cell type of the pregnant uterus, called the decidual stromal cell, is controlled. The dataset we wish to gain access to in this application will allow us to identify key genes whose regulation might be controlled by a specialized \\u00e2\\u0080\\u009cepigenetic\\u00e2\\u0080\\u009d mechanism we recently discovered through our work on the maternal-fetal interface in mice. Understanding whether this mechanism operates during human pregnancy will open up entirely new ways of thinking about how human pregnancy complications come about and would lay the foundation for discovering new therapeutics to improve human pregnancy outcome, such as ones that prevent preterm birth.\",\n\"Successful pregnancy involves the coordinated interactions between multiple cell types at the maternal-fetal interface.  One cell type is the decidual stromal cell (DSC), which comprises the key parenchymal component of the decidua, i.e. the specialized endometrial tissue that surrounds the implanted conceptus. We are interested in understanding how the DSC phenotype is regulated at the transcriptional and epigenetic level in both normal and abnormal human pregnancy.  This project was motivated by our previous finding that a specific histone modification (H3K27me3) silences a large number (~800) of genes in mouse decidual stromal cells (Science 2012, J. Clin. Invest. 2018).  These genes appear in part to be ones that, if inappropriately expressed, might lead to pregnancy complications such as preterm labor. We thus wish to determine whether H3K27me3 silences a similar set of genes in human DSCs.  However, this is a long-term goal and a more immediate aim is to identify genes that are transcriptionally downregulated in human DSCs. We hypothesize that such genes, while silenced in early gestation, might be (re)-induced in late gestation as a part of the process that leads to labor induction.  Moreover, we hypothesize that premature induction of these genes might be a cause of preterm labor.  Accordingly, we are gathering together all published datasets on gene expression profiles of early and late gestation decidual tissue, with the hope that a thorough meta-analysis will allow us to discern key genes of interest for further investigation. The dataset we seek here is a single-cell RNA-seq analysis of the late gestation human decidua experiencing normal parturition timing or preterm labor.  \",\n\"May13, 2020\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Fatumo, Segun \",\n\"LONDON SCH/HYGIENE &amp; TROPICAL MEDICINE\",\n\"Genomic determinant  of Complex Diseases in African ancestry individuals  \",\n\"expired\",\n\"Africa is undergoing a demographic transition leading to an increasing prevalence of non-communicable diseases (NCDs) and a growing cause of morbidity and premature mortality. In this study, our aim is to identify genetic variants that influence NCDs. This project has the potential to lead to advanced understanding of the causes of NCDs in Africa and paving the way for future improved prevention and treatment. \",\n\"Africa is undergoing a demographic transition leading to increasing prevalence of non-communicable diseases (NCDs) and growing cause of morbidity and premature mortality. NCD predictors are all partly heritable. We aiming to use African ancestry individual GWAS summary statistics to identify genetic associations loci, evaluate transferability of the polygenic risk score in continent African and performed Mendelian Randomisation analysis of NCDs (including CVD, Hypertension (HT), Type 2 Diabetes (T2D), Lipid traits, kidney function, post-trauma stress disorder (PTSD), Cancer. Our study provide opportunities to: (1) discover novel disease susceptibility loci; (2) refine (fine-map) association signals at new and existing loci; (3) evaluate transferability of polygenic risk score (4) assess causal relationships of NCDs in Africa using Mendelian randomization\",\n\"Oct27, 2021\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"ASHLEY-KOCH, ALLISON \",\n\"DUKE UNIVERSITY\",\n\"Replication of genetic risk for psychiatric disorders\",\n\"approved\",\n\"Genetic risk factors for psychiatric disorders are commonly associated with small effect sizes.  As a result, replication of genetic findings in an independent cohort is important to establish the validity of the findings.  This project will utilize results of psychiatric studies within the MVP cohort as a replication opportunity for other ongoing psychiatric genetic studies.\",\n\"Data from the psychiatric phenotypes of the Million Veterans project will be used to replicate genetic risk factors in non-MVP cohorts.  This will involve single genetic variant replications, as well as generation and evaluation of polygenic risk scores.  We are not planning on combining data from MVP with other studies, with the exception of possibly performing meta-analysis.  But no direct combining of data will occur.\",\n\"2021, Sep21\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kawai, Vivian \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Genetics and Inflammation\",\n\"approved\",\n\"We are broadly interested to study how inflammatory diseases shared genetic and clinical factors with other clinical phenotypes, and we will use different methods and large genetic studies to identify genetic factors that are shared among different phenotypes.  \",\n\"Objective: The overall goal of this proposal is to define clinical and genetic pleiotropy in inflammatory conditions such as RA, SLE, gout and others.  The availability of large genome-wide association studies (GWAS) for different phenotypes provides an unprecedented opportunity to use big data to make unexpected discoveries. Hence, the specific aims of this analysis are: 1) to determine if the genetic determinants of a risk factor or disease alter the risk of inflammatory diseases; and 2) to determine if the genetic determinants of inflammatory diseases alter risk factors for other diseases or other diseases themselves. Study Design & Analysis Plan: We will use summarized data from different GWAS datasets (including the MVP transethnic blood pressure study in over 750,000 individuals) and the Vanderbilt Medical Center Biobank BioVU (which is linked to the de-identified electronic medical record) to study the association between autoimmune and non-autoimmune traits using: 1) traditional association approaches, b) mendelian randomization methods, 3) genetic risk scores constructed from publicly available GWAS databases that predict the risk factor or disease, 4) phenome-wide association study approach, and 5) the use of methods to imputed molecular endophenotypes (like transcriptomics) to study their association with clinical phenotypes. Our results will be published and broadly share with the scientific community following the NIH policies. \",\n\"2019, Dec30\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kooperberg, Charles \",\n\"FRED HUTCHINSON CANCER CENTER\",\n\"Integrative Genomics into Genetic Association Studies of Blood Pressure and Stroke in African Americans\",\n\"expired\",\n\"We are applying novel methodologies to identify genetic associations between stroke and blood pressure phenotypes for African Americans and other racial/ethnic groups. We do this by finding models associating genetic data with data on other omics, and then by applying these methods to existing genome-wide association studies. Discovery of genetic variants that predispose to blood pressure and stroke is a crucial step toward understanding genetic mechanisms that may lead to novel prevention and treatment strategies.\",\n\"Stroke is the third leading cause of death among African Americans (AAs): they are twice as likely to die from stroke as European Americans (EAs), and their incidence rate is almost double that of EAs. Recent genomewide studies (GWAS) suggest there is a substantial genetic contribution to stroke risk in African ancestry populations, with heritability estimates of about 35%. However, to date, genetic studies in AAs are greatly lagging behind those in EAs despite their increased stroke burden. Among the risk factors for stroke, blood pressure is a major contributor: 4 in 10 AAs suffer from hypertension, 50% more than EAs. These disparities have been considered to be mediated by environmental and social determinants, yet they remain after adjusting for demographics, socioeconomic status, clinical characteristics, and modifiable health behaviors. Heritability analysis suggests African ancestry is associated with hypertension, with heritability estimates from 30\\u00e2\\u0080\\u009340% for systolic and diastolic blood pressure. However, genetic susceptibility to hypertension among AAs is less well studied compared to other ethnic groups. Therefore, there is considerable motivation for identifying the genetic components of stroke and high blood pressure in AAs.  Discovery of genetic variants that predispose to blood pressure and stroke is a crucial step toward understanding genetic mechanisms that may lead to novel prevention and treatment strategies. Yet, GWAS have thus far identified genetic loci that together account for only a small proportion of the heritable risk. Substantial efforts have been devoted to studying the association of genetic variation with gene expression and other molecular characteristics through large collaborative initiatives such as Genotype-Tissue Expression (GTEx) and Encyclopedia of DNA Elements. These initiatives have provided a deeper understanding of functional elements across the genome, which have been used to inform genetic association and identified many novel loci. However, most of the data in these studies have focused on European ancestry and little has been done in AAs. Our recent work in Nature shows that genetic discoveries in one population do not readily transfer to other populations.  The objective of this study is to identify variants predicting various genomic features (gene expression, methylation and protein) in AA samples that have been recently collected through Trans-Omics for Precision Medicine, the CommonMind Consortium, and GTEx, and to integrate this functional information into genetic association analysis of blood pressure and stroke in AAs. Insight into both molecular activity and genetic variation can inform association analysis and enable novel genome-wide discoveries. In particular, we propose to develop methods that leverage the data for EAs to improve power for identifying genetic variants that regulate various types of genomic features in AAs, and for integrating the genomic regulation models into GWAS with the ultimate goal to identify novel loci for stroke risk and blood pressure in AAs. To facilitate these aims we have assembled the largest number of AAs for genomic studies and AA stroke cases and blood pressure data for GWAS.  One of the main strategies that we will employ is to construct predictive models for the genomic features (gene expression, methylation and protein) based on GWAS data and apply these models to existing studies that have GWAS summary statistics for these features available. The data for which we are applying has such summary statistics. \",\n\"2022, May06\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kooperberg, Charles \",\n\"FRED HUTCHINSON CANCER CENTER\",\n\"Genetic risk of type 2 diabetes and related risk factors and complications\",\n\"approved\",\n\"We are requesting genome-wide association study (GWAS) summary statistics that will be used to construct multi-ancestry type 2 diabetes (T2D) polygenic risk scores (PRS), which will be used to determine whether genetic risk of T2D is predictive of T2D complications and whether genetic risk of T2D is mediated by T2D risk factors.\",\n\"Type 2 diabetes (T2D) is a major public health challenge that expected to notably rise in prevalence over the next couple decades, demonstrating the urgent need to improve T2D risk prediction and stratification. Polygenic risk scores (PRS) have shown promise in risk stratification of T2D, particularly when combined with non-genetic clinical predictors. Within the Population Architecture using Genomics and Epidemiology (PAGE) Study, we recently found that a multi-ancestry T2D PRS performed similarly or better than population specific T2D PRS and accounted for up to 15.3% of familial relative risk of T2D across diverse populations (Polfus, Darst et al. HGG Advances 2021). However, given the large number of modifiable risk factors for T2D, further characterizing these scores could lead to a better understanding of the mechanisms impacted by genetic risk of T2D. Moreover, it remains unclear whether T2D PRS could hold prognostic value for diabetes-related complications in individuals who have T2D.  Our primary study question will investigate whether a multi-ancestry type 2 diabetes (T2D) polygenic risk score (PRS) is associated with T2D-related phenotypic risk factors and complications across diverse populations. Our secondary study question will investigate whether the effect of the T2D PRS on T2D risk is mediated by T2D risk factors. We are requested genome-wide association study (GWAS) summary statistics to construct these PRS, which be subsequently calculated and investigated in the PAGE Study. Summary statistics may be combined with T2D GWAS summary statistics generated in the PAGE Study, but no additional risks will be posed to research participants. This is a joint research project with Dr. Burcu Darst at the University of Southern California (Project #29826).\",\n\"2021, Sep16\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kochi, Yuta \",\n\"TOKYO MEDICAL AND DENTAL UNIVERSITY\",\n\"Analysis of genetic factors of severe COVID-19 through eQTL and sQTL of immune cells\",\n\"approved\",\n\"In order to clarify the genetic factors for the severe manifestation of the COVID-19, several genetic studies including the one led by COVID Human Genetic Effort have been performed. Although these studies identified several genetic factors, the whole picture of genetic determinants for the severe disease is not fully drawn. Because these genetic studies had limitations for statistical power, genetic factors with moderate effects remained unidentified. Another challenge in the genetics of COVID-19 is to dissect the causal mechanism. Most of the genetic variants for other immune-related diseases are now considered to affect gene expression or alternative splicing. Therefore, it is essential to integrate genetic data with expression data to dissect the causal mechanism of disease. We performed eQTL/sQTL studies that examined the relation between genetic variants and expression/splicing using immune cell subsets. Through our experiences in the research field, we aim to elucidate the disease mechanism of severe COVID-19 by integrating genetic data and eQTL/sQTL data. We also aim to establish a prediction model for severe disease manifestation using individual\\u00e2\\u0080\\u0099s genome data by evaluating the accumulation of disease-associated eQTL/sQTL. \",\n\"In order to dissect the genetic factors for the severe manifestation of the COVID-19, several genome-wide association studies (GWAS) including the one led by COVID Human Genetic Effort have been performed. Although these GWAS identified several risk loci such as a genomic segment on chromosome 3 inherited from Neanderthal, the whole picture of genetic determinants for the severe disease is not fully drawn. Because these GWAS have set a strict threshold for the genome-wide significance (i.e. P=5x10E-8), majority of risk loci with moderate effects did not reach this threshold and remained unidentified. Another challenge in the genetics of COVID-19 is to dissect the causal mechanism in the GWAS loci. Most of GWAS loci identified in other immune-related diseases, such as autoimmune diseases and allergic diseases, are now considered to be expression quantitative trait locus (eQTL) or splicing QTL (sQTL), where genetic variants regulate gene expression or alternative splicing. In fact, we performed GWAS for clinically amyopathic dermatomyositis, which manifests severe lung disease similar to COVID-19, and we identified that a SNP having an sQTL effect for the WDFY4 gene caused the disease (Kochi Y et al, Ann Rheum Dis 2018). Therefore, it is essential to integrate GWAS data with eQTL/sQTL data to dissect the causal mechanism of disease. To this end, several eQTL/sQTL studies such as GTEx, DICE, and ImmVar project have been performed. We also performed eQTL/sQTL studies using immune cell subsets (Ishigaki K and Kochi Y et al, Nat Genet 2017). Moreover, we established a method to evaluate polygenic burden of these eQTLs in individuals and evaluated the signal pathways that are activated in the disease (Nat Genet 2017). Through our experiences in the research field, we aim to elucidate the disease mechanism of severe COVID-19 by integrating GWAS data and eQTL/sQTL. We also aim to establish a prediction model for severe disease manifestation using individual\\u00e2\\u0080\\u0099s genome data by evaluating the polygenic burden of disease associated eQTL/sQTL with moderate effects.    We combine the present dataset (whole genome sequencing data and GWAS data of COVID-19) with the summary statistics obtained from the eQTL/sQTL data (GTEx, DICE, ImmVar and our own) that are publicly available. Our analysis will not create any additional risks to participants. \",\n\"2020, Dec04\",\n\"phs002245.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kim, Dong Ki \",\n\"SEOUL NATIONAL UNIVERSITY HOSPITAL\",\n\"Causal factors for diabetes and cardiovascular diseases: A Mendelian randomization study\",\n\"closed\",\n\"We are planning to use the genome-wide association study results for diabetes mellitus in our study. Summary statistics are generated from genome-wide association study results, identifying associations between genetic information and diseases, and in the summary-level data, individuals are not identifiable. The summary statistics can be implemented in a method of \\\"Mendelian randomization\\\" analysis, which stratifies genetic risk of an exposure and investigating the diseases risk according to the genetic predisposition. The method has strength in identifying causal risk factors for complex diseases. Mendelian randomization analysis can be performed by summary-level data, one for the exposure and one for the disease outcome. The data from phs001672 includes a genome-wide association study results for diabete mellitus, so the data will be utilized as the outcome summary statistics of our Mendelian randomization study.\",\n\"This study aimed to identify causal factors for diabetes and cardiovascular diseases through a Mendelian randomization analysis. The study will perform two-sample Mendelian randomization analysis, and we will implement the data from phs001672 as the outcome summary statistics for the Mendelian randomization study. The genetic instrument for the Mendelian randomization will be introduced from external sources, but the information would not be directly cooperated into the dataset but only the summary-level data of the data from phs001672 will be utilized. Thus, the study would not result in additional risks to participants included in the phs01672 data. The data of particular interest has been published in Nature Genetics volume 52, pages 680\\u00e2\\u0080\\u0093691(2020), and we are planning to utilize the GWAS summary statistics for type 2 diabetes mellitus of individuals of European ancestry (pha004945.1) as the outcome of our two sample Mendelian randomization analysis. Two sample Mendelian randomization analysis will be performed by a statistical package in R. \",\n\"Sep29, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kilpelainen, Tuomas \",\n\"UNIVERSITY OF COPENHAGEN\",\n\"Mendelian randomization analyses of fatty liver disease and metabolic disorders\",\n\"closed\",\n\"Fatty liver disease is a common disease characterized by excessive fat accumulation in the liver, which may ultimately result in cirrhosis or cancer. We will study whether fatty liver disease is causally associated with blood lipid levels and other metabolic risk markers and disease risk, such as type 2 diabetes and heart disease. The results will show whether fatty liver disease drives the development of metabolic disease, or whether metabolic disease accelerates the development of fatty liver disease. \",\n\"Fatty liver disease is the most frequent chronic liver disease in Western countries, affecting more than 30% of the general population. A large body of observational studies suggest that fatty liver disease is associated with metabolic disease and inflammation. However, observational studies suffer from confounding and reverse causality, and it thus remains unclear whether fatty liver disease is causally associated with metabolic disease and inflammation, or vice versa. As genotypes are randomly allocated at conception, genetic alleles can be used to assign individuals according to higher or lower levels of an exposure trait in a randomized manner. This approach, Mendelian randomization, helps to assess causal relationships by minimizing the influence of confounding and reverse causality on observational results.   In this project we will test bidirectional causal associations between fatty liver disease and metabolic and inflammatory profiles using two-sample Mendelian randomization approach applied on summary statistics from genome-wide association studies (GWAS). As part of this study, we request access to the dbGaP summary statistics for the Million Veteran Program GWAS for LDL cholesterol, HDL cholesterol, total cholesterol and triglycerides, published in 2018 [Klarin et al. Nat Genet 2018], to study the bidirectional causal associations between lipid levels and fatty liver disease (liver inflammation, steatosis and fibrosis). To maximise the sample size for HDL cholesterol and triglycerides, we will meta-analyze the summary statistics from the Million Veteran Program with the summary statistics available from lipids meta-analyses published by the Global Lipids Genetics Consortium in 2013 [Willer CJ et al Nat Genet 2013]. As only summary statistics and no individual-level data will be used in these combined analyses, no additional risks will be created for the participants. \",\n\"2020, Dec21\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kilpelainen, Tuomas \",\n\"UNIVERSITY OF COPENHAGEN\",\n\"Genetic regulation of weight gain and risk of diabetes and cardiovascular disease\",\n\"closed\",\n\"Obesity is associated with increased risk of diabetes, heart disease, and stroke. However, many obese individuals, often called \\u00e2\\u0080\\u009cmetabolically healthy obese\\u00e2\\u0080\\u009d, seem protected from these diseases. Conversely, some individuals who are normal weight have an increased metabolic risk similar to the obese. This paradox may be partly explained genetic differences. Indeed, recent genetic studies have identified multiple DNA variants associated with higher body fat but lower insulin resistance and risk of diabetes and heart disease. Many of these variants are linked to storing relatively more fat on the hip than on the waist. In the present project, we will examine whether DNA variants showing such \\\"favorable adiposity\\\" effect are associated with long-term weight gain that does not lead to increased risk of diabetes, heart disease, or stroke. \",\n\"The current obesity pandemic is a major threat to public health systems worldwide. The majority of obesity-related health care costs are due to cardiometabolic impairments such as insulin resistance, dyslipidemia, and hypertension, which increase the risk of type 2 diabetes and cardiovascular disease. However, many obese individuals seem resistant to cardiometabolic complications, the \\\"metabolically healthy obese\\\". Conversely, some normal weight individuals suffer from cardiometabolic comorbidities similar to the obese, the \\\"metabolically obese normal-weight\\\". Genetic mechanisms may partly explain this paradox. Recent genome-wide association studies have identified multiple genetic loci that are associated with increased overall body fat and subcutaneous fat deposition, but lower cardiometabolic risk. At present, it remains unclear whether genetic predisposition to such \\\"favorable adiposity\\\" may uncouple long-term weight gain from cardiometabolic risk during adulthood.  The current obesity pandemic is a major threat to public health systems worldwide. The majority of obesity-related health care costs are due to cardiometabolic impairments such as insulin resistance, dyslipidemia, and hypertension, which increase the risk of type 2 diabetes and cardiovascular disease. However, many obese individuals seem resistant to cardiometabolic complications, the \\\"metabolically healthy obese\\\". Conversely, some normal weight individuals suffer from cardiometabolic comorbidities similar to the obese, the \\\"metabolically obese normal-weight\\\". Genetic mechanisms may partly explain this paradox. Recent genome-wide association studies have identified multiple genetic loci that are associated with increased overall body fat and subcutaneous fat deposition, but lower cardiometabolic risk. At present, it remains unclear whether genetic predisposition to such \\\"favorable adiposity\\\" may uncouple long-term weight gain from cardiometabolic risk during adulthood.  The overall objective of the present project is to examine whether genetic predisposition to a \\\"metabolically healthy obese\\\" phenotype diminishes the impact of weight gain on risk of cardiometabolic disease. We will construct a \\\"favorable adiposity\\\" genetic score by combining all genetic loci associated with increased body fat but decreased insulin resistance (defined by decreased fasting insulin and triglycerides, and increased HDL cholesterol) by using genome-wide summary statistics from the largest available genome wide association studies. To generate this genetic score, we are requesting dbGaP summary statistics data for HDL cholesterol and triglycerides for European-ancestry individuals from the Million Veteran Program GWAS, published in 2018 (Klarin D et al Nat Genet 2018). To maximise the sample size for HDL cholesterol and triglycerides, we will meta-analyze the summary statistics from the Million Veteran Program with the summary statistics available from lipids meta-analyses published by the Global Lipids Genetics Consortium in 2013 (Willer CJ et al Nat Genet 2013). As only summary statistics and no individual-level data will be used in these combined analyses, no additional risks will be created for the participants. We will subsequently use the genetic score derived from the variants identified to be associated with insulin resistance, to test whether the scores diminishes the impact of weight gain on the levels of cardiometabolic risk markers and risk of type 2 diabetes and cardiovascular disease in a meta-analysis of data from six Danish prospective cohorts, locally available to us at the University of Copenhagen.  \",\n\"Feb27, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kelly, Tanika \",\n\"TULANE UNIVERSITY OF LOUISIANA\",\n\"Genomic determinants underlying cardiovascular disease susceptibility\",\n\"approved\",\n\"Cardiovascular disease is a leading cause of death globally. Our genetic makeup influences our susceptibility to cardiovascular disease and its contributing risk factors. Identification of the genes involved in these processes has been difficult for scientists. However, recent advances in technology and statistical methods, as well as public availability of genomic data from numerous large population-based studies, now provides scientists with the opportunity to more comprehensively examine genes and genetic factors related to cardiovascular disease. In the proposed study, we will explore the association between genetic variants across the genome for their association with cardiovascular disease and its risk factors in many large studies. Through the proposed research, we may identify important genes for cardiovascular disease, which could lead to advancements in its treatment and prevention.\",\n\"Cardiovascular disease (CVD) is a leading cause of death globally. Hypertension, diabetes, chronic kidney disease, obesity, and dyslipidemia, as well as their related cardiometabolic traits, associate with increased risk of CVD. Recent genome-wide association studies (GWAS) have identified important loci related to these phenotypes. However, our understanding of their genomic etiology remains limited. The objective of the current study is to identify genes and variants that influence CVD-related traits among studies accessed through dbGaP as well as in the NHLBI-funded Genetic Epidemiology Network of Salt-Sensitivity study. We will use exome-sequencing, genome-wide genotype, and GWAS meta-analysis results data from the Atherosclerosis Risk In Communities study, the Cardiovascular Health Study, the Chronic Renal Insufficiency Cohort Study, the International Consortium of Blood Pressure, the Finland-United States Investigation of NIDDM Genetics study, the Framingham Heart Study, the Jackson Heart Study, the Multi-ethnic Study of Atherosclerosis, the Coronary Artery Risk Development in Young Adults Study, and the Women\\u00e2\\u0080\\u0099s Health Initiative. To achieve our overall objective, we will carry-out the following specific aims: 1.) To identify novel common and low-frequency variants that influence CVD-related traits. We will use several approaches to identify such variants, including GWAS, meta-analysis, and candidate gene study using GWAS results, genome-wide genotype and exome sequencing data. Single-marker and pathway-based analyses will facilitate novel variant discovery. 2.) To identify novel rare variants that may have functional effects on CVD-related traits. We will use state-of-the-art statistical methods to examine the collective effects of rare variants in aggregate analyses using exome sequencing data. 3.) To examine the collective effects of known genomic variants for CVD-related traits to predict risk of CVD in these large, population based studies. By helping to elucidate the biological pathways underlying CVD and its related risk factors, these findings may aid in the prevention CV\",\n\"2021, May15\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kaikkonen-M\\u00c3\\u00a4\\u00c3\\u00a4tt\\u00c3\\u00a4, Minna \",\n\"UNIVERSITY OF EASTERN FINLAND\",\n\"Functional interpretation of enhancer variants associated with CAD\",\n\"approved\",\n\"Coronary artery disease (CAD) is the leading cause of death in the Western world. To improve disease prevention, diagnosis and treatment, we need to gain better understanding of the genetic and environmental factors that promote the progression of the disease. Hundreds of SNPs have been significantly associated with CAD. However, the vast majority of such variants are located in the noncoding sequences with yet undefined role. Large fraction of them do fall within enhancers specific for disease relevant cell types. In this study, we aim to identify and functionally characterize enhancer variants affecting cell type specific gene expression. Shedding light into the molecular mechanisms of these variants is hoped to provide insight into the genetic basis of CAD. \",\n\"Coronary artery disease (CAD) is the leading cause of death in the Western world. To improve disease prevention, diagnosis and treatment, we need to gain better understanding of the genetic and environmental factors that promote the progression of the disease. Hundreds of SNPs have been significantly associated with CAD. However, the vast majority of such variants are located in the noncoding sequences with yet undefined role. Large fraction of them do fall within enhancers specific for disease relevant cell types. In this study, we aim to identify and functionally characterize enhancer variants affecting cell type specific gene expression. To provide mechanistic insight for CAD associated regions, GTEx data is used to identify target genes using eQTL and allele-specific gene expression analysis. In addition, GTEx data is used to build tissue-specific gene regulatory networks that are compared to our cell-type specific gene modules and for biomarker analysis. GTEx data is analyzed independently of other datasets. The Million Veteran Program (MVP) summary statistics and credible sets from the latest CAD GWAS are used to prioritize and select regulatory elements for functional characterization using CRISPR and massively parallel reporter assays. This could allow fine-mapping of risk variants and identification of causal disease mechanisms.\",\n\"Apr15, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Fatumo, Segun \",\n\"LONDON SCH/HYGIENE &amp; TROPICAL MEDICINE\",\n\"Investigating Predictive Utility of Polygenic Risk Scores of Chronic Kidney Disease in Africans\",\n\"approved\",\n\"Currently, more than 50 million people in Africa have Chronic Kidney Disease (CKD) making it the continent with the greatest burden of CKD in the world. With the rapid advances in genotyping technologies, it is anticipated that the advent of the global knowledge in CKD genomics could be translated to clinical benefits. A key application of genomic data is in disease risk prediction, which can potentially contribute to precision medicine such as individualized preventive strategies and early diagnosis of diseases. In this project, we aim to develop an approach called polygenic risk score that can help to identify a sub-group of people in Africa who may be at higher risk of kidney disease in order to provide earlier intervention and treatments.  \",\n\"Currently, more than 50 million people in Africa have Chronic Kidney Disease (CKD) making it the continent with the greatest burden of CKD in the world. With the rapid advances in genotyping technologies, it is anticipated that the advent of global knowledge in CKD genomics could be translated to clinical benefits. A key application of genomic data is in disease risk prediction, which can potentially contribute to precision medicine such as individualized preventive strategies and early diagnosis of diseases. In view of the limited transferability of Polygenic Risk Score (PRS) across ethnicities, in this project, we will develop the PRS for prediction of CKD within African populations. We will evaluate best approaches for using a diverse discovery dataset from trans-ethnic Genome-wide association studies (GWAS) meta-analysis and a target dataset individuals from continental Africa to develop and evaluate a novel PRS for the prediction of CKD leveraging on diverse datasets of trans-ethnic Genome-wide association studies summary statistics from multi-ethnic groups. \",\n\"2021, Apr20\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"KALLAPUR, SUHAS \",\n\"UNIVERSITY OF CALIFORNIA LOS ANGELES\",\n\"Single cell transcriptomics of the fetal membranes and placenta\",\n\"closed\",\n\"We are requesting access to the dataset so that we can compare our data that we will generate in Rhesus monkey placenta with those from human. This will ensure rigor of study and avoid duplication.\",\n\"We have performed single cell RNA seq analyses of preterm pregnant Rhesus macaques. We are requesting access to the human transcriptomic data set to compare our results in the Rhesus with those of human. We do not plan to combine datasets. If there are important data that are correlated, we may publish the human data finding as confirmatory. In such a case we will acknowledge the source and the publication. The published data will be of a global nature and not specific to any individual or study participant. \",\n\"Feb01, 2022\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Jung, Jeesun \",\n\"NIH\",\n\"Multi-omic and Mendelian randomization analyses of alcohol use\",\n\"approved\",\n\"Genetic variants associated with alcohol use can be used to describe the biology of alcohol addiction and identify relationships between alcohol addiction and other traits. We will use genetic variants associated with alcohol use in Million Veterans Program (MVP) studies to perform computational research aimed at these aforementioned goals.\",\n\"Summary statistics from Million Veterans Program (MVP) genome-wide associations studies (GWAS) of maximum habitual alcohol intake, AUDIT-C, and problematic alcohol use will be used to perform multi-omic and Mendelian randomization analyses of alcohol use. Our multi-omic analyses will seek to describe transcriptomic, cell-type, and protein associations with alcohol use. Our Mendelian randomization analyses will seek to identify potential causal relationships between alcohol use and clinical and biological phenotypes. We will perform both discovery and replication analyses using MVP data. The aim of our analyses is to elucidate the biology of alcohol addiction and identify therapeutic targets for alcohol use disorder.\",\n\"2023, Mar02\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Jonas, Katherine \",\n\"STATE UNIVERSITY NEW YORK STONY BROOK\",\n\"Exploring the effect of dimensional phenotypes on gene discovery\",\n\"approved\",\n\"When scientists want to identify the genetic factors associated with psychological illnesses, they most often divide people into people with the illness (cases) and people without the illness (controls).  However, this method of grouping people can be inefficient, because cases with very severe symptoms are not distinguished from cases with mild symptoms, nor are controls with subthreshold symptoms differentiated from people with no symptoms.  In this study, we will compare the traditional, case-control method to a method in which every person is rated on a continuous scale. We hypothesize that a continuous scale will be a more efficient method of identifying genetic risk factors for psychological illnesses.\",\n\"The field of molecular psychiatric genetics has made enormous advances in the last decade, predominantly through large-scale genome-wide association studies (GWASs) dependent on categorical, case-versus-control analyses embedded within traditional diagnostic classification systems. A growing body of evidence suggests that a quantitative and transdiagnostic approaches to phenotype measurement can further enhance the discovery power of GWASs. The current study provides a comprehensive review of such phenotypic approaches in GWASs across all major psychiatric and neurodevelopmental disorders. We identify seven themes and recommendations that emerge from the literature, encompassing the issues of statistical power, disorder prevalence, psychopathology measurement, and novel phenotypes emerging from hierarchical \\u00e2\\u0080\\u0098splitting\\u00e2\\u0080\\u0099 and \\u00e2\\u0080\\u0098lumping\\u00e2\\u0080\\u0099 approaches. We also explore the effect of dimensional phenotypes on heritability and validity.  Our objective in requesting summary statistics from the MVP GWAS is to compare the heritability and genetic correlations of the PTSD, MDD, and GAD phenotypes reported in Stein et al. (2021) and Levey et al. (2021) with the case-control phenotypes for PTSD, MDD, and GAD reported in Nievergelt et al (2019), Howard et al (2019), and and Purves et al (2020), respectively.  Heritability and genetic correlations will be estimated using LD score regression (Bulik-Sullivan, 2015). The heritability and genetic correlation estimates will be used to evaluate the utility of dimensional, quantitative phenotypes for GWAS.\",\n\"Jun29, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Johnson, Eric \",\n\"RESEARCH TRIANGLE INSTITUTE\",\n\"Genome-wide association study of opioid addiction using public controls\",\n\"rejected\",\n\"Approximately 50% of the risk for heroin abuse is attributable to genetic factors. However, few genes have been consistently associated with heroin abuse, and it is likely that a number of genes contributing to heroin abuse have not been identified. The proposed study would test associations between heroin abuse and genetic variants across the genome, with the goal of uncovering biological mechanisms that underlie drug addiction. More broadly, we also examine genetic risks for substance use disorders regardless of specific drug used.\",\n\"The goal of the proposed study is to identify novel genetic determinants of opioid addiction (OA) and related substance use disorder (SUD) phenotypes by conducting genome-wide association studies (GWAS), including meta-analyses across cohorts, as well as multi-omic gene network studies for SUD phenotypes that combine GWAS with other omics (e.g., RNAseq from postmortem brain tissue) in multiplex data.  Although GWAS studies have successfully identified genetic variants contributing to other SUD phenotypes (e.g. nicotine dependence), there are few GWAS of rarer high-risk substance abuse.  For GWAS, we propose to use a case-control study design, drawing OA cases from the Urban Health Study (UHS) and other case-only cohorts and matching population controls from dbGaP deposited studies.  We have published GWAS using such a case / population control approach (Johnson, et al, 2015; Hancock et al. 2015; Gaddis et al. 2022) and have recently extended our array-dependent matching protocol to use cohorts with whole genome sequencing data (Mathur et al. 2022).  In the current period we will continue to use these previously acquired cohorts and add dbGaP studies as population controls for newly genotyped OA cases to study SUD phenotypes. We will also request new dbGaP studies with SUD phenotype data for replication and/or meta-analyses as they become available.  We will continue to participate in the Addiction group of the Psychiatric Genetic Consortium (PGC): analyzing these data in large meta-analyses with other investigators. We also lead the Genetics of Opioid Addiction Consortium (GENOA), which combined summary statistics from 13 cohorts (Gaddis et al., 2022). As part of NIDA Smoke Screen Initiative, several new cohorts that are informative for OA have been or are in the process of being genotyped, which we will request under this project. This projects work is now funded under P50 DA054071: Integrative Omics Center for Accelerating Neurobiological Understanding of Opioid Addiction (IOmics-OA).\",\n\"Jul07, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Jin, Shanzhao \",\n\"BIOMAP (BEIJING) INTELLIGENCE TECHNOLOGY LIMITED\",\n\"Integrated analysis of multi-omics data on human synovial tissue and PBMC in Rheumatoid Arthritis (RA)\",\n\"approved\",\n\"The main goal of this study is to reveal the underlining molecular mechanisms between treatment-responded and treatment resistant patients with RA. Specifically, we want to identify the critical factors that drive immune balance in RA patients with treatment. To explore potential targets or treatment strategies for RA, we plan to define and validate the cell subpopulations, cell states and gene pathways associated with RA pathogenicity and treatment response.\",\n\"The Target Discovery Research team at BioMap aims at leveraging multi-omic data for the novel drug discovery. Specifically, we are interested in auto-immune diseases such as Rheumatoid Arthritis. Our research objectives are to compare the RA synovial biopsy, PBMC, OA and healthy control datasets in public databases (including dbGaP) with our in-house data.    We want to make full use of datasets from dbGaP, particularly those from the Accelerating Medicines Partnership (AMP), as meaningful and significant references for our in-house single-cell RNA-seq (scRNA-seq) and spatial transcriptome datasets. First, we intend to integrate all the available datasets to uniformly define and validate cell subpopulations in RA tissues by different studies and use them as a solid reference for our in-house scRNA-seq dataset from synovial tissues and PBMC. Second, to identify the priority or major cell-cell interaction network hubs in RA progression and remission during treatment, including those cell types contributing to tissue repair/anti-inflammatory and tissue damage/pro-inflammatory, respectively. Finally, we will detect cellular responses to perturbations to identify relationships between genes and RA in silicon and in vitro. Overall, we request access to datasets from projects phs000983.v1.p1, phs001262.v1.p1.c2, phs001529.v1.p1.c1, phs001457.v1.p1, phs001459.v1.p1 and phs001615.v1.p1. The comparison and combination of these datasets with our in-house data will provide potential insights for RA pathogenesis and immunotherapies.   BioMap aims to accelerate novel drug discovery for unmet medical needs with secure IT systems and data storage. We will comply with all regulations for data access as set out in dbGaP, and we state that our institution's IT security requirements and policies are sufficient to protect the confidentiality and integrity of the NIH controlled-access data. \",\n\"2023, Feb14\",\n\"phs002780.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Jaffe, Andrew \",\n\"BLACKTHORN THERAPEUTICS, INC.\",\n\"Genetic targets of mood disorders in MVP\",\n\"rejected\",\n\"Recent classes of methods have been developed to \\u00e2\\u0080\\u009cimpute\\u00e2\\u0080\\u009d gene expression levels in hard-to-ascertain tissues like brain using just DNA sequence variation, which can identify new targets for different mood disorders when casting wider nets of molecular associations in brain tissue. \",\n\"Recent classes of methods have been developed to \\u00e2\\u0080\\u009cimpute\\u00e2\\u0080\\u009d gene expression levels in hard-to-ascertain tissues like brain using just DNA sequence variation, for example predixcan [PMID: 26258848] and TWAS [PMID: 26854917]. While these algorithms have already published and publicly disseminated genotype/SNP \\u00e2\\u0080\\u009cweights\\u00e2\\u0080\\u009d to predict gene expression levels across all tissues in the Genotype By Tissue Expression (GTEx) project [PMID: 32913098], only two expression summarizations \\u00e2\\u0080\\u0093 gene-level and splicing-level are routinely considered. We have previous shown that other expression features like exons and exon-exon junction counts may be more strongly related to genetic variation than gene-level summarizations [PMID: 30050107], which require generating new \\u00e2\\u0080\\u009cweights\\u00e2\\u0080\\u009d at these features. Here we propose to cast a wider net of molecular/eQTL characterizations with the goal of identifying new targets in the genetic risk of different mood disorders identified by MVP. \",\n\"2021, Sep01\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"JANES, KEVIN \",\n\"UNIVERSITY OF VIRGINIA\",\n\"Comparing SARS-CoV-2 Heart Infection to Other Viral Heart Infections\",\n\"approved\",\n\"Like other organs, the human heart can be infected by viruses. Previously, we identified different ways that human hearts can react to the presence of virus. However, it is not known whether these reactions to virus are harmful or helpful to the heart. In this study, we are aiming to compare each newly identified heart reaction to a known heart reaction. SARS-CoV-2 is well recognized for its ability to infect the human heart and cause damage. Therefore, we intend to compare the heart reactions that we identified to how the heart responds to SARS-CoV-2 infection. We anticipate that this project will advance our understanding of viral heart infections and how the body responds to them. We will publish the results of this study to make them available to the public.\",\n\"Several viruses are known to infect the human heart. These infections can either be short-term and acute in nature or persist in a long-term chronic state. Chronic viral heart infection is associated with progression to heart failure. Using other publicly available RNA-seq datasets, we have identified undiagnosed infections in patient hearts. We then found that patient hearts harboring viral RNA showed signs of adapting to the viral infection in distinct ways. However, it is unknown whether these heart adaptations are harmful or helpful to the human heart. Addressing this question requires a known heart adaptation. SARS-CoV-2 is a well-known heart pathogen that causes myocarditis. The objective of this project is to use RNA-seq profiles from patient hearts with severe SARS-CoV-2 infection to build a reference frame to compare with our identified heart adaptations to other viral infections. Differentially expressed gene lists are available comparing COVID+ hearts to control hearts. However, to perform a rigorous comparison between SARS-CoV-2 infected hearts and our previously identified heart adaptations, we must process the data in exactly the same way. Therefore, we are requesting the raw sequencing reads. All data will be analyzed and stored through the Information Technology Services at the University of Virginia. Upon completion of this study, we will disseminate the results by posting a pre-print of the work, followed by formal publication.\",\n\"Jan06, 2023\",\n\"phs002258.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kooperberg, Charles \",\n\"FRED HUTCHINSON CANCER RESEARCH CENTER\",\n\"Learning Genetic Architecture of Complex Traits Across Populations\",\n\"closed\",\n\"Genome-wide association studies (GWAS) have become a primary tool for understanding the genetic basis of complex traits and diseases.  The goal of this research is to augment our understanding of cardiovascular diseases in minority populations, so that GWAS can benefit all individuals. Specifically, by leveraging information across diverse populations, our analyses aim to identify minority-relevant genetic variants and to build polygenic risk scores (PRS) for minority populations. \",\n\"Understanding the genetic factors influencing complex traits in minority populations is challenging because the sample sizes are often much smaller in Genome-wide association Studies (GWAS) of minority populations, compared to GWAS in European populations. As described below, we have previously developed minority-centric association mapping and genetic risk prediction approaches. The key insight underlying these approaches is to leverages multi-ethnic and trans-ethnic information, while acknowledging heterogeneity across populations. The objective of this dbGaP application is to apply and extend these methods to identify identifying minority-relevant genetic factors underlying plasma lipids, stroke and VTE in African Americans and Hispanics. We will also build population-specific polygenic risk score (PRS) for lipids and CAD (stroke and VTE). For association mapping, we will apply an empirical Bayes approach developed by Dr. Tang\\u00e2\\u0080\\u0099s group, XPEB (Coram et al.2015;  PMID: 25892113). This method considers a study design, in which summary-level statistics from two GWAS are available: one in the minority population of primary interests (referred to as the focal population), and the other in a reference population of much larger sample size (referred to as the base population). As an example, to identify genetic loci associated with low-density lipid cholesterol (LDL) in African Americans, we will generate the focal GWAS summary-statistics by meta-analyzing an African American LDL study performed in the Women\\u00e2\\u0080\\u0099s Health Initiative (WHI) with the African American LDL analysis in the Million Veterans Program. (All investigators on this, and the collaborating, applications have access to WHI). For base GWAS, we will use MVP\\u00e2\\u0080\\u0099s European-stratified LDL summary statistics.  For PRS, we will apply a mixed-effects model approach developed by Dr. Tang\\u00e2\\u0080\\u0099s group, XP-BLUP (Coram et al. 2017; PMID: 28757202). This method considers a study design, in which individual-level genotype-phenotype data is available in a minority cohort (the focal GWAS), while only summary-level GWAS statistics are available in the base GWAS. In the proposed study, WHI individual-level African Americans and Hispanic American GWAS data are available to all investigators. Analysis: For association mapping, we will apply XPEB (implemented in an open source R package)  for each of the lipids and CAD traits, and using African Americans or Hispanics as the focal population.  For PRS, XP-BLUP program, implementing the mixed-effects model, is available on github. Specifically, we will include age and genetic-PC as covariate for all analysis. In addition, trait-specific covariates will be adjusted as needed.    Validation/Replication: For minority-relevant loci identified using XPEB, we will seek replication in the appropriate minority-populations. Because we anticipate some loci identified by XPEB feature moderate/weak effect sizes, other minority-cohorts are under powered. As an alternative, we will examine consistency in allelic effects and assess the improvement in disease risk prediction. For minority-specific PRS, we evaluate the predictive performance on an independent cohort of matching minority population (African Americans or Hispanics). We will compare PRS built by XP-BLUP with PRS directly transferred from European cohorts.  The DUL states \\u00e2\\u0080\\u009cuse of this data is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry. Use of the data includes methods development research (e.g., development of software or algorithms).\\u00e2\\u0080\\u009d Our analysis will use the population-specific GWAS results produced by the MVP; no analysis is proposed to study the population origin or ancestry.  The proposed research will be performed in collaboration with Dr. Hua Tang\\u00e2\\u0080\\u0099s group at Stanford University. Dr. Tang is submitting a dbGaP application through Stanford.\",\n\"Apr29, 2021\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Koyuturk, Mehmet \",\n\"CASE WESTERN RESERVE UNIVERSITY\",\n\"Identification of causal drivers of labor in human pregnancy\",\n\"approved\",\n\"The process of parturition involves the transformation of the quiescent myometrium (uterine smooth muscle) to the highly contractile laboring state.  This is thought to be driven by changes in gene expression in myometrial cells.  Despite the existence of multiple myometrial gene expression studies, the gene expression changes that initiate labor are largely unknown.  This makes it difficult to understand and treat labor disorders such as preterm labor.   We will use this large-scale and high-resolution dataset to identify the drivers of labor in term and pre-term pregnancy. Such markers can potentially lead to the development of therapies for preventing and/or controlling pre-term birth, a public health problem that affects millions of infants and their families, many throughout their entire lifespan.\",\n\"While the interplay between progesterone signaling, inflammatory processes, and cAMP signaling in the onset of labor is relatively well-understood, the key drivers of this process are yet to be characterized. We are developing algorithms to construct functional state descriptors using single cell data and making causal inference to identify key drivers of cell state change using these functional state descriptors. As part of our ongoing collaboration with Dr. Sam Mesiano (Case Western Reserve Dept. of Reproductive Biology), we will use this dataset to identify functional state descriptors for cells representing different tissues, different cell types, and different times (labor vs. non-labor, term vs. pre-term).  We will then use counterfactual based causal inference to identify key drivers that drive state change. The high-resolution provided by this dataset will enable application of causal inference algorithms from observational data.\",\n\"Aug24, 2020\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kragstrup, Tue \",\n\"UNIVERSITY OF AARHUS\",\n\"JAK-STAT pathways in venous thromboembolism\",\n\"closed\",\n\"JAK inhibitors are drugs targeting the JAK-STAT pathway. These drugs have proven effective in the treatment of several immune mediated inflammatory diseases such as rheumatoid arthritis and psoriatic arthritis. Drug adverse effects are recognized both during clinical trials and post marketing. Side effects of the JAK inhibitors during the clinical trials were herpes zoster. Post marketing surveillance has now warranted a warning on several of the JAK inhibitors for thromboembolic events. These side effects are not fully understood. This study will investigate whether there is a genetic basic for the putative implication of JAK-STAT signaling in the development of VTE. We expect that this study will help improve the use of JAK inhibitors and drive development to explore drugs with less side effects.\",\n\"Background: Venous thromboembolism (VTE) is a collective term for deep vein thrombosis (DVT) and pulmonary embolism (PE). Janus kinase (JAK) inhibitors are a group of small molecules that block one or more of the intracellular tyrosine kinases: JAK1, JAK2, JAK3 and TYK2. These compounds can block different cytokine signalling pathways, and they therefore have an immunosuppressive effect. JAK inhibitors are used in treatment of autoimmune and rheumatological diseases, such as rheumatoid arthritis, psoriasis, inflammatory bowel disease, and ulcerative colitis. Drug adverse effects are recognized both during clinical trials and post marketing. Side effects of the JAK inhibitors during the clinical trials were herpes zoster. Post marketing surveillance has now warranted a warning on several of the JAK inhibitors for thromboembolic events. In 2017 The European Medicines Agency revised the summary of product characteristics for Baricitinib to include PE and DVT. In 2019 The US FDA issued a \\u00e2\\u0080\\u009cblack box\\u00e2\\u0080\\u009d warning, that patients treated with Jakinibs had experienced thrombosis, including DVT and PE. These side effects are not understood. Therefore, investigating the mechanism behind the association between JAK inhibitor use and VTE is needed to keep JAK inhibitors attractive.  Aim: We want to use the requested dataset to assess the genetic burden in the JAK/STAT signaling pathway and downstream STAT transcription factor regulomes in VTE.  Methods: We will determine if gene sets containing JAK/STAT pathway genes and/or STAT target genes are enriched with VTE genetic risk using generalized gene-set analysis of VTE GWAS summary statistics with MAGMA (https://ctg.cncr.nl/software/magma). \",\n\"2021, Mar06\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Krishnamurthy, Ashok \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"Testing RADx Data Hub and correlating ILI reports againts COVID-19 cases\",\n\"approved\",\n\"Understanding the correlation between the number of reports of flu-like symptoms and counts of COVID-19 cases.\",\n\"I am co-PI on the RADx Data Hub Project, which is creating a cloud-based infrastructure for the discovery and analysis of data related to COVID-19 testing. I am interested in understanding how checks for Influenza Like Illnesses (ILI) predict the number of COVID-19 cases. I am primarily also testing if the workflow to request a RADx dataset works.\",\n\"2022, Nov16\",\n\"phs002538.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Li, Jingjing \",\n\"UNIVERSITY OF CALIFORNIA, SAN FRANCISCO\",\n\"Functional Validation of Genetics Variants in Spontaneous Preterm Birth Using Single-Cell Genomic Data\",\n\"approved\",\n\"Preterm birth (PTB) affects ~11% newborns in the nation, and has a strong genetic basis. We have recently identified many new genes implicated in PTB, and we herein request this single-cell gene expression data to study the biological functions of our newly identified PTB-associated genes. Specifically, we will ask what specific cell types these genes expression, and whether these genes will be dysregulated in PTB clinical samples. This study will help identify novel mechanisms in PTB.\",\n\"Preterm birth (PTB) affects 15 million births worldwide annually, which is the leading cause of neonatal mortality and morbidity. PTB has a strong genetic component; however, its genetic basis has been largely unclear. We recently studied large-scale genome-wide association study (GWAS) datasets, developed a set of novel algorithms to integrate GWAS and many other functional genomic datasets, and identified candidate loci in PTB. As an immediate step, we need to experimentally validate these associated genes for their roles in PTB. We therefore request this dataset to study gene expression of the placenta at a single-cell resolution. For our identified genes in PTB, we will use this dataset to identify their expression specificities in particular cell types, and also identify their differential expression in PTB. This study will identify novel genetic elements in PTB, and will help foster developing prognostic and treatment strategies for PTB.\",\n\"Feb06, 2020\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Li, Bin \",\n\"UNIVERSITY OF SCIENCE AND TECHNOLOGY OF CHINA\",\n\"Transcriptional landscape of the human placenta in ageing\",\n\"expired\",\n\"Older women are at high risk of miscarriage. Therefore, studying the changes in the maternal-fetal interface during aging is important to reveal age-related risks. We plan to apply the data from this application (phs001886.v1.p1) to this study to reveal changes in the placenta during aging.\",\n\"Exploring the cell type composition and function of human placenta is of great significance for studying the conditions required for a successful pregnancy. The risk of miscarriage in older women is very high. At present, no one has compared the transcriptome changes in the placenta of women during aging. We plan to sequence the transcriptome of the placenta of the young and old mothers and compare their differences. We intend to use the applied data (phs001886.v1.p1) as a control group.\",\n\"2021, Aug13\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Levy, Daniel \",\n\"NIH\",\n\"Using MVP GWAS results for co-localization and mendelian randomization analysis\",\n\"approved\",\n\"To use genetic results from MVP analysis in conjunction with our genetic results from Framingham to identify important genetic signals for cardiovascular disease and its risk factors.\",\n\"We seek to use MVP GWAS results across available cardiometabolic traits to conduct co-localization and mendelian randomization analysis.\",\n\"2019, May30\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Levy, Daniel \",\n\"NIH\",\n\"NHLBI TOPMed Whole-genome Sequencing Program (TOPMed WGS)\",\n\"approved\",\n\"The goals of the TOPMed Whole Genome Sequencing program are to identify genetic risk (and protective) factors for disease, to identify subtypes of disease, and to develop targeted and personalized treatments for disease.  The program consists of multiple studies addressing these goals for phenotypes focused on heart, lung, blood and sleep disorders, which have a high impact on public health.  These studies will include individuals from diverse populations, including those historically under-represented in health research.\",\n\"Phenotypic and genomic data from TOPMed studies will be used for: (a) investigating genotype-phenotype associations; (b) variant summary statistics, (c) common controls in association tests, (d) reference panels to impute into un-sequenced individuals from TOPMed; and (e) population genetic studies, f) integrative genomic studies. TOPMed data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Data from an individual with disease-specific consent will not be used in analyses outside of that restriction, including indirect uses such as imputation reference panels, common controls for association studies and variant summary statistics. For example, an individual with consent restricted to lung function studies will not be used to impute data intended for analyses of diseases not clearly related to lung function.  When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish or otherwise broadly share any findings from this study with the scientific community. This application anticipates collaboration among TOPMed investigators from multiple institutions, who are named in a master list appended to the IRB document uploaded with this application. Within a set of applicants approved for access to the TOPMed Exchange Areas, data may be shared in a cloud environment or by direct transfer, provided that they all agree among themselves to allow such sharing, and that all have approved TOPMed Data Access Requests for the specific data sets to be shared. Otherwise, data will be shared only among Internal Collaborators \\u00e2\\u0080\\u0093 i.e. within an institution. Data from TOPMed may be combined with non-TOPMed data from the same or other studies (obtained from dbGaP or other sources), to improve the power for novel genetic discoveries, while respecting the consent of all participants. We expect that this activity creates no additional risks to participants. \",\n\"2021, Apr07\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Levanon, Erez \",\n\"BAR-ILAN UNIVERSITY\",\n\"A-to-I RNA Editing Levels in COVID-19 Infected Individuals\",\n\"approved\",\n\"Coronavirus disease 2019 (COVID-19) is a contagious disease caused by an infection with SARS-CoV-2. RNA editing is a mechanism that enables RNA modifications in the cell, mainly carried by the ADAR1 enzyme in humans. ADAR1 is also tightly linked to the immune system activation against viruses.  Our group investigates RNA editing levels in infected COVID-19 individuals. So far we discovered increased RNA editing levels in blood cells of COVID-19 patients compared to controls, and we would like to further investigate these findings in infected cells taken directly from the respiratory system. Our results will be published in public.\",\n\"Coronavirus disease 2019 (COVID-19) is a contagious disease caused by an infection with SARS-CoV-2. All viruses require host cellular machinery for their RNA and protein synthesis. The presence of viral RNA molecules inside host cells can activate different host immune components.   RNA editing is a mechanism that enables RNA modifications in the transcriptome. In humans, Adenosine-to-Inosine (A-to-I) is considered to be the most abundant RNA editing process, carried by the Adenosine Deaminase Acting on RNA (ADAR) family of enzymes, particularly by ADAR1.Furthermore, it is known that ADAR1 is tightly linked to Interferon and the immune system activity against viruses.    So far, our group found increased RNA editing levels by ADAR1 in blood cells of COVID-19 patients compared to controls. In other words, we found that SARS-CoV-2 has a substantial impact on host RNA. We assume that such an effect may be even greater in infected cells of the respiratory system. We truly believe that our research will contribute to further understanding the pathogenesis of COVID-19, and guarantee to publish the results to the scientific community.\",\n\"Mar03, 2022\",\n\"phs002258.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Lei, Wei \",\n\"AFFILIATED HOSPITAL OF SOUTHWEST MEDICAL UNIVERSITY\",\n\"post-GWAS analysis for smoking traits\",\n\"closed\",\n\"I want to perform the post-GWAS analysis for smoking traits, and based on the summary statistical results from smoking traits to conduct the gene-based analysis. Subsequently, combine the results of gene-based analysis and brainspan dataset to find the significant subnetworks associated with smoking traits.\",\n\"We plan to use the summary statistics from smoking traits to i) run gene-based analyses, e.g. to uncover significant genes with smoking traits and ii) combine the results of gene-based analysis and brainspan dataset to find significant networks associated with smoking traits. The preliminary plan of analyses is, 2-step. First, we want to run gene based analysis using summary statistics from smoking traits based on the VEGAS program. Second, integrate the results of gene based analysis and coexpression network to find the significant subnetworks associated with smoking traits.\",\n\"2021, Apr13\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Lei, Wei \",\n\"AFFILIATED HOSPITAL OF SOUTHWEST MEDICAL UNIVERSITY\",\n\"Post-GWAS analyses on the genetics of addictions, medical and psychiatric disorders, and behaviors\",\n\"approved\",\n\"Differences in genetic makeup contribute to variations in behavior, health, and diseases. We will use the summary information derived from the Million Veteran Program\\u00e2\\u0080\\u0099s genome-wide association studies (GWASs) to help understand the genetic basis of addictions, medical and psychiatric disorders, and behaviors.\",\n\"Differences in genetic makeup contribute to variations in behavior, health, and diseases. The Million Veteran Program (MVP) has generated rich data, including genome-wide association studies (GWASs)\\u00e2\\u0080\\u0099 summary statistics. We will use MVP GWAS summary statistics to conduct post-GWAS analyses, such as (but not limited to) computing genetic correlations, generating polygenic scores, converting to the gene-based association, conducting Mendelian randomization, carrying out meta-analyses, or performing pathway or network analysis to examine the genetic basis of behavior, health, and diseases, including addictions, medical and psychiatric disorders, and behaviors. We will analyze MVP GWAS summary statistics independently, combine them with GWAS summary statistics for traits from other sources, or link to gene expression data such as the Genotype-Tissue Expression (GTEx) data. We will not combine datasets but integrate the MVP GWAS summary statistics with summary information for other traits. Our overarching objective is to decipher the biology or pathways of behavior, health, and diseases through genetic summary information and gene expression data.\",\n\"2022, Nov29\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Lee, Phil \",\n\"MASSACHUSETTS GENERAL HOSPITAL\",\n\"Investigating the common genetic control mechanisms of brain transcriptome for neurodevelopmental and neurological disorders \",\n\"closed\",\n\"Genome-wide genetic studies of psychiatric disorders have yielded important findings of disease susceptibility loci in recent years. Nevertheless, the biology underlying complex brain disorders is still elusive at best. In the present study, we examine the influence of genetic variability in human cortical gene expression on the psychopathology underlying major neurodevelopmental and neurological disorders. \",\n\"This study investigates the etiologic role of expression quantitative trait loci (eQTLs) markers in neurodevelopmental and neurological disorders. We will first identify brain-specific eQTLs using publicly available gene expression (from GEO) and genome data (obtained through dbGaP) of healthy controls. We will take a meta/mega analytic strategy to examine the effects of brain eQTLs on various brain disorders, including substance abuse and mental health phenotypes. \",\n\"Aug20, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Lee, Phil \",\n\"MASSACHUSETTS GENERAL HOSPITAL\",\n\"Genetic effects shared across major brain disorders and ends-phenotypes \",\n\"approved\",\n\"Genetic variation influences the differences in human traits and health outcomes. We will use dbGAP datasets to improve our understanding of how genetic variation confers shared and unique effects to complex human traits, brain disorders, and related health outcomes. \",\n\"The goal of this proposal is to understand how genetic variation influences the differences of complex human traits and susceptibility to brain disorders and related health outcomes. Recent successes of large-scale genomics studies, including both rare and common variations, have uncovered an un-expected level of genetic risk sharing across major psychiatric disorders at a genome-wide level. Nevertheless, little is known of which and how certain genetic loci carry risk effects transcending traditional diagnostic boundaries. To fill in this critical gap, we will use dbGAP datasets: (1) to identify specific genetic risk variants and biological mechanisms shared across major psychiatric disorders, somatic disorders, and related health outcomes; (2) to systematically evaluate the characteristics of the identified risk variants, genes, and their regulatory function; and (3) to advance our understanding of the genetic architecture and relationships of the studied phenotypes. The study design includes the meta-analysis of GWAS summary statistics and a range of genomic data analyses (e.g., variants to gene-mapping, gene expression data analyses, and polygenic scoring analyses). Our target phenotypes include major neuropsychiatric and neurological disorders, human behaviors, endophenotypes (e.g., neuroimaging), somatic disorders, and related health outcomes (e.g., pain, suicide) in diverse genomic samples.  \",\n\"Nov01, 2021\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Lai, Dongbing \",\n\"INDIANA UNIV-PURDUE UNIV AT INDIANAPOLIS\",\n\"Genetic study of complex diseases\",\n\"approved\",\n\"Complex diseases are caused by many genetic variants. Most of them have small effects and are not genome-wide significant in recent large-scale genome-wide association studies (GWAS). The purpose of this project is using a novel strategy developed by our group, which focuses on variants having the same direction of effects in all study cohorts, to identify small effect variants by using GWAS summary statistics from the Million Veteran Program and other large-scale GWAS.\",\n\"Recent large-scale genome-wide association studies (GWAS) have identified many disease-associated variants, however, for most complex diseases, they only explained a small portion of heritability. Many variants having small effects and thus sub-threshold P-values in large-scale GWAS remain to be identified. Further increasing the sample sizes to make all disease related variants genome-wide significant is not feasible in near feature. Study cohorts in large-scale GWAS are typically ascertained using different strategies and/or using different diagnostic methods. While this heterogeneity will generate many study-specific findings, for disease-associated variants, they will have the same direction of effects in all study cohorts (i.e., they are concordant), even they are not genome-wide significant. Furthermore, if these concordant SNPs can explain large proportion of heritability and polygenic risk scores calculated using them have high predictability, then they are likely to be truly disease-associated. In our previous works, we used this strategy and identified many small effect variants related to alcohol use disorder and Alzheimer's disease and related dementia in diverse populations. In this study, our objectives are using the same strategy to identify small effect variants associated with other complex diseases such as hypertension, type 2 diabetes, cardiovascular diseases, etc. We will use GWAS summary statistics from the Million Veteran Program and other large-scale GWAS of the same traits to identify concordant variants, then meta-analysis will be performed by using concordant variants only. Meta-analysis results will be used to estimate heritability and calculate PRS. We will not combine any dataset and findings will be tested in independent datasets. The identified variants will help us understand the genetic etiologies of complex diseases. \",\n\"Mar30, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Lai, Dongbing \",\n\"INDIANA UNIV-PURDUE UNIV AT INDIANAPOLIS\",\n\"Genetic study of alcohol use disorder and related phenotypes\",\n\"approved\",\n\"Studies have shown that alcohol use disorders and related disorders are caused by many genes with small effects. The purposes of this project are performing innovative analysis to identify these genes and investigating their interactions with other genes and/or environmental factors. We will also perform polygenic risk scores analysis to evaluate an individual's overall genetic risk on these diseases.\",\n\"Large-scale genome-wide association studies of alcohol use disorder (AUD) and related diseases such as other substance use disorders and psychiatric diseases have identified many risk variants/genes. However, these variants/genes only explained a small portion of the heritability. Majority variants/genes having small effects and thus having sub genome-wide significance P-values remain to be discovered. In this project, we aim at performing innovative analyses to identify sub-threshold variants/genes; then for those variants/genes identified, we will investigate whether they interact with other variants/genes and/or environmental factors. We will also perform polygenic risk scores analysis to evaluate an individual's overall genetic risk on these diseases. The finding of this project can help us understand the etiologies of AUD and related diseases, and shed light on the development of personalized prevention and treatment methods.\",\n\"2019, Jun28\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Labrecque, Nathalie \",\n\"HOPITAL MAISONNEUVE-ROSEMONT\",\n\"Application of a newly developed Notch T-cell specific transcriptional signature\",\n\"approved\",\n\"The immune system is devoted to fighting infectious agents. This is achieved by white blood cells that patrol the organism to eliminate pathogens. T cells, a type of white blood cells, are central players in an efficient immune response. These T cells recognize foreign materials via a cell surface receptor, the T cell receptor (TCR). This recognition allows for the expansion and differentiation of T cells that are specific to the infectious agent. This differentiation step is crucial to allow the T cells to eliminate the pathogen. We have previously shown that the evolutionarily conserved Notch receptor controls the differentiation of T cells. Identifying T cells that have active or inactive Notch signaling could be helpful to develop therapies where we manipulate Notch to control the immune response of T cells. In this proposal, we will determine whether a tool that we have developed, a T-cell specific Notch molecular signature, is detected in patients with multisystem inflammatory syndrome (MIS-C) following SARS-CoV-2 infection. These patients have been shown to have an affected Notch pathway. This would help us validate a tool that could simplify the identification of biological contexts where manipulation of the Notch pathway in T cells could be targeted for therapy.\",\n\"The evolutionary conserved Notch signaling pathway controls the differentiation fate in many cell types. In particular, our team has found that Notch plays a crucial role in the differentiation of effector CD8 T cells in the context of acute infections. Others have also described that the Notch pathway controls the CD4 T cell response during graft-versus-host disease (GVHD). Thus, the Notch pathway is important in the response of peripheral T cells. However, bioinformatic tools to identify Notch signalling often fail to identify the activity of this pathway in different T-cell responses. This is illustrated in a recent publication entitled \\\"Notch1-CD22-Dependent Immune Dysregulation in the SARSCoV2-Associated Multisystem Inflammatory Syndrome in Children.\\\" In this publication, the authors demonstrate an important role for Notch in Multisystem Inflammatory Syndrome, but the scRNA-seq data does not identify this pathway (see Fig S1). We propose that this is because the currently available Notch signatures in public databases are derived from tumour-derived data and Notch signalling in peripheral T cells regulates a distinct set of genes.   To address this issue, we developed our own Notch signature specific to activated peripheral T cells. We have validated this signature in several contexts of Notch deficiency or overexpression in mice. We have also shown that this signature is specific to T cells, as there was very little overlap with currently available Notch signatures developed from cancer cell lines.   We wish to evaluate whether our newly generated Notch signature is enriched during different types of human T-cell responses. In the above-mentionned study of Benamar M et al. (phs003086.v1.p1), single-cell RNA-seq was used to identify robust Notch signaling in Tregs of patients having developed multisystem inflammatory syndrome (MIS-C) following SARS-CoV-2 infection. Our objective is to determine whether our T-cell Notch Signature is enriched in these patients\\u00e2\\u0080\\u0099 Tregs. To this end, the scRNA-seq will be analyzed using Seurat v 4.0.0. The top 3000 variable genes will be used for PCA calculations and the first 25 PCs will be used for UMAP and for cluster identification (obtained with Louvain algorithm with a resolution of 2). Enrichment of the T-cell Notch Signature on each individual cell will be measured with the AddModuleScore function from Seurat. We will not be evaluating phenotypic characteristics or genetic variants of the participants?\",\n\"Apr30, 2023\",\n\"phs003086.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"LOGUE, MARK \",\n\"VA BOSTON HEALTH CARE SYSTEM\",\n\"Early Cognitive Impairment as a Function of Alzheimer\\u00e2\\u0080\\u0099s Disease Genes and Trauma\",\n\"approved\",\n\"In this project we will examine Alzheimer\\u00e2\\u0080\\u0099s disease (AD) genetic risk and its relationship to cognitive impairment in middle and old age to determine the extent to which effects of AD genetic risk are observable in advance of the onset of dementia. We will also evaluate whether AD genetic risk influences PTSD symptomatology in aging veterans.  Finally, will look for evidence that combat trauma and traumatic brain injury (TBI) exacerbate the effects of AD genetic risk on AD. Early detection of AD risk has been noted as key to the identification of effective treatments for AD. Hence, this project has the potential to lead to better informed and more effective AD interventions. By examining how genetic factors interact with TBI and trauma we hope to pave the way for the development of diagnostic algorithms specifically tailored to a veteran population designed to identify those at risk for dementia\",\n\"We desire to obtain GWAS summary statistics and to  examine  results for specific variants and compute polygenic scores based on GWAS results based on GWAS summary statistics to examine the applicability of our GWAS results to other traits. \",\n\"2021, Sep09\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kullo, Iftikhar \",\n\"MAYO CLINIC ROCHESTER\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease. Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data.\",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry. Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community. However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets. \",\n\"2022, Dec15\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Kullo, Iftikhar \",\n\"MAYO CLINIC ROCHESTER\",\n\"Development and validation of polygenic risk score-based risk models for coronary heart disease in diverse racial/ethnic groups\",\n\"approved\",\n\"Risk scores that are made up of genetic markers in the human genome, i.e. polygenic risk scores (PRS), have been shown to identify individuals at 2-3 times higher risk of heart disease. However, application of these scores to racial/ethnic minorities is challenging since PRS were developed using white participants. Genetic differences across racial/ethnic groups require development/validation of PRS specifically for minority racial/ethnic groups to achieve the same level of performance. In this study, we aim to use dbGaP cohorts to develop/validate PRS specifically tailored for minority groups. This will allow accurate characterization of the genetic risk of heart and blood vessel diseases in the United States and better risk estimations for ethnically diverse groups.\",\n\"Since polygenic risk scores (PRS) for coronary heart disease (CHD) are derived from mainly European ancestry (EA) cohorts, there is an urgent need for the development of race/ethnicity specific PRS and validation of EA-derived PRS in minority groups such as African ancestry (AA) and Hispanic ethnicity (HE) individuals. This study is being submitted by the investigators involved in the NHGRI-funded electronic MEdical Records and Genomics (eMERGE) network. Our primary objective is to use dbGaP cohorts with participants of non-EA ancestry to validate PRS-based relative and absolute risk models derived using the eMERGE Phase III cohort for CHD  Genetic data (genotyping array, whole genome sequencing etc.) will be used to compute PRS for CHD. In addition to the genetic data, phenotypic data such as age, sex, serum lipid measurements, blood pressure measurements, history of diabetes mellitus, hypertension, use of lipid lowering and/or anti-hypertensive therapy, smoking status as available will be incorporated into the prediction models to fully account for both genetic and conventional risk factors. Models developed using eMERGE data will be tested in dbGaP cohorts to assess the strength of association of each risk factor, discriminatory ability, and calibration of the model in each racial/ethnic group. These models will be refined based on the assessment using dbGaP cohorts.       \",\n\"May20, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Isles, Anthony \",\n\"CARDIFF UNIVERSITY\",\n\"Investigating imprinted gene expression in the placenta at the single cell level\",\n\"expired\",\n\"We wish to study the activity of a group of genes called imprinted genes in the different cells of the placenta. We hope to use this dbGaP Single Cell Analysis of the Human Placenta dataset to identify in which specific cells of the placenta activity of imprinted genes is highest. This will give us an understanding of the possible role imprinted genes play in the placenta, and also hopefully help us understand what might be going wrong in disorders linked to imprinted genes, such as Prader-Willi syndrome. \",\n\"Genomic imprinting is the process whereby epigenetic imprints, established in the germline of the mother and father, survive epigenetic reprogramming in early development and are inherited by the somatic cells of the offspring. These imprints distinguish the maternally- and paternally-derived genomes and ultimately lead to some imprinted genes being expressed from the maternal allele only; whilst others are expressed from the paternal copy only. Imprinted genes have provided a template for understanding epigenetic processes throughout development generally. However, from a functional standpoint, although small in total number (~200 in the genome) imprinted genes, which are only found in mammalian animals (they are found in plants too), also pose a number of evolutionary questions and are also linked to several disorders, including Prader-Willi syndrome. My lab is interested in understanding the function of imprinted genes, and this project in particular takes advantage of a the growing number of single RNA-seq datasets to examine the enrichment of canonical imprinted gene expression as a group in the mouse and human tissues. These analyses identify \\u00e2\\u0080\\u0098expression hotspots\\u00e2\\u0080\\u0099 where strongly expressed imprinted gene are significantly over-represented in the transcriptome (i.e. more than by chance alone), which in turn suggests \\u00e2\\u0080\\u0098functional hotspots\\u00e2\\u0080\\u0099. We would like access to the dbGaP Single Cell Analysis of the Human Placenta dataset (phs001886.v2.p1) in order to examine imprinted gene expression and enrichment in the placenta. Although there are a number of studies examining imprinted gene expression in bulk placenta samples, placental cell type-specific RNA references are limited. Our analyses will involve screening the transcriptomes of different cell types (as identified and defined in Pique-Regi et al  eLife 2019;8:e52004) for the presence of imprinted gene expression, then comparing the relative proportion of imprinted genes in each cell type to establish if these genes, as a group, are enriched in any cell types specifically (see https://doi.org/10.1101/2020.07.27.222893 for methodological details).\",\n\"May24, 2021\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Irvin, Marguerite \",\n\"UNIVERSITY OF ALABAMA AT BIRMINGHAM\",\n\"Polygenic risk scores for chronic kidney disease in African Americans\",\n\"approved\",\n\"We are using the summary level genetic data to inform personalized risk for kidney disease in African Americans.  \",\n\"Genetic background contributes to risk for developing chronic kidney disease (CKD). The objective of this study is to develop polygenic risk scores (PRS) for CKD and explore their capacity to predict incident kidney failure in cohorts of African Americans (AAs). We will use summary statistics from a genome-wide association study (GWAS) of estimated glomerular filtration rate (eGFR) conducted by Hellwege et al. within the Million Veteran Program (MVP) to train PRS for CKD in ~1,900 individuals with whole genome sequenced (WGS) data in the Hypertension Genetic Epidemiology Network (HyperGEN). CKD status will be defined as having a baseline eGFR <60 ml/min/1.73m2, and incident kidney failure will be defined as having a follow-up eGFR <15 ml/min/1.73m2 or dialysis or kidney transplantation among individuals without kidney failure at baseline. We will use the PRScs software to train the PRS in HyperGEN, and we will develop risk models with additional adjustment for age, sex, and comorbidities associated with kidney injury (e.g. hypertension, diabetes). We will then evaluate the predictive performance of these risk models (e.g. c statistic), and the best performing risk algorithm (PRS + covariates) will be selected for validation in AAs with WGS data from the Jackson Heart Study (JHS, approved) and Women\\u00e2\\u0080\\u0099s Health Initiative (proposal under review). Once these analyses are completed, the validated risk model will be tested for its capacity to predict incident kidney failure in the REGARDS, HyperGEN, and JHS cohorts.\",\n\"2022, Jun10\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ingelsson, Erik \",\n\"STANFORD UNIVERSITY\",\n\"Molecular studies of cardiovascular disease, type 2 diabetes and related conditions\",\n\"closed\",\n\"We are broadly interested in what leads to cardiovascular diseases, type 2 diabetes and related conditions. We combine different methods, including large-scale studies in human populations, with studies in model systems including cell cultures and murine models. Our research aims at increasing understanding of cardiovascular diseases, type 2 diabetes and related conditions; to personalize treatment of patients with risk of these conditions; and to find new drugs to treat them. We ask permission to use data from dbGAP for our projects, which include discovery of novel genetic determinants and studies of causal factors of these diseases; prediction of new uses and side effects of drugs, with a focus on cardiovascular diseases, type 2 diabetes and related conditions using genetic methods; prioritization of genes for further studies in model systems by combination of genetic and gene expression data; and development of new methods to analyze molecular data.\",\n\"Our research program is broadly interested in the etiology of cardiovascular diseases (CVD), type 2 diabetes (T2D) and related conditions (such as obesity, dyslipidemia, insulin resistance) and risk factors (such as lifestyle factors and other environmental exposures). Our research is translational and interdisciplinary, combining large-scale population-based research - including genomic, metabolomic, transcriptomic, epigenomic and proteomic studies in large population-based studies - with functional work in model systems to reach new insights into the pathophysiology of cardiovascular disease and related conditions, identification of new biomarkers for improved risk prediction, and discovery of novel targets for drug development. Dr. Ingelsson has had a leading role in many of the large efforts identifying new loci associated with cardiovascular and metabolic traits, and has extensive experience from research on biomarkers and -omics methods, including development and application of prediction metrics and Mendelian randomization. Since 2014, he has also built a team working with characterization of loci established in GWAS using gene editing methods in model systems. The present proposal has several aims: (1) Discovery and characterization of molecular determinants of CVD, T2D and related conditions; (2) Studies of risk factors for these conditions using both traditional observational analyses and Mendelian randomization approaches; (3) Studies of loss-of-function variation in drug target genes to predict (a) potential side effects (using phenome-wide approaches) of drugs aimed at treating CVD, T2D or related traits; or (b) potential for drug repositioning to CVD- or T2D-indications of drugs used for other diseases; (4) Prioritization of candidate genes for functional studies using eQTL analyses; and (5) Development and application of novel methods to analyze molecular data in relation to CVD, T2D or related traits. We will publish or otherwise broadly share any findings from our studies arising from use of this data with the scientific community. We will honor all embargo dates associated with any of the publicly available datasets.\",\n\"2019, May15\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Golod, David \",\n\"GOLDFINCH BIOPHARMA, INC.\",\n\"Variant discovery in kidney disease\",\n\"expired\",\n\"Goldfinch Bio plans to utilize genomic variant data from large numbers of control subjects to confirm the potential pathogenicity of genomic variants and signatures found in kidney patient samples. These data will provide a means to identify dysregulated pathways in patients, new targets for therapeutic intervention, and characterize sub-sets of patients that are likely to respond to targeted molecular therapeutics.\",\n\"The objective of this study is to identify rare variants in kidney disease.  We are currently sequencing 2000 cases with nephrotic syndrome, as indicated by proteinuria.  Nephrotic syndrome can be caused by chronic disease such as hypertension and diabetes. To detect rare variants associated with nephrotic syndrome, we need whole genome sequences (WGS) from control populations to compare variant frequencies.  Because kidney disease is more prevalent in populations of African ancestry, we need controls of diverse ancestry.  We are also particularly interested in samples from population-based cohorts, to get the most accurate measures of population allele frequencies.  Thus, we will be combining all data sources into several analyses using a case-control design.  First, we will perform an unbiased common variant (> 0.5%) association study (i.e. GWAS) in each ancestry population, and then estimate overall effect sizes by meta-analyzing across all ancestry populations.  Second, we will search for rare variants (< 0.5%) with burden analyses, increasing power by filtering variants along several dimensions: class (disruptive, severity of missense), frequency, and within gene sets of interest.  Combining these datasets will not create any additional risks to the participants.  All raw data will be securely stored and will comply with the data use restrictions of each dataset.\",\n\"Feb18, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Glass, Christopher \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Integration of GWAS disease risk variants with with epigenomic annotations derived from myeloid cells (monocytes, monocyte-derived macrophages, microglia)\",\n\"closed\",\n\"Most GWAS signals reside in noncoding regions of the genome and often these regions harbor gene regulatory regions. Often it is unclear what specific cell type and what specific genes are affected by these disease risk variants. Previously we showed, that microglia harbor a majority of Alzheimer disease risk variants in their distal gene regulatory regions. Here, we propose to provide a catalogue of active regulatory regions in cells of the myeloid cell lineage (monocytes, monocyte-derived macrophages, and microglia) and their associations with disease risk variants identified by GWAS. We aim to extend our previous efforts to provide a catalogue of gene regulatory elements specific for myeloid cells for various disorders, ranging from neurodegenerative, cardiovascular, metabolic, and autoimmune diseases. To this aim, we will use GWAS summary statistics and overlay these with our comprehensive next-generation sequencing datasets (PLAC-seq, RNA-seq, ATAC-seq, ChIP-seq). \",\n\"The objective of this project is to integrate disease risk variants obtained from GWAS studies involving neurological diseases and peripheral diseases (autoimmune, metabolic and cardiovascular diseases) with epigenomic annotations derived from myeloid cells (monocytes, monocyte-derived macrophages, microglia) as well as neutrophils and T cells.   Here, we propose to integrate GWAS signals from neurological, metabolic, cardiovascular and autoimmune diseases) with (epi-) genomic datasets (RNA-seq, ATAC-seq, H3K27ac ChIP-seq, and PLAC-seq) derived from immune cell types as mentioned above. This will enable us to link genetic variants residing in distal regulatory regions of myeloid cells to gene regulation and possibly to disease susceptibility. We will determine the enrichment of genetic variants associated with complex traits in cell type-specific regulatory regions, we implement a linkage disequilibrium score (LDSC) regression analysis of heritability.  Taken together, our approach will advance the understanding of the genetic bases of the myeloid cell lineages are their linkage to systemic diseases.  We will combine dbGaP, UKBiobank and GWAS Catalog datasets. There are currently no planned collaborations with researchers at other institutions.  Data obtain from dbGaP is used for general research and health/medical/biomedical purpose. Results generated by this study will be made available for the larger scientific community. All obtained summary results from the below mentioned studies is not used to identify individuals or groups and is used only for the evaluation of SNP associations with the phenotypes provided. Use of eMERGE Network Imputed GWAS for 41 phenotypes (phs000888.v1.p1) will not include the study of population origins or ancestry. \",\n\"Dec22, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Gillespie, Nathan \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"The Structure of Genetic Risk Factors for Major Psychiatric Disorders v4\",\n\"approved\",\n\"By combining the aggregated results from large genome-wide association scans (GWAS) of psychiatric disorders and complex behaviors will will determine the overall size of the genetic correlations between them. \",\n\"Objective: Recent advances in genome-wide association scans (GWAS) of psychiatric disorders and statistical innovations now permit researchers to revisit hypotheses regarding the genetic structure of psychiatric disorders and sources of comorbidity. We will determine if the genetic structure observed in family studies between Externalizing, Internalizing (including anxiety) and Psychotic spectrum disorders can be confirmed using statistical methods that analyze GWAS-based genetic correlations. Methods: We combined publicly available GWAS summary statistics for 12 major psychiatric disorders (including the results based on the MVP analysis of anxiety), six substance use measures and two personality domains. Using the genomicSEM software package, we will estimate GWAS-based genetic correlations between all 12 disorders and 8 traits before applying confirmatory factor analysis (CFA) to identify the genetic structure.\",\n\"2022, Dec22\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Gevaert, Olivier \",\n\"STANFORD UNIVERSITY\",\n\"Image-based biomarkers for radiogenomics of COVID19\",\n\"approved\",\n\"Effective COVID-19 diagnosis and treatment rely on the information from multiple patient tests involving clinical, molecular, imaging, and genomic data. Radiomics, an expansion of computer-aided diagnosis, is a growing effort that involves these computerized image analyses in attempt to further relate quantitative image data to other \\u00e2\\u0080\\u009c-omic\\u00e2\\u0080\\u009d data such as clinical, pathologic, and genomic data. The integration of imaging data with genomic data allows for the study of associations between the imaging findings and the genomic measurements of the same disease site. Using the MIDC datasets of COVID-19, we will investigate relationships between computer-extracted quantitative radiomic COVID-19 features and various clinical, molecular, and genomics markers of prognosis and risk of recurrence, including gene expression profiles. We will assess the predictive ability of the quantitative COVID-19 CT/CXR features relative to multiple tasks: clinical stage, risk of recurrence, and genomics.\",\n\"Effective COVID-19 diagnosis and treatment rely on the information from multiple patient tests involving clinical, molecular, imaging, and genomic data. Radiomics, an expansion of computer-aided diagnosis, is a growing effort that involves these computerized image analyses in attempt to further relate quantitative image data to other \\u00e2\\u0080\\u009c-omic\\u00e2\\u0080\\u009d data such as clinical, pathologic, and genomic data [Zhu 2015]. The integration of imaging data (radiomics) with genomic data, referred to as \\u00e2\\u0080\\u009cimaging-genomics\\u00e2\\u0080\\u009d or \\u00e2\\u0080\\u009cradiogenomics\\u00e2\\u0080\\u009d, allows for the study of associations between the radiomic phenotypes and the genomic measurements of the same disease site. Radiomic phenotypes that are highly correlated with important clinical, molecular, or genomic biomarkers can potentially serve as diagnostic or prognostic tools for patient monitoring and assessing therapeutic response, and thus augment the utility of medical imaging as a non-invasive technology for COVID-19 care, like a \\u00e2\\u0080\\u009cvirtual digital biopsy\\u00e2\\u0080\\u009d. Using the MIDC datasets of COVID-19, we will investigate relationships between computer-extracted quantitative radiomic COVID-19 features and various clinical, molecular, and genomics markers of prognosis and risk of recurrence, including gene expression profiles. We will assess the predictive ability of the quantitative COVID-19 CT/CXR features relative to multiple tasks: clinical stage, risk of recurrence, and genomics. Aims: 1. To create a multi-omics dataset of cases, including radiomics of COVID-19 and non-COVID-19 patients from chest x-rays and/or CTs along with their molecular and genomic markers, which will be made openly available for discovery by computational biologists and machine learning experts. MIDRC has the chest x-ray image data corresponding to the EHR data in RED CORAL: PETAL Repository of Electronic Data COVID-19 Observational Study. Our first goal is to test the interoperability of MIDRC and the Biodata CATALYST. We will link the imaging data with the EHR data and investigate methods for use of data from both sources for machine learning development and assessment. 2. To determine if these quantitative radiomics (virtual biopsy) of COVID-19 CT/CXR can serve as a surrogate biomarker for assessing the molecular environment, enabling translation of virtual biopies.  Push/Pull Relationships: The multi-institutional chest radiograph and CT datasets of the MIDC will enable the rapid investigation of radiogenomics for prognosis, prediction, and monitoring of response of the COVID patient. This project will deliver discovery findings for COVID-19 lungs to expedite the understanding and development of machine intelligence as well as a clinical tool.\",\n\"2022, Oct04\",\n\"phs002363.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Garvin, Michael \",\n\"UT-BATTELLE, LLC-OAK RIDGE NATIONAL LAB\",\n\"Genome-wide association study of heroin abuse using public controls\",\n\"closed\",\n\"Approximately 50% of the risk for heroin abuse is attributable to genetic factors. However, few genes have been consistently associated with heroin abuse, and it is likely that a number of genes contributing to heroin abuse have not been identified. The proposed study would test associations between heroin abuse and genetic variants across the genome, with the goal of uncovering biological mechanisms that underlie drug addiction. More broadly, we also examine genetic risks for substance use disorders regardless of specific drug used.  \",\n\"The goal of the proposed study is to identify novel genetic determinants of heroin abuse and related substance use disorder (SUD) phenotypes by conducting genome-wide association (GWA) studies as well as candidate variant studies for SUD phenotypes.  Although GWA studies have successfully identified genetic variants contributing to other SUD phenotypes (e.g. nicotine dependence), there are few GWA studies of rarer high-risk substance abuse.  We propose to use a case-control study design, drawing heroin abuse cases from the Urban Health Study (UHS) which includes >12,000 people who inject drugs (PWID) enrolled from 1986 to 2005 (Kral et al., 2001).  We have recently completed genotyping of over 10,000 PWID samples on Illumina arrays (Omni1-quad and 2.5M) under grant R01 DA044014 (PI Eric Johnson).  Of these PWIDs, 66% are heroin abuse cases who injected 10+ times in the past 30 days (ONDC criteria of abuse). This cohort is 53% African American, 37% European American, and 10% Hispanic American. We use dbGaP studies genotyped on Illumina arrays to match to the UHS cases as population controls generating a UHScases+dbGaPcontrols cohort for GWAS. We have published two studies using this approach (Johnson, et al, 2015; Hancock et al. 2015). We use additional dbGaP studies with SUD phenotype data to test for replication of findings from UHS+dbGaPcontrols cohort or in meta-analyses. In the current period we will continue to use these previously acquired cohorts and add dbGaP studies as population controls for our newly genotyped UHS cases to study SUD phenotypes. We will also request new dbGaP studies with SUD phenotype data for replication and/or meta-analyses as they become available.  We will continue to participate in the Addiction group of the Psychiatric Genetic Consortium (PGC): analyzing these data in large meta-analyses with other investigators.  PGC quality control and analysis are being conducted at the MGH Simches Research Center \\u00e2\\u0080\\u0093 Harvard University. We are also leading the NIDA Genetics Consortium GWAS of Opioid Addiction, which combines summary statistics from 13 cohorts. As part of NIDA Smoke Screen Initiative, a number of new cohorts that are informative for heroin addiction have been genotyped on the Axiom_Smokesc1 array. Most of these cohorts are case only. To leverage these data we will need to extend our population control approach to include matching controls derived from whole genome sequencing (WGS) to array genotyped cases given the very low overlap between the Axiom_Smokesc1 array and other Axiom and Illumina array. We will develop a new protocol to match TopMed WGS cohorts as population controls to the Axiom_Smokesc1 array genotyped cases; the WGS data containing a complete overlap with the Axiom_Smokesc1. All results from the analyses using these data will be published and summary statistics shared. \",\n\"Jun22, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Gandal, Michael \",\n\"UNIVERSITY OF CALIFORNIA LOS ANGELES\",\n\"Building integrative CNS networks for genomic analysis of autism and schizophrenia\",\n\"closed\",\n\"Advances in genomics and genetics have opened a new window through which to understand psychiatric diseases, such as autism spectrum disorder (ASD) and schizophrenia, but also reveal that they are all very genetically heterogeneous. Gene networks have shown promise for organizing the complexity and heterogeneity of genetic factors that contribute to ASD. Here, through highly collaborative efforts using multiple public and private data sets we develop a rigorous, multi-level and experimentally tested framework for integrative network studies in neuropsychiatric diseases.\",\n\"Large-scale genomic investigations have begun to illuminate the genetic contributions to major psychiatric illnesses including autism spectrum disorder (ASD) and schizophrenia. In ASD,  rare large effect variants provide novel causal anchors to understand its neurobiological basis, and to understand convergence and divergence in disease mechanisms. These findings, not unlike other psychiatric disorders, also emphasize extreme genetic heterogeneity. We and others have shown that gene and protein networks provide an organizing framework for understanding heterogeneous psychiatric disease genetic risk in a unified biological context. The emergence of large-scale genomic and epigenetic data from human brain, and growing knowledge of genetic variation involved in ASD and other psychiatric diseases, coupled with advances in methodology, provides an unprecedented opportunity for comprehensive integrative analyses. We bring together a collaborative group of investigators, each with distinct areas of expertise critical for understanding psychiatric diseases, that have not been combined before, to develop a framework for integrative genomic network analysis. The work proposed here aims to develop a comprehensive framework and test optimal methods for integration of gene expression and protein-protein interaction (PPI) networks in the brain with genetic and epigenetic data \\u00e2\\u0080\\u0093 networks that will be iteratively refined using experimental data. We will construct networks representing multiple brain regions in adulthood and development through rigorous combination of multiple transcriptomic data sets from ASD and control brain, developing and validating methods for integration of splicing and expression levels within gene networks. These networks will be refined to inform tissue specific PPI inference, validated via experimental tissue-specific PPI. We will further identify causal drivers by integration of genetic and epigenetic data, identifying QTL effects on RNA, splicing and protein levels. Completion of these aims will lead to more valid and comprehensive CNS networks thereby significantly advancing our understanding of ASD associated variants and causal neurobiological pathways. \",\n\"Jun18, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"GUSEV, ALEXANDER \",\n\"DANA-FARBER CANCER INST\",\n\"Investigating the relationship between germline and somatic variation in common cancers\",\n\"approved\",\n\"The mechanisms of cancer are driven by germline (inherited) variants as well as somatic variants that occur in the cell. Though specific examples exist for both types of variation acting as a cancer driver, the genome-wide interplay between these two features remains an important area of research. The goal of this project is to investigate the impact of germline genetic variation on patterns of somatic variation and cellular activity such as gene expression and methylation measured in tumor/normal cells. The project is divided into three main aims. First, we will identify specific germline variants that are associated (and likely driving) cellular activity. Second, we will assess whether these variants can explain the loci identified in large-scale studies of cancer risk. Third, we will assess whether cancer risk variants have a significant effect on patterns of somatic changes.\",\n\"***We will NOT analyze genetic ancestry in studies that do not allow analysis of population origins and ancestry and in accordance with the study data use limitations*** ***For studies restricted to specific phenotypes, we will only analyze GWAS data from those phenotypes in accordance with the study data use limitations.***  It is well understood that many cancers have a substantial genetic component, which can effect risk, response to exposure, cancer etiology, and even molecular mechanisms. The goal of this project is to investigate the impact of germline genetic variation on patterns of somatic variation and tumor-specific cellular activity. We intend to broadly share your results with the scientific community by publishing our findings.  1. We will assess the impact of germline variants on molecular phenotypes (expression, methylation, etc) in tumor and normal individuals. We will apply traditional QTL and allelic imbalance methods as well as emerging methods. We will develop new methods to identify variants that have an interaction with tumor/normal status or tumor sub-type.  Separately, we will quantify and partition the genome-wide heritability and genetic correlation of these molecular features, within and across different cancers.  2. We will investigate the mechanistic impact of features identified in Aim 1 on GWAS data from cancer and related traits (where allowed by the study). We will use the genome-wide QTL signal as an annotation to assess enrichment of GWAS associations and heritability. We will use novel statistical methodology to identify QTLs whose molecular features are likely to be mediating the GWAS association.  3. We will investigate the relationship between somatic variation and polygenic burden scores from cancer GWAS studies (for example, between extreme high and low risk individuals). Specifically, we will look for association between genetic risk score and the distribution of somatic mutations/alterations and tumor-specific expression/methylation in each sample. We will investigate if patterns of somatic variation lead to functional enrichment of GWAS association statistics and heritability.  IMPORTANT: None of the analyses require merging controlled data with external data and we will never do so. For studies restricted to specific phenotypes, we will only analyze GWAS data from those phenotypes in accordance with the study data use limitations:  * For study \\\"VA Million Veteran Program (MVP) (phs001672.v3.p1)\\\" data will only be used for health/medical/biomedical purposes and will not be used to study population origins or ancestry. * For study \\\"Functional Significance of Prostate Cancer risk-SNPs (phs000985.v1.p1)\\\" we will only use corresponding data to study prostate cancer and cross-reference against prostate cancer GWAS data. * For study \\\"Stockholm-Tartu Atherosclerosis Reverse Network Engineering Task (STARNET) (phs001203.v1.p1)\\\" data will only be used for health/medical/biomedical purposes and will not be used to study population origins or ancestry. * For study \\\"Exome recapture and sequencing of prospectively characterized clinical specimens from cancer patients (phs001783.v1.p1)\\\" data will only be used to study cancer and for methods development.\",\n\"2019, Dec30\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"GOATE, ALISON \",\n\"ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI\",\n\"Evaluating the Causal relationship between modifiable risk factors and Alzheimer\\u00e2\\u0080\\u0099s disease\",\n\"closed\",\n\"The goal of this project is to validate proposed causal relationships and identify new causal relationships between modifiable risk factors and the Alzheimer\\u00e2\\u0080\\u0099s endophonome using Mendelian Randomization analysis. \",\n\"Objectives: The overarching objective of this project is to enhance our understanding of the bidirectional causal relationships between lifestyle, health, psychosocial, biomarker, and disease outcomes on the Alzheimer\\u00e2\\u0080\\u0099s endophenome using Mendelian randomization analysis. Mendelian Randomization (MR) uses genetic variants as proxies for exposures to provide an estimate of the causal association between an intermediate exposure and an outcome. Identifying modifiable risk factors that are causally related to Alzheimer\\u00e2\\u0080\\u0099s disease can be used to inform lifestyle interventions.   Study design & Analysis Plan: Using summarized data from genome-wide association studies we will conduct Mendelian Randomization analysis between potentially modifiable risk factors and Alzheimer's disease. Exposures will consist of GWAS for health lifestyle psychosocial and disease traits. GWAS datasets for the Alzheimer\\u00e2\\u0080\\u0099s endophenome will consist of Late-Onset Alzheimer\\u00e2\\u0080\\u0099s disease, Alzheimer\\u00e2\\u0080\\u0099s age of onset (AAOS), amyloid beta, tau and phosphorylated tau levels in CSF, neuropathological burden of neuritic plaques and neurofibrillary tangles, and hippocampal volume. For each exposure, we will construct instrumental variables (IV) using genome-wide significant loci. Where SNPs are in linkage disequilibrium (LD), LD clumping will be performed by excluding SNPs that have an r2 > 0.1 with another variant with smaller p-value association within a 250kb window. For genetic variants that are not present in the outcome GWAS, we will use proxy SNPs (r2 < 0.8) where available. Causal estimates will be estimated using inverse-variance weighted fixed-effects meta-analysis. Sensetivity analyses will be conducted using weighted median, weighted mode, and MR-Egger regression analysis.  \",\n\"2018, Dec20\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"GIZER, IAN \",\n\"UNIVERSITY OF MISSOURI-COLUMBIA\",\n\"Comorbidities among mental and physical disorders using data from Genome-Wide Association Studies (GWAS)\",\n\"approved\",\n\"Individuals with substance use problems are at higher risk for developing a host of medical conditions and vice versa. We will use genetic data to better understand the causes of these increased risks.\",\n\"This study will seek to examine the genetic overlap between addiction-related behaviors (e.g., alcohol, tobacco, illicit drug use) and other mental (e.g. antisocial behavior, depression and anxiety) and physical disorders (e.g., chronic pain conditions) at both the aggregate and single variant level. We will use methods that use either genotype-level data or summary statistics to determine the genetic relations between these phenotypes. For those methods that rely on summary statistic, we will use to meta-analytic approaches to combine datasets, including those outside of dbGaP. All data security protocols will be followed and because the focus will be on summary statistics, this should not present additional risks to study participants. We will use univariate and multivariate analytic association methods to identify loci that may have pleiotropic effects on these phenotypes. Finally, gene-based and pathway-based analyses will be conducted in an attempt to identify the biological mechanisms that contribute to the phenotypic correlations between these conditions. \",\n\"Dec18, 2018\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"GETZ, GAD \",\n\"BROAD INSTITUTE, INC.\",\n\"Studying the cellular dynamics of response to CAR-T therapy\",\n\"approved\",\n\"CAR-T cell therapy has revolutionized the treatment of patients suffering from a wide class of cancers of the blood (leukemias) and lymph node (lymphomas). In this type of treatment, a patient\\u00e2\\u0080\\u0099s own immune cells are engineered to be able to recognize and kill the tumor cells. Despite their success, still about half of patients with advanced lymphomas ultimately relapse after treatment, and the reasons for this are not well understood. By sequencing the RNA from these CAR-T cells and other immune cells from lymphoma patients treated with this therapy, we identified the features of these cells that are characteristic of patients who do, or do not, respond, and show how these change across the course of treatment. \",\n\"Our study aims to understand the mechanisms of response to CAR-T therapy in patients with large B-cell lymphomas through single-cell RNA sequencing. A preliminary version of our study can be found on medRxiv (https://doi.org/10.1101/2022.04.04.22273422). We obtained a total of 105 samples, including pre- and post-treatment peripheral blood and infusion products, from 32 patients treated with the first two FDA-approved CD19 CAR-T products: axicabtagene ciloleucel (axi-cel) and tisagenlecleucel (tisa-cel). Through analysis of the CAR-T cell and other immune populations, we identify the key transcriptional features displayed by responders and non-responders to each product, as well as their dynamics across treatment timepoints. To characterize T-cell subpopulations in our data, we labeled cells based on expression of commonly used markers in FACs-based literature. We are applying for access to this dataset in order to have a reference 10x dataset with both CITE-seq and RNA-sequencing data that can serve to validate and enhance our classification method. We will pre-process this dataset using a pipeline identical to the one used in our project, and then classifying the cells based on both (1) the RNA-based classification method we used, and two (2) a protein-based classification using the CITE-seq data, and then compare the concordance of the two classification schemes. This dataset may also be used as training data to better parameterize our model, which we could then apply to future datasets. The clinical side of this project is in collaboration with the labs of Marcela Maus at Massachusetts General Hospital and Catherine Wu at Dana Farber Cancer Institute, but only the Getz lab will handle the raw data applied for here.\",\n\"2022, May23\",\n\"phs002315.v1.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Fullerton, Janice \",\n\"UNIVERSITY OF NEW SOUTH WALES\",\n\"ENIGMA addiction project - Sydney node\",\n\"approved\",\n\"Genome wide association studies (GWAS) identify genetic variants associated with a trait such as problematic substance use. We will use two GWAS from dbGAP which targeted nicotine dependence and problematic alcohol use to generate individualized genetic risk scores for these traits in other samples. We will examine whether these risk scores are related to behavior (e.g. age of onset and frequency of substance use, and measures of impulsivity) or brain features detected using MRI, including measures of brain structure (e.g. cortical and subcortical grey matter volume and white matter tract integrity) and function.\",\n\"Polygenic risk scores (PRS) are a powerful tool to examine the genetic liability of a specific trait in another population. Our aim is to investigate the genetic liability to substance use disorders (e.g., alcohol) in independent population-based cohorts in relation to behavior measures and to identify brain intermediate phenotype measures associated with greater genetic risk for problematic alcohol use. PRS will be created using PLINK 2 software, based on the results of the Genome Wide Association Studies (GWAS) for problematic alcohol use (PMID 32451486 - phs001672.v7.p1) and other datasets including cigarettes per day (PMID 30643251). These PRS will be regressed onto behavioral phenotypes in local datasets (e.g., substance use measures including age of onset and frequency of use, and impulsivity measures) and onto brain measures including structure (cortical and subcortical measures and diffusivity) and function. This project does not require individual-level data from dbGAP from the mentioned studies (only GWAS summary statistics are required) and therefore personal identifying information for research participants in the original study is not requested, and the objectives of the current project are limited to health/medical/biomedical purposes, and does not include the study of population origins or ancestry.  Data will be stored on secure password-protected servers at Neuroscience Research Australia (NeuRA), a not-for-profit research institute of around 300 academic researchers affiliated with The University of New South Wales (UNSW). NeuRA's research grants, governance and ethics approvals are managed by UNSW. NeuRA's security systems are independently overseen by NeuRA's Director of IT Security, Andrew Cartwright, who has a conjoint appointment at UNSW, and specific training in high-level data security (e.g. 2022 NIST cybersecurity Framework practitioner, and 2020 Cyber security Foundation+ practitioner accreditatons) with institutional oversight. NeuRA's computing capacity is supported by computing infrastructure at UNSW (e.g. using a configuration on the main UNSW virtual private network concentrator, and access to UNSW data archive). NeuRA is fully resident on the UNSW data network, which allows network traffic to pass through the UNSW next-generation firewall.  The generation of the PRS in local datasets will occur on the NeuRA servers using imputed genotype data created using the ENIMGA genetics pipeline (1kGP p3v5 cookbook) by investigator Associate Professor Janice Fullerton (j.fullerton@neura.edu.au; ERA commons username: JANICE_FULLERTON) and stored locally. The PRS will be generated locally, and the results analysed in collaboration with local clinical collaborators at UNSW School of Psychiatry, Scientia Professor Philip Mitchell (p.mitchell@unsw.edu.au) and Dr Gloria Roberts (robertsg@unsw.edu.au), as per UNSW HREA ethical approval HC220120 (23Mar2022-22Mar2027). Summary data will then be contributed to the ENIGMA Addiction consortium (https://www.enigmaaddictionconsortium.com/), with meta-analysis led by Dr Renata Cupertino (Renata.Cupertino@uvm.edu), and project co-headed by Professor Hugh Garavan (Hugh.Garavan@uvm.edu) and Professor Scott Mackey (msmackey@uvm.edu) at the University of Vermont, Burlington, VT.   Data will be used for the sole use of the proposed project, which is in line with the research participants\\u00e2\\u0080\\u0099 informed consent. We acknowledge the confidentiality of the data, that the privacy of participants must be protected, and public posting of summary results are not allowed. We agree not to use the requested datasets to identify or contact individual participants from whom data and/or samples were collected. We certify our agreement with the expectations and terms of access, including the requirement of an additional request if cloud computing is deemed necessary. \",\n\"Aug30, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Franceschini, Nora \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"Polygenic risk scores for cardiovascular traits \",\n\"approved\",\n\"We will use the agenetic analysis results for cardiovascular traits to study genetic risk in additional populations. \",\n\"We propose to create polygenic risk scores using summary statistics from GWAS of cardiometabolic traits to predict cardio-renal outcomes in diverse populations. We will also use the summary statistics to replicate findings identified in GWAS of diverse populations. \",\n\"Aug31, 2021\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Fongang, Bernard \",\n\"UNIVERSITY OF TEXAS HLTH SCIENCE CENTER\",\n\"Gene expression and DNA methylation signatures of the cardiovascular health and neurocognitive endophenotypes\",\n\"approved\",\n\"Cardiovascular health is associated with dementia, but whether one could use cardiovascular risk factors drugs (blood pressure, for example) to prevent or treat dementia or cognition is controversial. We will use the requested datasets to determine the shared genetic, genomic, and epigenomic features between the cardiovascular risk factors and dementia endophenotypes. Our results will lay the foundation for future studies and help understand the common biological mechanisms between cardiovascular diseases and dementia.\",\n\"Cardiovascular diseases (CVD) and dementia have common risk factors. But whether CVD drugs could be used to treat or prevent dementia is controversial. We are requesting genetic, genomic, and epigenomic data that will be used to establish the common biological mechanisms of CVD risk factors and dementia endophenotypes. We will use biostatistical and bioinformatic models to (a) determine the genetic, genomic, and epigenomic signatures of CVD risk factors; (b) the genetic, genomic, and epigenetic signatures of dementia endophenotypes; (c) identify the shared features; (d) and build a predictive model that will establish the most critical elements influencing memory decline and the development of dementia. The requested data will be used for biomedical research only, and the analyses will be restricted to the aims described above. The data will be stored in our secure platforms and accessible only to the PI. Moreover, we will not attempt to reveal the identity/ancestry of any participant. Finally, findings from our study will be published and shared with the scientific community. The dbGap accession information with the relevant version number of the analyzed datasets will be included in the publications. The data hosted by dbGap are well suited for our project since the genetic, imaging, and clinical data are collected on the same participants.  This project will provide valuable information to the research community. Because brain changes related to dementia start decades before symptoms appear, understanding the molecular connections between CVD risk factors and neurocognitive endophenotypes could offer a chance to prevent or delay memory loss and other debilitating symptoms. \",\n\"Sep07, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Fletcher, Jason \",\n\"UNIVERSITY OF WISCONSIN-MADISON\",\n\"Using Summary Statistics from Multi-Ethnic GWAS to Develop and deploy new polygenic scores in social science surveys\",\n\"closed\",\n\"This research will leverage summary statistics from genetic analysis in the Million Veteran Program to create genetic-based predictions (polygenic scores) for PTSD and substance dependency in multi-ethnic social survey data\",\n\"This research will leverage summary statistics from genetic analysis in the Million Veteran Program to create genetic-based predictions (polygenic scores) for PTSD and substance dependency in multi-ethnic social survey data.  Our research team has developed new methods of utilizing summary statistics from large scale GWAS in order to create polygenic scores in genotyped social science surveys, such as the Health and Retirement Study and Add Health, among others.  Most current large scale GWAS is focused on individuals of European ancestry, which makes the creation of polygenic scores for other populations groups difficult and less predictive.  We will extend our methods that have focused on GWAS summary statistics in European ancestry analyses in order to create and use new polygenic scores created from non-European ancestry GWAS, like the Million Veterans Program.    These new polygenic scores will be used in follow up analyses in the HRS and Add Health, among other datasets, in order to explore how the polygenic scores are associated with a large range of health measures not included in the Million Veterans Program data.  \",\n\"Apr06, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Fitzgerald, Kathryn \",\n\"JOHNS HOPKINS UNIVERSITY\",\n\"Comorbidity and outcomes in multiple sclerosis: A polygenic score approach\",\n\"approved\",\n\"Health conditions like depression, heart disease, diabetes, obesity, or high blood pressure are common in people with multiple sclerosis (MS). A person with MS who has one of these conditions may be at higher risk of their MS worsening. For example, a person with MS and high blood pressure might have more MS disability than a person with MS who does not have high blood pressure. However, the results from studies so far have not been the same. Sometimes in studies where participants are not randomized (like the flip of a coin) to a study treatment, there can be bias. The bias could have played a role in the different conclusions. We want to reduce these biases by studying genetic factors that are related to a person\\u00e2\\u0080\\u0099s risk for diabetes or high blood pressure. Genetic factors do not change over a person\\u00e2\\u0080\\u0099s lifetime. There are many of these genetic factors that play a role in a person\\u00e2\\u0080\\u0099s risk for diabetes or high blood pressure. Together, they are called \\u00e2\\u0080\\u0098diabetes genetic scores\\u00e2\\u0080\\u0099 or \\u00e2\\u0080\\u0098high blood pressure genetic scores.\\u00e2\\u0080\\u0099 The goal of this study is to see if these genetic scores can predict MS progression. The genetic scores will be derived using summary information that from previously conducted studies as a part of phs001672 and the Million Veteran\\u00e2\\u0080\\u0099s Project. \",\n\"Background: The disease course in people with multiple sclerosis (PwMS) is heterogenous; emerging studies indicate that comorbidity could contribute to the observed variability in disease course. For example, cardiometabolic comorbidities like obesity, diabetes, hypertension and hyperlipidemia are overrepresented in PwMS; and are also associated with worse MS outcomes. The goal of this proposal is to improve our understanding of the contribution of comorbidity to disease outcomes in MS. We will accomplish these goals using a novel polygenic score (PGS) approach and will leverage four large pre-existing multi-national cohort studies of PwMS with detailed clinical, imaging, and genetic information. We will accomplish this goal using a novel polygenic score (PGS)8,9 approach and by leveraging four large pre-existing cohort studies of PwMS with detailed clinical, imaging, and genetic information. We will also use summary measures of polygenic inheritance (e.g., the weighted sum of the number of known genetic risk variants for a given disorder within each individual) for each cardiometabolic disease derived from large-scale genome wide association studies (GWAS; e.g., derived from summary statistics available from phs001672 and the Million Veteran\\u00e2\\u0080\\u0099s Project) as an instrument to study the effect of each comorbidity on changes in MS outcomes. By focusing on genetic factors, which do not change from birth, we can minimize confounding and effectively exclude reverse causation as an explanation for associations between comorbidity and outcomes. Thus, we mitigate some weaknesses of prior work. Our central hypothesis is that cardiometabolic and psychiatric comorbidity and related traits (e.g., lipid levels, BMI, among others), as measured using PGSs, predict clinical and radiological outcomes in PwMS.   Aim 1. To examine the association between psychiatric and cardiometabolic PGS with clinical, imaging and biomarker outcomes in people with MS. Hypothesis: Individuals with higher PGS for common cardiometabolic comorbidities will have worse disease outcomes, including faster disability progression (primary outcome) as well as more disease activity (relapses) and radiographic and biomarker evidence of disease worsening (secondary outcomes). Analysis Plan: We will generate PGS for (1) indicators of obesity including body mass index and waist circumference; (2) lipid levels including total cholesterol and low-density lipoprotein levels; (3) diabetes, insulin levels, and hemoglobin A1c; (4) hypertension including systolic blood pressure and diastolic blood pressure; (5) depression. PGS will be generated using established guidelines with adherence to strict reporting standards. We will pool resources from four well-characterized cohorts of PwMS including 2366 PwMS with detailed genetic information and rigorous follow-up averaging 2.8\\u00c2\\u00b11.0 years. The analytic model for each outcome will based on the distribution of the outcome in question and will consider adjustment for age, sex, genetic ancestry and MS therapy. PGS will be generated using PRS-CSx, which is a method that improves polygenic prediction in a multi-ethnic context through the inclusion of GWAS summary statistics from multiple populations. Summary statistics from populations of interest as available through phs001672 and the Million Veterans Project include individuals of European, African American, Hispanic, and Asian ancestry. The primary outcome will be change in the MS Functional Composite (MSFC). The MSFC is a validated metric of impairment, showing good longitudinal correlation with clinical disability and responsiveness to treatment. We will fit longitudinal (e.g., linear mixed effects [LME] models). We will consider adjustment for age, sex, genetic ancestry, and MS therapy. P-values will be adjusted for multiple comparisons using appropriate methods.61 Horizontal pleiotropy (e.g., the PGS may affect traits that influence the outcome, other than via the trait of interest) is a concern. For analyses showing potentially significant findings, we will replicate PGS analyses using Mendelian randomization and test for such bias therein.  \",\n\"Apr12, 2023\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Fisch, Kathleen \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Comprehensive transcriptomic characterization of decidual leukocytes at the maternal-fetal interface at single cell resolution\",\n\"approved\",\n\" A successful pregnancy requires crosstalk between maternal immune cells and fetal-derived cells in the placenta (trophoblast). The goal of our research is to characterize these cells at the maternal-fetal interface in early and late pregnancy. Specifically in this project, we are interested in understanding the maternal immune cells and how these cells change as pregnancy progresses. \",\n\"Our research objective is to characterize the decidual leukocytes at the maternal-fetal interface. We propose to do this by single-cell RNA sequencing of both early and late gestation placenta. Our group has already sequenced early gestation decidua and the dbGaP dataset we are requesting access to are from term placental tissues. We will compare the leukocytes from each of the datasets to understand how the population changes throughout pregnancy. The requested dataset will also be used for validation of the datasets we plan to generate when we complete our single-cell RNA sequencing of term placenta.\",\n\"2021, Apr16\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Golod, David \",\n\"GOLDFINCH BIOPHARMA, INC.\",\n\"Genetic correlation between types of kidney diseases\",\n\"expired\",\n\"Goldfinch Bio plans to utilize genetic data from across kidney disease types to identify pathways that are shared vs distinct to disease development. These data will provide a means to better characterize how genetic variation influences kidney disease types, identify dysregulated pathways that are specific to disease types, and develop precision medicines for therapeutic interventions\",\n\"The objective of this study is to characterize shared vs distinct contributions of common variants to kidney disease types. We have assembled a large whole genome sequencing dataset of ~23,000 multi-ethnic populations to identify common and rare variants that associate with focal segmental glomerulosclerosis (FSGS), a rare form of kidney disease. To compare and contrast genetic relationship between FSGS and other kidney disease or functionally-related traits, we need GWAS summary statistics from population-based cohorts of kidney disease or function. We are requesting access to estimated glomerular filtration rate (eGFR) GWAS summary statistics from the Million Veteran Program (MVP) of ~300,000 multi-ethnic participants. Our objectives are: to measure genetic relatedness between: 1) FSGS and chronic kidney disease; 2) FSGS and diabetic kidney disease; and 3) FSGS and non-diabetic kidney disease. First, we will assess genetic relatedness within each population (European or African-American) by using population-specific summary statistics, and then across-ancestries by using meta-analysis results from the two populations. In accordance with the data use limitations for the dataset, we are not requesting any individual-level data. We do not anticipate any additional risk to the participants.\",\n\"2020, Apr08\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Gray, Joshua \",\n\"U.S.UNIFORMED SERVICES UNIV  HLTH SCIS\",\n\"Taking stock of genes associated with problematic alcohol use: Protein-protein interactions yield disease-relevant biological networks\",\n\"approved\",\n\"Psychiatric disorders, including alcohol use disorder, are influenced by hundreds or even thousands of genes. We seek to apply a new set of algorithms that will organize genetic contributions to problematic alcohol use into biological subgroups. This will help us better understand the genetic contributions to alcohol use disorder. Furthermore, we will look up medications that interact with these biological subgroups in hopes of identifying potential new treatments for alcohol use disorder.\",\n\"A recent challenge, Disease Module Identification Dialogue on Reverse Engineering and Assessment (DREAM), identified top-performing algorithms that most effectively identify disease relevant modules from protein-protein interaction (PPI) data. These modules are meaningful functional units that are comprised of core disease associated genes and peripheral genes, which impact disease relevant processes despite not yet being implicated in GWASs directly. Thus, in order to elucidate pathways implicated in problematic alcohol use (PAU) as well as sequelae, such as alcohol-related liver disease, we will apply a winning algorithm from the DREAM challenge to organize alcohol associated genes from the most recent PAU GWAS into interpretable PPI modules. We will additionally apply other emerging network-based approaches for understanding genes linked to PAU in their broader biological context. Furthermore, we will utilize drug-protein interaction databases (e.g., https://www.broadinstitute.org/drug-repurposing-hub) to look for approved drugs that interact with proteins implicated by these modules. The overarching goal is to understand more about biological contributors to PAU and to potentially identify novel treatment targets and/or drugs for repurposing.  This will not be combined with datasets outside of dbGaP. We may present these results alongside those from a recent GSCAN study, but these summary statistics are publicly available (https://www.nature.com/articles/s41588-018-0307-5). We will only be downloading the summary statistics (i.e., SNPs and associated p values) and thus do not anticipate any additional risk to participants.  We are focusing on the genetic basis of problematic alcohol use (PAU) so the outcome is the same as the original project.\",\n\"Jun29, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Ingelsson, Erik \",\n\"GLAXOSMITHKLINE\",\n\"Discovery & Development of New Medicines & Vaccines using GWAS Summary Statistics\",\n\"approved\",\n\"Genome-wide association studies (GWAS) identify genetic variants associated with complex diseases which can be useful in identifying new therapeutics and optimizing the development of existing therapeutics.  We will combine results from multiple GWAS studies to create well-powered studies to conduct this research.  Scientists with expertise in human genetics will interpret the results.  \",\n\"We will use the GWAS summary statistics for the purposes of research focused on identifying genes associated with patient outcomes which can be used to discover or optimize development of new medicines and vaccines.  This research is conducted by performing meta-analyses of multiple genetic association studies to create a well-powered study of the trait of interest and colocalization with expression QTL results.  These GWAS summary statistics will be meta-analyzed with GWAS summary statistics derived from analysis of UK Biobank, FinnGen, 23andMe and other publicly available data sets and colocalization with expression QTL summary statistics including Blueprint and GTEX and other publicly available datasets.    This research would be spread across the GSK group of companies and may include use in research collaborations, but the GWAS summary statistics will not be passed to any third party. Any publication relating to this work would include appropriate attribution/credit.  We also plan to use the GWAS summary statistics to evaluate different strategies for fine and gene mapping, e.g. via colocalization to expression, splicing and other QTL datasets.  Erik Ingelsson is the SVP, Human Genetics which is responsible for Human Genetics analysis for GSK R&D organization.  Due to the corporate nature of GSK, Erik\\u00e2\\u0080\\u0099s team is distributed across the UK and US and employed by multiple GSK entities. He has direct line management and accountability for the work that these employees perform.    Chuck Smith is a VP, Data Platform and Solutions, in our Tech Research and Development department which is the IT department within GSK.  He is an IT director and has the authority and capacity to deliver the platform, processes and staff all relevant for the requested data.  \",\n\"2021, Jul12\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Grotzinger, Andrew \",\n\"UNIVERSITY OF COLORADO\",\n\"Psychiatric Genomics Consortium Cross-Disorder Group v3 (PGC-CDG3): Probing multivariate genetic architecture across psychiatric traits\",\n\"approved\",\n\"Psychiatric disorders overlap substantially at the genetic level. Cross-disorder genomic projects that aim to better characterize this genetic overlap then have the potential to produce key insights into the underpinnings of rampant psychiatric comorbidity. To this end, the current proposal aims to analyze the most up-to-date summary statistics across 14 major psychiatric disorders using four cutting-edge genomic methods: Genomic SEM, CC-GWAS, MiXER, and LAVA. These methods can be used to examine genetic overlap in aggregate across the genome, for specific classes of genes (e.g., genes that are expressed early in life), and at the level of individual genetic variants. By applying different methodoligcal approaches we aim to identify whether a convergent narrative emerges about the patterns of genetic overlap across disorders at varying levels of analysis. With respect to the number of disorders and utilized methods, this project aims to perform the comprehensive, cross-disorder psychiatric analysis to-date. We are specifically requesting permission to use the summary statistics that include the MVP cohort for: alcohol use disorder, opioid use disorder, anxiety disorder, major depressive disorder, and post-traumatic stress disorder.  \",\n\"Building on the work of the PGC Cross-Disorder Working Group (PGC CDG2) in clarifying the genetic structure of psychopathology, we now aim to leverage multiple, multivariate methods to better define the shared and disorder-specific genomic underpinnings across 14 major psychiatric disorders. At the genome-wide level, we will specifically apply MiXER, Genomic SEM, and CC-GWAS. Using Genomic SEM, we will produce a factor model based on the genetic correlations across the 14 disorders with the aim of identifying subgroups of disorders that more strongly cluster together. MiXER then takes an alternative approach to quantify polygenic overlap by estimating the total number of shared and trait-specific causal variants, while CC-GWAS calculates the overall \\u00e2\\u0080\\u009cgenetic distance\\u00e2\\u0080\\u009d across two traits based on differences in allele frequencies for cases. Collectively, these results will help us determine whether a convergent narrative emerges about the overall pattern of genetic overlap across these disorders.  At the functional level, we will apply two new methods: Stratified Genomic SEM and LAVA. Stratified Genomic SEM builds upon the findings from Genomic SEM to examine multivariate enrichment at the level of the psychiatric factors. LAVA then examines local genetic correlations at the level of individual loci. In conjunction, these two methods allow us to identify specific categories of genes and loci that drive genetic overlap across disorders. Finally, at the SNP-level of analysis, we will use CC-GWAS and multivariate GWAS in Genomic SEM to identify SNPs that are both highly pleiotropic and disorder-specific. The cumulative set of findings will offer critical insights into the multivariate architecture of psychiatric disorders utilizing cutting-edge methods at three levels of analysis (genome-wide -> functional -> SNP). Indeed, the proposed project reflects the most comprehensive cross-disorder, psychiatric analysis to-date. We are specifically requesting permission to use the summary statistics that include the MVP cohort for: alcohol use disorder, opioid use disorder, anxiety disorder, major depressive disorder, and post-traumatic stress disorder.   \",\n\"2021, Dec13\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Im, Kate \",\n\"GENO TECHNOLOGIES, INC.\",\n\"Investigating heritability of disease risk and methods development in family-based statistical genetics. \",\n\"approved\",\n\"To understand the genetic underpinning of diseases and traits, we will use this data to improve algorithms to identify genomic differences between individuals and understand how those differences can predispose different individuals to diseases or specific traits. \",\n\"We aim  to better understand the genetic underpinning of diseases and traits with an aim to prevent diseases and empower patients and their families. We use a combination of approaches including investigating variants in genes associated with diseases as well as constructing genome-wide risk scores for more complex diseases. We are particularly interested in utilizing data for conditions that are of interest in the setting of family planning. We will analyze these datasets in accordance with the data use statements for each study. As an example, our team is exploring the impact of multiple autoimmune conditions;  in this context we will not combine the data for one consent group (e.g. vitiligo only) with data for another consent group if forbidden by the research use statement. This data will be used for methods development which includes investigating the quality of genotype imputation methods, predicting parental haplotypes and their transmission, and calculating genomic risk scores for comparison amongst siblings. No genetic information will be released to any third-party outside of the company for any of these individuals. Results of this project will be presented at scientific meetings and published in journals.\",\n\"Jul20, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Huang, Frank \",\n\"CINCINNATI CHILDRENS HOSP MED CTR\",\n\"Identifying cell-type specific gene expression in preterm birth\",\n\"closed\",\n\"According to a recent investigation, more than 10 million preterm birth occur worldwide each year. We would like to analyze the data from women who are term and preterm parturition to find out the molecular mechanisms and risk genes that contribute to preterm birth.\",\n\"The placenta is of great significance for human parturition, while the cell types in the placenta are still puzzled, especially for preterm birth. Here we would like to apply for the access of single-cell RNA-seq data in the placenta. Our research mainly focuses on identifying the heterogeneous cell types of the placenta. We will also compare the gene expression of different cell types and their proposition between term and preterm birth sample. Moreover, we plan to apply the Seurat R package to study the cell type-specific differential expressed genes by comparing the single-cell RNA-seq data of term and preterm and identify if those genes are highly correlated to preterm birth. \",\n\"Apr28, 2020\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Holm, Hilma \",\n\"DECODE GENETICS, EHF\",\n\"Genome wide meta-analysis of cardiometabolic traits\",\n\"approved\",\n\"The goal of the study is to identify risk variants in the genome that associate with the risk of the cardiometabolic traits; peripheral artery disease, abdominal aortic aneurysm, venous thromboembolism, type 2 diabetes, blood pressure and blood lipids. Several studies have screened the entire genome of a large number of cases with these particular traits in order to look for association between millions of genetic variants and the disease/trait outcome (so called Genome Wide Association study). In order to gain more power to identify genetic risk variants we intend to combine the largest sample sets available for these traits and then perform our analyses.\",\n\"The goal of our research is to perform genetic discovery analyses of the following cardiometabolic traits; peripheral artery disease, abdominal aortic aneurysm, venous thromboembolism, type 2 diabetes, blood pressure and blood lipids. The requested data from dbGAP, i.e. summary level Genome wide Association (GWA) data from the MVP trans-ancestry meta-analyses of these cardiometabolic traits, will be combined with GWAS datasets from Iceland, UKBiobank, Denmark, Finngen, Intermountain USA and other available datasets.  We will perform association analyses, Mendelian randomization studies and polygenic risk score approaches. The data requested will include no personal identifiers only summary level data so this study will not create any additional risks to the participants.\",\n\"2023, Feb21\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Hillman, Sara \",\n\"UNIVERSITY COLLEGE LONDON\",\n\"Analysis of cell heterogeneity in human placenta \",\n\"expired\",\n\"During pregnancy there is a highly coordinated dialogue between mother and fetus and it is at the point of contact between both of them where very important communication is established to ensure the correct development of the baby. Recently, single-cell transcriptomics have allowed the study of the maternal-fetal interface at cellular resolution. We are requesting access to the data deposited at NIH dbGAP phs001886.v2.p1 to re-analyse the cellular composition of the tissues that encompasses the maternal-fetal interface using recent methods based on machine learning to classify cell types. The outcome of this proposal will increase our understanding of cellular composition at this important interface.\",\n\"The placenta is a dynamic and heterogenous organ including both maternal and fetal cell types. Single-cell RNA-seq has enabled the transcriptional profiling of this organ at single cell resolution [2-3, 5]. Using a combination of unsupervised clustering and label transfer, Pique-Regi et al., 2019 shown that there are 19 cell types in the basal plate, placental villous and chorionamniotic membranes.   Recently, new computational tools based on machine learning have been developed to probabilistically assign cell identities from sc-RNAseq datasets. We are requesting the libraries deposited at NIH dbGAP (accession number phs001886.v2.p1) to re-analyse the cellular composition in the placenta and benchmark some bioinformatics recent tools to classify cell types. Importantly, these tools have the potential of identifying novel cell types and resolve better the differences between term labor and pre-term labor.  Specifically, we will use supervised approaches (for instance Garnett[4] and CellAssign[6]) and unsupervised approaches (such as SCAFF[1]). We will analyse dbGap datasets separately and in combination with other datasets publicly available (Vento-Tormo et al., 2018).   The outcome of the proposed project will advance our understanding of the cellular composition at different trimesters during pregnancy and locations of the maternal and fetal interface. The analysis proposed will not create any additional risks to participants.  References  [1] Miao, Z., Nature Methods, 2020; 17(6), 621-628. DOI:10.1038/s41592-020-0825-9 [2] Pique-Regi R. Elife. 2019;8:e52004 DOI: 10.7554/eLife.52004 [3] Pique-Regi R. Elife. 2020; 9:e58716 DOI: 10.7554/eLife.58716 [4] Pliner, H. A. Nature Methods, 2019; 16(10), 983-986. DOI:10.1038/s41592-019-0535-3 [5] Vento-Tormo R. Nature. 2018;563(7731):347-53. DOI: 10.1038/s41586-018-0698-6  [6] Zhang, A. W. Nature Methods, 2019; 16(10),1007-1015. DOI:10.1038/s41592-019-0529-1 \",\n\"2021, Mar31\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Herse, Florian \",\n\"MAX DELBRUECK CENTRUM FOR MOLECULAR MED\",\n\"Placental cell atlas\",\n\"approved\",\n\"Preeclampsia (PE) is the sudden onset of high blood pressure after the 20th week of pregnancy. Severe PE causes intrauterine growth restriction (IUGR) of the fetus, preterm delivery or even stillbirth. PE is the leading cause of maternal and fetal morbidity and mortality. The underlying mechanism of PE is still unknown. However, it is evident that the placenta and its development (placentation) early in pregnancy plays a major role in the development of this disease. To analyze the underlying mechanisms in PE and to better understand human placentation, we recently conducted studies featuring placental tissues at single-cell resolution. These studies reveals the cell communication network as well as functionally distinct cell types of the maternal-fetal interface, namely placenta (tissue with fetal origin) and decidua (maternal tissue) and will describe human cell differentiation in first trimester and term placentas. Funded by the german research foundation \\u00e2\\u0080\\u009cDeutsche Forschungsgemeinschaft\\u00e2\\u0080\\u009d (DFG) (grants-No.: HE6249/7-1 and HE6249/7-2) our group plan to construct a placenta cell atlas resulting of first trimester and term placental and corresponding decidual tissue, including PE samples.\",\n\"Preeclampsia (PE) is the sudden onset of hypertension after the 20th week of pregnancy accompanied by proteinuria. Severe PE causes intrauterine growth restriction (IUGR) of the fetus, preterm delivery or even stillbirth. PE is the leading cause of maternal and fetal morbidity and mortality. The underlying pathomechanism of PE is still unknown. However, it is evident that the placenta plays a major role in the development of this disease. To analyze the underlying mechanisms in PE and to better understand human placentation, we recently conducted studies featuring placental tissues at single-cell resolution. These studies reveals the cell communication network as well as functionally distinct cell types of the maternal-fetal interface, namely placenta (fetal tissue) and decidua (maternal tissue) and describe human cell differentiation in first trimester and term placentas. Funded by the \\u00e2\\u0080\\u009cDeutsche Forschungsgemeinschaft\\u00e2\\u0080\\u009d (DFG) (grants-No.: HE6249/7-1 and HE6249/7-2) our group plan to construct a placenta cell atlas resulting of first trimester and term placental and corresponding decidual tissue, including PE samples: First trimester placenta control: 3 First trimester decidua control: 3 Term placenta control: 6 Term decidua control: 3 Term placenta PE: 3 Term decidua PE: 3  The data processing workflow consists of preprocessing via the CellRanger v3 workflow, followed by integration and comparisons using the R package Suerat v3, and further donstream analysis including differential expression, cell type abundance, receptor-ligand interactions and trajectory analysis.  Further bulk and single cell sequencing data from human tissue and cells will be integrated in the analysis to improve cell annotation and integrity of data.  \",\n\"2020, Sep03\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Heilig, Markus \",\n\"LINKOPING UNIVERSITY\",\n\"Polygenic risk for substance abuse as a predictor of substance abuse in individuals with versus without documented childhood maltreatment\",\n\"approved\",\n\"Many people who experience childhood maltreatment go on to develop issues with substance abuse, like alcohol or opioid addiction. Still, others who experience this sort of childhood trauma do not go on to have issues with substance abuse. One reason for this difference might be that those who go on to develop substance abuse have a higher genetic susceptibility for this disorder, and the stress of childhood maltreatment interacts with this genotype in a pathological way. In the research proposed here, we plan to test this hypothesis directly by looking at genetic risk for substance abuse in persons who have or have not experienced childhood maltreatment and then gone on to either have substance abuse issues or not.\",\n\"Three recent genome wide associate studies of opioid use disorder (https://jamanetwork.com/journals/jamapsychiatry/fullarticle/2766708), tobacco smoking (https://www.nature.com/articles/s41467-020-18489-3), and alcohol use disorder (https://www.nature.com/articles/s41593-020-0643-5) report summary statistics stored in dbGaP that we propose using for assessing polygenic risk (PRS) for substance abuse, broadly construed. We will assess polygenic risk in four groups: 1) individuals with no history of childhood maltreatment (CM) and substance abuse (SA); 2) persons with CM but no SA history; 3) individuals with SA but not CM history; and 4) persons with both SA and CM histories. The primary analyses of interest will compare PRS in SA+ versus SA- groups as well as in CM+&SA+ versus CM+&SA- subgroups. The summary statistics made available from dbGaP will be combined with the following data from our site: 1) individual genotype data derived from DNA collected onsite; 2) substance abuse histories determined from structured clinical interview; and 3) childhood maltreatment histories determined from accounts collected by state-sponsored agencies intervening at the time of abuse.\",\n\"2021, Jul28\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Hayek, Salim \",\n\"UNIVERSITY OF MICHIGAN AT ANN ARBOR\",\n\"Role of SuPAR in the Intersection Between Cardiovascular and Kidney Disease\",\n\"approved\",\n\"Chronic kidney disease affects 15% of people in the US, and those affected are more prone to heart disease. We recently found that patients with increased levels of a certain biomarker - a substance that is found in body fluids, in this case the blood of patients - known as suPAR, are more likely to have a poor prognosis than those with lower levels of the suPAR biomarker.  The goal of this project is to see whether suPAR itself contributes to heart disease and declining kidney function. We will use the suPAR levels found in patients who took part in the Multi-Ethnic Study of Atherosclerosis (MESA) to ensure the correlation with other early markers of heart disease and the progression to worsening heart disease.  Then, to see if suPAR could be a cause of these diseases, we will use a genome-wide association study of suPAR levels in MESA to see if there are genes that are different in people with high suPAR levels. We will use a larger group of people (through the UK Biobank and Million Veterans Program) to see if those genes are connected to heart disease or chronic kidney disease.  \",\n\"We aim to assess potential causality between suPAR and both Chronic Kidney Disease (CKD) and cardiovascular disease (CVD) using Mendelian randomization. We hypothesize patients with CVD or CKD will harbor genetic variants associated with high suPAR levels, suggesting suPAR is an effector of CKD and CVD. We have previously performed a genome-wide association study (GWAS) of suPAR in 3,244 healthy volunteers and identified genetic variants in the PLAU and PLAUR genes associated with suPAR levels. We will (a) validate our findings in a GWAS of MESA, followed by (b) a meta-analysis using all cohorts (including the healthy participants in the Trinity Student Study (TSS) and the Genes and Blood-Clotting (GABC) study). Lastly (c) we will perform a Mendelian randomization study in the UK Biobank with data for over 500,000 volunteers, to support suPAR as a cause of CKD and CVD.   People with CKD suffer a disproportionately high burden of CVD for reasons that are poorly understood. Inflammation represents the major pathophysiological process common to both atherosclerosis and CKD, and recently, a novel pathway of inflammation was uncovered linking both CVD and CKD to bone marrow myeloid cells producing a circulating signaling molecule: soluble urokinase plasminogen activator receptor (suPAR). suPAR levels predict adverse outcomes in both diseases, and outperform well-established markers of CVD risk, such as coronary calcium, C-reactive protein, high sensitivity troponin I and B-type natriuretic peptide. Pharmacological inhibition of suPAR in experimental models prevented kidney injury. These compelling data reveal the potential for suPAR not only as an excellent biomarker of risk, but also as a promising therapeutic target.   The overall goal of this project is to elucidate suPAR\\u00e2\\u0080\\u0099s potential causal role in the progression of atherosclerosis and its link to decline in kidney function through epidemiological insights. We will achieve this goal by leveraging three of the most significant contributions to cardiovascular science: the landmark Multi-Ethnic Study of Atherosclerosis (MESA); an NIH-funded prospective cohort in which enrollees underwent serial measurements of subclinical markers of atherosclerosis, the JUPITER trial, in which participants with high C-reactive protein levels were randomized to statin or placebo, and the UK Biobank, a data repository of over 500,000 volunteers. SuPAR levels will be measured in ~5,620 participants of MESA to determine whether levels correlate with early markers of CVD and predicts their progression independently of a decline in kidney function. To assess potential causality, a gene-wide association study of suPAR levels in MESA, followed by a Mendelian randomization analysis in the UK Biobank dataset will connect genetic determinants of suPAR levels to CVD and CKD. Lastly, to establish whether suPAR is a modifiable risk factor for CVD, levels will be measured serially in participants of the JUPITER trial randomized to rosuvastatin (n=200) or placebo (n=200), and the change in suPAR will be compared between groups. Whether the benefits of statins are dependent on suPAR levels will be assessed in the MESA cohort by comparing the survival of participants started on statins across suPAR quartiles, and determining whether suPAR is a modifier of the association between statins and outcomes. The proposed research has the potential to address the unmet need of close to 50 million people with CKD in the United States alone, identifying sorely needed therapeutic targets and management strategies in a field that has long been stagnant. \",\n\"2021, Jan17\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Hao, Xingjie \",\n\"HUAZHONG UNIVERSITY/SCIENCE/TECHNOLOGY\",\n\"Fine-mapping, genetic correlations and potential shared mechanisms between complex diseases in different populations\",\n\"approved\",\n\"Complex diseases were defined as diseases which were influenced by a combination of multiple genes and environmental factors, including asthma, diabetes, epilepsy, hypertension, manic depression, schizophrenia and so on. Single nucleotide polymorphisms (SNPs) are the most common type of genetic variants among people, some of which may affect diseases in some ways. We aim to reveal the pleiotropic SNPs that are associated independently with two or more distinct diseases and explore the common mechanisms for the SNPs affecting the diseases in different populations. In different populations, we will determine the underlying causal variants for each complex diseases, estimate genetic correlations between complex diseases that implying the effects of the tested genetic variants on the two tested complex diseases correlated or not and identify common disease-relevant cell-types in two or more complex diseases which suggesting potential shared mechanisms.\",\n\"There are potential shared genetic structures and pathogenic mechanisms between the complex diseases, which may be different across populations. The overall objective of this study is to reveal the pleiotropic SNPs that are associated independently with two or more distinct complex diseases and explore the common pathways and mechanisms for the SNPs affecting the complex diseases in different populations. With the GWAS summary-level data in different populations of Million Veteran Program (MVP), we plan to (1) develop and utilize new methods to perform fine-mapping for complex diseases leveraging linkage disequilibrium (LD) score in different populations, (2) estimate the genetic correlations between complex diseases in different populations using linkage disequilibrium score regression (LDSC) software, which may provide reference for trans-ancestries polygenic risk score (PRS) and (3) identify disease-relevant cell-types in different populations by conducting stratified LDSC regression to calculate partitioned heritability leveraging the 66 cell-type specific GenoSkyline-Plus annotation tracks, so that we may find potential shared mechanisms and the possible heterogeneity of genetic regulation across populations. We do not plan to combine the dataset from MVP with others. The GWAS summary statistics will not be passed to any third party. Any publication relating to this work would include appropriate attribution/credit.\",\n\"Jan03, 2022\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"HURD, YASMIN \",\n\"ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI\",\n\"Single cell RNA-seq analysis\",\n\"approved\",\n\"We are looking to use molecular data from the human placenta, specifically mRNA expression from single cells in the human placenta, to deconvolve (more detail below) mRNA expression from whole placenta tissue. The data we are requesting has mRNA expression in single cells. The data our lab has is mRNA expression in whole placentas. By comparing the single-cell mRNA expression to our whole-placenta mRNA expression, we can estimate the percentages of different cell types in our whole-placenta mRNA expression, and this is what \\\"deconvolution\\\" means. The placenta contains a variety of cell types, and these cell types are labeled in the single-cell mRNA expression data we are requesting. By examining these different cell types in the requested data, we can find \\\"marker mRNAs\\\" that are selectively expressed in the different cell types in the placenta. With these marker mRNAs, we can estimate the percentages of the same cell types in our whole-placenta mRNA expression based on these marker mRNAs. Our whole-placenta mRNA expression comes from rats that were administered cannabinoids or no cannabinoids (i.e., control group). The data we are requesting will help us identify which cell types, if any, are affected in their placental expression by cannabinoids. \",\n\"Our goal is to use the human placenta single-cell RNA-sequencing data provided in this dataset for single-cell deconvolution of bulk placenta RNA-sequencing data generated by our lab. The purpose of this computational procedure is to use a single-cell RNA-seq reference dataset to estimate cell type fractions in bulk RNA-seq samples. Single-cell deconvolution is conducted using software packages such as dampened weighted least squares (DWLS). Our placenta RNA-seq samples were generated from rat dams that were exposed to cannabinoids during pregnancy. Our study is examining the effects of prenatal cannabinoid exposure on the placenta transcriptome, and part of our investigation is to understand how cannabinoids affect the abundance of different cell types. We plan to use the requested data as a reference single-cell RNA-seq dataset for single-cell deconvolution to obtain cell type fractions. These cell type fractions are calculated for each sample, and we can use these cell type fractions to measure differences in estimated cell type abundance in our bulk RNA-seq samples. Further, the first step of the single-cell deconvolution process is to identify marker genes of different cell types in a single-cell RNA-seq dataset. Using the provided data would allow us to identify marker genes in different placental cell types. \",\n\"Feb17, 2023\",\n\"phs001886.v4.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"HETTEMA, JOHN \",\n\"TEXAS A&amp;M UNIVERSITY HEALTH SCIENCE CTR\",\n\"Identification of Susceptibility Genes for Anxiety and Related Disorders\",\n\"approved\",\n\"Anxiety and depression tend to co-occur over an individual\\u00e2\\u0080\\u0099s lifetime and in their family members. Existing studies suggest this may be due to shared genetic risk factors. This study will try to identify genes that increase risk across these conditions by integrating data from dbGaP datasets with analyses being conducted in other studies. \",\n\"Anxiety disorders to co-occur with each other and with related conditions like depression. Genetic epidemiological studies suggest this may be due to shared genetic risk factors. This study aims to identify genes that increase risk across these conditions (shared and disorder-specific) using genome-wide association study (GWAS) approaches. We will analyze individual anxiety disorder phenotypes (generalized anxiety, panic, phobias, PTSD, OCD) in association with genetic variants as well as combine these into an AnyAnxiety disorder status. Where available, we will include other psychiatric phenotypes like mood disorders, eating disorders, substance use, and personality disorders. We will also apply factor analytic methods to estimate dimensional severity scores as phenotypes. We will integrate results from dbGaP datasets with those from other GWAS via meta-analysis. There will be no overlap between dbGaP samples and external datasets, so no additional risk of identifying subjects will occur. When available, we will also incorporate non-clinical phenotypes like neuroimaging data into our analyses. Note that this is a continuation/extension of our prior project (Identification of Candidate Genes for Internalizing Phenotypes) after the PI changed institutions.\",\n\"Jul29, 2021\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"HAN, SHIZHONG \",\n\"LIEBER INSTITUTE, INC.\",\n\"Mapping PTSD GWAS risk loci to human brain genomic phenotypes\",\n\"rejected\",\n\"We are interested in mapping GWAS risk loci to nearby gene function in human postmortem brain genomics datasets. \",\n\"We are interested in mapping GWAS risk loci to molecular function in the human brain by performing integrative analyses of GWAS summary statistics and human postmortem tissue RNA sequencing (RNA-seq) and whole genome bisulfite sequencing (WGBS). We will take individual variants identified in GWAS for PTSD, and examine how the genotypes of these SNPs associate with nearby DNA methylation and RNA measurements from multiple brain regions in patients with PTSD and neurotypical controls.\",\n\"2019, Dec30\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"HAN, SHIZHONG \",\n\"JOHNS HOPKINS UNIVERSITY\",\n\"Integrative analysis of multi-omics datasets for alcoholism\",\n\"closed\",\n\"Alcoholism has a genetic basis, but identification specific genes or risk variants contributing to the disorder has been a challenging task. Over the last ten years, the research field has accumulated large genetic and genomic datasets that are publicly available, such as those from dbGaP, which provides an unprecedented opportunity to risk gene discovery. We will apply for the access to these datasets and develop advanced statistical models to re-analyze these datasets, with the goal of identifying risk genes underlying alcoholism.\",\n\"Our goal is to identify risk genes underlying alcoholism. We request GWAS summary statistics for alcohol-related traits from MVP. We will perform gene, pathway and network-based analysis using various existing statistical genetic methods and computational tools. We will also develop novel network-based approaches that integrate gene expression information in the brain tissues. We agree to abide by the NIH Code of Conduct for Data Use and the use restrictions associated with each GWAS dataset.\",\n\"2019, Dec30\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Guo, Yan \",\n\"XI'AN JIAOTONG UNIVERSITY\",\n\"Genomic analysis for human complex diseases\",\n\"rejected\",\n\"Many human complex diseases/traits have strong genetic determination. The aim of this project is to search new genetic factors and better understand genetic architectures for common complex diseases/traits through genomic analyses or related methodologies. We are seeking to the access to multiple datasets in dbGAP that contain metabolic disorders, hematologic malignant tumors, cardiovascular diseases, and their related phenotypes. Our results may be helpful in uncovering genetic basis for human complex diseases/traits, and in prevention, diagnosis, and therapy of diseases. \",\n\"Many human complex diseases/traits are highly heritable, but the underlying genes are largely unknown. The aim of the present study is to identify susceptibility loci and improve understanding of the genetic architectures of complex diseases/traits by using available genetic data from multiple datasets in dbGAP through our comprehensive statistical analyses and study design approaches. We have extensive experience in analyzing GWAS and sequencing data. In this application, we are requesting the access to the dbGAP datasets which contain at least one of the following phenotypes of our interest, including metabolic disorders (e.g., osteoporosis, diabetes, and obesity), hematologic malignant tumors, cardiovascular diseases/traits, lung function, lipid traits, psychiatric traits, and their related anthropometric phenotypes and biochemical measures. These data will be used separately and/or in combination by integrating various data types to achieve our goals.  For studies that allow only disease specific research, the data will only be used to develop models specific to the disease and investigate the genetic basis of the disease. The use of the data is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry. The specific objectives are to 1) identify novel loci for the complex traits/diseases of our interest; 2) perform joint analyses for multiple phenotypes to identify potential pleiotropic genes; 3) identify novel gene-gene and gene-environment interactions; 4) conduct gene network-based GWAS analyses that consider multiple genes simultaneously; 5) evaluate related methodologies for analyzing complex traits/diseases. We acknowledge the Use Restrictions for the requested data sets, and will abide by them and conducting research consistent with them. These data will be used only for research purposes. They will not be used to determine the individual identity of any person or of their relationship to another person. We will publish and broadly share any findings from our study with the scientific community and peer-reviewed journals.\",\n\"2020, Jan13\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Guo, Shicheng \",\n\"JOHNSON/JOHNSON/PHARM/RES/ DEVELOPMENT\",\n\"GWAS to identify disease genes, drug targets and novel therapeutic approaches\",\n\"approved\",\n\"This study plans to identify genetic differences between people with and without certain disease or traits. Identification of genetic differences associated with disease will help us understand the biology mechanism and help us to develop new drugs for disease treatment. Meanwhile, disease risk prediction can be performed to show the risk that people will be affected by this disease. Finally, genetic difference also explains the different response to drugs which will help pharmacist to select the best choice of the drug to decease the side-effects of the drugs. \",\n\"Large genetic cohorts provide an opportunity to identify disease susceptibility genes and novel drug target for the disease therapeutics, especially meta-analysis between different studies and populations could provide increased power to identify novel susceptibility genes and drug targets which have low allele frequency. The aim of this project is to identify potential drug targets with eMERGE and other population genetics data including UK-Biobank, FinnGen and Japanese biobank. We will apply multiple computational approach (SAIGE, SAIGE-GENE, Robust-SKAT, Robust-SKAT-O, GENESIS, REGENIE, RVTESTS) to identify disease/traits associated genetic variants and to reveal novel disease mechanism so that novel drug target could be identified. Comparison and evaluation the performance for these different computational approaches will be performed in which the difference of the positive associations and computational resource requirement will be reported. In this study, GWAS study will be performance to all the 41 phenotypes in eMERGE, UKB and FinnGen with uniform statistical method and computational pipelines to avoid non-necessary method bias. We will also apply tissue-specific gene expression, eQTL, sQTL to infer the potential tissue-of-origin for the disease. Multiple genetic fine-mapping approach will be conducted to compare the performance and multiple multiple-omics (GTEx, GEO and blueprint) based network approach will be tried to identify disease related molecule modules and tissue-specific regulatory circuits. Our research aims are totally same with primary study aims of eMERGE, UKB and FinnGen and we will not have any external collaborators in this study.\",\n\"2020, Nov13\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"Gui, Hongsheng \",\n\"HENRY FORD HEALTH SYSTEM\",\n\"Investigation of substance use on risk of suicidal attempt: a Mendelian randomization study\",\n\"approved\",\n\"Is there any connection between substance addiction and suicidal attempt? Can we identify patients with substance use disorder so to prevent their suicide completion? We seek to use genomic data published by Million Veteran Project to answer this two questions. Different from environment risk factors, genomics data is inherent and stable across lifespan. Therefore insights gained from secondary analyses of genomics data will provide us an unbiased and powerful approach to delineate the underlying relationship between substance use and suicide.\",\n\"The project aims to study the correlation and causation relationship between substance uses and suicidal attempts. The primary outcome phenotype of interest is suicidal attempt, and the exposure phenotype include opioid use disorder, cannabis use disorder, and alcohol use disorder reported in Million Veteran Project (MVP). Genetic correlation, polygenic risk score, and 2-sample Mendelian randomization will be used to investigate the underlying causal links between outcome and exposure phenotype. Instrumental variables will be selected from summary statistics of genome-wide association studies (GWAS). To overcome health disparity across populations, we will investigate both European descent samples and African descent samples. Other cross-phenotype analyses include multi-trait analysis of GWAS, genomic structural equation modeling, and case-case genome-wide association study will also be used to identify shared or distinct genes and pathways. All analyses will be based on summary statistics instead of using genotype data, and this will help to protect the confidentiality of samples in MVP. This study will better delineate the relationship between substance use disorder and suicidal attempt through genomics perspective, and will help to scrutinize potential high-risk suicidal individuals for early intervention and management.  \",\n\"2021, Jul28\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }],\n [\"xiongjian, luo \",\n\"KUNMING INSTITUTE OF ZOOLOGY\",\n\"Integrative analysis of Post-traumatic stress disorder\",\n\"approved\",\n\"Our study will help to find the causal risk variants and genes of PTSD. Further functional exploration of these causal risk variants and genes will uncover the genetic regulation mechanisms and potential disease etiology of PTSD. These results will further improve the disease diagnosis and treatment.\",\n\"Post-traumatic stress disorder (PTSD) is a severe mental health problem which develop by someone who exposure to a life-threatening events. The Million Veteran Program conducted a PTSD GWAS study (dbGaP Study Accession:phs001672.v3.p1) identified several risk loci of PTSD. We will utilize the PTSD summary statistics for further investigation the causal risk variants and genes of PTSD. Based on the PTSD GWAS summary statistics, we will pinpoint the causal mutations by utilize the public available epigenomic data and chip-seq data from FANTOM, ENCODE, and Roadmap projects. Furthermore, we will also prioritizing PTSD genes by integrating the PTSD summary statistics with brain eQTL data by Sherlock, SMR. TWAS (transcriptome-wide association analysis) will also help to find the key genes by integrate PTSD GWAS summary statistics and brain transcriptome data. We claimed that we will only use the summary statistics data for research only and have no intention to request the individual participants data, and there are no additional collaborations in this project. \",\n\"Jun18, 2020\",\n\"phs001672.v10.p1\",\n{\n            'v': 1,\n            'f': \"1\",\n        }]],\n        columns: [[\"string\", \"Requestor\"], [\"string\", \"Affiliation\"], [\"string\", \"Project\"], [\"string\", \"Request status\"], [\"string\", \"Public Research Use Statement\"], [\"string\", \"Technical Research Use Statement\"], [\"string\", \"Date of approval\"], [\"string\", \"accession\"], [\"number\", \"Number of requests\"]],\n        columnOptions: [],\n        rowsPerPage: 10,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n    ",
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Requestor</th>\n",
       "      <th>Affiliation</th>\n",
       "      <th>Project</th>\n",
       "      <th>Request status</th>\n",
       "      <th>Public Research Use Statement</th>\n",
       "      <th>Technical Research Use Statement</th>\n",
       "      <th>Date of approval</th>\n",
       "      <th>accession</th>\n",
       "      <th>Number of requests</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>234</th>\n",
       "      <td>Rose, Peter</td>\n",
       "      <td>UNIVERSITY OF CALIFORNIA, SAN DIEGO</td>\n",
       "      <td>Analysis and Evaluation of RADx-rad Datasets</td>\n",
       "      <td>approved</td>\n",
       "      <td>The National Institutes of Health launched the...</td>\n",
       "      <td>This request supports the RADx-rad program out...</td>\n",
       "      <td>2023, Mar07</td>\n",
       "      <td>phs002522.v1.p1, phs002523.v1.p1, phs002524.v1...</td>\n",
       "      <td>48</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>Anwar, Mohd Mozharul</td>\n",
       "      <td>NIH</td>\n",
       "      <td>Exploration of Wearable Device Data for COVID-19</td>\n",
       "      <td>approved</td>\n",
       "      <td>Wearable devices collect various physiological...</td>\n",
       "      <td>The objective of the proposed research is to e...</td>\n",
       "      <td>Mar27, Dec19, 2023, Feb07, 2022</td>\n",
       "      <td>phs002516.v1.p1, phs002519.v1.p1, phs002523.v1...</td>\n",
       "      <td>11</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>Ciofani, Danielle</td>\n",
       "      <td>BROAD INSTITUTE, INC.</td>\n",
       "      <td>Confirmation of RAS approval workflow for RADx...</td>\n",
       "      <td>approved</td>\n",
       "      <td>I will be conducting testing to confirm that I...</td>\n",
       "      <td>I am a co-I of the RADx Data Hub program, and ...</td>\n",
       "      <td>Nov14, 2022</td>\n",
       "      <td>phs002516.v1.p1, phs002533.v1.p1, phs002534.v1...</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>Miguez, Maria-Jose</td>\n",
       "      <td>FLORIDA INTERNATIONAL UNIVERSITY</td>\n",
       "      <td>System analysis for COVID humoral response</td>\n",
       "      <td>approved</td>\n",
       "      <td>Multisystem inflammatory syndrome in children ...</td>\n",
       "      <td>Multisystem inflammatory syndrome in children ...</td>\n",
       "      <td>Feb10, 2023</td>\n",
       "      <td>phs002781.v1.p1, phs002945.v1.p1</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>Chan, Kei Hang</td>\n",
       "      <td>BROWN UNIVERSITY</td>\n",
       "      <td>NHLBI TOPMed Whole-genome Sequencing Program (...</td>\n",
       "      <td>approved</td>\n",
       "      <td>We use epidemiological, statistical, and bioin...</td>\n",
       "      <td>Many complex traits such as diabetes and cardi...</td>\n",
       "      <td>2022, Dec15</td>\n",
       "      <td>phs002299.v1.p1, phs002363.v1.p1</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>HAN, SHIZHONG</td>\n",
       "      <td>JOHNS HOPKINS UNIVERSITY</td>\n",
       "      <td>Integrative analysis of multi-omics datasets f...</td>\n",
       "      <td>closed</td>\n",
       "      <td>Alcoholism has a genetic basis, but identifica...</td>\n",
       "      <td>Our goal is to identify risk genes underlying ...</td>\n",
       "      <td>2019, Dec30</td>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>Guo, Yan</td>\n",
       "      <td>XI'AN JIAOTONG UNIVERSITY</td>\n",
       "      <td>Genomic analysis for human complex diseases</td>\n",
       "      <td>rejected</td>\n",
       "      <td>Many human complex diseases/traits have strong...</td>\n",
       "      <td>Many human complex diseases/traits are highly ...</td>\n",
       "      <td>2020, Jan13</td>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>Guo, Shicheng</td>\n",
       "      <td>JOHNSON/JOHNSON/PHARM/RES/ DEVELOPMENT</td>\n",
       "      <td>GWAS to identify disease genes, drug targets a...</td>\n",
       "      <td>approved</td>\n",
       "      <td>This study plans to identify genetic differenc...</td>\n",
       "      <td>Large genetic cohorts provide an opportunity t...</td>\n",
       "      <td>2020, Nov13</td>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>Gui, Hongsheng</td>\n",
       "      <td>HENRY FORD HEALTH SYSTEM</td>\n",
       "      <td>Investigation of substance use on risk of suic...</td>\n",
       "      <td>approved</td>\n",
       "      <td>Is there any connection between substance addi...</td>\n",
       "      <td>The project aims to study the correlation and ...</td>\n",
       "      <td>2021, Jul28</td>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>xiongjian, luo</td>\n",
       "      <td>KUNMING INSTITUTE OF ZOOLOGY</td>\n",
       "      <td>Integrative analysis of Post-traumatic stress ...</td>\n",
       "      <td>approved</td>\n",
       "      <td>Our study will help to find the causal risk va...</td>\n",
       "      <td>Post-traumatic stress disorder (PTSD) is a sev...</td>\n",
       "      <td>Jun18, 2020</td>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>312 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "<google.colab.data_table.DataTable object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "#@title Data access requests grouped by requestor\n",
    "def get_download_url(accession):\n",
    "    return \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/GetAuthorizedRequestDownload.cgi?study_id=\" + accession\n",
    "\n",
    "def get_authorized_requests(studies):\n",
    "    authorized_requests = pd.DataFrame()\n",
    "\n",
    "    for _, row in tqdm(studies.iterrows(), total=studies.shape[0]):\n",
    "        try:\n",
    "            df = pd.read_csv(get_download_url(row[\"accession\"]), \n",
    "                             usecols=[\"Requestor\", \"Affiliation\", \"Project\", \"Date of approval\", \"Request status\", \n",
    "                                      \"Public Research Use Statement\", \"Technical Research Use Statement\"],\n",
    "                            sep=\"\\t\")\n",
    "            df[\"accession\"] = row[\"accession\"]\n",
    "            df[\"name\"] = row[\"name\"]\n",
    "            authorized_requests = pd.concat([authorized_requests, df], ignore_index=True)\n",
    "        except:\n",
    "            print(f\"Skipping: {row['accession']} - no data access through dbGaP.\")\n",
    "                                        \n",
    "    return authorized_requests\n",
    "\n",
    "requests = get_authorized_requests(studies)\n",
    "\n",
    "# group requests to create a summary view\n",
    "if requests.shape[0] > 0: \n",
    "    # summary = requests.groupby([\"Requestor\", \"Affiliation\", \"Project\", \"Date of approval\", \"Request status\",\n",
    "    #                             \"Public Research Use Statement\", \"Technical Research Use Statement\"], \n",
    "    #                             as_index=False)[\"accession\"].agg(', '.join)\n",
    "    summary = requests.groupby([\"Requestor\", \"Affiliation\", \"Project\", \"Request status\",\n",
    "                                \"Public Research Use Statement\", \"Technical Research Use Statement\"], \n",
    "                                as_index=False)[[\"Date of approval\",\"accession\"]].agg(\", \".join)\n",
    "\n",
    "    # keep only unique dates\n",
    "    summary[\"Date of approval\"] = summary[\"Date of approval\"].str.split(\", \").apply(set).str.join(\", \")\n",
    "\n",
    "    # show most frequent requests first\n",
    "    summary[\"Number of requests\"] = summary[\"accession\"].str.count(\",\") + 1\n",
    "    summary.sort_values(by=\"Number of requests\", ascending=False, inplace=True)\n",
    "\n",
    "# show results\n",
    "print()\n",
    "print()\n",
    "print(\"Number of data access requests :\", requests.shape[0])\n",
    "\n",
    "if requests.shape[0] > 0:\n",
    "    print(\"Number of unique requestors    :\", len(requests[\"Requestor\"].unique()))\n",
    "    print(\"Number of unique studies       :\", len(requests[\"accession\"].unique()))\n",
    "    display(data_table.DataTable(summary, include_index=False, num_rows_per_page=10))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "2f136e9c-e208-4cc1-95f7-895266e4adbb",
   "metadata": {
    "cellView": "form",
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 3901
    },
    "id": "2f136e9c-e208-4cc1-95f7-895266e4adbb",
    "outputId": "9c125b48-c428-4d28-86ce-5aac96233919",
    "tags": []
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of data access requests : 380\n",
      "Number of unique requestors    : 273\n",
      "Number of unique studies       : 70\n"
     ]
    },
    {
     "data": {
      "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/99dac6621f6ae8c4/data_table.js\";\n\n      window.createDataTable({\n        data: [[\"Aday, Aaron \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Peripheral Artery Disease and Major Depressive Disorder\",\n\"Mar03, 2023\",\n\"approved\",\n\"People with depression are at higher risk of developing coronary artery disease, or cholesterol buildup in the heart arteries. Prior studies suggest that some changes in our DNA they make us more likely to develop depression are the same as some changes that make us more likely to develop coronary artery disease. Cholesterol buildup can affect other arteries, including the leg arteries. This disease is called peripheral artery disease. We do not know if the changes in our DNA that make us more likely to develop depression also make us more likely to develop peripheral artery disease. The goal of this proposed research is to answer this question.\",\n\"Overview Major depressive disorder (MDD) is phenotypically and genetically correlated with coronary artery disease (CAD), but there are limited data suggesting these correlations extend to peripheral artery disease (PAD). We propose to (1) test the association of a PAD polygenic risk score (PRS) with MDD and related biomarkers, (2) stratify such analyses by covariates important in PAD pathogenesis, including sex and diabetes, and (3) use previously published meta-GWAS summary statistics to identify genes and pathways involved in comorbid MDD and PAD.  Study Population The VUMC biobank, BioVU, will serve as the discovery population.  Genotyping and quality control  A subset of BioVU patients (n = 24,262) was genotyped as part of various institutional and investigator-initiated projects on the Illumina MEGAEX platform, which contains more than two million markers. Genotypes were imputed using SHAPEIT/IMPUTE4 with the 1000 genomes phase I reference panel, and variants with INFO < 0.3 were excluded. A subset of SNPs in linkage disequilibrium was used to calculate relatedness and principal components of ancestry using multidimensional scaling in PLINK v1.9. We restricted to a homogenous population of European descent defined by principal components of ancestry to avoid population stratification effects, leaving 18,385 individuals for analyses. Samples were genotyped in five batches, and variants were removed if allele frequencies differed significantly (P < 5 \\u00c3\\u0097 10-5) between any batch and the rest of the sample. Finally, we filtered multiallelic and structural variants, converted dosage data to hard genotype calls, and excluded variants with certainty <0.9 or INFO < 0.95, resulting in 5,218,407 high quality SNPs across the autosomes.  Phenotype data  To facilitate the investigation of comorbidity patterns across the medical phenome, we will map ICD-9 billing codes in the EHR to phecodes. ICD-9 codes will be mapped to 1814 phecode categories according to the Phecode Map v1.2. Patients will be assigned to the case group for a given phecode if they have at least two different ICD-9 codes that mapped to a given phecode, or if they had at least two separate occurrences of a single ICD-9 code that mapped to the given phecode.  PAD is focus of our analysis. Age in cases will be defined by the age of first PAD-defining feature in the EHR, while in controls, age will be defined by the age at last encounter. Data on PAD risk factors were also extracted from the EHR either from structured data or via text-mining algorithms. PAD risk factors included body mass index (BMI), hypertension, type 2 diabetes diagnosis, pre-medication blood levels of high- and low-density lipoprotein cholesterol (HDL and LDL) and of triglycerides, smoking history, and socio-economic status.  We will define MDD by the phecode 296.22 \\u00e2\\u0080\\u009cMajor depressive disorder.\\u00e2\\u0080\\u009d We will also define broad categories of mental illness, to adjust for the possibility that any degree of mental illness could increase PAD risk simply due to lifestyle factors that may be correlated with having the mental illness itself. We will define a milder depressive symptoms phenotype by one or more depression ICD-9 codes (296.2 and 296.3, including all fifth digit subclassifications, and 311), and as a final test of the robustness of associations, we will exclude patients with any psychiatric symptoms. Statistical analyses  Polygenic scores for MDD will be computed using the PRSice software package. In targeted analyses of PAD, we will select the P threshold at which the polygenic score is most strongly associated with PAD risk (i.e., at an P threshold different than 0.05). Risk ratios (odds or hazard) will reflect the risk increase per 1-SD increase in the polygenic score.  Phenome-wide association studies will be conducted with the PheWAS R package v0.12. We will require phecodes to have at least 100 cases, and we will include covariates for sex, median age across the EHR, genotyping batch, and the first 10 principal components of ancestry.  Targeted analyses of PAD in BioVU will employ logistic regression. Minimally adjusted models will include covariates for sex, age, genotype batch, and the first ten principal components of ancestry. Fully adjusted models will additionally include covariates for BMI, hypertension, smoking, type 2 diabetes, blood measurements of HDL, LDL, and triglycerides, highest level of education, a polygenic score for PAD, and depressive symptoms.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Adebamowo, Sally \",\n\"UNIVERSITY OF MARYLAND BALTIMORE\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"Dec14, 2022\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease. Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data. \",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry. Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community. However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"AGRAWAL, ARPANA \",\n\"WASHINGTON UNIVERSITY\",\n\"Neurobiological bases of psychiatric traits\",\n\"Apr08, 2020\",\n\"approved\",\n\"Genetic variation contributes to health outcomes. We will use the results generated by the Million Veteran Program to further our understanding of how polygenic liability influences health-related outcomes.\",\n\"Genetic variation contributes to variability in behavior and health outcomes. We will use summary statistics generated by genome-wide association studies (GWASs) conducted within the Million Veteran Program dataset to examine how genetic variation (e.g., polygenic risk scores, trait-related genetically-conferred differences in gene expression) contributes to individual differences in biology, behavior, and health-related outcomes in diverse samples. The overall objective of this study is to use MVP summary statistics independently and alongside other GWAS summary statistics to identify meaningful pathways (e.g., biology, behavior) through which genetic variation contributes to health outcomes. The study design will include use of these summary statistics to inform studies of health outcomes, meta-analysis of these summary statistics with similar summary statistics from other sources, and for a variety of post hoc genomic analyses such as polygenic risk scoring. We do not plan to combine datasets but rather to combine this summary information with other summary information. Phenotypes will include summary statistics for GWAS related to health, including addictions, as well as behavior and biology.   \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Almasy, Laura \",\n\"CHILDREN'S HOSP OF PHILADELPHIA\",\n\"Polygenic effects on complex traits across development\",\n\"Jun22, 2022\",\n\"approved\",\n\"Genetic markers are an important source of variation in human health and behavior. We will use the results generated from genome-wide association studies (GWASs) conducted in the Million Veteran Program (MVP) to advance our understanding of how multiple genes may influence medical, psychiatric, and substance use outcomes in children and adults.\",\n\"Summary statistics from genome-wide association studies (GWAS) conducted within the Million Veteran Program (MVP) will be used to examine the genetic architecture of complex traits across development. We will use the MVP GWAS summary statistics to compute genetic correlations, create polygenic scores, and employ other statistical techniques, i.e., Mendelian randomization, to examine the genetic contribution to an array of phenotypes, including medical, psychiatric, and substance use symptoms and disorders. The findings will then be tested in independent data sets that include the same medical, psychiatric, and substance use phenotypes in samples that include a broad age span. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Andreassen, Ole \",\n\"UNIVERSITY OF OSLO\",\n\"TOP study\",\n\"Apr21, 2020\",\n\"approved\",\n\"Severe mental disorders such as schizophrenia, bipolar disorders and major depression are severe mental illnesses with high degree of heritability. Still, the disease mechanisms are not yet known. The study will combine genetic data from samples from several populations, to investigate genetic risk factors. Further, the study will investigate overlapping genetic factors among several psychiatric disorders, including schizophrenia, bipolar disorder, major depression, autism and ADHD, and with important comorbid mental disorders such as substans use disorders, somatic disease such as cancer, and cardiovascular disease, and link to mental traits, such as sleep patterns. \",\n\"The overall research objective is to gain more knowledge of genetic factors involved in development of severe mental disorders (eg  neurodevelopmental disorders, schizophrenia, bipolar and depression disorder and substance use disorders), and overlapping phenotypes - to learn more about the pathophysiology and improve clinical treatment. The objective is to identify genes related to these disorders, and potential overlapping phenotypes and shared genetic architecture and pleiotropy mechanisms:  1) Focus on gene discovery with association studies, including newly identified genes for neuronal migration, neurotransmission, synapse function, and immunological factors. Here we will use use gene variants recently identified in association studies and investigate if these findings are replicated in independent samples. We will use diagnoses together with phenotype characteristics. 2) Focus on overlapping genetic architecture among psychiatric disorders and mental traits. There are indications that the same genetic factors could lead to different clinical phenotypes. We wish to investigate if the genetic risk factors are overlapping between schizophrenia, bipolar disorders and major depressive disorder, and between other mental disorders such as substance use disorders, eating disorders, anxiety, autism, OCD, PTSD, ADHD or with dementia, another brain disorder with comorbid psychiatric symptoms, and a range of mental traits such as cognition, mood and sleep characteristics. 3) Focus on comorbid substance use, including smoking. Both in schizophrenia and bipolar disorder, there is a 2-3 fold higher prevalence of smoking and 50% comorbidity with substance use disorders in general, including alcohol. We will investigate potential overlapping genes between substance use traits and disorders  and severe mental illness, both in terms of overlapping genetic architecture, and specific overlapping gene loci. We will also investigate if these risk genes individually or polygenic effects, lead to higher risk of secondary diseases, such as cancer or other somatic disorders, which from epidemiological evidence seems to be relatively high in severe mental illness.  4) Focus on somatic comorbidity, including cardiovascular disease. There is a severely increased mortality in mental illness, especially schizophrenia and bipolar disorder, but also major depression. We will investigate potential overlapping genes between cardiovascular disease and severe mental illness. We will also investigate if such genes are enriched in patients with comorbid heart diseases. There is some epidemiological evidence for overlapping pathophysiology, and unmedicated early phase patients seem to have indicated abnormal heart conditions in severe mental illness.   For these approaches it is necessary to have a good estimates of the genetic variation in the general population.  We will focus on diseases involving adults (comorbidity etc.) but also compare with ADHD and autism since these are neurodevelopmental disorders which can lead to later adult psychopathology.   We will use standard statistical tools for analysis of gene variants associated with diseases and phenotypes, and use novel statistical tools for pleiotropy based on annotation and enrichment methods, and mathematical modeling including MiXeR, condFDR and multivariate approaches, such as MOSTest and new prediction tools, including PHS.   The proposed research is consistent with Use Restrictions for the requested dataset, since the data will be used for gaining more knowledge about disease mechanisms of the severe mental disorders, and strictly for the purpose outlined in this application, and we will follow the regulations about data security, publication and IP regulations.  Will not share data with other persons than the listed personnel from Univ of Oslo.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Andreassen, Ole \",\n\"UNIVERSITY OF OSLO\",\n\"Enrichment methods to improve gene discovery in neuropsychiatric disorders\",\n\"May25, 2022\",\n\"approved\",\n\"We will develop and apply novel statistical genetics methods to improve gene discovery in neuropsychiatric disorders and comorbidities, and improve risk prediction / stratification based on genetic data. This will provide more knowledge about genetic factors related to neuropsychiatric diseases, including functional understanding and risk prediction may have implications for clinical use, including stratification. All the results of the project will be made available in accordance with IRB and data protection regulations\",\n\"Neuropsychiatric disorders are highly heritable, but few genes have yet been detected. We propose to leverage a new statistical framework based on annotation and pleiotropy enrichment, to improve gene discovery in genome-wide association studies (GWAS), reveal genetic architecture and improve risk prediction in neuropsychiatric disorders.   There are certain regions of the genome that are more likely to have an effect (genic annotation), and we will use this information to improve statistical methods. Further, there are overlapping phenotypes in neuropsychiatric disorders, with similar symptoms profiles in several diseases. Further, there is a comorbidity with other disorders. It is likely that such overlap can be explained by common underlying genetic factors (genetic pleiotropy).  Our methodology has the potential to improve the power of traditional GWAS, increase the likelihood of identifying replicable gene variants and improve phenotypic prediction from SNP data (generalization performance). We will now apply our statistical approach to psychiatric disorders.  The overall aim is to discover new genes in schizophrenia, bipolar disorder, major depression, ADHD, autism, anxiety, dementia and related neurodegenerative disorders, substance use disorders, including alcohol use with the following sub-aims:  1) Assess the validity of our statistical approach using completed GWAS p-values in psychiatric disorders 2) Leverage genomic annotation information to improve gene discovery in psychiatric disorders 3) Leverage associated phenotypes (genetic pleiotropy) to improve gene discovery in psychiatric disorders 4) Use the information obtained in 1-3 to improve SNP-based prediction (generalization performance) in neuropsychiatric disorders, and identify functional pathways.  Methods:  We will use in-house pipeline for annotation of different genetic markers. We will leverage overlapping phenotypes for pleiotropy enrichment (immune-, cardiovascular- and neurological diseases, cancer and anthropometry), and diverse genetic categories for annotation enrichment. We will use a statistical framework based on bayesian approach, including Q-Q plots, FDR and replication rate, and  mixture model (CM3, BGMG, univariate causative mixture models (MiXeR), bivariate MiXeR, Finemap and GSA MiXeR, the recent MOSTest multivariate method and extensions, including functional information and mediation (mendelian randomization) approaches.  All analyses are done at Univ. of Oslo. No data are shared with third parties. We will provide summary statistics (anonymous data) as part of publications, if the IRB approvals allow. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"ASHLEY-KOCH, ALLISON \",\n\"DUKE UNIVERSITY\",\n\"Replication of genetic risk for psychiatric disorders\",\n\"Sep21, 2021\",\n\"approved\",\n\"Genetic risk factors for psychiatric disorders are commonly associated with small effect sizes.  As a result, replication of genetic findings in an independent cohort is important to establish the validity of the findings.  This project will utilize results of psychiatric studies within the MVP cohort as a replication opportunity for other ongoing psychiatric genetic studies.\",\n\"Data from the psychiatric phenotypes of the Million Veterans project will be used to replicate genetic risk factors in non-MVP cohorts.  This will involve single genetic variant replications, as well as generation and evaluation of polygenic risk scores.  We are not planning on combining data from MVP with other studies, with the exception of possibly performing meta-analysis.  But no direct combining of data will occur.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"ASSIMES, THEMISTOCLES \",\n\"STANFORD UNIVERSITY\",\n\"Mendelian Randomization Studies of Atherosclerosis Related Traits and Chronic Diseases\",\n\"Apr03, 2023\",\n\"approved\",\n\"Some human traits (risk factors) that predispose to heart attacks and strokes have been found to be related not only to these atherosclerosis related conditions but also a variety of other diseases or traits.  It is often not clear if these risk factors cause these conditions or are a consequence of these conditions.  In this study, we will use genetics to helo determine whether all, some, none of these relationships are causal. We will take advantage of the wide range of chronic disease outcomes available in dbGAP to examine these relationships for several risk factors of atherosclerosis. \",\n\"Some CVD risk factors have not only been linked to CVD but also to a number of other chronic complex diseases.  The objective of this study is to implement a Mendelian Randomization (MR) approach to identify causal associations between any known or purported atherosclerosis related trait (e.g. lipid, glucose, and insulin levels, age at natural menopause, etc.) and any other chronic disease that has been linked to that same trait (e.g. atherosclerosis related outcomes, other CV diseases, cancers, fractures and osteoporosis, neurological disorders, etc.).  Established genetic variants for exposures of interest will serve as MR instruments.  We will then test for association between these variants and a set of chronic diseases. For each study, the set of diseases will be defined by previous publications or our own novel hypotheses.  For example, one hypothesis could be to test for causality using MR between LDL levels and not only multiple cancers (lung, breast,endometrial), but also CHD, stroke, and Alzheimer's disease.  We may test several hypotheses in this fashion separately with each study providing us with insights on the pleiotropic link between a given atherosclerosis related exposure and a set of diseases.  Individual dbGAP datasets will ALWAYS be used in compliance with their data use certificates and their data use limitation to only examine causal (genetic) associations between an exposure of interest and the phenotype(s) listed in the data use limitation for that specific dataset.  The data requested in the application will be used only for genetic studies to learn about, prevent, or treat chronic diseases in general. Data use will be limited to research and will not be used in any commercial applications or patents.  The data requested in this proposal may be combined with non dbGAP datasets but these non dbGAP datasets will always be de-identified datasets of IRB approved research studies on chronic diseases that have been performed by other institutions or organizations which do not deposit to dbGAP (e.g. UK Biobank).   We do not believe that this creates any additional risks to participants. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"ASSIMES, THEMISTOCLES \",\n\"STANFORD UNIVERSITY\",\n\"Genetic Determinants of atherosclerosis and risk factors for atherosclerosis, addition of deidentified data from other studies\",\n\"Jul30, 2020\",\n\"approved\",\n\"The Division of Cardiovascular Medicine at Stanford University has a long history of involvement in human genetic studies related to atherosclerosis. Atherosclerosis, or hardening of the arteries, is responsible for many serious health conditions including heart attacks, strokes, and leg amputations.  The inherited basis of all of these rather common conditions is very complex with multiple DNA mutations each appearing to have subtle effect on risk.  In this context, the challenge of identifying mutations that predispose to these conditions can only be overcome through the analysis of multiple large data-sets including those available through dbGAP.  The development of novel methods to analyze genetic data is also important.  Members plan to responsibly use the data available on dbGAP to support the evidence for association between one or more mutations and conditions related to atherosclerosis in other studies they are involved in.  This exercise will allow them to plan the next phase of population genetic studies. They will also use the data to help plan basic science experiments that will shed light on why some mutations predispose to atherosclerosis and to test novel statistical algorithms developed by Stanford faculty. Individuals using this data will publish or otherwise broadly share any findings from their studies with the scientific community.  They also will honor all embargo dates associated with any of the publicly available datasets.\",\n\"The Division of Cardiovascular Medicine at Stanford University has a long history of involvement in human genetic studies related to atherosclerosis. Faculty in the division have served as PIs or co-investigators in multiple NIH grants with a primary aim of identifying genetic determinants of cardiovascular medicine related traits including but not limited to hypertension, high cholesterol, high blood pressure, diabetes, obesity, smoking, insulin resistance, coronary atherosclerosis, peripheral arterial disease including aortic aneurysms, and cardiomyopathies. Members of the division are currently active in several related GWAS consortia. Members are also collaborating with bio-statisticians at Stanford who are interested in developing novel methods of analyzing genetic data but need individual level data in large numbers to carry out such projects effectively.  Finally, faculty members in the division also work with molecular biologists at Stanford who are trying to unravel the mechanism behind some of the most intriguing genetic associations discovered to date related to CVD .  Project members will use the genetic and phenotype data to augment evidence for association with a CVD related phenotypes derived from other human genetic studies.  They will also use the multi-ethnic data to better understand the haplotype structure in established regions of association with CVD The data will also be used to test novel statistical methodologies or applications of existing methodologies developed by Stanford faculty and to plan future genetic studies including studies that will test the predictive value of multi locus genetic risk scores of CVD.  Individuals using this data will publish or otherwise broadly share any findings from their studies with the scientific community.  They also will honor all embargo dates associated with any of the publicly available datasets.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Auer, Paul \",\n\"MEDICAL COLLEGE OF WISCONSIN\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"Mar29, 2023\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease. Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data.\",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry. Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community. However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Auton, Adam \",\n\"23ANDME, INC.\",\n\"Discovery & Development of New Medicines & Vaccines using GWAS Summary Statistics\",\n\"Jul05, 2021\",\n\"approved\",\n\"Genome-wide association studies (GWAS) identify genetic variants associated with complex diseases which can be useful in identifying new therapeutics and optimizing the development of existing therapeutics.  We will combine results from multiple GWAS studies to create well-powered studies to conduct this research.  Scientists with expertise in human genetics will interpret the results.   \",\n\"We will use the GWAS summary statistics for the purposes of research focused on identifying genes associated with patient outcomes which can be used to discover or optimize development of new medicines and vaccines.  This research is conducted by performing meta-analyses of multiple genetic association studies to create a well-powered study of the trait of interest and colocalization with expression QTL results. These GWAS summary statistics will be meta-analyzed with GWAS summary statistics derived from analysis of UK Biobank, FinnGen, 23andMe and other publicly available data sets and colocalization with expression QTL summary statistics including Blueprint and GTEX and other publicly available datasets.     We also plan to use the GWAS summary statistics to evaluate different strategies for fine and gene mapping, e.g. via colocalization to expression, splicing and other QTL datasets.  This research may include use of results in research collaborations, but the GWAS summary statistics will not be passed to any third party. Any publication relating to this work would include appropriate attribution/credit. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Auton, Adam \",\n\"23ANDME, INC.\",\n\" Study of blood lipid levels in a multi-ethnic cohort\",\n\"Sep14, 2020\",\n\"closed\",\n\"The goal of this project is to explore the genetics of blood lipid levels in more detail by combining them with results from the 23andMe research cohort. Combining these summary statistics will lead to increased power to find new associations and accurately pinpoint disease-causing genes. \",\n\"This proposal aims to use MVPLipids data as an independent source of GWAS summary statistics for blood lipid levels to further explore the genetics of blood lipid levels in the multi-ethnic research cohort.   We will use the provided summary statistics in combination with association statistics from GWASes on the 23andMe cohort to perform meta-analysis and co-localization. Since the power of association and fine-mapping tests increases with sample size, we expect that combining the summary statistics may lead to the discovery of new associations. We will also use the summary statistics to predict lipid levels in the multi-ethnic cohort to examine if the associations have predictive utility.  The use of this data is requested solely for health/medical/biomedical purposes, and does not include any study of population origins or ancestry of the MVPLipids data. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"BACANU, SILVIU-ALIN \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"Opiate use disorders\",\n\"Jul13, 2020\",\n\"approved\",\n\"We plan to use the result from present study to i) run secondary analyses, e.g. to uncover genes, proteins associated with OUD and ii) replicate variant/gene/protein/pathways results found by other means.\",\n\"We plan to use the summary statistics from opiate use disorder (OUD) study to i) run secondary analyses, e.g. to uncover genes, proteins associated with OUD and ii) replicate variant/gene/protein/pathways results found by other means. The preliminary plan of analyses is, for now, 2-step. First, we want to run new TWAS/PWAS gene/protein and pathway analyses using only summary statistics from this opiate use disorder (OUD) study. Second, replicate the findings obtained by imputing OUD phenotypically in UK Biobank.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"BACANU, SILVIU-ALIN \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"Cross species alcohol trait analyses\",\n\"Aug07, 2020\",\n\"rejected\",\n\"We intend to analyze this human study together with our animal ones to discover genes that regulate alcohol traits in humans and animals.\",\n\"We will use summary statistics from this study to impute agnostic and transcriptomic gene statistics for  this study.  Subsequently we will used a weighted FDR approach to combine these human results with the animal model results from Virginia Commonwealth University Alcohol Research Center. Each animal model will use the summary stats from the other animal models to prioritize genes.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Barr, Peter \",\n\"SUNY DOWNSTATE MEDICAL CENTER\",\n\"Prediction of Alcohol Use Disorders across Ancestral Populations\",\n\"Nov09, 2021\",\n\"approved\",\n\"Polygenic scores are becoming of interest for medical practitioners and clinicians. The current analysis will examine the clinical utility of polygenic risk scores for identifying those at extreme risk for substance use disorders. We will focus on individuals of both European and African ancestry to address the diversity gap in genomic research.\",\n\"Polygenic scores derived from GWAS summary statistics are becoming increasingly useful in medical research and there is hope that these will provide actionable information for clinical intervention in regards to substance use disorders. However, the majority of the studies focus on individuals of European ancestry. Here, we plan on using summary statistics from large scale GWAS of alcohol use disorders (AUD), alcohol consumption, and correlated phenotypes (e.g., major depression, other substance use disorders) in both European and African ancestry samples in the Million Veterans Program (MVP) to create polygenic risk scores in  holdout samples with available phenotypic information on substance use disorders. Where possible, we will meta-analyze MVP results with other available summary statistics to increase the predictive power of polygenic scores. The goal of this research will focus on whether the extreme ends of the polygenic score distribution for alcohol-related phenotypes provide any clinically relevant information in individuals of European or African ancestries. We will focus on a variety of substance use disorders including alcohol, cannabis, cocaine, nicotine, and others. Data will be used only for research purposes and we will make no attempt to reveal the individual participants. The proposed research is consistent with the data use limitation noted in the Data Use Certification Agreement for MVP. This project will be in collaboration with Dr. Danielle Dick at Rutgers University as part of ongoing efforts to better understand genetic risk in diverse populations (project ID: 21833).\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Batini, Chiara \",\n\"UNIVERSITY OF LEICESTER\",\n\"The genetics of smoking and drinking behaviours in underrepresented populations \",\n\"Jul28, 2021\",\n\"approved\",\n\"Smoking is a leading risk factor for many of the top ten causes of death world-wide, and  smoking prevalence varies among world regions, with 80% of the 1.3 billion smokers worldwide living in low and middle-income countries. Similarly to smoking, alcohol use is a very important risk factor for disease and disability. Both behaviours have shown a genetic background but most studies have been performed in European individuals. We are now conducting studies in African and admixed individuals and the MVP data will be used to verify the robustness of our findings. \",\n\"Smoking is a leading risk factor for many of the top ten causes of death world-wide, and  smoking prevalence varies among world regions, with 80% of the 1.3 billion smokers worldwide living in low and middle-income countries. A reduction in smoking prevalence is expected in most areas by 2025, with the African and the Eastern Mediterranean regions showing an opposite trend. To date, the largest genome-wide association study (GWAS) of smoking behaviour traits includes 1.2 million individuals and highlights genes coding for proteins involved in neurotransmission of nicotine, dopamine and glutamate. However, as in most GWASs, it only includes individuals of European origin failing on the representation of human diversity worldwide. Similarly to smoking, alcohol use is a very important risk factor for disease and disability. Evidence from large GWASs, mostly in European populations, suggest that genetic susceptibility underpins alcohol use and high genetic correlations have been observed between alcohol and smoking behaviours.   In order to disentangle the role of smoking and drinking behaviours on disease susceptibility we are studying the genetic architecture of both traits in different international cohorts (AWI-Gen, Uganda Genome Resource and Mexican Biobank) as well as in the different ancestral backgrounds represented in UK Biobank, with a special interest in individuals of African and admixed ancestry, by far the least represented in genomic studies.   We are now looking for datasets to perform replication analyses for smoking and drinking behaviour traits in individuals of African and admixed ancestries. The results will be compared with those for European individuals to understand the shared genetic architecture among populations and among traits. The traits of interest include smoking initiation and cessation, alcohol use disorder diagnosis and AUDIT-C score, and any trait closely related to these. To date, MVP is the only available cohort with the ancestries of interest represented and with comparable phenotypes, and with sample sizes that would allow a robust replication analysis for our study.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Below, Jennifer \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Meta-analyses of Type 2 Diabetes and Lipid Traits in Hispanic/Latino Populations\",\n\"Jul13, 2020\",\n\"approved\",\n\"Genetic studies of type 2 diabetes and blood lipid traits have focused primarily on European ancestry populations, though Hispanic/Latino populations have higher rates of these diseases. We are performing a study that will aggregate data from many smaller studies of diabetes and blood lipids in Hispanic/Latino populations to increase our ability to understand genetic factors that contribute to disease.\",\n\"Genetic studies of type 2 diabetes (T2D) and lipid traits in populations of European ancestry have uncovered novel, biologically relevant pathways. However, the predominant focus on Non-Hispanic white ancestral populations limits the global relevance and therefore translational potential of these large-scale genome-wide association study (GWAS) efforts, particularly in populations overlooked by many ongoing GWAS initiatives, but most burdened by T2D and dyslipidemia and their downstream consequences, i.e. Hispanic/Latinos. Therefore, we have assembled the DIAMANTE Hispanic/Latino ancestry group and the Global Hispanic Lipids Consortium to perform large-scale meta-analyses and functionally oriented TWAS analyses of T2D and lipid traits in H/L populations.  We currently have summary statistics from nineteen contributing GWAS of T2D, HDL cholesterol, LDL cholesterol, total cholesterol, and/or triglycerides. We are requesting summary statistics for the T2D and lipids analyses in MVP to incorporate into our meta-analyses. This will involve combining summary statistics generated from MVP data with summary statistics generated in other datasets, however this should pose no additional risk to any of the participants. To examine the meta-regression results in a functionally oriented context, we will then use genetically regulated expression prediction models to determine association of each phenotype with predicted gene expression levels.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Beulens, Joline \",\n\"VRIJE UNIVERSITEIT\",\n\"The causal association of lifestyle factors associated with peripheral artery disease: a Mendelian Randomization approach\",\n\"Mar15, 2021\",\n\"closed\",\n\"Peripheral artery disease (PAD) is one of the most common types of cardiovascular disease. It is a multifactorial disease, meaning it is caused by a combination of genetic and lifestyle factors. Even though many factors are known to play a role in the onset of PAD, classic epidemiological study designs lack the ability of determining causal relationships between risk factors and PAD since they are limited by bias. Therefore, we aim to evaluate the causal relationship between lifestyle related risk factors and the development of PAD via a Mendelian Randomization (MR) approach. Summary statistics of the lifestyle related risk factors type 2 diabetes, lipid metabolism, obesity, smoking, alcohol use, coffee consumption, physical activity, sleep and educational level will be used as instruments, information will be extracted from publically available GWASs. Summary statistics of GWAS of PAD from the MVP program will be used in the MR study as a outcome variable. By determining the causal relationships between lifestyle factors and PAD, a solid basis for targeted disease prevention strategies can be created.\",\n\"Peripheral artery disease (PAD) is one of the most common types of cardiovascular disease, and is estimated to affect over 200 million people worldwide with a prevalence rising sharply with age. The epidemiology of PAD has been studied comprehensively, and a plethora of genetic and lifestyle factors are linked to the development of PAD. However, since PAD can long exist in an asymptomatic state and the severity of symptoms can vary significantly between individuals, it is difficult to accurately define a precise moment of disease onset. For that reason, many studies investigating PAD risk factors are based on cross-sectional associations. Although such studies are potentially informative, bias such as confounding and reverse causation are introduced, which prevents researchers from drawing causal conclusions. To overcome this limitation, we aim to investigate type 2 diabetes, lipid metabolism, obesity, smoking, alcohol use, coffee consumption, physical activity, sleep and educational level as lifestyle related risk factors for PAD by using a mendelian randomization (MR) design, that will enable us to investigate causal relationships between PAD and putative risk factors.  A 2-sample MR design will be used to investigate the relationship between lifestyle factors and PAD. In a MR design, single nucleotide polymorphisms (SNPs), genetic variants that are robustly associated with potentially modifiable risk factors, are used as an instrumental variable. Since SNPs are randomly allocated at meiosis, they are largely free of the classic forms of bias that are observed in observational research (e.g. confounding, reverse causation, measurement error) and can therefore be used as proxies to explore causal effects on an outcome, providing certain assumptions are met.  We will conduct an MR study by using summary statistics from genome wide association studies (GWAS). Data from publically available GWASs will be used to select instrumental SNPs. An SNP will be used as an instrument if associated with the risk factor at the genome wide significance level (P<5x10-8). When linkage disequilibrium (r2>0.1) occurs, the SNP with the strongest correlation with the risk factor will be used. SNPs correlated with type 2 diabetes, lipid metabolism, obesity, smoking, alcohol use, coffee consumption, physical activity, sleep and educational level will be used as instruments. The summary statistics from PAD as analyzed in the MVP GWAS will be used as an outcome variable.  The random-effects inverse variance weighted (IVW) method will be used as main analysis method to combine the Wald estimates for each SNPs (SNP outcome estimates divided by SNP exposure estimate where the intercept is constrained to zero) into one causal estimate of each risk factor. The IVW method will provide an unbiased estimate in the absence of horizontal pleiotropy or when horizontal pleiotropy is uncorrelated with SNP-confounder associations and is balanced. Sensitivity analyses will be performed to assess the validity and robustness of the IVW by assessing the presence of possible pleiotropy. Firstly, we will perform complementary analyses using the weighted median and MR-egger regression methods which are relatively robust under different assumptions about pleiotropy, although at the cost of statistical power. Secondly, the MR pleiotropy residual sum and outlier test will be used to identify outlying SNPs reflecting likely pleotropic biases, in which it will be checked whether exclusion of the outlying SNP changes the causal estimate.  All analyses will be performed in RStudio (The R Foundation for Statistical Computing). Results will be reported as odds ratios (OR) with corresponding 95% confidence intervals. Since only summary statistics and no individual-level data will be used in these combined analyses, participants are not exposed to any additional risks.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Bhatt, Ishan \",\n\"NORTHERN ARIZONA UNIVERSITY\",\n\"Investigating the genetic basis of tinnitus and hearing disorders\",\n\"Jun29, 2020\",\n\"closed\",\n\"Tinnitus, the phantom perception of sound in the absence of an external sound source, is a prevalent hearing disorder. The primary goal of this study is to identify detailed phenotypic and genotypic profiles of tinnitus and related hearing health concerns (e.g., noise-induced hearing loss). This research will be useful to construct a genetic risk profile that can be used by health-care providers and educators to identify individuals genetically at risk for tinnitus. \",\n\"Title: Investigating the genetic basis of tinnitus and hearing disorders Objectives of the proposed project: The primary objective of the proposed project is to identify the genetic factors underlying tinnitus, commonly known as ringing in ears/head and tinnitus-related distress. This project will also investigate the genetic factors underlying other hearing disorders that are common in individuals exposed to occupational noise (e.g., noise-induced hearing loss). Study design: Over the past decade, genome-wide association studies (GWAS) have identified numerous genetic variants associated with common diseases and traits [1]. The small fraction of disease heritability explained, thus far, by GWAS findings leads to prediction models that generally fail to accurately and comprehensively predict which individuals will develop the disease in the future [2]. However, it is projected that comprehensive polygenic risk prediction models will provide clinically useful predictions of disease risk if the limitations of current GWAS can be overcome [3]. The present study will evaluate the genotype-phenotype relationship for tinnitus, tinnitus-related distress, and hearing disorders that are common in individuals exposed to occupational noise (e.g., noise-induced hearing loss). We will employ machine learning strategies to identify the genetic association with hearing disorders. The primary dataset we will evaluate this approach is on the Veterans Administration (VA) Million Veteran Program (phs001672.v3.p1). Analysis plan: We will extract SNP genotypes from complete genomes by using Plink, a whole-genome association analysis toolset designed to perform a range of large-scale analyses. We will extract the phenotype variables related to tinnitus, tinnitus-related distress, and other hearing measures (e.g., hearing sensitivity). We will identify individuals with and without tinnitus. We will start by performing odds ratios, regression, and principal component analyses to detect an initial set of SNPs associated with tinnitus groups. Using the SNPs as predictor variables and tinnitus groups as a response, SNPs exhibiting a p-value<0.05 for the ORs, principal component correlations, or regression t-tests will be regarded as significantly associated to hearing loss. P-values between 0.05 and 0.07 will be regarded as marginally significant. After identifying the possible associated SNPs, with either significant or marginally significant associations, we will build mixed models for additional regression analyses in order to identify possible confounding factors that may amplify or cancel the effect of a given SNP over the prediction power of the statistical models. Finally, we will identify possible cooperation effects among SNPs by combining them as interaction terms in the mixed models. We will apply machine learning techniques such as random multi-model inference, parameter exchange, and model selection in order to test all possible combinations and interactions among all SNPs analyzed in a feasible computational time. For comparing the quality and accuracy of the statistical models, we will use Aikaike Information Criterion (AIC), Bayesian information criterion (BIC), Area Under the Curve (AUC), ANOVA, and likelihood-ratio tests. We will employ other appropriate methods to identify a confluence of genetic and other health-related factors for tinnitus and related hearing-health concerns (e.g., tinnitus-related distress and noise-induced hearing loss). SNPs with the p-value <5\\u00c3\\u009710-8 in both samples will be regarded as a statistically significant association with the phenotypes. SNPs with the p-value <0.05 and >5\\u00c3\\u009710-8 in both samples will be regarded as a potential association with the phenotypes.  References: 1. Welter, D., et al., The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. Nucleic Acids Res, 2014. 42(Database issue): p. D1001-6.  2. Wray, N.R., et al., Pitfalls of predicting complex traits from SNPs. Nat Rev Genet, 2013. 14(7): p. 507- 15.  3. Chatterjee, N., et al., Projecting the performance of risk prediction based on polygenic analyses of genome-wide association studies. Nat Genet, 2013. 45(4): p. 400-5, 405e1-3. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Bick, Alexander \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"NHLBI TOPMed Whole-genome Sequencing Program (TOPMed WGS)\",\n\"Mar10, 2021\",\n\"approved\",\n\"The goals of the TOPMed Whole Genome Sequencing program are to identify genetic risk (and protective) factors for disease, to identify subtypes of disease, and to develop targeted and personalized treatments for disease.  The program consists of multiple studies addressing these goals for phenotypes focused on heart, lung, blood and sleep disorders, which have a high impact on public health.  These studies will include individuals from diverse populations, including those historically under-represented in health research.\",\n\"Phenotypic and genomic data from TOPMed studies will be used for: (a) investigating genotype-phenotype associations; (b) variant summary statistics, (c) common controls in association tests, (d) reference panels to impute into un-sequenced individuals from TOPMed; and (e) population genetic studies.  TOPMed data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Data from an individual with disease-specific consent will not be used in analyses outside of that restriction, including indirect uses such as imputation reference panels, common controls for association studies and variant summary statistics. For example, an individual with consent restricted to lung function studies will not be used to impute data intended for analyses of diseases not clearly related to lung function.  When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish or otherwise broadly share any findings from this study with the scientific community.  This application anticipates collaboration among TOPMed investigators from multiple institutions, who are named in a master list appended to the IRB document uploaded with this application. Within a set of applicants approved for access to the TOPMed Exchange Areas, data may be shared in a cloud environment or by direct transfer, provided that they all agree among themselves to allow such sharing, and that all have approved TOPMed Data Access Requests for the specific data sets to be shared. Otherwise, data will be shared only among Internal Collaborators \\u00e2\\u0080\\u0093 i.e. within an institution.  Data from TOPMed may be combined with non-TOPMed data from the same or other studies (obtained from dbGaP or other sources), to improve the power for novel genetic discoveries, while respecting the consent of all participants. We expect that this activity creates no additional risks to participants.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Bountress, Kaitlin \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"Investigating Shared Risk Factors for Comorbidity \",\n\"Aug11, 2020\",\n\"approved\",\n\"Traumatic events are common, and increase risk for adverse outcomes. Increased alcohol use/consumption (AC), alcohol use disorder (AUD), recreational cannabis use (RCU), cannabis use disorder (CUD), opioid use disorder (OUD), cigarettes per day (Cpd) current and past, smoking initiation (SmkInit), trajectories, and cessation (SmkQuit), and posttraumatic stress disorder (PTSD) may occur following trauma, and may co-occur. Additionally, there is evidence suggesting that shared genetic factors may partially account for this co-occurrence. This project seeks to use summary statistics for PTSD,  and AUD, RCU, CUD, OUD, Cpd current and past,  SmkInit, smoking trajectories, and SmkQuit as well as PTSD and RCU, from the Million Veteran Program (MVP) in order to understand the potential shared molecular genetic underpinnings of the PTSD-AC/AUD and PTSD-RCU comorbidity. This research will help to move forward the literature on potential shared risk factors for this debilitating comorbidity.\",\n\"The focus of this project is on understanding the etiology of comorbid conditions, specifically Posttraumatic Stress Disorder (PTSD), alcohol and substance  phenotypes and positive phenotypes (e.g., positive subjective well-being, life satisfaction, openness to experience).   Trauma is common, and associated with a host of deleterious outcomes. Posttraumatic Stress Disorder (PTSD) is one of the most common disorders resulting from trauma, tending to co-occur with alcohol consumption (AC), alcohol use disorder (AUD), and recreational cannabis use (RCU), cannabis use disorder (CUD), opioid use disorder (OUD) and cigarette consumption (e.g., cigarettes per day). Much of the research on the comorbidity of PTSD and alcohol phenotypes has focused on PTSD-AUD. As increased consumption of alcohol is associated with AUD, research is needed to determine whether the same etiologic processes underlying PTSD-AUD comorbidity are those underlying PTSD and AC. Similarly, there is limited research focused on the etiology of PTSD and RCU, CUD and cigarette smoking. .  Thus, the proposed study will use large-scale de-identified data and employ cutting-edge methods to broadly investigate the genetic architecture of PTSD, AUD-related phenotypes, AC, RCU, CUD, OUD, current and past cigarettes per day (Cpd), smoking (SmkInit), trajectories, and cessation (SmkQuit),  and their comorbidity. Specifically, this project proposes to use GWAS summary statistics for PTSD, AUD, maximum alcohol intake, AUDIT-C (consumption), and lifetime RCU, lifetime OUD, current and past cigarettes per day (Cpd), smoking initiation (SmkInit), trajectories, and cessation (SmkQuit)  from a number of large consortia datasets, including Million Veteran Program (MVP) in order to better understand: 1) the genetic correlations between PTSD-AUD, PTSD-AC, and PTSD-RCU, PTSD-CUD, PTSD-OUD, PTSD-Cpd current and past, PTSD-SmkInit, PTSD-smoking trajectories, and PTSD-SmkQuit, 2) whether aggregate genetic risk for AUD/AC predict PTSD, whether aggregate genetic risk for PTSD predicts AUD/AC, whether aggregate genetic risk for RCU predicts PTSD, and whether aggregate genetic risk for PTSD predicts RCU, whether aggregate genetic risk for PTSD predicts CUD, whether aggregate genetic risk for PTSD predicts OUD, whether aggregate genetic risk for PTSD predicts past and current Cpd, whether aggregate genetic risk for PTSD predicts SmkInit, whether aggregate genetic risk for PTSD predicts smoking trajectories, and whether aggregate genetic risk for PTSD predicts SmkQuit, 3) if PTSD and AUD/AC are related to one another in a causal manner,  and if PTSD and RCU are related to one another in a causal manner, if PTSD and CUD are related to one another in a causal manner, if PTSD and OUD are related to one another in a causal manner, if PTSD and current and past Cpd  are related to one another in a causal manner, if PTSD and SmkInit are related to one another in a causal manner, if PTSD and smoking trajectories are related to one another in a causal manner, if PTSD and SmkQuit  are related to one another in a causal manner and 4) the genetic factor structure of PTSD, alcohol, and cannabis, opioid, and cigarette smoking phenotypes. In order to accomplish these goals, we will employ the use of linkage disequilibrium score regression (LDSR; 1), polygenic risk scores PRS (2), mendelian randomization (MR; 3), and genomic structural equation modeling (gSEM; 4). A combination of Unix and R/RStudio will be utilized in order to conduct the proposed analyses. Only summary statistics will be utilized. Analyses will be split by ancestry, specifically for European Ancestry and African Ancestry separately, as well as split by sex. The proposed study represents an important step forward in clarifying risk factors for this co-occurrence.  These analyses will be conducted by researchers at Virginia Commonwealth University. Dr. Kaitlin Bountress is the PI for the proposed study. Dr. Bountress, with collaboration from colleagues (including Drs. Ananda Amstadter, Todd Webb, Kenneth Kendler, Silviu Bacanu, Nathan Gillespie) and graduate students will complete these analyses.   **Update to application: we will be extending these analyses to include other substance outcomes; we are requesting summary stats for: recreational cannabis use (RCU), problematic alcohol use (PAU), opioid use disorder(OUD), cannabis use disorder (CUD), cigarettes per day (CPD; current and past), SmkTrajectory, SmkInit, SmkQuit.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Brumpton, Ben \",\n\"NORWEGIAN UNIV OF SCIENCE &amp; TECHNOLOGY\",\n\"Investigating the causal effects of childhood and adulthood adiposity on later life mental and sleep health outcomes: a mendelian randomisation study\",\n\"Dec05, 2022\",\n\"approved\",\n\"We want to see the effect of childhood and adulthood adiposity on later life depression, anxiety and insomnia. As anxiety is one of our outcome we will use anxiety data to see whether childhood adiposity has its direct effect on later life anxiety or its effect goes through adulthood adiposity. For our research purpose we need genetic data of anxiety as it is not freely available.\",\n\"Anxiety is one of our outcome. We will use anxiety summary statistics in our two sample mendelian randomisation study. We are not going to combine the data with any other data set. Objective: To investigate the causal effects of childhood adiposity on later life depression, anxiety and insomnia independent of adulthood adiposity. Method: Two-sampled univariable and multivariable Mendelian randomization (MR). Analysis plan: Investigate the total effect of childhood body size and direct effect after accounting for adulthood body size on depression, anxiety and insomnia. We are using summary statistics of childhood and adulthood adiposity, depression, insomnia from different large GWASH consortium. As anxiety data is not freely available so, we want to request for summary statistic of anxiety.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Cebola, Ines \",\n\"IMPERIAL COLLEGE OF SCIENCE, TECHNOLOGY AND MEDICINE\",\n\"The contribution of human cell types to NAFLD\",\n\"Mar26, 2023\",\n\"approved\",\n\"Non-alcoholic fatty liver disease (NAFLD) is a progressive disease estimated to affect up to a third of the World's population that arises from a combination of genetic and life style factors. Despite being a significant contributor to higher mortality and a leading cause of liver transplantation and liver cancer in its more advanced cases, many of the molecular mechanisms underlying NAFLD are still not known. This lack of knowledge is explained, at least in part, by the high cellular heterogeneity of the liver, which is comprised of many different cell types but where about 80% of the cell mass is comprised of a single cell type - hepatocytes. As a result of this, many of the current and previous studies were only able to investigate molecular mechanisms of NAFLD in hepatocytes. This gap in knowledge is particularly acute in the investigation of genetic mechanisms.   In this project, we are using isolated liver cells to develop maps of active genes and other regions of the genome that act as 'DNA switches' to turn genes on. We envision that by looking at individual and less explored types of cells from the human liver in combination with DNA variants that change our risk of developing NAFLD, we will understand better why some people are particularly susceptible to develop NAFLD.\",\n\"Our study seeks to address distinct research hypotheses that were not the focus of the primary studies. Specifically, we seek to investigate what is the contribution of different human cell populations to nonalcoholic fatty liver disease (NAFLD) genetic risk, including the 'NAFLD proxy traits'. To this end, we request access to the full summary-level association data from the multiancestry, EA, AA, HISP and ASN analyses for unexplained chronic ALT elevation as published by Marijana et al. (https://doi.org/10.1038/s41588-022-01078-z), which is to date the largest GWAS dataset for a NAFLD proxy trait. The association data will be used to test the enrichment of associated variants at specific categories of cis-regulatory elements, which are currently being mapped by my team using different experimental models and sets of primary samples. These datasets include ATAC-seq and ChIP-seq/CUT&RUN/CUT&Tag from primary hepatocytes, liver sinusoidal endothelial cells, and human pancreatic islets, and single-nuclei ATAC-seq/CUT&Tag from human liver samples. We will use publicly available ENCODE datasets for other tissues and cell types as controls. With these analyses, we will identify genomic features and cell types relevant to NAFLD and NAFLD-related traits.   Enrichment of variants at specific cis-regulatory element categories will be tested using fgwas (https://github.com/joepickrell/fgwas), GREGOR (https://genome.sph.umich.edu/wiki/GREGOR), GARFIELD (https://www.ebi.ac.uk/birney-srv/GARFIELD/), and LD-score regression (https://github.com/bulik/ldsc).  The data will be tested separately from other genetic association datasets and all data analysis will be carried out in our secure high-throughput computing facility (https://www.imperial.ac.uk/admin-services/ict/self-service/research-support/rcs/), we will not use cloud computing for this project.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"CHANG, SUHUA \",\n\"PEKING UNIVERSITY\",\n\"Association between sleep traits and the risk of posttraumatic stress disorder: a mendelian randomization study\",\n\"Dec30, 2019\",\n\"closed\",\n\"Increasing evidence shows that pretrauma factors, such as sleep traits, are associated with increased risk for PTSD following traumatic exposure, pointing to a potential modifiable target for prevention. However, the causality and direction of these associations remains unclear. The 2-sample MR was performed to examine whether sleep traits have a causal effect on risk of PTSD, and vice versa. As genotypes are randomly allocated at conception, they are largely free from confoundings that could bias the typical observational findings. In addition, the MR method reduces the potential bias that are common in conventional observational studies such as confounding, reverse causation and measurement error, provided certain assumptions are met. Summary statistics for genetic variants were identified from the largest available, nonoverlapping GWAS of sleep traits (n = up to 1331010 individuals) and PTSD reexperiencing symptoms from the genome-wide association meta-analysis conducted in the US Million Veteran Program from predominantly European descent.\",\n\"Research objectives and study design: Increasing evidence shows that pretrauma factors, such as sleep traits, are associated with increased risk for posttraumatic stress disorder (PTSD) following traumatic exposure, pointing to a potential modifiable target for prevention. However, the causality and direction of these associations remains unclear. The global lifetime prevalence of PTSD is approximately 1.3 to 12.2%, with a 12-month prevalence ranging from 0.2 to 3.8%. Evidence from epidemiologic studies, which showed that PTSD was not an inevitable consequence of trauma, also suggested that many variables previously considered to be the outcomes of trauma are likely to be pretrauma risk factors associated with PTSD. Although much of the literature on PTSD risk has focused on the effects of cognitive ability and educational attainment (years of schooling), less studies investigate the potential adverse effects of sleep disturbance and traits such as sleep duration, chronotype (morning or evening preference) and insomnia. The 2-sample mendelian randomization (MR) were performed to examine whether sleep traits have a causal effect on risk of PTSD, and vice versa. MR analyses use genetic variants robustly associated with potentially modifiable risk factors (e.g., sleep traits) as proxies to explore causal effects on diseases (e.g., PTSD). As genotypes are randomly allocated at conception, they are largely free from confoundings that could bias the typical observational findings. This, as well as the invariant nature of genetic variants, provides an analogy to randomized clinical trail, in which exposure is allocated randomly and is not affected by subsequent disease. The MR method reduces the potential bias that are common in conventional observational studies such as confounding, reverse causation and measurement error, provided certain assumptions are met. We retrieved summary statistics for the association between genetic instruments and self-reported sleep traits from the respective genome-wide association studies (GWAS) conducted among individuals of European ancestry. We tend to retrieved summary statistics for PTSD re-experiencing symptoms from the genome-wide association meta-analysis conducted in the US Million Veteran Program (146,660 European Americans and 19,983 African Americans). Analysis plan: Two sample MR analyses were conducted using the R package TwoSampleMR, which harmonizes exposure and outcome data sources containing information on selected instrument SNPs (effect alleles, effect sizes (\\u00c3\\u009f statistics), standard errors, P values and effect alleles frequencies). For each direction of potential effect, we used an random-effects inverse variance weighted (IVW) meta-analysis as main analysis method to combined the Wald estimates for each SNPs (SNP outcome estimates divided by SNP exposure estimate where the intercept is constrained to zero). IVW method can provided an unbiased estimate in the absence of horizontal pleiotropy or when horizontal pleiotropy is uncorrelated with SNP-confounder associations and is balanced. To account for horizontal pleiotropy, we performed complementary analyses using the weighted median and MR-egger regression methods which are relatively robust under different assumptions about pleiotropy, although at the cost of statistical power. To further detect and correct the obtained causal estimates, we performed MR-Pleiotropy Residual Sum and Outlier (MR-PRESSO) test to identify and remove any outliers reflecting likely pleiotropic biases. We also conducted tests for significant results using meta-analytic methods to detect heterogeneous outcomes, including leave-one-out analyses, Cochran\\u00e2\\u0080\\u0099s Q statistics and the MR Egger intercept test of deviation from null. To assess the robustness of causal estimates obtained using SNPs associated with self-reported sleep traits, we conducted MR analyses using genetic variants associated with objective accelerometer-based sleep traits from GWAS in UK biobank (up to 85830), including 11 independent SNPs associated with nocturnal sleep duration (to be compared with self-reported sleep duration), 6 SNPs associated with timing of the least active 5 h (L5 timing; to be compared with self-reported chronotype of morning preference) and 21 SNPs associated with number of nocturnal sleep episodes (to be compared with self-reported insomnia symptoms). All analyses were performed using R version 3.4.3 (The R Foundation for Statistical Computing). \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Chasman, Daniel \",\n\"BRIGHAM AND WOMEN'S HOSPITAL\",\n\"Causality and/or shared physiology linking plasma lipids to migraine\",\n\"May17, 2019\",\n\"closed\",\n\"Many studies have observed correlations of elevated levels of circulating LDL-cholesterol (LDL-C) and triglycerides with migraine. However, whether these lipids are causal for migraine is not known. The proposed research would use genetic methods to understand whether the correlation is due to causality.  The outcome of the research is expected to be improved understanding of the biological mechanisms underlying migraine. If lipids are determined to contribute to the cause of migraine, the results would suggest that existing drugs that lower LDL-C and triglycerides may be part of an effective therapeutic strategy for migraine. \",\n\"The comorbidity of migraine and incident cardiovascular disease (CVD) has motivated a search to understand potential shared pathophysiologic mechanisms.  Some of these explorations have focused on one of the most important classes of CVD risk factors, plasma lipid levels for which associations between migraine with lipids have been consistent.  These associations implicate elevated LDL cholesterol (LDLC) and triglycerides but not HDL cholesterol in migraine. Within this context, a recent randomized trial among migraineurs found that the combination of simvastatin, an LDLC lowering agent, and vitamin D significantly reduced migraine frequency compared to placebo. These results raise two non-exclusive possibilities for potential links between the plasma lipids and migraine: there may be a causal (or reverse causal) relationships and/or there may be shared underlying biology. We will implement recent analytic approaches leveraging genetics to evaluate the extent to which each of these two scenarios may be relevant.  The first analytic approach termed \\u00e2\\u0080\\u009cMendelian Randomization\\u00e2\\u0080\\u009d, exploits the randomized and irreversible meiotic assortment of alleles influencing lipid levels to assess whether lifelong exposure to a genetically determined risk factor may be associated with migraine, thereby implying causality.  Using individual level information for migraine, lipid levels, and genome-wide genotype data from the Women\\u00e2\\u0080\\u0099s Genome Health Study (5,122 migraineurs/23,294 total), the UK Biobank (14,392 migraineurs/~500,000 total), we will perform MR, including tests of reverse causality, to compare observational effects with genetically inferred causal effects, which would be further characterized by mediation analysis.  These analyses will be complemented by MR analysis using summary statistics from extremely well-powered, genome-wide association studies of migraine (>56,000 migraineurs, Gormley et al. Nat. Gen. 2016v48p856) and lipid levels (~350,000 individuals Klarin et al. Nat. Gen. 2018v50p1514).  In the second analytic approach, we will estimate the extent of shared genetic influence, or equivalently \\u00e2\\u0080\\u009cshared heritability\\u00e2\\u0080\\u009d or \\u00e2\\u0080\\u009cgenetic covariance\\u00e2\\u0080\\u009d, between each lipid risk factor and migraine in these same data sets while also identifying specific genetic variants with shared associations.  Moreover, we will use computational approaches to infer which tissue types and/or biological pathways may be most relevant to the shared genetic effects. Due to the robustness of the analytic approaches and the statistical power of available data, the proposed research is expected to reach definitive conclusions about relationships between migraine and lipids, and is likely to advance understanding of migraine pathophysiology if not also to suggest novel therapeutic strategies for migraine.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Chen, Chia-Yen \",\n\"BIOGEN IDEC U S CORPORATION\",\n\"Drug target identification using cross-trait meta-analysis\",\n\"Jun10, 2020\",\n\"expired\",\n\"We seek to combine the results of large-scale GWAS from MVP cohort and additional publicly available GWAS summary statistics through meta-analyses and downstream bioinformatic analyses to identify potential new targets. To do so, we will perform meta-analysis using different statistical methods and evaluate results for diseases and traits across different categories. We expect these meta-analysis results will inform us on novel disease loci with different effects across diseases, which will potentially lead to new drug targets. \",\n\"Large-scale genome-wide association studies (GWAS) have provided unprecedented opportunities for new drug target identification through unbiased, hypothesis-free genome-wide search. We plan to leverage the summary statistics of large-scale GWAS from MVP cohort and additional publicly available GWAS summary statistics, of the same traits and other genetically correlated traits, to identify potential new targets. We will evaluate the results from 3 different classes of meta-analysis methodologies, including 1) conventional inverse-variance weighted meta-analysis, designed for GWAS of the same traits; 2) multi-trait analysis of GWAS (MTAG; PMID:29292387), designed for combining GWAS of genetically correlated traits; 3) Association analysis based on subsets (ASSET; PMID:22560090) for dissecting concordant/discordant genetic effects across different traits. We will test these methods on diseases and traits across different disease categories and genetic architectures to evaluate the robustness of the methods when applied on real data. For the validated genome-wide meta-analyses, we will perform downstream bioinformatic analyses to gain insight into the molecular basis of the novel loci, including transcriptome-wide association analysis (TWAS), colocalization, Mendelian randomization, gene-based and pathway-based analysis, and drug database annotation. We will also evaluate if these meta-analyses can improve polygenic score prediction. We expect these results will inform us on novel disease loci and shared loci between diseases/traits with concordant/discordant effects, which will potentially lead to new drug target identification.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Cheung, Ching Lung \",\n\"UNIVERSITY OF HONG KONG\",\n\"Mendelian Randomization of Lipids\",\n\"Apr23, 2020\",\n\"expired\",\n\"Circulating lipid plays an important role in multiple chronic diseases. The current study will elucidate whether circulating lipids play a primary role in disease development or as a marker of chronic diseases.\",\n\"Circulating lipid is an important risk factor and mediator of multiple chronic diseases. Thus, it is important to evaluate if circulating lipid plays an important role in contributing to the pathogenesis of chronic diseases. We aim to evaluate the role of circulating lipid in multiple chronic diseases via Mendelian Randomization (MR) approach. Summary statistics of GWAS of lipids from the MVP programme will be used in the MR study. Summary statistics of circulating lipids will either be used as the primary variables or confounding factors that will be adjusted in the multivariable analysis. The findings will be important to elucidate if lipids play a role in pathogenesis of different chronic diseases, and whether lipids are mediated in the relationship between other risk factors and chronic diseases.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Chiang, Charleston \",\n\"UNIVERSITY OF SOUTHERN CALIFORNIA\",\n\"Leveraging the Evolutionary History to Improve Identification of Trait-Associated Alleles and Risk Stratification Models of Cardiometabolic Traits in Diverse Human Populations\",\n\"May17, 2022\",\n\"approved\",\n\"It is imperative to extend human genetic studies of metabolic traits and diseases to include diverse populations around the world. In this study we will gather diverse populations not standardly included in existing GWAS studies (i.e. those outside of the European and East Asian ancestry populations, and those from the major ethnic minority populations such as the African Americans and Latino Americans) to perform a genetic study. We will evaluate the currently available genomic resource to see if it is helpful in genetic studies of understudied populations, explore alternative approaches that could overcome the (presumed) lack of genomic resources for these populations, and combine data from multiple populations to identify genes associated with diseases like type-2 diabetes.\",\n\"In order to personalize medicine for everyone, we need to combine an improved understanding of environmental and lifestyle contributions to disparity with a better understanding of the genetic etiology of complex diseases in diverse, often understudied, populations. For metabolic diseases such as type-2 diabetes and obesity, they are prevalent among Oceanian and Asian populations. While there are some large-scale efforts to study these traits and related diseases in East Asia, other parts of the world (such as Southeast Asia or the Pacific) have been ignored. Because each of these individual populations are understudied and cohort recruitments have been difficult to extend to large scale, it is necessary to aggregate multiple cohorts to obtain substantial statistical power. Furthermore, genetic studies of these understudied populations brings the opportunity to identify population- or region-specific variants of large effect, such as the variant found in CREBRF gene (Minster et al. Nat Genet 2016). Identification of these variants would ideally require sequencing data, which is also rare for understudied population. We will thus also evaluate the imputation accuracy for these population using the best available resources currently available, including TOPMED and GenomicAsia 100K reference panels. We will then apply imputation-based GWAS meta-analysis design to identify novel variant association to quantitative traits such as fasting glucose and insulin levels, BMI, Waist-to-hip ratio, and binary traits such as type-2 diabetes and obesity. We will also evaluate the efficacy of risk stratification models based on polygenic risk scores in different populations, and attempt to create an efficacious model across continental or subcontinental populations. We will further investigate the population structure in these cohorts, and explore its impact in confounding or empowering genetic mapping studies. Our study design has been approved by an expedited IRB review and all of our findings will be published to share with the larger scientific community and the public.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Chimusa, Rugamika \",\n\"UNIVERSITY OF CAPE TOWN\",\n\"African Genomic Risk Assessment\",\n\"Jul20, 2020\",\n\"closed\",\n\"Various models for genetic risk prediction have been possible due to numerous genome-wide association studies mostly of European-ancestry with some studies reaching up to hundreds of thousands study subjects. Clinical utility of disease risk prediction has largely been explored in populations of European-ancestry. While it is well-known that the prediction varies according to genetic ancestry of the population, current genetic risk prediction models have not been evaluated in populations of non-European ancestry. This raises questions as to whether the clinical utility within the predictive power of these methods could be equitable across populations and diseases, it is currently unclear how to accurately predict health and disease risk in populations of non-European ancestry. Here, we will independently assess current polygenic risk score tools through various human phenotypes including cancers, Chronic obstructive pulmonary disease, Autism Spectrum and Cardiovascular, Heart, Lung and heart diseases) from multi-ethics and simulated data; will develop a Bayesian posterior inference that uses a combination of (a) likelihood function that jointly sampling from training large reference data (General Research use data sets) and targeted training data from a given disease data.\",\n\"Various models for genetic risk prediction have been possible due to numerous genome-wide association studies mostly of European-ancestry with some studies reaching up to hundreds of thousands study subjects. Clinical utility of disease risk prediction has largely been explored in populations of European-ancestry. While it is well-known that the prediction varies according to genetic ancestry of the population, current genetic risk prediction models have not been evaluated in populations of non-European ancestry. This raises questions as to whether the clinical utility within the predictive power of these methods could be equitable across populations and, it is currently unclear how to accurately predict health and disease risk in populations of non-European ancestry.  This project aims to:  a. Assess current polygenic risk score (PRS) tools. We aim at running various PRS tools on (1) various simulated data mimicking various population genetics model (we will use datasets from General Research ); (2) various type of Cancer phenotypes; (3) Chronic obstructive pulmonary disease (COPD); (4) Autism Spectrum Disorder; (5) Cardiovascular Disease, Heart, Lung and heart diseases. To evaluate the accuracy and predictive power of current PRS tools on accessed data sets with differing sample size, phenotype from same and/or different ethnic groups, each accessed data set with specific phenotype from specific a particular population will be assessed independently. The data set with phenotype information will NOT be merged, however accessed large-scale (only General Research use) data sets based on similar ethnic groups as for data set with specific phenotype will be be used as reference training for some PRS tool. The General Research Use data sets with 1000 Genomes data will merged after rephasing and imputation processes and the final merged data set will be used as training reference panel in PRS analysis and also from  developed new tool in each specific phenotypic data set (each cancer type , Chronic obstructive pulmonary disease (COPD); Autism Spectrum Disorder;  Cardiovascular Disease, Heart, Lung and heart diseases, respectively).  b. Develop a Bayesian posterior inference that uses a combination of (1) a likelihood function that jointly sample from training large reference data and training data from the target population by modelling linkage disequilibrium and (2) a finite mixture of normal distributions prior on the marker\\u00e2\\u0080\\u0099s effects, local-specific ancestry and individual non-genetic risk factors.  c. To improve the power for risk prediction and independently apply the developed methods to various type of phenotype data sets from diverse ethics, We will develop an approach for integrating large-scale (at least 50 k samples) data sets with general purpose (Not data with phenotype information to avoid cross phenotype merge and potential risks to participants) across diverse and multi-ethics to serve as training data set in the model. Each cancer type data data set, Chronic obstructive pulmonary disease (COPD) data set; Autism Spectrum Disorder data set; and Cardiovascular Disease, Heart, Lung, Blood diseases and heart disease data set will be independently analysed (as targeted training data set within GWAS summary statistics, respectively) using a separate large-scale multi-ethnic training reference data (as described above). We aim to dissimulate broadly our findings with respect to each phenotype and publish our work to the scientific community through high impact factor journal.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Choi, Murim \",\n\"SEOUL NATIONAL UNIVERSITY\",\n\"Colocalization analysis of GWAS and eQTL in NAFLD\",\n\"Oct04, 2022\",\n\"approved\",\n\"Non-alcoholic fatty liver disease (NAFLD) is a complex disorder that are caused by multiple factors. Many genes, variants, diet, and other lifelog history are known to contribute to the development of NAFLD in a complicated manner. Nevertheless, it is crucial to understand genetic cause of NAFLD to precisely predict differential susceptibility to the disease across different individuals. To achieve this goal, we established a bioinformatic pipeline called NAFLD-specific expression quantitative trait loci (eQTL) pipeline and proved its utility in identifying potential drug targets for certain individuals with specific genetic variants (Yoo et al., J Hepatol 2021). Using the NAFLD GWAS data available from this data set, we plan to compare GWAS signals with those from our eQTL screening. The result will help us to understand genetic causation of NAFLD that may lead to the personalized disease prediction and therapy.\",\n\"Non-alcoholic fatty liver disease (NAFLD) is a complex disorder involving function of many genes and variants in addition to contribution from environmental and other complicating clinical factors. Nevertheless, it is crucial to understand genetic etiology of NAFLD to precisely predict disease susceptibility and individual-level variation. To achieve this goal, we established a NAFLD-specific expression quantitative trait loci (eQTL) pipeline and proved its utility in identifying potential drug targets that may behave differentially among individuals of different genotype (Yoo et al., J Hepatol 2021). Using the NAFLD GWAS data available from this data set, we plan to compare GWAS signals with those from our eQTL screening. The result will help us to understand genetic causation of NAFLD that may lead to the personalized disease prediction and therapy.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Churilla, James \",\n\"UNIVERSITY OF NORTH FLORIDA\",\n\"Sedentary time and heart failure and heart failure sub-type\",\n\"Apr12, 2023\",\n\"approved\",\n\"This study plans to investigate the potential causal pathway between sedentary time and various types of heart failure. This study also plans to investigate whether physical activity plays a role in this relationship. This study plans to utilize data from several data sources and combine the necessary data for analyses. Adults 40 years of age and older with data for all variables needed for the analyses will be included. This study will illustrate whether sedentary behavior is a causal component in heart failure and if physical activity plays a role in this relationship.\",\n\"Faculty from the Kinesiology and Lifestyle Medicine Program at the University of North Florida (UNF) possess a history of physical activity, cardiometabolic, and heart failure research. Previous work out of UNF has shown a significant relationship between sedentary time and incident heart failure with preserved ejection fraction (HFpEF) utilizing Multi Ethnic Study of Atherosclerosis (MESA) data. Other studies have identified one locus for HFpEF and 13 loci for heart failure with reduced ejection fraction (HFrEF). For this study, our team aims to investigate the potential causal pathway between sedentary time and HFrEF and HFpEF through a Mendelian randomization analysis. A secondary aim will be to investigate physical activity as a potential mediator or moderator. The Mendelian randomization will utilize data from three separate genome wide association studies (GWAS): 1) Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention; 2) Genome-wide association and Mendelian randomization analysis provide insights into the pathogenesis of heart failure; 3) Genetic architecture of heart failure with preserved versus reduced ejection fraction. (data being requested). Inclusion criteria will include adults >40 years of age with valid data for all variable in the analyses. Primary independent variables will be sedentary time and physical activity. Our dependent variable will be heart failure sub-types. We will utilize Million Veterans Project genomic association summary results for ejection fraction to identify heart failure sub-types. Summary statistics and risk estimates will be produced utilizing frequency and regression procedures in R. Our team will utilize genetic and phenotype data to add to the current evidence in the area of sedentary behavior and heart failure. Our team will publish our findings and abide by all embargo timelines associated with all datasets.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Clark, Shaunna \",\n\"TEXAS A&amp;M UNIVERSITY HEALTH SCIENCE CTR\",\n\"Integrating methylomic and genomic summary statistics to better characterize alcohol-related DNA methylation\",\n\"Mar17, 2020\",\n\"approved\",\n\"Despite extensive anti-drinking efforts, ~6.2% of US adults qualify for an alcohol use disorder and many more engage in habitual or problematic drinking. Mounting evidence suggests that DNA methylation may play a role in drinking behaviors. However, many DNA methylation may be impacted by variation that occurs on the genetic level. We propose to use summary statistics from alcohol genetic studies in order to determine if our alcohol-related DNA methylation findings are simply picking up on genetic signal or if they represent a unique biological signature not related to underlying genetics. \",\n\"Despite extensive anti-drinking efforts, ~6.2% of US adults qualify for an alcohol use disorder and many more engage in habitual or problematic drinking. Mounting evidence suggests that DNA methylation may play a role in drinking behaviors. However, many DNA methylation sites are regulated by common genetic variation. The research objective is to utilize summary statistics from alcohol-related genome-wide association studies (GWASs) to determine if alcohol-related methylation is simply picking up on genetic signal or represents a unique biological signature separate from common genetic variation.   This project will analytically combine the summary statistics from alcohol GWAS and summary statistics from alcohol methylation-wide association studies (MWASs). The combination of the GWAS and MWAS results does not create any additional risk to participants as we will only be using summary statics and do not need any participant-level information to conduct the project. Specifically, we will test whether the top findings from the alcohol GWAS occur more frequently among the top findings of our MWAS (i.e., are enriched) than what is expected by chance. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Claussnitzer, Melina \",\n\"BROAD INSTITUTE, INC.\",\n\"Multi-phenotype mapping for cardiometabolic phenotypes\",\n\"Apr07, 2020\",\n\"expired\",\n\"Cardiometabolic diseases such as type 2 diabetes, obesity and coronary artery disease appear to have shared etiology, but the underlying genetic factors remain largely unknown. To identify genetic loci that may have pleiotropic effects on cardiovascular and metabolic traits, we seek to study GWAS summary statistics to learn about common disease aetiologies.\",\n\"We plan to combine summary statistics from the MVP cohort with other existing summary statistics to improve power to assess multi-phenotype patterns of genetic associations with metabolic diseases and traits, and inform priors for cell types and mechanisms for individual GWAS loci. We will use multi-trait analysis of GWAS (MTAG), a method for joint analysis of summary statistics from genome-wide association studies (GWAS) of different traits, to gain insight into individual GWAS loci and globally. We will use our own compute resources and will follow procedures outlined in NIH communication (NOT-OD-19-023) for using summary statistic data from human genetic studies.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Coleman, Jonathan \",\n\"KING'S COLLEGE LONDON\",\n\"Use of genomic summary statistics for neuropsychiatric genetic epidemiology\",\n\"Jan03, 2022\",\n\"approved\",\n\"Research has identified many places in the human genome where certain changes in the DNA are found more often in people with brain-related conditions (like depression and Alzheimer\\u00e2\\u0080\\u0099s disorder) compared to people who do not have the condition. The results of research like this from different groups of scientists can be used to examine how similar the genetic effects on one condition are to the genetic effects on another. We will use results from the Million Veteran Program to examine how similar genetic effects on various conditions studies by the Million Veteran Program scientists are to the brain-related conditions we study.\",\n\"Genome-wide association studies (GWAS) have implicated numerous regions of the genome as associated with complex brain-related phenotypes, including normative behaviour, psychiatric illness, and neurological degeneration and dysfunction. Studies of the genetic overlap between traits have benefited from statistical methods that use GWAS summary statistics. We will use summary statistics from GWAS conducted by the Million Veteran Program to study how genetic variation contributes to brain-related traits, including normal behaviours such as personality, psychiatric disorders such as depression and anxiety disorders, and neurological dysfunction, including Alzheimer\\u00e2\\u0080\\u0099s disorder. Ultimately, we aim to understand better both the prediction of such traits, and their biology. We will examine the role of genetic influences in isolation and also within the context of environmental risk factors. We intend to use the summary statistics obtained in this request (alongside other summary statistics from other sources) to create polygenic risk scores within the individual level genetic datasets available to our research groups. We will also use summary statistics to conduct between-trait analyses at the summary level, through methods including genetic correlation analysis (e.g. in LDSC), genomic mixture analysis (e.g. in MiXer), and multivariate GWAS and genomic structural equation modelling (e.g. in GenomicSEM). \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Coombes, Brandon \",\n\"MAYO CLINIC ROCHESTER\",\n\"Polygenic risk prediction of psychiatric, addiction, and treatment response outcomes   \",\n\"Oct30, 2020\",\n\"approved\",\n\"Although individual genetic variations are unlikely to have large effects on complex traits such as substance use disorders, the collective effects of relevant genetic variations may be quite important. We will use results from the Million Veteran Program studies to examine how genetic variation/genetic risk contributes to individual differences in biology, behavior, and health-related outcomes in studies of psychiatric and addiction outcomes. \",\n\"Although individual genetic variations are unlikely to have large effects on complex traits such as substance use disorders, the collective effects of relevant genetic variations may be quite important. We will use summary statistics generated by genome-wide association studies (GWASs) conducted within the Million Veteran Program dataset to examine how genetic variation/genetic risk contributes to individual differences in biology, behavior, and health-related outcomes in studies of psychiatric, addiction, and treatment response outcomes. The overall objective of this study is to use MVP summary statistics to create polygenic risk scores in independent samples to further our understanding of psychiatric and addiction outcomes. We will also meta-analyze these summary statistics with independent GWASs of similar outcomes. Phenotypes will include summary statistics for GWAS related to addiction (e.g. problematic alcohol use, opioid use disorder, etc.), psychiatric outcomes (PTSD, resilience, depression, etc.), as well as outcomes for health, behavior and biology.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Corces, Michael \",\n\"J. DAVID GLADSTONE INSTITUTES\",\n\"Investigating the role of noncoding SNPs in neuropsychiatric disorders\",\n\"Nov02, 2021\",\n\"approved\",\n\"Small variations in non-coding sequences of the human genome can increase susceptibility to disease. We will use data from the Million Veteran Program to highlight specific changes that are relevant to neuropsychiatric disorders, and further our understanding of the biological mechanisms connecting these variants to disease symptoms. We hope to identify new therapeutic targets that will aid in combatting these personally and socially impactful disorders.\",\n\"The broad research approach of our laboratory is to use insights from the epigenome to inform our understanding of human diseases, with the goal of identifying novel avenues for therapeutic intervention. We are interested in how the genome, transcriptome, and epigenome intersect and how inherited genetic variation interfaces with lived experiences and environmental factors to produce durable and functional gene regulatory alterations that lead to disease. We are particularly interested in neurological and neuropsychiatric diseases.  A conceptual tenet of our group is that single nucleotide polymorphisms (SNPs) associated with diseases can lie in regions of the genome that regulate gene expression, driving cellular and ultimately organismal phenotypes from the molecular level. To apply this idea toward understanding human disease, we will use summary statistics from genome-wide association studies (GWASs) conducted as part of the Million Veteran Program to identify candidate noncoding SNPs that may play a role in the pathogenesis of complex neuropsychiatric disorders such as Major Depressive Disorder (MDD), Generalized Anxiety Disorder (GAD) and Post-Traumatic Stress Disorder (PTSD).  We will overlay these summary data onto our own single-cell ATAC-seq, bulk ATAC-seq, and three-dimensional chromatin conformation data from primary brain samples, allowing us to identify noncoding SNPs that fall in cell-type and cell-state specific regions of accessible chromatin and interact with other known regulatory elements. Highlighting these putative functional SNPs will allow us to derive hypotheses about the biological mechanisms mediating associations between genetic variants and symptom phenotypes. The proposed combination of the requested data and our own data will not create any additional risk to participants. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Crowley, James \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"Genetic study of trichotillomania\",\n\"Mar06, 2021\",\n\"expired\",\n\"We are performing a genetic study of compulsive hair pulling, also known as trichotillomania (TTM). This study has both a rare variant component and common variant component. For the rare variant component, we have access to whole exome sequencing data for 100 TTM cases and are searching for an appropriate control group of neurologically normal subjects with whole exome sequencing data. The dataset we apply for here appears to be an ideal control group for case-control analyses. We will test whether subjects with TTM have a greater burden of rare, deleterious coding variants relative to controls.   For the common variant component, we have access to genome wide SNP data for these same 100 cases. We would like to test whether these TTM cases have an elevation in polygenic risk scores for anxiety and other psychiatric traits. The Million Veteran Program (MVP) dataset provides a well-powered GWAS on anxiety and we intend to use summary statistics to compute polygenic risk scores for the TTM cases. We will test whether subjects with TTM have a higher burden of common alleles for anxiety.\",\n\"We are performing a genetic study of compulsive hair pulling, also known as trichotillomania (TTM). This study has both a rare variant component and common variant component. For the rare variant component, we have access to whole exome sequencing data for 100 TTM cases and are searching for an appropriate control group of neurologically normal subjects with whole exome sequencing data. The dataset we apply for here appears to be an ideal control group for case-control analyses. We will test whether subjects with TTM have a greater burden of rare, deleterious coding variants relative to controls.   For the common variant component, we have access to genome wide SNP data for these same 100 cases. We would like to test whether these TTM cases have an elevation in polygenic risk scores for anxiety and other psychiatric traits. The Million Veteran Program (MVP) dataset provides a well-powered GWAS on anxiety and we intend to use summary statistics to compute polygenic risk scores for the TTM cases. We will test whether subjects with TTM have a higher burden of common alleles for anxiety.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"CULE, MADELEINE \",\n\"CALICO LIFE SCIENCES, LLC\",\n\"Genetic risk factors of thoracic and abdominal aneurysm\",\n\"Jul06, 2022\",\n\"approved\",\n\"We are studying the genetic basis of risk factors for cardiovascular disease, and have conducted studies which identified risk factors for aneurysm in a large British cohort. We wish to compare our findings in a large British cohort with similar findings in the Million Veteran\\u00e2\\u0080\\u0099s cohort\",\n\"We are investigating the etiology of cardiovascular disease, including thoracic and abdominal aortic aneurysm, using the features derived from MRI scans in the UK Biobank resource. We have conducted genetic studies of these risk factors and plan to investigate the genetic correlation between these traits and aneurysm outcomes in the MVP (as described in Klarin et al, Circulation, https://doi.org/10.1161/CIRCULATIONAHA.120.047544) . We also plan to investigate whether the genetic signals found in the two traits colocalize to implicate the same genes/variants. No attempt will be made to identify individual participants.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Czamara, Darina \",\n\"MAX PLANCK INSTITUTE OF PSYCHIATRY\",\n\"Neurobiological correlates of PTSD polygenic risk scores in dimensional samples of stress-related psychiatric disorders\",\n\"Jul28, 2021\",\n\"closed\",\n\"Genome wide association studies (GWAS) have identified many genes associated with psychiatric diseases. However, it is not known how the identified risk genes contribute to maladaptive physiological responses, brain functioning and behaviour. In our analyses, we will GWAS data for post-traumatic stress disorder to compute polygenic risk scores. These scores reflect a small part of our study participant\\u00c2\\u00b4s risk for having a psychiatric disorder. We will then analyze whether the polygenic risk scores are associated with physiological responses (for example stronger pupil dilations to threatening stimuli), brain functioning (for example activation of rewarding brain regions in response to rewarding stimuli) and behavior (for example having trouble to get out of bed in the morning). This will help us understand how the genetic predisposition shapes mental health. \",\n\"The GWAS data will be used for the calculation of polygenic risk scores, mainly in the ongoing BeCOME study (registered on ClinicalTrials.gov, TRN: NCT03984084) that is designed to identify biology-based classes of stress, affective and anxiety related mental disorders and recruits healthy controls and patients with different degrees and varying comorbidities of stress-related psychiatric disorders. Polygenic risk scores will be used to predict pathology and task performance in the BeCOME study, either as a main predictor or in interaction with other variables of interest, such as childhood trauma. The planned analyses do not convey additional risks to the participants. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"da Silva Couto Alves, Alexessander \",\n\"UNIVERSITY OF SURREY\",\n\"Mendelian Randomization of Birthweight on Adult Blood Lipids\",\n\"Jul06, 2020\",\n\"closed\",\n\"Cardiovascular diseases are complex caused by a combination of genetics and environmental factors. The identification of environmental causal factors contributing to risk of cardiovascular disease is hampered by confounding factors.  Mendelian randomization is a statistical approach that is less prone to confounding. To conduct mendelian randomization only summary statistics are required. No need for individual-level genetic information to be shared, thus volunteer genetic data is never put at risk. We request the adult lipid summary statistics corresponding to the 20 analyses listed below: pha004662.1 pha004663.1 pha004664.1 pha004665.1 pha004827.1 pha004828.1 pha004829.1 pha004830.1 pha004831.1 pha004832.1 pha004833.1 pha004834.1 pha004835.1 pha004836.1 pha004837.1 pha004838.1  \",\n\"I would like to access the GWAS summary statistics reported on the paper (Klarin 2018 NatGen) to conduct mendelian randomization analyses. We aim to understand the causal effect of developmental phenotypes such as birthweight on the risk of cardiovascular markers and risk factors such as adult blood lipids. The focus of this study is birth weight and adult lipids. References Klarin 2018. Genetics of blood lipids among ~300,000 multi-ethnic participants of the Million Veteran Program. Nature Genetics.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Daly, Mark \",\n\"BROAD INSTITUTE, INC.\",\n\"VA-MVP Cohorts for Replication Efforts in Finnish T2D Analyses\",\n\"Oct05, 2020\",\n\"closed\",\n\"We aim to use the summary statistics from the Veterans Administration (VA) Million Veteran Program (MVP) cohorts from their recent meta-analysis for replication for our T2D association study within the FinnGen Study.\",\n\"We have been pursuing our Type 2 diabetes (T2D) project to use genome-wide association techniques to discover variants enriched through the Finnish founding bottleneck (eligible to have stronger biological effects than their frequency would generally indicate). We have already completed over 29,000 Finnish T2D cases, and are comparing these to physiologic results from the Botnia Study which has compiled physiologic results after oral glucose tolerance testing on over 3000 Finns without T2D. We propose the use of the case control data from the recent international Type 2 diabetes meta analysis (~228,000/~1,180,000) to replicate potential risk and protective variants that are found to be associated in this Finnish population sample. While we expect many of these to be much rarer in either the either European or international samples, they will often exist and the case-control distribution can be quite informative as replication given the sample numbers involved.  The specific samples of interest are from analyses pha004943.1, pha004944.1,  pha004945.1 , pha004946.1, and pha004947.1.  (Vujkovic M, Keaton JM, Lynch JA, et al. Nat Genet. 2020;52(7):680-691.)\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Damrauer, Scott \",\n\"UNIVERSITY OF PENNSYLVANIA\",\n\"Cross trait genetics of cardiometabolic diseases, their causes and consequence\",\n\"Nov23, 2022\",\n\"approved\",\n\"We aim to is to analyze the genetic architecture of a wide variety of cardiometabolic diseases and their risk factors. We are interested in utilizing numerous data sources to identify DNA variants associated with our diseases and risk factors of interest to better understand how genes confer disease risk. We will conduct genome-wide association studies to link genetic sequence variants with risk of cardiometabolic disease and risk factors and integrate across datatypes mapping to identify novel variant associations with gene expression through a method known as expression quantitative trail locus (eQTL) mapping. Similar methods will be applied to link genetic variants to other intermediate traits including epigenetic markers, protein levels, and metabolite levels, which may mediate cardiometabolic disease and its risk factors. These integrative approaches will enable us to elaborate powerful datasets for downstream causal inference experiments to evaluate the casual links between risk factors, intermediate phenotypes/traits, and cardiometabolic diseases.\",\n\"The purpose of the proposed activity is to analyze the genetic architecture of cardiometabolic diseases including but not limited to: CAD, PAD, cIMT, CAC, AAC, VTE, AAA, TAA, CHF, stroke, carotid stenosis, and carotid plaque, as well as the risk factors for these diseases including but not limited to diabetes, dyslipidemia, hypertension, smoking, obesity, alcohol use, and psychiatric disorders. We are interested in identifying DNA sequence variants that associate with these diseases, understanding the molecular consequences of these variants, and identifying how these variants influence the relationship between risk factors and disease. We will leverage publicly available, deidentified individual-level, and summary statistics data sources for the above diseases and risk factors as well as all of MVP and GTEx. We will apply computational methods to:  1. identify the relationships between DNA sequence variants and cardiometabolic diseases/risk factors 2. integrate these datasets with other summary-level data that are both internally generated, as well as existing in the public domain 3. apply causal inference methods to understand the relationships and mechanisms that link DNA sequence variation with disease First, we will conduct genome-wide association study (GWAS) meta-analyses to link genetic sequence variants with risk of cardiometabolic disease and its risk factors. Next, we will integrate across datatypes, for example linking publicly available genotyping and/or genome sequencing data with gene expression data to perform expression quantitative trait locus (eQTL) mapping, enabling the identification of novel variants associated with gene expression. As cell-type specific eQTL data has been shown to be more powerful for discovery, we will integrate individual level GTEx data with single cell transcriptomics to compute interaction eQTLs. Similar methods will be applied to link genetic variants to other intermediate traits, including epigenetic markers, protein levels, and metabolite levels, which may mediate cardiometabolic disease and its risk factors. These integrative approaches will enable us to elaborate powerful datasets for causal inference experiments. Finally, we will apply causal inference methods including but not limited to Mendelian Randomization to evaluate the casual links between risk factors, intermediate phenotypes/traits, and cardiometabolic diseases.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Daskalakis, Nikolaos \",\n\"MCLEAN HOSPITAL\",\n\"Integrating genetics of neuroimaging into functional characterization of neuropsychiatric GWAS\",\n\"Oct05, 2020\",\n\"approved\",\n\"We will study the genetic risk architecture of mental disorders, with a focus on the portion of the human genome influencing imaging of the brain. \",\n\"We will integrate neuroimaging genetics with neuropsychiatric risk by applying our neuroimaging models to the GWAS from neuropsychiatric traits. Our polygenic models of neuroimaging phenotypes have specific SNP weights that can be applied to neuropsychiatric GWAS summary statistics to aggregate the SNP association statistics to neuroimaging phenotype association statistics. Using causal mediation analysis, we will distinguish causal associations from false positives. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Davis, Lea \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Multi-ethnic and ancestry-specific polygenic scores in blood lipids and other laboratory test results\",\n\"May17, 2019\",\n\"approved\",\n\"Electronic health records linked to genetic data hold great promise for improving human health.  Our research in the Vanderbilt University Medical Center biobank, BioVU, harnesses the hundreds of laboratory test results collected in routine clinical care to discover novel gene-biomarker associations.  A major focus of our project is to include patients of diverse ancestries, a population largely overlooked by the last decade of genetic studies.  In an initial experiment, we are requesting results from a new multi-ethnic genetic study of blood lipids in the Million Veteran Program participants.  We will use these results to quantify genetic risk factors for blood lipids across BioVU patients (~20% of non-European ancestry), and to test these for ancestry-specific associations with lipids and other biomarkers in the biobank.  We are additionally requesting results from a study of problematic alcohol use in the Million Veteran Program participants. We will use these results to quantify genetic risk factors for problematic alcohol use across BioVU patients (~20% of non-European ancestry), and to test these for ancestry-specific associations with buprenorphine treatment duration in opioid use disorder patients in the biobank.\",\n\"Genome-wide association studies (GWAS) over the past decade have revealed thousands of variants associated with common complex diseases.  These studies have powered analyses with polygenic scores, which summarize the effects of multiple genetic variants into a single measure, and which can be tested for association with the same, or a different trait in any genotyped sample, using only summary statistics from an external GWAS.  Previous GWAS, however, have overwhelmingly been in European-ancestry individuals, hampering our ability to generate valid polygenic scores in people of other ancestries.  We are requesting access to blood lipid GWAS summary statistics from the Million Veteran Program (MVP) of ~300,000 multi-ethnic participants.  Our objectives are:  1) To develop multi-ethnic and ancestry-specific polygenic scores for blood lipids from the MVP summary statistics; and 2) To test these polygenic scores for association with blood lipids and other laboratory test results in patients of different ancestries from the Vanderbilt University Medical Center biobank, BioVU. 3) To develop multi-ethnic and ancestry-specific polygenic scores for Problematic Alcohol Use from the MVP summary statistics; and 4) To test these polygenic scores for association with buprenorphine treatment duration (BTD) in Opioid Use Disorder (OUD) patients of different ancestries from the Vanderbilt University Medical Center biobank, BioVU   BioVU includes >285,000 patients (~20% of non-European ancestry) with DNA samples linked to electronic health records (EHR) that span 10 years, on average.  Genotypes were measured on the Illumina MEGAEX array of >2 million SNP markers, and ethnicity was assigned by the principal components of ancestry.  Using the MPV summary statistics, our next step is to derive multi-ethnic and ancestry-specific polygenic scores (e.g., using the software PRSice, LDpred, and/or PRS-CS), and to test these scores for association with blood lipids and other laboratory test results (e.g., C-reactive protein) in the EHR of BioVU patients, stratified by ethnicity.  Results will be shared broadly in conference presentations and published manuscripts.  Despite the benefits associated with buprenorphine in the treatment for Opioid Use Disorder (OUD), retention remains poor. Research on the impact of co-occurring medical issues on buprenorphine treatment duration (BTD) has yielded mixed results. It is also unknown whether genetic risk in the form of polygenic scores (PGS) for OUD are associated with BTD. The objectives of our research are to test the association between somatic and psychiatric comorbidities and BTD, and to determine whether PGS for OUD and several other OUD-related traits are associated with BTD. We would like to request the GWAS summary statistics from Zhou et al. (2020), to generate PGS of Problematic Alcohol Use and apply them to individuals with genotype data from Vanderbilt University Medical Center's (VUMC) biobank.  In accordance with the data use limitations for the dataset, we are not requesting any individual-level data.  We do not anticipate collaborating with other institutions on this project. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"DE VRIES, PAUL \",\n\"UNIVERSITY OF TEXAS HLTH SCI CTR HOUSTON\",\n\"Mendelian randomization to assess the effect of hemostatic factors on disease phenotypes\",\n\"Nov10, 2020\",\n\"approved\",\n\"The proteins that affect blood clotting may affect our risk of certain diseases, such as heart disease and stroke. Learning more about whether these proteins are disease risk factors could affect the way we try to prevent and treat disease. We can use genetics to research the effect of these clotting proteins on diseases. This approach is called \\\"Mendelian randomization\\\", and is done by comparing the effect of genetic variants on blood levels of a clotting protein to the effect of those genetic variants on disease risk. If genetic variants that increase blood levels of the clotting protein also increase the risk of a disease, then that tells us that the clotting protein might be influencing the risk of disease. To carry out this research, we request access to results from the MVP study. The results from MVP can tell us about the effects of genetic variants on disease, which we can then use to do Mendelian randomization.\",\n\"Circulating levels of hemostatic factors such as fibrinogen, Factor VII, Factor VIII, and von Willebrand factor are hypothesized to influence the risk of venous thromboembolism as well as the risk of diseases characterized by arterial thrombosis such as coronary artery disease, ischemic stroke, and peripheral artery disease. Many hemostatic factors are also thought to influence the risk of bleeding disorders such as hemorrhagic stroke. It is also hypothesized that specific hemostatic proteins may have effects on disease that are not characterized by thrombosis or bleeding, such as type 2 diabetes or chronic kidney disease. Mendelian randomization is an approach that uses genetic variants associated with an exposure as instrumental variables to test the causal association of that exposure with one or more disease phenotypes. Mendelian randomization can be conducted using summary statistics from genome-wide association studies (GWAS). Within the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE)consortium we already used this approach to investigate the causal effect of several hemostatic factors on venous and arterial thrombosis. We now propose to use summary statistics from MVP GWAS to conduct Mendelian randomization analyses of hemostatic factors on a variety of disease outcomes. When appropriate we will meta-analyze summary statistics from MVP with summary statistics from our GWAS. From the disease phenotypes that currently have MVP summary statistics available on dbGaP we propose to use venous thromboembolism, peripheral artery disease, eGFR, and type 2 diabetes. We are also interested in using summary statistics for coronary artery disease and stroke, if these become available. We also intend to use the MVP summary statistics to create polygenic scores for blood pressure and lipid levels, and examine how those scores associate with longitudinal trajectories of blood pressure and lipid levels, as well as with cardiovascular outcomes. Given that we will be working only with summary statistics, this proposed research poses no additional risks to MVP participants. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Del-Aguila, Jorge \",\n\"MERCK AND COMPANY\",\n\"Target identification using human genetics for peripheral artery disease (PAD)\",\n\"Apr09, 2020\",\n\"closed\",\n\"Peripheral artery disease (PAD), a condition where fatty deposits build up in the arteries and restrict blood flow.  When untreated, PAD can lead to amputation, additional cardiovascular disease and, in the worst scenario, death.  We aim to use the human genetic analysis of PAD from the Million Veterans Program (MVP) to identify dysfunctional genes that lead to PAD using a standard analytical pipeline in the field of human genetics.\",\n\"Peripheral artery disease (PAD) is a leading cause of cardiovascular morbidity and mortality, however current medical treatments mostly target risk factors for PAD, such as diabetes, hypercholesterolemia and high blood pressure and not PAD itself, or involve invasive surgical procedures.  As part of Merck's efforts to identify novel drug targets to treat disease, we aim to identify promising targets for PAD by analyzing the GWAS PAD summary statistics from the Million Veteran Program (MVP) recently published by Klarin et al (Nature Genetics 2019).  This effort is based on the observation that human genetic evidence linking a target to disease increases the probability of success to advance a drug through development by Nelson et al (Nature Genetics 2015).  Our analysis pipeline aims to identify the most likely causal gene at each of the genome-wide significant loci (P<5e-8).  The pipeline contains statistical analyses to perform 1) fine-mapping of the 95% credible set to identify most likely causal variants, 2) conditional analysis to detect if multiple independent associations exist, 3) colocalization analysis with summary eQTL data from GTEx and summary pQTL data from Sun 2018 to identify putative causal genes.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Deloukas, Panagiotis \",\n\"UNIV/LONDON-QUEEN MARY&amp; WESTFIELD COLL\",\n\"Investigating the casual relationship between thyroid function and serum lipid levels.\",\n\"May06, 2019\",\n\"rejected\",\n\"In this study we aim to investigate the causal relationships between thyroid function and blood lipid levels using the Mendelian Randomization approach.  \",\n\"Observational studies reported an association between thyroid function and serum lipid levels. This study will use a two-sample Mendelian Randomization (MR) approach to examine whether a causal relationship underlies this association. MR uses genetic variants as proxies to evaluate the casual effect of an exposure on the outcome of interest. In this two-sample MR we will use summary-level statistics from the largest genome-wide association studies (GWAS) on thyroid function traits and blood lipid levels. Causal estimates will be assessed using inverse-variance weighted method, followed by sensitivity analyses. No participant-level data will be used in this project.   \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Demontis, Ditte \",\n\"UNIVERSITY OF AARHUS\",\n\"Polygenic effects on psychiatric disorders, comorbidity risk and severe outcomes \",\n\"Oct25, 2022\",\n\"approved\",\n\"Common genetic variation contributes considerably to human behavior and complex diseases like psychiatric disorders. We will use the results generated from genome-wide association studies (GWAS) conducted in the Million Veteran Program (MVP) to advance our understanding of how the polygenic architecture affects the risk of psychiatric disorders and comorbidities in the Danish iPSYCH cohort.\",\n\"Common genetic variation contributes to variability in behavior and disease risk. We will use the summary statistics generated by genome-wide association studies (GWASs) of individuals in the Million Veteran Program (and meta-analyses with other cohorts)  to examine how the polygenic architecture contributes to individual differences in biology, comorbidity patterns and disease outcomes in the Danish iPSYCH cohort consisting of 130K genotyped samples with major psychiatric disorders (80K individuals) and population-based controls (50K individuals). The MVP data will be used as basis for generating polygenic risk scores for the individuals in the iPSYCH cohort for subsequent analyses of how the polygenic risk load associates with overall risk for various health outcomes and comorbidities as well as analyses stratifying cases by their polygenic risk load to evaluate risks within polygenic risk score bins. Additionally, the MVP data will be used for Mendelian randomization analyses and mediation analyses to test for causality and direct and indirect effects of one phenotype on another. The described analyses will be used to evaluate the genetic contribution to risk focusing on psychiatric disorders (including substance use disorder phenotypes). \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Derks, Eske \",\n\"QUEENSLAND INSTITUTE OF MEDICAL RESEARCH\",\n\"Incorporating depression PRS into network analyses of depressive symptoms\",\n\"Apr11, 2022\",\n\"expired\",\n\"Depression is a highly prevalent psychiatric disorder that causes substantial burden worldwide. Patients with depression present in many different ways, with a large number of different symptom profiles being observed. In this project we seek to investigate how genetic risk for depression is related to different patterns and networks of depression symptoms.\",\n\"Depression is a clinically heterogeneous disorder. The network approach to psychopathology, which conceptualises mental disorders as a system of causal interactions between symptoms, embraces the large heterogeneity observed within psychiatric disorders. In this project, we will incorporate depression PRS into a network analysis of depressive symptoms in order to assess how PRS is associated with particular symptoms while accounting for between symptom relationships.   We will construct a depression PRS from large-scale depression GWAS data, and conduct a phenotypic network analysis of depressive symptoms in the UK Biobank. We will explore how the network structure of symptoms differs between individuals with high vs. low PRS. We will not combine GWAS data of the Million Veterans Program with other datasets (e.g., PGC) ourselves. We do request permission to use the meta-analyzed datasets (i.e., MVP and PGC) that are currently being generated in a project jointly led by the MVP and PGC. We do not believe this creates any additional risks to participants as the meta-analysis is restricted to summary statistics only).\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Dewan, Andrew \",\n\"YALE UNIVERSITY\",\n\"Genetic studies on peripheral artery disease (PAD)\",\n\"Aug26, 2022\",\n\"approved\",\n\"Peripheral artery disease (PAD) is a vascular disease with severe morbidity and mortality. Additional data to validate our findings of genetic risk variants for PAD could improve our understanding of the etiology. We will use the summary statistics being requested and summary statistics generated from the PAD case-control data being requeested as validation for our findings from an independent dataset and to construct new genetic models of PAD that incorporate information from multiple genetic variants.  \",\n\"Our objectives for this research is the discovery and validation of genetic risk factors for peripheral artery disease (PAD). We currently have genotype and phenotype data from the UK Biobank. We plan to use PAD summary statistic data from dbGAP in two ways: 1) validate findings from the analysis of UK Biobank; and 2) conduct polygenic risk score (PRS) analysis by using the GWAS summary statistics from million veteran program to construct models for us to then apply to the UK Biobank data (not a part of this application) and the other PAD case-control phenotype and genotype data being requested from dbGaP. No individual genotype or phenotype data from any of the dbGaP datasets we are requesting will be combined with the UK Biobank data or any other external data. The only combination of data will potentially the combination of summary statistics from dbGaP datasets and UK Biobank summary statistics for PAD that we generate in a meta-analysis framework. This does not increase risk or create new risks to the participants in any of the dbGaP studies we are requesting. Results from this study will be published in a peer-reviewed journal. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Dewey, Frederick \",\n\"FORESITE LABS, LLC\",\n\"Systematic analysis of the relationship between genetic drug target modulation and clinical outcomes\",\n\"Oct06, 2022\",\n\"approved\",\n\"We will combine aggregated information from many different studies from dbGAP and internal and external databases to gain a better understanding of the relationship between genetics and outcomes such as disease and its response to treatment. We plan to use the data to improve inferences made with genetic information, leading to more reliable risk prediction and more efficient drug development. We also aim to reduce harm and improve outcomes when developing new treatments. A secondary benefit is that treatments developed in this manner might be more targeted and therefore more cost-effective. Thirdly, we aim to replicate, improve and develop robust methods for understanding the relationship between genetic targets, disease biology, and health-related outcomes.\",\n\"This application will utilize the MVP genotype-phenotype association summary statistics to support ongoing research in discovery and validation of therapeutic targets and biomarkers. These include co-localization, Mendelian randomization analyses, and other summary statistic analyses in combination with data from internal and external databases, and development and validation of polygenic risk score (PRS) approaches applicable to multiple ancestries.  All analyses derived from the requested datasets will only be used in aggregated form and will abide by dataset-specific limitations. Access to the data will be limited to the collaborators named in our application (i.e. our staff and Dr. Connor Emdin of FL2022-001) and will abide by all applicable data use certifications. The proposed research does not involve patient identification and is consistent with the data use agreement and consent requirements. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"DICK, DANIELLE \",\n\"RBHS-ROBERT WOOD JOHNSON MEDICAL SCHOOL\",\n\"Prediction of Alcohol Use Disorders across Ancestral Populations\",\n\"Feb06, 2023\",\n\"approved\",\n\"Polygenic scores are becoming of interest for medical practitioners and clinicians. The current analysis will examine the clinical utility of polygenic risk scores for identifying those at extreme risk for substance use disorders. We will focus on individuals of both European and African ancestry to address the diversity gap in genomic research.\",\n\"Polygenic scores derived from GWAS summary statistics are becoming increasingly useful in medical research and there is hope that these will provide actionable information for clinical intervention in regards to substance use disorders. However, the majority of the studies focus on individuals of European ancestry. Here, we plan on using summary statistics from large scale GWAS of alcohol use disorders (AUD) and alcohol consumptions in both European and African ancestry samples in the Million Veterans Program (MVP) to create polygenic risk scores in a diverse sample of individuals deeply affected with AUD. The goal of this research will focus on whether the extreme ends of the polygenic score distribution for alcohol phenotypes provide any clinically relevant information in individuals of European or African descent. We will focus on a variety of substance use disorders including alcohol, cannabis, cocaine, nicotine, and others. Data will be used only for research purposes and we will make no attempt to reveal the individual participants. The proposed research is consistent with the data use limitation noted in the Data Use Certification Agreement for MVP.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"DICK, DANIELLE \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"Prediction of Alcohol Use Disorders across Ancestral Populations\",\n\"Aug08, 2019\",\n\"closed\",\n\"Polygenic scores are becoming of interest for medical practitioners and clinicians. The current analysis will examine the clinical utility of polygenic risk scores for identifying those at extreme risk for substance use disorders. We will focus on individuals of both European and African ancestry to address the diversity gap in genomic research.\",\n\"Polygenic scores derived from GWAS summary statistics are becoming increasingly useful in medical research and there is hope that these will provide actionable information for clinical intervention in regards to substance use disorders. However, the majority of the studies focus on individuals of European ancestry. Here, we plan on using summary statistics from large scale GWAS of alcohol use disorders (AUD) and alcohol consumptions in both European and African ancestry samples in the Million Veterans Program (MVP) to create polygenic risk scores in a diverse sample of individuals deeply affected with AUD. The goal of this research will focus on whether the extreme ends of the polygenic score distribution for alcohol phenotypes provide any clinically relevant information in individuals of European or African descent. We will focus on a variety of substance use disorders including alcohol, cannabis, cocaine, nicotine, and others. Data will be used only for research purposes and we will make no attempt to reveal the individual participants. The proposed research is consistent with the data use limitation noted in the Data Use Certification Agreement for MVP. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"DRACHEVA, STELLA \",\n\"JAMES J PETERS VA  MEDICAL CENTER\",\n\"Integration of the opioid use disorder GWAS dataset with brain cell type-specific ATAC-seq and H3K27ac ChIP-seq data \",\n\"Apr10, 2023\",\n\"approved\",\n\"Opioid Use Disorder (OUD) is a problematic pattern of opioid use that leads to substantial impairment and disability. A tenfold increase in opioid analgesic prescriptions between 1990 and 2010 in the U.S. contributed to an epidemic of opioid abuse and overdose deaths. A recent large-scale genome-wide association study (GWAS) identified many gene regions (loci) that are associated with OUD. As in other complex disorders, most of the OUD risk loci reside within non-coding regions of the human genome where they are predicted to alter the activity of gene regulatory elements (GREs), which regulate the expression of nearby genes.  The effect of a GRE on gene expression is dependent on the cell and tissue type, making it difficult to understand the functional impact of GWAS risk loci.  Linking the disease risk variants with epigenetic maps of GREs in relevant tissues/cell types yielded formidable results in other psychiatric disorders  and holds promise for OUD.  Here we propose to integrate the OUD GWAS with cell-type-specific epigenetic data that were generated in our laboratory. The proposed studies should lead to the discovery of the OUD risk variants and the affected genes that are likely to be causal for OUD and act in specific brain cell types.\",\n\"We are requesting summary-level association data from the study by Kember et al. 2022. The goal of the proposed research is to understand if cell-type-specific non-coding regulatory elements (e.g., enhancers, promoters) in the orbital frontal cortex (OFC; the brain region that has been implicated in drug addiction) are enriched for opioid use disorder (OUD) risk SNPs.   Most functional noncoding SNPs are predicted to exert their effects through the alteration of gene expression via perturbation of transcription factor binding and/or regulatory element function. Such regulatory elements are highly cell type specific, suggesting that the resultant effects of noncoding SNPs (including risk SNPs) would be equally cell type specific.  As a part of our NIH-funded study (R01DA04324; NIH/NIDA; Cell Specificity of the Human Heroin Epigenome), we obtained transcriptional (RNA-seq) and epigenetic (ATAC-seq and H3K27ac ChIP-seq) data in the OFC samples from subjects who died of heroin overdose and control subjects (N=30 per group). The data were obtained in the nuclei from three major brain cell types\\u00e2\\u0080\\u0094inhibitory GABAergic neurons (GABA), excitatory glutamatergic neurons (Glu), and oligodendrocytes (OLIG)\\u00e2\\u0080\\u0094that were separated from postmortem brains using fluorescent-activated-nuclear sorting.   Our cell-type-specific ATAC-seq and H3K27ac ChIP-seq data, which are indicative of open chromatin and active enhancer/promoter regions, respectively, allowed us a comprehensive nomination of the putative functional regulatory elements that are active in each of the major brain cell types (GABA, Glu, OLIG) in OFC. These critical data will be used to test the functional importance of genetic risk loci in OUD.  To this end, we aim to perform linkage disequilibrium score regression (LDSR) analysis using summary-level GWAS data from Kember et al. 2022, as well as publicly available GWAS data from other brain disorders and traits that are not directly related to brain cell types. In addition to testing whether the cell-type-specific regulatory elements in OFC are enriched for OUD risk SNPs, our multiomic data set should allow for the nomination of specific functional OUD risk SNPs, which could be tested in the future studies.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Duan, Rui \",\n\"HARVARD SCHOOL OF PUBLIC HEALTH\",\n\"A federated transfer learning framework for improving genetic risk prediction in ancestry-diverse populations\",\n\"May31, 2022\",\n\"approved\",\n\"The limited representation of minorities and disadvantaged populations in large-scale clinical and genomics research has become a barrier to translating precision medicine research into practice, which may further exacerbate known health disparities, and there is an urgent need for methods in the field of statistics and ML to address biases and unfairness caused by such lack of representation. In this project, we aim to improve knowledge discovery and risk prediction in underrepresented populations through two strategies: (1) leverage the shared knowledge across populations through novel transfer learning approaches that allow rapid evolution of predictive models as they are \\u00e2\\u0080\\u009ctransferred\\u00e2\\u0080\\u009d between data sets to jumpstart the modeling process; (2) develop efficient and safe information-sharing strategies to facilitate biomedical data integration, improve the diversity, and reduce the potential for model overfitting. \",\n\"Polygenic risk scores (PRS) have shown promising potentials for early disease detection, prevention and intervention. However, as existing genome-wide studies were predominantly conducted in European-ancestry (EA) populations, the performance of PRS is much poor in non-EA populations compared with EA populations, which may exacerbate existing health disparities. The goal of our proposed work is to develop novel transfer and federated learning methods to improve the performance of PRS in ancestry-diverse populations. We will (1) develop a transfer learning framework based on penalized high-dimensional regressions, which transfer the shared genetic architectures learned from individual-level datasets and available GWAS summary statistics to a target population while accounting for the heterogeneity across populations; (2) develop communication-efficient federated learning algorithms to increase the training sample size of the target population by incorporating multiple study cohorts with communication efficiency and individual-level privacy protection. Completion of this project will lead to novel tools that produce high quality PRS with better predictive performance for underrepresented non-EA populations, which can ultimately help advance precision medicine and reduce the disparity in PRS-based research.   We will apply the proposed methods to combine summary statistics obtained from the applied datasets as well as summary statistics obtained from datasets outside of dbGAP, including the UK biobank, and construct PRS for both quantitative traits and binary traits including cancers, cardiovascular diseases, Alzheimer's disease, and mental disorders, targeting underrepresented populations. We will use metrics such as R squared and AUC to evaluate the prediction models for quantitative traits and binary traits respectively.   We intend to publish methodology papers in the field of statistical genetics and machine learning to share our methods to the scientific community. We will also publish our study results of PRS models to genetic study community and precision medicine community. We will publish open-source software packages in github and cran. Our proposed methods can be generalized to other areas in health and biomedical research to facilitate knowledge discovery and risk prediction in a minority sub-population, and can help reduce health disparities and advance precision medicine. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Dubin, Ruth \",\n\"UNIVERSITY OF CALIFORNIA, SAN FRANCISCO\",\n\"The Impact of Renal Dysfunction on Biological Mechanisms of Cardiovascular Disease\",\n\"Jan15, 2020\",\n\"expired\",\n\"Over 20 million people in the United States have chronic kidney disease, and all of them are at high risk for development or progression of heart disease. Unfortunately, we have few effective therapies for this group of patients, because we have limited understanding of the biology of their disease. In this project, we will utilize a unique dataset of 1100 proteins that have been measured on about 1000 patients with heart disease; after following these 1000 patients for 10 years, we know which ones had a heart attack or were hospitalized for heart failure. We will select proteins that are associated with a risk of either of these clinical events. We anticipate that we will find new proteins that cause cardiovascular disease in patients with kidney disease. The level of such a protein is likely to vary person to person, and some of this variation depends on the person's genes. We will confirm the importance of any new protein using genetic information collected from thousands of patients and available publically on research sites.\",\n\"    Studies in both community-based and high-risk populations have confirmed that chronic kidney disease(CKD)confers at least a two-fold increase in cardiovascular (CV) risk. Approximately 10% of the $200 billion 2008 United States Medicare budget was devoted to patients with concomitant CKD and cardiovascular disease (CVD).Despite this enormous public health impact, we have a limited understanding of the biology fueling accelerated CVD phenotypes in patients with CKD.      The question we wish to answer is \\u00e2\\u0080\\u009cWhich biomarkers/pathways explain the risk of cardiovascular disease in patients with chronic kidney disease?\\u00e2\\u0080\\u009d Previous investigations typically focus on a single biomarker\\u00e2\\u0080\\u0099s association with adverse CV outcomes, and use multivariable adjustment to clarify the role of renal dysfunction in this association. This approach is limited, not only due to its myopic single-marker focus, but also because confounders cannot be distinguished from mediators by multivariable adjustment alone.      Our group has recently published the first large-scale aptamer proteomics study of CV risk prediction, in partnership with Somalogic (Boulder CO).The Heart and Soul Cohort includes 1024 participants with stable coronary artery disease, of whom 978 had proteomics (1124 proteins) measured at baseline. Approximately 30% had CKD (defined as estimated glomerular filtration rate (eGFR) <60ml/min/1.73m2 or albuminuria >30mg/g), and prior work has shown that cystatin C, a sensitive measure of renal dysfunction, is independently associated with atherosclerotic CVD and heart failure (HF) in the cohort. In the current project, rather than risk prediction, we will investigate novel associations between proteins and either atherosclerotic CVD or HF in the Heart and Soul Cohort, and then corroborate findings with GWAS analyses performed for baseline and longitudinal CVD in the Framingham cohort.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Duncan, Laramie \",\n\"STANFORD UNIVERSITY\",\n\"Investigating a Male Protective Effect in PTSD Using an Ancestry-adjusted Polygenic Risk Score Meta-analysis\",\n\"Dec22, 2020\",\n\"approved\",\n\"Many psychiatric diseases show differences in prevalence by sex. For example, autism has a female to male prevalence ratio of ~1:4. Significant research has gone into understanding why these prevalence differences exist. For autism, there is evidence of a Female Protective Effect (FPE), meaning that more genetic risk factors are required for females to develop autism as compared to males. PTSD has a female to male sex prevalence ratio of 2:1. The aim of this project is to determine if there is a Male Protective Effect (MPE) in PTSD; that is, if males require more genetic risk factors to develop PTSD than females. To do so, we will use datasets from the Psychiatric Genetics Consortium PTSD group. In order to assess genetic risk for PTSD of each individual, we will use a technique called polygenic scoring. Polygenic scoring yields a single number\\u00e2\\u0080\\u0094a genetic score\\u00e2\\u0080\\u0094that quantifies an individual\\u00e2\\u0080\\u0099s genetic propensity for a given phenotype (e.g. PTSD). We can compare the polygenic scores of the males with PTSD to the females with PTSD in our datasets to determine if an MPE exists.\",\n\"Many psychiatric diseases show differences in prevalence by sex. For example, autism has a female to male prevalence ratio of ~1:4. Significant research has gone into understanding why these prevalence differences exist. For autism, there is evidence of a Female Protective Effect (FPE), meaning that more genetic risk factors are required for a female to develop autism as compared to a male. PTSD has a female to male sex prevalence ratio of 2:1. The aim of this project is to determine if there is a Male Protective Effect (MPE) in PTSD; that is, if males require more genetic risk factors to develop PTSD than females. In order to assess genetic risk for PTSD of each individual, we will use a technique called polygenic scoring. Polygenic scoring yields a single number\\u00e2\\u0080\\u0094a genetic score\\u00e2\\u0080\\u0094that quantifies an individual\\u00e2\\u0080\\u0099s genetic propensity for a given phenotype (e.g. PTSD). The predictive power of psychiatric polygenic risk scores has dramatically increased in the last several years, meaning we can now more accurately assess genetic risk for PTSD across males and females.  In the proposed analysis we will compare polygenic risk scores in female cases to male cases.  To adjust  for ancestry, we will regress out 10 principal components from the polygenic risk scores of each participant on a sample-by-sample basis.  This will leave us with ancestry-adjusted residuals of the polygenic risk scores, which will be used to determine the presence of an MPE in each sample.  We will then conduct a meta-analysis across the individual datasets which range in sample size from 208 to 6,565 participants. We will also use the PTSD polygenic risk scores in the typical manner (i.e. for prediction of PTSD and related phenotypes). Our ability to determine the presence of an MPE in PTSD  greatly depends on the quality of the polygenic risk score used. Thus, we are requesting access to the most recent GWAS of PTSD, that from the Million Veteran Program of over 250,000 participants, with which we can derive the most powerful polygenic risk score for PTSD.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Elam, Kit \",\n\"INDIANA UNIVERSITY BLOOMINGTON\",\n\"Examining Polygenic Effects on Longitudinal Depression Symptoms\",\n\"May13, 2022\",\n\"approved\",\n\"We plan to examine genetic influences on longitudinal depression symptoms in two independent samples. We will investigate whether these influences are related to individual characteristics, substance use, and family factors, or possibly buffered by a family-centered intervention. We will examine this in African American and European American participants. \",\n\"We request access to aggregate summary statistics (not individual-level data) from the bi-ancestral GWAS of depression in the Million Veteran Program sample (MVP; Levey et al., 2021; phs001672). We will not combine the requested data with other datasets outside dbGaP. Aggregate summary statistics will be used to create polygenic scores (PGS) for depression in African American (AA) and European American (EA) participants using PRS-CSx in two independent datasets. Participants in the Early Steps and Project Alliance 1 (PAL1) samples were randomized to a family-based intervention or control condition and assessed over time. Both samples primarily consist of African American (AA) and European American (EA) participants.   Our primary objective is to examine PGS prediction of longitudinal depression symptoms in two independent samples (Early Steps: age 2-16; PAL1 age 16-31). Guided by preliminary analyses, a secondary objective is to examine for moderation and/or mediation of genetic effects on depression by individual, peer/familial, and sociodemographic characteristics. We will examine for the moderating role of substance use as well as an outcome of depression. We will also examine for buffering effects of the intervention condition. Our final objective is to examine if pathways differ across AA/EA subgroups. The proposed research is consistent with the data use limitations as we plan to leverage aggregate GWAS data to examine effects on longitudinal health outcomes but do not plan to study population origins or ancestry.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Ellinor, Patrick \",\n\"BROAD INSTITUTE, INC.\",\n\"Genetics of atrial fibrillation\",\n\"Aug09, 2022\",\n\"approved\",\n\"Atrial fibrillation (AF) is a cardiac arrhythmia affecting 3 Million Americans. The causes of the disease are not clear, and current treatment methods are only partially successful.  Our research group has studied the genetic causes of AF over the last decade, and genetic screening has helped us identify mutations causing the disease in families with AF, as well as more common genetic variants linked with this disease. Recently, whole genome- and exome sequencing have made it possible to sequence large regions of the genome, and these types of studies increase our power to detect genetic variants associated with AF. dbGaP is a unique source of genetic and phenotypic data. We propse to analyze available exome sequencing, genome sequencing, exome chip and genome wide genotyping data in individuals with and without AF. We hope that our work will allow us to identify new genes and pathways, that will eventually lead to identification of new molecular targets for the treatment of AF.\",\n\"In this project we seek access to exome sequencing data, whole genome sequence data, exome chip, and genome wide genotyping data from studies with data on atrial fibrillation (AF) that are registered in dbGaP. The primary goal of the current proposal will be to identify novel genetic variants associated with atrial fibrillation by integrating exome sequence data, exome chip and genome wide genotyping data. We plan to combine the raw sequence data available in individuals with exome sequence data and AF and plan on a joint recalling of these datasets. When available, exome chip and genome wide genotyping data will be used to facilitate quality control metrics in each study. Cases will consist of individuals with AF in the requested cohorts using currently available phenotypic data in dbGap; referents will consist of individuals without AF in each cohort.  We do not believe that the combination of these genetic data will impose any new risks on the participant in the cohorts accessed through dbGap. The data obtained from dpGaP will not be shared with any additional collaborators. We request de-identified data, and will not attempt to decode any personal information using the requested data. The obtained data will be used for genetic research that facilitates discovery of variation related to cardiovascular disease (CVD), and for no other purposes. We plan to include cohorts from other continents in the combined genetic analyses, but the data included from the dbGaP will not be shared with other collaborators, and all analyses will be performed by our research group led by the PI. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Erdos, Michael Reynolds\",\n\"NIH\",\n\"Genetic drivers of type 2 diabetes in diverse ancestries\",\n\"Dec01, 2021\",\n\"approved\",\n\"Type 2 diabetes (T2D) is a leading contributor to global morbidity and mortality. Previous genetic studies have demonstrated the role of genetic predisposition in T2D risk and successfully identified >240 loci associated with T2D. However, most lie in non-coding regions of the genome, masking the underlying \\u00e2\\u0080\\u009ceffector gene\\u00e2\\u0080\\u009d. Additionally, these studies were primarily performed in individuals of European ancestry, threatening to exacerbate existing health disparities among non-European communities disproportionately suffering from T2D as genetic findings are integrated into clinical interventions. This study aims to expand our genetic understanding of T2D in non-European ancestry groups by combining data from diverse cohorts to perform a trans-ancestry genetic analysis of T2D. Such a study has the potential to make substantial progress towards eliminating existing health disparities by proposing genetically-derived preventative interventions (e.g., polygenic scores) while simultaneously advancing our understanding of the molecular drivers of T2D by nominating likely-causal variants.\",\n\"Type 2 diabetes (T2D) is a leading contributor to global morbidity and mortality. T2D is characterized by hyperglycemia resulting from \\u00c3\\u009f-cell dysfunction and impaired insulin-stimulated response, but the molecular and cellular pathways driving these observations remain elusive. Previous genetic studies have demonstrated the role of genetic predisposition in T2D risk and successfully identified >240 loci associated with disease. However, most lie in non-coding regions of the genome, masking the underlying \\u00e2\\u0080\\u009ceffector gene\\u00e2\\u0080\\u009d. Additionally, these studies were primarily performed in individuals of European ancestry, threatening to exacerbate existing health disparities among non-European communities disproportionately suffering from T2D-related health complications as genetic findings are integrated into clinical interventions. Expanding our genetic understanding into non-European ancestries is essential as these studies will mitigate such health disparities and improve our ability to identify causal loci and mechanisms.  This study will combine data from diverse cohorts, with an emphasis on non-European ancestry groups, to perform a trans-ancestry common variant genome-wide association study of T2D, identifying trans-ancestry and ancestry-specific T2D-associated loci. We will fine-map these loci to identify likely causal variants and perform experimental validation of the strongest candidates using genome engineering techniques. Finally, we will generate and validate polygenic scores in each ancestry major group. Together, this study has the potential to make substantial progress towards eliminating existing health disparities by proposing genetically-derived preventative interventions while simultaneously advancing our understanding of the molecular drivers of T2D.  We are requesting the T2D summary statistics from the Million Veterans Program (MVP) to include in our meta-analysis. Due to the diversity represented in MVP, this dataset will be crucial to maximize our sample size of non-European ancestries. All data will be stored on the NHGRI Trek cluster, which is secured behind the NIH firewall. All approved users have received data security training. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Evans, Luke \",\n\"UNIVERSITY OF COLORADO\",\n\"Association mapping and the genetic architecture of complex traits.\",\n\"Sep18, 2020\",\n\"approved\",\n\"I am requesting the genotype and phenotype data of the datasets to perform analyses on the genetics of biometric measures (e.g., height, BMI), and behavior and substance use such as nicotine and alcohol dependence and anxiety. Specifically, I will use a range of methods to assess the overall genetic contribution to these complex traits (i.e., their heritability), identify variants and genes that influence these traits (through GWAS and TWAS), the genetic relationships among traits (their genetic correlations), and assess functional enrichment of associations by leveraging external annotation and expression data. The analyses will lead to a greater understanding of the genetics of complex traits, including the genetic etiology of nicotine and alcohol use and internalizing disorders such as depression and anxiety.\",\n\"The data will be used for health/medical/biomedical purposes, including studies of the genetic basis of biometric and health measurements, focusing on behavior and substance use phenotypes, in particular depression, anxiety, alcohol use and nicotine use. The first study is a study on the genetics of smoking behaviors. I will use the data for GWAS as well as transcriptome-wide association studies (TWAS). I will combine the GWAS and TWAS summary statistics from these datasets with other approved dbGaP datasets such as NESARC-III, HRS, and Add Health, along with internal datasets here at CU Boulder and the UK Biobank via meta-analysis of the SNP- and gene-level effect sizes and p-values estimated within each study separately. TWAS will be performed using dbGaP-approved access GTEx, HBCC, and NABEC expression data to obtain SNP effects on expression, and these estimated SNP effects will be used with the individual studies to estimate gene expression-trait associations within each study. I will combine the resulting summary statistics from each dataset via meta-analysis.  Analyses will include GWAS and TWAS, association enrichment of functional annotations, genetic correlation of traits, and SNP-heritability estimation. The proposed research falls within the General Research Use consent of the requested data. All data are de-identified, and no attempt will be made to re-identify individual participants. I have received approval following an expedited review of my proposed research by the CU IRB (document attached, full proposal available upon request). The data will be stored at the CU Research Computing, a supercomputing facility with robust security here at CU Boulder and with dedicated staff to maintain it.  We intend to publish any findings, and intend to share our findings with the scientific community via presentations at scientific conferences (e.g., WCPG, ASHG, BGA, SRNT). \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Fatumo, Segun \",\n\"LONDON SCH/HYGIENE &amp; TROPICAL MEDICINE\",\n\"Genomic determinant  of Complex Diseases in African ancestry individuals  \",\n\"Oct27, 2021\",\n\"expired\",\n\"Africa is undergoing a demographic transition leading to an increasing prevalence of non-communicable diseases (NCDs) and a growing cause of morbidity and premature mortality. In this study, our aim is to identify genetic variants that influence NCDs. This project has the potential to lead to advanced understanding of the causes of NCDs in Africa and paving the way for future improved prevention and treatment. \",\n\"Africa is undergoing a demographic transition leading to increasing prevalence of non-communicable diseases (NCDs) and growing cause of morbidity and premature mortality. NCD predictors are all partly heritable. We aiming to use African ancestry individual GWAS summary statistics to identify genetic associations loci, evaluate transferability of the polygenic risk score in continent African and performed Mendelian Randomisation analysis of NCDs (including CVD, Hypertension (HT), Type 2 Diabetes (T2D), Lipid traits, kidney function, post-trauma stress disorder (PTSD), Cancer. Our study provide opportunities to: (1) discover novel disease susceptibility loci; (2) refine (fine-map) association signals at new and existing loci; (3) evaluate transferability of polygenic risk score (4) assess causal relationships of NCDs in Africa using Mendelian randomization\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Fatumo, Segun \",\n\"LONDON SCH/HYGIENE &amp; TROPICAL MEDICINE\",\n\"Investigating Predictive Utility of Polygenic Risk Scores of Chronic Kidney Disease in Africans\",\n\"Apr20, 2021\",\n\"approved\",\n\"Currently, more than 50 million people in Africa have Chronic Kidney Disease (CKD) making it the continent with the greatest burden of CKD in the world. With the rapid advances in genotyping technologies, it is anticipated that the advent of the global knowledge in CKD genomics could be translated to clinical benefits. A key application of genomic data is in disease risk prediction, which can potentially contribute to precision medicine such as individualized preventive strategies and early diagnosis of diseases. In this project, we aim to develop an approach called polygenic risk score that can help to identify a sub-group of people in Africa who may be at higher risk of kidney disease in order to provide earlier intervention and treatments.  \",\n\"Currently, more than 50 million people in Africa have Chronic Kidney Disease (CKD) making it the continent with the greatest burden of CKD in the world. With the rapid advances in genotyping technologies, it is anticipated that the advent of global knowledge in CKD genomics could be translated to clinical benefits. A key application of genomic data is in disease risk prediction, which can potentially contribute to precision medicine such as individualized preventive strategies and early diagnosis of diseases. In view of the limited transferability of Polygenic Risk Score (PRS) across ethnicities, in this project, we will develop the PRS for prediction of CKD within African populations. We will evaluate best approaches for using a diverse discovery dataset from trans-ethnic Genome-wide association studies (GWAS) meta-analysis and a target dataset individuals from continental Africa to develop and evaluate a novel PRS for the prediction of CKD leveraging on diverse datasets of trans-ethnic Genome-wide association studies summary statistics from multi-ethnic groups. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Fitzgerald, Kathryn \",\n\"JOHNS HOPKINS UNIVERSITY\",\n\"Comorbidity and outcomes in multiple sclerosis: A polygenic score approach\",\n\"Apr12, 2023\",\n\"approved\",\n\"Health conditions like depression, heart disease, diabetes, obesity, or high blood pressure are common in people with multiple sclerosis (MS). A person with MS who has one of these conditions may be at higher risk of their MS worsening. For example, a person with MS and high blood pressure might have more MS disability than a person with MS who does not have high blood pressure. However, the results from studies so far have not been the same. Sometimes in studies where participants are not randomized (like the flip of a coin) to a study treatment, there can be bias. The bias could have played a role in the different conclusions. We want to reduce these biases by studying genetic factors that are related to a person\\u00e2\\u0080\\u0099s risk for diabetes or high blood pressure. Genetic factors do not change over a person\\u00e2\\u0080\\u0099s lifetime. There are many of these genetic factors that play a role in a person\\u00e2\\u0080\\u0099s risk for diabetes or high blood pressure. Together, they are called \\u00e2\\u0080\\u0098diabetes genetic scores\\u00e2\\u0080\\u0099 or \\u00e2\\u0080\\u0098high blood pressure genetic scores.\\u00e2\\u0080\\u0099 The goal of this study is to see if these genetic scores can predict MS progression. The genetic scores will be derived using summary information that from previously conducted studies as a part of phs001672 and the Million Veteran\\u00e2\\u0080\\u0099s Project. \",\n\"Background: The disease course in people with multiple sclerosis (PwMS) is heterogenous; emerging studies indicate that comorbidity could contribute to the observed variability in disease course. For example, cardiometabolic comorbidities like obesity, diabetes, hypertension and hyperlipidemia are overrepresented in PwMS; and are also associated with worse MS outcomes. The goal of this proposal is to improve our understanding of the contribution of comorbidity to disease outcomes in MS. We will accomplish these goals using a novel polygenic score (PGS) approach and will leverage four large pre-existing multi-national cohort studies of PwMS with detailed clinical, imaging, and genetic information. We will accomplish this goal using a novel polygenic score (PGS)8,9 approach and by leveraging four large pre-existing cohort studies of PwMS with detailed clinical, imaging, and genetic information. We will also use summary measures of polygenic inheritance (e.g., the weighted sum of the number of known genetic risk variants for a given disorder within each individual) for each cardiometabolic disease derived from large-scale genome wide association studies (GWAS; e.g., derived from summary statistics available from phs001672 and the Million Veteran\\u00e2\\u0080\\u0099s Project) as an instrument to study the effect of each comorbidity on changes in MS outcomes. By focusing on genetic factors, which do not change from birth, we can minimize confounding and effectively exclude reverse causation as an explanation for associations between comorbidity and outcomes. Thus, we mitigate some weaknesses of prior work. Our central hypothesis is that cardiometabolic and psychiatric comorbidity and related traits (e.g., lipid levels, BMI, among others), as measured using PGSs, predict clinical and radiological outcomes in PwMS.   Aim 1. To examine the association between psychiatric and cardiometabolic PGS with clinical, imaging and biomarker outcomes in people with MS. Hypothesis: Individuals with higher PGS for common cardiometabolic comorbidities will have worse disease outcomes, including faster disability progression (primary outcome) as well as more disease activity (relapses) and radiographic and biomarker evidence of disease worsening (secondary outcomes). Analysis Plan: We will generate PGS for (1) indicators of obesity including body mass index and waist circumference; (2) lipid levels including total cholesterol and low-density lipoprotein levels; (3) diabetes, insulin levels, and hemoglobin A1c; (4) hypertension including systolic blood pressure and diastolic blood pressure; (5) depression. PGS will be generated using established guidelines with adherence to strict reporting standards. We will pool resources from four well-characterized cohorts of PwMS including 2366 PwMS with detailed genetic information and rigorous follow-up averaging 2.8\\u00c2\\u00b11.0 years. The analytic model for each outcome will based on the distribution of the outcome in question and will consider adjustment for age, sex, genetic ancestry and MS therapy. PGS will be generated using PRS-CSx, which is a method that improves polygenic prediction in a multi-ethnic context through the inclusion of GWAS summary statistics from multiple populations. Summary statistics from populations of interest as available through phs001672 and the Million Veterans Project include individuals of European, African American, Hispanic, and Asian ancestry. The primary outcome will be change in the MS Functional Composite (MSFC). The MSFC is a validated metric of impairment, showing good longitudinal correlation with clinical disability and responsiveness to treatment. We will fit longitudinal (e.g., linear mixed effects [LME] models). We will consider adjustment for age, sex, genetic ancestry, and MS therapy. P-values will be adjusted for multiple comparisons using appropriate methods.61 Horizontal pleiotropy (e.g., the PGS may affect traits that influence the outcome, other than via the trait of interest) is a concern. For analyses showing potentially significant findings, we will replicate PGS analyses using Mendelian randomization and test for such bias therein.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Fletcher, Jason \",\n\"UNIVERSITY OF WISCONSIN-MADISON\",\n\"Using Summary Statistics from Multi-Ethnic GWAS to Develop and deploy new polygenic scores in social science surveys\",\n\"Apr06, 2020\",\n\"closed\",\n\"This research will leverage summary statistics from genetic analysis in the Million Veteran Program to create genetic-based predictions (polygenic scores) for PTSD and substance dependency in multi-ethnic social survey data\",\n\"This research will leverage summary statistics from genetic analysis in the Million Veteran Program to create genetic-based predictions (polygenic scores) for PTSD and substance dependency in multi-ethnic social survey data.  Our research team has developed new methods of utilizing summary statistics from large scale GWAS in order to create polygenic scores in genotyped social science surveys, such as the Health and Retirement Study and Add Health, among others.  Most current large scale GWAS is focused on individuals of European ancestry, which makes the creation of polygenic scores for other populations groups difficult and less predictive.  We will extend our methods that have focused on GWAS summary statistics in European ancestry analyses in order to create and use new polygenic scores created from non-European ancestry GWAS, like the Million Veterans Program.    These new polygenic scores will be used in follow up analyses in the HRS and Add Health, among other datasets, in order to explore how the polygenic scores are associated with a large range of health measures not included in the Million Veterans Program data.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Fongang, Bernard \",\n\"UNIVERSITY OF TEXAS HLTH SCIENCE CENTER\",\n\"Gene expression and DNA methylation signatures of the cardiovascular health and neurocognitive endophenotypes\",\n\"Sep07, 2022\",\n\"approved\",\n\"Cardiovascular health is associated with dementia, but whether one could use cardiovascular risk factors drugs (blood pressure, for example) to prevent or treat dementia or cognition is controversial. We will use the requested datasets to determine the shared genetic, genomic, and epigenomic features between the cardiovascular risk factors and dementia endophenotypes. Our results will lay the foundation for future studies and help understand the common biological mechanisms between cardiovascular diseases and dementia.\",\n\"Cardiovascular diseases (CVD) and dementia have common risk factors. But whether CVD drugs could be used to treat or prevent dementia is controversial. We are requesting genetic, genomic, and epigenomic data that will be used to establish the common biological mechanisms of CVD risk factors and dementia endophenotypes. We will use biostatistical and bioinformatic models to (a) determine the genetic, genomic, and epigenomic signatures of CVD risk factors; (b) the genetic, genomic, and epigenetic signatures of dementia endophenotypes; (c) identify the shared features; (d) and build a predictive model that will establish the most critical elements influencing memory decline and the development of dementia. The requested data will be used for biomedical research only, and the analyses will be restricted to the aims described above. The data will be stored in our secure platforms and accessible only to the PI. Moreover, we will not attempt to reveal the identity/ancestry of any participant. Finally, findings from our study will be published and shared with the scientific community. The dbGap accession information with the relevant version number of the analyzed datasets will be included in the publications. The data hosted by dbGap are well suited for our project since the genetic, imaging, and clinical data are collected on the same participants.  This project will provide valuable information to the research community. Because brain changes related to dementia start decades before symptoms appear, understanding the molecular connections between CVD risk factors and neurocognitive endophenotypes could offer a chance to prevent or delay memory loss and other debilitating symptoms. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Franceschini, Nora \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"Polygenic risk scores for cardiovascular traits \",\n\"Aug31, 2021\",\n\"approved\",\n\"We will use the agenetic analysis results for cardiovascular traits to study genetic risk in additional populations. \",\n\"We propose to create polygenic risk scores using summary statistics from GWAS of cardiometabolic traits to predict cardio-renal outcomes in diverse populations. We will also use the summary statistics to replicate findings identified in GWAS of diverse populations. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Fullerton, Janice \",\n\"UNIVERSITY OF NEW SOUTH WALES\",\n\"ENIGMA addiction project - Sydney node\",\n\"Aug30, 2022\",\n\"approved\",\n\"Genome wide association studies (GWAS) identify genetic variants associated with a trait such as problematic substance use. We will use two GWAS from dbGAP which targeted nicotine dependence and problematic alcohol use to generate individualized genetic risk scores for these traits in other samples. We will examine whether these risk scores are related to behavior (e.g. age of onset and frequency of substance use, and measures of impulsivity) or brain features detected using MRI, including measures of brain structure (e.g. cortical and subcortical grey matter volume and white matter tract integrity) and function.\",\n\"Polygenic risk scores (PRS) are a powerful tool to examine the genetic liability of a specific trait in another population. Our aim is to investigate the genetic liability to substance use disorders (e.g., alcohol) in independent population-based cohorts in relation to behavior measures and to identify brain intermediate phenotype measures associated with greater genetic risk for problematic alcohol use. PRS will be created using PLINK 2 software, based on the results of the Genome Wide Association Studies (GWAS) for problematic alcohol use (PMID 32451486 - phs001672.v7.p1) and other datasets including cigarettes per day (PMID 30643251). These PRS will be regressed onto behavioral phenotypes in local datasets (e.g., substance use measures including age of onset and frequency of use, and impulsivity measures) and onto brain measures including structure (cortical and subcortical measures and diffusivity) and function. This project does not require individual-level data from dbGAP from the mentioned studies (only GWAS summary statistics are required) and therefore personal identifying information for research participants in the original study is not requested, and the objectives of the current project are limited to health/medical/biomedical purposes, and does not include the study of population origins or ancestry.  Data will be stored on secure password-protected servers at Neuroscience Research Australia (NeuRA), a not-for-profit research institute of around 300 academic researchers affiliated with The University of New South Wales (UNSW). NeuRA's research grants, governance and ethics approvals are managed by UNSW. NeuRA's security systems are independently overseen by NeuRA's Director of IT Security, Andrew Cartwright, who has a conjoint appointment at UNSW, and specific training in high-level data security (e.g. 2022 NIST cybersecurity Framework practitioner, and 2020 Cyber security Foundation+ practitioner accreditatons) with institutional oversight. NeuRA's computing capacity is supported by computing infrastructure at UNSW (e.g. using a configuration on the main UNSW virtual private network concentrator, and access to UNSW data archive). NeuRA is fully resident on the UNSW data network, which allows network traffic to pass through the UNSW next-generation firewall.  The generation of the PRS in local datasets will occur on the NeuRA servers using imputed genotype data created using the ENIMGA genetics pipeline (1kGP p3v5 cookbook) by investigator Associate Professor Janice Fullerton (j.fullerton@neura.edu.au; ERA commons username: JANICE_FULLERTON) and stored locally. The PRS will be generated locally, and the results analysed in collaboration with local clinical collaborators at UNSW School of Psychiatry, Scientia Professor Philip Mitchell (p.mitchell@unsw.edu.au) and Dr Gloria Roberts (robertsg@unsw.edu.au), as per UNSW HREA ethical approval HC220120 (23Mar2022-22Mar2027). Summary data will then be contributed to the ENIGMA Addiction consortium (https://www.enigmaaddictionconsortium.com/), with meta-analysis led by Dr Renata Cupertino (Renata.Cupertino@uvm.edu), and project co-headed by Professor Hugh Garavan (Hugh.Garavan@uvm.edu) and Professor Scott Mackey (msmackey@uvm.edu) at the University of Vermont, Burlington, VT.   Data will be used for the sole use of the proposed project, which is in line with the research participants\\u00e2\\u0080\\u0099 informed consent. We acknowledge the confidentiality of the data, that the privacy of participants must be protected, and public posting of summary results are not allowed. We agree not to use the requested datasets to identify or contact individual participants from whom data and/or samples were collected. We certify our agreement with the expectations and terms of access, including the requirement of an additional request if cloud computing is deemed necessary. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Gandal, Michael \",\n\"UNIVERSITY OF CALIFORNIA LOS ANGELES\",\n\"Building integrative CNS networks for genomic analysis of autism and schizophrenia\",\n\"Jun18, 2020\",\n\"closed\",\n\"Advances in genomics and genetics have opened a new window through which to understand psychiatric diseases, such as autism spectrum disorder (ASD) and schizophrenia, but also reveal that they are all very genetically heterogeneous. Gene networks have shown promise for organizing the complexity and heterogeneity of genetic factors that contribute to ASD. Here, through highly collaborative efforts using multiple public and private data sets we develop a rigorous, multi-level and experimentally tested framework for integrative network studies in neuropsychiatric diseases.\",\n\"Large-scale genomic investigations have begun to illuminate the genetic contributions to major psychiatric illnesses including autism spectrum disorder (ASD) and schizophrenia. In ASD,  rare large effect variants provide novel causal anchors to understand its neurobiological basis, and to understand convergence and divergence in disease mechanisms. These findings, not unlike other psychiatric disorders, also emphasize extreme genetic heterogeneity. We and others have shown that gene and protein networks provide an organizing framework for understanding heterogeneous psychiatric disease genetic risk in a unified biological context. The emergence of large-scale genomic and epigenetic data from human brain, and growing knowledge of genetic variation involved in ASD and other psychiatric diseases, coupled with advances in methodology, provides an unprecedented opportunity for comprehensive integrative analyses. We bring together a collaborative group of investigators, each with distinct areas of expertise critical for understanding psychiatric diseases, that have not been combined before, to develop a framework for integrative genomic network analysis. The work proposed here aims to develop a comprehensive framework and test optimal methods for integration of gene expression and protein-protein interaction (PPI) networks in the brain with genetic and epigenetic data \\u00e2\\u0080\\u0093 networks that will be iteratively refined using experimental data. We will construct networks representing multiple brain regions in adulthood and development through rigorous combination of multiple transcriptomic data sets from ASD and control brain, developing and validating methods for integration of splicing and expression levels within gene networks. These networks will be refined to inform tissue specific PPI inference, validated via experimental tissue-specific PPI. We will further identify causal drivers by integration of genetic and epigenetic data, identifying QTL effects on RNA, splicing and protein levels. Completion of these aims will lead to more valid and comprehensive CNS networks thereby significantly advancing our understanding of ASD associated variants and causal neurobiological pathways. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Garvin, Michael \",\n\"UT-BATTELLE, LLC-OAK RIDGE NATIONAL LAB\",\n\"Genome-wide association study of heroin abuse using public controls\",\n\"Jun22, 2022\",\n\"closed\",\n\"Approximately 50% of the risk for heroin abuse is attributable to genetic factors. However, few genes have been consistently associated with heroin abuse, and it is likely that a number of genes contributing to heroin abuse have not been identified. The proposed study would test associations between heroin abuse and genetic variants across the genome, with the goal of uncovering biological mechanisms that underlie drug addiction. More broadly, we also examine genetic risks for substance use disorders regardless of specific drug used.  \",\n\"The goal of the proposed study is to identify novel genetic determinants of heroin abuse and related substance use disorder (SUD) phenotypes by conducting genome-wide association (GWA) studies as well as candidate variant studies for SUD phenotypes.  Although GWA studies have successfully identified genetic variants contributing to other SUD phenotypes (e.g. nicotine dependence), there are few GWA studies of rarer high-risk substance abuse.  We propose to use a case-control study design, drawing heroin abuse cases from the Urban Health Study (UHS) which includes >12,000 people who inject drugs (PWID) enrolled from 1986 to 2005 (Kral et al., 2001).  We have recently completed genotyping of over 10,000 PWID samples on Illumina arrays (Omni1-quad and 2.5M) under grant R01 DA044014 (PI Eric Johnson).  Of these PWIDs, 66% are heroin abuse cases who injected 10+ times in the past 30 days (ONDC criteria of abuse). This cohort is 53% African American, 37% European American, and 10% Hispanic American. We use dbGaP studies genotyped on Illumina arrays to match to the UHS cases as population controls generating a UHScases+dbGaPcontrols cohort for GWAS. We have published two studies using this approach (Johnson, et al, 2015; Hancock et al. 2015). We use additional dbGaP studies with SUD phenotype data to test for replication of findings from UHS+dbGaPcontrols cohort or in meta-analyses. In the current period we will continue to use these previously acquired cohorts and add dbGaP studies as population controls for our newly genotyped UHS cases to study SUD phenotypes. We will also request new dbGaP studies with SUD phenotype data for replication and/or meta-analyses as they become available.  We will continue to participate in the Addiction group of the Psychiatric Genetic Consortium (PGC): analyzing these data in large meta-analyses with other investigators.  PGC quality control and analysis are being conducted at the MGH Simches Research Center \\u00e2\\u0080\\u0093 Harvard University. We are also leading the NIDA Genetics Consortium GWAS of Opioid Addiction, which combines summary statistics from 13 cohorts. As part of NIDA Smoke Screen Initiative, a number of new cohorts that are informative for heroin addiction have been genotyped on the Axiom_Smokesc1 array. Most of these cohorts are case only. To leverage these data we will need to extend our population control approach to include matching controls derived from whole genome sequencing (WGS) to array genotyped cases given the very low overlap between the Axiom_Smokesc1 array and other Axiom and Illumina array. We will develop a new protocol to match TopMed WGS cohorts as population controls to the Axiom_Smokesc1 array genotyped cases; the WGS data containing a complete overlap with the Axiom_Smokesc1. All results from the analyses using these data will be published and summary statistics shared. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Gillespie, Nathan \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"The Structure of Genetic Risk Factors for Major Psychiatric Disorders v4\",\n\"Dec22, 2022\",\n\"approved\",\n\"By combining the aggregated results from large genome-wide association scans (GWAS) of psychiatric disorders and complex behaviors will will determine the overall size of the genetic correlations between them. \",\n\"Objective: Recent advances in genome-wide association scans (GWAS) of psychiatric disorders and statistical innovations now permit researchers to revisit hypotheses regarding the genetic structure of psychiatric disorders and sources of comorbidity. We will determine if the genetic structure observed in family studies between Externalizing, Internalizing (including anxiety) and Psychotic spectrum disorders can be confirmed using statistical methods that analyze GWAS-based genetic correlations. Methods: We combined publicly available GWAS summary statistics for 12 major psychiatric disorders (including the results based on the MVP analysis of anxiety), six substance use measures and two personality domains. Using the genomicSEM software package, we will estimate GWAS-based genetic correlations between all 12 disorders and 8 traits before applying confirmatory factor analysis (CFA) to identify the genetic structure.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"GIZER, IAN \",\n\"UNIVERSITY OF MISSOURI-COLUMBIA\",\n\"Comorbidities among mental and physical disorders using data from Genome-Wide Association Studies (GWAS)\",\n\"Dec18, 2018\",\n\"approved\",\n\"Individuals with substance use problems are at higher risk for developing a host of medical conditions and vice versa. We will use genetic data to better understand the causes of these increased risks.\",\n\"This study will seek to examine the genetic overlap between addiction-related behaviors (e.g., alcohol, tobacco, illicit drug use) and other mental (e.g. antisocial behavior, depression and anxiety) and physical disorders (e.g., chronic pain conditions) at both the aggregate and single variant level. We will use methods that use either genotype-level data or summary statistics to determine the genetic relations between these phenotypes. For those methods that rely on summary statistic, we will use to meta-analytic approaches to combine datasets, including those outside of dbGaP. All data security protocols will be followed and because the focus will be on summary statistics, this should not present additional risks to study participants. We will use univariate and multivariate analytic association methods to identify loci that may have pleiotropic effects on these phenotypes. Finally, gene-based and pathway-based analyses will be conducted in an attempt to identify the biological mechanisms that contribute to the phenotypic correlations between these conditions. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Glass, Christopher \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Integration of GWAS disease risk variants with with epigenomic annotations derived from myeloid cells (monocytes, monocyte-derived macrophages, microglia)\",\n\"Dec22, 2020\",\n\"closed\",\n\"Most GWAS signals reside in noncoding regions of the genome and often these regions harbor gene regulatory regions. Often it is unclear what specific cell type and what specific genes are affected by these disease risk variants. Previously we showed, that microglia harbor a majority of Alzheimer disease risk variants in their distal gene regulatory regions. Here, we propose to provide a catalogue of active regulatory regions in cells of the myeloid cell lineage (monocytes, monocyte-derived macrophages, and microglia) and their associations with disease risk variants identified by GWAS. We aim to extend our previous efforts to provide a catalogue of gene regulatory elements specific for myeloid cells for various disorders, ranging from neurodegenerative, cardiovascular, metabolic, and autoimmune diseases. To this aim, we will use GWAS summary statistics and overlay these with our comprehensive next-generation sequencing datasets (PLAC-seq, RNA-seq, ATAC-seq, ChIP-seq). \",\n\"The objective of this project is to integrate disease risk variants obtained from GWAS studies involving neurological diseases and peripheral diseases (autoimmune, metabolic and cardiovascular diseases) with epigenomic annotations derived from myeloid cells (monocytes, monocyte-derived macrophages, microglia) as well as neutrophils and T cells.   Here, we propose to integrate GWAS signals from neurological, metabolic, cardiovascular and autoimmune diseases) with (epi-) genomic datasets (RNA-seq, ATAC-seq, H3K27ac ChIP-seq, and PLAC-seq) derived from immune cell types as mentioned above. This will enable us to link genetic variants residing in distal regulatory regions of myeloid cells to gene regulation and possibly to disease susceptibility. We will determine the enrichment of genetic variants associated with complex traits in cell type-specific regulatory regions, we implement a linkage disequilibrium score (LDSC) regression analysis of heritability.  Taken together, our approach will advance the understanding of the genetic bases of the myeloid cell lineages are their linkage to systemic diseases.  We will combine dbGaP, UKBiobank and GWAS Catalog datasets. There are currently no planned collaborations with researchers at other institutions.  Data obtain from dbGaP is used for general research and health/medical/biomedical purpose. Results generated by this study will be made available for the larger scientific community. All obtained summary results from the below mentioned studies is not used to identify individuals or groups and is used only for the evaluation of SNP associations with the phenotypes provided. Use of eMERGE Network Imputed GWAS for 41 phenotypes (phs000888.v1.p1) will not include the study of population origins or ancestry. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"GOATE, ALISON \",\n\"ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI\",\n\"Evaluating the Causal relationship between modifiable risk factors and Alzheimer\\u00e2\\u0080\\u0099s disease\",\n\"Dec20, 2018\",\n\"closed\",\n\"The goal of this project is to validate proposed causal relationships and identify new causal relationships between modifiable risk factors and the Alzheimer\\u00e2\\u0080\\u0099s endophonome using Mendelian Randomization analysis. \",\n\"Objectives: The overarching objective of this project is to enhance our understanding of the bidirectional causal relationships between lifestyle, health, psychosocial, biomarker, and disease outcomes on the Alzheimer\\u00e2\\u0080\\u0099s endophenome using Mendelian randomization analysis. Mendelian Randomization (MR) uses genetic variants as proxies for exposures to provide an estimate of the causal association between an intermediate exposure and an outcome. Identifying modifiable risk factors that are causally related to Alzheimer\\u00e2\\u0080\\u0099s disease can be used to inform lifestyle interventions.   Study design & Analysis Plan: Using summarized data from genome-wide association studies we will conduct Mendelian Randomization analysis between potentially modifiable risk factors and Alzheimer's disease. Exposures will consist of GWAS for health lifestyle psychosocial and disease traits. GWAS datasets for the Alzheimer\\u00e2\\u0080\\u0099s endophenome will consist of Late-Onset Alzheimer\\u00e2\\u0080\\u0099s disease, Alzheimer\\u00e2\\u0080\\u0099s age of onset (AAOS), amyloid beta, tau and phosphorylated tau levels in CSF, neuropathological burden of neuritic plaques and neurofibrillary tangles, and hippocampal volume. For each exposure, we will construct instrumental variables (IV) using genome-wide significant loci. Where SNPs are in linkage disequilibrium (LD), LD clumping will be performed by excluding SNPs that have an r2 > 0.1 with another variant with smaller p-value association within a 250kb window. For genetic variants that are not present in the outcome GWAS, we will use proxy SNPs (r2 < 0.8) where available. Causal estimates will be estimated using inverse-variance weighted fixed-effects meta-analysis. Sensetivity analyses will be conducted using weighted median, weighted mode, and MR-Egger regression analysis.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Golod, David \",\n\"GOLDFINCH BIOPHARMA, INC.\",\n\"Variant discovery in kidney disease\",\n\"Feb18, 2020\",\n\"expired\",\n\"Goldfinch Bio plans to utilize genomic variant data from large numbers of control subjects to confirm the potential pathogenicity of genomic variants and signatures found in kidney patient samples. These data will provide a means to identify dysregulated pathways in patients, new targets for therapeutic intervention, and characterize sub-sets of patients that are likely to respond to targeted molecular therapeutics.\",\n\"The objective of this study is to identify rare variants in kidney disease.  We are currently sequencing 2000 cases with nephrotic syndrome, as indicated by proteinuria.  Nephrotic syndrome can be caused by chronic disease such as hypertension and diabetes. To detect rare variants associated with nephrotic syndrome, we need whole genome sequences (WGS) from control populations to compare variant frequencies.  Because kidney disease is more prevalent in populations of African ancestry, we need controls of diverse ancestry.  We are also particularly interested in samples from population-based cohorts, to get the most accurate measures of population allele frequencies.  Thus, we will be combining all data sources into several analyses using a case-control design.  First, we will perform an unbiased common variant (> 0.5%) association study (i.e. GWAS) in each ancestry population, and then estimate overall effect sizes by meta-analyzing across all ancestry populations.  Second, we will search for rare variants (< 0.5%) with burden analyses, increasing power by filtering variants along several dimensions: class (disruptive, severity of missense), frequency, and within gene sets of interest.  Combining these datasets will not create any additional risks to the participants.  All raw data will be securely stored and will comply with the data use restrictions of each dataset.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Golod, David \",\n\"GOLDFINCH BIOPHARMA, INC.\",\n\"Genetic correlation between types of kidney diseases\",\n\"Apr08, 2020\",\n\"expired\",\n\"Goldfinch Bio plans to utilize genetic data from across kidney disease types to identify pathways that are shared vs distinct to disease development. These data will provide a means to better characterize how genetic variation influences kidney disease types, identify dysregulated pathways that are specific to disease types, and develop precision medicines for therapeutic interventions\",\n\"The objective of this study is to characterize shared vs distinct contributions of common variants to kidney disease types. We have assembled a large whole genome sequencing dataset of ~23,000 multi-ethnic populations to identify common and rare variants that associate with focal segmental glomerulosclerosis (FSGS), a rare form of kidney disease. To compare and contrast genetic relationship between FSGS and other kidney disease or functionally-related traits, we need GWAS summary statistics from population-based cohorts of kidney disease or function. We are requesting access to estimated glomerular filtration rate (eGFR) GWAS summary statistics from the Million Veteran Program (MVP) of ~300,000 multi-ethnic participants. Our objectives are: to measure genetic relatedness between: 1) FSGS and chronic kidney disease; 2) FSGS and diabetic kidney disease; and 3) FSGS and non-diabetic kidney disease. First, we will assess genetic relatedness within each population (European or African-American) by using population-specific summary statistics, and then across-ancestries by using meta-analysis results from the two populations. In accordance with the data use limitations for the dataset, we are not requesting any individual-level data. We do not anticipate any additional risk to the participants.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Gray, Joshua \",\n\"U.S.UNIFORMED SERVICES UNIV  HLTH SCIS\",\n\"Taking stock of genes associated with problematic alcohol use: Protein-protein interactions yield disease-relevant biological networks\",\n\"Jun29, 2020\",\n\"approved\",\n\"Psychiatric disorders, including alcohol use disorder, are influenced by hundreds or even thousands of genes. We seek to apply a new set of algorithms that will organize genetic contributions to problematic alcohol use into biological subgroups. This will help us better understand the genetic contributions to alcohol use disorder. Furthermore, we will look up medications that interact with these biological subgroups in hopes of identifying potential new treatments for alcohol use disorder.\",\n\"A recent challenge, Disease Module Identification Dialogue on Reverse Engineering and Assessment (DREAM), identified top-performing algorithms that most effectively identify disease relevant modules from protein-protein interaction (PPI) data. These modules are meaningful functional units that are comprised of core disease associated genes and peripheral genes, which impact disease relevant processes despite not yet being implicated in GWASs directly. Thus, in order to elucidate pathways implicated in problematic alcohol use (PAU) as well as sequelae, such as alcohol-related liver disease, we will apply a winning algorithm from the DREAM challenge to organize alcohol associated genes from the most recent PAU GWAS into interpretable PPI modules. We will additionally apply other emerging network-based approaches for understanding genes linked to PAU in their broader biological context. Furthermore, we will utilize drug-protein interaction databases (e.g., https://www.broadinstitute.org/drug-repurposing-hub) to look for approved drugs that interact with proteins implicated by these modules. The overarching goal is to understand more about biological contributors to PAU and to potentially identify novel treatment targets and/or drugs for repurposing.  This will not be combined with datasets outside of dbGaP. We may present these results alongside those from a recent GSCAN study, but these summary statistics are publicly available (https://www.nature.com/articles/s41588-018-0307-5). We will only be downloading the summary statistics (i.e., SNPs and associated p values) and thus do not anticipate any additional risk to participants.  We are focusing on the genetic basis of problematic alcohol use (PAU) so the outcome is the same as the original project.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Grotzinger, Andrew \",\n\"UNIVERSITY OF COLORADO\",\n\"Psychiatric Genomics Consortium Cross-Disorder Group v3 (PGC-CDG3): Probing multivariate genetic architecture across psychiatric traits\",\n\"Dec13, 2021\",\n\"approved\",\n\"Psychiatric disorders overlap substantially at the genetic level. Cross-disorder genomic projects that aim to better characterize this genetic overlap then have the potential to produce key insights into the underpinnings of rampant psychiatric comorbidity. To this end, the current proposal aims to analyze the most up-to-date summary statistics across 14 major psychiatric disorders using four cutting-edge genomic methods: Genomic SEM, CC-GWAS, MiXER, and LAVA. These methods can be used to examine genetic overlap in aggregate across the genome, for specific classes of genes (e.g., genes that are expressed early in life), and at the level of individual genetic variants. By applying different methodoligcal approaches we aim to identify whether a convergent narrative emerges about the patterns of genetic overlap across disorders at varying levels of analysis. With respect to the number of disorders and utilized methods, this project aims to perform the comprehensive, cross-disorder psychiatric analysis to-date. We are specifically requesting permission to use the summary statistics that include the MVP cohort for: alcohol use disorder, opioid use disorder, anxiety disorder, major depressive disorder, and post-traumatic stress disorder.  \",\n\"Building on the work of the PGC Cross-Disorder Working Group (PGC CDG2) in clarifying the genetic structure of psychopathology, we now aim to leverage multiple, multivariate methods to better define the shared and disorder-specific genomic underpinnings across 14 major psychiatric disorders. At the genome-wide level, we will specifically apply MiXER, Genomic SEM, and CC-GWAS. Using Genomic SEM, we will produce a factor model based on the genetic correlations across the 14 disorders with the aim of identifying subgroups of disorders that more strongly cluster together. MiXER then takes an alternative approach to quantify polygenic overlap by estimating the total number of shared and trait-specific causal variants, while CC-GWAS calculates the overall \\u00e2\\u0080\\u009cgenetic distance\\u00e2\\u0080\\u009d across two traits based on differences in allele frequencies for cases. Collectively, these results will help us determine whether a convergent narrative emerges about the overall pattern of genetic overlap across these disorders.  At the functional level, we will apply two new methods: Stratified Genomic SEM and LAVA. Stratified Genomic SEM builds upon the findings from Genomic SEM to examine multivariate enrichment at the level of the psychiatric factors. LAVA then examines local genetic correlations at the level of individual loci. In conjunction, these two methods allow us to identify specific categories of genes and loci that drive genetic overlap across disorders. Finally, at the SNP-level of analysis, we will use CC-GWAS and multivariate GWAS in Genomic SEM to identify SNPs that are both highly pleiotropic and disorder-specific. The cumulative set of findings will offer critical insights into the multivariate architecture of psychiatric disorders utilizing cutting-edge methods at three levels of analysis (genome-wide -> functional -> SNP). Indeed, the proposed project reflects the most comprehensive cross-disorder, psychiatric analysis to-date. We are specifically requesting permission to use the summary statistics that include the MVP cohort for: alcohol use disorder, opioid use disorder, anxiety disorder, major depressive disorder, and post-traumatic stress disorder.   \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Gui, Hongsheng \",\n\"HENRY FORD HEALTH SYSTEM\",\n\"Investigation of substance use on risk of suicidal attempt: a Mendelian randomization study\",\n\"Jul28, 2021\",\n\"approved\",\n\"Is there any connection between substance addiction and suicidal attempt? Can we identify patients with substance use disorder so to prevent their suicide completion? We seek to use genomic data published by Million Veteran Project to answer this two questions. Different from environment risk factors, genomics data is inherent and stable across lifespan. Therefore insights gained from secondary analyses of genomics data will provide us an unbiased and powerful approach to delineate the underlying relationship between substance use and suicide.\",\n\"The project aims to study the correlation and causation relationship between substance uses and suicidal attempts. The primary outcome phenotype of interest is suicidal attempt, and the exposure phenotype include opioid use disorder, cannabis use disorder, and alcohol use disorder reported in Million Veteran Project (MVP). Genetic correlation, polygenic risk score, and 2-sample Mendelian randomization will be used to investigate the underlying causal links between outcome and exposure phenotype. Instrumental variables will be selected from summary statistics of genome-wide association studies (GWAS). To overcome health disparity across populations, we will investigate both European descent samples and African descent samples. Other cross-phenotype analyses include multi-trait analysis of GWAS, genomic structural equation modeling, and case-case genome-wide association study will also be used to identify shared or distinct genes and pathways. All analyses will be based on summary statistics instead of using genotype data, and this will help to protect the confidentiality of samples in MVP. This study will better delineate the relationship between substance use disorder and suicidal attempt through genomics perspective, and will help to scrutinize potential high-risk suicidal individuals for early intervention and management.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Guo, Shicheng \",\n\"JOHNSON/JOHNSON/PHARM/RES/ DEVELOPMENT\",\n\"GWAS to identify disease genes, drug targets and novel therapeutic approaches\",\n\"Nov13, 2020\",\n\"approved\",\n\"This study plans to identify genetic differences between people with and without certain disease or traits. Identification of genetic differences associated with disease will help us understand the biology mechanism and help us to develop new drugs for disease treatment. Meanwhile, disease risk prediction can be performed to show the risk that people will be affected by this disease. Finally, genetic difference also explains the different response to drugs which will help pharmacist to select the best choice of the drug to decease the side-effects of the drugs. \",\n\"Large genetic cohorts provide an opportunity to identify disease susceptibility genes and novel drug target for the disease therapeutics, especially meta-analysis between different studies and populations could provide increased power to identify novel susceptibility genes and drug targets which have low allele frequency. The aim of this project is to identify potential drug targets with eMERGE and other population genetics data including UK-Biobank, FinnGen and Japanese biobank. We will apply multiple computational approach (SAIGE, SAIGE-GENE, Robust-SKAT, Robust-SKAT-O, GENESIS, REGENIE, RVTESTS) to identify disease/traits associated genetic variants and to reveal novel disease mechanism so that novel drug target could be identified. Comparison and evaluation the performance for these different computational approaches will be performed in which the difference of the positive associations and computational resource requirement will be reported. In this study, GWAS study will be performance to all the 41 phenotypes in eMERGE, UKB and FinnGen with uniform statistical method and computational pipelines to avoid non-necessary method bias. We will also apply tissue-specific gene expression, eQTL, sQTL to infer the potential tissue-of-origin for the disease. Multiple genetic fine-mapping approach will be conducted to compare the performance and multiple multiple-omics (GTEx, GEO and blueprint) based network approach will be tried to identify disease related molecule modules and tissue-specific regulatory circuits. Our research aims are totally same with primary study aims of eMERGE, UKB and FinnGen and we will not have any external collaborators in this study.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Guo, Yan \",\n\"XI'AN JIAOTONG UNIVERSITY\",\n\"Genomic analysis for human complex diseases\",\n\"Jan13, 2020\",\n\"rejected\",\n\"Many human complex diseases/traits have strong genetic determination. The aim of this project is to search new genetic factors and better understand genetic architectures for common complex diseases/traits through genomic analyses or related methodologies. We are seeking to the access to multiple datasets in dbGAP that contain metabolic disorders, hematologic malignant tumors, cardiovascular diseases, and their related phenotypes. Our results may be helpful in uncovering genetic basis for human complex diseases/traits, and in prevention, diagnosis, and therapy of diseases. \",\n\"Many human complex diseases/traits are highly heritable, but the underlying genes are largely unknown. The aim of the present study is to identify susceptibility loci and improve understanding of the genetic architectures of complex diseases/traits by using available genetic data from multiple datasets in dbGAP through our comprehensive statistical analyses and study design approaches. We have extensive experience in analyzing GWAS and sequencing data. In this application, we are requesting the access to the dbGAP datasets which contain at least one of the following phenotypes of our interest, including metabolic disorders (e.g., osteoporosis, diabetes, and obesity), hematologic malignant tumors, cardiovascular diseases/traits, lung function, lipid traits, psychiatric traits, and their related anthropometric phenotypes and biochemical measures. These data will be used separately and/or in combination by integrating various data types to achieve our goals.  For studies that allow only disease specific research, the data will only be used to develop models specific to the disease and investigate the genetic basis of the disease. The use of the data is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry. The specific objectives are to 1) identify novel loci for the complex traits/diseases of our interest; 2) perform joint analyses for multiple phenotypes to identify potential pleiotropic genes; 3) identify novel gene-gene and gene-environment interactions; 4) conduct gene network-based GWAS analyses that consider multiple genes simultaneously; 5) evaluate related methodologies for analyzing complex traits/diseases. We acknowledge the Use Restrictions for the requested data sets, and will abide by them and conducting research consistent with them. These data will be used only for research purposes. They will not be used to determine the individual identity of any person or of their relationship to another person. We will publish and broadly share any findings from our study with the scientific community and peer-reviewed journals.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"GUSEV, ALEXANDER \",\n\"DANA-FARBER CANCER INST\",\n\"Investigating the relationship between germline and somatic variation in common cancers\",\n\"Dec30, 2019\",\n\"approved\",\n\"The mechanisms of cancer are driven by germline (inherited) variants as well as somatic variants that occur in the cell. Though specific examples exist for both types of variation acting as a cancer driver, the genome-wide interplay between these two features remains an important area of research. The goal of this project is to investigate the impact of germline genetic variation on patterns of somatic variation and cellular activity such as gene expression and methylation measured in tumor/normal cells. The project is divided into three main aims. First, we will identify specific germline variants that are associated (and likely driving) cellular activity. Second, we will assess whether these variants can explain the loci identified in large-scale studies of cancer risk. Third, we will assess whether cancer risk variants have a significant effect on patterns of somatic changes.\",\n\"***We will NOT analyze genetic ancestry in studies that do not allow analysis of population origins and ancestry and in accordance with the study data use limitations*** ***For studies restricted to specific phenotypes, we will only analyze GWAS data from those phenotypes in accordance with the study data use limitations.***  It is well understood that many cancers have a substantial genetic component, which can effect risk, response to exposure, cancer etiology, and even molecular mechanisms. The goal of this project is to investigate the impact of germline genetic variation on patterns of somatic variation and tumor-specific cellular activity. We intend to broadly share your results with the scientific community by publishing our findings.  1. We will assess the impact of germline variants on molecular phenotypes (expression, methylation, etc) in tumor and normal individuals. We will apply traditional QTL and allelic imbalance methods as well as emerging methods. We will develop new methods to identify variants that have an interaction with tumor/normal status or tumor sub-type.  Separately, we will quantify and partition the genome-wide heritability and genetic correlation of these molecular features, within and across different cancers.  2. We will investigate the mechanistic impact of features identified in Aim 1 on GWAS data from cancer and related traits (where allowed by the study). We will use the genome-wide QTL signal as an annotation to assess enrichment of GWAS associations and heritability. We will use novel statistical methodology to identify QTLs whose molecular features are likely to be mediating the GWAS association.  3. We will investigate the relationship between somatic variation and polygenic burden scores from cancer GWAS studies (for example, between extreme high and low risk individuals). Specifically, we will look for association between genetic risk score and the distribution of somatic mutations/alterations and tumor-specific expression/methylation in each sample. We will investigate if patterns of somatic variation lead to functional enrichment of GWAS association statistics and heritability.  IMPORTANT: None of the analyses require merging controlled data with external data and we will never do so. For studies restricted to specific phenotypes, we will only analyze GWAS data from those phenotypes in accordance with the study data use limitations:  * For study \\\"VA Million Veteran Program (MVP) (phs001672.v3.p1)\\\" data will only be used for health/medical/biomedical purposes and will not be used to study population origins or ancestry. * For study \\\"Functional Significance of Prostate Cancer risk-SNPs (phs000985.v1.p1)\\\" we will only use corresponding data to study prostate cancer and cross-reference against prostate cancer GWAS data. * For study \\\"Stockholm-Tartu Atherosclerosis Reverse Network Engineering Task (STARNET) (phs001203.v1.p1)\\\" data will only be used for health/medical/biomedical purposes and will not be used to study population origins or ancestry. * For study \\\"Exome recapture and sequencing of prospectively characterized clinical specimens from cancer patients (phs001783.v1.p1)\\\" data will only be used to study cancer and for methods development.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"HAN, SHIZHONG \",\n\"LIEBER INSTITUTE, INC.\",\n\"Mapping PTSD GWAS risk loci to human brain genomic phenotypes\",\n\"Dec30, 2019\",\n\"rejected\",\n\"We are interested in mapping GWAS risk loci to nearby gene function in human postmortem brain genomics datasets. \",\n\"We are interested in mapping GWAS risk loci to molecular function in the human brain by performing integrative analyses of GWAS summary statistics and human postmortem tissue RNA sequencing (RNA-seq) and whole genome bisulfite sequencing (WGBS). We will take individual variants identified in GWAS for PTSD, and examine how the genotypes of these SNPs associate with nearby DNA methylation and RNA measurements from multiple brain regions in patients with PTSD and neurotypical controls.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"HAN, SHIZHONG \",\n\"JOHNS HOPKINS UNIVERSITY\",\n\"Integrative analysis of multi-omics datasets for alcoholism\",\n\"Dec30, 2019\",\n\"closed\",\n\"Alcoholism has a genetic basis, but identification specific genes or risk variants contributing to the disorder has been a challenging task. Over the last ten years, the research field has accumulated large genetic and genomic datasets that are publicly available, such as those from dbGaP, which provides an unprecedented opportunity to risk gene discovery. We will apply for the access to these datasets and develop advanced statistical models to re-analyze these datasets, with the goal of identifying risk genes underlying alcoholism.\",\n\"Our goal is to identify risk genes underlying alcoholism. We request GWAS summary statistics for alcohol-related traits from MVP. We will perform gene, pathway and network-based analysis using various existing statistical genetic methods and computational tools. We will also develop novel network-based approaches that integrate gene expression information in the brain tissues. We agree to abide by the NIH Code of Conduct for Data Use and the use restrictions associated with each GWAS dataset.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Hao, Xingjie \",\n\"HUAZHONG UNIVERSITY/SCIENCE/TECHNOLOGY\",\n\"Fine-mapping, genetic correlations and potential shared mechanisms between complex diseases in different populations\",\n\"Jan03, 2022\",\n\"approved\",\n\"Complex diseases were defined as diseases which were influenced by a combination of multiple genes and environmental factors, including asthma, diabetes, epilepsy, hypertension, manic depression, schizophrenia and so on. Single nucleotide polymorphisms (SNPs) are the most common type of genetic variants among people, some of which may affect diseases in some ways. We aim to reveal the pleiotropic SNPs that are associated independently with two or more distinct diseases and explore the common mechanisms for the SNPs affecting the diseases in different populations. In different populations, we will determine the underlying causal variants for each complex diseases, estimate genetic correlations between complex diseases that implying the effects of the tested genetic variants on the two tested complex diseases correlated or not and identify common disease-relevant cell-types in two or more complex diseases which suggesting potential shared mechanisms.\",\n\"There are potential shared genetic structures and pathogenic mechanisms between the complex diseases, which may be different across populations. The overall objective of this study is to reveal the pleiotropic SNPs that are associated independently with two or more distinct complex diseases and explore the common pathways and mechanisms for the SNPs affecting the complex diseases in different populations. With the GWAS summary-level data in different populations of Million Veteran Program (MVP), we plan to (1) develop and utilize new methods to perform fine-mapping for complex diseases leveraging linkage disequilibrium (LD) score in different populations, (2) estimate the genetic correlations between complex diseases in different populations using linkage disequilibrium score regression (LDSC) software, which may provide reference for trans-ancestries polygenic risk score (PRS) and (3) identify disease-relevant cell-types in different populations by conducting stratified LDSC regression to calculate partitioned heritability leveraging the 66 cell-type specific GenoSkyline-Plus annotation tracks, so that we may find potential shared mechanisms and the possible heterogeneity of genetic regulation across populations. We do not plan to combine the dataset from MVP with others. The GWAS summary statistics will not be passed to any third party. Any publication relating to this work would include appropriate attribution/credit.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Hardy, Dale \",\nNaN,\n\"A Multi-Omics risk score analysis for obesity by ancestry\",\n\"Sep21, 2021\",\n\"approved\",\n\"Obesity is one of the most common heritable conditions that is an important predictor of health status and chronic diseases. Studies have reported that African Americans have diets that increase the risk of obesity and chronic disease conditions. We will integrate omics technologies and investigate by pooled meta-analysis, associations of an ancestry-and sex-specific Multi-Omics Risk Score on obesity whether high levels of physical activity can mitigate these risks.\",\n\"We will analyze our data by both cross-sectional and prospective longitudinal study designs to measure the effect of an integrated genomic-biological Multi-Omics Risk Score (MORS) from different Omics technologies (whole genome sequencing, transcriptomics, proteomics, methylomics, and metabolomics) on obesity (BMI, waist circumference, waist-to-hip ratio, and waist-to-height ratio) adjusted for covariates and the PCA score in African American and European American males and females. In additional models we will model the additional effect due to interaction and mediation of MORS with macronutrient intake (total fat, polyunsaturated fat, monounsaturated fat, saturated fat, animal fat, protein, carbohydrates, sucrose, fiber, and total calories) on obesity. To evaluate the variation in risk, we will use cluster-specific models for correlated data to capture the variance-covariance between participants. We will use two cluster-specific approaches: random-effects models and random slope models. In variance-components models using the random intercept approach, the random intercept represents personal characteristics or latent factors which are not included in the model. In other models, we will add random slopes to the random intercept models to allow the covariates to vary between clusters. We will use the likelihood ratio test to test if there in a difference in models with random slope vs. without random slope. In summary, in cross-sectional model, education levels will be considered as the random effect for latent values applies to participants with a common but unobserved value of the random effect. In longitudinal models, we will compute the cumulative effect the participants\\u00e2\\u0080\\u0099 ID considered as the random effect. We will broadly share our findings with the scientific community through publications, oral and poster presentations, guest lectures, and other means as presented.   Analytical methods and statistical analysis conducted for each specific aim In Aims 1A, We will integrate ancestry-specific Multi-Omics technologies within common functional pathways and investigate whether top clusters (most significant p value) are associated with obesity within sex groups.  Random effects cross-sectional Linear model (BMI/ body weight)= \\u00c3\\u009f1 MORS + \\u00c3\\u009f2covariates + \\u00c3\\u009f3PCApopulation structure + random intercept (by ancestry & sex)   Random effects longitudinal multivariable model (Obesityij)= \\u00c3\\u009f1MORS + \\u00c3\\u009f2covariatesij + \\u00c3\\u009f3PCApopulation structure + random interceptij (by ancestry & sex)    Specific Aim 1B: Examine molecular clustering genetic pathways by ancestry and sex.  We will examine pathways in this study by racial ancestries using ingenuity pathway analysis.  Specific Aim 2A: Determine interaction and mediation of common signature MORS clusters with macronutrient intake to further influence risks within sex-specific analysis by racial ancestry and whether high physical activity can mitigate these risks. For Aim 2A, we will test whether there are additive interactions (biological interactions) between the MORSs and macronutrients on the additive risk scale. For cross-sectional and longitudinal mediation models, we will determine if macronutrient intake acts as a mediator (intervening variable). We will evaluate the overall fit of the model using the model fit statistics. We will use the Bonferroni correction for false-positive tests for multiple testing. In these models we will examine interaction and mediation of MORS with macronutrients and whether physical activity can decrease the increased risks. Our multivariable statistical models are below. Ij denotes repeated measures over multiple timepoints.  Longitudinal interaction multivariable model (Obesity)= \\u00c3\\u009f1MORS + \\u00c3\\u009f2macronutrientij + \\u00c3\\u009f3 MORS*macronutrient + \\u00c3\\u009f4covariatesij + \\u00c3\\u009f5PCApopulation structure  + \\u00c3\\u009f6random effect (by ancestry & sex)   Specific Aim 2B: Determine dose-response risks of common signature MORS clusters and from interaction across common pathways.  We will test risks for dose-response of MORS and dose-response interaction of MORS by macronutrient on obesity. Our longitudinal Models below. Ij denotes repeated measures over multiple timepoints.   Longitudinal dose-response model Risk(Dose response) obesityij = \\u00c3\\u009f1MORS (dose) + \\u00c3\\u009f2macronutrientij + \\u00c3\\u009f3MORS (dose)*macronutrientij  + \\u00c3\\u009f4covariatesij + \\u00c3\\u009f5PCApopulation structure + random effectij + random slope ij (by ancestry & sex)  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Hayek, Salim \",\n\"UNIVERSITY OF MICHIGAN AT ANN ARBOR\",\n\"Role of SuPAR in the Intersection Between Cardiovascular and Kidney Disease\",\n\"Jan17, 2021\",\n\"approved\",\n\"Chronic kidney disease affects 15% of people in the US, and those affected are more prone to heart disease. We recently found that patients with increased levels of a certain biomarker - a substance that is found in body fluids, in this case the blood of patients - known as suPAR, are more likely to have a poor prognosis than those with lower levels of the suPAR biomarker.  The goal of this project is to see whether suPAR itself contributes to heart disease and declining kidney function. We will use the suPAR levels found in patients who took part in the Multi-Ethnic Study of Atherosclerosis (MESA) to ensure the correlation with other early markers of heart disease and the progression to worsening heart disease.  Then, to see if suPAR could be a cause of these diseases, we will use a genome-wide association study of suPAR levels in MESA to see if there are genes that are different in people with high suPAR levels. We will use a larger group of people (through the UK Biobank and Million Veterans Program) to see if those genes are connected to heart disease or chronic kidney disease.  \",\n\"We aim to assess potential causality between suPAR and both Chronic Kidney Disease (CKD) and cardiovascular disease (CVD) using Mendelian randomization. We hypothesize patients with CVD or CKD will harbor genetic variants associated with high suPAR levels, suggesting suPAR is an effector of CKD and CVD. We have previously performed a genome-wide association study (GWAS) of suPAR in 3,244 healthy volunteers and identified genetic variants in the PLAU and PLAUR genes associated with suPAR levels. We will (a) validate our findings in a GWAS of MESA, followed by (b) a meta-analysis using all cohorts (including the healthy participants in the Trinity Student Study (TSS) and the Genes and Blood-Clotting (GABC) study). Lastly (c) we will perform a Mendelian randomization study in the UK Biobank with data for over 500,000 volunteers, to support suPAR as a cause of CKD and CVD.   People with CKD suffer a disproportionately high burden of CVD for reasons that are poorly understood. Inflammation represents the major pathophysiological process common to both atherosclerosis and CKD, and recently, a novel pathway of inflammation was uncovered linking both CVD and CKD to bone marrow myeloid cells producing a circulating signaling molecule: soluble urokinase plasminogen activator receptor (suPAR). suPAR levels predict adverse outcomes in both diseases, and outperform well-established markers of CVD risk, such as coronary calcium, C-reactive protein, high sensitivity troponin I and B-type natriuretic peptide. Pharmacological inhibition of suPAR in experimental models prevented kidney injury. These compelling data reveal the potential for suPAR not only as an excellent biomarker of risk, but also as a promising therapeutic target.   The overall goal of this project is to elucidate suPAR\\u00e2\\u0080\\u0099s potential causal role in the progression of atherosclerosis and its link to decline in kidney function through epidemiological insights. We will achieve this goal by leveraging three of the most significant contributions to cardiovascular science: the landmark Multi-Ethnic Study of Atherosclerosis (MESA); an NIH-funded prospective cohort in which enrollees underwent serial measurements of subclinical markers of atherosclerosis, the JUPITER trial, in which participants with high C-reactive protein levels were randomized to statin or placebo, and the UK Biobank, a data repository of over 500,000 volunteers. SuPAR levels will be measured in ~5,620 participants of MESA to determine whether levels correlate with early markers of CVD and predicts their progression independently of a decline in kidney function. To assess potential causality, a gene-wide association study of suPAR levels in MESA, followed by a Mendelian randomization analysis in the UK Biobank dataset will connect genetic determinants of suPAR levels to CVD and CKD. Lastly, to establish whether suPAR is a modifiable risk factor for CVD, levels will be measured serially in participants of the JUPITER trial randomized to rosuvastatin (n=200) or placebo (n=200), and the change in suPAR will be compared between groups. Whether the benefits of statins are dependent on suPAR levels will be assessed in the MESA cohort by comparing the survival of participants started on statins across suPAR quartiles, and determining whether suPAR is a modifier of the association between statins and outcomes. The proposed research has the potential to address the unmet need of close to 50 million people with CKD in the United States alone, identifying sorely needed therapeutic targets and management strategies in a field that has long been stagnant. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Heilig, Markus \",\n\"LINKOPING UNIVERSITY\",\n\"Polygenic risk for substance abuse as a predictor of substance abuse in individuals with versus without documented childhood maltreatment\",\n\"Jul28, 2021\",\n\"approved\",\n\"Many people who experience childhood maltreatment go on to develop issues with substance abuse, like alcohol or opioid addiction. Still, others who experience this sort of childhood trauma do not go on to have issues with substance abuse. One reason for this difference might be that those who go on to develop substance abuse have a higher genetic susceptibility for this disorder, and the stress of childhood maltreatment interacts with this genotype in a pathological way. In the research proposed here, we plan to test this hypothesis directly by looking at genetic risk for substance abuse in persons who have or have not experienced childhood maltreatment and then gone on to either have substance abuse issues or not.\",\n\"Three recent genome wide associate studies of opioid use disorder (https://jamanetwork.com/journals/jamapsychiatry/fullarticle/2766708), tobacco smoking (https://www.nature.com/articles/s41467-020-18489-3), and alcohol use disorder (https://www.nature.com/articles/s41593-020-0643-5) report summary statistics stored in dbGaP that we propose using for assessing polygenic risk (PRS) for substance abuse, broadly construed. We will assess polygenic risk in four groups: 1) individuals with no history of childhood maltreatment (CM) and substance abuse (SA); 2) persons with CM but no SA history; 3) individuals with SA but not CM history; and 4) persons with both SA and CM histories. The primary analyses of interest will compare PRS in SA+ versus SA- groups as well as in CM+&SA+ versus CM+&SA- subgroups. The summary statistics made available from dbGaP will be combined with the following data from our site: 1) individual genotype data derived from DNA collected onsite; 2) substance abuse histories determined from structured clinical interview; and 3) childhood maltreatment histories determined from accounts collected by state-sponsored agencies intervening at the time of abuse.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"HETTEMA, JOHN \",\n\"TEXAS A&amp;M UNIVERSITY HEALTH SCIENCE CTR\",\n\"Identification of Susceptibility Genes for Anxiety and Related Disorders\",\n\"Jul29, 2021\",\n\"approved\",\n\"Anxiety and depression tend to co-occur over an individual\\u00e2\\u0080\\u0099s lifetime and in their family members. Existing studies suggest this may be due to shared genetic risk factors. This study will try to identify genes that increase risk across these conditions by integrating data from dbGaP datasets with analyses being conducted in other studies. \",\n\"Anxiety disorders to co-occur with each other and with related conditions like depression. Genetic epidemiological studies suggest this may be due to shared genetic risk factors. This study aims to identify genes that increase risk across these conditions (shared and disorder-specific) using genome-wide association study (GWAS) approaches. We will analyze individual anxiety disorder phenotypes (generalized anxiety, panic, phobias, PTSD, OCD) in association with genetic variants as well as combine these into an AnyAnxiety disorder status. Where available, we will include other psychiatric phenotypes like mood disorders, eating disorders, substance use, and personality disorders. We will also apply factor analytic methods to estimate dimensional severity scores as phenotypes. We will integrate results from dbGaP datasets with those from other GWAS via meta-analysis. There will be no overlap between dbGaP samples and external datasets, so no additional risk of identifying subjects will occur. When available, we will also incorporate non-clinical phenotypes like neuroimaging data into our analyses. Note that this is a continuation/extension of our prior project (Identification of Candidate Genes for Internalizing Phenotypes) after the PI changed institutions.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Holm, Hilma \",\n\"DECODE GENETICS, EHF\",\n\"Genome wide meta-analysis of cardiometabolic traits\",\n\"Feb21, 2023\",\n\"approved\",\n\"The goal of the study is to identify risk variants in the genome that associate with the risk of the cardiometabolic traits; peripheral artery disease, abdominal aortic aneurysm, venous thromboembolism, type 2 diabetes, blood pressure and blood lipids. Several studies have screened the entire genome of a large number of cases with these particular traits in order to look for association between millions of genetic variants and the disease/trait outcome (so called Genome Wide Association study). In order to gain more power to identify genetic risk variants we intend to combine the largest sample sets available for these traits and then perform our analyses.\",\n\"The goal of our research is to perform genetic discovery analyses of the following cardiometabolic traits; peripheral artery disease, abdominal aortic aneurysm, venous thromboembolism, type 2 diabetes, blood pressure and blood lipids. The requested data from dbGAP, i.e. summary level Genome wide Association (GWA) data from the MVP trans-ancestry meta-analyses of these cardiometabolic traits, will be combined with GWAS datasets from Iceland, UKBiobank, Denmark, Finngen, Intermountain USA and other available datasets.  We will perform association analyses, Mendelian randomization studies and polygenic risk score approaches. The data requested will include no personal identifiers only summary level data so this study will not create any additional risks to the participants.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Im, Kate \",\n\"GENO TECHNOLOGIES, INC.\",\n\"Investigating heritability of disease risk and methods development in family-based statistical genetics. \",\n\"Jul20, 2020\",\n\"approved\",\n\"To understand the genetic underpinning of diseases and traits, we will use this data to improve algorithms to identify genomic differences between individuals and understand how those differences can predispose different individuals to diseases or specific traits. \",\n\"We aim  to better understand the genetic underpinning of diseases and traits with an aim to prevent diseases and empower patients and their families. We use a combination of approaches including investigating variants in genes associated with diseases as well as constructing genome-wide risk scores for more complex diseases. We are particularly interested in utilizing data for conditions that are of interest in the setting of family planning. We will analyze these datasets in accordance with the data use statements for each study. As an example, our team is exploring the impact of multiple autoimmune conditions;  in this context we will not combine the data for one consent group (e.g. vitiligo only) with data for another consent group if forbidden by the research use statement. This data will be used for methods development which includes investigating the quality of genotype imputation methods, predicting parental haplotypes and their transmission, and calculating genomic risk scores for comparison amongst siblings. No genetic information will be released to any third-party outside of the company for any of these individuals. Results of this project will be presented at scientific meetings and published in journals.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Ingelsson, Erik \",\n\"GLAXOSMITHKLINE\",\n\"Discovery & Development of New Medicines & Vaccines using GWAS Summary Statistics\",\n\"Jul12, 2021\",\n\"approved\",\n\"Genome-wide association studies (GWAS) identify genetic variants associated with complex diseases which can be useful in identifying new therapeutics and optimizing the development of existing therapeutics.  We will combine results from multiple GWAS studies to create well-powered studies to conduct this research.  Scientists with expertise in human genetics will interpret the results.  \",\n\"We will use the GWAS summary statistics for the purposes of research focused on identifying genes associated with patient outcomes which can be used to discover or optimize development of new medicines and vaccines.  This research is conducted by performing meta-analyses of multiple genetic association studies to create a well-powered study of the trait of interest and colocalization with expression QTL results.  These GWAS summary statistics will be meta-analyzed with GWAS summary statistics derived from analysis of UK Biobank, FinnGen, 23andMe and other publicly available data sets and colocalization with expression QTL summary statistics including Blueprint and GTEX and other publicly available datasets.    This research would be spread across the GSK group of companies and may include use in research collaborations, but the GWAS summary statistics will not be passed to any third party. Any publication relating to this work would include appropriate attribution/credit.  We also plan to use the GWAS summary statistics to evaluate different strategies for fine and gene mapping, e.g. via colocalization to expression, splicing and other QTL datasets.  Erik Ingelsson is the SVP, Human Genetics which is responsible for Human Genetics analysis for GSK R&D organization.  Due to the corporate nature of GSK, Erik\\u00e2\\u0080\\u0099s team is distributed across the UK and US and employed by multiple GSK entities. He has direct line management and accountability for the work that these employees perform.    Chuck Smith is a VP, Data Platform and Solutions, in our Tech Research and Development department which is the IT department within GSK.  He is an IT director and has the authority and capacity to deliver the platform, processes and staff all relevant for the requested data.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Ingelsson, Erik \",\n\"STANFORD UNIVERSITY\",\n\"Molecular studies of cardiovascular disease, type 2 diabetes and related conditions\",\n\"May15, 2019\",\n\"closed\",\n\"We are broadly interested in what leads to cardiovascular diseases, type 2 diabetes and related conditions. We combine different methods, including large-scale studies in human populations, with studies in model systems including cell cultures and murine models. Our research aims at increasing understanding of cardiovascular diseases, type 2 diabetes and related conditions; to personalize treatment of patients with risk of these conditions; and to find new drugs to treat them. We ask permission to use data from dbGAP for our projects, which include discovery of novel genetic determinants and studies of causal factors of these diseases; prediction of new uses and side effects of drugs, with a focus on cardiovascular diseases, type 2 diabetes and related conditions using genetic methods; prioritization of genes for further studies in model systems by combination of genetic and gene expression data; and development of new methods to analyze molecular data.\",\n\"Our research program is broadly interested in the etiology of cardiovascular diseases (CVD), type 2 diabetes (T2D) and related conditions (such as obesity, dyslipidemia, insulin resistance) and risk factors (such as lifestyle factors and other environmental exposures). Our research is translational and interdisciplinary, combining large-scale population-based research - including genomic, metabolomic, transcriptomic, epigenomic and proteomic studies in large population-based studies - with functional work in model systems to reach new insights into the pathophysiology of cardiovascular disease and related conditions, identification of new biomarkers for improved risk prediction, and discovery of novel targets for drug development. Dr. Ingelsson has had a leading role in many of the large efforts identifying new loci associated with cardiovascular and metabolic traits, and has extensive experience from research on biomarkers and -omics methods, including development and application of prediction metrics and Mendelian randomization. Since 2014, he has also built a team working with characterization of loci established in GWAS using gene editing methods in model systems. The present proposal has several aims: (1) Discovery and characterization of molecular determinants of CVD, T2D and related conditions; (2) Studies of risk factors for these conditions using both traditional observational analyses and Mendelian randomization approaches; (3) Studies of loss-of-function variation in drug target genes to predict (a) potential side effects (using phenome-wide approaches) of drugs aimed at treating CVD, T2D or related traits; or (b) potential for drug repositioning to CVD- or T2D-indications of drugs used for other diseases; (4) Prioritization of candidate genes for functional studies using eQTL analyses; and (5) Development and application of novel methods to analyze molecular data in relation to CVD, T2D or related traits. We will publish or otherwise broadly share any findings from our studies arising from use of this data with the scientific community. We will honor all embargo dates associated with any of the publicly available datasets.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Irvin, Marguerite \",\n\"UNIVERSITY OF ALABAMA AT BIRMINGHAM\",\n\"Polygenic risk scores for chronic kidney disease in African Americans\",\n\"Jun10, 2022\",\n\"approved\",\n\"We are using the summary level genetic data to inform personalized risk for kidney disease in African Americans.  \",\n\"Genetic background contributes to risk for developing chronic kidney disease (CKD). The objective of this study is to develop polygenic risk scores (PRS) for CKD and explore their capacity to predict incident kidney failure in cohorts of African Americans (AAs). We will use summary statistics from a genome-wide association study (GWAS) of estimated glomerular filtration rate (eGFR) conducted by Hellwege et al. within the Million Veteran Program (MVP) to train PRS for CKD in ~1,900 individuals with whole genome sequenced (WGS) data in the Hypertension Genetic Epidemiology Network (HyperGEN). CKD status will be defined as having a baseline eGFR <60 ml/min/1.73m2, and incident kidney failure will be defined as having a follow-up eGFR <15 ml/min/1.73m2 or dialysis or kidney transplantation among individuals without kidney failure at baseline. We will use the PRScs software to train the PRS in HyperGEN, and we will develop risk models with additional adjustment for age, sex, and comorbidities associated with kidney injury (e.g. hypertension, diabetes). We will then evaluate the predictive performance of these risk models (e.g. c statistic), and the best performing risk algorithm (PRS + covariates) will be selected for validation in AAs with WGS data from the Jackson Heart Study (JHS, approved) and Women\\u00e2\\u0080\\u0099s Health Initiative (proposal under review). Once these analyses are completed, the validated risk model will be tested for its capacity to predict incident kidney failure in the REGARDS, HyperGEN, and JHS cohorts.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Jaffe, Andrew \",\n\"BLACKTHORN THERAPEUTICS, INC.\",\n\"Genetic targets of mood disorders in MVP\",\n\"Sep01, 2021\",\n\"rejected\",\n\"Recent classes of methods have been developed to \\u00e2\\u0080\\u009cimpute\\u00e2\\u0080\\u009d gene expression levels in hard-to-ascertain tissues like brain using just DNA sequence variation, which can identify new targets for different mood disorders when casting wider nets of molecular associations in brain tissue. \",\n\"Recent classes of methods have been developed to \\u00e2\\u0080\\u009cimpute\\u00e2\\u0080\\u009d gene expression levels in hard-to-ascertain tissues like brain using just DNA sequence variation, for example predixcan [PMID: 26258848] and TWAS [PMID: 26854917]. While these algorithms have already published and publicly disseminated genotype/SNP \\u00e2\\u0080\\u009cweights\\u00e2\\u0080\\u009d to predict gene expression levels across all tissues in the Genotype By Tissue Expression (GTEx) project [PMID: 32913098], only two expression summarizations \\u00e2\\u0080\\u0093 gene-level and splicing-level are routinely considered. We have previous shown that other expression features like exons and exon-exon junction counts may be more strongly related to genetic variation than gene-level summarizations [PMID: 30050107], which require generating new \\u00e2\\u0080\\u009cweights\\u00e2\\u0080\\u009d at these features. Here we propose to cast a wider net of molecular/eQTL characterizations with the goal of identifying new targets in the genetic risk of different mood disorders identified by MVP. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Johnson, Eric \",\n\"RESEARCH TRIANGLE INSTITUTE\",\n\"Genome-wide association study of opioid addiction using public controls\",\n\"Jul07, 2020\",\n\"rejected\",\n\"Approximately 50% of the risk for heroin abuse is attributable to genetic factors. However, few genes have been consistently associated with heroin abuse, and it is likely that a number of genes contributing to heroin abuse have not been identified. The proposed study would test associations between heroin abuse and genetic variants across the genome, with the goal of uncovering biological mechanisms that underlie drug addiction. More broadly, we also examine genetic risks for substance use disorders regardless of specific drug used.\",\n\"The goal of the proposed study is to identify novel genetic determinants of opioid addiction (OA) and related substance use disorder (SUD) phenotypes by conducting genome-wide association studies (GWAS), including meta-analyses across cohorts, as well as multi-omic gene network studies for SUD phenotypes that combine GWAS with other omics (e.g., RNAseq from postmortem brain tissue) in multiplex data.  Although GWAS studies have successfully identified genetic variants contributing to other SUD phenotypes (e.g. nicotine dependence), there are few GWAS of rarer high-risk substance abuse.  For GWAS, we propose to use a case-control study design, drawing OA cases from the Urban Health Study (UHS) and other case-only cohorts and matching population controls from dbGaP deposited studies.  We have published GWAS using such a case / population control approach (Johnson, et al, 2015; Hancock et al. 2015; Gaddis et al. 2022) and have recently extended our array-dependent matching protocol to use cohorts with whole genome sequencing data (Mathur et al. 2022).  In the current period we will continue to use these previously acquired cohorts and add dbGaP studies as population controls for newly genotyped OA cases to study SUD phenotypes. We will also request new dbGaP studies with SUD phenotype data for replication and/or meta-analyses as they become available.  We will continue to participate in the Addiction group of the Psychiatric Genetic Consortium (PGC): analyzing these data in large meta-analyses with other investigators. We also lead the Genetics of Opioid Addiction Consortium (GENOA), which combined summary statistics from 13 cohorts (Gaddis et al., 2022). As part of NIDA Smoke Screen Initiative, several new cohorts that are informative for OA have been or are in the process of being genotyped, which we will request under this project. This projects work is now funded under P50 DA054071: Integrative Omics Center for Accelerating Neurobiological Understanding of Opioid Addiction (IOmics-OA).\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Jonas, Katherine \",\n\"STATE UNIVERSITY NEW YORK STONY BROOK\",\n\"Exploring the effect of dimensional phenotypes on gene discovery\",\n\"Jun29, 2022\",\n\"approved\",\n\"When scientists want to identify the genetic factors associated with psychological illnesses, they most often divide people into people with the illness (cases) and people without the illness (controls).  However, this method of grouping people can be inefficient, because cases with very severe symptoms are not distinguished from cases with mild symptoms, nor are controls with subthreshold symptoms differentiated from people with no symptoms.  In this study, we will compare the traditional, case-control method to a method in which every person is rated on a continuous scale. We hypothesize that a continuous scale will be a more efficient method of identifying genetic risk factors for psychological illnesses.\",\n\"The field of molecular psychiatric genetics has made enormous advances in the last decade, predominantly through large-scale genome-wide association studies (GWASs) dependent on categorical, case-versus-control analyses embedded within traditional diagnostic classification systems. A growing body of evidence suggests that a quantitative and transdiagnostic approaches to phenotype measurement can further enhance the discovery power of GWASs. The current study provides a comprehensive review of such phenotypic approaches in GWASs across all major psychiatric and neurodevelopmental disorders. We identify seven themes and recommendations that emerge from the literature, encompassing the issues of statistical power, disorder prevalence, psychopathology measurement, and novel phenotypes emerging from hierarchical \\u00e2\\u0080\\u0098splitting\\u00e2\\u0080\\u0099 and \\u00e2\\u0080\\u0098lumping\\u00e2\\u0080\\u0099 approaches. We also explore the effect of dimensional phenotypes on heritability and validity.  Our objective in requesting summary statistics from the MVP GWAS is to compare the heritability and genetic correlations of the PTSD, MDD, and GAD phenotypes reported in Stein et al. (2021) and Levey et al. (2021) with the case-control phenotypes for PTSD, MDD, and GAD reported in Nievergelt et al (2019), Howard et al (2019), and and Purves et al (2020), respectively.  Heritability and genetic correlations will be estimated using LD score regression (Bulik-Sullivan, 2015). The heritability and genetic correlation estimates will be used to evaluate the utility of dimensional, quantitative phenotypes for GWAS.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Jung, Jeesun \",\n\"NIH\",\n\"Multi-omic and Mendelian randomization analyses of alcohol use\",\n\"Mar02, 2023\",\n\"approved\",\n\"Genetic variants associated with alcohol use can be used to describe the biology of alcohol addiction and identify relationships between alcohol addiction and other traits. We will use genetic variants associated with alcohol use in Million Veterans Program (MVP) studies to perform computational research aimed at these aforementioned goals.\",\n\"Summary statistics from Million Veterans Program (MVP) genome-wide associations studies (GWAS) of maximum habitual alcohol intake, AUDIT-C, and problematic alcohol use will be used to perform multi-omic and Mendelian randomization analyses of alcohol use. Our multi-omic analyses will seek to describe transcriptomic, cell-type, and protein associations with alcohol use. Our Mendelian randomization analyses will seek to identify potential causal relationships between alcohol use and clinical and biological phenotypes. We will perform both discovery and replication analyses using MVP data. The aim of our analyses is to elucidate the biology of alcohol addiction and identify therapeutic targets for alcohol use disorder.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kaikkonen-M\\u00c3\\u00a4\\u00c3\\u00a4tt\\u00c3\\u00a4, Minna \",\n\"UNIVERSITY OF EASTERN FINLAND\",\n\"Functional interpretation of enhancer variants associated with CAD\",\n\"Apr15, 2022\",\n\"approved\",\n\"Coronary artery disease (CAD) is the leading cause of death in the Western world. To improve disease prevention, diagnosis and treatment, we need to gain better understanding of the genetic and environmental factors that promote the progression of the disease. Hundreds of SNPs have been significantly associated with CAD. However, the vast majority of such variants are located in the noncoding sequences with yet undefined role. Large fraction of them do fall within enhancers specific for disease relevant cell types. In this study, we aim to identify and functionally characterize enhancer variants affecting cell type specific gene expression. Shedding light into the molecular mechanisms of these variants is hoped to provide insight into the genetic basis of CAD. \",\n\"Coronary artery disease (CAD) is the leading cause of death in the Western world. To improve disease prevention, diagnosis and treatment, we need to gain better understanding of the genetic and environmental factors that promote the progression of the disease. Hundreds of SNPs have been significantly associated with CAD. However, the vast majority of such variants are located in the noncoding sequences with yet undefined role. Large fraction of them do fall within enhancers specific for disease relevant cell types. In this study, we aim to identify and functionally characterize enhancer variants affecting cell type specific gene expression. To provide mechanistic insight for CAD associated regions, GTEx data is used to identify target genes using eQTL and allele-specific gene expression analysis. In addition, GTEx data is used to build tissue-specific gene regulatory networks that are compared to our cell-type specific gene modules and for biomarker analysis. GTEx data is analyzed independently of other datasets. The Million Veteran Program (MVP) summary statistics and credible sets from the latest CAD GWAS are used to prioritize and select regulatory elements for functional characterization using CRISPR and massively parallel reporter assays. This could allow fine-mapping of risk variants and identification of causal disease mechanisms.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kawai, Vivian \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Genetics and Inflammation\",\n\"Dec30, 2019\",\n\"approved\",\n\"We are broadly interested to study how inflammatory diseases shared genetic and clinical factors with other clinical phenotypes, and we will use different methods and large genetic studies to identify genetic factors that are shared among different phenotypes.  \",\n\"Objective: The overall goal of this proposal is to define clinical and genetic pleiotropy in inflammatory conditions such as RA, SLE, gout and others.  The availability of large genome-wide association studies (GWAS) for different phenotypes provides an unprecedented opportunity to use big data to make unexpected discoveries. Hence, the specific aims of this analysis are: 1) to determine if the genetic determinants of a risk factor or disease alter the risk of inflammatory diseases; and 2) to determine if the genetic determinants of inflammatory diseases alter risk factors for other diseases or other diseases themselves. Study Design & Analysis Plan: We will use summarized data from different GWAS datasets (including the MVP transethnic blood pressure study in over 750,000 individuals) and the Vanderbilt Medical Center Biobank BioVU (which is linked to the de-identified electronic medical record) to study the association between autoimmune and non-autoimmune traits using: 1) traditional association approaches, b) mendelian randomization methods, 3) genetic risk scores constructed from publicly available GWAS databases that predict the risk factor or disease, 4) phenome-wide association study approach, and 5) the use of methods to imputed molecular endophenotypes (like transcriptomics) to study their association with clinical phenotypes. Our results will be published and broadly share with the scientific community following the NIH policies. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kelly, Tanika \",\n\"TULANE UNIVERSITY OF LOUISIANA\",\n\"Genomic determinants underlying cardiovascular disease susceptibility\",\n\"May15, 2021\",\n\"approved\",\n\"Cardiovascular disease is a leading cause of death globally. Our genetic makeup influences our susceptibility to cardiovascular disease and its contributing risk factors. Identification of the genes involved in these processes has been difficult for scientists. However, recent advances in technology and statistical methods, as well as public availability of genomic data from numerous large population-based studies, now provides scientists with the opportunity to more comprehensively examine genes and genetic factors related to cardiovascular disease. In the proposed study, we will explore the association between genetic variants across the genome for their association with cardiovascular disease and its risk factors in many large studies. Through the proposed research, we may identify important genes for cardiovascular disease, which could lead to advancements in its treatment and prevention.\",\n\"Cardiovascular disease (CVD) is a leading cause of death globally. Hypertension, diabetes, chronic kidney disease, obesity, and dyslipidemia, as well as their related cardiometabolic traits, associate with increased risk of CVD. Recent genome-wide association studies (GWAS) have identified important loci related to these phenotypes. However, our understanding of their genomic etiology remains limited. The objective of the current study is to identify genes and variants that influence CVD-related traits among studies accessed through dbGaP as well as in the NHLBI-funded Genetic Epidemiology Network of Salt-Sensitivity study. We will use exome-sequencing, genome-wide genotype, and GWAS meta-analysis results data from the Atherosclerosis Risk In Communities study, the Cardiovascular Health Study, the Chronic Renal Insufficiency Cohort Study, the International Consortium of Blood Pressure, the Finland-United States Investigation of NIDDM Genetics study, the Framingham Heart Study, the Jackson Heart Study, the Multi-ethnic Study of Atherosclerosis, the Coronary Artery Risk Development in Young Adults Study, and the Women\\u00e2\\u0080\\u0099s Health Initiative. To achieve our overall objective, we will carry-out the following specific aims: 1.) To identify novel common and low-frequency variants that influence CVD-related traits. We will use several approaches to identify such variants, including GWAS, meta-analysis, and candidate gene study using GWAS results, genome-wide genotype and exome sequencing data. Single-marker and pathway-based analyses will facilitate novel variant discovery. 2.) To identify novel rare variants that may have functional effects on CVD-related traits. We will use state-of-the-art statistical methods to examine the collective effects of rare variants in aggregate analyses using exome sequencing data. 3.) To examine the collective effects of known genomic variants for CVD-related traits to predict risk of CVD in these large, population based studies. By helping to elucidate the biological pathways underlying CVD and its related risk factors, these findings may aid in the prevention CV\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kilpelainen, Tuomas \",\n\"UNIVERSITY OF COPENHAGEN\",\n\"Genetic regulation of weight gain and risk of diabetes and cardiovascular disease\",\n\"Feb27, 2020\",\n\"closed\",\n\"Obesity is associated with increased risk of diabetes, heart disease, and stroke. However, many obese individuals, often called \\u00e2\\u0080\\u009cmetabolically healthy obese\\u00e2\\u0080\\u009d, seem protected from these diseases. Conversely, some individuals who are normal weight have an increased metabolic risk similar to the obese. This paradox may be partly explained genetic differences. Indeed, recent genetic studies have identified multiple DNA variants associated with higher body fat but lower insulin resistance and risk of diabetes and heart disease. Many of these variants are linked to storing relatively more fat on the hip than on the waist. In the present project, we will examine whether DNA variants showing such \\\"favorable adiposity\\\" effect are associated with long-term weight gain that does not lead to increased risk of diabetes, heart disease, or stroke. \",\n\"The current obesity pandemic is a major threat to public health systems worldwide. The majority of obesity-related health care costs are due to cardiometabolic impairments such as insulin resistance, dyslipidemia, and hypertension, which increase the risk of type 2 diabetes and cardiovascular disease. However, many obese individuals seem resistant to cardiometabolic complications, the \\\"metabolically healthy obese\\\". Conversely, some normal weight individuals suffer from cardiometabolic comorbidities similar to the obese, the \\\"metabolically obese normal-weight\\\". Genetic mechanisms may partly explain this paradox. Recent genome-wide association studies have identified multiple genetic loci that are associated with increased overall body fat and subcutaneous fat deposition, but lower cardiometabolic risk. At present, it remains unclear whether genetic predisposition to such \\\"favorable adiposity\\\" may uncouple long-term weight gain from cardiometabolic risk during adulthood.  The current obesity pandemic is a major threat to public health systems worldwide. The majority of obesity-related health care costs are due to cardiometabolic impairments such as insulin resistance, dyslipidemia, and hypertension, which increase the risk of type 2 diabetes and cardiovascular disease. However, many obese individuals seem resistant to cardiometabolic complications, the \\\"metabolically healthy obese\\\". Conversely, some normal weight individuals suffer from cardiometabolic comorbidities similar to the obese, the \\\"metabolically obese normal-weight\\\". Genetic mechanisms may partly explain this paradox. Recent genome-wide association studies have identified multiple genetic loci that are associated with increased overall body fat and subcutaneous fat deposition, but lower cardiometabolic risk. At present, it remains unclear whether genetic predisposition to such \\\"favorable adiposity\\\" may uncouple long-term weight gain from cardiometabolic risk during adulthood.  The overall objective of the present project is to examine whether genetic predisposition to a \\\"metabolically healthy obese\\\" phenotype diminishes the impact of weight gain on risk of cardiometabolic disease. We will construct a \\\"favorable adiposity\\\" genetic score by combining all genetic loci associated with increased body fat but decreased insulin resistance (defined by decreased fasting insulin and triglycerides, and increased HDL cholesterol) by using genome-wide summary statistics from the largest available genome wide association studies. To generate this genetic score, we are requesting dbGaP summary statistics data for HDL cholesterol and triglycerides for European-ancestry individuals from the Million Veteran Program GWAS, published in 2018 (Klarin D et al Nat Genet 2018). To maximise the sample size for HDL cholesterol and triglycerides, we will meta-analyze the summary statistics from the Million Veteran Program with the summary statistics available from lipids meta-analyses published by the Global Lipids Genetics Consortium in 2013 (Willer CJ et al Nat Genet 2013). As only summary statistics and no individual-level data will be used in these combined analyses, no additional risks will be created for the participants. We will subsequently use the genetic score derived from the variants identified to be associated with insulin resistance, to test whether the scores diminishes the impact of weight gain on the levels of cardiometabolic risk markers and risk of type 2 diabetes and cardiovascular disease in a meta-analysis of data from six Danish prospective cohorts, locally available to us at the University of Copenhagen.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kilpelainen, Tuomas \",\n\"UNIVERSITY OF COPENHAGEN\",\n\"Mendelian randomization analyses of fatty liver disease and metabolic disorders\",\n\"Dec21, 2020\",\n\"closed\",\n\"Fatty liver disease is a common disease characterized by excessive fat accumulation in the liver, which may ultimately result in cirrhosis or cancer. We will study whether fatty liver disease is causally associated with blood lipid levels and other metabolic risk markers and disease risk, such as type 2 diabetes and heart disease. The results will show whether fatty liver disease drives the development of metabolic disease, or whether metabolic disease accelerates the development of fatty liver disease. \",\n\"Fatty liver disease is the most frequent chronic liver disease in Western countries, affecting more than 30% of the general population. A large body of observational studies suggest that fatty liver disease is associated with metabolic disease and inflammation. However, observational studies suffer from confounding and reverse causality, and it thus remains unclear whether fatty liver disease is causally associated with metabolic disease and inflammation, or vice versa. As genotypes are randomly allocated at conception, genetic alleles can be used to assign individuals according to higher or lower levels of an exposure trait in a randomized manner. This approach, Mendelian randomization, helps to assess causal relationships by minimizing the influence of confounding and reverse causality on observational results.   In this project we will test bidirectional causal associations between fatty liver disease and metabolic and inflammatory profiles using two-sample Mendelian randomization approach applied on summary statistics from genome-wide association studies (GWAS). As part of this study, we request access to the dbGaP summary statistics for the Million Veteran Program GWAS for LDL cholesterol, HDL cholesterol, total cholesterol and triglycerides, published in 2018 [Klarin et al. Nat Genet 2018], to study the bidirectional causal associations between lipid levels and fatty liver disease (liver inflammation, steatosis and fibrosis). To maximise the sample size for HDL cholesterol and triglycerides, we will meta-analyze the summary statistics from the Million Veteran Program with the summary statistics available from lipids meta-analyses published by the Global Lipids Genetics Consortium in 2013 [Willer CJ et al Nat Genet 2013]. As only summary statistics and no individual-level data will be used in these combined analyses, no additional risks will be created for the participants. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kim, Dong Ki \",\n\"SEOUL NATIONAL UNIVERSITY HOSPITAL\",\n\"Causal factors for diabetes and cardiovascular diseases: A Mendelian randomization study\",\n\"Sep29, 2020\",\n\"closed\",\n\"We are planning to use the genome-wide association study results for diabetes mellitus in our study. Summary statistics are generated from genome-wide association study results, identifying associations between genetic information and diseases, and in the summary-level data, individuals are not identifiable. The summary statistics can be implemented in a method of \\\"Mendelian randomization\\\" analysis, which stratifies genetic risk of an exposure and investigating the diseases risk according to the genetic predisposition. The method has strength in identifying causal risk factors for complex diseases. Mendelian randomization analysis can be performed by summary-level data, one for the exposure and one for the disease outcome. The data from phs001672 includes a genome-wide association study results for diabete mellitus, so the data will be utilized as the outcome summary statistics of our Mendelian randomization study.\",\n\"This study aimed to identify causal factors for diabetes and cardiovascular diseases through a Mendelian randomization analysis. The study will perform two-sample Mendelian randomization analysis, and we will implement the data from phs001672 as the outcome summary statistics for the Mendelian randomization study. The genetic instrument for the Mendelian randomization will be introduced from external sources, but the information would not be directly cooperated into the dataset but only the summary-level data of the data from phs001672 will be utilized. Thus, the study would not result in additional risks to participants included in the phs01672 data. The data of particular interest has been published in Nature Genetics volume 52, pages 680\\u00e2\\u0080\\u0093691(2020), and we are planning to utilize the GWAS summary statistics for type 2 diabetes mellitus of individuals of European ancestry (pha004945.1) as the outcome of our two sample Mendelian randomization analysis. Two sample Mendelian randomization analysis will be performed by a statistical package in R. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kooperberg, Charles \",\n\"FRED HUTCHINSON CANCER CENTER\",\n\"Genetic risk of type 2 diabetes and related risk factors and complications\",\n\"Sep16, 2021\",\n\"approved\",\n\"We are requesting genome-wide association study (GWAS) summary statistics that will be used to construct multi-ancestry type 2 diabetes (T2D) polygenic risk scores (PRS), which will be used to determine whether genetic risk of T2D is predictive of T2D complications and whether genetic risk of T2D is mediated by T2D risk factors.\",\n\"Type 2 diabetes (T2D) is a major public health challenge that expected to notably rise in prevalence over the next couple decades, demonstrating the urgent need to improve T2D risk prediction and stratification. Polygenic risk scores (PRS) have shown promise in risk stratification of T2D, particularly when combined with non-genetic clinical predictors. Within the Population Architecture using Genomics and Epidemiology (PAGE) Study, we recently found that a multi-ancestry T2D PRS performed similarly or better than population specific T2D PRS and accounted for up to 15.3% of familial relative risk of T2D across diverse populations (Polfus, Darst et al. HGG Advances 2021). However, given the large number of modifiable risk factors for T2D, further characterizing these scores could lead to a better understanding of the mechanisms impacted by genetic risk of T2D. Moreover, it remains unclear whether T2D PRS could hold prognostic value for diabetes-related complications in individuals who have T2D.  Our primary study question will investigate whether a multi-ancestry type 2 diabetes (T2D) polygenic risk score (PRS) is associated with T2D-related phenotypic risk factors and complications across diverse populations. Our secondary study question will investigate whether the effect of the T2D PRS on T2D risk is mediated by T2D risk factors. We are requested genome-wide association study (GWAS) summary statistics to construct these PRS, which be subsequently calculated and investigated in the PAGE Study. Summary statistics may be combined with T2D GWAS summary statistics generated in the PAGE Study, but no additional risks will be posed to research participants. This is a joint research project with Dr. Burcu Darst at the University of Southern California (Project #29826).\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kooperberg, Charles \",\n\"FRED HUTCHINSON CANCER RESEARCH CENTER\",\n\"Learning Genetic Architecture of Complex Traits Across Populations\",\n\"Apr29, 2021\",\n\"closed\",\n\"Genome-wide association studies (GWAS) have become a primary tool for understanding the genetic basis of complex traits and diseases.  The goal of this research is to augment our understanding of cardiovascular diseases in minority populations, so that GWAS can benefit all individuals. Specifically, by leveraging information across diverse populations, our analyses aim to identify minority-relevant genetic variants and to build polygenic risk scores (PRS) for minority populations. \",\n\"Understanding the genetic factors influencing complex traits in minority populations is challenging because the sample sizes are often much smaller in Genome-wide association Studies (GWAS) of minority populations, compared to GWAS in European populations. As described below, we have previously developed minority-centric association mapping and genetic risk prediction approaches. The key insight underlying these approaches is to leverages multi-ethnic and trans-ethnic information, while acknowledging heterogeneity across populations. The objective of this dbGaP application is to apply and extend these methods to identify identifying minority-relevant genetic factors underlying plasma lipids, stroke and VTE in African Americans and Hispanics. We will also build population-specific polygenic risk score (PRS) for lipids and CAD (stroke and VTE). For association mapping, we will apply an empirical Bayes approach developed by Dr. Tang\\u00e2\\u0080\\u0099s group, XPEB (Coram et al.2015;  PMID: 25892113). This method considers a study design, in which summary-level statistics from two GWAS are available: one in the minority population of primary interests (referred to as the focal population), and the other in a reference population of much larger sample size (referred to as the base population). As an example, to identify genetic loci associated with low-density lipid cholesterol (LDL) in African Americans, we will generate the focal GWAS summary-statistics by meta-analyzing an African American LDL study performed in the Women\\u00e2\\u0080\\u0099s Health Initiative (WHI) with the African American LDL analysis in the Million Veterans Program. (All investigators on this, and the collaborating, applications have access to WHI). For base GWAS, we will use MVP\\u00e2\\u0080\\u0099s European-stratified LDL summary statistics.  For PRS, we will apply a mixed-effects model approach developed by Dr. Tang\\u00e2\\u0080\\u0099s group, XP-BLUP (Coram et al. 2017; PMID: 28757202). This method considers a study design, in which individual-level genotype-phenotype data is available in a minority cohort (the focal GWAS), while only summary-level GWAS statistics are available in the base GWAS. In the proposed study, WHI individual-level African Americans and Hispanic American GWAS data are available to all investigators. Analysis: For association mapping, we will apply XPEB (implemented in an open source R package)  for each of the lipids and CAD traits, and using African Americans or Hispanics as the focal population.  For PRS, XP-BLUP program, implementing the mixed-effects model, is available on github. Specifically, we will include age and genetic-PC as covariate for all analysis. In addition, trait-specific covariates will be adjusted as needed.    Validation/Replication: For minority-relevant loci identified using XPEB, we will seek replication in the appropriate minority-populations. Because we anticipate some loci identified by XPEB feature moderate/weak effect sizes, other minority-cohorts are under powered. As an alternative, we will examine consistency in allelic effects and assess the improvement in disease risk prediction. For minority-specific PRS, we evaluate the predictive performance on an independent cohort of matching minority population (African Americans or Hispanics). We will compare PRS built by XP-BLUP with PRS directly transferred from European cohorts.  The DUL states \\u00e2\\u0080\\u009cuse of this data is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry. Use of the data includes methods development research (e.g., development of software or algorithms).\\u00e2\\u0080\\u009d Our analysis will use the population-specific GWAS results produced by the MVP; no analysis is proposed to study the population origin or ancestry.  The proposed research will be performed in collaboration with Dr. Hua Tang\\u00e2\\u0080\\u0099s group at Stanford University. Dr. Tang is submitting a dbGaP application through Stanford.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kooperberg, Charles \",\n\"FRED HUTCHINSON CANCER CENTER\",\n\"Integrative Genomics into Genetic Association Studies of Blood Pressure and Stroke in African Americans\",\n\"May06, 2022\",\n\"expired\",\n\"We are applying novel methodologies to identify genetic associations between stroke and blood pressure phenotypes for African Americans and other racial/ethnic groups. We do this by finding models associating genetic data with data on other omics, and then by applying these methods to existing genome-wide association studies. Discovery of genetic variants that predispose to blood pressure and stroke is a crucial step toward understanding genetic mechanisms that may lead to novel prevention and treatment strategies.\",\n\"Stroke is the third leading cause of death among African Americans (AAs): they are twice as likely to die from stroke as European Americans (EAs), and their incidence rate is almost double that of EAs. Recent genomewide studies (GWAS) suggest there is a substantial genetic contribution to stroke risk in African ancestry populations, with heritability estimates of about 35%. However, to date, genetic studies in AAs are greatly lagging behind those in EAs despite their increased stroke burden. Among the risk factors for stroke, blood pressure is a major contributor: 4 in 10 AAs suffer from hypertension, 50% more than EAs. These disparities have been considered to be mediated by environmental and social determinants, yet they remain after adjusting for demographics, socioeconomic status, clinical characteristics, and modifiable health behaviors. Heritability analysis suggests African ancestry is associated with hypertension, with heritability estimates from 30\\u00e2\\u0080\\u009340% for systolic and diastolic blood pressure. However, genetic susceptibility to hypertension among AAs is less well studied compared to other ethnic groups. Therefore, there is considerable motivation for identifying the genetic components of stroke and high blood pressure in AAs.  Discovery of genetic variants that predispose to blood pressure and stroke is a crucial step toward understanding genetic mechanisms that may lead to novel prevention and treatment strategies. Yet, GWAS have thus far identified genetic loci that together account for only a small proportion of the heritable risk. Substantial efforts have been devoted to studying the association of genetic variation with gene expression and other molecular characteristics through large collaborative initiatives such as Genotype-Tissue Expression (GTEx) and Encyclopedia of DNA Elements. These initiatives have provided a deeper understanding of functional elements across the genome, which have been used to inform genetic association and identified many novel loci. However, most of the data in these studies have focused on European ancestry and little has been done in AAs. Our recent work in Nature shows that genetic discoveries in one population do not readily transfer to other populations.  The objective of this study is to identify variants predicting various genomic features (gene expression, methylation and protein) in AA samples that have been recently collected through Trans-Omics for Precision Medicine, the CommonMind Consortium, and GTEx, and to integrate this functional information into genetic association analysis of blood pressure and stroke in AAs. Insight into both molecular activity and genetic variation can inform association analysis and enable novel genome-wide discoveries. In particular, we propose to develop methods that leverage the data for EAs to improve power for identifying genetic variants that regulate various types of genomic features in AAs, and for integrating the genomic regulation models into GWAS with the ultimate goal to identify novel loci for stroke risk and blood pressure in AAs. To facilitate these aims we have assembled the largest number of AAs for genomic studies and AA stroke cases and blood pressure data for GWAS.  One of the main strategies that we will employ is to construct predictive models for the genomic features (gene expression, methylation and protein) based on GWAS data and apply these models to existing studies that have GWAS summary statistics for these features available. The data for which we are applying has such summary statistics. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kragstrup, Tue \",\n\"UNIVERSITY OF AARHUS\",\n\"JAK-STAT pathways in venous thromboembolism\",\n\"Mar06, 2021\",\n\"closed\",\n\"JAK inhibitors are drugs targeting the JAK-STAT pathway. These drugs have proven effective in the treatment of several immune mediated inflammatory diseases such as rheumatoid arthritis and psoriatic arthritis. Drug adverse effects are recognized both during clinical trials and post marketing. Side effects of the JAK inhibitors during the clinical trials were herpes zoster. Post marketing surveillance has now warranted a warning on several of the JAK inhibitors for thromboembolic events. These side effects are not fully understood. This study will investigate whether there is a genetic basic for the putative implication of JAK-STAT signaling in the development of VTE. We expect that this study will help improve the use of JAK inhibitors and drive development to explore drugs with less side effects.\",\n\"Background: Venous thromboembolism (VTE) is a collective term for deep vein thrombosis (DVT) and pulmonary embolism (PE). Janus kinase (JAK) inhibitors are a group of small molecules that block one or more of the intracellular tyrosine kinases: JAK1, JAK2, JAK3 and TYK2. These compounds can block different cytokine signalling pathways, and they therefore have an immunosuppressive effect. JAK inhibitors are used in treatment of autoimmune and rheumatological diseases, such as rheumatoid arthritis, psoriasis, inflammatory bowel disease, and ulcerative colitis. Drug adverse effects are recognized both during clinical trials and post marketing. Side effects of the JAK inhibitors during the clinical trials were herpes zoster. Post marketing surveillance has now warranted a warning on several of the JAK inhibitors for thromboembolic events. In 2017 The European Medicines Agency revised the summary of product characteristics for Baricitinib to include PE and DVT. In 2019 The US FDA issued a \\u00e2\\u0080\\u009cblack box\\u00e2\\u0080\\u009d warning, that patients treated with Jakinibs had experienced thrombosis, including DVT and PE. These side effects are not understood. Therefore, investigating the mechanism behind the association between JAK inhibitor use and VTE is needed to keep JAK inhibitors attractive.  Aim: We want to use the requested dataset to assess the genetic burden in the JAK/STAT signaling pathway and downstream STAT transcription factor regulomes in VTE.  Methods: We will determine if gene sets containing JAK/STAT pathway genes and/or STAT target genes are enriched with VTE genetic risk using generalized gene-set analysis of VTE GWAS summary statistics with MAGMA (https://ctg.cncr.nl/software/magma). \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kullo, Iftikhar \",\n\"MAYO CLINIC ROCHESTER\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"Dec15, 2022\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease. Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data.\",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry. Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community. However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kullo, Iftikhar \",\n\"MAYO CLINIC ROCHESTER\",\n\"Development and validation of polygenic risk score-based risk models for coronary heart disease in diverse racial/ethnic groups\",\n\"May20, 2020\",\n\"approved\",\n\"Risk scores that are made up of genetic markers in the human genome, i.e. polygenic risk scores (PRS), have been shown to identify individuals at 2-3 times higher risk of heart disease. However, application of these scores to racial/ethnic minorities is challenging since PRS were developed using white participants. Genetic differences across racial/ethnic groups require development/validation of PRS specifically for minority racial/ethnic groups to achieve the same level of performance. In this study, we aim to use dbGaP cohorts to develop/validate PRS specifically tailored for minority groups. This will allow accurate characterization of the genetic risk of heart and blood vessel diseases in the United States and better risk estimations for ethnically diverse groups.\",\n\"Since polygenic risk scores (PRS) for coronary heart disease (CHD) are derived from mainly European ancestry (EA) cohorts, there is an urgent need for the development of race/ethnicity specific PRS and validation of EA-derived PRS in minority groups such as African ancestry (AA) and Hispanic ethnicity (HE) individuals. This study is being submitted by the investigators involved in the NHGRI-funded electronic MEdical Records and Genomics (eMERGE) network. Our primary objective is to use dbGaP cohorts with participants of non-EA ancestry to validate PRS-based relative and absolute risk models derived using the eMERGE Phase III cohort for CHD  Genetic data (genotyping array, whole genome sequencing etc.) will be used to compute PRS for CHD. In addition to the genetic data, phenotypic data such as age, sex, serum lipid measurements, blood pressure measurements, history of diabetes mellitus, hypertension, use of lipid lowering and/or anti-hypertensive therapy, smoking status as available will be incorporated into the prediction models to fully account for both genetic and conventional risk factors. Models developed using eMERGE data will be tested in dbGaP cohorts to assess the strength of association of each risk factor, discriminatory ability, and calibration of the model in each racial/ethnic group. These models will be refined based on the assessment using dbGaP cohorts.       \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Lai, Dongbing \",\n\"INDIANA UNIV-PURDUE UNIV AT INDIANAPOLIS\",\n\"Genetic study of alcohol use disorder and related phenotypes\",\n\"Jun28, 2019\",\n\"approved\",\n\"Studies have shown that alcohol use disorders and related disorders are caused by many genes with small effects. The purposes of this project are performing innovative analysis to identify these genes and investigating their interactions with other genes and/or environmental factors. We will also perform polygenic risk scores analysis to evaluate an individual's overall genetic risk on these diseases.\",\n\"Large-scale genome-wide association studies of alcohol use disorder (AUD) and related diseases such as other substance use disorders and psychiatric diseases have identified many risk variants/genes. However, these variants/genes only explained a small portion of the heritability. Majority variants/genes having small effects and thus having sub genome-wide significance P-values remain to be discovered. In this project, we aim at performing innovative analyses to identify sub-threshold variants/genes; then for those variants/genes identified, we will investigate whether they interact with other variants/genes and/or environmental factors. We will also perform polygenic risk scores analysis to evaluate an individual's overall genetic risk on these diseases. The finding of this project can help us understand the etiologies of AUD and related diseases, and shed light on the development of personalized prevention and treatment methods.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Lai, Dongbing \",\n\"INDIANA UNIV-PURDUE UNIV AT INDIANAPOLIS\",\n\"Genetic study of complex diseases\",\n\"Mar30, 2023\",\n\"approved\",\n\"Complex diseases are caused by many genetic variants. Most of them have small effects and are not genome-wide significant in recent large-scale genome-wide association studies (GWAS). The purpose of this project is using a novel strategy developed by our group, which focuses on variants having the same direction of effects in all study cohorts, to identify small effect variants by using GWAS summary statistics from the Million Veteran Program and other large-scale GWAS.\",\n\"Recent large-scale genome-wide association studies (GWAS) have identified many disease-associated variants, however, for most complex diseases, they only explained a small portion of heritability. Many variants having small effects and thus sub-threshold P-values in large-scale GWAS remain to be identified. Further increasing the sample sizes to make all disease related variants genome-wide significant is not feasible in near feature. Study cohorts in large-scale GWAS are typically ascertained using different strategies and/or using different diagnostic methods. While this heterogeneity will generate many study-specific findings, for disease-associated variants, they will have the same direction of effects in all study cohorts (i.e., they are concordant), even they are not genome-wide significant. Furthermore, if these concordant SNPs can explain large proportion of heritability and polygenic risk scores calculated using them have high predictability, then they are likely to be truly disease-associated. In our previous works, we used this strategy and identified many small effect variants related to alcohol use disorder and Alzheimer's disease and related dementia in diverse populations. In this study, our objectives are using the same strategy to identify small effect variants associated with other complex diseases such as hypertension, type 2 diabetes, cardiovascular diseases, etc. We will use GWAS summary statistics from the Million Veteran Program and other large-scale GWAS of the same traits to identify concordant variants, then meta-analysis will be performed by using concordant variants only. Meta-analysis results will be used to estimate heritability and calculate PRS. We will not combine any dataset and findings will be tested in independent datasets. The identified variants will help us understand the genetic etiologies of complex diseases. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Lee, Phil \",\n\"MASSACHUSETTS GENERAL HOSPITAL\",\n\"Investigating the common genetic control mechanisms of brain transcriptome for neurodevelopmental and neurological disorders \",\n\"Aug20, 2020\",\n\"closed\",\n\"Genome-wide genetic studies of psychiatric disorders have yielded important findings of disease susceptibility loci in recent years. Nevertheless, the biology underlying complex brain disorders is still elusive at best. In the present study, we examine the influence of genetic variability in human cortical gene expression on the psychopathology underlying major neurodevelopmental and neurological disorders. \",\n\"This study investigates the etiologic role of expression quantitative trait loci (eQTLs) markers in neurodevelopmental and neurological disorders. We will first identify brain-specific eQTLs using publicly available gene expression (from GEO) and genome data (obtained through dbGaP) of healthy controls. We will take a meta/mega analytic strategy to examine the effects of brain eQTLs on various brain disorders, including substance abuse and mental health phenotypes. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Lee, Phil \",\n\"MASSACHUSETTS GENERAL HOSPITAL\",\n\"Genetic effects shared across major brain disorders and ends-phenotypes \",\n\"Nov01, 2021\",\n\"approved\",\n\"Genetic variation influences the differences in human traits and health outcomes. We will use dbGAP datasets to improve our understanding of how genetic variation confers shared and unique effects to complex human traits, brain disorders, and related health outcomes. \",\n\"The goal of this proposal is to understand how genetic variation influences the differences of complex human traits and susceptibility to brain disorders and related health outcomes. Recent successes of large-scale genomics studies, including both rare and common variations, have uncovered an un-expected level of genetic risk sharing across major psychiatric disorders at a genome-wide level. Nevertheless, little is known of which and how certain genetic loci carry risk effects transcending traditional diagnostic boundaries. To fill in this critical gap, we will use dbGAP datasets: (1) to identify specific genetic risk variants and biological mechanisms shared across major psychiatric disorders, somatic disorders, and related health outcomes; (2) to systematically evaluate the characteristics of the identified risk variants, genes, and their regulatory function; and (3) to advance our understanding of the genetic architecture and relationships of the studied phenotypes. The study design includes the meta-analysis of GWAS summary statistics and a range of genomic data analyses (e.g., variants to gene-mapping, gene expression data analyses, and polygenic scoring analyses). Our target phenotypes include major neuropsychiatric and neurological disorders, human behaviors, endophenotypes (e.g., neuroimaging), somatic disorders, and related health outcomes (e.g., pain, suicide) in diverse genomic samples.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Lei, Wei \",\n\"AFFILIATED HOSPITAL OF SOUTHWEST MEDICAL UNIVERSITY\",\n\"Post-GWAS analyses on the genetics of addictions, medical and psychiatric disorders, and behaviors\",\n\"Nov29, 2022\",\n\"approved\",\n\"Differences in genetic makeup contribute to variations in behavior, health, and diseases. We will use the summary information derived from the Million Veteran Program\\u00e2\\u0080\\u0099s genome-wide association studies (GWASs) to help understand the genetic basis of addictions, medical and psychiatric disorders, and behaviors.\",\n\"Differences in genetic makeup contribute to variations in behavior, health, and diseases. The Million Veteran Program (MVP) has generated rich data, including genome-wide association studies (GWASs)\\u00e2\\u0080\\u0099 summary statistics. We will use MVP GWAS summary statistics to conduct post-GWAS analyses, such as (but not limited to) computing genetic correlations, generating polygenic scores, converting to the gene-based association, conducting Mendelian randomization, carrying out meta-analyses, or performing pathway or network analysis to examine the genetic basis of behavior, health, and diseases, including addictions, medical and psychiatric disorders, and behaviors. We will analyze MVP GWAS summary statistics independently, combine them with GWAS summary statistics for traits from other sources, or link to gene expression data such as the Genotype-Tissue Expression (GTEx) data. We will not combine datasets but integrate the MVP GWAS summary statistics with summary information for other traits. Our overarching objective is to decipher the biology or pathways of behavior, health, and diseases through genetic summary information and gene expression data.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Lei, Wei \",\n\"AFFILIATED HOSPITAL OF SOUTHWEST MEDICAL UNIVERSITY\",\n\"post-GWAS analysis for smoking traits\",\n\"Apr13, 2021\",\n\"closed\",\n\"I want to perform the post-GWAS analysis for smoking traits, and based on the summary statistical results from smoking traits to conduct the gene-based analysis. Subsequently, combine the results of gene-based analysis and brainspan dataset to find the significant subnetworks associated with smoking traits.\",\n\"We plan to use the summary statistics from smoking traits to i) run gene-based analyses, e.g. to uncover significant genes with smoking traits and ii) combine the results of gene-based analysis and brainspan dataset to find significant networks associated with smoking traits. The preliminary plan of analyses is, 2-step. First, we want to run gene based analysis using summary statistics from smoking traits based on the VEGAS program. Second, integrate the results of gene based analysis and coexpression network to find the significant subnetworks associated with smoking traits.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Levy, Daniel \",\n\"NIH\",\n\"NHLBI TOPMed Whole-genome Sequencing Program (TOPMed WGS)\",\n\"Apr07, 2021\",\n\"approved\",\n\"The goals of the TOPMed Whole Genome Sequencing program are to identify genetic risk (and protective) factors for disease, to identify subtypes of disease, and to develop targeted and personalized treatments for disease.  The program consists of multiple studies addressing these goals for phenotypes focused on heart, lung, blood and sleep disorders, which have a high impact on public health.  These studies will include individuals from diverse populations, including those historically under-represented in health research.\",\n\"Phenotypic and genomic data from TOPMed studies will be used for: (a) investigating genotype-phenotype associations; (b) variant summary statistics, (c) common controls in association tests, (d) reference panels to impute into un-sequenced individuals from TOPMed; and (e) population genetic studies, f) integrative genomic studies. TOPMed data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Data from an individual with disease-specific consent will not be used in analyses outside of that restriction, including indirect uses such as imputation reference panels, common controls for association studies and variant summary statistics. For example, an individual with consent restricted to lung function studies will not be used to impute data intended for analyses of diseases not clearly related to lung function.  When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish or otherwise broadly share any findings from this study with the scientific community. This application anticipates collaboration among TOPMed investigators from multiple institutions, who are named in a master list appended to the IRB document uploaded with this application. Within a set of applicants approved for access to the TOPMed Exchange Areas, data may be shared in a cloud environment or by direct transfer, provided that they all agree among themselves to allow such sharing, and that all have approved TOPMed Data Access Requests for the specific data sets to be shared. Otherwise, data will be shared only among Internal Collaborators \\u00e2\\u0080\\u0093 i.e. within an institution. Data from TOPMed may be combined with non-TOPMed data from the same or other studies (obtained from dbGaP or other sources), to improve the power for novel genetic discoveries, while respecting the consent of all participants. We expect that this activity creates no additional risks to participants. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Levy, Daniel \",\n\"NIH\",\n\"Using MVP GWAS results for co-localization and mendelian randomization analysis\",\n\"May30, 2019\",\n\"approved\",\n\"To use genetic results from MVP analysis in conjunction with our genetic results from Framingham to identify important genetic signals for cardiovascular disease and its risk factors.\",\n\"We seek to use MVP GWAS results across available cardiometabolic traits to conduct co-localization and mendelian randomization analysis.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Li, Wei \",\n\"UNIVERSITY OF CALIFORNIA-IRVINE\",\n\"Characterizing the role of alternative polyadenylation in human immune cells\",\n\"Jun23, 2021\",\n\"approved\",\n\"Approximately 70% of human gene transcripts have utilized different polyA sites to generate alternative 3\\u00e2\\u0080\\u0099UTR region. Recently, it has been shown that marked 3\\u00e2\\u0080\\u0099UTR shortening event were observed upon innate immune response. We have developed a novel bioinformatics tool to identify 3\\u00e2\\u0080\\u0099 untranslated region (3\\u00e2\\u0080\\u0099 UTR) from RNA-seq. With access to the requested RNA-Seq and genotyping data from Immunological Variation Consortium, we will be able to provide a comprehensive landscape of 3\\u00e2\\u0080\\u0099 UTR usage across different immune cells, in addition, we will also investigate how genetic variations are associated with these 3\\u00e2\\u0080\\u0099UTR usage.\",\n\"For more than 70% of human genes, their 3' UTR can be shortened or lengthened under diverse physiological conditions. Since 3\\u00e2\\u0080\\u0099 UTRs contain many important cis-regulatory elements, such as miRNA binding sites, gene with shorter 3\\u00e2\\u0080\\u0099 UTRs will no longer be repressed by miRNA, leading to higher expression. In general, widespread 3\\u00e2\\u0080\\u0099UTR shortening have been observed upon active innate immune response. However, the critical target genes subject to differential 3\\u00e2\\u0080\\u0099 UTR usages in other immune cell types and their underlying genetic mechanism regulating dynamic 3\\u00e2\\u0080\\u0099 UTR usages during immune activation remain poorly understood. Therefore, we previously developed a novel bioinformatics, termed DaPars, for dynamic analysis of alternative polyadenylation from RNA-seq (Xia, Nature Commun., 2014), which can effectively profile 3\\u00e2\\u0080\\u0099UTR usages from RNA-seq data. The objective of this project is to apply our new developed DaPars2 algorithm to various RNA-seq data to provide a comprehensive landscape of dynamic 3\\u00e2\\u0080\\u0099UTR usages across human immune cells and to infer the potential upstream genetic variations modulating 3\\u00e2\\u0080\\u0099 UTR usage. Together, these differential 3\\u00e2\\u0080\\u0099 UTR usage and their associated genetic variants represent an illustrative case of genomic \\u00e2\\u0080\\u009cdark matter\\u00e2\\u0080\\u009d beyond coding regions, and thus may provide a new layer for understanding autoimmune or inflammatory diseases.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Li, Xin \",\n\"SHANGHAI INSTITUTES FOR BIOLOGICAL SCIS\",\n\"Statistical methods for decoding the genetic architecture of rare disease\",\n\"Jan19, 2021\",\n\"expired\",\n\"We seek to uncover the role of rare non-coding variation in gene regulation and complex traits. The mechanism how regulatory variants cause human disease is largely unknown. We plan to utilize modern machine learning techniques to train a trans-omics model, with the goal of better interpreting the function of the non-coding genome. \",\n\"Most verified disease-causing mutations have been limited to protein coding regions, with the function of non-coding variation remains largely unknown. The efficient interpretation of non-coding variants among human populations holds a substantial potential for furthering our understanding of disease etiology and finding new therapeutic targets. However, investigation of noncoding variation has been significantly barred by the lack of explicit genetic code, the requirement of extra-large sample sizes and deep whole genome sequencing. UDN and other rare disease cohorts with whole genome sequencing provides us with the unique opportunity to unveil a comprehensive landscape of the functional impact of rare non-coding variants.   In this study we set out to develop a new machine learning model to characterize the roles of many types of rare non-coding variants across multiple regulatory cascades. Utilizing solved and unresolved cases in the cohort, we seek to train the model over a variety of genomic and epigenomic features, aiming at identifying potential rare non-coding variants that are more prone to regulatory modification and pathogenesis. A successful outcome of this study will build a detailed catalog of all types of rare non-coding variants and their regulatory effects, forming the basis for prioritizing rare variants for further phenotypic characterization. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Li, Yun \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"Nov22, 2022\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease. Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data.\",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry. Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community. However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Linial, Michal \",\n\"HEBREW UNIVERSITY OF JERUSALEM\",\n\"Polygenic Risk Prediction for complex disease by gene-based association studies\",\n\"Sep16, 2021\",\n\"closed\",\n\"We will search for genes and protein mutations that cause depression and other diseases (e.g. chronic pain, risk of addiction, or chronic diseases such as diabetes) using the genetic data in the VHA omics dataset. We will look not just at changes in the genome, but also if these mutations affect the created proteins significantly, as a new way of finding more important factors.\",\n\"Our lab focuses on genomic and proteomic aspects of biology. We will use the VA dataset to search for genes and their associated protein level mutations that associate with diseases and other physiological troubles - we will focus on the target of the original study (depression), but will also examine other included covariates if available (e.g. chronic pain, schizophrenia, bipolar disorder) using the genetic/omics data in the VHA omics dataset. We aim to combine multiple biological risk factors to find novel risk factors for complex diseases. Use will be entirely computational, to find associations between the recorded data and recorded diseases.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Linnstaedt, Sarah \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"Assessing genetic overlap and protective factors for common co-morbid posttraumatic sequelae\",\n\"Jun10, 2020\",\n\"closed\",\n\"Results from the Million Veterans Program will be used to assess the genetic overlap and protective factors influencing vulnerability to posttraumatic stress, chronic pain, and depression using polygenic risk scores in a civilian trauma cohort.\",\n\"Clinical comorbidity between different psychiatric symptoms following trauma is common. The purpose of our proposed research is to investigate the genetic overlap between vulnerability to posttraumatic stress (and relevant symptom clusters), chronic pain, and depression using polygenic risk scores. Polygenic risk scores will be generated using PRSice polygenic risk score software, summary statistics from the Million Veteran Program and other large genome-wide association studies, and individual genetic information from our emergency department cohort. Statistical analyses will be used to determine how well polygenic risk scores predict outcomes and whether protective factors (e.g. social support, socioeconomic status) offset this genetic risk. Datasets will not be combined but will be used in addition to other GWAS summary information.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Liu, Dajiang \",\n\"PENNSYLVANIA STATE UNIV HERSHEY MED CTR\",\n\"method development for statistical fine-mapping analysis using multiple traits. \",\n\"Feb02, 2023\",\n\"approved\",\n\"We will develop software tools to identify genetic variants that elevate the risk for human diseases. Identifying causal variants is of utmost importance to understand the underlying disease biology and gain insights on how to develop therapeutics to reverse the effects of risk variants, which can lead to treatment. Here, we expect to develop methods to jointly analyze multiple health outcomes, and more accurately identify causal variants.  \",\n\"GWAS to date have been extremely successful for identifying genetic regions associated with various complex traits. Yet, due to extensive linkage disequilibrium, the causal variants from most GWAS loci remain unclear. Statistical fine mapping methods are often used to pinpoint causal variants and assign probability to each variant measuring how likely it is causal. Current statistical fine. mapping methods have limited success. Due to shared genetic basis between traits, borrowing strength from multiple traits will improve the estimates of genetic effects and in turn improve fine mapping. For this project, we seek to develop useful methods to improve fine-mapping combining summary statistics from multiple traits. We will release results and distribute software implementing the proposed methods. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Liu, Jin \",\n\"NATIONAL UNIVERSITY OF SINGAPORE\",\n\"Statistical methods for genetic studies in health-related outcomes\",\n\"Jan28, 2019\",\n\"closed\",\n\"Genetic studies have been developed rapidly over past twenty years, including genome-wide association studies (GWAS). Rigorous statistical methods will be developed to modulate the relationships among multiple GWAS and a scalable algorithm will be developed to handle these data sets. We plan to use applied data sets in dbGaP to demonstrate the usefulness of our methods. \",\n\"Genetic studies have been developed rapidly over past twenty years, including genome-wide association studies (GWAS).  1. Objective: A rigorous statistical method will be developed to modulate the relationships among multiple health-related traits and a scalable algorithm will be developed to handle admixture of Hispanic population to boost findings of risk variants or factors for health-related traits, and lung disease trait. 2. Study design: We plan to use machine learning techniques following statistical principles to explore the unique property of these data sets, which will potentially produce solid results for interpretation. A statistically rigorous method will be developed to handle large genomic data as current state-of-art for these data set is lack of scalability. We will also explore the integrative analysis using this data set with other external genomic data set to boost our findings. 3. Analysis plan: We plan to analyze multiple health-related traits and lung disease traits in the datasets requested. 4. The analysis of this data set and methodology development will be conducted in my lab.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"LOGUE, MARK \",\n\"VA BOSTON HEALTH CARE SYSTEM\",\n\"Early Cognitive Impairment as a Function of Alzheimer\\u00e2\\u0080\\u0099s Disease Genes and Trauma\",\n\"Sep09, 2021\",\n\"approved\",\n\"In this project we will examine Alzheimer\\u00e2\\u0080\\u0099s disease (AD) genetic risk and its relationship to cognitive impairment in middle and old age to determine the extent to which effects of AD genetic risk are observable in advance of the onset of dementia. We will also evaluate whether AD genetic risk influences PTSD symptomatology in aging veterans.  Finally, will look for evidence that combat trauma and traumatic brain injury (TBI) exacerbate the effects of AD genetic risk on AD. Early detection of AD risk has been noted as key to the identification of effective treatments for AD. Hence, this project has the potential to lead to better informed and more effective AD interventions. By examining how genetic factors interact with TBI and trauma we hope to pave the way for the development of diagnostic algorithms specifically tailored to a veteran population designed to identify those at risk for dementia\",\n\"We desire to obtain GWAS summary statistics and to  examine  results for specific variants and compute polygenic scores based on GWAS results based on GWAS summary statistics to examine the applicability of our GWAS results to other traits. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Loos, Ruth \",\n\"ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI\",\n\"The genetic susceptibility to type 2 diabetes across diverse ancestries\",\n\"Jul13, 2020\",\n\"closed\",\n\"The Mount Sinai Biobank, BioMe, is an electronic health record linked biorepository composed of over 60,000 individuals who seek care through the Mount Sinai Medical Center. A majority of the BioMe participants are of non-European ancestry, making us well situated to study traditionally under-represented populations in genetics. The UK Biobank is a large biobank in the United Kingdom that is composed predominantly of individuals of European ancestry, and contains ~5% non-European ancestry individuals. We are requesting access to the Million Veteran Program type 2 diabetes GWAS summary statistics for African Americans, Europeans, Hispanic Americans and the multi-ethnic cohort. Using the summary statistics, we will generate type 2 diabetes polygenic risk scores in an ancestry specific manner in BioMe and the UK Biobank, and use them to test the association for diabetes and related traits.\",\n\"The majority of genome-wide association studies (GWAS) have been done in individuals of European ancestry, hindering our ability to create polygenic risk scores (PRS) in other ancestries. We are requesting access to the African American, European, Hispanic American and multi-ethnic type 2 diabetes (T2D) GWAS summary statistics from the Million Veteran Program (MVP) to generate ancestry-specific PRSs and examine their role in the Mount Sinai Biobank, BioMe, and the UK Biobank (UKBB). BioMe is a multi-ethnic electronic medical record linked biorepository of over 60,000 individuals. Within BioMe, we have > 32,000 whole exome sequencing samples, the majority of which are from individuals of African American and Hispanic ancestry. The UKBB is a large biobank in the United Kingdom that is composed predominantly of individuals of European ancestry and contains ~5% non-European ancestry individuals. Within each biobank, our objectives are:  1. To generate ancestry specific type 2 diabetes PRSs and test their association with T2D in each each ancestry  2. To examine the association of rare variants in known genes known to affect T2D risk and the risk of T2D in each ancestry 3. To investigate the interaction between the PRSs and rare variants risk of T2D and  4. Repeat analyses in 1-2 for diabetes related outcomes, such as retinopathy, neuropathy and nephropathy, as well as diabetes related biomarkers, such as glucose and hemoglobin A1C. 5. Repeat analyses 1-4 using the multi-ethnic summary statistics for comparison with the ancestry specific results. We are not requesting individual level data. We do not expect to collaborate with other institutions.   NOTE: We are only requesting summery statistics related to this paper: Discovery of 318 new risk loci for type 2 diabetes and related vascular outcomes among 1.4 million participants in a multi-ancestry meta-analysis. Vujkovic M, Keaton JM, Lynch JA, Miller DR, Zhou J, Tcheandjieu C, Huffman JE, Assimes TL, Lorenz K, Zhu X, Hilliard AT, Judy RL, Huang J, Lee KM, Klarin D, Pyarajan S, Danesh J, Melander O, Rasheed A, Mallick NH, Hameed S, Qureshi IH, Afzal MN, Malik U, Jalal A, Abbas S, Sheng X, Gao L, Kaestner KH, Susztak K, Sun YV, DuVall SL, Cho K, Lee JS, Gaziano JM, Phillips LS, Meigs JB, Reaven PD, Wilson PW, Edwards TL, Rader DJ, Damrauer SM, O'Donnell CJ, Tsao PS; HPAP Consortium; Regeneron Genetics Center; VA Million Veteran Program, Chang KM, Voight BF, Saleheen D.  We are not requesting individual level data. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Lu, Yingchang \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Cardiometabolic trait associations for variants associated with total body fat percentage and fat-free mass index.\",\n\"Oct09, 2019\",\n\"approved\",\n\"In order to understand the physiology underlying body fat mass and body fat-free mass regulations, we conducted a genetic study on body fat percentage and fat-free mass index among >400,000 individuals of European ancestry.  We identify 1,137 genetic variants associated with these two traits. We intend to understand the effects of these 1,137 variants on cardiometabolic traits and diseases.\",\n\"Large-scale genome-wide association study (GWAS) of body mass index (BMI) have recently identified 716 independent variants [PMID: 30124842], which together explained 5% of the variance of BMI. Although BMI is a widely used indicator of overweight and obesity, it does not distinguish body fat mass from lean mass. In order to identify novel loci and gain a deeper understanding of genetic regulations of body adiposity and lean mass, we conducted a genome and exome-wide association studies of both body fat percentage and fat-free mass index among >400,000 individuals of European ancestry. In total, we identified 1,137 independent variants associated with these two traits at P < 5E-9. We identified 6 clusters among the 1,137 identified variants, based on their association patterns with 5 standard anthropometric traits, and have found that these clusters show discordant genetic associations with various cardio-metabolic outcomes and continuous traits. We would therefore like to request association results of cardiometabolic traits including lipids and other traits for 1,137 variants analyzed on MVP participants. We are not going to combined requested data with other data. This project will not create any additional risks to participants.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Luo, Yiming \",\n\"COLUMBIA UNIVERSITY HEALTH SCIENCES\",\n\"Mendelian randomization of autoimmune and cardiovascular diseases\",\n\"Mar14, 2023\",\n\"approved\",\n\"Both autoimmune and cardiovascular diseases are significant public health burden. This study is to investigate their casual relationship and shared susceptibility using population genetic data. \",\n\"Both autoimmune and cardiovascular diseases are significant public health burden. Recent studies have suggested that inflammation due to immune dysregulation is a causal factor of cardiovascular diseases. This study is to perform Mendelian randomization and other post-GWAS analyses (such as genetic correlation analysis, colocalization, cross-phenotypic meta-analysis) among autoimmune and cardiovascular diseases. This study will only use summary statistics but not individual-level genotype data. All the analyses will be performed in password-protected desktops/laptops and only study investigators can access the data. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Ma, Jiantao \",\n\"TUFTS UNIVERSITY BOSTON\",\n\"Causal omics biomarkers of nonalcoholic fatty liver disease\",\n\"Jan25, 2023\",\n\"approved\",\n\"Nonalcoholic fatty liver disease (NAFLD) is the most common chronic liver disease and a significant risk factor of heart disease. Dietary interventions are crucial to the prevention and treatment of NAFLD. Early detection of NAFLD will allow us to identify individuals at high risk of NAFLD; therefore, we can design effective dietary interventions to prevent the development of NAFLD and NAFLD-associated diseases. We will use information derived from genetic association studies of blood molecules (e.g., genes in white cells) and NAFLD to identify causal biomarkers of NAFLD, which may be used as biomarkers for early identification of NAFLD.  \",\n\"Description: We propose to examine the potential causal associations of omics biomarkers with nonalcoholic fatty liver disease (NAFLD) using Mendelian randomization analysis and colocalization analysis. Background: NAFLD is a common chronic liver disease, which is projected to be the first indication of liver transplantation in 2030. NAFLD is also tightly associated with cardiometabolic diseases and risk factors such as obesity and type 2 diabetes; therefore, NAFLD has been called as the liver manifestation of metabolic syndrome. The prevalence of NAFLD is approximately 25% to 30% in the US. Healthy diet is an important lifestyle factor for NAFLD prevention and among the first line treatment of NAFLD. Early detection of NAFLD is crucial to identify individuals at high risk of comorbidities of NAFLD and allow targeted lifestyle medication such as dietary interventions. A large number of omics biomarkers, e.g., transcriptomic and proteomic biomarkers, have been associated with NAFLD. Recently, a large genome-wide association (GWAS), conducted in 218,595 discovery samples and validated in 108,471 individuals, has identified 77 NAFLD-associated genomic loci. Using the summary results from this NAFLD GWAS and GWAS of omics biomarkers, i.e., quantitative trait loci, we can test which omics biomarkers are likely causal to NAFLD. Such analysis will facilitate the identification of omics biomarkers that can be used for early detection of NAFLD.  Aim: The primary objective of this study is to identify potential causal omics biomarkers such as transcriptomic and proteomic biomarkers of NAFLD.  Method: We will fulfill our objective by analyzing existing expression quantitative loci (eQTLs) and protein quantitative loci (pQTLs) in publicly available datasets (e.g., the MR base; https://www.mrbase.org/) and the summary results of the recent NAFLD GWAS (available through dbGAP under accession number phs001672.v7.p1). We will conduct the Mendelian randomization (MR) analysis, implemented using R SampleMR package, and the colocalization analysis, implemented using R HyPrColoc package. In MR analysis, we will perform the primary analysis using the inverse variance weighted (IVW) method and sensitivity analysis using the MR-Egger method. We will test for potential horizontal pleiotropy by examining the MR-Egger intercept P value. For colocalization analysis, we will use a deterministic Bayesian algorithm to calculate the posterior probability of full colocalization (PPFC). Default prior configuration parameters (prior.1=1e-4 and prior.c=0.02) and threshold of 0.7 for PPFC will be used. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Mackey, Scott \",\n\"UNIVERSITY OF VERMONT &amp; ST AGRIC COLLEGE\",\n\"Polygenic Risk Scores for Nicotine Dependence and Problematic Alcohol Use\",\n\"Feb17, 2021\",\n\"approved\",\n\"Genome wide association studies (GWAS) identify genetic variants associated with a trait such as problematic substance use. We will use two GWAS from dbGAP which targeted nicotine dependence and problematic alcohol use to generate individualized genetic risk scores for these traits in large cohort samples. We will examine whether these risk scores are related to behavior (e.g. age of onset and frequency of substance use, and measures of impulsivity) or brain features detected using MRI, including measures of brain structure (e.g. cortical and subcortical grey matter volume and white matter tract integrity) and function. \",\n\"Research Use Statement: Polygenic risk scores (PRS) derived from GWAS are a powerful tool to examine the genetic liability of a specific trait in a sample not included the GWAS sample. Our aim is to investigate the genetic liability of nicotine and alcohol use in population-based cohorts in relation to behavioral measures and intermediate brain phenotypes associated with problematic alcohol use or nicotine dependence. PRS will be create using the PLINK 2 software based on the results of the Genome Wide Association Studies (GWAS) for nicotine dependence (PMID 33144568 - phs001532.v2.p1) and problematic alcohol use (PMID 32451486 - phs001672.v3.p). These PRS will be regressed onto behavioral phenotypes (e.g. substance use measures including age of onset and frequency of use, and impulsivity measures) and onto MRI brain measures including structure (cortical and subcortical volume measures and white matter diffusivity) and function. This project does not require individual-level data from the mentioned studies and will be consistent with their data use limitations, using the data only by the terms of the model Data Use Certification onto research, and not for profit organizations. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Maher, Brion \",\n\"JOHNS HOPKINS UNIVERSITY\",\n\"The Role of Polygenic Alcohol Risk in Opioid Use in Multiple Baltimore Cohorts\",\n\"Dec21, 2020\",\n\"approved\",\n\"Opioid misuse is associated with serious negative consequences, including increased risk for abuse or dependence, compromised physical and mental health, emergency room visits, and overdose deaths. To date, targeting physician overprescribing has been the predominant public health and policy approach to curtailing the prescription opioid epidemic (Compton et al., 2019); however, opioid misuse is a complex phenomenon and effective prevention of this problem requires the consideration of the multiple pathways through which opioid misuse occurs. One individual specific factor that may influence the likelihood of misusing opioids is genetic predispositions. Indeed, it is well established that opioid misuse is genetically influenced, with heritability estimates ranging from .49-.76 (Sun et al., 2019). Given genetic overlap that has been observed across substance use disorders, it is possible that the genetics associated with problematic alcohol use predicts opioid misuse; however, there has been little research conducted in this area. Accordingly, the present study sought to examine whether polygenic risk for problematic alcohol use predicts opioid misuse in two diverse samples. This work has the potential to aid in risk stratification and the identification of at-risk individuals for screening and more effective preventions and treatments for opioid misuse. \",\n\"This research proposal aims to identify whether the genetics of problematic substance use predicts polysubstance use in 2 samples of predominantly low-income, urban African Americans. Based on summary statistics from the MVP (Levey et al., 2020), we plan on creating polygenic risk scores (PRS) for problematic alcohol, tobacco and other drug use using several approaches (e.g., pruning/thresholding and Bayesian methods). The first sample includes three cohorts of participants that were originally recruited in first grade as part of an elementary school-based intervention and followed until adulthood (~ age 30) (Ialongo et al., 1999). We plan to conduct analyses examining whether problematic alcohol use PRS predicts trajectories of heroin use and non-prescription narcotic use (e.g., Oxycodone), lifetime heroin and non-prescription narcotic use, and opioid use disorders (OUD). For the trajectory analyses, we plan on using person-centered approaches (e.g., growth mixture modeling). To examine whether problematic alcohol use PRS predicts lifetime opioid use and OUD status, we will conduct logistic regression analyses. The second sample includes individuals that were recruited to understand the impact of injection drug use on HIV course (Vlahov et al., 1991). We plan on examining whether PRS for problematic alcohol use predict frequency of injection drug use over time. The analytic plan will involve conducting generalized estimation equation models. We will not conduct analyses that are outside of the proposed data use agreement. The analyses conducted will only involve JHU investigators, including Drs. Brion Maher, Nicholas Ialongo, and Jill Rabinowitz.   We are requesting access to phs001672.v3.p1 and phs001532.v2.p1.  We are requesting a continuation of this project to complete our analyses. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"MANICHAIKUL, ANI \",\n\"UNIVERSITY OF VIRGINIA\",\n\"Systematic integration of multi-omics data for the study of coronary artery disease and subclinical atherosclerosis\",\n\"Feb24, 2023\",\n\"approved\",\n\"Large scale genetics studies have identified numerous genetic variants associated with disease. The goal of our project is to combine prior genetic findings with new molecular data in order to identify the molecular mechanisms of disease.\",\n\"Coronary artery disease (CAD) is a leading cause of death and disability worldwide. Large-scale genome-wide association studies (GWAS) of CAD from international consortia, including CARDIoGRAM, C4D and the UK Biobank have identified over 150 CAD-associated genetic loci, but the biological mechanisms and pathways implicated by GWAS are still not fully understood. Multi-omics integration combining GWAS results with molecular data provides a useful avenue to identify biomarkers and elucidate potential causal molecular mechanisms of the disease. Our group has access to molecular \\u00e2\\u0080\\u0098omics data through our work in the NHLBI Trans-omics for Precision Medicine (TOPMed) program. These data include transcriptomics, proteomics, metabolomics and methylomics on circulating samples from the population-based Multi-Ethnic Study of Atherosclerosis [MESA]. Currently, we are applying for MVP and CHARGE GWAS summary statistics from dbGaP with the goal of integrating those GWAS summary data with molecular QTL resources and identifying candidate molecular targets downstream of the GWAS signals. Our work will help to elucidate biological mechanisms of cardiovascular disease traits previously studied by GWAS.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Manning, Alisa \",\n\"BROAD INSTITUTE, INC.\",\n\"Exploration and use of MVP Summary Statistics for diabetes genetics research studies\",\n\"Feb14, 2022\",\n\"approved\",\n\"The purpose of this application is to explore the MVP summary statistics available in dbGAP and extract MVP summary statistics for various regions and various traits to use in on-going diabetes genetics research studies. We generally intend to perform co-localization analyses with MVP summary statistics and our own genetic summary statistics.\",\n\"The purpose of this application is to explore the MVP summary statistics available in dbGAP and extract MVP summary statistics for various regions and various traits to use in on-going diabetes genetics research studies. We generally intend to perform co-localization analyses with MVP summary statistics and our own genetic summary statistics. We will perform these analyses using cloud computing (the BioData Catalyst platform) and will not redistribute any summary statistics.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Manning, Alisa \",\n\"BROAD INSTITUTE, INC.\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"Nov07, 2022\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease.  Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data.\",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry.  Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community.   However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets.  External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Martin, Hilary \",\n\"SANGER INSTITUTE\",\n\"The genetic architecture and genomic prediction of serum lipid levels and type 2 diabetes in diverse populations\",\n\"Apr08, 2020\",\n\"approved\",\n\"This research will study the link between a person\\u00e2\\u0080\\u0099s genetic makeup and their lifelong health in South Asian individuals, and compare it with better studied European individuals. By using the large-scale lipid GWAS in MVP, it will contribute to better ways to predicting serum lipid levels and risk of diseases including cardiovascular diseases for which lipids (such as LDL cholesterol) are important risk factors. We plan to use the information gathered from a South Asian cohort to develop polygenic risk scores that work for British Bangladeshi and British Pakistani people, and compare the performance of similar scores constructed for European subjects. This will contribute to better understanding of the genetic differences between the two major populations, proper characterisation of polygenic score portability for various traits, and improvements in health through better and earlier prediction of disease.\",\n\"Large-scale genome-wide association studies (GWASs) have identified numerous genetic variants that are associated with human common complex traits. Polygenic risk scores (PRSs), which are constructed using GWAS results, have been shown to be successful in identifying individuals at high disease risk. Most of the GWASs and genomic prediction studies have been conducted in European samples, and we plan to evaluate the extent to which PRSs derived from these GWASs are useful for predicting complex traits and diseases in a South Asian cohort (East London Genes and Health; ELGH). We will compare PRSs for common diseases (such as T2D, asthma, and coronary atherosclerosis), as well as for quantitative traits (such as serum lipid levels, height, weight, and BMI) in ELGH versus a European cohort. In order to construct powerful polygenic scores for lipids, we are applying for the access to the summary statistics of lipid GWAS performed in the Million Veteran Program (MVP), which has so-far the largest sample size (around 300,000). We are also applying for the access to the summary statistics of the T2D GWAS meta-analysis which includes the MVP cohort. We will construct PRSs for total cholesterol, LDL cholesterol, HDL cholesterol, triglycerides, and T2D in ELGH using various methods, and evaluate their accuracy in this population and in various subgroups thereof.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"McCarthy, Mark \",\n\"UNIVERSITY OF OXFORD\",\n\"Deconstructing mechanisms involved in type 2 diabetes predisposition using trait association pattern mapping\",\n\"Nov29, 2018\",\n\"closed\",\n\"We lead international efforts to use genetics to understand the processes that are involved in the development of type 2 diabetes. One very useful strategy for doing this is to take the DNA sequence differences that we have shown to be associated with type 2 diabetes risk, and examine the effects of those same DNA differences on other diabetes-related measurements such as body mass index, and lipid levels. These analyses are most powerful when the data sets are large and it is for this reason that we request access to the recent data made available by the lipid analysis in the Millions Veterans Project. \",\n\"We are interested in what leads to type 2 diabetes (T2D) and related conditions. We have led international efforts that have identified over 400 type 2 diabetes (T2D) risk loci. Most map to non-coding sequence and mechanisms of action are often unclear. One strategy for defining the physiological processes through which these variants act is to examine their patterns of association to other diabetes-related metabolic and anthropometric traits. Over the past years, we have shown that, by integrating information on quantitative trait associations and tissue-specific regulatory annotation, genetic variants influencing type 2 diabetes (T2D) predisposition can be characterised in terms of the pathophysiological processes through which they operate. This allows a deconstruction of T2D pathophysiology that addresses phenotypic and clinical heterogeneity, promotes mechanistic insights, and reveals novel translational opportunities. We have extended this effort to the T2D-risk variants discovered through our expanded meta-analyses (both European-specific and trans-ethnic) in over a million individuals. For this, we are seeking access to summary statistics for SNPs associated with lipid traits.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"McVean, Gil \",\n\"GENOMICS LIMITED\",\n\"Robustness of genetic association and polygenic risk scores to non-European ancestries\",\n\"Jul05, 2021\",\n\"approved\",\n\"Genetic data has the potential to transform health care, by better predicting diseases, understanding and recommending treatment options and better prevention. This is particularly critical for cardiovascular diseases, for which lifestyle alterations and drugs (e.g. statins) are available for at-risk individuals.  However, much of the data underpinning this revolution have been generated in individuals of European descent. These data cannot be readily used in non-European populations, which limits broad adoption and widens an already existing gap in access to modern genomic technologies. With this project, we are interested in assessing how these genetic predictors can be used in individuals of diverse ancestry. We want to improve the design of these tools so that they become more precise, and usable by a wider range of individuals.\",\n\"There is a growing sense in the genomics community that genetic-based prediction can provide effective information in health care. In particular, polygenic risk scores (PRS) have gathered a lot of attention in the scientific community. PRS are designed to aggregate information from a large number of genetic variants, which, once combined, provide effective predictions on a wide range of traits. We are particularly interested in developing these PRS for diseases that are actionable and/or have ongoing screening programmes which makes diabetes, eye diseases, osteoporosis/fractures, and cardiovascular diseases (lipids studies, hypertensions, myocardial infarction) major foci of this application.   However, opportunities for improved care also exist for a broader range of human traits and diseases, for example, autoimmune conditions for which wrong diagnosis or lack of diagnosis severely impact a large number of individuals. Genome-wide association (GWA) studies provide the data supporting the design of these PRS. However, PRS typically rely on keeping the pattern of linkage disequilibrium (LD) constant. Because the majority of genetic studies have been performed in individuals of European descent, these PRS are not readily exportable to other populations, which limits their widespread adoption.  Given these limitations, we are keen to explore the robustness of PRS and association summary statistics across European and non-European populations. For the common diseases for which genetic prediction is a relevant and practical opportunity, this work will be performed in three stages: We will initially refine the construction of PRS by utilising summary statistics data from studies performed in populations of European descent. We will download data from genome-wide association (GWA) studies with individual-level genotype in European and non-European cohorts and generate PRS data, comparing association statistics between European and non-European data. We will explore alternative PRS generation techniques that leverage individual-level data, such as machine learning techniques (LASSO, neural networks) and understand how these prediction algorithms are impacted by varying ancestries.  This analysis will help us assess the robustness of PRS and improve their design in order to be effective in a broader set of populations. The design of these PRS will combine the novel data accessed as part of this project with the summary statistics that we have previously aggregated for a range of genetic studies. Where permissible in accordance with data use terms, we also combine individual-level data (both from dbGaP and other sources) to understand the interplay between genetics and phenotype with more granularity.  We intend to publish various aspects of our research and share results accordingly. We have already presented some of our prediction work at international conferences (e.g. ASHG 2019) and our paper quantifying the role of genetics for the prediction of cardiovascular disease is currently in press (Validation of an Integrated Risk Tool, Including Polygenic Risk Score, for Atherosclerotic Cardiovascular Disease in Multiple Ethnicities and Ancestries in The American Journal of Cardiology). We will follow a similar process across diseases, for conference and journal submissions, following the pace of our research work.  Only the collaborators named in our dbGaP application (i.e. our staff) will have access to our database. We will absolutely respect all data use certifications and data use limitations when conducting our analyses. In the case of disease-specific limitations, we will limit our analyses accordingly using technical and administrative means.  The IRB certificate is expected to be renewed in August, and the updated certificate will be uploaded as soon as it has been obtained.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"McVean, Gil \",\n\"GENOMICS LIMITED\",\n\"Integrative analysis of genetic association data for individual genome interpretation and translational biological research - part 2\",\n\"Sep19, 2021\",\n\"approved\",\n\"We will combine information from many different studies in dbGaP, and many other studies in our own database, to try to learn even more about the biology of health and sickness than the individual studies can alone. We will develop new statistical methods to help do this. We will compare our results with other scientific studies in our database in many different ways, for example by looking at which genes are turned on and off in different cells, to better understand how DNA varies in different people. Ultimately we hope that our work will improve two things about modern medicine. First that drug companies will be better informed about choosing what to concentrate on when making new medicines. And secondly, that an individual person\\u00e2\\u0080\\u0099s DNA can be used to help in making a diagnosis, or choosing treatment options when they go to the doctor.\",\n\"There has been limited translation of important GWA study findings for complex diseases and other phenotypes into healthcare practice. We wish to use certain dbGaP datasets as part of our ongoing efforts to build and integrate a curated database of GWAS and related biological and phenotypic data. As a company, we hope to obtain scientific and commercial value by using insights from the database to develop proprietary analytical tools, and performing analyses of the database to better inform and improve healthcare eg. improvements to drug development and genomics-based precision medicine.   Our planned analyses will rely on systematic processing of dbGaP datasets into summary (non-individual level) statistics, prior to incorporation into our database. We will then interrogate the database to look for association between genetic markers and different combinations and summaries of phenotypic data. Where appropriate, we will perform a GWAS for each phenotype, and a phenotype\\u00c2\\u00ad-wide association study at each genetic variant. Using statistical methods we will summarize the evidence that each genetic variant/locus has an association with available phenotypes. The data generated will be integrated across the genome and phenotypes to build a model showing how genetic variation shapes human biology. We will also use the data to develop, refine, and optimise robust algorithms that form the basis of our analytical tools. As part of this, some of the datasets will be used to compute correlations between genetic markers, especially in an ancestry specific manner- an output typically referred to as linkage disequilibrium maps. These linkage disequilibrium maps can then be used to better conduct our analyses in a manner that properly accounts for variant correlations across each genetic ancestry group. Furthermore, dimensionality reduction techniques such as principal component analysis may be applied to the data in order to summarise the genetic space into a lower dimension map, a process typically used for genetic ancestry assignment. We expect that our tools will be utilised in research laboratories, clinics, and elsewhere. Our tools, used alone or with the database, could be used to generate precision medicine analysis for many traits, aiding interpretation of extant clinical diagnostic information.  We will share results from various aspects of our research work e.g. via publicly accessible repositories, following the pace of our analyses. Our hope is that this will enable the scientific community ready access to foundational analyses that they can draw insight from and integrate into the body of work of the wider scientific community.  Only the collaborators named in our dbGaP application (ie. our staff) will have access to our database. We will absolutely respect all data use certifications and data use limitations when conducting our analyses. In the case of disease-specific limitations, we will limit our analyses accordingly using technical and administrative means.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"McVean, Gil \",\n\"GENOMICS LIMITED\",\n\"Robustness of genetic association and polygenic risk scores to non-European ancestries (cancer)\",\n\"Oct29, 2019\",\n\"closed\",\n\"Genetic data has the potential to transform health care, by better predicting diseases, understanding and recommending treatment options and better prevention. This is particularly critical for cancers, which greatly benefit from early detection, and for cardiovascular diseases, for which lifestyle alterations and drugs (e.g. statins) are available for at risk individuals. However, much of the data  underpinning this revolution have been generated in individuals of European descent. These data cannot be readily used in non-European populations, which limits broad adoption and widens an already existing gap in access to modern genomic technologies. With this project we are interested in assessing how these genetic predictors can be used in individuals of diverse ancestry. We want to improve the design of these tools so that they become more precise, and usable by a wider range of individuals.\",\n\"There is a growing consensus in the genomics community that genetic-based prediction can provide effective information in health care. In particular, polygenic risk scores (PRS) have gathered a lot of attention in the scientific community. PRS are designed to aggregate information from a large number of genetic variants, which, once combined, provide effective predictions for a wide range of traits. We are particularly interested in developing these PRS for diseases that are actionable and/or have ongoing screening programmes, which makes common cancers (breast, prostate, colorectal, ovarian, lung) a major focus of our work.   Genome-wide association (GWA) studies provide the data supporting the design of these PRS. However, PRS typically rely on keeping the pattern of linkage disequilibrium (LD) constant. Because the majority of genetic studies have been performed in individuals of European descent, these PRS are not readily exportable to other populations, which limits their widespread adoption.  Given these limitations, we are keen to explore the robustness of PRS and association summary statistics across European and non-European populations. For the common diseases for which genetic prediction is a relevant and practical opportunity, this work will be performed in three stages: We will initially refine the construction of PRS by utilising summary statistics data from studies performed in populations of European descent. We will download data from genome-wide association (GWA) studies with individual-level genotypes in European and non-European cohorts and generate PRS data, comparing association statistics between European and non-European data. We will explore alternative PRS generation techniques that leverage individual-level data, such as machine learning techniques (LASSO, neural networks) and understand how these prediction algorithms are impacted by varying ancestries.  This analysis will help us assess the robustness of PRS and improve their design in order to be effective in a broader set of populations. The design of these PRS will combine the novel data accessed as part of this project with the summary statistics that we have previously aggregated for a range of genetic studies. Where permissible in accordance with data use terms, we also combine individual-level data (both from dbGaP and other sources) to understand the interplay between genetics and phenotype with more granularity.  We intend to publish various aspects of our research and share results accordingly. We have already presented some of our prediction work at international conferences (e.g. ASHG 2019, 2021) and we are in the process of organising our work for submission in peer-reviewed publications. We will continue to follow a similar process across cancer types, for conference and journal submissions, following the pace of our research work.  Only the collaborators named in our dbGaP application (i.e. our staff) will have access to our database. We will absolutely respect all data use certifications and data use limitations when conducting our analyses. In the case of disease-specific limitations, we will limit our analyses accordingly using technical and administrative means.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"McVean, Gil \",\n\"GENOMICS LIMITED\",\n\"Integrative analysis of genetic association data for individual genome interpretation and translational biological research\",\n\"May20, 2019\",\n\"closed\",\n\"We will combine information from many different studies in dbGaP, and many other studies in our own database, to learn even more about the biology of health and sickness than can be derived from any individual study. We will develop new statistical methods to help do this. We will compare our results with other scientific studies in our database in many different ways, for example by looking at which genes are turned on and off in different cells, to better understand how DNA varies in different people. Ultimately we hope that our work will improve two things about modern medicine. Firstly, we hope that drug companies will be better informed about choosing what to concentrate on when making new medicines. And secondly, that an individual person\\u00e2\\u0080\\u0099s DNA can be used to help in making a diagnosis, or choosing treatment options when they go to the doctor\",\n\"There has been limited translation of important GWA study findings for complex diseases and other phenotypes into healthcare practice. We wish to use certain dbGaP datasets as part of our ongoing efforts to build and integrate a curated database of GWAS and related biological and phenotypic data. As a company, we hope to obtain scientific and commercial value by using insights from the database to develop proprietary analytical tools, and performing analyses of the database to better inform and improve healthcare eg. improvements to drug development and genomics-based precision medicine.   Our planned analyses will rely on systematic processing of dbGaP datasets into summary (non-individual level) statistics, prior to incorporation into our database. We will then interrogate the database to look for association between genetic markers and different combinations and summaries of phenotypic data. Where appropriate, we will perform a GWAS for each phenotype, and a phenotype\\u00c2\\u00ad-wide association study at each genetic variant. Using statistical methods we will summarize the evidence that each genetic variant/locus has an association with available phenotypes. The data generated will be integrated across the genome and phenotypes to build a model showing how genetic variation shapes human biology. We will also use the data to develop, refine, and optimise robust algorithms that form the basis of our analytical tools. As part of this, some of the datasets will be used to compute correlations between genetic markers, especially in an ancestry specific manner- an output typically referred to as linkage disequilibrium maps. These linkage disequilibrium maps can then be used to better conduct our analyses in a manner that properly accounts for variant correlations across each genetic ancestry group. Furthermore, dimensionality reduction techniques such as principal component analysis may be applied to the data in order to summarise the genetic space into a lower dimension map, a process typically used for genetic ancestry assignment. We expect that our tools will be utilised in research laboratories, clinics, and elsewhere. Our tools, used alone or with the database, could be used to generate precision medicine analysis for many traits, aiding interpretation of extant clinical diagnostic information.  Only the collaborators named in our dbGaP application (ie. our staff) will have access to our database. We will absolutely respect all data use certifications and data use limitations when conducting our analyses. In the case of disease-specific limitations, we will limit our analyses accordingly using technical and administrative means. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Mehta, Divya \",\n\"QUEENSLAND UNIVERSITY OF TECHNOLOGY\",\n\"Multi-omics analysis to identify novel genes for PTSD\",\n\"Apr11, 2022\",\n\"expired\",\n\"Combining the MVP data with our in-house Australian veteran\\u00e2\\u0080\\u0099s data, we aim to identify novel genes associated with posttraumatic stress disorder (PTSD). This study leverages existing datasets and cutting-edge imputation-based methodologies to uncover robust genes and pathways associated with PTSD through an integrative multi-omics approach. Our team has previously used such a study design to identify novel genes for cancer and we now wish to implement our statistical pipeline on PTSD. These results will be novel and will be submitted to an international peer-review journal. For any publications arising from these analyses, the MVP team will be included as co-authors.\",\n\"Aim: The overarching aim of the proposed study is to develop a bioinformatics pipeline to identify novel PTSD genes through multi-omics integration of secondary and in-house omics data   Objectives:  Objective 1: Identify SNP-regulated genes and methylation sites using validated imputation-based methods - First, using the Million veteran project (MVP) GWAS summary results, we will conduct an imputation-based transcriptome-wide association study (TWAS) using S-PrediXcan (Barbeira et al., 2018) and FUSION (Gusev et al., 2016) to predict the genetically regulated genes that are differentially expressed between PTSD and controls. These genes will be validated by differential gene expression analysis using gene expression measured in our in-house clinically diagnosed Vietnam war combat veterans with and without PTSD (n=300), this cohort is also part of the PGC-PTSD combined cohort (Mehta et al., 2018). Next, using the MVP GWAS summary results and EstiMeth (Freytag et al., 2018) we will perform an imputation-based methylome-wide association study (MWAS) to predict the genetically regulated DNA methylation sites that are differentially methylated between individual with and without PTSD. Validation of these results will again be performed using the DNA methylation measurements from our in-house Vietnam war combat veterans with and without PTSD (Mehta et al., 2017). While the imputation-based approaches identify the genes/methylation sites associated with PTSD, they do not provide information about colocalization of GWAS SNPs and expression quantitative trait loci (eQTLs) or methylation quantitative trait loci (mQTLs). We conduct colocalization genetic analysis using COLOC (Giambartolomei et al., 2014) and EpiCOLOC (Zhou et al., 2020) to identify shared causal variants between GWAS SNPs and eQTLs/mQTLs. We will also conduct mendelian randomization to examine the causal influence of GWAS and colocalized eQTL/mQTL SNPs on PTSD. Lastly, characterisation through enrichment analysis such as tissue specific enrichment, enrichment in other major psychiatric disorders (depression, schizophrenia, bipolar, alcohol use disorder), and pathway analysis will also be conducted for the identified SNPs. Objective 2: Once we have identified functional SNPs that are associated with changes in PTSD gene expression and/or DNA methylation profiles in Objective 1, we will use these SNPs to develop a unique polygenic risk score. The developed model will be validated on Vietnam war veteran dataset (Mehta et al., 2018). Phenotypes will include PTSD and depression to validate if the PRS from PTSD (using only robust functional SNPs from MVP) explain variance in PTSD and/or depressive symptoms. The requested data will be used for health/medical/biomedical purposes, i.e., to identify PTSD-specific biomarkers that can provide further understanding of biological mechanisms underlying PTSD and develop PRS model to identify individuals at risk. The requested data will not be combined with other dbGaP dataset or other sources but rather the results from the independent analysis will be verified in measured gene expression and methylation data sets and tested for enrichment in the summary results of other psychiatric disorders. We will only work with de-identified data and will not attempt to identify any of the participants. Our proposed research does not include the study of population origins or ancestry. The genomic summary results will not be posted publicly. Any findings from the study will be published or otherwise broadly shared with the scientific community. For any publications arising from these analyses, the MVP team will be included as co-authors.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Melas, Philippe \",\n\"KAROLINSKA INSTITUTE\",\n\"Pharmacogenetic responses to a monoamine stabilizer (OSU6162) in individuals with alcohol use disorder\",\n\"Apr18, 2023\",\n\"approved\",\n\"Alcohol use disorders (AUD) affect approximately 5% of the world population and impose a major socio-economic burden on the global mental health. Thus, there is a pressing need to develop new medications and identify biomarkers that predict treatment response. Our project aims to enhance our understanding of the molecular and genetic biosignatures that predict treatment response to a novel compound in AUD treatment, called OSU6162. To this end, we use a cohort of AUD individuals who were treated with OSU6162 and calculate polygenic risk scores (PRS), which provide a measure of alcohol use-related disease risk due to their genes. Next, we evaluate whether PRS scores for AUD-related phenotypes correlate with treatment response to OSU6162. Examining whether PRS scores can be used to predict effectiveness in treatment response holds promise for the future since it will allow the prescription of medications to those patients who will have the highest likelihood of responding positively, commonly referred to as personalized medicine.\",\n\"Objectives of the proposed research: Researchers at our institution have previously published a randomized controlled trial evaluating the effects of a monoamine stabilizer (OSU6162) on reducing alcohol-related phenotypes in individuals with alcohol dependence (PMID: 26482910). I am the PI of a follow-up project that uses the same human cohort to conduct pharmacogenetic studies related to personalized medicine. Specifically, our objective is to calculate polygenic risk scores (PRS) for alcohol use-related phenotypes, including comorbid diagnoses (e.g., depression), and examine whether they correlate with therapeutic responses to OSU6162.  Study design: The current study design builds on a completed randomized controlled trial that followed a double blind, placebo-controlled design, where alcohol dependent participants (N=56) were randomized to receive OSU6162 or placebo (PMID: 26482910). The study was approved by the regional ethical review board in Stockholm and the Swedish Medical Products Agency, registered in the European Clinical Trials Database (EudraCT;2011-003133-34), monitored by Karolinska Trial Alliance and conducted in accordance with Good Clinical Practice and the Declaration of Helsinki.  Analysis plan, including the phenotypic characteristics that will be evaluated in association with genetic variants: We have collected genomic data from all study participants using the Illumina Infinium Global Screening Array-24 v2.0. Our PRS analyses, for which we request dbGAP access, will focus on phenotypes including (i) alcohol consumption/AUDIT-C, (ii) alcohol use disorder and (iii) problematic alcohol use, and (iv) depression. Additional PRS analyses, outside dbGAP, will be conducted for Drinks per week (PMID: 36477530). All PRS-related phenotypes will be analyzed independently in relation to OSU6162-response.  Explanation of how the proposed research is consistent with the data use limitations for the requested dataset(s): To calculate the abovementioned PRS scores, we request access to the Veterans Administration (VA) Million Veteran Program (MVP) Summary Results from Omics Studies (dbGAP study accession phs001672.v9.p1). Our proposed research is consistent with the data use limitations and in line with the already published GWAS studies on: (i) alcohol consumption/AUDIT-C and alcohol use disorder (PMID: 30940813), (ii) problematic alcohol use (PMID: 32451486), and (iii) depression (PMID: 34045744). More specifically, the use of the data will be limited to health/medical/biomedical purposes and will not include the study of population origins or ancestry. Moreover, there will be no public posting of genomic summary results, and controlled-access data will only be available to the applicant (Philippe Melas).   Brief description of any planned collaboration with researchers at other institutions, including the name of the collaborator(s) and their institution(s): No such collaborations are planned. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Menzel, Stephan \",\n\"KING'S COLLEGE LONDON\",\n\"Confirmation of genetic loci for HbF levels and stroke risk in sickle cell disease\",\n\"Sep18, 2020\",\n\"rejected\",\n\"Whilst sickle cell disease (SCD) is a well-recognised monogenic disorder, the individual patient experience can vary vastly. We believe a certain component of this is due to changes in other genes that influence the complex biology of SCD. We are investigating two aspects of SCD. Firstly, genetic factors that may predispose patients to stroke \\u00e2\\u0080\\u0093 a devastating complication that can affect up to 10% of children with SCD. By looking closely at those who develop a stroke a very young age, in combination with the data available publically, we hope to be able to develop new tests that will help to predict patients most at risk and also, by better understanding key mechanisms involved, develop new methods of treating or preventing stroke in this group. Secondly, we are investigating genetic factors influencing haemoglobin F levels. Higher haemoglobin F levels are known to alleviate disease severity in SCD. We have been researching this for some time and believe the additional data available here will help strengthen our investigations helping us to draw more confident conclusions.\",\n\"The wide phenotypic variation seen in sickle cell disease (SCD) no doubt has significant polygenic influences. Known modifiers of haemoglobin F and alpha-thalassaemia deletions convey a degree of this genetic variability, but do not account for its entirety.  Indeed, known modifiers of haemoglobin F do not even account for all the variability of haemoglobin F levels. Studies investigating such genetic factors are consistently undermined by sample size. At Kings College London, we have established a large database of patients with SCD in whom carefully curated phenotypic data is paired with SNParray data and also a smaller cohort in whom whole exome sequencing (WES) has been performed. We have two facets of investigation for which we believe the dbGaP data will be of great benefit. Firstly, we are investigating genetic factors associated with cerebrovascular disease in SCD. We have performed WES in a cohort of selected patients who have had a stroke at a very young age as we believe genetic influences are likely to be more prominent in this subset. The conclusions drawn from this small sample size will be strengthened by validation in the dbGaP datasets, with those from the SWITCH study providing positive cases and those from the HUSTLE study providing negative cases. Our second interest in the dbGaP dataset relates to common genetic variants controlling stroke risk and Haemoglobin F levels in sickle cell disease. Using our SNParray dataset, we have identified new loci for both traits that we wish to validate in other cohorts. The baseline phenotype data provided contains sufficient information to allow us to utilise the sequencing data for this purpose. Having read the DUC, we are confident that our proposal complies with the data usage limitations, and the anonymity of the participants will not be compromised. We hope this work will improve understanding of two key aspects of SCD, namely cerebrovascular disease and haemoglobin F regulation and provide important new biomarkers in both regards.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Mercader, Josep \",\n\"BROAD INSTITUTE, INC.\",\n\"Discovery and fine mapping of cardiometabolic traits in diverse populations\",\n\"Nov19, 2020\",\n\"approved\",\n\"Genome-wide association studies have been widely used to identify genetic regions of the genome that modify the risk for a disease. Most of the studies have been performed in European Ancestry participants, but studies in non-European ancestry have been limited. We are planning to extend these studies to diverse populations to better understand the pathophysiology of type 2 diabetes and related traits.\",\n\"We have performed genome-wide association studies in non-European populations. We would like to use additional datasets to replicate and fine-map the the signals identified in our discovery cohorts.  Objectives: The objective of this study is to replicate variants identified in other datasets from diverse ancestries. Study design: We will select variants identified in other studies and perform lookups of the association summary statistics from the MVP data. We will meta-analyze the MVP results with the summary statistics of our study results to assess significance. Analysis Plan: We have performed several meta-analyses of T2D in diverse ancestries. We will select variants with p value < 1e-6 and we will seek for replication in the corresponding ancestry if available within MVP T2D summary statistics. We will consider significant replication if p value is equal to 0.05/number of variants tested. We will consider genome-wide significance when the meta-analysis between the discovery and the replication reaches a p-value below 5e-8. We will perform the meta-analysis using Fixed Effects Invesrse Variance meta-analysis using the software METAL.  We do not plan to combine the requested datasets with other datasets outside of dbGaP. We plan to meta-analyze results obtained from dbGaP datasets with those obtained in our discovery cohorts, but we plan only to meta-analyze summary statistics, and never combine individual-level data. The proposed research is consistent with the Data Use Limitations. We will only use this dataset for biomedical or health research.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Miller, Clint \",\n\"UNIVERSITY OF VIRGINIA\",\n\"Dissecting the causal regulatory mechanisms of coronary artery disease \",\n\"Mar05, 2020\",\n\"approved\",\n\"Coronary artery disease and heart attacks are the leading causes of death globally. While we have identified a number of modifiable risk factors in the environment that are responsible for heart disease, there are also non-modifiable risk factors that can be identified in our genes. Based on family studies and large-scale population studies we have uncovered genetic risk factors that may contribute to our risk for developing heart disease. These subtle changes in genes may interact with our environment and lead to subtle but substantial changes in risk over our lifetime. Studies that enroll organ donors have been absolutely critical to our understanding of how these changes alter physiological processes in our body and ultimately may inform how to effectively target these pathways to better prevent and treat heart disease.\",\n\"Coronary artery disease (CAD) and myocardial infarction (MI) remain the leading causes of mortality worldwide despite advances in lifestyle modification and therapeutic interventions. Approximately 40% of the risk of developing coronary artery disease may be attributed to genetic risk factors, and large-scale genome-wide association studies (GWAS) have now identified a large proportion of this estimated heritable risk through 95 genome-wide significant loci. These loci are often identified in the form of single nucleotide polymorphisms (SNPs) which are co-inherited withs hundreds of other variants in linkage disequilibrium, presenting a challenge to elucidate causal functional variants responsible for the association. Furthermore, the nearest annotated gene is often not the causal gene, posing additional challenges to understand the mechanisms linking the variant to the disease-relevant phenotype. Expression quantitative trait locus (eQTL) and allele-specific expression (ASE) mapping in relevant cell types and tissues has been critical to prioritize candidate causal variants and genes, through co-localization and cis and trans regulatory network approaches. We have recently identified a number of regulatory variants associated with CAD by integrating genomic, epigenomic, and transcriptomic profiles in coronary artery smooth muscle cells (1). In particular, we have utilized the STARNET dataset to prioritize these loci. This dataset and other CAD and MI GWAS datasets will be further analyzed using a combination of omics datasets and experimental functional follow-up approaches, ultimately to identify novel mechanisms associated with CAD risk amenable to prevention and therapeutic development. We do not anticipate these studies to pose any risk to the study participants.   1. Miller CL, Pjanic M, Wang T, Nguyen T, Cohain A, Lee JD, Perisic L, Hedin U, Kundu RK, Majmudar D, Kim JB, Wang O, Betsholtz C, Ruusalepp A, Franzen O, Assimes TL, Montgomery SB, Schadt EE, Bj\\u00c3\\u00b6rkegren J, Quertermous T. Integrative functional genomics identifies regulatory mechanisms at coronary heart disease loci. Nature Communications. 2016 Jul 8;7:12092.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Miller, Melissa \",\n\"PFIZER, INC.-RESEARCH &amp; DEVELOPMENT DIV\",\n\"Evaluating SNP-Phenotype Associations from Million Veteran's Program Summary Statistics to Support Confidence in Rationale for Therapeutic Targets\",\n\"May11, 2020\",\n\"approved\",\n\"Pfizer Research and Development utilize many tools to enable novel target discovery across many different diseases, including human genetics. To that end, we request access to the summary statistics data associated with this dbGAP study in order to understand how single nucleotide polymorphisms, and the genes in which they are contained, are associated to disease to support therapeutic programs, as well as to understand other endpoints associated with our targets that might point to safety or efficacy concerns. Finally, we are interested in understanding the pathways that are linked to disease through the evaluation of genes associated to disease from GWAS studies.\",\n\"The Human Genetics and Computational Biology and Biomedicine groups at Pfizer utilize GWAS and other genetic-disease association data in order to better facilitate novel target discovery and target validation across research and development efforts across our portfolio. To that end, our group has collated various genetics summary statistics datasets in order to understand how traits are correlated genetically, the pathways underlying the genetic risk for disease, and potential causal relationships between certain traits and diseases. We plan to utilize the summary data contained within this study phs001672 to evaluate relationships between the genetics of traits for which we have internal data through such approaches as LD score regression and GCTA, as well as evaluate causal relationships between these traits and circulating proteins/biomarkers through Mendelian Randomization analyses. Additionally, we plan to use these data to perform pheWAS studies on variants which are associated to other disease traits in order to assess potential safety/efficacy issues. Finally, we plan to use these data to test novel algorithms, such as fgwas, for identifying causal genes at GWAS loci through the integration of genomics, transcriptomics, epigenomics, and other various omic's datasets. We request to evaluate all the various phenotypes captured by the Million Veteran's Program to date, as this can provide insight on disease etiology as well as links to secondary indications for targets and mechanisms of interest to our researchers.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"ming, li \",\n\"KUNMING INSTITUTE OF ZOOLOGY\",\n\"The relationship between human behavior and lifespan\",\n\"Jan18, 2023\",\n\"approved\",\n\"We aimed to explore the relationship between human behavior and lifespan by leveraging genetics data. Research result will help to increase awareness of human behavior and lifespan.\",\n\"We plan to study the relationship between human behavior, like depression, drinking, smoking and lifespan by analyzing the GWAS summary statistics data. We will conduct a series of genetic analyses including linkage disequilibrium score regression (LDSC), Mendelian randomization, and polygenic overlap quantification and identification to understand the genetic architectures of human behavior and lifespan. The applied dbGap data will be analyzed independently and will not be combined with other datasets.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Momenan, Abdolreza \",\n\"NIH\",\n\"PRS study of substance use disorders\",\n\"Jun10, 2022\",\n\"approved\",\n\"Genome wide association studies (GWAS) identify genetic variants associated with a trait such as problematic substance use. We will use two GWAS from dbGAP which targeted nicotine dependence and problematic alcohol use to generate individualized genetic risk scores for these traits in other samples. We will examine whether these risk scores are related to behavior (e.g. age of onset and frequency of substance use, and measures of impulsivity) or brain features detected using MRI, including measures of brain structure (e.g. cortical and subcortical grey matter volume and white matter tract integrity) and function.\",\n\"Polygenic risk scores (PRS) are a powerful tool to examine the genetic liability of a specific trait in another population. Our aim is to investigate the genetic liability to substance use disorders (e.g., alcohol) in population-based cohorts in relation to behavior measures and to identify brain intermediate phenotypes measures associated with greater genetic risk for problematic alcohol use. PRS will be create using the PLINK 2 software based on the results of the Genome Wide Association Studies (GWAS) problematic alcohol use (PMID 32451486 - phs001672.v3.p). These PRS will be regressed onto behavioral phenotypes (e.g., substance use measures including age of onset and frequency of use, and impulsivity measures) and onto brain measures including structure (cortical and subcortical measures and diffusivity) and function. This project does not require individual-level data from the mentioned studies and it is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Mullins, Niamh \",\n\"ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI\",\n\"Genetic and causal relationships between problematic alcohol use, depression and suicidality\",\n\"Mar28, 2022\",\n\"approved\",\n\"Problematic alcohol use, depression and suicidality commonly co-occur and represent substantial public health problems. This research project aims to leverage genetic analyses and data from the Million Veteran Program and the Collaborative Study on the Genetics of Alcoholism (COGA) cohort, to investigate the genetic, biological and potentially causal relationships between these phenotypes. \",\n\"Objectives: The objectives of the proposed research are to understand the genetic, biological and causal relationships between alcohol, depression and suicide-related phenotypes. Study design: We request approval to access summary statistics from published genome-wide association studies (GWAS) of major depression and problematic alcohol use conducted in the Million Veteran Program study. These GWAS results will be used in the Collaborative Study on the Genetics of Alcoholism (COGA) cohort, to calculate polygenic risk scores and genetic correlations, assess causal relationships using Mendelian randomization, and conduct GWAS meta-analyses. Analysis plan: Phenotypes of interest in the COGA study will be alcohol dependence, alcohol use disorder, problematic alcohol use, depression and suicide-related phenotypes (such as deliberate self-harm, suicidal ideation and suicide attempt). Analyses will be conducted in individuals of European and African-American ancestries, controlling for ancestry, relatedness and other relevant cohort/analysis-specific covariates. Analyses will be performed using the latest statistical genetics software, such as PRS-CS (for polygenic risk scoring), LD score regression (for genetic correlations), GSMR and others (for Mendelian randomization) and METAL (for GWAS meta-analyses). Data will be stored securely and with restricted access on the computing server at the Icahn School of Medicine at Mount Sinai; they will be used for research purposes only, and no attempt will be made to identify participants. This research will be conducted in collaboration with Dr. Peter Barr from SUNY Downstate Medical Center, who has prior dbGaP approval for these data.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Natarajan, Pradeep \",\n\"BROAD INSTITUTE, INC.\",\n\"Analysis of publicly available genomic data and cardiometabolic phenotypes\",\n\"Apr01, 2021\",\n\"approved\",\n\"The goals of our study is to use genetic and non-genetic factors, in concert with downstream biologic mediators, to characterize the determinants of cardiometabolic disease.\",\n\"We seek to identify the genetic and non-genetic factors for cardiometabolic disease, to identify disease subtypes, and to develop personalized treatment strategies to manage and prevent disease. These studies involve cardiovascular and metabolic traits and outcomes. Exposures include environmental, dietary, lifestyle, and genetic factors. Genetic factors include DNA sequence variation derived from array genotypes, targeted genotyping, targeted sequencing, whole exome sequencing, and whole genome sequencing. We seek to integrate orthogonal \\\"-omics,\\\" such as gene expression / transcriptomics, metabolomics, and proteomics, to better inform genetic analyses and characterize downstream mediators of disease.  We will conduct genotype-phenotype associations, include methods for rare and common variation, using both publicly available annotation sets and information from orthogonal \\\"-omics\\\" datasets generated within the same individual. In addition, genetic data may be used (subject to consent restrictions) for (a) variant summary statistics, (b) common controls in association tests, (c) population genetic studies.. Imputation would only be performed for specific variants of interest for replication of non-sequenced participants so we will not be generating an imputation panel. Datasets from the UK Biobank and FinnGen study will be combined with datasets provided by dbGaP for various analyses to fulfill the aims of this research.  Genetic and non-genetic data will not be used to identify individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Furthermore, when an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish scientific findings and, generally, broadly share any findings from these studies with the scientific community.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Ng, Maggie \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Genetic study of cardiometabolic diseases in diverse populations\",\n\"Sep03, 2020\",\n\"approved\",\n\"Cardiometabolic diseases and associated complications disproportionally affect minority populations in the US. These diseases may be partly contributed by shared genetic risks. Our study will examine genome-wide association data in diverse populations from Million Veteran Program and other studies to identify genetic variants associated with these diseases. We will aggregate the genetic risk scores of these diseases in single or multiple ethnicity to test for their clinical utility in disease prediction and prognosis. We will identify and evaluate the putative functions of genetic variants that are shared among multiple cardiometabolic diseases. \",\n\"Cardiometabolic diseases and related traits, such as type 2 diabetes, hypertension, dyslipidemia, obesity, coronary artery disease, stroke, chronic kidney disease, end-stage renal disease, retinopathy, neuropathy, and non-alcoholic fatty liver disease are highly heritable. Many of these cardiometabolic diseases and traits cluster within individuals and share genetic risks. Racial disparity has been observed in these diseases which disproportionally affect individuals of non-European ancestry. Our research program is interested in the genetic architecture of cardiometabolic diseases and related traits in diverse populations and actively participate in several related genome-wide association study (GWAS) consortia. Our objectives in this project are: 1) to perform ancestry-specific and trans-ancestry GWAS meta-analysis of cardiometabolic diseases and traits in diverse populations; 2) to develop ancestry-specific and trans-ancestry polygenic risk scores of cardiometabolic diseases and traits and evaluate their clinical utility; 3) to perform multi-trait GWAS and variant-trait clustering analyses to identify pleiotropic loci for cardiometabolic diseases and traits in diverse populations. These analyses will combine summary statistics generated from Million Veteran Program (MVP) and other studies, wherever available. This should not pose additional risks to participants. Results will be shared broadly in conference presentations and published manuscripts. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Niarchou, Maria \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"The medical and genetic correlates of short versus long buprenorphine treatment medication in the electronic health record\",\n\"Aug24, 2022\",\n\"approved\",\n\"Buprenorphine is one of the three medications approved by the FDA for the treatment of Opioid Use Disorder (OUD). Despite the benefits associated with buprenorphine in the treatment for OUD, retention remains poor. It is unknown whether and how genetic risk in the form of polygenic scores (PGS) for OUD is associated with the duration of buprenorphine treatment. Using the summary statistics from the GWAS of OUD (Zhou et al. 2020) we will be able to test whether there is a role for genetic liability of OUD in the duration of buprenorphine treatment. The findings will help us better understand the genetic mechanisms underlying buprenorphine treatment duration. We would like to note here that although genetics can play a role in OUD, the primary cause of OUD is environmental rather than genetic. Moreover, although PGS can be used as a tool to better understand the genetic mechanisms underlying BTD, there currently remain many technical and ethical limitations associated with the use of PGS in the clinical context. We will include these statements in our manuscript.\",\n\"Buprenorphine, which is a partial opioid agonist, is one of three medications approved by the U.S. Food and Drug Administration (FDA) for the treatment of opioid use disorder (OUD). Studies indicate that buprenorphine treatment is associated with reduced opioid use and higher treatment retention rates compared with placebo. Moreover, patients undergoing buprenorphine treatment have lower risk for all cause and overdose mortality and fewer opioid-related emergency department visits. Despite the benefits of buprenorphine, retention in buprenorphine treatment remains poor. Previous research on the impact of co-occurring somatic, psychiatric and other substance use disorders on buprenorphine treatment duration (BTD) has yielded mixed results. For example, some studies have found that, among other factors, psychiatric diagnosis is associated with greater odds of retention. However, other studies indicate that higher retention is associated with lower rates of psychiatric diagnoses, including substance use disorder diagnoses, while other studies have found no association between retention and comorbidity profiles. In order to increase buprenorphine retention rates and treatment efficacy for OUD, further research is needed to obtain a better understanding of the characteristics of patients who stay in buprenorphine treatment. Another factor that may differentiate patients who are retained in buprenorphine treatment is the genetic liability for physical and mental health conditions. There are now large GWAS for OUD, as well as traits closely related to OUD, including chronic pain, problematic alcohol use (PAU), depression, loneliness, and externalizing psychopathology. Taking into account that thousands of genetic variants of small effect sizes contribute to complex traits, the aims of our study were to test the association between somatic and psychiatric comorbidities and BTD, and to determine whether PGS for OUD and other OUD related traits contribute to the variation in BTD. The details of our study are the following:  Design: Case-control study.  Setting: Vanderbilt University Medical Center\\u00e2\\u0080\\u0099s (VUMC) electronic health records (EHR). Participants: N=6,049 individuals with buprenorphine prescription that lasted for less than 6 months (short-BTD) and N=1,919 individuals with buprenorphine prescriptions that lasted for at least 6 months (long-BTD).  Exposures: Phenotypes were based on mapping International Classification of Disease versions 9 and 10 codes.   We would like to request the GWAS summary statistics of OUD from Zhou et al. 2020 to  generate PGS for OUD and apply them to 545 individuals that have been prescribed buprenorphine and have genotype data available from the Vanderbilt Biobank.  Main outcomes: Long versus short BTD.  Our findings will help us better understand the factors that may be involved on whether a patient remains on buprenorphine treatment or not. We would like to note here that although genetics can play a role in OUD, the primary cause of OUD is environmental rather than genetic. Moreover, although PGS can be used as a tool to better understand the genetic mechanisms underlying BTD, there currently remain many technical and ethical limitations associated with the use of PGS in the clinical context. We will include these statements in our manuscript.   \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Nievergelt, Caroline \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Psychiatric Genomics Consortium for PTSD meta-analysis\",\n\"Feb10, 2020\",\n\"approved\",\n\" Up to 50% of the variance determining who develops Posttraumatic Stress Disorder following a severe trauma may be genetic; however, most successful analyses to date suggest that risk arises from multiple common genetic variants of small effect size, and that very large sample sizes will be needed for detection. This proposal aims to uncover the genomic architecture of PTSD through large-scale, collaborative genome-wide association studies. Identifying the pathways underlying PTSD will lead to an improved neurobiological understanding, enhanced prevention, and improved treatment of this debilitating and prevalent syndrome.\",\n\"Posttraumatic stress disorder (PTSD) occurs only in vulnerable individuals after exposure to severe traumatic events. This risk is due, in part, to 40-50% heritability of differential vulnerability. Due to increasing collaborations across the field of PTSD genomics and the advent of new analytical tools, it is a very exciting time for PTSD genetic risk discovery. The purpose of this application is to facilitate meta-analyses of genome-wide association study (GWAS) data for symptoms and diagnosis of PTSD. We propose to conduct large-scale meta-analyses through the PTSD group of the Psychiatric Genomics Consortium (PGC). Currently our multi-ethnic data collection includes genotypes from 60 studies with a total N of over 200K combined cases and trauma-exposed controls. We hypothesize that with an increased sample size and deeper phenotype characterization, the PGC-PTSD will accelerate our current understanding of the genetic architecture of PTSD. Our primary Aim is to expand the PGC-PTSD data resource to produce the most powerful GWAS analyses. Addition of the MVP PTS symptoms dataset is a next step. Only those named on the project or appropriate personnel will have access to the data and derivative data for analysis. Only meta-analysis results will be published and shared broadly with the research community. Meta-analyses across GWAS on PTS symptoms and PTSD diagnoses will be performed using the standard PGC pipeline RICOPILI on the LISA server. GWAS downstream analyses will be performed using the FUMA pipeline. Lastly, we will use polygenic risk scores (PRS) generated using PRcise to provide insights into relationships to other traits and advance causal inference. Identifying the genetic pathways underlying PTSD will lead to improved neurobiological understanding, enhanced prevention, and improved treatment of this debilitating and prevalent syndrome. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Okada, Yukinori \",\n\"OSAKA UNIVERSITY\",\n\"The genetic architecture of quantitative traits and their correlation across phenotypes between populations.\",\n\"Feb21, 2019\",\n\"closed\",\n\"Although association of genotypes with phenotypes is extensively investigated,  the majority of the studies are targeting European ancestry individuals. Here we would like to elucidate the differences between the genetic architecture of quantitative traits between European and East Asian population, and would also like to assess their correlation with other phenotypes. We would like to  evaluate the shared and distinct genetic architecture between populations.\",\n\"GWAS has been clarifying the polygenic architecture of complex human traits. However, the majority of the studies are targeting European ancestry individuals. Here we would like to elucidate the differences between the genetic architecture of quantitative traits between European and East Asian population, and would also like to assess their correlation with other phenotypes. We have been conducting large-scale GWASs of quantitative traits including cholesterol-related traits in Japanese population. By constructing polygenic risk scores and associating them to the end-phenotypes such as late-onset complex diseases or health outcomes in both of the populations, we would like to elucidate shared and distinct effect of the exposure to the outcome. For this purposes we look for the access to the summary statistics of cholesterol-related traits in MVP.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Olsen, Morten \",\n\"UNIVERSITY OF COPENHAGEN\",\n\"A genome wide association study for identification of genetic loci associated with venous thromboembolism\",\n\"Apr11, 2022\",\n\"expired\",\n\"This study aims to Identify common genetic variants associated with venous thromboembolism (VTE) as well as develop/calculate polygenic risk score for venous thromboembolism and compare risk estimates in the highest polygenic risk with known risk factors for VTE.\",\n\"The objective is to identify common genetic variants associated with venous thromboembolism (VTE). We will do this by using summary statistics from VTE together with our main study of a danish cohort.   The main study design is a genome-wide association study, where we will use a large number of single nucleotide polymorphisms (SNPs) to identify genetic regions that associate with VTE and the components and complications to it and related diagnoses. We will include patients with a known pathogenic variant as well as patients where the pathogenic variant is unknown. We will use expected genotype counts, and the logistic model will be adjusted for sex, age, principal components.    Using GWAS summary statistics, we will develop/calculate polygenic risk score (PRS) for venous thromboembolism and compare the risk of high PRS (e.g. top 5 %) with the risks associated with known risk factors (e.g. Factor V Leiden, cancer, major surgery, oral contraceptives/hormone re-placement therapy). We will also explore how this PRS associates with known for blood cell and coagulation-markers. For these analyses, we will be using VTE summary statistics from the Million Veteran Program in a meta-analysis with similar summary statistics from other biobank datasets. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Osburn, William \",\n\"JOHNS HOPKINS UNIVERSITY\",\n\"Role of VTE-associated SNP in the liver\",\n\"Nov02, 2021\",\n\"approved\",\n\"Every year up to 600,000 individuals suffer from blood clotting disorders such as deep vein thrombosis and/or pulmonary embolism. There is a need for new therapies to treat or prevent these disorders. Several large genetic studies have identified genetic markers that are found more frequently in individuals with blood clotting disorders. While the effect of mutations in a number of these genetic markers is obvious (e.g., marker is within a gene with a known role in blood clotting), these studies have also identified genetic markers whose role in blood clotting disorders is not clear. Therefore, this study will pinpoint specific genetic variants that could be causing increased risk of blood clotting disorders and investigate their role in production of coagulation proteins using human liver cell lines and standard molecular biology experimental approaches.  \",\n\"Venous thromboembolism (VTE) including deep vein thrombosis and/or pulmonary embolism significantly contributes to morbidity and mortality affecting up to 600,000 individuals a year in the United States. New prevention and therapeutic approaches are needed to lessen the burden of VTE-associated disease. Several genome-wide association studies have identified genetic loci that are associated with risk of VTE. While some of the genetic risk loci involve gene products with known activity along coagulation or hemostasis pathways, several novel loci near or within genes having no known role in coagulation were identified, including genes that encode transcription factors, epigenetic proteins and proteins involved in translation. Since most coagulation factors are secreted by the liver and the expression of genes containing or adjacent to the identified risk loci have been documented in human liver and hepatocyte cell lines, we will focus on the effect of the identified single nucleotide polymorphisms (SNP) on transcription, translation, and secretion of proteins in the coagulation pathway in liver. Therefore, we will do fine mapping of the genetic risk loci in those data sets to identify SNP that are potentially causal and study these SNP in vitro using CRISPR-Cas9 approaches in HepG2 cells, a human hepatocellular carcinoma cell line. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"PALMER, ROHAN \",\n\"EMORY UNIVERSITY\",\n\"Systems Genetics of Alcohol and Other Substance Use Disorders \",\n\"Dec05, 2019\",\n\"rejected\",\n\"This proposal requests assessment and genotype data in dbGAP for training and research purposes. These data will be used to investigate gene networks that influence susceptibility to alcohol and other drug use and misuse. Drug Addiction is a complex disease affecting many Americans. Despite the fact that genetic differences largely account for individual differences in susceptibility to addiction, analyses using dense SNP coverage of the entire human genome only account for a small portion of the liability to disease. The K01 and DP1 grants builds upon the idea of placing more emphasis on biologically relevant genes by using novel systems biology approaches in combination with statistical and algorithmic procedures that capture gene-gene interactions. Moreover, our analytical pipeline leverages the increased power gained in prediction by modeling not just addiction outcomes, but also comorbid psychopathologies, such as ADHD and depression, which have overlapping genetic liabiity with drug addiction. The rich phenotypic, genotypic, and transcriptomic data available in dbGAP and these these data will be used to identify and test gene-networks that confer susceptibility to alcohol, tobacco, cannabis, heroin, and other substances of abuse.\",\n\"Data (e.g., SAGE, COGA, and The Genetic Architecture of Smoking) will be used to facilitate the identification of genetic loci for drug use and to develop novel computational tools for genomic prediction. Specifically, to develop, modify, and apply novel computational techniques that facilitate the modeling of the joint and multiplicative effects of genetic variants. Each dataset affords access to phenotypes not captured in a single study. Sociodemographic, biological, and other phenotypes related to drug addiction will be examined as covariates/moderators. Primary analyses will be conducted in non-overlapping external samples that are sufficiently large (e.g., UKbiobank, Estonian Biobank, Australian, THL Biobank) and deCODE with replication analyses being conducted across the dbGAP samples with available GWAS data (e.g., SAGE, AddHealth, COGA, etc.). When possible (i.e., only when allowed by the Data Use Agreement), dbGAP datasets that have genotype data and relevant phenotypes will be merged (when the Data Use Agreement allows) to increase statistical power after first imputing the genomic data within ancestry using the Michigan Imputation Server. dbGAP samples that are just summary statistics (e.g., PGC summary files) will be used to inform the primary and replication analyses. We will also meta-analyze results across available datasets using summary statistics from within sample GWAS. Transcriptomic evidence from datasets, such as GTEX will also be used to help prioritize genomic regions for analysis. dbGAP samples of various ancestries are requested to aid in the prioritization of genomic regions as recent studies indicate that different populations confer varying power to estimate effects across different regions of the genome. It is important to note that analyses are being conducted within ancestral groups and will only be reported as such; this minimizes any possibility of stigmatization as our focus is on convergent findings across groups. It is hypothesized that: (Hyp 1) neuronal and metabolism pathways and their associated genes will be well represented in identified gene-network models; (Hyp 2) Bayesian and network-based models will be more predictive of than GWAS association methods; (Hyp 3) genes that comprise subnetworks will be consistent across related phenotypes, though  individual variants within genes may be different; (Hyp 4) genetic effects for related traits (e.g., depression, ADHD) will enhance the identification and makeup of the gene network structure. Replication analyses will be limited to independent non-dbGAP sample samples with similar phenotypic information (e.g., the CADD and NESARC). Externally de-identified data from other repositories will not be combined with these data, but the resulting summary statistics from our analyses will be meta-analyzed across relevant data. (Hyp 5a) A subset of loci that contribute to the additive genetic variance of substance use and disorders can be identified by prioritizing information from prior experimental and eQTL studies. (Hyp5b) The \\\"subset of loci\\\" and the integrated data can be leveraged to infer risk in other samples.  Additional Notes: Individualized dbGAP data will not be shared with any external collaborators. These individuals will only have access to summary statistics. If data are needed they will submit a separate application for access. Requests for phenotypes that are phenotypically and genetically correlated with substance abuse is specifically for the purpose of understanding the genetic architecture of these traits (e.g., PGC summary data for schizophrenia, bipolar, ADHD) as their pleitropy can be leveraged to enhance the detection of loci for substance use outcomes. Requests for access to genotypes and phenotypes of cancer-related datasets are to study individual differences in levels of tobacco/nicotine consumption. Lastly, these data will not being used to associate ancestry/race with any health or clinical outcomes. Specifically, data from Europeans, Latino, and African Americans are requested to conduct within-ancestry analyses in order to increase power to detect regional genetic association effects as stratifying analyses by population and identifying convergent findings has proven to offer greater statistical power (see Brick et al., 2019, Addiction Biology for details). \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"PALMER, ROHAN \",\n\"EMORY UNIVERSITY\",\n\"Evaluation of Polygenic Scores for Opioid Use in European American Populations \",\n\"Sep09, 2021\",\n\"approved\",\n\"This proposal requests summary statistics from existing GWAS on MVP samples for training and research purposes. These data will be used to investigate how gene sets identified in the parent GWAS are associated with opioid use outcomes in a clinical sample undergoing treatment of opioid use disorder.\",\n\"This project leverages the MVP GWAS summary statistics to address important questions concerning the generalizability and utility of polygenic scores for opioid use and opioid overdose risk. We will investigate how opioid PGSs are associated with opioid outcomes and opioid related behaviors (i.e., depression and risk taking), in a population undergoing methadone maintenance therapy. Moderators of PGS effects will also be examined using sociocultural measures of participant environment (i.e., PGS x environment). All summary statistics provided will be limited to research use only. Resulting publications will be honor all data use requirements and acknowledge relevant MVP related funding sources. No risk to MVP participants are expected. The overarching goal is to understand how genetic risk for opioid phenotypes is associated with problematic use in European Americans.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"PALMER, ROHAN \",\n\"EMORY UNIVERSITY\",\n\"Evaluation of Polygenic Scores for Alcohol Use in African American Populations\",\n\"Jun11, 2021\",\n\"approved\",\n\"Alcohol use and disorders are enduring public health problems in the United States with more than 86.3% of people older than 18 having drank alcohol at some point in their life and 5.8% (i.e., 14.4 million adults) meeting criteria for an alcohol use disorder. Large-scale genomewide associations studies indicate that individual differences in alcohol use and disorder are partly explained by genetic differences. Polygenic scores  provide a means to index alcohol and alcohol use disorder risk in the general population, but few studies have examined their utility in underrepresented groups, such as African Americans. Our study aims to examine the utility of alcohol PGSs in order to understand how they perform within groups of individuals exposed to varying levels of traumatic experiences  lifetime.\",\n\"This project leverages the MVP GWAS summary statistics to address important questions concerning the generalizability and utility of polygenic scores for alcohol use and problems. The provided summary statistics will be used to develop polygenic score in an African American cohort of men and women exposed to trauma. We will investigate how alcohol PGSs are associated with alcohol outcomes and alcohol related behaviors, such as anxiety and depression. Moderators of PGS effects will also be examined using sociocultural measures of participant evivornment (i.e., PGS x environment). All summary statistics provided will be limited to research use only. Resulting publications will be honor all data use requirements and acknowledge relevant MVP related funding sources. No risk to MVP participants are expected. The overarching goal is to understand how genetic risk for alcohol phenotypes is associated with problematic use in African Americans. We will employ recent MVP GWAS findings for AUDIT-C and AUDIT-P. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"PALMER, ROHAN \",\n\"EMORY UNIVERSITY\",\n\"Characterizing Phenotypic and Etiological Relationships between Alcohol & Cannabis Misuse and Executive Functioning\",\n\"May13, 2022\",\n\"approved\",\n\"Alcohol and cannabis use disorders are persistent in United States due to both genetic and environmental factors. While polygenic scores provide a means to index alcohol and alcohol use disorder risk in the general population, advancements in identifying genes for complex behavioral traits have been slow. In parallel, early impairment of executive functioning has been implicated as a precursor and genetic correlate of substance involvement and may aid in understanding individual differences in misuse; nonetheless, few have examined how polygenic scores for executive functioning increase our ability to predict risk of alcohol and cannabis use disorders. Specifically, new methods have suggested that if we leverage the observed genetic factors commonly shared between behaviors (i.e., pleiotropy) that we will increase the predictive utility of polygenic scores for behavioral outcomes, such as alcohol and cannabis use disorders. The proposed research examines how domains of executive functioning influence developmental patterns of alcohol and cannabis use using a variety of population and genetic approaches. The information obtained from this project will clarify the relationship between domains of executive functioning and the development of substance use, while providing solid groundwork to elucidate novel biologically informed targets which will contribute to more personalized prevention and intervention approaches. \",\n\"The current proposal is the third part of a dissertation study that seeks to demonstrate the utility of executive functioning as a precursor\\u00e2\\u0080\\u0094as well as vulnerability factor\\u00e2\\u0080\\u0094for alcohol and cannabis use and disorders using both sophisticated phenotypic and genetic methods. Specifically, the MVP GWAS findings for AUDIT-C and ICD diagnoses of abuse/dependence will be utilized to develop polygenic scores (PGS) for both alcohol use and alcohol use disorders among adolescents and young adults from 3 separate cohorts across two independent longitudinal samples in the third part of the study at large. Single and multi-trait models containing derived PGS for executive functioning, alcohol/ cannabis use and disorders, and impulsivity will be employed to predict class membership for alcohol and cannabis use trajectories, as well as lifetime alcohol/cannabis use disorders. Underlying this study is the notion that one genetic locus can influence multiple traits (i.e., pleiotropy), suggesting that increases in variance can be gained by including related traits/behaviors in our PGS models more variation among alcohol/cannabis use profiles than single-trait models. Primarily, we will examine how class membership for alcohol and cannabis use trajectories is a mediator between the PGSs and developing alcohol/cannabis use disorder. All summary statistics provided will be limited to research use only. Resulting publications will be honor all data use requirements and acknowledge relevant MVP related funding sources. No risks to MVP participants are expected.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Pare, Guillaume \",\n\"MCMASTER UNIVERSITY\",\n\"Genetic and Epigenetic Determinants of Cardiometabolic Disease\",\n\"Mar01, 2019\",\n\"approved\",\n\"Cardiometabolic diseases such as atherosclerosis and diabetes are leading causes of death and disability globally. While the environmental risk factors for these conditions are well understood, less is known about the effect of alterations in DNA on regulating disease risk. The aim of our study is to 1) identify rare genetic variants in the protein coding regions of our genome, which are responsible for regulating essential structural and functional cellular processes, 2) elucidate the role of common genetic variants through polygenic risk score analysis, and 3) characterize the interplay between genetic and epigenetic signals involved in relevant disease processes. In doing so, we can comprehensively assess the genes and pathways involved in cardiometabolic disease onset and leverage our findings to help discriminate an individual\\u00e2\\u0080\\u0099s risk for disease onset. We will compare the frequency of rare protein-coding variants in diseased and reference populations and also examine differences in epigenetic signatures. In order to effectively accomplish our study aims, we are requesting access to individual-level data from pertinent case-control and longitudinal cohort studies. \",\n\"Cardiometabolic diseases such as Atherosclerotic Cardiovascular Disease (ACVD) and Type 2 Diabetes Mellitus (T2DM) represent leading causes of mortality and morbidity worldwide. Large-scale seminal analyses have demonstrated that an individual\\u00e2\\u0080\\u0099s genetic and epigenetic architecture can confer profound risk for the development of both disorders. We propose that assessing the impact of common variants, rare protein-altering variants, and methylation patterns on disease risk can provide invaluable insight into novel molecular pathways underlying cardiometabolic disease etiology. The primary objectives of our analysis will be to 1) identify the burden of rare protein-altering mutations responsible for very early ACVD and T2DM at the exome-wide level, 2) characterize the role of common variants through polygenic risk scores, 3) investigate epigenetic signatures of cardiometabolic conditions through an integration of genotyping and methylation data. To fulfill our primary objectives with sufficient statistical power, we request access to individual-level data from study populations encompassing the MIGen ExS cohorts, the Heart GO cohorts of the NHLBI GO ESP, select cohorts from the NHLBI TOPMed project, select cohorts of the T2D-GENES Project, and study cohorts inclusive of genotype and DNA methylation data.  Case exomes will be incorporated in a case-only association analysis using summary statistics within the ExAC as an external control dataset. Rare variant burden testing will be used to delineate gene-based association signals and effect sizes. These parameters can be used to facilitate gene prioritization in association analyses using locally sequenced cases. For epigenetic analysis, genotype and DNA methylation data will be integrated in order to characterize the epigenetic signatures associated with cardiometabolic disease. Genetic risk scores can then be used to identify causal relationships through Mendelian randomization. Both analyses will be preceded by stringent quality control procedures to prevent spurious associations. We foresee absolutely no risk for study participants within the requested cohorts as data will remain anonymized on secure servers only accessible to study personnel. All individual-level data requested from dbGaP will not be combined with external datasets. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Patrick, Matthew \",\n\"UNIVERSITY OF MICHIGAN AT ANN ARBOR\",\n\"Shared Genetic Risk Between Psoriasis and its Comorbidities through the Million Veteran Program\",\n\"Nov09, 2021\",\n\"closed\",\n\"Psoriasis is an inflammatory skin disease that substantially affects the health and well-being of many patients. Beyond its impact on the skin, psoriasis also has a broader impact on inflammation across the body. As such, patients with psoriasis are also at risk from a wide range of other inflammatory diseases. Understanding which diseases a patient is at risk from is important to identify how best to treat the patient. We therefore intend to use data from the Million Veteran Program (MVP) to identify the risks of different diseases, based on their genetics.\",\n\"We intend to use data from the Million Veteran Program (MVP) to investigate the shared genetic risk factors between psoriasis and its comorbidities. We have conducted a large GWAS meta-analysis for psoriasis, involving over 11,000 psoriasis individuals (Patrick et al. 2018, Nat. Commun.) and have experience investigating shared genetics using GWAS summary statistics. For example, recently we identified IFIH1 as a shared genetic locus between psoriasis and coronary artery disease (Patrick et al. 2021, J. Investig. Dermatol.). The MVP cohort includes relevant traits, such as HDL and LDL cholesterol, peripheral artery disease and venous thromboembolism, that will help us to understand the relationship between psoriasis and cardiovascular disease at a level of finer granularity, as well as reveal important information about the relationship between psoriasis and its other comorbidities.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Pendergrass, Rion \",\n\"GENENTECH, INC.\",\n\"Using Summary Statistics from Existing Datsets to Seek Replication of Genetic Associations \",\n\"Oct22, 2020\",\n\"approved\",\n\"In performing Genome Wide Association Studies (GWAS) it is important to seek replication for significant genetic associations within other datasets. It is also critical to do so across GWAS of different ancestries, to determine what is similar and different. The Million Veterans Program (MVP) has performed a series of GWAS across multiple ancestries, with a range of phenotypic outcomes. We intend to use these summary results to compare with GWAS that we are performing for similar phenotypes. \",\n\"We will not combine the requested datsets with other datasets outside of dbGAP  A series of GWAS were completed within the Million Veterans Program (MVP), and association studies are listed here: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/GetListOfAllObjects.cgi?study_id=phs001672.v4.p1&object_type=analysis These studies provide valuable genetic association results that can be used for comparing to genetic association results we have through GWAS of other datasets.  Of particular interest is comparing our GWAS results to the multi-ancestry results of the MVP, as the MVP is more ancestrally diverse than our European-American focused cohorts. Differing linkage disequilibrium and allele frequencies may highlight important differences and similarities between the results of our GWAS and the MVP. We will be comparing summary statistics across these datasets with our other GWAS results, no individual level data is being combined or compared.   While we are comparing association results across study by ancestry, we are not studying population origins or ancestral history.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Perera, Minoli \",\n\"NORTHWESTERN UNIVERSITY AT CHICAGO\",\n\"Venous Thromboembolism in African Americans \",\n\"Sep29, 2021\",\n\"closed\",\n\"We plan to use the publicly available GWAS data to first validate genetic mutations we have found to be associated to clot formation in African Americans. We will then use this GWAS data to prioritize genetic mutations that are more likely to change the amount of protein a cell makes. We will then test these mutations for association with clot formation in our clinical data set of African Americans.\",\n\"Venous thromboembolism (VTE) is a complex disease encompassing deep vein thrombosis (DVT), pulmonary embolism (PE), or a combination of both. In the US, each year over 900,000 individuals are hospitalized due to VTE and over 300,000 patients die. African Americans (AAs) have the highest incidence and mortality rates of DVT and PE. Warfarin is used to treat and prevent DVT and/or PE. However, warfarin dose requirement has been shown to be higher among AAs as well as DVT/PE patients. The research objective is to identify genetic variants associated with DVT/PE in African Americans. We have conducted a candidate gene study in African Americans on warfarin to identify SNP associations to DVT/PE. We plan to use the requested GWAS data set to validate our significant findings in an independent data set. We will then use bioinformatic methods to restrict the GWAS finding to only those that are expression quantitative trait loci (eQTLs) in several relevant tissues to enrich for truly functional signals. We will then genotypes these eQTLs in our African American cohort and again test for association to DVT/PE. Results from these studies will published in peer reviewed journals.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Pers, Tune \",\n\"UNIVERSITY OF COPENHAGEN\",\n\"Identification of Cell Populations Associated with Type 2 Diabetes\",\n\"Apr01, 2021\",\n\"approved\",\n\"Type 2 Diabetes (T2D) is a growing health concern worldwide. To study the genetic influence on health and disease we will leverage summary statistic data generated from the genome wide association study from the Million Veteran Program and single-cell profiles to identify the tissue and cell populations that are enriched for genetic susceptibility associated with T2D.\",\n\"The objective of this proposal is to leverage large-scale human genetic data and single-cell profiling to identify candidate cell populations that are enriching for genes co-localizing with genetic variants associated with Type 2 diabetes (T2D). We are requesting access to the African American, European, Hispanic American and multi-ethnic type 2 diabetes (T2D) GWAS summary statistics from the Million Veteran Program (MVP). We have developed a computational framework to identify whether a given cell population overall enriches for genes near variants identified in Genome Wide Association Studies (GWAS). We and others have shown that integration of GWAS data with human microarray-based gene expression data and single-cell RNA-sequencing data can help to pinpoint likely cell populations and tissues underlying complex traits and diseases. Towards this aim, we will integrate summary statistics associated with T2D generated from the GWAS conducted within the MVP dataset with publicly available and inhouse single-cell expression datasets to identify candidate cell populations enriching for human genetic variants associated with T2D. For these analyses we are requesting the summary statistics data (phs001672.v4.p1) published in Nature Genetics volume 52, pages 680\\u00e2\\u0080\\u0093691(2020; PMID 32541925). We are not requesting individual level data. There are no planned collaborations, and the data will be exclusively used by our group.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Pirastu, Nicola \",\n\"UNIVERSITY OF EDINBURGH\",\n\"Understanding the causal determinants of complex traits through the use of two sample mendelian randomization\",\n\"Oct19, 2019\",\n\"closed\",\n\"We will be using the MVP genome-wide association study results to understand the factors which are causal to complex disease using a technique called Mendelian Randomization.\",\n\"One of the main goals of epidemiology is to understand how to improve people\\u00e2\\u0080\\u0099s health by modifying causal risk factors. Identifying these causal factors has been extremely challenging due to the difficulty in designing proper longitudinal studies or randomised controlled traits. Recently with the increase of strongly powered GWAS studies, an alternative has aroused: Mendelian randomization. In Mendelian randomization (MR), the genetic variants strongly associated with a specific phenotype are used as instrumental variables to assess causal relationships between an exposure trait and an outcome. In order to avoid bias, MR requires that the effects of the SNPs on the exposure trait and on the outcome trait are measured on different sets of samples. Our study broadly aims at understanding the causal relationships amongst traits by using two-sample Mendelian randomization. In particular, MVP represents a very well powered cohort completely independent from UK biobank and thus results from GWAS are particularly well suited to be used in combination with the results coming from UK biobank to be used both as exposures and outcomes. For this goal we need to access full summary statistics from the GWAS published on the MPV on Blood pressure (DBP, SBP and PP) and Blood lipids (HDL, LDL, Total Cholesterol and Triglycerides).  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"POLIMANTI, RENATO \",\n\"YALE UNIVERSITY\",\n\"Investigating the genetic overlap of human traits and diseases\",\n\"Jul22, 2020\",\n\"approved\",\n\"Many individuals either suffer from or have family members suffering from a major health diagnosis. Much of the distress caused by these diagnoses arises from a lack of understanding of the \\u00e2\\u0080\\u009cwhy\\u00e2\\u0080\\u009d this event happened to them or their family and how this event affects their personal risk for the disorder. Because the genetic risk for complex traits is distributed throughout the healthy and affected population, our study aims to put disease risk into context and provide the community with answers to why certain features of the human phenome (e.g., diseases and other deleterious traits) persist in the population and how they influence other phenotypes. Additionally, we will compare illnesses affecting different age groups to understand how genetic risk changes over the lifetime and in particular in more vulnerable individuals such as children and the elderly.\",\n\"We will investigate how genome-wide pleiotropic variants connect trait domains across the human phenome. This aim will uncover how the polygenic architecture of disease relates to other phenotypes of the human condition to uncover gene and gene-network therapeutic targets. The causes and consequences of pleiotropy will be investigated using sophisticated statistical approaches such as heritability partitioning for various annotations of the genome, effect size distributions, genomic structural equation modeling, bivariate genetic correlation, latent causal variable,  Mendelian randomization, polygenic risk scoring, genotype networking, and multiple tools for drug-repositioning analyses. To understand the dynamics underlying diseases with different ages of onset, we will focus particularly on those related to childhood and elderly age of onset, comparing them to other illnesses. This will permit us to dissect how genetic variants are associated with pathological associated with disorders characterizing different age groups such as dementias in the elderly. Data use will be limited to research and will not be used in any commercial applications or patents. We will publish or otherwise broadly share any findings from their studies with the scientific community. We also will honor all embargo dates associated with any of the publicly available datasets. We do not believe that this creates any additional risks to participants.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Posthuma, Danielle \",\n\"VRIJE UNIVERSITEIT\",\n\"Investigating genetic heterogeneity in dimensions and subgroups of alcohol misuse\",\n\"Dec13, 2021\",\n\"approved\",\n\"Alcohol misuse, consisting of both heavy consumption and pathological patterns of alcohol use, is one of the leading contributors to the global public health burden. Alcohol misuse is heritable, but there are likely different genes and biological processes involved for different individuals and/or different types of alcohol misuse. In this study, we will conduct genetic analyses across these subgroups/subtypes to see what genetic influences are shared or unique. We will compare these results with those from more traditional measures of alcohol misuse (e.g. alcohol use disorder diagnoses) to determine whether a consideration of these different dimensions can lead to improved understanding of alcohol misuse and targeted implementations in precision medicine. \",\n\"Alcohol misuse, consisting of both heavy consumption and pathological patterns of alcohol use, is one of the leading contributors to the global public health burden. Individual risk for alcohol misuse is substantially heritable, yet there has been slow progress in identifying the associated genetic factors and translating them into clinical practice. However, emerging evidence indicates that biological processes differ across dimensions of alcohol misuse (e.g. consumption versus problems) and may also differ across subgroups of individuals (e.g. with or without certain environmental exposures). Better understanding of these processes has the potential to inform precision medicine applications to best target a variety of etiological processes in the development of alcohol misuse.  In this study, we will use biobank data to investigate the genetic etiology of various dimensions and subgroups of alcohol misuse. We will use GWAS and in silico tools to identify associated genetic loci, and compare these with loci identified by more traditional measures of alcohol misuse such as overall consumption and alcohol use disorder (AUD) diagnoses, using publicly available summary statistics. To this end, we aim to include the summary statistics from the Million Veterans Program GWAS studies of alcohol-related measures (AUD, problematic alcohol use [PAU], AUDIT, maximum alcohol use). We will use these summary statistics to compare overlap in loci from our discovery sample, using tools such as functional annotation and (local) genetic correlation. In case of substantial genetic overlap between studies, we will meta-analyze discovery sample and MVP summary statistics (and/or other publicly available summary statistics of alcohol-related GWASs) to increase variant discovery power.  We will use this data in line with the participants\\u00e2\\u0080\\u0099 consent. We will not access any individual level data and will not attempt to identify participants or their ancestries. This research is being conducted for health/biomedical science purposes and the results will be published in scientific journals and shared with the scientific community to advance the understanding of the etiology of alcohol misuse. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Priest, James \",\n\"STANFORD UNIVERSITY\",\n\"Pre-genotyped Public Datasets for the Study of Cardiovascular Disease\",\n\"Sep27, 2019\",\n\"closed\",\n\"To understand why different individuals have diseases such as abnormalities of the heart we look at the DNA or genetic code for differences or mutations. To know if a mutation is important or unusual we need to look for similar genetic mutations in other people who do and do not have the disease. The computer software we use for finding mutations has improved significantly over the last number of years. So we would like to use the publicly available genetic information in the NIH dbGAP from patients and perform our own analysis, so that we can use this data as control subjects for the genetic information we have here at Stanford on patients with heart abnormalities.  \",\n\"Ongoing genetic studies of rare inherited cardiovascular disease within the Stanford University School of Medicine require an appropriate control populations for comparison. The rare cardiovascular diseases under study include cardiomyopathies, arrhythmias, and a variety congenital malformations.   For exome or genome sequencing studies performed by next generation sequencing (NGS), the assumptions regarding alignment and variant calling of publicly available datasets are varied and largely unknown. We will obtain raw fastq or bam files from exome data on upwards of 2000 individuals generated from ARIC, SWISS, PCGC,and other projects which have both healthy controls, and also individuals with some of the cardiovascular diseases under investigation. We will perform alignment and genotyping on these dbGAP subjects in the same highly sensitive manner as our study population. This will maximize sensitivity for rare variant detection in a well-phenotyped control population and minimize the systematic error when performing case-control type comparisons.  A second approach using array-based data will test methods for using external control populations for case-control comparisons where only the cases (probands) have been collected. Probands with cardiovascular disease from the pediatric cardiac genomics consortium will be matched with individuals without in the UK Biobank study or the study populations from dbGAP. Individuals will be matched using a principal components analysis (PCA) on the combined sample, computed over common autosomal SNPs which pass variant quality control. Probands which are outliers in any component relative to the distribution of Biobank individuals with self-reported ancestry will be removed, and a second PCA to be used for matching will be computed in this subsample. Matches will be selected by taking the samples nearest each proband by Euclidean distance in the space of the first four components of the new PCA.   The data obtained from dbGAP may be combined with the collected data at Stanford, the UK Biobank study, and the Pediatric Cardiac Genomics Consortium study for the purposes of improving methods for genetic inference. This re-analysis will not pose any additional risk to the subjects because all of the data (dbGAP, Stanford, UK Biobank, Pediatric Cardiac Genomics Consortium) is received in a de-identified fashion, stored and analyzed under dbGAP-compliant security architecture.  The datasets from dbGAP will be used as healthy controls for the purposes of case-control comparison to rare inherited cardiovascular disease. The datasets from PCGC will be used as a replication cohort and \\u00e2\\u0080\\u009cdisease controls\\u00e2\\u0080\\u009d for ongoing studies of congenital heart disease.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"PRITCHARD, JONATHAN \",\n\"STANFORD UNIVERSITY\",\n\"Human genetic variation and complex trait risk\",\n\"May26, 2020\",\n\"rejected\",\n\"We are studying the causes of diseases like heart attack, diabetes, and asthma. In this project, we want to look at the role genes play in these diseases, and particularly on how different genes matter in different contexts. We are focusing on how ancestry changes which genes matter, and how much genes matter in general, as our primary goal, and we also want to look at how these common diseases relate to one another. We will first look at genetic factors for each disease, using both our knowledge of people with the disease and its risk factors. Because more people have each risk factor than have the disease, this allows us to build better genetic models. Then we will use these models to predict disease in people with parents from different populations, which is particularly difficult for genetic testing, by finding which genes came from each population. Finally, we will compare the genetic models to what doctors now use from family history and test whether genetic models can improve disease models using these new methods. \",\n\"This project aims to understand the links between genetic variation and risk for complex diseases. In particular we are developing new methods to analyze complex trait associations in European, admixed and non-European individuals. We will focus on complex traits, including biomarkers and cardiometabolic (heart disease, diabetes, obesity, and complications, including metabolic diseases of pregnancy), sleep, and pulmonary (asthma, COPD, etc, risk factors, and complications) traits. Our work will involve both development of novel methods related to estimating polygenic scores in admixed individuals as well as environmental perturbation modeling and technical, environmental, and genetic sources of poor portability of polygenic scores. In some cases we may analyze these data jointly with data from other sources including UK Biobank. We will not attempt to link individuals from different studies and we do not anticipate that this creates any additional risk to participants. At this time the work does not involve collaboration with researchers at other institutions. Some of the data sets provide notes indicating restricted usage, including that come data can only be used to investigate specific types of conditions; in our analytical work we will abide by all usage regulations. We plan to publish our findings in peer-reviewed academic journal articles.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Ray, Debashree \",\n\"JOHNS HOPKINS UNIVERSITY\",\n\"Genetic determinants of diabetes and related traits\",\n\"Nov22, 2022\",\n\"approved\",\n\"We will use summary information from Million Veteran Program dataset to replicate genetic risk factors of diabetes and other related outcomes.\",\n\"We will use genome-wide association study (GWAS) summary statistics from the Million Veteran Program (MVP) dataset to explore if genetic determinants identified from our in-house data on diabetes and related traits are replicated in MVP. Phenotypes of interest are diabetes and any related trait/outcome. We will not combine the MVP dataset with other datasets outside of dbGaP; we will only combine summary information of MVP with other summary information such as that used for meta-analysis when replicating results. Further, we will use this MVP summary dataset to illustrate novel statistical methodologies based on summary statistics. There is no additional risks to participants.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Reilly, Muredach \",\n\"COLUMBIA UNIVERSITY HEALTH SCIENCES\",\n\"Genetic determinants of cardiovascular diseases across multiple de-identified biobank data sets\",\n\"Oct31, 2022\",\n\"approved\",\n\"Cardiovascular diseases are a serious and widespread threat to human health. A major goal of biomedicine is to better understand gene-disease associations. Analysis of rare genetic changes, especially potential changes that lead to a loss in function, can help us understand gene function and have implications for the development of medicines. Our groups, both based in the Columbia University Division of Cardiology, largely focus on cardiometabolic diseases such as atherosclerosis and heart attack. The MIGen data provides large-scale, high-quality information about DNA sequence for people who had a heart attack at unusually-young ages, and healthy comparison people who have not had a heart attack. We will use bioinformatic analysis methods to identify changes in DNA, including in sequence and copy number. We will use this analyzed data to look for associations of particular DNA changes or groups of changes with cardiovascular diseases (such as early-onset heart attack). Our existing data is in a South Asian population (Pakistan), and we would like to compare the results with a largely Caucasian population (MIGen) to see if racial and ethnic ancestry changes the strength or direction of effect of genetic changes. \",\n\"A major goal of biomedicine is to better understand gene-disease associations. Phenotypic analysis of rare functional variants, especially putative loss of function variants (PLoF), can help inform gene function and guide drug discovery efforts. Our groups, both based in the Columbia University Irving Medical Center Department of Medicine, Division of Cardiology, largely focus on cardiometabolic diseases such as atherosclerosis and myocardial infarction. Dr. Saleheen also serves as Scientific Director of the Pakistan Genomic Resource (PGR), which is a large-scale biobank (>180,000 participants) with genomic DNA, whole exome sequencing (>80,000 participants), basic biochemistry, anthropometric traits, EKG, and an extensive questionnaire that includes personal and family medical history, diet, tobacco use, alcohol consumption, and physical activity measures. The STAMPEED data provides large-scale, high-quality genotyping in a case-control study of early-onset myocardial infarction cases and matched controls. We will use bioinformatic pipelines to annotate variants of interest, carriers of rare functional variants, and to identify copy number variations. Using the annotations, we will perform genome-wide association tests on variants of interest as well as gene burden associations for rare PLoF, missense, and other functional variants to check for additive and recessive associations.   We will combine and compare the following datasets with those in this application: Pakistan Genomic Resource (PGR), UK Biobank, Biobank Japan, CARDIoGRAMplusC4D, All of Us, and the Munich Vascular Biobank. Together, these studies provide data on ~1.5 million participants (~300,000 cases) for genome-wide common variants. We will also perform ancestry analysis and meta-analysis to compare the effects of variants in the different populations, with two exceptions. We will not perform ancestry analysis for the datasets phs001101.v1.p1 (\\\"Malmo Diet and Cancer Study\\\") and phs001672.v8.p1 (\\\"Veteran's Administration (VA) Million Veteran Program (MVP) Summary Results from Omics Studies\\\") as both of these datasets bar such analyses.   Access to this dataset will aid the discovery and multi-ethnic validation of genetic mutations and copy number variants associated with cardiometabolic diseases. Project personnel using this data will publish or otherwise broadly share any findings from these studies with the scientific community, honoring all possible embargo dates associated with any publicly-available datasets.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Reitz, Christiane \",\n\"COLUMBIA UNIVERSITY HEALTH SCIENCES\",\n\"Shared heritability of cholesterol levels and non-Mendelian early-onst Alzheimer's Disease\",\n\"Apr07, 2020\",\n\"rejected\",\n\"We are requesting these data on lipid genetics to help us examine the role of cholesterol in Alzheimer's disease.\",\n\"Investigator:  Christiane Reitz, MD PhD Associate Professor of Neurology and Epidemiology Depts. of Neurology, Epidemiology, Sergievsky Center, Taub Institute for Research on the Aging Brain Columbia University 630 W 168th Street, New York, NY 10032  Phone: (212) 305-0865 Fax: (212) 305-2426 Email: cr2101@cumc.columbia.edu  Objective of the Proposed Research Alzheimer disease (AD) is the leading cause of dementia and occurs in all ethnic and racial groups with a wide range of onset age. Early-onset forms of AD (EOAD), characterized by presentation with first symptoms under age 65, represent as much as 10% of all AD. There is mounting evidence that, among EOAD, only 10% is explained by known mutations in the APP, PSEN1 and PSEN2 genes although previous studies have shown that EOAD is significantly enriched for genetic risk factors. Thus, while some genetic variants influencing EOAD have been identified, the majority of EOAD remains unexplained. One hypothesis for the etiology of EOAD is that other risk factors, such as vascular disease or neuropsychiatric phenotypes, increase susceptibility to AD and \\u00e2\\u0080\\u009cpush\\u00e2\\u0080\\u009d the age of onset earlier.    Study Design To begin assessing this hypothesis, the EOAD Working Group of the Alzheimer\\u00e2\\u0080\\u0099s Disease Genetics Consortium (ADGC; EOAD WG leaders: Reitz/Beecham/Cruchaga), will evaluate genetic correlations between vascular traits, neuropsychiatric traits, and EOAD capitalizing on available GWAS array data in over 13,000 meticulously phenotyped subjects with EOAD. While there is strong evidence that cardio- and cerebrovascular disease is associated with late-onset Alzheimer\\u00e2\\u0080\\u0099s disease, there is a paucity of data on their effect on age of onset of AD and contribution to early-onset AD. The analyses are designed to begin to assess the genetic association of EOAD and age at onset of AD with blood lipid levels, a major risk factor for vascular disease.   Aim1.  Evaluation of Genetic Correlations. To dissect the shared and distinct genetic architecture across EOAD subphenotypes (ie. EOAD with different blood lipid levels) we will apply LD score regression (LDSC). LDSC provides a principled framework to partition genetic covariance of complex traits from genome summary statistics while accounting for LD structure from reference populations, and is computationally tractable even at large sample sizes and genome-wide scope. To apply LDSC, we will remove individuals with genetic relatedness coefficient =0.05 and filter out structural variants and variants with missing rate >1%, a MAF < 1% in reference populations (gnomAD; 1000G), strand ambiguous SNPs, variants in regions with exceptionally long-range LD and variants in pericentromeric regions enriched for sequence gaps. We will then estimate genome-wide heritability and pairwise genetic correlations between vascular endpoints and the adjusted summary statistics for these subphenotypes. Results from LDSC will subsequently be further analyzed using GNOVA (GeNetic cOVariance Analyzer).  We will also use PRS, GTCA and MTAG to analyze the overall relationships between these traits.   Aim 2: Joint Analysis of Correlated Traits. If underlying genetic correlations exist between EOAD and blood lipid levels this is evidence that there may be loci that influence multiple traits. To assess this, we will perform a joint analysis of genetically correlated phenotypes employing MTAG (\\u00e2\\u0080\\u009cmulti-trait analysis of genome-wide association summary statistics\\u00e2\\u0080\\u009d) which allows for efficient meta-analysis of related traits, even when samples overlap. We will use MTAG to analyze variation across EOAD and lipid levels to identify variants or genes that are part of the causal pathway of EOAD and dyslipidemia traits. FUSION/TWAS and UTMOST will be employed to infer the overlap in tissue-specific gene expression between these diseases.   External Collaborators Gary Beecham, PhD Associate Professor Director, Division of Research Informatics in the Center for Genetic Epidemiology and Statistical Genetics, John P. Hussman Institute for Human Genomics Email: gbeecham@med.miami.edu  Carlos Cruchaga, PhD Professor of Psychiatry and Neurology  Director NeuroGenomics and Informatics  Knight-ADRC and DIAN Genetics Core leader Email: cruchagac@wustl.edu \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Reitz, Christiane \",\n\"COLUMBIA UNIVERSITY HEALTH SCIENCES\",\n\"Shared hereditability of peripheral artery disease and Non Mendelian early onset Alzheimer's disease\",\n\"Nov10, 2020\",\n\"rejected\",\n\"We are requesting these data on peripheral artery disease genetics to help us examine its role in Alzheimer's disease.\",\n\"Investigator:  Christiane Reitz, MD PhD Associate Professor of Neurology and Epidemiology Depts. of Neurology, Epidemiology, Sergievsky Center, Taub Institute for Research on the Aging Brain Columbia University 630 W 168th Street, New York, NY 10032  Phone: (212) 305-0865 Fax: (212) 305-2426 Email: cr2101@cumc.columbia.edu  Objective of the Proposed Research Alzheimer disease (AD) is the leading cause of dementia and occurs in all ethnic and racial groups with a wide range of onset age. Early-onset forms of AD (EOAD), characterized by presentation with first symptoms under age 65, represent as much as 10% of all AD. There is mounting evidence that, among EOAD, only 10% is explained by known mutations in the APP, PSEN1 and PSEN2 genes although previous studies have shown that EOAD is significantly enriched for genetic risk factors. Thus, while some genetic variants influencing EOAD have been identified, the majority of EOAD remains unexplained. One hypothesis for the etiology of EOAD is that other risk factors, such as vascular disease or neuropsychiatric phenotypes, increase susceptibility to AD and \\u00e2\\u0080\\u009cpush\\u00e2\\u0080\\u009d the age of onset earlier.    Study Design To begin assessing this hypothesis, the EOAD Working Group of the Alzheimer\\u00e2\\u0080\\u0099s Disease Genetics Consortium (ADGC; EOAD WG leaders: Reitz/Beecham/Cruchaga), will evaluate genetic correlations between vascular traits, neuropsychiatric traits, and EOAD capitalizing on available GWAS array data in over 13,000 meticulously phenotyped subjects with EOAD. While there is strong evidence that cardio- and cerebrovascular disease is associated with late-onset Alzheimer\\u00e2\\u0080\\u0099s disease, there is a paucity of data on their effect on age of onset of AD and contribution to early-onset AD. The analyses are designed to begin to assess the genetic association of EOAD and age at onset of AD with peripheral artery disease, and assess whether both traits have shared genetic heritability.   Aim1.  Evaluation of Genetic Correlations. To dissect the shared and distinct genetic architecture across EOAD subphenotypes (ie. EOAD with different blood lipid levels) we will apply LD score regression (LDSC). LDSC provides a principled framework to partition genetic covariance of complex traits from genome summary statistics while accounting for LD structure from reference populations, and is computationally tractable even at large sample sizes and genome-wide scope. To apply LDSC, we will remove individuals with genetic relatedness coefficient =0.05 and filter out structural variants and variants with missing rate >1%, a MAF < 1% in reference populations (gnomAD; 1000G), strand ambiguous SNPs, variants in regions with exceptionally long-range LD and variants in pericentromeric regions enriched for sequence gaps. We will then estimate genome-wide heritability and pairwise genetic correlations between vascular endpoints and the adjusted summary statistics for these subphenotypes. Results from LDSC will subsequently be further analyzed using GNOVA (GeNetic cOVariance Analyzer).  We will also use PRS, GTCA and MTAG to analyze the overall relationships between these traits.   Aim 2: Joint Analysis of Correlated Traits. If underlying genetic correlations exist between EOAD and blood lipid levels this is evidence that there may be loci that influence multiple traits. To assess this, we will perform a joint analysis of genetically correlated phenotypes employing MTAG (\\u00e2\\u0080\\u009cmulti-trait analysis of genome-wide association summary statistics\\u00e2\\u0080\\u009d) which allows for efficient meta-analysis of related traits, even when samples overlap. We will use MTAG to analyze variation across EOAD and lipid levels to identify variants or genes that are part of the causal pathway of EOAD and dyslipidemia traits. FUSION/TWAS and UTMOST will be employed to infer the overlap in tissue-specific gene expression between these diseases.   External Collaborators Gary Beecham, PhD Associate Professor Director, Division of Research Informatics in the Center for Genetic Epidemiology and Statistical Genetics, John P. Hussman Institute for Human Genomics Email: gbeecham@med.miami.edu  Carlos Cruchaga, PhD Professor of Psychiatry and Neurology  Director NeuroGenomics and Informatics  Knight-ADRC and DIAN Genetics Core leader Email: cruchagac@wustl.edu  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Rice, John \",\n\"WASHINGTON UNIVERSITY\",\n\"Replication and Polygenic Risk Score Analysis for Alcohol Use and Dependence\",\n\"Oct09, 2019\",\n\"approved\",\n\"Alcohol use disorders represent a major public health problem.  We will use the results from this this large-scale study to examine replication in various datasets.  Importantly, we will do separate analyses of European (EA) and African American (AA) samples comparing results from this study and COGA.  We will use all the results from this study to construct a genetic risk score separately for EUs and AAs and apply this to the data.\",\n\"We will use data with smoking and alcohol phenotypes including the case-control data from COGA (The Collaborative Study on the Genetics of Alcoholism) to examine replication in European Americans (EA) and African American (AA) for the GWAS significant findings for alcohol consumption and dependence found in this Million Veterans Project (MVP).  We will also generate Polygenic Risk Scores (PRS) for the EU and AA samples to examine proportion of variance explained for each phenotype.  We will compare the PRS derived from the EUs of the MVP to that derived from the AAs when compared in the AA sample.  We will also compare the PRS derived from the UK Biobank with that from the MVP in the EU sample, and do bivariate analyses using both scores.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Rice, Kenneth \",\n\"UNIVERSITY OF WASHINGTON\",\n\"NIH PRIMED Consortium Coordinated Application\",\n\"Dec05, 2022\",\n\"approved\",\n\"Polygenic risk scores (PRS), are a genetic estimate of a person\\u00e2\\u0080\\u0099s risk for specific diseases. PRS are calculated by comparing the genomic data of people with and without a particular disease.  Research shows that early approaches to calculating PRS, developed from mostly European ancestry populations, are not effective when used in diverse populations. Researchers have used available genomic datasets to develop the ability to calculate PRS for numerous conditions, such as diabetes, and to identify people who are at high risk. This allows clinicians to use PRS in combination with a person\\u00e2\\u0080\\u0099s lifestyle and environmental factors to tailor their medical management. The Polygenic RIsk MEthods in Diverse populations Consortium will use existing datasets from dbGaP and elsewhere to improve PRS by studying much larger numbers of non-European individuals and developing methods to better adjust for ancestry. The data will help the consortium to develop better methods for using datasets of different types, such as those with only summary-level data, or datasets with missing genotype data.\",\n\"The Polygenic Risk Methods in Diverse Populations (PRIMED) Consortium will use phenotypic and genomic data from existing datasets to generate and refine polygenic risk scores (PRS) for populations of diverse race, ethnicities, and genetic ancestry.  Uses of the requested data will include, but are not limited to:  developing PRS and improve the ability of PRS to predict health measures and disease risk across diverse populations;  developing new methods for genetic risk prediction across diverse populations; adjusting for measures of local and/or global ancestry in populations of diverse genetic ancestry;  integrating both summary statistics and individual-level datasets;  statistically imputing genotypes for individuals with incomplete data; PRIMED Consortium investigators will not use the requested data to investigate individual identity in any analyses. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, ancestry, or population history, that data will be excluded from analyses that address those issues as outcomes. In PRIMED, PRS will be developed across different domains, such as cardiometabolic diseases, diabetes, cancer, smoking, and inflammation. Datasets with disease-specific DULs will only be used in the development or improvement of PRS for the corresponding disease. We intend to publish or otherwise broadly share any findings from this study with the scientific community.   However, genomic summary results from datasets with a \\u00e2\\u0080\\u009csensitive\\u00e2\\u0080\\u009d designation will not be disseminated beyond publications to support study\\u00e2\\u0080\\u0099s conclusions. This request anticipates collaboration among PRIMED investigators from multiple institutions, who are named in an external collaborators lists and are submitting coordinated dbGaP data access requests. External Collaborators to this request consist of investigators eligible to enter into PRIMED Consortium-wide data sharing. A list of currently eligible PRIMED investigators can be found at https://primedconsortium.org/eligibility. Data from PRIMED may be combined with data from other studies to improve the power for novel genetic discoveries or other advances. We do not anticipate any additional risk to participants when combining datasets. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Richards, Brent \",\n\"SIR MORTIMER B. DAVIS JEWISH GEN HOSP\",\n\"Identification of causal genes for disease\",\n\"Dec06, 2022\",\n\"approved\",\n\"Human genetics can contribute to understanding diseases and providing insights into potential therapeutic targets. Indeed, 66% of new drugs approved by the FDA had evidence from human genetics supporting their efficacy.   Therefore, we propose utilizing the human genetics datasets available from the Million Veterans Program, the CHARGE Consortium and the GTex project to identify the genetic causes of disease and potential therapeutic targets.   We will apply these methods to the summarized information to identify disease-causing genes. First, we will use Mendelian randomization methods; second, we will use a statistical mapping strategy to identify causative genes. Third, we will integrate the results of these published studies with DNA sequencing to increase the probability that the identified gene is the causative gene.  This study is expected to help identify the genetic mechanism of the diseases and targets for new treatments. \",\n\"Objectives: We propose to use summary-level genome-wide association test statistics from published genome-wide association studies and individual-level data from expression studies to identify genes that play a causal role in diseases. We are motivated to do so because genes that play a causal role in diseases are favourable targets for the development of medicines to treat such diseases. Here we propose to be granted access to summary-level results from genome-wide association studies (GWAS) and individual-level data from expression studies to fulfill this goal. We will also not post these summary-level results in any repository. Thus, the risks to participants due to data handling and analysis are reduced.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Richards, John Brent \",\n\"FIVE PRIME SCIENCES INCORPORATED\",\n\"Identifying genetic causes of common diseases to accelerate development of therapeutics\",\n\"Apr11, 2022\",\n\"expired\",\n\"To develop new and effective therapies for diseases, it is important to understand what has caused the disease. In some cases, the cause of the disease may be due to the genetic effects. Identifying these specific genes using human genetics can therefore help to identify the targets for medicines that could be used in the future to help treat disease.  We are therefore proposing to use the human genetic datasets that have already been published by the CHARGE consortium and the Million Veterans Program to identify the genetic causes of disease. We are not requesting to have information for each individual person, rather, we are only requesting access to the summarized information for the relationship between genetic variation and risk of disease.  We will use this summarized information to apply three methods to identify genes causing diseases. First we will use Mendelian randomization, second we will use statistical mapping strategies to identify causal genes and third, we will converge results from these published studies with DNA sequencing studies to increase the probability that the identified genes are causal.  Success in this program could help to identify the targets for new therapies for the diseases studied by the CHARGE consortium and the Million Veterans Program. \",\n\"Objectives: We propose to use summary-level genome-wide association test statistics from published genome-wide association studies to identify genes that play a causal role in diseases. We are motivated to do so because genes that play a causal role in diseases are favourable targets for the development of medicines to treat such diseases. Here we propose to be granted access to summary-level results from genome-wide association studies (GWAS) to fulfill this goal. We do not request access to individual-level data. We will also not post these summary-level results in any repository. Thus, the risks to participants due to data handling and analysis are reduced.  Combining datasets: We will not combine datasets. However, we will qualitatively summarize results from separate datasets, including those in this dbGAP application with other types of data. For example, we will use exome sequencing summary-level data from other cohorts and test if the results from the GWAS summary statistics we are requesting in this dbGAP application are consistent across both datasets.   Different hypotheses and outcomes: We are not focusing on outcomes or hypotheses that were not the focus of the original studies.  Study Design: Our study design involves using already generated GWAS summary statistics to identify causal genes at implicated loci by undertaking three different Aims. In our first Aim, we will use Mendelian randomization (MR) analyses. Such analyses test the effect of an exposure, such as a protein, or gene expression product on the etiology of a disease using genetic variants as instrumental variables. The main advantage of using such an approach, is that it can estimate the causal effect of an exposure, such as a particularly protein level, upon a disease outcome.  In our second Aim, we will use a method that we have recently developed to rank the probability of causality of any gene at a GWAS locus, called the Effector Index. This index provides a probability of causality between zero and one for each gene at a GWAS locus, thus allowing for a prioritization of these genes at associated loci.   For our third Aim, we will combine GWAS summary statistics with results from whole-exome sequencing studies to generate synergistic evidence, by combining these two types of datasets, to further increase confidence that a particular gene influences disease incidence.  We will undertake analyses across the outcomes that have been published through the CHARGE Consortium and the Million Veteran\\u00e2\\u0080\\u0099s Program GWASs.  Brief Analysis Plan: Aim 1: MR. We will use human genetics to identify protein and expression markers using MR\\u00e2\\u0080\\u0094an established genetic epidemiology method. In MR, SNPs are used as proxies for modifiable exposures, such as protein and expression biomarkers. MR circumvents inherent limitations of traditional observational studies, such as confounding and reverse causation. As such, MR can provide evidence supporting, or contradicting, the causality of biomarkers in disease susceptibility.  Aim 2: Causal genes at GWAS loci. We will use different methods to identify potentially causal genes at GWAS loci. These include the Effector Index, our recently published algorithm to meet this need, as well as activity by contact, Pops, Locus2Gene mapping and other recently published methods to identify causal gene(s) at GWAS loci. We will then compare these approaches and qualitatively synthesize results. We will also compare these methods to known positive control genes for common diseases, such as Mendelian causes of rare forms of the disease, as well as targets of successfully developed medicines.  Aim 3: Combining results with Exome Sequencing. Last, we will compare results from Aim 1 and Aim 2 to recently published large-scale exome sequencing programs, such as those generated from UK Biobank. When there is convergence of MR results, highly probably genes at GWAS loci and exome sequencing, such targets are likely to be causal for the disease in question.   Summary By using summary-level GWAS test results, we hope to use the above tools to identify high-yield targets for drug development that could render new therapies for the diseases that have been studied by the CHARGE consortium and the Million Veterans Program. Doing so, could help to alleviate the morbidity and mortality caused by these diseases, thereby realizing in part the goals of these research programs. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Rietschel, Marcella \",\n\"CENTRAL INSTITUTE OF MENTAL HEALTH\",\n\"Analysis of shared and specific genetic risk of Post-traumatic Stress Disorder and Borderline Personality Disorder\",\n\"Feb24, 2022\",\n\"approved\",\n\"Post-traumatic Stress Disorder (PTSD) and Borderline Personality Disorder co-ocour often, and share common risk factors, such as the experience of traumatic events. We will use the results generated by the Million Veteran Program on PTSD to further investigate shared and distinct disorder mechanisms of the two disorders.  \",\n\"Post-traumatic Stress Disorder (PTSD) and Borderline Personality Disorder (BPD) have comorbid presentations and shared features. Molecular genetic analyses indicate both are highly polygenic disorders. Studying the common and distinct genetic risk factors of both disorders, represents a promising approach to understand shared and specific biological and behavioural disorder mechanisms. We have carried out the largest genome-wide association study (GWAS) on BPD. In the proposed project, we will use summary statistics of GWAS of PTSD and its symptom subdomains carried out in the Million Veteran Program (MVP) (Stein et al., 2021, Nature Genetics) to investigate shared (and distinct) genetic factors of PTSD with BPD. PTSD summary statistics from MVP and BPD summary statistics will be analyzed in regard to their association with each other, and with other related GWAS data, e.g. comorbid mental disorders, or relevant risk factors (e.g. early trauma) (e.g. genetic correlations, GenomicSEM). Additionally, additional analysis will be performed to leverage the shared genetic risk to facilitate discovery of (shared) genetic risk factors (e.g. Multi-trait analysis (MTAG), colocalization analysis). \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Rietschel, Marcella \",\n\"CENTRAL INSTITUTE OF MENTAL HEALTH\",\n\"Analysis of shared genetic risk factors for psychiatric disorders\",\n\"Sep01, 2021\",\n\"approved\",\n\"Psychiatric disorders are often comorbid and are thought to have shared genetic underpinnings. We plan to use results generated by the Million Veteran Program to explore the relationship between comorbid disorders such as depression and addiction. \",\n\"Many psychiatric disorders have comorbid presentations and shared features. Major depressive disorder (MDD) is a serious and complex illness with heterogeneous presentation and high polygenicity. Our previous and ongoing research has used genome wide association techniques in major depressive disorder and other psychiatric disorders, examining questions of comorbidity and shared genetic factors between disorders.  We plan to use summary statistics generated by genome-wide association studies (GWASs) from the Million Veteran Program (MVP), to study how MDD relates to different disorders at the genetic level. MDD summary statistics from MVP will be analysed together with other GWASs and genetic data to investigate disease comorbidity (e.g., with Bipolar Disorder and Addiction), using a variety of genomic analyses techniques such as genetic correlation, polygenic risk scoring and other post-hoc genomic techniques.   \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Ripke, Stephan \",\n\"CHARITE, UNIVERSITAETSMEDIZIN BERLIN\",\n\"C4 imputation on schizophrenia cohorts of the Psychiatric Genomics Consortium (PGC)\",\n\"Sep30, 2022\",\n\"approved\",\n\"The complement component 4 (C4) of the Major Histocompatibility Complex (MHC) region in chromosome 6 exists as two functionally distinct genes (isotypes), C4A and C4B; both vary in structure and copy number. C4 variation has been shown to play a role in schizophrenia risk. We will try to get a more precise picture of the C4 related risk using an updated and much larger data collection from schizophrenia patients and healthy individuals.\",\n\"We will use the reference panel to impute the core cohorts of the Psychiatric Genomics Consortium (PGC) schizophrenia working group, replicating the results from Sekar et al. \\\"Schizophrenia risk from complex variation of complement component 4\\\"(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4752392/). This updated collection comprises 90 cohorts of European (EUR) and East Asian (ASN) ancestry from the PGC, totalling 67,390 cases and 94,015 controls.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"RITCHIE, MARYLYN \",\n\"UNIVERSITY OF PENNSYLVANIA\",\n\"Exploring the genetic architecture of complex traits through genome-wide, phenome-wide, and polygenic risk analyses\",\n\"Jan25, 2021\",\n\"approved\",\n\"Our goal is to identify the role of genetics in complex human disease.  In some diseases, we expect a single gene, or a small set of genes, will be important for disease risk.  In other diseases, we expect that it is a combination of numerous genes across the genome that work together to lead to increased disease risk.  Additionally, because we observe that people often experience multiple diseases throughout their lifetime, often at the same time, we want to improve our understanding of the relationship between diseases at the genetic level.  This means, there may be specific genes or biological pathways that influence multiple parts of the body at the same time and lead to multiple diseases presenting in an individual.  In our study, we want to explore this across people will common diseases, like hypertension, type 2 diabetes, or elevated blood lipids, and improve our understanding of the role that genetics plays in these disease areas.  \",\n\"One of the primary goals in precision medicine research is to identify the underlying genetic architecture of complex traits.  There are a number of study designs and analysis designs that have been developed to conduct research in this domain.  Genome-wide association studies (GWAS), Phenome-wide association studies (PheWAS), and Polygenic Risk Scores (PRS) are three analysis designs that are commonly used.  Integrating the results across these different analyses can be incredibly useful to begin to deconstruct complex trait architecture.  Looking for which genes associate across multiple diseases or disease categories, or does the PRS score for one trait associate with other disease categories \\u00e2\\u0080\\u0093 these are common questions that we are interested in for understanding complex traits.  In this project, we would like to access the MVP GWAS Summary statistics to integrate with summary statistics and data from other datasets that we are currently using for phenome-wide analyses.  First, the Penn Medicine BioBank (PMBB) is a medical biobank from Penn Medicine that includes genetic data and EHR data.  Second, the UK Biobank is a population-based research study from the United Kingdom; around 500,000 participants have genetic data and clinical data available for analysis.  Third, the electronic Medical Records and Genomics network (eMERGE) is an NHGRI funded initiative that combines electronic health record and genetic data from over 125,000 participants across ten health systems in the United States.  In our study, we plan to conduct PheWAS meta-analyses to look for pleiotropy (where one gene or PRS score is associated with multiple diseases), and PRS analyses to look for variability in PRS performance by age, sex, ancestry; and to look for biological pathways that overlap across different disease categories.  The MVP summary statistics will be incredibly useful for meta-analysis with our existing datasets and generating PRS to validate in our datasets where we have access to de-identified, individual level data. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"RIVAS, MANUEL \",\n\"STANFORD UNIVERSITY\",\n\"Large-scale inference of population cohorts\",\n\"Oct28, 2019\",\n\"closed\",\n\"We plan to combine summary statistics from MVP cohort with other existing summary statistics to improve power to detect genetic associations, to assess multi-phenotype patterns of genetic associations, and to improve polygenic risk scores. We plan to visualize our association results in Global Biobank Engine (https://biobankengine.stanford.edu). We will use our own compute resources and will follow procedures outlined in NIH communication (NOT-OD-19-023) for using summary statistic data from human genetic studies. \",\n\"We plan to combine summary statistics from MVP cohort with other existing summary statistics to improve power to detect genetic associations, to assess multi-phenotype patterns of genetic associations, and to improve polygenic risk scores. We plan to visualize our association results in Global Biobank Engine (https://biobankengine.stanford.edu). We will use our own compute resources and will follow procedures outlined in NIH communication (NOT-OD-19-023) for using summary statistic data from human genetic studies.   We have developed novel statistical methods for analyzing summary statistics across multiple variants, multiple phenotypes, and multiple studies (https://www.biorxiv.org/content/10.1101/257162v5, https://www.biorxiv.org/content/10.1101/630079v1). These will be applied to the data. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Robinson-Cohen, Cassianne \",\n\"VANDERBILT UNIVERSITY MEDICAL CENTER\",\n\"Fetuin-A and Diabetes: A Mendelian Randomization Study\",\n\"Mar01, 2022\",\n\"closed\",\n\"The protein fetuin-A has been linked with risk of diabetes, but despite intensive investigator it's unclear whether this association is one of cause and effect. The proposed study has direct significance for understanding pathologies related to diabetes and may lead to the identification of novel drugs and therapies to prevent this prevalent condition.\",\n\"Observational studies have shown that fetuin-A, a hepatic-origin protein, is associated with risk of type 2 diabetes, but it is unknown whether this association is causal. We propose to study this association using a Mendelian randomization study with single nucleotide polymorphisms within the fetuin-A\\u00e2\\u0080\\u0093encoding AHSG gene, looked up within Million Veteran Program diabetes GWAS. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"ROMANOSKI, CASEY \",\n\"UNIVERSITY OF ARIZONA\",\n\"Fine-Mapping CAD GWAS MVP loci using endohtelial cells molecular data\",\n\"Feb08, 2023\",\n\"approved\",\n\"In this study, we propose to investigate and identify which variations in DNA pose risk of developing Coronary Artery Disease (CAD) through changing how blood vessel cells work. Our lab has generated a lot of information about the molecular properties of blood vessel cells, under healthy and disease-prone conditions. We will utilize this information together with information from the statistical results of the large population-based study of CAD risk. Together, our goal is to discover specific ways in which blood vessel cells modulate the progression of CAD. \",\n\"The objectives of this project will be to fine-map genetic signals underlying CAD with molecular etiology in vascular endothelial cells. The study design is to intersect summary statistics from the CAD GWAS with genomic loci of interest in endothelial cells to nominate causal SNPs, regulatory elements, and target genes that modulate susceptibility to CAD. Methods used to intersect GWAS and endothelial data will take several forms, including by genomic position as well as using statistical methods like co-localization. Loci of interest from endothelial cells will be defined as those that: 1) demonstrate accessibility in cell-type and activation state-specific patterns; 2) have molecular QTLs for accessibility, TF binding or histone methylation; and 3) have eQTLs. We will not utilize any personally identifiable information in our analysis and will work solely with summary statistics. Therefore, we will not pose any risks to participants. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"ROSAND, JONATHAN \",\n\"MASSACHUSETTS GENERAL HOSPITAL\",\n\"Joint Genetic Contribution to Vascular-Related Phenotypes \",\n\"Dec28, 2020\",\n\"approved\",\n\"Vascular conditions (myocardial infarction and stroke) constitute one of the leading causes of death and disability worldwide. Several recent genetic studies demonstrate that genetic variation has an important role in the occurrence from these diseases.  The main feature of modern genotyping technologies is their impressive ability to generate massive amounts of information related to genetic variation across the genome. While these studies have been very successful in identifying genetic variants that increase the risk of each vascular condition considered separately, interesting opportunities are available to discover genetic variants that contribute to causing vascular conditions in general. The aim of this proposal is to combine data from several studies on different vascular diseases in order to uncover these variants. We seek to add value to already existing resources, leveraging the notion that a single entity, blood vessels, is affected in vascular diseases, regardless of the final organ involved (the heart in myocardial infarction and the brain in stroke).\",\n\"Biologically related conditions share common genetic predisposing factors. This is relevant for vascular conditions, as they share a single anatomical target (blood vessels), final mechanism of injury (occlusion or rupture of vessels) and predisposing non-genetic risk factors (diabetes, hypertension, hypercholesterolemia, smoking, obesity). Large amounts of data are available for vascular conditions, risk factor and related endophentoypes. Likewise, novel informatic and statistical methods designed to analyze this data are available regularly. We intend to publish/broadly share our findings with the scientific community when available.  The aims of this proposal are: 1) to evaluate whether genetic architecture of vascular conditions and their risk factors overlap; 2) develop/test novel analytical techniques designed to assess joint genetic contribution to vascular outcomes and risk factors; 3) identify genetic loci that contribute to these traits; 4) assess if loci identified in #2, (separately/in aggregate) improve prediction models for occurrence, severity and outcome of vascular phenotypes; 5) to develop tools to manipulate/automate genetic and phenotypic data related to vascular phenotypes  We will combine sequencing, genotype + phenotype data related to several of ischemic vascular phenotypes (myocardial infarction, ischemic stroke, acute peripheral occlusion), hemorrhagic vascular phenotypes (hemorrhagic stroke), non-vascular phenotypes with known vascular component (Alzheimer\\u00e2\\u0080\\u0099s Disease), vascular risk factors (diabetes, hypertension, hypercholesterolemia, smoking, obesity) and relevant endophenotypes (white matter hyperintensities). In this combined dataset we plan to conduct single-variant, gene-based, pathway-based, polygenic risk score, and single-trait and joint heritability analyses, as well test newly developed statistical tools aimed at comparing genome-wide data across related phenotypes.   Data is individually/jointly analyze with data outside of dbGaP. External dataset may be meta-analyzed with dbGaP data or serve as replication cohorts. This plan will not create additional risk. Data use is consistent with all set Data Use Limitations and will not be combined with other phenotypes. We intend to publish results and share findings.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Ross, Elsie \",\n\"STANFORD UNIVERSITY\",\n\"Development and validation of peripheral artery disease polygenic risk scores\",\n\"Apr06, 2020\",\n\"approved\",\n\"Peripheral artery disease (PAD) affects 8-12 million Americans and has an estimated annual health care cost of $21 billion in the United States. PAD increases risk of heart attack, stroke, and sudden cardiovascular death but often goes undiagnosed. Because prevention measures can significantly improve long-term outcomes in PAD, low rates of diagnosis represent missed opportunities to prevent costly and deadly consequences of disease. Thus, there is a critical need to develop methods to improve PAD risk assessment, diagnosis and treatment. Our overall objective in using the MVP and ARIC data sets is to develop a first ever PAD-specific polygenic risk score and evaluate the utility of incorporating this score into existing clinical data. A polygenic risk score will allow us to combine multiple risk factors identified in the genome into a summary score that can be used to identify who is at high or low risk of PAD.  \",\n\"Peripheral artery disease (PAD) affects 8-12 million Americans and has an estimated annual health care cost of $21 billion in the United States. PAD confers a 2 to 6-fold risk of major adverse cardiovascular events (MACE) but often goes undiagnosed. Because secondary prevention measures can significantly improve long-term outcomes in PAD, low rates of diagnosis represent missed opportunities to prevent costly and deadly consequences of disease. Thus, there is a critical need to develop methods to improve PAD risk assessment, diagnosis and treatment. Our overall objective in using the MVP and ARIC data sets is to develop a first ever PAD-specific polygenic risk score, validate its performance and evaluate the utility of incorporating this score into an EHR-based PAD risk assessment tool.  PAD-specific PRSs have not previously been developed due to difficulty in identifying risk alleles unique to PAD. Recently the MVP GWAS for PAD was published and is of unprecedented size, making it now possible to calculate potentially useful PAD-specific PRSs. Summary statistics for PRS development will be derived from the latest PAD GWAS study, conducted using data from the Million Veterans Program, which includes >31,000 PAD cases and >211,000 controls. Using the MVP GWAS study we will develop PAD PRSs using Bayesian polygenic prediction algorithms. Fit of PRSs will be assessed with AUC and pseudo-R2 metrics. These data will then be combined with our existing electronic health record model to evaluate whether PRS improves PAD risk classification. After developing multiple well-performing PRSs we will validate their performance and utility using data from The Atherosclerosis Risk in Communities (ARIC) Study.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"ROVARIS, DIEGO \",\n\"UNIVERSITY OF SAO PAULO\",\n\"The Biological Relationship between ADHD and Depression in Contrasting Exposomes\",\n\"Jun06, 2022\",\n\"approved\",\n\"In psychiatry, it is common to diagnose patients with more than one mental health issue. Clinical depression, for example, is often accompanied by ADHD diagnosis (attention deficit hyperactivity disorder). With novel genomic tools, it is possible to find genetic signals that correlate with the presence of mental health issues and may help explain what biological mechanisms are behind them. The environment and society in which these patients are inserted are also of extreme importance since it plays an important role in the well-being and mental health of an individual. In our study, we aim to look deeper into the biology behind the correlation between depression and ADHD looking for genetic signals that are shared between the two, signals unique to depression and signals unique to ADHD. We would like to understand if and how the environmental variables influence these diagnosis as well as which one of these are most relevant. We will compare data from volunteers in our country, Brazil, to data that has been collected by many other countries in the world exploring the differences and similarities between genomics, environment, and diagnosis. \",\n\"This statement has been written to request access to the summary statistics resulting from the Genome Wide Association Study performed by Levey et. al in \\u00e2\\u0080\\u009cBi-ancestral depression GWAS in the Million Veteran Program and meta-analysis in >1.2 million individuals highlight new therapeutic directions\\u00e2\\u0080\\u009d (phs001672.v1.p).  The objective of our proposed research is to better understand the biological association between Major Depressive Disorder (MDD) and Attention Deficit and Hyperactivity Disorder (ADHD) in a clinical ADHD sample collected in Porto Alegre, Brazil, by the Adult ADHD Outpatient Program (PRODAH) in the hospital of the Federal University of Rio Grande of Sul. The sample consists of \\u00cb\\u009c600 ADHD cases and \\u00cb\\u009c600 controls with extensive clinical, sociodemographic, and biological data (e.g., GWAS, EWAS and neuroimaging) additional to longitudinal data in two waves over thirteen years of clinical follow-up.  We plan on using the beforementioned summary statistics to perform 1) polygenic risk score analysis in our sample to explore the biological relationships between ADHD and depression as well as potential synergism between the ADHD and depression PRSs on the severity of ADHD. Additionally, we will 2) explore local correlations between ADHD and depression (using only summary statics data) to find the genomic regions that better explain the comorbidity between ADHD and depression and evaluate them in our clinical sample. Since we request the summary statistics, which does not identify the participants information, our research will be consistent with the data usage limitations.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Ruigrok, Ynte \",\n\"UNIVERSITY MEDICAL CENTER UTRECHT\",\n\"Determining the phenotypical effects of proteins associated with abdominal aortic aneurysm\",\n\"Apr07, 2022\",\n\"closed\",\n\"We request access to the aggregated GWAS data on AAA, which will be used to prioritize promising protein drug targets using in silico experimentation.\",\n\"We will employ the principle of genetically guided drug development to identity and prioritize druggable proteins for an association with abdominal aortic aneurysm (AAA) and related cardio-metabolic traits relevant to determine the benefit/harms profile of on-target perturbation.   Specifically, we propose to leverage in-house developed software and statistical tools for drug target Mendelian randomization (MR), colocalization, and polygenic scores (PGS), to identify drug target (tissue specific splice site, mRNA expression, or protein concentration) effects on the before mentioned outcomes. Targets will be further prioritized by identifying potential repurposing opportunities through linkage with ChEMBL, and the British National Formulary (BNF). Tissue specificity will be assessed sourcing Human Protein Atlas (HPA) data, and target pathways explored by interfacing with REACTOME.   The phenotypical consequences of prioritized drug targets will be shared to inform (de novo) drug development through open access scientific publications, presentations, and through dissemination of a drug target catalogue, concatenating evidence the previously described analyses.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"SALEM, RANY \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Genetic Association Analyses of Sequencing Data, Complex Traits and Methods Development Using Biomedical Repositories\",\n\"Jul27, 2022\",\n\"approved\",\n\"The goal of this project is to better understand how genes influence disease and traits. Improved understanding of how genes influence human biology can help prevent disease, reduce severity, and improve quality of life. We are requesting data for many genetic sequencing studies and will analyze them separately and in combination, using improved statistical methods that we and others develop. We will study body size and shape, related childhood growth traits, the aging process, asthma and lung function, diabetes and metabolic syndrome including related traits and their complications, blood measures (including cholesterol and metabolic products), kidney function, immune disorders and complications, cancer, heart and vascular disease and related traits, and how gene activity differs by tissue. We will ask how genes influence traits over time, use genetics to assess cause and effect, predict disease risk, study how genes influence multiple traits, and analyze genetic factors individually and in combination. Results of this project will be presented at scientific meetings and published in journals.\",\n\"*Research Use Statement (RUS)  In this project, we aim to improve understanding and provide insights into the genetic architecture of diseases and complex traits utilizing whole exome- and whole genome- sequencing from multiple cohorts in dbGaP. The application and improvement in both statistical methodology and study design approaches is an important path to accomplish this goal. Improving insights into the genetic basis of complex traits could help guide new therapies. We believe this project is well aligned with the goals of dbGAP. We are requesting data for a large number of study cohorts, that will be used separately and in combination (integrating genotype, expression, phenotype and sequence data for a specific trait) to achieve these goals. To maximize the utility of these datasets, we propose to explore methodological issues that can inform the association analysis of select phenotypes: anthropometric phenotypes, childhood growth, the aging process, asthma and lung-function, diabetes and glucose metabolism, metabolic syndrome and its components, diabetes and MetSy morbidity, cognitive function, hematological and biochemical measures (including lipid and metabolites), kidney function and disease, rheumatological disorders and musculoskeletal complications, cancer and related disorders, and cardiovascular function and disease, including BP and HTN. Several important genetic association questions will be explored with existing and novel statistical genetics techniques in the following areas: Longitudinal Analysis, Mendelian Randomization, Risk prediction, multi-traits and Pleiotropy, eQTL, and Statistical Genetics. Access to the data will be limited to members of the Salem lab and labs of listed collaborators (all affiliated with UCSD) who have completed both human subjects and data safety training. All users will abide to terms of data use certification restrictions and limit analyses accordingly, and will honor all publication embargo dates. Findings will be disseminated and shared with the scientific community by presenting results at national scientific meetings and publishing in peer-reviewed journals. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Sanchez Roige, Sandra \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Genetic studies of substance use disorders\",\n\"Jun29, 2022\",\n\"approved\",\n\"We aim to investigate the genetics of substance use disorders (SUD) and its overlap with other comorbid conditions. We will use the results generated by the Million Veteran Program and the iNDiGO Consortium to further our understanding of how polygenic liability influences different substance use disorder phenotypes measured across cohorts that differ in their ascertainment (population- and hospital-based cohorts), and how it overlaps with other known comorbid conditions, such as impulsivity, externalizing, chronic pain. This work will inform genetic studies of substance use disorders and the complex web of medical comorbidities associated with such a debilitating condition.\",\n\"We are requesting to use GWAS summary statistics from various sources (the Million Veterans Program, MVP, and the iNDiGO consortium) to investigate the genetic signal of many substance use disorders (SUDs), and its correlation with other comorbid traits. We have assembled multiple substance use phenotypes, ranging from exposure to chronic use, dependence and misuse, in several population- and hospital-based cohorts, and performed genome-wide association studies (GWAS) of each of the measured phenotypes (not available in dbGaP). We will use standard statistical tools for polygenic analyses (e.g. polygenic risk scores, LDSC genetic correlations) to calculate the amount of genetic overlap across the different SUD definitions and other comorbid traits, such as chronic pain. We will also seek to perform meta-analyses of the correlated traits. We do not plan to combine datasets but rather use the provided summary statistics alongside our summary information. Phenotypes will include summary statistics for substance use disorders (opioids, alcohol, tobacco, cannabis, cocaine) and comorbid traits, such as impulsivity, gambling, externalizing. The use of this data is requested solely for health, medical and biomedical purposes and will not involve any commercial applications or patents. We will only be downloading the summary statistics and thus do not anticipate any additional risk to participants. The summary statistics will be used only within the PI\\u00e2\\u0080\\u0099s institution. We will publish or broadly share any findings with the scientific community. The proposed research is consistent with the Use Restrictions for the requested dataset, and we will follow all regulations therein. This work will validate alternatives to rigorous clinical ascertainment for genetic studies of substance use disorders, which could serve to achieve large sample sizes in a cost-efficient manner, advancing our understanding of substance use disorders. This work will also examine the genetic overlap between substance use disorders and other comorbid conditions, which could have significant clinical implications. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Sanchez Roige, Sandra \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Genetic studies of opioid use disorders\",\n\"Dec09, 2020\",\n\"closed\",\n\"Genetic factors influence opioid use disorders, but genetic studies of opioid use disorders have been severely limited due to the complexity of assessing opioid misuse. We will use the results generated by the Million Veteran Program to further our understanding of how polygenic liability influences different opioid use phenotypes measured across cohorts that differ in their ascertainment (population- and hospital-based cohorts). This work will inform genetic studies of opioid use disorders that use questionnaire-based or hospital-based ascertainment as a complementary alternative to traditional clinical ascertainment.\",\n\"Genetic factors contribute to variability in opioid use and abuse, but studies of opioid use disorders have been severely limited due to insufficiently powered samples, high phenotypic heterogeneity, and controls with undocumented exposure history. We have assessed multiple opioid use phenotypes, ranging from exposure to chronic use, dependence and abuse, in several population- and hospital-based cohorts, and performed genome-wide association studies (GWAS) of each of the measured phenotypes (not available in dbGaP). We are requesting to use GWAS summary statistics from the Million Veterans Program (MVP) to determine how genetic variation captured by our different opioid use definitions contributes to individual differences in opioid use disorders as defined by the MVP, which will serve as the \\u00e2\\u0080\\u009cgold standard\\u00e2\\u0080\\u009d. The objectives of this project are two-fold. First, we will use standard statistical tools for polygenic analyses (e.g. polygenic risk scores, LDSC genetic correlations) to calculate the amount of genetic overlap across the different opioid definitions. Next, we will use the MVP data to investigate whether the findings derived from our samples replicate in the MVP cohort. We do not plan to combine datasets but rather use the provided summary statistics alongside our summary information. Phenotypes will include summary statistics for opioids and comorbid traits, such as impulsivity, gambling, externalizing. The use of this data is requested solely for health, medical and biomedical purposes and will not involve any commercial applications or patents. We will only be downloading the summary statistics and thus do not anticipate any additional risk to participants. The summary statistics will be used only within the PI\\u00e2\\u0080\\u0099s institution. We will publish or broadly share any findings with the scientific community. The proposed research is consistent with the Use Restrictions for the requested dataset, and we will follow all regulations therein. This work will validate alternatives to rigorous clinical ascertainment for genetic studies of opioid use disorders, which could serve to achieve large sample sizes in a cost-efficient manner, advancing our understanding of opioid use disorders.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Schmitz, Lauren \",\n\"UNIVERSITY OF WISCONSIN-MADISON\",\n\"The Impact of Socioeconomic Environments and Polygenic Risk on Alcohol Use\",\n\"Apr07, 2020\",\n\"closed\",\n\"Although there are very few genetic differences between people, researchers have been able calculate which of these small differences in people\\u00e2\\u0080\\u0099s DNA are related to certain health behaviors like drinking. When aggregated, these small differences form a polygenic risk score (PGS). PGSs can help researchers understand whether people are predicted to be at a greater risk for alcohol abuse based on their DNA. Our study will use these PGSs to understand how they interact with social environments and experiences (e.g., job loss) to shape alcohol consumption or abuse. For example, the degree to which job loss affects alcohol use may vary by an individual\\u00e2\\u0080\\u0099s underlying genetic predisposition. \",\n\"Social science research has rapidly incorporated genetic data into quantitative analyses to better understand how the small DNA variations between people can affect health. For many health behaviors, researchers have found null results of the effects of major life events (e.g., job loss, divorce, retirement) on health behaviors, such as drinking. However, with the advent of polygenic risk scores, researchers can examine if there is variability in drinking behaviors based on genetic risk.   This study will utilize gene by environment (GxE) methods to assess how different environmental exposures related to socioeconomic status affect drinking behaviors when interacted with an alcohol consumption polygenic score (PGS). The GWAS weights from the Million Veterans Project for alcohol use will be utilized to build whole-genome PGSs and incorporated into GxE statistical frameworks to examine whether drinking behaviors among young adults in the Add Health study and older adults in the Health and Retirement Study (HRS) vary by underlying genetic predisposition. The environmental exposures will be related to socioeconomic status and life events, while the phenotypes will be alcohol consumption or an indicator for alcohol use disorder (AUD).  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Schork, Andrew \",\n\"REGION HOVEDSTADEN\",\n\"Characterizing pleiotropy between cardio-metabolic and psychiatric phenotypes\",\n\"Oct22, 2020\",\n\"approved\",\n\"Psychiatric patients are known to exhibit excessive cardio-metabolic comorbidity which can lead to premature mortality and exacerbate poor health outcomes.  It is critical to understand the etiology of these comorbidities as this could improve care of an already at-risk population.  We will integrate results of large genetic studies of cardio-metabolic and psychiatric traits in the MVP study with a prospective longitudinal cohort of population controls and psychiatric patients to better understand the genetic contributions to cardio-metabolic health of psychiatric patients.\",\n\"We aim to systematically and comprehensively describe the shared genetic etiology between psychiatric and cardio-metabolic phenotypes.  We will use state-of-the-field and novel methods for assessing the overall genetic sharing (i.e. genetic correlation), testing the cross-predictive implications for (i.e., polygenic risk score analysis), and mapping individually pleiotropic loci (i.e., GWAS-like approaches) for cardio-metabolic and psychiatric phenotypes.   We will also look at predictors and mediators of longitudinal psychiatric and cardiovascular outcomes of individual population controls and psychiatric patients from the iPSYCH case-cohort study.  This study will integrate an extensive, broad catalog of GWAS summary statistics that include contributions from the \\u00e2\\u0080\\u009cVeterans Administration (VA) Million Veteran Program (MVP) Summary Results from Omics Studies\\u00e2\\u0080\\u009d project (phs001672.v4.p1), additional public GWAS data sources, and in-house data generated as part of the iPSYCH case-cohort study (PMID: 28924187).  We will pursue purely summary statistics comparison-based analysis to describe broad genetic sharing among these and other cardio-metabolic and psychiatric GWAS summary results.  We will also use the data obtained here to calculate polygenic risk assessments for the iPSYCH data cohort to study determinants of and predictors of heterogeneity for longitudinal psychiatric and cardio-metabolic outcomes.  This study will push forward our understanding of the etiology of excessive cardio-metabolic co-morbidities among psychiatric patients, extending the biobank\\u00e2\\u0080\\u0099s aims towards improved understanding health and disease in veterans to other populations, consistent with the aims of this resource..  This research will be conducted in-house as described above, with no plans to share or use data more broadly, and is in accordance with the Use Restrictions for this resource and will follow all regulations regarding data security, IP, and publication.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Schwarz, Emanuel \",\n\"CENTRAL INSTITUTE OF MENTAL HEALTH\",\n\"Identification of Genetic Schizophrenia Subgroups\",\n\"Feb09, 2023\",\n\"approved\",\n\"Psychotic disorders are among the most serious mental illnesses and major causes of disability, absence from work and early retirement. A major problem is that current diagnostic criteria are based on symptom and clinical course criteria, which, almost by definition, do not map to the underlying biology. As a consequence, genetic illness associations regularly cross diagnostic boundaries and are hallmarked by small effect sizes due to the polygenic disease architecture and heterogeneity of diagnostic constructs. In this project, we will use the dbGaP data to identify and validate biological subgroups of schizophrenia patients. We will investigate the polygenic nature of such subgroups and explore their relevance for brain function using brain imaging data. This study will yield important insights into the biology of schizophrenia and related mental illness, and lay the groundwork for larger-scale follow-up experiments.\",\n\"A major problem for schizophrenia research is that current diagnostic criteria are based on symptom and clinical course criteria, which, almost by definition, do not map to the underlying biology. As a consequence, genetic illness associations regularly cross diagnostic boundaries and are hallmarked by small effect sizes due to the polygenic disease architecture and heterogeneity of diagnostic constructs. Our group is developing machine learning approaches to identify accurate classifiers using data from individual, as well as multiple modalities of high-dimensional data. We are integrating external information sources on biological systems-knowledge (such as protein-protein interaction, co-expression information, as well as expression data from the BrainCloud or GTX datasets) to stratify the high-dimensional data space. We have already shown that using this strategy, developed classifiers outperform conventional machine learning strategies on genome-wide methylation data and are now in the process of adapting these approaches to genome-wide association data. For this, we will apply algorithms on dbGaP data (such as the Molecular Genetics of Schizophrenia datasets). We will furthermore integrate these datasets with other datasets (i.e. methylation or gene expression) to derive multi-modal classifiers or perform cross-modality prediction. Profiles will further be mapped to differences in brain structure or function in an in-house MRI dataset. These integrative analyses will not create any additional risks to participants. We will employ graph theoretic and machine learning methods to identify subgroups and their polygenic profiles. We will further test developed methods on other (related) mental illness, such as alcohol use disorders, as a basis for the identification of illness-specific and overlapping biological hallmarks. This research may yield important insights into the biology of schizophrenia related mental illness.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Shah, Sonia \",\n\"UNIVERSITY OF QUEENSLAND\",\n\"Using statistical genomics approaches to identify potential therapeutic compounds for treatment of Abdominal Aortic Aneurysm\",\n\"Jul06, 2022\",\n\"approved\",\n\"A genome-wide association studies (GWAS) aims to identify the genes that are relevant to the development of disease, which gives us better understanding of disease biology. Many cutting-edge statistical genomics methods have been developed that can be applied to data generated from GWAS for various purposes, including identifying drug targets for the disease as well as identifying existing drugs that could be repurposed for the disease in questions. This study aims to use such methods to identify potential drug targets and therapeutic compounds for the treatment of AAA. \",\n\"Objective: Using statistical genomics approaches to identify potential therapeutic compounds for treatment of Abdominal Aortic Aneurysm  Analysis Plan: Using methods such as MetaXScan that integrate expression quantitative trait loci (eQTL) data with AAA GWAS summary data that predict gene expression changes between cases and controls, we will first generate a gene expression signature for AAA using blood eQTL and GTEx aorta data. Next, we will use this gene expression signature query the connectivity map, a database of compound gene expression signatures, to identify compounds that are most likely to reverse AAA disease gene expression signatures, and may therefore show potential for treatment of AAA. We will also use Mendelian randomisation approaches to identify potential drug targets for AAA.  All analyses only require GWAS summary data, and no individual level data is requested. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Sherva, Richard \",\n\"BOSTON UNIVERSITY MEDICAL CAMPUS\",\n\"Genome-wide association study of substance dependence traits\",\n\"Jul13, 2020\",\n\"closed\",\n\"The risk for substance use disorders has a known genetic component. A genome-wide association study, in which millions of single base pair mutations throughout the human genome are tested for association with a trait, are one way to identify the specific genes responsible for this genetic risk. In this study, we will attempt to identify genes associated with the risk of alcohol, cannabis, opioids, and cocaine dependence.  \",\n\"Our research objectives are to identify susceptibility genes for substance use disorders, including alcohol, cannabis, opioids, and cocaine dependence through genetic association. The data requested will serve as a replication sample to validate associations observed in two samples: the Yale-Penn sample and SAGE (both already in dbGaP). Our primary analyses will use the DSM-IV criteria counts for the substances listed as ordinal outcome measures. We will use SNP minor allele dosage as a predictor variable. Yale-Penn and SAGE have been combined as a discovery sample and we will first test a small number of regions in the dataset we are requesting in an attempt to replicate these findings. We will also conduct a full GWAS in the requested data. Results will be combined with the discovery sample via meta-analysis. We will impute the requested data to the current 1000 Genomes reference panel.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Silveira, Patricia \",\n\"MCGILL UNIVERSITY\",\n\"Validation of a polygenic score of insulin receptor associated gene networks\",\n\"Feb10, 2022\",\n\"approved\",\n\"Complex traits such as mental health outcomes are polygenic.  This idea has led to the use of a genomic risk profile that captures the genetic burden as reflected by a set of \\u00e2\\u0080\\u0098risk\\u00e2\\u0080\\u0099 alleles for specific psychiatric disorders. In the current project we want to test developed a methodology for a computing total genetic risk based on gene networks of biological relevance.\",\n\"An individual\\u00e2\\u0080\\u0099s cumulative genetic risk, for a trait, can be summarized by the polygenic risk score (PRS); by multiplying the measured number of risk alleles at a locus by the effect size of the association between that genotype and outcome and summing over all SNPs at a certain significance threshold. Despite holding much promise the caveat of the PRS is that they do not provide information about the underlying network in which these genes operate nor the neurobiology of the disease. To overcome this we have developed a methodology for a \\u00e2\\u0080\\u0098refined\\u00e2\\u0080\\u0099 polygenic risk scores (PRS) of biological relevance. Insulin modulates synaptic plasticity, especially in dopaminergic neurons, influencing food intake, mood, impulsivity/inhibitory control, cognition and memeory. We extracted insulin receptor (IR) related gene network based on genes co-expressed with the IR in key brain regions such as the PFC or hippocampus. Based on the gene networks extracted we created two refined PRS-IRs (termed PRS-IRPFC & PRS-IRhippo). These were associated with the functional readouts of the specific brain regions- impulsivity for the PFC and memory related tasks for the hippocampus- in children. The Maternal Adversity, Vulnerability And Neurodevelopment cohort-a longitudinal study of Canadian mother-child dyads-was used for this analysis. In the current project we will validate these findings in an independent adult cohort.   The aim of this study is twofold: Validate the refined PRS-IRs in an independant dataset. Use an adult cohort to access whether PRS-IRs capture the genetic risk for relevant psychopathology(substance use for impulsivity associated PRS-IRPFC,Alzheimer\\u00e2\\u0080\\u0099s disease for memory associated PRS-IRhippo).  Datasets selected 1.GenADA-These which will be used to validate of the PRS-IRhippo. 2. Study of Addiction: Genetics and Environment (SAGE)-This will be used to validate of the PRS-IRPFC.  Study Plan: SNP genotypes from the GenADA and SAGE datasets will be used to compute the PRS-IRhippo and PRS-IRPFC respectively.The association between the PRS-IRs and appropriate affection status will be assessed. Confounders such a gender and study design will be controlled for.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"SMOLLER, JORDAN \",\n\"BROAD INSTITUTE, INC.\",\n\"Psychiatric Genomics Consortium Cross-Disorder Group v3 (PGC-CDG3): Probing multivariate genetic architecture across psychiatric traits\",\n\"Dec13, 2021\",\n\"rejected\",\n\"Psychiatric disorders overlap substantially at the genetic level. Cross-disorder genomic projects that aim to better characterize this genetic overlap then have the potential to produce key insights into the underpinnings of rampant psychiatric comorbidity. To this end, the current proposal aims to analyze the most up-to-date summary statistics across 14 major psychiatric disorders using four cutting-edge genomic methods: Genomic SEM, CC-GWAS, MiXER, and LAVA. These methods can be used to examine genetic overlap in aggregate across the genome, for specific classes of genes (e.g., genes that are expressed early in life), and at the level of individual genetic variants. By applying different methodoligcal approaches we aim to identify whether a convergent narrative emerges about the patterns of genetic overlap across disorders at varying levels of analysis. With respect to the number of disorders and utilized methods, this project aims to perform the comprehensive, cross-disorder psychiatric analysis to-date. We are specifically requesting permission to use the summary statistics that include the MVP cohort for: alcohol use disorder, opioid use disorder, anxiety disorder, major depressive disorder, and post-traumatic stress disorder.\",\n\"Building on the work of the PGC Cross-Disorder Working Group (PGC CDG2) in clarifying the genetic structure of psychopathology, we now aim to leverage multiple, multivariate methods to better define the shared and disorder-specific genomic underpinnings across 14 major psychiatric disorders. At the genome-wide level, we will specifically apply MiXER, Genomic SEM, and CC-GWAS. Using Genomic SEM, we will produce a factor model based on the genetic correlations across the 14 disorders with the aim of identifying subgroups of disorders that more strongly cluster together. MiXER then takes an alternative approach to quantify polygenic overlap by estimating the total number of shared and trait-specific causal variants, while CC-GWAS calculates the overall \\u00e2\\u0080\\u009cgenetic distance\\u00e2\\u0080\\u009d across two traits based on differences in allele frequencies for cases. Collectively, these results will help us determine whether a convergent narrative emerges about the overall pattern of genetic overlap across these disorders.  At the functional level, we will apply two new methods: Stratified Genomic SEM and LAVA. Stratified Genomic SEM builds upon the findings from Genomic SEM to examine multivariate enrichment at the level of the psychiatric factors. LAVA then examines local genetic correlations at the level of individual loci. In conjunction, these two methods allow us to identify specific categories of genes and loci that drive genetic overlap across disorders. Finally, at the SNP-level of analysis, we will use CC-GWAS and multivariate GWAS in Genomic SEM to identify SNPs that are both highly pleiotropic and disorder-specific. The cumulative set of findings will offer critical insights into the multivariate architecture of psychiatric disorders utilizing cutting-edge methods at three levels of analysis (genome-wide -> functional -> SNP). Indeed, the proposed project reflects the most comprehensive cross-disorder, psychiatric analysis to-date. We are specifically requesting permission to use the summary statistics that include the MVP cohort for: alcohol use disorder, opioid use disorder, anxiety disorder, major depressive disorder, and post-traumatic stress disorder.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"SO, Hon-Cheong \",\n\"CHINESE UNIVERSITY OF HONG KONG\",\n\"Dissecting the genetic basis of complex diseases and phenotype prediction\",\n\"Feb08, 2019\",\n\"approved\",\n\"The advance of genotyping technologies enable us to interrogate a large number of genetic variants at the same time. Individual genetic variants have limited prediction power, but by combining information from multiple variant, predictive accuracy may be improved. In this project we aim to test various statistical and machine learning strategies to improve risk prediction in diseases by using genomics data, and other omics or clinical data if available. We will also attempt to discover new variants for diseases and study the interaction of genetic variants. \",\n\"With the advance of genotyping technologies and large-scale genomic studies, an increasing number of susceptibility variants have been found for complex diseases. However, individual genetic variants have limited prediction power and a multi-locus approach is often used for risk or disease phenotype prediction. In this project we aim to test different supervised and unsupervised statistical and machine learning methodologies, such as regularized regression, decision trees and support vector machines, in individualized disease risk prediction. The prediction will be based on genomics profiles, and other \\\"omics\\\" (e.g. transcriptomics, methylomics) or clinical data, if such data is available. The performance of different predictive models will be assessed and compared. We also aim to employ statistical learning methods to uncover new susceptibility variants and genetic interactions, and to understand the genetic architecture of complex diseases.   In this project we aim to request data access to a range of complex diseases. For studies that only allow disease-specific research, the data will only be used to study the genetic basis for the specific disease/trait of interest and develop predictive models for that disease.    The data accessed will be used only for biomedical research purposes. We will not attempt to reveal the individual identity/ancestry of any person or of their relationship to another person. Each dataset will only be used in line with the consent of the participants. Findings from our study will be published and shared with the scientific community. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Sofer, Tamar \",\n\"BETH ISRAEL DEACONESS MEDICAL CENTER\",\n\"Polygenic scores for a range of phenotypes across population background groups. \",\n\"May05, 2023\",\n\"approved\",\n\"This research will use results from genetic association analysis performed in a diverse population from the Million Veterans Program, to generate \\u00e2\\u0080\\u009cinstructions\\u00e2\\u0080\\u009d to create summaries of genetic data that explain and predict various phenotypes, such as blood pressure, lipids, and kidney function. These gene-based prediction equations will be adapted to and evaluated across different population groups. In the long term, this work will help increase the availability of genetic risk prediction tools to underrepresented minorities.\",\n\"The goal of the proposed research is to develop polygenic risk scores (PRSs) based on different background groups and combined (African Americans, Hispanic Americans, etc) that are associated with a range of phenotypes, such as blood pressure, kidney function, and lipids, and study their usefulness for explaining these phenotypes and for prediction such phenotypes across the different background groups.   For this purpose, we will use summary statistics from the GWAS of performed in MVP, including GWAS of blood pressure, and other phenotypes, to select genetic variants and estimates weights for the PRSs. To select the genetic variants, we will use methods that take into account the summary statistics from testing each genetic variant, and also how the various genetic variants correlate with one another (linkage disequilibrium). Linkage disequilibrium will be assessed from different, independent datasets, that will serve as \\u00e2\\u0080\\u009creference panels\\u00e2\\u0080\\u009d.  We will evaluate the PRSs association with these traits in independent datasets.  This research will help determine how to construct PRSs that one day may be used in clinic for risk prediction, while benefitting populations that are underrepresented in current genetic medical research. This research purpose is consistent with the data use agreement, because this is a biomedical research that will not be used to study the population origin or ancestry.   The summary statistics (the data that we will download) will be used only in the PI\\u00e2\\u0080\\u0099s institution. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Sofer, Tamar \",\n\"BRIGHAM AND WOMEN'S HOSPITAL\",\n\"Polygenic scores for a range of phenotypes across population background groups. \",\n\"May22, 2019\",\n\"approved\",\n\"This research will use results from genetic association analysis performed in a diverse population from the Million Veterans Program, to generate \\u00e2\\u0080\\u009cinstructions\\u00e2\\u0080\\u009d to create summaries of genetic data that explain and predict various phenotypes, such as blood pressure, lipids, and kidney function. These gene-based prediction equations will be adapted to and evaluated across different population groups. In the long term, this work will help increase the availability of genetic risk prediction tools to underrepresented minorities.\",\n\"The goal of the proposed research is to develop polygenic risk scores (PRSs) based on different background groups and combined (African Americans, Hispanic Americans, etc) that are associated with a range of phenotypes, such as blood pressure, kidney function, and lipids, and study their usefulness for explaining these phenotypes and for prediction such phenotypes across the different background groups.   For this purpose, we will use summary statistics from the GWAS of performed in MVP, including GWAS of blood pressure, and other phenotypes, to select genetic variants and estimates weights for the PRSs. To select the genetic variants, we will use methods that take into account the summary statistics from testing each genetic variant, and also how the various genetic variants correlate with one another (linkage disequilibrium). Linkage disequilibrium will be assessed from different, independent datasets, that will serve as \\u00e2\\u0080\\u009creference panels\\u00e2\\u0080\\u009d.  We will evaluate the PRSs association with these traits in independent datasets.  This research will help determine how to construct PRSs that one day may be used in clinic for risk prediction, while benefitting populations that are underrepresented in current genetic medical research. This research purpose is consistent with the data use agreement, because this is a biomedical research that will not be used to study the population origin or ancestry.   The summary statistics (the data that we will download) will be used only in the PI\\u00e2\\u0080\\u0099s institution. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Sorensen, Peter \",\n\"UNIVERSITY OF AARHUS\",\n\"Using correlated trait information for improved gene mapping and prediction of complex traits and diseases\",\n\"Nov23, 2022\",\n\"approved\",\n\"Current approaches for estimating an individual's genetic disease predisposition only explain a small proportion of the disease risk. We will increase mapping power and prediction accuracy for complex diseases such as Type 2 Diabetes (T2D) and Coronary Artery Disease (CAD) by developing statistical and bioinformatic approaches that take advantage of the rapid accumulation of functional genomic information in biological databases, and availability of large independently collected genotype and phenotype datasets for multiple complex diseases. Our project will provide genetic markers and functional marker groups associated with T2D and CAD disease risk that points towards biological important processes underlying the disease. This information can be used to identify potential drug targets in the early stages of the drug discovery process and can help priorities which of these drug targets should be further investigated. Furthermore utilizing genomic information in precision medicine for stratifying individuals based on their inherent genetic disease predisposition has great potential to identify high-risk patients enabling early-lifestyle and/or pharmacological interventions.\",\n\"The aim is to increase power of detecting genetic variants underlying disease risk by developing bioinformatics and statistical modeling approaches that 1) combines information on multiple correlated traits across ancestries, and 2) use information on functional marker groups to capture weak genetic signals within and across traits. We will apply novel statistical methodology to well-established and high-quality human genetic databases for T2D and CAD and we therefore believe that the project could generate significant new knowledge and potentially uncover novel therapeutic mechanisms and/or drug targets and more accurate genomic predictions for cardiometabolic diseases.  The discovery of novel genetic variants associated with complex diseases can help identify biomarkers and potential drug targets for the disease including 1) genes and regulatory elements located nearby disease-associated variants or biological pathway, and 2) biomarkers (transcriptome, metabolome and proteome) whose association signals are linked to disease associated signals.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Speed, Doug \",\n\"UNIVERSITY OF AARHUS\",\n\"Using MVP summary statistics to construct polygenic risk scores and perform gene-based association testing\",\n\"Nov02, 2021\",\n\"approved\",\n\"Given an individual's DNA, it should be possible to acccurately predict their risk of developing different diseases, and decide the best course of medication should they become ill. However, for this to be possible, we must better understand the genetic factors that influence disease risk. The Million Veterans Project has performed genetic studies for a number of traits (e.g., cholesteral and coronary artery diseases). We will use the results from these studies to improve our understanding of the genetic basis of complex diseases.\",\n\"We would like access to the summary statistics produced by recent GWAS by the Million Veterans Project (MVP). Specifically, we are interested in the summary statistics for cholesterol (LDL and HDL), type 2 diabetes and when available coronary artery disease. We have two main uses for these summary statistics.   1. Construct polygenic risk scores (PRS) for these traits. Based on data from UK Biobank, we believe our software LDAK (www.ldak.org) is able to produce the most accurate PRS. We would like to test our software on MVP summary statistics. We will use these summary statistics to construct PRS, then test the accuracy of these on UK Biobank data (i.e., measure how well the PRS constructed from MVP can be used to predict cholesterol levels, type 2 diabetes and coronary artery disease in UK Biobank data).  2 - Perform gene-based association testing. It is normal to test SNPs (genetic variants) individually for association with the trait. However, LDAK includes a tool for testing groups of SNPs jointly (e.g., SNPs corresponding to a gene). We would like to test the performance of this tool on MVP summary statisitics. In particular, see whether it is possible to detect causal loci using our gene-based test that are not detected from single-SNP analysis.  We are only requesting summary statistics (not individual-level data) so there are minimal risks (e.g., it is almost impossible to identify individuals, and we will not try to). In order to run our two tools, we require summary statistics and a reference panel (the latter is used to estimate SNP-SNP correlations). For this we will use genetic data from the 1000 Genome Project or UK Biobank. Once we have constructed PRS, we will test their performance using data from UK Biobank (i.e, the PRS allow us to predict traits for UK Biobank individuals, then we can compare these predicted trait values to their actual values).\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Speliotes, Elizabeth \",\n\"UNIVERSITY OF MICHIGAN\",\n\"Analysis of Nonalcoholic Fatty Liver Disease(NAFLD) and Cardiometabolic Traits in dbGAP Datasets\",\n\"Dec19, 2022\",\n\"approved\",\n\"With increasing obesity, Nonalcoholic Fatty Liver Disease (NAFLD) has reached epidemic proportions. Genetic variants have been found to associate with both NAFLD and cardiometabolic traits/diseases. We will use dbGAP datasets with measures of NAFLD to 1. identify new common genetic variants that associate with NAFLD 2. fine map and characterize NAFLD associated variants by performing larger more powered GWAS analyses in diverse ancestries with measures of NAFLD 3. test variants that have been found to reproducibly associate with cardiometabolic traits/diseases for effects on NAFLD and variants that associate with NAFLD for effects on cardiometabolic traits/diseases and 4. determine whether the effects of NAFLD-associated SNPs are altered by environmental variables. This work will define disease risk categories and environmental triggers of disease development, gene targets for therapeutic intervention, and equalize the benefits of genetic discoveries to diverse ancestries. \",\n\"Nonalcoholic Fatty Liver Disease (NAFLD) is a spectrum of disease that includes steatosis, steatohepatitis, and fibrosis/cirrhosis in the liver. Genetic variants have been found to associate with both NAFLD and cardiometabolic traits/diseases. These will be cross sectional analyses that will use dbGAP datasets where we have CT-measured hepatic steatosis (Amish, AGES, Family Heart Study, Framingham Heart Study, IRAS, Cardia, GENOA, MESA, Dutch Nelson, Jackson Heart Study, and COPD Gene) and genome wide data available (on over 22,000 individuals) to identify new common variants that associate with CT measured NAFLD using regression analyses controlling for confounders (age, gender, alcoholic drinks, principal components) in the individual studies. We will carry out fixed effect meta analysis across groups and replicate promising associations in histology diagnosed NAFLD in cases from the FLIP consortium and the Geisinger bariatric sample matched to publically available controls. We will also see whether these variants reproduce for effects on NAFLD related diagnoses in UKBiobank and Michigan Genomics Initiative (MGI) datasets that we have now also obtained. We will also meta analyze our data with association data from UKBiobank and MGI with dbGAP datasets to increase our power to identify reproducible associations.  Variants that reach genome wide significance or that replicate will be considered real associations. We will use cross ancestry meta analysis and  conditional analyses to help identify causal variants at associated loci. We will test variants that have been found to reproducibly associate with cardiometabolic traits/diseases for effects on NAFLD  and also test variants that have been found to reproducibly associate with NAFLD for effects on cardiometabolic traits/diseases in dbGAP datasets, histology datasets, and the UKbiobank and MGI datasets. We will determine whether the effects of NAFLD-associated SNPs are altered by environmental variables including anthropometric, serum laboratory, and dietary variables. We will expand our analyses of BMI and NAFLD to include analyses of the genetic and epidemiological contributions to all continuous and dichotomous human traits and diseases using regression and data mining methods.  We will examine the effects of associated variants on affecting gene expression in STARNET and other eQTL datasets as they become available. This work will define disease risk categories and gene targets for therapeutic intervention.  Please see 2014 approval for method details.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Spracklen, Cassandra \",\n\"UNIVERSITY OF MASSACHUSETTS AMHERST\",\n\"Overlapping MVP T2D Summary Statistics with caQTL and eQTL results in Pancreatic Islets\",\n\"Nov16, 2021\",\n\"approved\",\n\"I would like to use the summary results files so that I can see the full set of results for all of the DNA variants tested for association with T2D (not just the most significant variants reported in the main publication). I will also use the summary data to generate graphs/plots of the results.\",\n\"I will use the T2D summary statistics from the trans-ancestry and single-ancestry analyses to look-up single-variant results from my current work overlapping T2D GWAS hits with caQTL and eQTL results from human pancreatic islets. I also intend to use the summary statistic files to generate Locus Plots for visualization of the GWAS signal.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Stallings, Michael \",\n\"UNIVERSITY OF COLORADO\",\n\"Comparing shared genetic risk of substance use disorder across associated phenotypes\",\n\"Oct04, 2022\",\n\"approved\",\n\"Substance use disorders are common and can occur in many different types of people with different diagnostic and personality profiles. We are interested in investigating how these different profiles overlap and/or diverge with respect to genetically-informed risk for addiction to substances.\",\n\"Substance use disorders (SUDs) are both phenotypically and genetically correlated with several pathological and behavioral phenotypes. However, it is not clear to what extent genetic liability for SUDs is common across individuals who share a high predisposition for SUDs but who differ with respect to other risk factors. The objective of the proposed study is to use Genomic Structural Equation Modeling (Genomic SEM) to demonstrate potential genetic heterogeneity in liability to SUDs. To create our models, we intend to use summary SUD and mental health data from other consortia alongside summary SUD and mental health data from MVP. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Su, Jinni \",\n\"ARIZONA STATE UNIVERSITY-TEMPE CAMPUS\",\n\"Gene-Environment Interactions and Alcohol Use and Related Outcomes \",\n\"Jul19, 2022\",\n\"approved\",\n\"Alcohol use disorders are prevalent and associated with significant personal, interpersonal, and societal costs. Understanding the risk and protective factors associated with alcohol use and related outcomes is important to inform prevention and intervention efforts for those who are at risk. Some people are genetically predisposed to have higher risk of developing alcohol use disorders. This research aims to identify malleable environmental factors across multiple levels (e.g., family, peer, school, neighborhood, culture) that may exacerbate or buffer genetic risk. \",\n\"Alcohol use disorders (AUD) are prevalent and associated with significant personal, interpersonal, and societal costs. Understanding how genetic and environmental influences interact (i.e., gene-environment interaction) to contribute to the development of AUD among diverse populations is critical for developing prevention and interventions. The overarching goal of this research is to understand how genetic risk of AUD interact with environmental factors to influence alcohol use and related outcomes (e.g., depression, anxiety, externalizing behaviors, etc.) in adolescents of racially/ethnically diverse backgrounds. We will conduct secondary data analysis of the Adolescent Brain Cognitive Development Study, which enrolled a large racially/ethnically diverse sample of children (N = 11,875; 52.1% White, 15.0% Black, 20.3% Hispanic, 2.1% Asian, and 10.5% other [e.g., biracial]; 47.8% female; aged 9-10 at baseline assessment) and follows them up over time. We will conduct multiple regression analysis and structural equation models to examine how genetic risk of AUD(characterized by genome-wide polygenic scores using summary statistics from genome-wide association study of AUD) interacts with environmental factors across multiple levels (e.g., parenting, peer deviance, school belonging, neighborhood disadvantage, cultural values) in predicting alcohol use and related behavioral and emotional health outcomes in adolescents. We are requesting access to the summary statistics from genome-wide association study of alcohol phenotypes with the Million Veteran Program study, which will be used to calculate polygenic scores for our project.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"SUGIURA, AYUMU \",\n\"TAKEDA PHARMACEUTICAL COMPANY LTD\",\n\"Integrative analysis of PTSD and multiple neurological disorders\",\n\"Oct23, 2019\",\n\"approved\",\n\"Understanding genetic background of patients with Posttraumatic stress disorder (PTSD) is a critical step to elucidate the molecular mechanism of the disease. We would like to use dbGaP data to identify genetic markers, sub-populations of PTSD, and associations with other neurological disorders. It could help us to understand genetic background contributing crucial biological processes and potential roles in disease phenotypes. In the future, the obtained knowledge will be used to develop more effective and safer treatment for PTSD and other neurological disorders.\",\n\"Understanding genetic background of patients with Posttraumatic stress disorder (PTSD) is a critical step to elucidate the molecular mechanism of the disease. We will perform GWAS of PTSD to identify genetic markers, and conduct post-GWAS analysis using some tools and public resources such as eQTL and pQTL databases. Moreover, we will apply several analytical methods such as co-localization and Mendelian randomization to PTSD GWAS for identifying genetic markers especially associate with other neurological disorders. Potentially, a genetic marker associates with sub-populations could be identified. These genetic markers should be very helpful to clarify crucial biological processes of the sub-populations and detect sub-populations having specific genetic background. These analyses might be able to reveal the biological basis of PTSD and relationships with other neurological disorders. Also, we could superimpose in-house data on these results, and it potentially helps us to investigate responder/non-responder or safety markers associate with the sub-populations of PTSD. It could also help us to identify potential drug targets for PTSD patients especially belonging to the sub-populations. Furthermore, an integration of these results with other knowledge and genomics data such as epigenome and gene expression might help us to obtain better understanding of PTSD at the molecular level. We will strictly conduct these studies according to Data Use Restriction. No violation of Use Restriction is expected.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Sumner, Jennifer \",\n\"UNIVERSITY OF CALIFORNIA LOS ANGELES\",\n\"Joint Contributions of PTSD and Polygenic Risk for High Blood Pressure to Blood Pressure and Hypertension\",\n\"Apr16, 2020\",\n\"closed\",\n\"PTSD has been associated with elevated blood pressure, but the role of genetic factors is unclear. This study will examine whether the link between PTSD and elevated blood pressure is greater for individuals with greater genetic vulnerability to high blood pressure.\",\n\"Posttraumatic stress disorder (PTSD) is associated with increased risk of elevated blood pressure (BP) and hypertension (e.g., Buckley & Kaloupek, 2001; Burg et al., 2017; Sumner et al., 2016). However, the role of genetic factors is unclear. Polygenic risk scores for BP have been studied in both European ancestry (EA) and trans-ancestry genome-wide association studies (GWASs) (Smith et al., 2015). These polygenic risk scores have been found to explain a small but significant percent of the variance in systolic BP, diastolic BP, and hypertension, above and beyond traditional risk factors (Fava et al., 2013; Franceschini et al., 2013; Giri et al., 2018; Warren et al., 2017). The aims of this PGC-PTSD secondary analysis proposal are to examine the interaction of PTSD with polygenic BP risk to BP and hypertension outcomes. We hypothesize that associations of PTSD with elevated BP and hypertension will be stronger for those with greater polygenic BP risk.  Studies from the PGC-PTSD working group with relevant data will contribute to this project. We are requesting access to the MVP trans-ethnic BP GWAS summary statistics so that we can compute polygenic risk scores for systolic and diastolic BP. Some of the contributing studies include individuals of European ancestry, but others are more diverse (e.g., individuals of African or Hispanic ancestry). Having access to the summary statistics from the MVP trans-ethnic BP GWAS will be useful for studying these diverse ancestry samples.   In each study with relevant data, we will calculate polygenic risk scores for BP. Then we will use hierarchical linear regression to examine the main effects and interaction of PTSD and BP polygenic risk score on BP (separate regression models will be conducted for systolic and diastolic BP). Population substructure (PCs), age, sex, PTSD, and BP polygenic risk score will be entered in the first step of the regression model. The PTSD x BP polygenic risk score interaction will then be entered in the second step of the regression model. We will also consider including PC x BP polygenic risk score and PC x PTSD interactions to properly control for potential confounders. We will then meta-analyze results across the different studies. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Sundstr\\u00c3\\u00b6m, Johan \",\n\"UPPSALA UNIVERSITY\",\n\"Investigating the causal relationship between lung function and atherosclerosis\",\n\"Apr16, 2020\",\n\"approved\",\n\"Individuals with impaired lung function are at increased risk of CVD. The relationship between lung function and CVD persists even after adjustments for shared risk factors such as smoking and inflammation, and has also been observed in studies restricted to never-smokers. However, whether impaired lung function leads to CVD or vice versa is currently not known. We will investigate this causal relationship, which might reveal the underlying pathophysiology required for the development of therapeutic interventions, using summary statistics from published large scale genome-wide association studies.\",\n\"Individuals with impaired lung function are at increased risk of CVD. The relationship between lung function and CVD persists even after adjustments for shared risk factors such as smoking and inflammation, and has also been observed in studies restricted to never-smokers. However, whether impaired lung function leads to CVD or vice versa is currently not known. Investigating the causal relationship might reveal the underlying pathophysiology that could be useful for the development of therapeutic interventions. To investigate the causal relationship between lung function (forced vital capacity and forced expired volume in 1 second) and atherosclerosis (carotid artery internal media thickness, presence of carotid plaque) and peripheral artery disease, we will perform a two-sample Mendelian randomization analysis. We will use summary statistics from published large scale genome-wide association studies to study this relationship bidirectionally, i.e. we will use genetic variants associated with lung function to assess the effect of lung function on atherosclerosis or peripheral artery disease and genetic variants associated with atherosclerosis to assess the effect of atherosclerosis or peripheral artery disease on lung function. Adjustment for smoking behavior will be performed using multivariable Mendelian randomization and, depending on access to external data, using subgroup analysis in never-smokers. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"TALL, ALAN \",\n\"COLUMBIA UNIVERSITY HEALTH SCIENCES\",\n\"Causal Analysis of Lipid Traits and Other Metabolic Syndrome Traits in Coronary Artery Disease\",\n\"Apr16, 2020\",\n\"closed\",\n\"Coronary artery disease is the leading cause of death in the United States, accounting for 28% of overall mortality. Low-density lipoprotein cholesterol, high-density lipoprotein cholesterol, and triglycerides have been identified as risk factors for coronary artery disease. Therapies targeting high-density lipoprotein cholesterol and triglycerides have so far failed to improve coronary artery disease outcomes. It is unclear whether individuals with low high-density lipoprotein cholesterol and high triglycerides cause coronary artery disease, or whether correlated metabolic traits such as obesity, blood pressure, and type 2 diabetes mediate these effects. We will use genetic variants that modify high-density lipoprotein cholesterol and triglycerides identified in the Million Veterans Program study to assess whether these traits cause coronary artery disease. This study will determine whether high-density lipoprotein cholesterol or triglycerides represent suitable targets for the next generation of preventative therapies in coronary artery disease. \",\n\"Coronary artery disease is a significant cause of morbidity and mortality in the United States, particularly in individuals with high low-density lipoprotein cholesterol (LDL-C), low high-density lipoprotein cholesterol (HDL-C), and high triglycerides (TG). The objective of this research is to determine the causal effects of variation in HDL-C and TG on coronary artery disease risk using Mendelian randomization. Previous studies have used genetic variants in LDL-C, HDL-C, and TG to model causal effects of these traits and reported strong evidence for a causal role of LDL-C, moderate evidence for a causal role of TG, and no evidence for a causal role of HDL-C in coronary artery disease. However, we have found that more than 20% of genetic variants affecting HDL-C and TG also modify other metabolic traits that influence coronary artery disease risk, such as body mass index (BMI), systolic blood pressure (SBP), and type 2 diabetes risk (T2D). We have acquired external datasets for BMI, SBP, and T2D genetic associations.   We will develop new multivariable Mendelian randomization methods that account for these pleiotropic effects by regressing coronary artery disease risk on LDL-C, HDL-C, TG, BMI, SBP, and T2D in a multivariate model using MR-inverse variance weighted and MR-Egger methods. We will also perform sensitivity analyses in which variants with pleiotropic effects on specific traits are excluded. Analyses will be performed in R 3.6.3 and will not require the use of cloud computing. We will use the Million Veterans Project genomic association summary results for blood lipids (LDL-C, HDL-C, and TG) to identify a sufficiently large set of lipid trait variants to model coronary artery disease risk effects of each trait after adjustment for pleiotropy. Use of this specific disease dataset will clarify the causal role of HDL-C and TG in coronary artery disease and establish one or more of these traits as targets for therapeutic development in coronary artery disease. The use of this data for methods development is consistent with the allowed use specified in the Data Use Restrictions. We will not perform research on population origins or ancestry. In accordance with the specifications of the Data Use Certification Agreement, we are not requesting any individual-level data. Results will be widely shared in published manuscripts. We will not collaborate with any external collaborators on this project.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Tang, Hua \",\n\"STANFORD UNIVERSITY\",\n\"Learning Genetic Architecture of Complex Traits Across Populations\",\n\"Apr29, 2021\",\n\"approved\",\n\"Genome-wide association studies (GWAS) have become a primary tool for understanding the genetic basis of complex traits and diseases.  The goal of this research is to advance our understanding of cardiovascular diseases in minority populations, so that GWAS can benefit all individuals. Specifically, by leveraging information across diverse populations, our analyses aim to identify minority-relevant genetic variants and to build polygenic risk scores (PRS) for minority individuals.  \",\n\"Objective. Understanding the genetic factors influencing complex traits in minority populations is challenging because the sample sizes areoften much smaller in Genome-wide association Studies (GWAS) of minority populations, compared to GWAS in European populations. As described below, we have previously developed minority-centric association mapping and genetic risk prediction approaches. The key insight underlying these approaches is to leveragesmulti-ethnic and trans-ethnic information, while acknowledging heterogeneity across populations. The objective of this dbGaP application is to apply and extend these methods to identify identifying minority-relevant genetic factors underlying plasma lipids, stroke and VTE in African Americans and Hispanics. We will also build population-specific polygenic risk score (PRS) for lipids and CAD (stroke and VTE).    Study Design. For association mapping, we will apply an empirical Bayes approach developed by Dr. Tang\\u00e2\\u0080\\u0099s group, XPEB (Coram et al.2015; \\u00c2\\u00a0PMID: 25892113). This method considers a study design, in which summary-level statistics from two GWAS are available: one in the minority population of primary interests (referred to as the focal population), and the other in a reference population of much larger sample size(referred to as the base population). As an example, to identify genetic loci associated with low-density lipid cholesterol (LDL) in African Americans, we will generate the focal GWAS summary-statistics by meta-analyzing an African American LDL study performed in the Women\\u00e2\\u0080\\u0099s Health Initiative (WHI) with the African American LDL analysis in the Million Veterans Program. (All investigators on this, and the collaborating, applications have access to WHI).For base GWAS, we will use MVP\\u00e2\\u0080\\u0099s European-stratified LDL summary statistics. For PRS, we will apply a mixed-effects model approach developed by Dr. Tang\\u00e2\\u0080\\u0099s group, XP-BLUP (Coram et al. 2017; PMID:\\u00c2\\u00a028757202).This method considers a study design, in which individual-level genotype-phenotype data is available in a minority cohort (the focal GWAS), while only summary-level GWAS statistics are available in the base GWAS. In the proposed study, WHI individual-level African Americans and Hispanic American GWAS data are available to all investigators (and through dbGaP).    Analysis plan. For association mapping, we will apply XPEB(implemented in an open source R package) \\u00c2\\u00a0for each of the lipids and CAD traits, andusing African Americans or Hispanics as the focal population. \\u00c2\\u00a0For PRS, XP-BLUP program, implementing the mixed-effects model, is available on github. Specifically, we will include age and genetic-PCas covariate for all analysis. In addition, trait-specific covariates will be adjusted as needed. \\u00c2\\u00a0Validation/Replication: For minority-relevant loci identified using XPEB, we will seek replication in the appropriate minority-populations. Because we anticipate some loci identified by XPEB feature moderate/weak effect sizes, other minority-cohorts are under powered.As an alternative, we will examine consistency in allelic effects and assess the improvement in disease risk prediction. For minority-specific PRS, we evaluate the predictive performance on an independent cohort of matching minority population (African Americans or Hispanics). We will compare PRS built byXP-BLUP with PRS directly transferred from European cohorts.    The DUL states \\u00e2\\u0080\\u009cuse of this data is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry. Use of the data includes methods development research(e.g., development of software or algorithms).\\u00e2\\u0080\\u009d Our analysis will use the population-specific GWAS results produced by the MVP; no analysis is proposed to study the population origin or ancestry. \\u00c2\\u00a0Therefore, this application is consistent with the DUL.   Collaboration with researchers at other institutions, if appropriate.The proposed research will be performed in collaboration with Drs. Charles Kooperberg and Alex Reiner\\u00e2\\u0080\\u0099s groups at the Fred Hutchinson Cancer Research Center (FHCRC). \\u00c2\\u00a0Drs.Kooperberg/Reiner are submitting a dbGaP application through FHCRC.\\u00c2\\u00a0\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"TAYLOR, KENT \",\n\"LA BIOMED RES INST/ HARBOR UCLA MED CTR\",\n\"NHLBI TOPMed Whole-genome Sequencing Program (TOPMed WGS)\",\n\"Jun07, 2021\",\n\"approved\",\n\"The goals of the TOPMed Whole Genome Sequencing program are to identify genetic risk (and protective) factors for disease, to identify subtypes of disease, and to develop targeted and personalized treatments for disease.  The program consists of multiple studies addressing these goals for phenotypes focused on heart, lung, blood and sleep disorders, which have a high impact on public health.  These studies will include individuals from diverse populations, including those historically under-represented in health research.\",\n\"TOPMed Generic Research Use Statement  NOTE TO REVIEWER OF THIS APPLICATION: I AM COLLABORATING WITH MULTIPLE WORKING GROUPS IN MESA, TOPMED, AND CHARGE. I CERTIFY THAT AT NO TIME WILL I SHARE DATA DOWNLOADED FROM DBGAP WITH OTHER INVESTIGATORS NOT LISTED IN THIS APPLICATION. IN THESE GROUPS WE PERFORM ALL OF OUR ANALYSES BY SHARING SUMMARY DATA ONLY AND PERFORMING META-ANALYSES.  Revised 12/21/17  Phenotypic and genomic data from TOPMed studies will be used for: (a) investigating genotype-phenotype associations; (b) variant summary statistics, (c) common controls in association tests, (d) reference panels to impute into un-sequenced individuals from TOPMed; and (e) population genetic studies.  TOPMed data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Data from an individual with disease-specific consent will not be used in analyses outside of that restriction, including indirect uses such as imputation reference panels, common controls for association studies and variant summary statistics. For example, an individual with consent restricted to lung function studies will not be used to impute data intended for analyses of diseases not clearly related to lung function.  When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish or otherwise broadly share any findings from this study with the scientific community.  This application anticipates collaboration among TOPMed investigators from multiple institutions, who are named in a master list appended to the IRB document uploaded with this application. Within a set of applicants approved for access to the TOPMed Exchange Areas, data may be shared in a cloud environment or by direct transfer, provided that they all agree among themselves to allow such sharing, and that all have approved TOPMed Data Access Requests for the specific data sets to be shared. Otherwise, data will be shared only among Internal Collaborators \\u00e2\\u0080\\u0093 i.e. within an institution.  Data from TOPMed may be combined with non-TOPMed data from the same or other studies (obtained from dbGaP or other sources), to improve the power for novel genetic discoveries, while respecting the consent of all participants. We expect that this activity creates no additional risks to participants. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"TAYLOR, KENT \",\n\"LA BIOMED RES INST/ HARBOR UCLA MED CTR\",\n\"TOPMED Data Analyses\",\n\"Aug24, 2021\",\n\"approved\",\n\"Data will be used to develop a genetic test to predict cardiovascular disease in woman and African Americans.\",\n\"Background We have recently demonstrated that genomics risk scores for cardiovascular disease, based on the results of GWAS, do not predict cardiovascular phenotypes in women from African and European descent (Hajek et al, 2018. Circ Genom Precis Med. 2018 Oct;11(10):e002324.  doi: 10.1161/CIRCGEN.118.002324.  Objectives for Project * Development of regularized matrix decomposition methods for analyses of multiple phenotype and omic datasets * Identification of genomic variants for prediction of cardiovascular outcomes in the MESA cohort with focus on women from African and European descent * Perform confirmation studies of matrix results using women from the Framingham Heart Study, the WHI, and the Jackson Heart Study  Study Design Case control omic analyses of epidemiological studies  Analysis Plan In all analyses, covariates are age and BMI. Gender and race are not included because the we will be analyze women by African or European descent separately. 1. regularized matrix decomposition will be applied to the MESA cohort (R package glmnet). This analysis perform a 10-fold cross validation of prediction error in order the determine the penalty to be used in the final analysis to determine genetic risk score. 2. The resulting genetic risk score will be applied to the FHS and JHS cohorts validation using logistic regression.  I recognize that the development of a genetic risk score will be of value in the clinical setting. However, I have every intent to publish the results of the study described here and make results available to my colleagues and to the public. The datasets additional to TOPMed are for confirmation studies in two-stage analytic methods. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Tcheandjieu, Catherine \",\n\"J. DAVID GLADSTONE INSTITUTES\",\n\"genetic determinant for ASCVD\",\n\"Jan18, 2023\",\n\"approved\",\n\"my proposal require access to genetic results published by the million Veteran to create methods for disease risk prediction that is applicable and beneficial for people from all ancestral background. \",\n\"Despite decades of research, Atherosclerosis cardiovascular diseases (ASCVDs) remain the leading cause of death worldwide. The lack of progress is at least partially related to the disproportionally higher incidence of ASCVD in minority groups. A higher prevalence of non-genetic ASCVD risk factors in some minority groups is a major contributor to this disparity and requires substantial attention. However, an advanced understanding of the genetic susceptibility to ASCVD in minority groups will also help to achieve progress. Unfortunately, while genetic determinants of ASCVD have been identified among individuals of European ancestry including the development of polygenic risk scores (PRS), very little is known about the genetic risk of ASCVD for other populations. This is consequential, because current PRSs, as shown in my previous work as well as others, do not have transferability across ancestry groups. Thus, although PRSs show promise as a precision medicine tool to advance diagnosis, prevention, and treatment of ASCVD, they also have a strong potential in their current state to exacerbate preexisting ASCVD disparities. Based on my previous study, I hypothesize that studying populations with diverse genetic backgrounds will not only reduce inequity in the applicability of genomic precision medicine but will also substantially improve our understanding of the underlying ASCVD biology. My goal is to leverage genetic diversity across global populations to (1) better understand the biology of ASCVD and (2) develop novel approaches for ASCVD precision medicine. I would like to access the summary statistic of various phenotype from the multi-ancestry cohort of the Million veteran project to create Polygenic risk score for ASCD that also include their risk factors. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Thom, Christopher \",\n\"CHILDREN'S HOSP OF PHILADELPHIA\",\n\"Genetic interactions between blood traits and complex disease. \",\n\"Feb23, 2023\",\n\"approved\",\n\"Blood cell biology is incredibly important yet not well understood. Complex physiology impacts blood cell formation and function in vivo, but is rarely considered in the context of blood trait genetics. The goals of this project are to genetically define how blood traits and complex disease traits are related, including a directional estimation to determine if disease traits impact blood traits, or vice versa. After defining trait associations, we will use statistical genetics approaches to define loci and genes that can account for the interrelatedness of traits in order to understand underlying genetic mechanisms. Further, we will use established data sets to test gene-specific hypotheses based on our model system work. In sum, this project will better define blood cell-intrinsic and systemic mechanisms that contribute to genetic inheritance of blood traits and related complex disease. Our findings will be relevant for understanding blood cell biology, complex disease physiology, and related therapeutic opportunities.  \",\n\"Mechanisms governing blood cell formation and function are critical yet incompletely understood. In vivo, blood cell formation (hematopoiesis) is influenced by myriad systemic and environmental factors. Blood cell biology also influences complex disease development, severity and physiology. Genome wide association studies (GWAS), patient studies, and model systems interrogating have defined thousands of genomic loci that impact blood traits. However, molecular mechanisms and trait interactions are not well defined. The goal of this project is to use statistical genetics approaches to discover novel trait associations and integrated multi-omics methods to define processes that influence, or are influenced by, blood cell physiology.   Prior work based on publicly available GWAS defined novel functional loci and related target genes, which were subsequently validated using murine and stem cell model systems. We have used statistical genetics approaches, including colocalization and Mendelian randomization (MR) methods, to define novel links between blood, cardiovascular, metabolic, autoimmune, psychiatric, oncologic, and behavioral (e.g., tobacco and alcohol use) traits. The project defined herein will expand on these prior studies and related hypotheses.    Using the MR framework, we will comprehensively define associations between quantitative blood traits (e.g., blood cell counts and percentages), blood cell function (platelet reactivity), and disease outcomes. These disease outcomes will include cardiovascular (coronary artery disease, heart failure, stroke, pulmonary embolism, peripheral artery disease, atrial fibrillation), metabolic (type 2 diabetes, lipid and cholesterol levels), immune/autoimmune (asthma, eczema, rheumatoid arthritis, lupus), psychiatric (schizophrenia, bipolar disorder, neurodegenerative), and oncologic disease predisposition, as well as anthropometric (height, body mass index, waist-to-hip ratio) and behavioral/lifestyle (tobacco use, alcohol use, circadian rhythms, socioeconomic status, education level) traits. After establishing causal inferences between traits, we will define relevant organ systems, cell types, genes and genomic loci using genetic colocalization and statistical fine-mapping approaches.    Prior GWAS and our studies have also prioritized genes that we anticipate directly influence blood cell formation and/or function, including but not limited to Tropomyosin 1 (TPM1). We will specifically query for coding mutations in this and other high priority target genes among whole exome/whole genome sequencing cohorts to identify naturally occuring mutations and related phenotypes that can help explain gene functions to hematopoiesis and other systems.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Thorgeirsson, Thorgeir \",\n\"DECODE GENETICS, EHF\",\n\"Genetic Risk Factors For Addiction\",\n\"May17, 2019\",\n\"rejected\",\n\"We aim to understand the genetic factors conferring risk of nicotine dependence, alcoholism, and addiction in general. We plan to accomplish this by combining data sets available from dbGAP with our own data sets. We will then search for variants associated with various kinds of addiction in the combined data sets. Both common and rare variants, and both single base changes in the genetic code and structural polymorphisms such as duplications and deletions will be examined.\",\n\"Our project aims to use genetic information to aid in understanding the biological processes involved in addiction. To do this, we plan to identify genetic risk factors for addiction by using the large data sets resulting from the combination of information from the requested data sets and from Iceland. Phenotypes to be studied include nicotine (tobacco) addiction, alcohol and substance use disorders. A combined addiction phenotype will also be examined, using data sets permitted to be used for this more general phenotype. Both the requested data sets and the Icelandic data sets will be imputed using reference sets from sequencing projects such as the 1000-genome project; association analyses will be conducted; and the results from the different data sets will be combined using standard meta-analyses techniques. The proposed research is consistent with Use Restrictions for the requested data sets, and we will follow all regulations therein.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Torkamani, Ali \",\n\"SCRIPPS RESEARCH INSTITUTE\",\n\"Identification of Genomic Risk Factors and Prediction of Cardiovascular Disease through Integrative Network Analysis and Machine Learning\",\n\"Oct09, 2020\",\n\"approved\",\n\"Our overarching goal is to pursue integrative multi-omics and systems level analyses of whole genome sequencing data from deeply phenotyped cohorts to define novel coronary artery disease biology, and to develop coronary artery disease risk prediction models that combine these discoveries with other polygenic and clinical features. While prior studies have shown basic coronary artery disease prediction models provide modest clinical utility, improved, comprehensive models can make this a reality of clinical practice. This information can be used to help guide therapy decisions and provide an impetus for optimizing lifestyle modifications, thereby improving health outcomes and clinical efficiency.\",\n\"Genome-wide association studies (GWAS) and other genetic studies have identified hundreds of genetic variants associated with coronary artery disease (CAD). Polygenic risk scores (PRS) limited to known CAD loci have been shown to be useful for the identification of high-risk individuals who could benefit most from initiation of statin therapy or lifestyle changes. However, currently developed PRSs have some limitations. These models cannot accurately predict which individuals will develop disease, and fail to: (1) incorporate rare pathogenic genetic variants, (2) consider sub-genome-wide significant genetic variants implicated in CAD pathogenesis through integrative analysis, (3) incorporate traditional clinical risk factors, (4) consider higher level interactions between loci, and (5) evaluate the utility of these models in non-European individuals. These limitations, individually and in combination, limit the clinical utility of PRSs in individual risk prediction and can only be overcome by the development of more sophisticated risk models. The goals of this proposal are: (1) to combined polygenic, rare variant, and clinical risk factor score for multi-ethnic prediction of coronary artery disease and myocardial infarction. (2) To identify novel genomic associations for coronary artery disease and myocardial infarction through multi-omic network enrichment analyses. (3) To integrate these novel genomic associations, and known genetic and non-genetic risk factors for prediction of coronary artery disease, myocardial infarction, and therapeutic intervention outcomes into standard PRS, regularized regression, and machine learning frameworks.  Analyses may include the Wellcomme Trust Case Control Consortium, CARDIoGRAMplusC4D, UKBiobank and other non-dbGAP data. No attempt will be made to cross reference individuals across databases.  Results of this research will be published and distributed broadly.  Note(1): general research use datasets will be used for imputation and to train other genomic encoding approaches.  Note(2): Our previous Research Use Statement and personnel included collaborators from Intel Corp, however, they have not been able to navigate the process to acquire ERA Commons + dbGAP accounts leading to rejection of our data request. Intel Corp is removed from this request and access to data.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Tripathy, Shreejoy \",\n\"CENTRE FOR ADDICTION AND MENTAL HEALTH\",\n\"Studying common genetic liability to substance use disorders across disparate substances\",\n\"Dec03, 2021\",\n\"approved\",\n\"Addiction and substance use disorders have a partially genetic basis. Genetic studies of substance use disorders have traditionally focused on a single substance at a time, for instance in studies of alcohol dependence, nicotine dependence, cannabis use disorder or opioid use disorder. However, recent research has shown that many of the same genetic factors may predispose people to all of these different substance use disorders. We plan to combine the results of multiple prior genetic studies of substance use disorders across different substances to try to find genetic variants and genes that influence substance use in general. Along the way, we hope to learn more about how substance use disorders influence and are influenced by other psychiatric and non-psychiatric disorders.\",\n\"Addiction and substance use disorders have a partially genetic basis. Genetic studies of substance use disorders have traditionally focused on a single substance at a time, for instance in studies of alcohol dependence, nicotine dependence, cannabis use disorder or opioid use disorder. However, recent research has shown that predisposition to these disparate substance use disorders can be accurately modelled by a single underlying genetic factor. We plan to combine the requested dbGAP summary statistics with other public summary statistics from outside dbGAP (e.g. from the Psychiatric Genomics Consortium) to perform a genome-wide association study of this underlying factor, a process which should not create any additional risks to participants as it preserves patient confidentiality. Through genetic correlation, Mendelian Randomization and post-GWAS gene prioritization analyses, we further plan to explore the interplay of substance use with other psychiatric and non-psychiatric traits, and uncover candidate causal genes involved in genetic liability to addiction and substance use disorders.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Uher, Rudolf \",\n\"DALHOUSIE UNIVERSITY\",\n\"Genomic predictors of developmental psychopathology\",\n\"Jan25, 2023\",\n\"approved\",\n\"Mental illness often develops in adolescence or in young adulthood and continues to affect people through their life. To effectively prevent the development of mental illness, we need to predict which young people may be at risk. We proposing to use the Million Veterans Project as a source of genetic predictors that we will apply to a new group of young people with information on mental disorders. This results will evalute if a genetic test may help predict which young person is at risk and who may benefit form early preventive interventions.\",\n\"The aim of this project is to predict which young people are at high risk for developing mental disorders, such as major depressive disorder, bipolar disorder, and psychosis. We will test a set of polygenic scores as predictors of developmental psychopathology in a cohort of youth with prospectively determined diagnoses of mental disorders. We would like to use genetic data from the Million Veterans Program to derive the polygenic scores for anxiety and for substance use. Although he genome-wide analyses of these traits have been published, we are required to obtain access to the dataset in order to obtain the summary statistics. We will apply the resulting polygenic risk scores to a cohort of over 200 youth to help predict anxiety, ADHD, depression, bipolar disorder, and substance use disorders. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Ursini, Gianluca \",\n\"LIEBER INSTITUTE, INC.\",\n\"Diabetes GRS and completed suicide\",\n\"Jun02, 2021\",\n\"rejected\",\n\"Our data on gene-expression in completed suicide brains point to energy metabolism as a contributor to the suicidal phenotype at the molecular level; additionally, our cases of suicide appear to have a significantly lower BMI than the other individuals. The proposed project plans to employ the genetic variants associated with risk for diabetes to construct a diabetes genetic score of risk and investigate its association, as a proxy of altered energy metabolism, with completed suicide in our sample of post-mortem human brains. The results of this study may help to identify biomarkers of suicide risk that could inform prevention.\",\n\"Research on genetic components of suicide relies mostly on hypothesis-driven approaches, aimed at suicide attempters. Our data on gene-expression in completed suicide brains point to energy metabolism as a contributor to the suicidal phenotype at the molecular level; additionally, our cases of suicide appear to have a significantly lower BMI than the other individuals. This project intends to leverage summary statistics from the MPV study to construct a diabetes Genomic Risk score (GRS) using the alleles associated with diabetes (European ancestry cohort) at different threshold of significance, to investigate whether the aggregate effect of GWAS-derived diabetes variants, as a proxy of altered energy metabolism, contributes to the risk for completed suicide in our sample of post-mortem human brains of European ancestry, including suicides. We hypothesize that diabetes GRS contributes to the prediction of case-control status in our cohort.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"VAN DEN OORD, EDWIN \",\n\"VIRGINIA COMMONWEALTH UNIVERSITY\",\n\"Integrating methylomic and genomic summary statistics to better characterize alcohol-related DNA methylation\",\n\"May11, 2020\",\n\"closed\",\n\"Despite extensive anti-drinking efforts, ~6.2% of US adults qualify for an alcohol use disorder and many more engage in habitual or problematic drinking. Mounting evidence suggests that DNA methylation may play a role in drinking behaviors. However, many DNA methylation sites are regulated by common genetic variation. The research objective is to utilize summary statistics from alcohol-related genome-wide association studies (GWASs) to determine if alcohol-related methylation is simply picking up on genetic signal or represents a unique biological signature separate from common genetic variation.   This project will analytically combine the summary statistics from alcohol GWAS and summary statistics from alcohol methylation-wide association studies (MWASs). The combination of the GWAS and MWAS results does not create any additional risk to participants as we will only be using summary statics and do not need any participant-level information to conduct the project. Specifically, we will test whether the top findings from the alcohol GWAS occur more frequently among the top findings of our MWAS (i.e., are enriched) than what is expected by chance.  \",\n\"Despite extensive anti-drinking efforts, ~6.2% of US adults qualify for an alcohol use disorder and many more engage in habitual or problematic drinking. Mounting evidence suggests that DNA methylation may play a role in drinking behaviors. However, many DNA methylation sites are regulated by common genetic variation. The research objective is to utilize summary statistics from alcohol-related genome-wide association studies (GWASs) to determine if alcohol-related methylation is simply picking up on genetic signal or represents a unique biological signature separate from common genetic variation.   This project will analytically combine the summary statistics from alcohol GWAS and summary statistics from alcohol methylation-wide association studies (MWASs). The combination of the GWAS and MWAS results does not create any additional risk to participants as we will only be using summary statics and do not need any participant-level information to conduct the project. Specifically, we will test whether the top findings from the alcohol GWAS occur more frequently among the top findings of our MWAS (i.e., are enriched) than what is expected by chance.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"van der Sluis, Sophie \",\n\"VRIJE UNIVERSITEIT\",\n\"Genetic relationship between 13 psychiatric disorders\",\n\"Nov10, 2020\",\n\"closed\",\n\"This project concerns a study on the genetic relatedness and distinctness of 13 psychiatric traits through an extensive series of analyses, focusing not only on the genetic overlap between these traits but also on their genetic uniqueness.\",\n\"This project concerns a study on the genetic relatedness and distinctness of 13 psychiatric traits through an extensive series of genetic analyses. Contrary to related projects, we will not only focus on the genetic overlap between these traits but also on their genetic uniqueness. The main objective of the project is to identify genes or gene sets that explain comorbidity between psychiatric disorders, and to identify genes or gene sets that are unique to these disorders. This study concern the analyses of GWAS summary statistics only. We will calculate (partial and local) genetic correlations, conduct genomic structural equation modelling and Mendelian randomization to scrutinize the genetic relatedness between the included disorders. If we have access to multiple GWAS summary statistics on the same trait in non-overlapping samples, then we will meta-analyze these sum stats first and then continue with the summary statistics from the meta-analysis.  For this project, we have obtained access to GWAS summary statistics of major psychiatric disorders such as schizophrenia, depression, anxiety, PTSD, anorexia, OCD, ADHD and Tourette syndrome. We wish to include alcohol use as a trait in this project as well because it is highly comorbid with many other psychiatric disorders, which makes studying the genetic overlap and uniqueness, and direction of causation, specifically interesting. The summary statistics that we here ask permission to are very valuable as these concern data on a large and well described sample, which would greatly improve our statistical power to detect overlapping and unique genes/gene sets for this disorder. All analyses in this project will be conducted by people from the CTG department at the VU (post doc Mats Nagel in particular), but we will collaborate with prof Murray Stein and prof Joel Gelernter, via whom we gained access to the latest GWAS summary statistics on PTSD from the VA Million Veteran Program.  The requested data will only be used for the purpose described here, i.e., to gain understanding on the (shared and unshared) genetics of mental health disorders. The data will not be used to study population origins or ancestry. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Voight, Benjamin \",\n\"UNIVERSITY OF PENNSYLVANIA\",\n\"Post-GWAS analyses for Million Veteran Program Summary Data\",\n\"Dec21, 2020\",\n\"approved\",\n\"The Million Veteran Program is a large-scale genetic biobank with over 500,000 participants with genotype and phenotype data available from diverse ancestries. Summary results from genome-wide association scans (GWAS) performed by the Program are available (dbGAP). These summary data are not human subjects data, are de-identified, and carry no personal health identifying information. We want to utilize the summary data to perform additional genetic analyses, to better characterize the genetic basis and architecture of traits analyzed by MVP, in order to identify causal variants, genes, tissues, and risk factors for these traits. \",\n\"We would like to use MVP summary data across the range of traits analyzed to date to perform a battery of computational post-GWAS analyses. These analyses are designed, broadly, to identify additional novel loci, or to triangulate on the causal variant, gene, and tissues that underlying the observed associations. These analyses also have the objective to characterize trait heritability and genetic correlation with additional traits, but also to perform causal inference studies with other, publicly available data sets to generate causal evidence between traits and disease endpoints.  The batter of analyses includes: causal inference studies via Mendelian Randomization, multivariate GWAS, conditional analysis, genetic fine-mapping, genetic co-localization analysis with GTEx and other QTL mapping data sets, and genetic correlation and heritability studies. At the time of this writing, summary data includes, but is not limited to: type 2 diabetes, lipid levels, blood pressure, eGFR, alcohol and substance use/abuse, PAD, and VTE.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Vrieze, Scott \",\n\"UNIVERSITY OF MINNESOTA\",\n\"GWAS & Sequencing Consortium of Alcohol and Nicotine\",\n\"Jul27, 2020\",\n\"approved\",\n\"Our goals are to identify genetic risk (and protective) factors for alcohol and nicotine use, to identify genetic risk factors that co-occur across multiple substance use behaviors, and to identify biological targets for future functional experiments or drug development relating to addiction or substance use.  To do this, we are aggregating data across dozens of studies to accumulate sample sizes of >1 million individuals to discover genes and other aspects of our DNA that are related to smoking and alcohol use behaviors. \",\n\"Genome-wide association meta-analyses have detected a handful of loci associated with heaviness of smoking, smoking initiation and cessation, alcohol use, and alcohol dependence, but many associated variants remain to be discovered. We are requesting access to studies from dbGaP to conduct genome-wide meta-analyses to better characterize the genetic etiology of nicotine and alcohol use. Phenotypic and genomic data from dbGaP studies will be used to: a) discover new loci associated with smoking and drinking phenotypes, b) detect shared genetic etiology across smoking and drinking phenotypes, and c) implicate novel gene sets and biological systems in the pathophysiology of addiction and use. Phenotypic data will include subject responses to questions about age of initiation of smoking, smoking cessation, heaviness of smoking, whether an individual has ever smoked regularly, and alcohol consumption (e.g., drinks per week, binge drinking). These studies will include individuals from diverse populations, including those historically under-represented in health research. We plan to publish all results in a peer-reviewed journal and share summary statistics from any meta-analyses with the broader scientific community.  Data from dbGaP may be meta-analyzed with data from other studies (obtained from dbGaP and other sources) to improve the power for novel genetic discoveries, while respecting the consent of all participants. However, each dataset from dbGaP will be analyzed independently prior to the meta-analysis. We expect that this activity creates no additional risks to participants. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Wang, Frances \",\n\"UNIVERSITY OF PITTSBURGH AT PITTSBURGH\",\n\"ENIGMA-Addiction\",\n\"Jul18, 2022\",\n\"approved\",\n\"Genome wide association studies (GWAS) identify genetic variants associated with a trait such as problematic substance use. We will use two GWAS from dbGAP which targeted nicotine dependence and problematic alcohol use to generate individualized genetic risk scores for these traits in other samples. We will examine whether these risk scores are related to behavior (e.g. age of onset and frequency of substance use, and measures of impulsivity) or brain features detected using MRI, including measures of brain structure (e.g. cortical and subcortical grey matter volume and white matter tract integrity) and function.\",\n\"Polygenic risk scores (PRS) are a powerful tool to examine the genetic liability of a specific trait in another population. Our aim is to investigate the genetic liability to substance use disorders (e.g., alcohol) in population-based cohorts in relation to behavior measures and to identify brain intermediate phenotypes measures associated with greater genetic risk for problematic alcohol use. PRS will be create using the PLINK 2 software based on the results of the Genome Wide Association Studies (GWAS) problematic alcohol use (PMID 32451486 - phs001672.v3.p). These PRS will be regressed onto behavioral phenotypes (e.g., substance use measures including age of onset and frequency of use, and impulsivity measures) and onto brain measures including structure (cortical and subcortical measures and diffusivity) and function. This project does not require individual-level data from the mentioned studies and it is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Wareham, Nicholas \",\n\"UNIVERSITY OF CAMBRIDGE\",\n\"Molecular basis of human metabolism and relevance to disease outcomes\",\n\"Sep20, 2021\",\n\"rejected\",\n\"Disorders of human metabolism such as type 2 diabetes, its risk factors and complications (e.g. heart attacks) affect millions of people worldwide. A key limiting factor in the development of preventive and therapeutic strategies is an incomplete understanding of the disease mechanisms. With this proposal we plan to combine information about the genetic makeup and the environmental risk factors to understand the causes of these metabolic diseases and help the development of strategies to prevent and treat these conditions.  \",\n\"Our research aims to better understand the genetic and molecular determinants of human metabolism, their interplay with the environment in shaping the risk of health outcomes in order to enable the development of safe and effective preventive or therapeutic strategies. To gain these insights, our approach is to integrate data from large-scale population-based studies, including genomics, multi-\\u00e2\\u0080\\u0098omics\\u00e2\\u0080\\u0099 measurements (e.g. metabolomics, proteomics, transcriptomics, etc.), detailed anthropometry, growth and development-related measurements,  nutritional measures and biomarkers, physical activity and other behavioural variables, environmental exposures, electronic health records, functional characterisation of genetic variants or genes in experimental models. Our team have had a leading role in large-scale studies identifying genetic determinants of metabolic traits (e.g. adiposity, insulin resistance, glycaemic measures) and diseases (e.g. type 2 diabetes) and the understanding of their role in human metabolism via the integration of genetic data and with phenotypic measurements. The aims of this proposal are: (a) discovery and characterization of genetic determinants of metabolic diseases and related conditions (as defined by the GERA as disease outcomes e.g. type 2 diabetes and cardiovascular disease) via genome-wide association studies or other relevant approaches; (b) observational and genetic epidemiology (e.g. Mendelian randomization) studies of the aetiologic relevance of risk factors to these diseases; (c) studies of naturally-occurring genetic variants known or suspected to affect the function of gene products in order to identify or validate drug targets for these conditions; (d) identification of causal genes and mechanisms for these conditions via fine-mapping, study of the co-localisation of quantitative trait loci or other integrative approach and (e) development and use of novel methods in molecular epidemiology.  We will combine the requested data with existing publicly available data in a meta-analysis of results. We will not meta-analyse individual level data across studies. There are no additional risks to participants.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"WENDT, FRANK \",\n\"UNIVERSITY OF TORONTO\",\n\"Genetic and Environmental Factors Associated with Internalizing Psychopathologies\",\n\"Dec20, 2022\",\n\"approved\",\n\"Many environmental and genetic factors associate with whether a person has a diagnosis of depression, anxiety, posttraumatic stress, and related disorders or diseases. This study is designed to investigate these various factors using sophisticated statistical approaches applied to large-scale aggregated data from dbGaP and external cohorts. The analytic approach investigates different types of genetic markers associated with the above traits, their interaction with one another and with environmental variables, and the ways in which these relationships may be used to mitigate disorder symptoms. Our goal is to deliver the scientific community with a collection of genetic positions and environmental variables with large effects on diagnosis. \",\n\"The internalizing spectrum of psychiatric disorders includes major depression, generalized anxiety, and posttraumatic stress disorders including their transdiagnostic features like neuroticisms and suicidal thoughts and behaviors. Large genetic studies support a heritable component of these traits. This project has three goals: (i) develop methods using genetic data to identify participants at high risk for disorder, (ii) uncover symptom heterogeneity in diverse populations, (iii) identify relevant brain mechanisms of psychopathology across the lifespan using large genome-wide association studies. Upon completion of our studies, we will have a better understanding of the various types of genetic loci that contribute to internalizing psychopathologies, including the potential for disseminating large-effect genetic variants and environments to the scientific community. In combination, these results may inform one path towards pharmacological and non-pharmacological intervention strategies.  Our study design relies of the use of genotype-phenotype association, imputation of various marker types, polygenic scoring, gene-by-environment genome-wide interaction testing, predicted gene expression, etc. For datasets that contain whole-exome and/or whole-genome sequencing data, we also plan to extract genotypes for multi-allelic loci across the genome using various read alignment strategies. All associations between phenotype~genotype and phenotype~environment will include baseline covariates like age and sex. Genetic analyses will consider genetic ancestry as an additional covariate. Because many of the requested datasets were not created to study internalizing traits, we also will include study ID as a covariate to mitigate the effects of targeted ascertainment in the diverse list of requested studies. Quality control protocols include detection of related individuals using genetic data, population stratification to inform genotype-phenotype relationships among diverse populations, and imputation quality thresholding for various genomic loci. Data from this dbGaP request may be either combined or meta-analyzed with other datasets including, but not limited to, UK Biobank, the Million Veteran Program, Biobank Japan, FinnGen, and internally curated genetic and psychiatric studies. We are requesting several datasets with the goal of merging and/or meta-analyzing across cohorts and with the external cohorts listed above, where appropriate, to improve statistical power of association. We anticipate no risk to participants as a consequence of meta-analysis.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Wingo, Aliza \",\n\"EMORY UNIVERSITY\",\n\"PWAS of PTSD in Veterans\",\n\"Apr08, 2020\",\n\"closed\",\n\"We will use the results of the genome-wide association study of PTSD with the protein data we generated to investigate the proteins in the brain that contribute to risk of PTSD.\",\n\"Objective: PTSD is a debilitating disorder and its treatment is ineffective in a large subset of the patients. Our goal is to identify brain proteins contributing to PTSD to facilitate effort in finding novel treatment for PTSD. Study design: We hypothesized that genetic variants lead to altered protein abundance in the brain, which in turn leads to susceptibility for PTSD after trauma exposure. To test this hypothesis, we will adapt a published pipeline (FUSION by Gusev et al 2016) that integrate the PTSD genome-wide association study (GWAS) summary statistics and assayed protein expression data to perform an imputed protein study of PTSD.  Analysis plan: Our group has generated brain proteomes from post-mortem brain samples. We will estimate brain protein expression weights from the brain proteomes and their corresponding genome-wide genotyping using Gusev et al 2016 FUSION method. Next, we will integrate the requested GWAS summary statistics with the protein weights to perform a proteomic analysis of PTSD following FUSION pipeline.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Wingo, Thomas \",\n\"EMORY UNIVERSITY\",\n\"Proteome-wide association study of Psychiatric Diseases in MVP\",\n\"Sep03, 2020\",\n\"approved\",\n\"We will use the results of the genome-wide association study of psychiatric disorders (e.g. anxiety, substance abuse) with the protein data to investigate the proteins in the brain that contribute to the risk of these diseases and facilitate findings for new treatments.  \",\n\"Research Objective.: Genetic variants that alter brain protein abundance may contribute to the development of different psychiatric diseases, such as depression, anxiety, and addiction to substance abuse (including opioids, cannabis, and alcohol). These disorders are debilitating and treatments are ineffective in a large subset of the patients. We propose to impute protein expression levels using summary statistics from the summary GWAS from MVP (e.g. Levy et al. 2020, AmJ Psychiatry; Zhou et al. 2020, Jama Psychiatry; Zhou et al 2020, Nature Neuroscience), to perform a proteome-wide association study (PWAS) of psychiatric diseases.   Our goal is to identify brain proteins contributing to these disorders to facilitate effort in finding novel treatments.  Study design: We will adapt a published pipeline (FUSION by Gusev et al 2016) that integrates the genome-wide association study (GWAS) summary statistics from MVP and assayed protein expression data to perform an imputed protein study of psychiatric disorders (e.g. anxiety and substance abuse).   Analysis plan: We will use as reference a brain proteome from post-mortem brain samples, already generated from our group. We will estimate brain protein expression weights from the brain proteomes and their corresponding genome-wide genotyping using Gusev et al 2016 FUSION method. Next, we will integrate the requested GWAS summary statistics with the protein weights to perform a proteomic analysis of psychiatric disorders following FUSION pipeline. We may also compare these results with the one obtained from other disorders (e.g. neurodegenerative diseases) or traits (personality, cognition, anthropometric) to identify overlapping and unique genes/pathways responsible for those disorders.   \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"WON, HYEJUNG \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"Elucidating neurobiological mechanisms underlying alcohol dependence \",\n\"Jan13, 2020\",\n\"closed\",\n\"Substance use disorders remain as a public health crisis. Genetic studies have elucidated genomic regions that are associated with alcohol dependence, while the functional impact of those regions remains to be studied. With our analyses, we hope to identify alcohol dependence risk genes from the genomic regions. Characterization of alcohol dependence risk genes will bring us closer to understanding the core biological mechanisms underlying alcohol dependence, and provide a foundation for developing treatment strategies for alcohol dependence. \",\n\"Genome-wide significant association studies (GWAS) have identified a plethora of genetic variants associated with complex traits such as substance use disorders (SUDs). However, most genetic risk variants reside in non-coding regions of the genome. Several gene mapping tools including MAGMA have been developed to assign GWAS SNPs to their cognate genes. We recently developed Hi-C coupled MAGMA (H-MAGMA), which harvests the capacity of MAGMA while leveraging chromatin interactions to assign non-coding SNPs to their cognate genes. In our previous study in which we applied H-MAGMA to psychiatric disorders, we have shown that H-MAGMA can dramatically increase the power of MAGMA by identifying more genes based on functional genomic evidence (Sey et al., bioRxiv 2019). Here, we plan to apply our H-MAGMA framework to identify genes associated with alcohol dependence. Once SNPs are mapped onto the genes, the subsequent functional analyses on the alcohol dependence-associated genes can elucidate the neurobiological mechanisms underlying alcohol dependence. Data from MVP will be used for research purposes only and no personal identifiers will be included in any of our analyses. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Won, Hong-Hee \",\n\"SUNGKYUNKWAN UNIVERSITY RES/BUS FDN\",\n\"Investigation of association between 14 micronutrients and 22 cancer outcomes using Mendelian randomization analyses\",\n\"Apr12, 2023\",\n\"approved\",\n\"Numerous observational studies and clinical trials have reported associations between micronutrients, namely vitamins and essential metals, and various health outcomes including cancers. Many people are interested in risk factors associated with various cancers. Many observational studies have reported the benefits of supplementing micronutrients on cancer risks, whereas randomized controlled trials (RCTs) more often reported a null effect. However, observational studies are inherently prone to confounding, which may result in inflated estimates, and RCTs are costly and often insufficiently powered for cancer outcomes requiring long-term follow-up and large cohort size. More evidence is needed to know whether there is actually causal relation. In this study, we will use genomic data to investigate which micronutrients can affect risks for various cancers.\",\n\"Identifying the precise causal risk factors of various cancer is important to reduce social and economic burdens. Numerous observational studies and clinical trials have reported associations between micronutrients, namely vitamins and essential metals, and various health outcomes including cancers. In this study, we will investigate causal relationships between associated risk factors such as micronutrients and cancers. To identify causal associations of micronutrients with cancers, we will use a two-sample Mendelian randomization (MR) approach using genome-wide association study (GWAS) summary statistics. We will not combine requested datasets with other datasets outside of dbGaP and we will analyze these datasets independently.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Won, Hong-Hee \",\n\"SUNGKYUNKWAN UNIVERSITY RES/BUS FDN\",\n\"Discovery of additional loci and improvement of disease prediction using the genetic relationship of polygenic traits\",\n\"Mar02, 2023\",\n\"approved\",\n\"Genetic variants associated with various common diseases are being actively discovered through GWAS. Typically, GWAS is conducted to target a single disease which prohibits the investigation of the complex network between diseases. Furthermore, lack of power limits finding all the true associations between genetic variants and disease. Our goal is to analyze various genetically correlated traits related to metabolic, psychiatric, and neurodegenerative disorders through a multivariate approach, followed by post-GWAS analyses to underpin their biological functions.\",\n\"Genome-wide association studies (GWAS) are fundamental research in genetics. The genetic variants associated with the common disease are being actively discovered. However, GWAS is typically conducted for a single disease which prohibits investigation of the complex interconnection between the diseases. Furthermore, GWAS of many diseases is still limited in terms of a sample size to fully reveal the underlying associations and the fact that causation cannot be determined. Recent methodologies such as multivariate GWAMA and Genomic SEM utilize genetic correlation between multiple diseases to define a common latent phenotype having shared genetic effects. The construction of such genetic architecture is useful to disentangle the relationship between interconnected diseases. Additionally, the statistical power to detect additional genetic variants will increase due to the larger sample size. Besides, it is also possible to discover additional genetic variants for the selected disease by conditioning the genetic effects of closely related diseases using conditional FDR. We aim to apply these approaches to various metabolic, psychiatric, and neurodegenerative disorders. The GWAS result will be followed by thorough post-analysis such as polygenic risk score calculation and evaluating its performance, assessing the shared proportion of genetic variants through MiXeR, detecting causal relationships through mendelian randomization, and so on. Newly discovered genetic variants and their biological relevance are expected to be useful for targeting new biomarkers and drugs. The requested data will be combined with external data through meta-analysis. There would be no harm to participants since it is impossible to identify individuals from summary statistics. All users agree to make results of studies using the data available to the larger scientific community and will abide by terms of data use certification restrictions and limit analyses accordingly and will honor all publication embargo dates. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Won, Hong-Hee \",\n\"SUNGKYUNKWAN UNIVERSITY RES/BUS FDN\",\n\"Causal inference between associated risk factors and Alzheimer disease, and identification of novel genetic variants for Alzheimer disease\",\n\"Jun22, 2022\",\n\"approved\",\n\"Many people are interested in preventing dementia. For example, people are wondering what diseases should be avoided, whether they should exercise, whether they should quit smoking, and whether they should be on a diet. Epidemiological studies on many people have shown that some diseases or lifestyle habits are associated with dementia, but more evidence is needed to know whether there is actually causal relation. In this study, we use epidemiological and genomic data to investigate which risk factors can control Alzheimer dementia.\",\n\"Identifying the precise causal risk factors of Alzheimer\\u00e2\\u0080\\u0099s disease (AD) is important to reduce social and economic burden. Candidate associated factors suggested by traditional epidemiological studies can be biased by reverse causation, confounding factors, and measurement errors. To overcome these problems and identify causal risk factors, the Mendelian randomization (MR) method has been proposed and is currently widely used. MR analysis is less likely to be influenced by social and economic bias because germline mutations are not affected by confounders.In this study, we will investigate causal inference between associated risk factors and AD, and identify novel genetic variants for AD. First, we will identify associated risk factors for AD from independent healthcare datasets (Elderly cohort in South Korea, UK biobank, and Korean genome and epidemiology study). We will employ the time-varying Cox hazard model with outcome of overall survival free of AD. All candidate associated factors for AD will be tested as independent variables of interest. This step will be performed independently of the NIAGADS data. Second, we will identify causality of these associated factors for AD using one-sample Mendelian randomization (OSMR). The OSMR estimates a polygenic risk score (PRS) related to the significant associated factors for each sample using the PLINK software. Summary statistics of genetic association studies of associated factors from LD hub (http://ldsc.broadinstitute.org/ldhub/) will be used to estimate the PRS. Next, we will test for the association of the PRS with incident AD using a logistic-regression analysis adjusted for covariates such as age and sex in the NIAGADS data. Third, we will investigate biological pathways shared by the causally-related phenotypes. The DEPICT tool can be used to prioritize and suggest associated biological pathways by identifying the most likely causal genes of identified loci using various databases.Our study will provide evidence for the causal risk factors for AD.  Use of this data is limited to health/medical/biomedical purposes, does not include the study of population origins or ancestry. All users will abide to terms of data use certification restrictions and limit analyses accordingly and will honor all publication embargo dates. Findings will be disseminated and shared with the scientific community by presenting results at national scientific meetings and publishing in peer-reviewed journals. This genetic study will conduct to learn about, prevent, or treat health problems; will not use at profit purpose. The aims of study will limit to health research on dementia, especially genetic basis. This study is related to AD, a neurodegenerative disease. This study will be conducted in the non-profit organization (SungKyunKwan Universty).\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"WRAY, Naomi \",\n\"UNIVERSITY OF QUEENSLAND\",\n\"Study of the genetic contribution to aortic aneurysm\",\n\"May11, 2022\",\n\"approved\",\n\"Aortic aneurysm has a strong genetic contribution but due to the inadequate sample sizes, our understanding of AA is lacking. Meta-analysis of the summary statistics from the Million Veteran Program with data from other programs will increase our ability to understand the genetic architecture of AA.\",\n\"Aortic aneurysm is a common cardiovascular disease that could be lethal if managed inappropriately. The ability to predict the onset and progression of AA is important as the management strategies for different stages of AA is different. Taking genetic factors into the prediction is meaningful as evidence showed that there is a strong genetic component in AA \\u00e2\\u0080\\u0093 twin studies showed heritability for AA could be as high as 70%. However, only ~25 loci have shown to be related to AA so far, potentially attributable to inadequate sample size to detect the difference. In our study, we are planning to meta-analyse all available GWAS sum statistics with our own data(N ~ 7500). Incorporating summary data from MVP would be invaluable to our research because of its large sample size. With MVP data and other sumstat from FinnGenn, Biobank Japan, and our own research, the total sample size will increase to N=25,000. We hope this will increase our power to detect significant SNPs related to AA.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"WRAY, Naomi \",\n\"UNIVERSITY OF QUEENSLAND\",\n\"The effect of blood pressure drugs on PTSD - a Mendelian randomisation analysis\",\n\"Oct05, 2020\",\n\"closed\",\n\"We aim to use genetic information to try and predict the effect of different classes of blood pressure drugs on PTSD risk and PTSD symptoms\",\n\"Observational studies suggest that some blood pressure drugs may have beneficial effects on PTSD symptoms. This project will use GWAS summary data from the Million Veteran Program as well as expression quantitative trait loci summary data from GTEX, PsychENCODE and eQTLGen to conduct a two-sample summary-based Mendelian randomisation analysis to predict the potential effect of different classes of blood pressure drugs on PTSD risk and risk of re-experiencing PTSD symptoms.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Wu, Lang \",\n\"UNIVERSITY OF HAWAII AT MANOA\",\n\"Identifying methylation and protein biomarkers of Type 2 diabetes using genetic instruments\",\n\"Jun10, 2022\",\n\"approved\",\n\"Type 2 diabetes brings huge public health burden. We propose to conduct studies using genetic instruments to better understand methylation and protein biomarkers influencing risk of Type 2 diabetes.\",\n\"It remains largely unknown for which DNA methylation and protein biomarkers may influence risk of Type 2 diabetes (T2D). Recently, there has been a new approach of using genetic instruments to evaluate associations between exposures and diseases. For example, novel disease susceptibility genes may be identified by evaluating associations of genetically-determined gene expression with disease risk (transcriptome wide association studies (TWAS) design). To identify novel DNA methylation and protein biomarkers associated with T2D, we plan to perform research using relevant T2D GWAS summary statistics data and data from several studies (e.g., MESA etc) containing relevant genetic and molecular information well suited for our research goals. In brief, we will develop molecular prediction models to represent the genetically-determined exposure of interest, and assess their associations with T2D. We hope our work will generate new knowledge for T2D. We plan to broadly share our results with the scientific community by publishing the findings. Data from the requested datasets will only be used in research consistent with the data use limitations of the corresponding datasets and will not be combined with other datasets of other phenotype.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Wuest, Stefan \",\n\"UNIVERSITY OF REGENSBURG\",\n\"LawStress: Psychological, endocrine, central nervous and genetic predictors of chronic stress responses in daily life \\u00e2\\u0080\\u0093 a prospective-longitudinal quasi-experiment\",\n\"May23, 2022\",\n\"approved\",\n\"Chronic stress increases the risk for several disorders, but the mechanisms mediating this effect are not fully understood. Our project aims at an exploratory analysis of the predictive value of a polygenic score (PGS) regarding phenotypes related to chronic stress responses in daily life. In a longitudinal study with a sample of healthy law students, we investigated a stress group experiencing a 13 months stress phase (preparation for a major exam) and a control group. At baseline, several predictor variables have been assessed, including the neural response to a stress induction in a MRI scanner. Subsequently, perceived stress levels, anxiety and depression symptoms as well as indicators of the regulation of the hormone cortisol have been assessed repeatedly over 13 months. With the planned polygenic score modelling we would like to study if the genetic liability to depression is related to these variables in individuals experiencing two different environments (with and without particular stress exposure).    \",\n\"There is ample evidence for an association between chronic stress and several disorders, including depression, anxiety, cardiovascular diseases as well as immune function related pathologies. However, biopsychological mechanisms mediating this link are not fully understood. Our project aims at investigating predictors of chronic stress responses in daily life. In a prospective-longitudinal study with a quasi-experimental design and a sample of healthy law students, we investigated a stress group experiencing a 13 months stress phase (preparation for the first state examination for law students) and a control group. We will use summary statistics generated by genome-wide association studies (GWASs) conducted within the Million Veteran Program dataset to investigate the combined predictive value of a polygenic score (PGS) and the environmental stressor (state examination) on changes in depression symptoms and other stress related phenotypes (e.g. momentary perceived stress levels, markers of hypothalamic-pituitary-adrenal axis regulation and responses to a standardized fMRI stress induction paradigm).  These exploratory analyses will be conducted in a multilevel framework to control for individual differences in intercept and slopes of the dependent variables. We are not requesting individual level data.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"xiongjian, luo \",\n\"KUNMING INSTITUTE OF ZOOLOGY\",\n\"Integrative analysis of Post-traumatic stress disorder\",\n\"Jun18, 2020\",\n\"approved\",\n\"Our study will help to find the causal risk variants and genes of PTSD. Further functional exploration of these causal risk variants and genes will uncover the genetic regulation mechanisms and potential disease etiology of PTSD. These results will further improve the disease diagnosis and treatment.\",\n\"Post-traumatic stress disorder (PTSD) is a severe mental health problem which develop by someone who exposure to a life-threatening events. The Million Veteran Program conducted a PTSD GWAS study (dbGaP Study Accession:phs001672.v3.p1) identified several risk loci of PTSD. We will utilize the PTSD summary statistics for further investigation the causal risk variants and genes of PTSD. Based on the PTSD GWAS summary statistics, we will pinpoint the causal mutations by utilize the public available epigenomic data and chip-seq data from FANTOM, ENCODE, and Roadmap projects. Furthermore, we will also prioritizing PTSD genes by integrating the PTSD summary statistics with brain eQTL data by Sherlock, SMR. TWAS (transcriptome-wide association analysis) will also help to find the key genes by integrate PTSD GWAS summary statistics and brain transcriptome data. We claimed that we will only use the summary statistics data for research only and have no intention to request the individual participants data, and there are no additional collaborations in this project. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"YANG, BAO-ZHU \",\n\"YALE UNIVERSITY\",\n\"Post-GWAS analyses on the genetics of addictions, medical and psychiatric disorders, and behaviors\",\n\"Dec05, 2022\",\n\"approved\",\n\"Differences in genetic makeup contribute to variations in behavior, health, and diseases. We will use the summary information derived from the Million Veteran Program\\u00e2\\u0080\\u0099s genome-wide association studies (GWASs) to help understand the genetic basis of addictions, medical and psychiatric disorders, and behaviors.\",\n\"Differences in genetic makeup contribute to variations in behavior, health, and diseases. The Million Veteran Program (MVP) has generated rich data, including genome-wide association studies (GWASs)\\u00e2\\u0080\\u0099 summary statistics. We will use MVP GWAS summary statistics to conduct post-GWAS analyses, such as (but not limited to) computing genetic correlations, generating polygenic scores, converting to the gene-based association, conducting Mendelian randomization, carrying out meta-analyses, or performing pathway or network analysis to examine the genetic basis of behavior, health, and diseases, including addictions, medical and psychiatric disorders, and behaviors. We will analyze MVP GWAS summary statistics independently, combine them with GWAS summary statistics for traits from other sources, or link to gene expression data such as the Genotype-Tissue Expression (GTEx) data. We will not combine datasets but integrate the MVP GWAS summary statistics with summary information for other traits. Our overarching objective is to decipher the biology or pathways of behavior, health, and diseases through genetic summary information and gene expression data. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Yang, Qiong \",\n\"BOSTON UNIVERSITY MEDICAL CAMPUS\",\n\"Mendelian randomization study of causal relationships between serum proteins and cardiovascular phenotypes\",\n\"Aug09, 2019\",\n\"closed\",\n\"We will correlate the genetic association results of proteins and cardiovascular disease risk factors (lipid levels and blood pressure) to uncover new biological pathways.\",\n\"Recent advances in aptamer-based proteomic technologies provide high sample throughput, rendering the profiling of low abundance analytes in epidemiological cohorts far more feasible.  We have recently completed genome-wide association studies of ~1300 aptamers in about 3,000 individuals , which provides opportunity to study the potential causal relationships between proteins and  other cardiovascular phenotypes using the Mendelian randomization methods. In this study, we will interrogate the genetic association results of protein profiles in our study and those of lipid levels and blood pressure phenotypes in Million Veteran Program (MVP) to infer causal relationships using various two sample Mendelian randomization techniques.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Yazdi, Puya \",\n\"GENIUS LABS COMPANY\",\n\"Validation of novel robust relative risk score models for common diseases across multiple populations\",\n\"Apr19, 2023\",\n\"approved\",\n\"Disease risk is determined by the combined influences of genetics and environment. The genetic component of disease risk can be estimated through the use of robust polygenic risk score (PRS) models, which estimate the risk of developing a certain disease, relative to the general population.Through this project, we aim to test and improve the risk models we have previously developed for multiple common diseases and biomarkers--measurements used to infer health status and risk. We are assessing these models for robustness, accuracy, and sensitivity across multiple populations. We are primarily concerned with models of relative risk for cardiovascular diseases, diabetes, hypercholesterolemia, dementia, stroke, and the various biomarkers associated with these disease states. This work will further our understanding of how genomic differences can influence individuals\\u00e2\\u0080\\u0099 relative risks of various diseases.  \",\n\"We have developed several methods of generating PRS models: from GWAS summary statistics alone, summary statistics combined with individual level data in a hybrid model, and a novel absolute risk model which incorporates levels of various biomarkers as priors. Validation of these PRS models against simulated data  indicates the models are robust and potentially applicable to populations of EUR and non-EUR ancestry. However, without testing against real datasets unassociated with the training and validation sets, we cannot be confident in these claims. We propose to use the genotype and phenotype information included in these dbGAP datasets to test our PRS models. We aim to use the information about the models\\u00e2\\u0080\\u0099 limitations gained from this testing phase to improve these models to make them more broadly applicable.   We further propose to utilize a new linear mixed model we have developed for association testing, to identify additional disease associated variants from these datasets.   We may compare the SNP frequencies and other statistics with outside databases, but we will use the data provided in accordance with the Data Use Limitations and Certification of each study.   We plan to publish the software associated with these PRS models under an open-source license, to contribute to the improved discovery of disease-significant genetic variants, and greater understanding of the differences in genetic architecture between populations.   \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Zhang, Haitao \",\n\"NIH\",\n\"Genetic association and polygenic risk scores of substance use disorders and related psychiatric diseases\",\n\"Jan19, 2021\",\n\"approved\",\n\"The proposal will utilize the large, well-characterized dbGaP summary results and datasets to derive Polygenic risk scores (PRS) as genetic components in epidemiologic studies and prediction model in substance use disorders and related psychiatric diseases. We will also validate the findings of our own genetic association studies on various psychiatric disorders and substance use disorders, such as mood disorder, anxiety disorder, Posttraumatic stress disorder, personality disorder, eating disorder, alcohol use disorder, nicotine use disorder, and drug use disorders. \",\n\"Polygenic risk scores (PRS) derived from dbGaP summary statistics will be used as genetic components in epidemiologic studies and prediction model in substance use disorders and related psychiatric diseases. In addition, we will utilize dbGaP summary results to further replicate/elucidate susceptibility genes identified in our genetic epidemiology research. We will combine the dbGaP summary results with our dataset in the process of analysis. Because we only use dbGaP summary results in population level in our research analysis, it will NOT create any additional risks to participants in dbGaP projects. The genetic data in our lab are generated from the genotyping array of Saliva DNA, with the aim of discovering potential genetic variants in various psychiatric disorders and substance use disorders, such as mood disorder, anxiety disorder, Posttraumatic stress disorder, personality disorder, eating disorder, alcohol use disorder, nicotine use disorder and drug use disorders. We intend to request dbGaP datasets to further validate the genetic variants identified from our own genetic studies. To address the issue of the different genotyping array between dbGaP datasets and our own studies, imputation approach will be used to get fully shared genetic data between different GWA studies. Meta-analysis will be primarily performed to identify genes contributing to these mental disorders. Since psychiatric disorders may have some shared as well as specific genetic effects, we plan to carry out an additional meta-analysis to identify genetic factors contributing to shared genetic components between two or more disorders by combining the genetic data from GWA studies on different traits. These meta-analyses will utilize large samples and thus gain more power to detect genes predisposed to complex disorders. Despite the important roles of copy number variations (CNVs) in causing many human complex diseases, especially psychiatric diseases, CNVs were not well studied in published GWA studies. The CNVs will be called using combined methods in our study plan and the association of CNVs with psychiatric disorders will be further tested. In addition, gene x gene interactions and gene x environmental interactions may be involved in the development of complex disorders and thus these interactions will be also evaluated. Given that susceptibility genes are likely to have small effects on the complex diseases, we will also test the effects of biological pathway and gene network on psychiatric disorders. Here we are more interested in potential common biological pathway shared among different psychiatric disorders. In addition, we will evaluate the genetic correlation between the genetic values for two or more diseases or disorders. We will also integrate data from GWAS and eQTL studies to identify genes whose expression level is associated with psychiatric disorders. Combining datasets and performing comprehensive genetic analysis from multiple genetic studies hold great promise in a better understanding of the genetic causes of psychiatric disorders and their comorbid disorders and thereby lead to better ways of prevention, diagnosis, and treatment. These dbGaP data will be used only for research purposes. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Zhang, Shijie \",\n\"TIANJIN MEDICAL UNIVERSITY\",\n\"Identification and prioritization of causal regulatory variants for cardiovascular disease based on GWAS data\",\n\"Oct17, 2022\",\n\"approved\",\n\"GWAS have successfully identified many disease/trait associated variants, but the reported leading associated variants are unlikely to be the actual causal variants. Associated loci may contain hundreds of variants spanning large regions and encompass several genes, making the biological interpretation of GWAS loci more difficult. We plan to combine eQTL data, GWAS data and epigenomic data to identify and prioritize functional eQTLs and causal variants that explain the CAD pathogenesis. Using experimental validation, we will investigate the biological mechanism of functional variants and uncover how these genetic variants affect gene expression. The study could provide a comprehensive resource of casual regulatory variants in CAD development. \",\n\"Over a hundred of coronary artery disease (CAD) genetic risk loci have been identified in the last decade through genome wide association studies (GWASs). A number of studies have experimentally validated regulatory effects of risk allele for complex diseases at individual loci. Yet for most of these associations the true causal variants remain unknown, especially for the associated variants located in the noncoding genomic region. Million Veteran Program (MVP), a nationwide cohort drawn from an integrated healthcare system serving a diverse population including many Black and Hispanic people. Previously, we have developed several computational methods to predict and prioritize regulatory variants in context-dependent manner (Li MJ et.al. Genome Biology. 2017 16;18(1):52). We plan to (1) collect public eQTL and GWAS data. (2) perform colocalization analysis based on eQTLs and the result of GWAS meta-analysis of CAD, with additional information of chromatin states, epigenetic markers as well as chromosome interactions, to identify and prioritize functional eQTL and causal variants. (3) Perform experimental validations to investigate the regulatory effect and potential mechanism of several identified causal regulatory variants for CAD susceptibility.. We expect that our study could provide new insights into the genetic basis and pathogenesis of CAD. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Zhang, Zhengdong \",\n\"ALBERT EINSTEIN COLLEGE OF MEDICINE\",\n\"Evolutionary analysis of aging and age-related disorders in a large multi-ethnic cohort\",\n\"Feb10, 2021\",\n\"rejected\",\n\"Identification of genetic factors underlying longevity is of paramount importance for the healthcare of the ageing population. It is believed that the specific pattern of disease coincidence in the ageing population is partly influenced by the genetic makeup of the respective population. Adaptation over the several thousand years has shaped the population gene pool and resulted in signatures of selection to be preserved in the genome of populations. Through analysis of signatures of selection across different age bins, we aim to capture the impact of ongoing selection on human longevity and assess the survival burden of genetic variants contributing to human quantitative traits. The ultimate goal of our project is to identify pro-longevity variants that enable survival to older ages and disentangle variants that increase the chance of mortality.\",\n\"Identification of ethnic-specific comorbidity patterns and evolutionary processes underlying age-specific mortality is of practical interest for the healthcare system. Evidence in favor of hard selective sweeps across the human genome, and often, evolutionary scans of the genome for identification of selective sweeps is confounded by population history.  Through the application of diverse methods for identification of selective sweeps across the genome of the ageing population we envisage identification of loci that are under directional natural selection across the genome of the healthy ageing population.  By aggregating data from our large in-house cohort of centenarian Ashkenazi Jewish population we plan to 1) investigate patterns of allele frequency changes across different age groups while controlling for confounding factors such as ethnicity and population structure; 2) we will assess the degree of epistasis between many pairs of SNPs in the context of longevity through composite likelihood modeling, and; 3) investigate the burden of recessive and loss of function mutations in advanced age through the development of robust statistical methods  Through investigation of ethnic-specific comorbidity patterns we aim to discover covariance between age-related traits and identify pleiotropic effects that are shared between anthropomorphic traits that correlate with human life span.  Ultimately by extending the statistical method recently developed by Jacobs et al. we aim to scan the entire genome for known and novel signatures of selection. By utilizing the GERA, the Framingham, and the Million Veteran cohorts, we will comprehensively and systematically analyze genetic variants influencing human survival in the extant human population.  Confidentiality of research participants is ensured by using anonymized data. The requested data will be deposited to the firewall-secured institutional storage server and will be accessible to the authorized applicants through the secured UNIX shell environment. \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"ZHU, XIAOFENG \",\n\"CASE WESTERN RESERVE UNIVERSITY\",\n\"Statistical analysis of high-throughput genotype and sequencing data\",\n\"Jul30, 2020\",\n\"approved\",\n\"There is pressing need for new analytical tools to extract meaningful information from the existing large amounts of genomic data. We plan to apply our developed novel statistical methods to analyze biological mechanisms using existing genomics data and for detecting rare and common genetic variants of associated with Alzheimer's disease. \",\n\"Alzheimer's disease related Phenotypic and genomic data from the dbgap data will be used for investigating genotype-phenotype associations to detect genetic variants associated with the disease; and (b) test new statistical methods for genetic association analysis. Data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses.  We will not make the data available to anyone outside of the named collaborators on the DAR.   We will report any findings in scientific journals.  \",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"ZHU, XIAOFENG \",\n\"CASE WESTERN RESERVE UNIVERSITY\",\n\"Genetic analyses of sleep apnea and hypertension related traits\",\n\"Jul16, 2020\",\n\"approved\",\n\"Obstructive sleep apnea (OSA) is a common disease which consists of repeated awakenings while gasping for air. OSA is an important contributor to several diseases, including cardiovascular disease. High blood pressure (BP) is a common condition associated with heart, brain, and kidney diseases.  We have established that there is a strong genetic component contributing to OSA and blood pressure, independent of obesity. By combining data from our Cleveland Family Study with other studies, we hope to assemble the largest whole genome sequencing data of OSA and blood pressure performed to date. We will combine family-based genetic signals and cutting-edge tools developed by our group to improve the chances of finding genetic mutations contributing to OSA and blood pressure. The combination of datasets outside dbGaP, which will be analyzed both independently and together with dbGaP datasets in meta-analysis, will not create any additional risks to participants.\",\n\"The extent and nature of sleep is an important determinant for both physical and mental health. Sleep disorders and disruptions of normal sleep architecture and duration have adverse health outcomes, increasing propensity to hypertension, cardiovascular disease, diabetes, stroke, depression and other conditions. The most common sleep disorders are Obstructive Sleep Apnea (OSA) and insomnia. OSA is characterized by recurrent episodes of pharyngeal obstruction during sleep, exposing the affected individual to severe nightly cardiovascular stress. Insomnia characterized by difficulties falling or staying asleep, and is associated with perturbations in physiological systems. Disturbed sleep architecture, from OSA, insomnia or other etiologies, may similarly contribute to health problems. OSA, insomnia and physiological disturbances in sleep may thus contribute to a wide range of health effects, supporting the need to identify the genetic bases for these interrelated disorders and traits. We aim to identify genes/variants underlying OSA traits.  High blood pressure (BP) is a common condition associated with heart, brain, and kidney diseases. The BP heritability estimated from family data ranges from 30 to 60%.  Recent genome-wide association studies (GWAS) have identified over 100 common BP variants that together explain ~3.5% of BP variation. Thus, the heritability of BP remains missing and most of the genetic variants remain to be discovered. It is expected that low frequency and rare variants may be able to explain a portion of the BP \\u00e2\\u0080\\u009cmissing heritability\\u00e2\\u0080\\u009d. Because of lower sensitivity to genetic heterogeneity, linkage analysis of family data can be used to identify low frequency and rare variants. Thus, we aim to identify novel low-frequency and rare variants associated with BP traits using TOPMed whole genome sequencing data.  The research will be published in peer-reviewed journals.\",\n\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\"],\n [\"Kochi, Yuta \",\n\"TOKYO MEDICAL AND DENTAL UNIVERSITY\",\n\"Analysis of genetic factors of severe COVID-19 through eQTL and sQTL of immune cells\",\n\"Dec04, 2020\",\n\"approved\",\n\"In order to clarify the genetic factors for the severe manifestation of the COVID-19, several genetic studies including the one led by COVID Human Genetic Effort have been performed. Although these studies identified several genetic factors, the whole picture of genetic determinants for the severe disease is not fully drawn. Because these genetic studies had limitations for statistical power, genetic factors with moderate effects remained unidentified. Another challenge in the genetics of COVID-19 is to dissect the causal mechanism. Most of the genetic variants for other immune-related diseases are now considered to affect gene expression or alternative splicing. Therefore, it is essential to integrate genetic data with expression data to dissect the causal mechanism of disease. We performed eQTL/sQTL studies that examined the relation between genetic variants and expression/splicing using immune cell subsets. Through our experiences in the research field, we aim to elucidate the disease mechanism of severe COVID-19 by integrating genetic data and eQTL/sQTL data. We also aim to establish a prediction model for severe disease manifestation using individual\\u00e2\\u0080\\u0099s genome data by evaluating the accumulation of disease-associated eQTL/sQTL. \",\n\"In order to dissect the genetic factors for the severe manifestation of the COVID-19, several genome-wide association studies (GWAS) including the one led by COVID Human Genetic Effort have been performed. Although these GWAS identified several risk loci such as a genomic segment on chromosome 3 inherited from Neanderthal, the whole picture of genetic determinants for the severe disease is not fully drawn. Because these GWAS have set a strict threshold for the genome-wide significance (i.e. P=5x10E-8), majority of risk loci with moderate effects did not reach this threshold and remained unidentified. Another challenge in the genetics of COVID-19 is to dissect the causal mechanism in the GWAS loci. Most of GWAS loci identified in other immune-related diseases, such as autoimmune diseases and allergic diseases, are now considered to be expression quantitative trait locus (eQTL) or splicing QTL (sQTL), where genetic variants regulate gene expression or alternative splicing. In fact, we performed GWAS for clinically amyopathic dermatomyositis, which manifests severe lung disease similar to COVID-19, and we identified that a SNP having an sQTL effect for the WDFY4 gene caused the disease (Kochi Y et al, Ann Rheum Dis 2018). Therefore, it is essential to integrate GWAS data with eQTL/sQTL data to dissect the causal mechanism of disease. To this end, several eQTL/sQTL studies such as GTEx, DICE, and ImmVar project have been performed. We also performed eQTL/sQTL studies using immune cell subsets (Ishigaki K and Kochi Y et al, Nat Genet 2017). Moreover, we established a method to evaluate polygenic burden of these eQTLs in individuals and evaluated the signal pathways that are activated in the disease (Nat Genet 2017). Through our experiences in the research field, we aim to elucidate the disease mechanism of severe COVID-19 by integrating GWAS data and eQTL/sQTL. We also aim to establish a prediction model for severe disease manifestation using individual\\u00e2\\u0080\\u0099s genome data by evaluating the polygenic burden of disease associated eQTL/sQTL with moderate effects.    We combine the present dataset (whole genome sequencing data and GWAS data of COVID-19) with the summary statistics obtained from the eQTL/sQTL data (GTEx, DICE, ImmVar and our own) that are publicly available. Our analysis will not create any additional risks to participants. \",\n\"phs002245.v1.p1\",\n\"Genetic Determinants of Susceptibility to Severe COVID-19 Infection\"],\n [\"Picardi, Ernesto \",\n\"INSTITUTE OF BIOMEMBRANES/BIOENERGETICS\",\n\"The role of Y chromosome anomalies in Covid-19 severity and outcome\",\n\"Jul30, 2021\",\n\"rejected\",\n\"The wide range of clinical outcomes following SARS-Cov-2 infection, from asymptomatic up to severe and/or long lasting infections and death, is well known as well as its correlation to the host genetic profile (PMID: 34237774).  Age-related mosaic loss of chromosome Y (MLY) has been associated with a wide spectrum of human diseases including cancer, Alzheimer's disease, and cardiovascular disease as well as to increased mortality rate (PMID:32020362). We plan to extensively investigate the possible correlation of MLY with the clinical outcome of Covid-19 by analysing the genomic profile of a large cohort of infected subjects and controls stratified according to clinical status, age, and other relevant features.\",\n\"The wide range of clinical outcomes following SARS-Cov-2 infection, from asymptomatic up to severe and/or long lasting infections and death, is well known as well as its correlation to the host genetic profile (PMID: 34237774).  Age-related mosaic loss of chromosome Y (MLY) has been associated with a wide spectrum of human diseases including cancer, Alzheimer's disease, and cardiovascular disease as well as to increased mortality rate (PMID:32020362). We plan to extensively investigate the possible correlation of MLY with the clinical outcome of Covid-19 by analysing the genomic profile of a large cohort of infected subjects and controls stratified according to clinical status, age, and other relevant features.\",\n\"phs002245.v1.p1\",\n\"Genetic Determinants of Susceptibility to Severe COVID-19 Infection\"],\n [\"Polyakov, Yuriy \",\n\"DUALITY TECHNOLOGIES, INC.\",\n\"Secure Genome Wide Association Studies using Homomorphic Encryption\",\n\"Dec16, 2020\",\n\"approved\",\n\"Our goal is to develop the methodology for secure genome wide association studies where all computations are performed on encrypted data. In other words, the genomic data would never be decrypted during computations. We are planning to use homomorphic encryption as the cryptographic tool for secure computations.\",\n\"The goal of our research is to develop a solution (methodology) for efficient secure genome wide association studies using the techniques based on homomorphic encryption. We submitted two prototypes to the IDASH competition track 2 (http://www.humangenomeprivacy.org/2018/competition-tasks.html) and now want to further develop these prototypes to support larger (practical) scales. If we are able to efficiently perform all computations for real datasets in the encrypted domain, we will submit a paper to Nature Biotech. As part of the same study, we are planning to develop a solution for genetic risk prediction. In summary, our goal is to support secure GWAS analysis of encrypted data.  Our research is funded by NIH SBIR (Phase I).  We are planning to analyze these datasets independently.  Update from 11/18/2020.   We published two papers based on the analysis of these data:  Blatt, M., Gusev, A., Polyakov, Y., Goldwasser, S., \\\"Secure Large-Scale Genome-Wide Association Studies using Homomorphic Encryption\\\", Proceedings of the National Academy of Sciences (PNAS), 2020, Vol. 117, No. 21 [https://eprint.iacr.org/2020/563]. Blatt, M., Gusev, A., Polyakov, Y., Rohloff, K., and Vaikuntanathan, V., \\\"Optimized Homomorphic Encryption Solution for Secure Genome-Wide Association Studies\\\", BMC Medical Genomics, 2020, Vol. 13, Suppl. 7, No. 83 [https://eprint.iacr.org/2019/223].   We now also have DARPA funding to extend our solution for privacy-preserving GWAS to a multi-party setting.  Update from 12/9/2020  We plan to apply the same privacy-preserving GWAS approach to the COVID dataset. The methodology will be the same as what we have used for the AMD dataset.\",\n\"phs002245.v1.p1\",\n\"Genetic Determinants of Susceptibility to Severe COVID-19 Infection\"],\n [\"Worthey, Elizabeth \",\n\"UNIVERSITY OF ALABAMA AT BIRMINGHAM\",\n\"Understanding the molecular underpinnings of ME/CFS - Long Covid\",\n\"Apr14, 2023\",\n\"approved\",\n\"Myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS) is a disabling and complex illness. There is significant overlap between ME/CFS and Long Covid and many researchers and clinicians with expertise in these diseases believe they are one and the same. ME/CFS/Long Covid ranges from mild to severe, and can cause more symptoms in some people than in others. It can affect many organs and systems and in addition to chronic fatigue is associated with tender lymph nodes in the neck or armpits, frequent sore throats, digestive issues, like irritable bowel syndrome, chills and night sweats, allergies and sensitivities to foods, odors, chemicals, light, or noise, muscle weakness, shortness of breath, and irregular heartbeat. Although ME/CFS/Long Covid is a relatively common disorder there are few approved and available treatments. We will combine dbGAP ME/CFS and related datasets (e.g. long covid and ehlers danlos) with internally generated data to better understand the mechanisms underlying development and progression of disease. We will also use these datasets to predict drugs that may can address the impact of variation in ME/CSF/Long Covid with the goal of identifying therapeutic candidates for testing in in vitro and in vivo ME/CFS/Long Covid-relevant systems.\",\n\"Myalgic encephalomyelitis/chronic fatigue syndrome (ME/CFS) is a disabling and complex illness. ME/CFS ranges from mild to severe, and can cause more symptoms in some people than in others. There is notable overlap is symptoms and disease course/outcomes between these diseases. Symptoms can affect many organs and systems and in addition to chronic fatigue is associated with tender lymph nodes in the neck or armpits, frequent sore throats, digestive issues, like irritable bowel syndrome, chills and night sweats, allergies and sensitivities to foods, odors, chemicals, light, or noise, muscle weakness, shortness of breath, and irregular heartbeat. Although ME/CFS is a relatively common disorder there are few approved and available treatments. Recently the dramatic increase in numbers of patients seen with this disorder due to covid infection has provided an impetus to data sharing that seems likely to finally support a better understanding of disease pathology through comprehensive clinical data/omic/other dataset based studies into causation/outcomes/therapeutics. We will combine dbGAP ME/CFS and related datasets (e.g. long covid and ehlers danlos) with internally generated RNA-Seq, WES, and WGS datasets and apply computational biology and machine/deep learning approaches to better understand the mechanisms underlying development and progression of disease. We have a well phenotyped ME/CFS cohort at UAB that also includes a variety of metabolic, immune, and imaging datasets. We will use this data to develop a model that can classify patients based on attributes, which will be used to look for differing underlying molecular causes of disease or risk factors in the different groups. We will also use these datasets to perform signal inversion in expression datasets to predict drugs with inverted expression profiles that may can address the impact of variation in ME/CSF with the goal of identifying therapeutic candidates for testing in in vitro and in vivo NF1-relevant systems.  It is our goal to apply computational biology and AI based approaches to existing genomic and transcriptomic datasets in combination with in-house derived data to benefit patients by defining molecular signatures of ME/CFS. No individual participants will be identifiable from the analysis that we perform. Our project will comply with all data use limitations for dbGaP studies. The dbGaP datasets will be used in accordance with the user specifications for the de-identified data. We will not seek or pursue findings that are beyond the respective scope of each data set. The data will be used for research purposes only. Any results of the study will be disseminated to the scientific community through publications and presentations, with the requisite acknowledgment of the NIH-designated data repositories and the specific dataset accession numbers.\",\n\"phs002245.v1.p1\",\n\"Genetic Determinants of Susceptibility to Severe COVID-19 Infection\"],\n [\"Xu, Lin \",\n\"UT SOUTHWESTERN MEDICAL CENTER\",\n\"Leveraging new deep learning algorithms to predict functional noncoding variants\",\n\"Dec04, 2020\",\n\"approved\",\n\"Artificial intelligence (AI) is revolutionizing and reshaping the health-care systems by enabling new algorithms and computer-based decision-support systems to increase diagnostic accuracy, design novel therapeutic strategies with better effectiveness, improve clinical workflow, decrease human resource costs and optimize treatment choices. My goal is to integrate cutting-edge artificial intelligence technology and preclinical drug tests as an experimental validation step, in order to establish a new framework for developing \\u00e2\\u0080\\u009cgame-changing\\u00e2\\u0080\\u009d strategy to identify previously known functional noncoding variants in various human diseases, in order to identify new biomarkers for disease diagnosis and explore new mechanisms of disease development through mutations in noncoding regions.\",\n\"In this study we will develop algorithms to predict gene and variant functions in a variety of human diseases. With the access to the dbGaP sequencing datasets, we will leverage our algorithms to identify potential functional variants. My lab will release our results to the public after publications are accepted. Our scripts used for this study will be also be released on GitHub. My lab will not combine requested datasets with other datasets outside of dbGaP. Data downloaded from this resource will be stored safely and avoid to be accessed by others without permission of PI. Use of the data will be limited to not-for-profit organizations. phs000259, phs000735, phs000970, phs000260 are related to the human microbiome. Sequence data generated from microbiome samples will be used only for the purpose of studying the microbial components of the microbiome and will not be used to make inferences about the human host DNA itself. phs002245 is for COVID-19 infection. We will follow these restriction for data usage and storage.\",\n\"phs002245.v1.p1\",\n\"Genetic Determinants of Susceptibility to Severe COVID-19 Infection\"],\n [\"JANES, KEVIN \",\n\"UNIVERSITY OF VIRGINIA\",\n\"Comparing SARS-CoV-2 Heart Infection to Other Viral Heart Infections\",\n\"Jan06, 2023\",\n\"approved\",\n\"Like other organs, the human heart can be infected by viruses. Previously, we identified different ways that human hearts can react to the presence of virus. However, it is not known whether these reactions to virus are harmful or helpful to the heart. In this study, we are aiming to compare each newly identified heart reaction to a known heart reaction. SARS-CoV-2 is well recognized for its ability to infect the human heart and cause damage. Therefore, we intend to compare the heart reactions that we identified to how the heart responds to SARS-CoV-2 infection. We anticipate that this project will advance our understanding of viral heart infections and how the body responds to them. We will publish the results of this study to make them available to the public.\",\n\"Several viruses are known to infect the human heart. These infections can either be short-term and acute in nature or persist in a long-term chronic state. Chronic viral heart infection is associated with progression to heart failure. Using other publicly available RNA-seq datasets, we have identified undiagnosed infections in patient hearts. We then found that patient hearts harboring viral RNA showed signs of adapting to the viral infection in distinct ways. However, it is unknown whether these heart adaptations are harmful or helpful to the human heart. Addressing this question requires a known heart adaptation. SARS-CoV-2 is a well-known heart pathogen that causes myocarditis. The objective of this project is to use RNA-seq profiles from patient hearts with severe SARS-CoV-2 infection to build a reference frame to compare with our identified heart adaptations to other viral infections. Differentially expressed gene lists are available comparing COVID+ hearts to control hearts. However, to perform a rigorous comparison between SARS-CoV-2 infected hearts and our previously identified heart adaptations, we must process the data in exactly the same way. Therefore, we are requesting the raw sequencing reads. All data will be analyzed and stored through the Information Technology Services at the University of Virginia. Upon completion of this study, we will disseminate the results by posting a pre-print of the work, followed by formal publication.\",\n\"phs002258.v1.p1\",\n\"Shotgun Transcriptome and Isothermal Profiling of SARS-CoV-2 Infection    Reveals Unique Host Responses, Viral Diversification, and Drug    Interactions\"],\n [\"Levanon, Erez \",\n\"BAR-ILAN UNIVERSITY\",\n\"A-to-I RNA Editing Levels in COVID-19 Infected Individuals\",\n\"Mar03, 2022\",\n\"approved\",\n\"Coronavirus disease 2019 (COVID-19) is a contagious disease caused by an infection with SARS-CoV-2. RNA editing is a mechanism that enables RNA modifications in the cell, mainly carried by the ADAR1 enzyme in humans. ADAR1 is also tightly linked to the immune system activation against viruses.  Our group investigates RNA editing levels in infected COVID-19 individuals. So far we discovered increased RNA editing levels in blood cells of COVID-19 patients compared to controls, and we would like to further investigate these findings in infected cells taken directly from the respiratory system. Our results will be published in public.\",\n\"Coronavirus disease 2019 (COVID-19) is a contagious disease caused by an infection with SARS-CoV-2. All viruses require host cellular machinery for their RNA and protein synthesis. The presence of viral RNA molecules inside host cells can activate different host immune components.   RNA editing is a mechanism that enables RNA modifications in the transcriptome. In humans, Adenosine-to-Inosine (A-to-I) is considered to be the most abundant RNA editing process, carried by the Adenosine Deaminase Acting on RNA (ADAR) family of enzymes, particularly by ADAR1.Furthermore, it is known that ADAR1 is tightly linked to Interferon and the immune system activity against viruses.    So far, our group found increased RNA editing levels by ADAR1 in blood cells of COVID-19 patients compared to controls. In other words, we found that SARS-CoV-2 has a substantial impact on host RNA. We assume that such an effect may be even greater in infected cells of the respiratory system. We truly believe that our research will contribute to further understanding the pathogenesis of COVID-19, and guarantee to publish the results to the scientific community.\",\n\"phs002258.v1.p1\",\n\"Shotgun Transcriptome and Isothermal Profiling of SARS-CoV-2 Infection    Reveals Unique Host Responses, Viral Diversification, and Drug    Interactions\"],\n [\"Avillach, Paul \",\n\"HARVARD MEDICAL SCHOOL\",\n\"NHLBI BioData Catalyst (BDC) \\u00e2\\u0080\\u0093 Infrastructure and Methods Development\",\n\"Mar25, 2021\",\n\"approved\",\n\"The NHLBI BioData Catalyst is a cloud-based platform, or technical framework, for tools, applications, and workflows. The NHLBI BioData Catalyst is being developed to provide secure workspaces for researchers to share, store, cross-link, and compute large sets of data generated from biomedical research. In this project, not only will approved investigators use controlled-access datasets from GTEx, NHLBI, and NHLBI TOPMed studies be utilized but an investigator will also be able to upload their own data into the system and will perform the following activities: Developing and testing cloud-based platforms and user interfaces to store, manage and interact with biomedical data and analysis tools; Establishing community-endorsed guidelines for Findable, Accessible, Interoperable and Reusable (FAIR) digital objects (data, software tools, etc.); Developing the ability to access controlled-access data through novel user authorization and authentication approaches; Developing analysis workflows that are portable across multiple cloud service providers; Agile software development; and Analyzing and evaluating the products and processes of the NHLBI BioData Catalyst for cost, utility, efficiency, usability, and adherence to FAIR guidelines. \",\n\"This Project is submitted by investigators involved in the NHLBI-supported BioData Catalyst  (https://biodatacatalyst.nhlbi.nih.gov/), a cloud-based infrastructure where heart, lung, blood, and sleep (HLBS) researchers can go to find, search, access, share, cross-link, and compute on large scale datasets. The NHLBI BioData Catalyst will provide tools, applications, and workflows to enable those capabilities in secure workspaces. In this project, not only controlled-access datasets from GTEx, NHLBI, and NHLBI TOPMed studies  will be utilized but an investigator will also be able to upload their own data into the system, and the following activities will be performed:  \\u00e2\\u0080\\u00a2 Developing and testing cloud-based platforms and user interfaces to store, manage and interact with biomedical data and analysis tools;  \\u00e2\\u0080\\u00a2 Establishing community-endorsed guidelines for Findable, Accessible, Interoperable and Reusable (FAIR) digital objects (data, software tools, publications, etc.);  \\u00e2\\u0080\\u00a2 Developing the ability to access controlled-access data through novel user authorization and authentication approaches;  \\u00e2\\u0080\\u00a2 Developing analysis workflows that are portable across multiple cloud service providers;  \\u00e2\\u0080\\u00a2 Agile software development; and  \\u00e2\\u0080\\u00a2 Analyzing and evaluating the products and processes of the NHLBI BioData Catalyst for cost, utility, efficiency, usability, and adherence to FAIR guidelines.   This list is not comprehensive and is subject to changes as the development of this cloud-based data ecosystem for storage and computing on biomedical digital objects evolves over the development and production phases of deployment. If there are significant modifications or changes to the scope of the project, this Research Use Statement must be amended and resubmitted for NHLBI Data Access Committee (DAC) review.   The investigators understand that any of the requested controlled-access datasets have Data Use Limitations (DULs) based on the terms of the informed consent of the study participants and/or stipulated by the submitting institution.  All approved investigators will abide by the DULs of each selected dataset, the Data Use Certification Agreement, NIH Genomic Data Sharing Policy (https://grants.nih.gov/grants/guide/notice-files/NOT-OD-14-124.html), Genomic Data User Code of Conduct (https://osp.od.nih.gov/wp-content/uploads/Genomic_Data_User_Code_of_Conduct.pdf), the NIH Security Best Practices for Controlled-Access Data Subject to the NIH Genomic Data Sharing (GDS) Policy (https://osp.od.nih.gov/wp-content/uploads/NIH_Best_Practices_for_Controlled-Access_Data_Subject_to_the_NIH_GDS_Policy.pdf), and the NIH Notice for Use of Cloud Computing Services for Storage and Analysis of Controlled-Access Data Subject to the NIH Genomic Data Sharing Policy (https://grants.nih.gov/grants/guide/notice-files/NOT-OD-15-086.html).   Under no circumstances will investigators listed in this Project seek to perform hypothesis-driven biomedical research, or develop methods to identify statistical patterns and assist in their own interpretation of the requested datasets or of any derived data for research purposes. Should the need arise to use the requested data for scientific, use-case driven biomedical research purposes, investigators will submit a separate dbGaP project application. At no time shall the Institution, Investigator, or Approved Users conduct Data Transfer or Data Access of the data and any derived data out of the approved cloud environments; nor shall the Institution, PI, or Approved Users attempt to recreate or replicate the data and any derived data for use outside of the approved cloud environments; this includes, but is not limited to, capturing screenshots or videos of data or derived data, or locally transferring software results or runs that may enable recapitulation of the data or derived data. At no time shall the Institution, PI, or Approved Users use the data or derived data for any other purpose outside of or beyond the scope of the work they are doing for the NHLBI BioData Catalyst. No ownership rights of the data (including derived or derivative data) are granted to the Institution, Investigator, Approved Users, or the cloud service providers in the development of the NHLBI BioData Catalyst. \",\n\"phs002299.v1.p1\",\n\"PETAL Network: Outcomes Related to COVID-19 Treated With    Hydroxychloroquine Among Inpatients With Symptomatic Disease (ORCHID)    Trial\"],\n [\"Avillach, Paul \",\n\"HARVARD MEDICAL SCHOOL\",\n\"Exploration of the ORCHID randomized controlled trial (RCT) results\",\n\"Jan04, 2021\",\n\"closed\",\n\" In a preliminary step, we will use data from the  ORCHID trial to reproduce the analyses performed in the original paper by Self, WH to get  baseline results enabling further comparisons. Additionally we will analyze the primary and secondary outcomes using other treatments as treatment groups (ie information available regarding premedication and in-hospital use of corticosteroids, tocilizumab, sarilumab, interferon, immunomodulators, azithromycin).  We will also compare the lab trajectories between the different treatment groups (labs information available up to the 5th day), and correlation between lab trajectories and the main outcome. We will study the secondary outcomes and safety events with a modified Intention-To-Treat approach (mITT), that is constituting the treatment groups looking at the treatment that has actually been received by the individuals as opposed to the treatment originally planned to be given (i.e. differences arising from individuals that dropped from the study or that did not received the complete treatment regimen as planned in the initial protocol). \",\n\"We will use data from the PETAL Network: Outcomes Related to COVID-19 Treated With Hydroxychloroquine Among Inpatients With Symptomatic Disease (ORCHID) Trial to reproduce and perform new analyses. In a preliminary step, we will reproduce the analyses performed in the original paper by Self, WH,et al.,  Effect of Hydroxychloroquine on Clinical Status at 14 Days in Hospitalized Patients With COVID-19, to get baseline results enabling further comparisons. This includes every analysis included in the Clinical Trial statistical analysis protocol. Additionally, we will perform several new analyses based on the data collected for this trial, as ancillary studies. 1) We will use the information regarding the concomitantly administered treatments received by the patients during the trial to look for an association between the administration of these treatments and any of the outcomes (i.e. using information available regarding premedication and in-hospital use of corticosteroids, tocilizumab, sarilumab, interferon, immunomodulators, azithromycin). Since these treatments were prescribed based on the actual health status of the patients, as opposed to the random allocation of hydroxychloroquine/placebo, the analyses will need to be adjusted on clinical and biological characteristics of these patients to take into account confounding factors. This adjustment could be made using multivariate models or propensity score matching, which allows to study the average treatment effect. Moreover, we will look for interaction between the concomitant drugs and the treatment groups, regarding both the clinical outcomes and the safety events. 2) We will compare the lab trajectories between the different treatment groups (labs information available up to the 5th day), and the correlation of these trajectories with the clinical outcomes. We will be able to compare these trajectories with external work in the literature studying the lab trajectories for COVID-19 patients, namely one performed by the 4CE consortium. 3) We will study the secondary outcomes and safety events with a modified Intention-To-Treat approach (mITT), that is constituting the treatment groups looking at the treatment that has actually been received by the individuals as opposed to the treatment originally planned to be given (i.e. differences arising from individuals that dropped from the study or that did not received the complete treatment regimen as planned in the initial protocol). The original statistical analysis plan includes an mITT for the primary outcome only. The mITT is especially suited for the safety outcomes analyses. Studying the subpopulation of patients which were excluded of the study after randomization, for worsening of their condition. We plan to look for specific characteristics of this population regarding the outcomes, the safety events and the concomitant treatment used.\",\n\"phs002299.v1.p1\",\n\"PETAL Network: Outcomes Related to COVID-19 Treated With    Hydroxychloroquine Among Inpatients With Symptomatic Disease (ORCHID)    Trial\"],\n [\"Chan, Kei Hang \",\n\"BROWN UNIVERSITY\",\n\"NHLBI TOPMed Whole-genome Sequencing Program (TOPMed WGS): Using molecular and genetic epidemiological approach to dissect the mechanism of complex diseases\",\n\"Dec15, 2022\",\n\"approved\",\n\"We use epidemiological, statistical, and bioinformatics to dissect the molecular mechanism of complex diseases using genetic, clinical and medication information from several well-established cohort studies. We will analyze the data each TOPMed cohort, e.g. WHI, FHS, JHS and CHS separately.\",\n\"Many complex traits such as diabetes and cardiovascular diseases encompass the interplay between genes and the environment. We hypothesize that the so-called \\u00e2\\u0080\\u009cmissing heritability\\u00e2\\u0080\\u009d is explained by the following two main reasons: First, many subtle genetic variations that are well under the genome-wide significance level coordinate with polymorphisms of strong genomic effects to impact diverse biological processes. Secondly, gene-environment (GxE) interactions, clearly demonstrated by the observation that genetically identical individuals often differ extensively in phenotypes, play a fundamentally important role in both obesity and T2D development. We will conduct an integrative genomics, modification assessment, and mendelian randomization studies that leverage genetic, genomic, clinical, medication, and phenotypic data from several large-scale multi-ethnic cohorts in the TOPMed Consortium to dissect the mechanisms of various complex diseases.  Research Objectives: With the requested data from the TOPMed Consortium, we aim to (including but not limited to the following): 1) evaluate the modification of medication use on biomarker-related genetic variants with risk of complex traits such as diabetes and cardiovascular diseases  2) assess the causal association between biomarkers and risk of complex traits such as diabetes and cardiovascular diseases using mendelian randomization approach 3) identify shared and ethnic-specific biological pathways and gene networks that are perturbed by genetic risks of complex traits between ethnic groups  4) identify shared and ethnic-specific gene network x environmental risk factors interactions that contribute to risk of complex traits between ethnic groups 5) identify shared and ethnic-specific biological pathways and gene networks that are perturbed by genetic effect of medication use between ethnic groups. 6) assess the association between potential biomarkers targeted by medication use and risk of complex traits using mendelian randomization approach with medication related genetic loci as instruments. 7) build disease prediction model using demographic data, lifestyle risk factors, biomarkers, and omics data.  Publication Statement: We will ensure that the research conducted using the datasets will only be published according to the regulations specified in the Data Use Certification (DUC).   Phenotypic and genomic data from TOPMed studies will be used for: (a) investigating genotype-phenotype associations; (b) variant summary statistics, (c) common controls in association tests, (d) reference panels to impute into un-sequenced individuals from TOPMed; and (e) population genetic studies.   TOPMed data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Data from an individual with disease-specific consent will not be used in analyses outside of that restriction, including indirect uses such as imputation reference panels, common controls for association studies and variant summary statistics. For example, an individual with consent restricted to lung function studies will not be used to impute data intended for analyses of diseases not clearly related to lung function. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish or otherwise broadly share any findings from this study with the scientific community.   TOPMed Generic Research Use Statement:  This application anticipates collaboration among TOPMed investigators from multiple institutions, who are named in a master list appended to the IRB document uploaded with this application. Within a set of applicants approved for access to the TOPMed Exchange Areas, data may be shared in a cloud environment or by direct transfer, provided that they all agree among themselves to allow such sharing, and that all have approved TOPMed Data Access Requests for the specific data sets to be shared. Otherwise, data will be shared only among Internal Collaborators \\u00e2\\u0080\\u0093 i.e. within an institution.   Data from TOPMed may be combined with non-TOPMed data from the same or other studies (obtained from dbGaP or other sources), to improve the power for novel genetic discoveries, while respecting the consent of all participants. We expect that this activity creates no additional risks to participants.   \",\n\"phs002299.v1.p1\",\n\"PETAL Network: Outcomes Related to COVID-19 Treated With    Hydroxychloroquine Among Inpatients With Symptomatic Disease (ORCHID)    Trial\"],\n [\"Manning, Alisa \",\n\"BROAD INSTITUTE, INC.\",\n\"Exploration of the ORCHID randomized controlled trial (RCT) results\",\n\"Feb10, 2021\",\n\"closed\",\n\"We will use data from the ORCHID trial to reproduce the analyses performed in the original paper by Self, WH. Based on our process of using a Jupyter notebook created by BioData Catalyst collaborators we decide which additional analyses we will perform and we will revise our dbGap application.\",\n\"We will use data from the PETAL Network: Outcomes Related to COVID-19 Treated With Hydroxychloroquine Among Inpatients With Symptomatic Disease (ORCHID) Trial to reproduce and perform new analyses. In a preliminary step, we will reproduce the analyses performed in the original paper by Self, WH,et al., Effect of Hydroxychloroquine on Clinical Status at 14 Days in Hospitalized Patients With COVID-19, to get baseline results enabling further comparisons. This includes every analysis included in the Clinical Trial statistical analysis protocol. The original statistical analysis plan includes an mITT for the primary outcome only. We will leverage a Jupyter notebook created by BioData Catalyst collaborators to replicate the analysis.\",\n\"phs002299.v1.p1\",\n\"PETAL Network: Outcomes Related to COVID-19 Treated With    Hydroxychloroquine Among Inpatients With Symptomatic Disease (ORCHID)    Trial\"],\n [\"Ahuja, Sunil \",\n\"UNIVERSITY OF TEXAS HLTH SCIENCE CENTER\",\n\"Markers of immunologic resilience\",\n\"Sep16, 2022\",\n\"approved\",\n\"People vary in their susceptibility to develop diseases with features of autoinflammation (e.g., cardiovascular disease, Alzheimer\\u00e2\\u0080\\u0099s disease) versus autoimmunity (e.g., multiple sclerosis, lupus). The reasons for highly-variable disease susceptibility and progression are unknown and currently being explored. One possibility is that disease susceptibility relates to interpersonal differences (e.g., genotypes, epigenetics) in risk of a lowering in immunologic resilience in response to common environmental antigenic challenges accrued with age. Potentially, persons that fail to preserve immunologic resilience in response to chronic low-grade or spontaneous high-grade microbial exposures are at higher risk of developing diseases with features of autoinflammation, whereas the opposite is associated with a heightened risk of developing autoimmune diseases as a trade-off. In the proposed study, we will develop methods to assess immunologic resilience by use of genomic and transcriptomic profiling from the GTex, Framingham Heart Study, T2D-GENES, and Normal Twins & Major Depression cohorts.\",\n\"The primary objective of our studies is to develop quantitative measures of immunologic resilience, biomarkers of immunosenescence, and to identify genetic determinants of immunologic health. In ongoing studies, we have evaluated 20,000+ humans and results demonstrate that chronic antigenic stimuli have a negative impact on immunologic integrity. Using the laboratory variables of CD4+ and CD8+ T cell counts, we developed a method to grade immunologic resilience (immunologic health grades). This immune grading system highly correlated with adverse clinical outcomes such as poor vaccine responsiveness, COVID-19 severity, chronic disease progression, risk of cancer development and recurrence, and all-cause mortality.  Next, by using high-dimensional flow cytometry approaches, we identified immunologic traits (e.g., CD28-CD8dim senescent T cells, CD25++CD8bright proliferating T-cells, CD8+NKT-like cells) that associate with immunologic health grades, chronological age, and the interaction of the two. The identification and studies of these traits have contributed to improvement of our routine FACS analyses. In recent studies, we have identified cell subsets associated with IHG (e.g., CD8+ TEMRA, CD8+CD38+), COVID-19 hospitalization (CD4+ Treg suppressive, CD4+ EM), COVID-19 mortality (CD8+HLA-DR+, CD8+ EM anergic), and COVID-19 sVNT %inhibition (IgD- B cells). Additionally, key cytokines (e.g., ITAC/CXCL11, TNFa) were found to be associated with IHG and COVID-19 disease progression. Furthermore, by integrating RNA-seq transcriptomes and these immunologic traits, we developed transcriptomic signatures that highly associate with the immunologic health grades, T cell responsiveness & dysfunction, plasma IL-6 levels, COVID-19 hospitalization, and all-cause mortality. We used these genomic indexes to interrogate 24,000+ NCBI GEO and Array Express datasets and determined associations with different diseases (e.g., HIV, CMV, COVID-19, tuberculosis, lupus, respiratory diseases, aging) and the levels of these signatures in various cell types/tissues. Recently in our COVID-19 study, newly derived transcriptomic signatures associated with COVID-19 disease progression were found to be associated with the immune health grades in several diseases (e.g., COVID-19, lupus, therapy-naive and treated HIV), influenza severity independent of age, 30- and 120-day mortality independent of age and sex in hospitalized COVID-19 patients, and all-cause mortality independent of age and sex in the Framingham Heart Study.  Genetic polymorphisms associated with COVID-19 severity were assessed in our COVID-19 mechanistic cohort (n=1,826), with a prominent association of 3p21.31 rs17713054G>A proximal to the LZTFL1 gene. This marker was associated with a 2-fold excess of COVID-19 all-cause mortality hazard with each 10-year increase in age, with a dosage effect in European-Americans associating with accelerated mortality rates. Gene expression signatures were identified in the upper airways and peripheral blood to proxy the effect of genotype on COVID-19 outcomes, with multiple signatures associating with viral load in independent cohorts. These biomarkers and signatures will be further evaluated in the Normal Twins & Major Depression study (phs000486.v1.p1) and the T2D-GENES study (phs000847.v1.p1) upon renewal.  In addition, we have developed signatures of immune-checkpoint and immune-checkpoint blockade, which we would like to evaluate/validate in different cancer models and also assess the effectiveness of current treatment protocols.  With this updated background, we request access to these dbGaP datasets with exceptionally thorough phenotyping to identify the genetic determinants that associate with previously derived transcriptomic signatures in various cell types/tissues and previously studied genetic biomarkers. The study will test for associations between single nucleotide polymorphisms and our transcriptomic signatures, & then examine the allele-specific effects of each SNP and the individual genes used within each index. We will not be combining the dbGaP datasets with outside datasets and our study does not create any additional risks to participants due to de-identified status of the data. Where available, we are requesting access to BAM files, genotype calls, OMNI SNP arrays, Affymetrix expression data, allele-specific expression, gene expression matrices, sample attributes, subject phenotypes, laboratory measurements, & disease outcomes.\",\n\"phs002300.v1.p1\",\n\"Successful Clinical Response in Pneumonia Therapy (SCRIPT)\"],\n [\"Chan, Kei Hang \",\n\"BROWN UNIVERSITY\",\n\"NHLBI TOPMed Whole-genome Sequencing Program (TOPMed WGS): Using molecular and genetic epidemiological approach to dissect the mechanism of complex diseases\",\n\"Dec15, 2022\",\n\"approved\",\n\"We use epidemiological, statistical, and bioinformatics to dissect the molecular mechanism of complex diseases using genetic, clinical and medication information from several well-established cohort studies. We will analyze the data each TOPMed cohort, e.g. WHI, FHS, JHS and CHS separately.\",\n\"Many complex traits such as diabetes and cardiovascular diseases encompass the interplay between genes and the environment. We hypothesize that the so-called \\u00e2\\u0080\\u009cmissing heritability\\u00e2\\u0080\\u009d is explained by the following two main reasons: First, many subtle genetic variations that are well under the genome-wide significance level coordinate with polymorphisms of strong genomic effects to impact diverse biological processes. Secondly, gene-environment (GxE) interactions, clearly demonstrated by the observation that genetically identical individuals often differ extensively in phenotypes, play a fundamentally important role in both obesity and T2D development. We will conduct an integrative genomics, modification assessment, and mendelian randomization studies that leverage genetic, genomic, clinical, medication, and phenotypic data from several large-scale multi-ethnic cohorts in the TOPMed Consortium to dissect the mechanisms of various complex diseases.  Research Objectives: With the requested data from the TOPMed Consortium, we aim to (including but not limited to the following): 1) evaluate the modification of medication use on biomarker-related genetic variants with risk of complex traits such as diabetes and cardiovascular diseases  2) assess the causal association between biomarkers and risk of complex traits such as diabetes and cardiovascular diseases using mendelian randomization approach 3) identify shared and ethnic-specific biological pathways and gene networks that are perturbed by genetic risks of complex traits between ethnic groups  4) identify shared and ethnic-specific gene network x environmental risk factors interactions that contribute to risk of complex traits between ethnic groups 5) identify shared and ethnic-specific biological pathways and gene networks that are perturbed by genetic effect of medication use between ethnic groups. 6) assess the association between potential biomarkers targeted by medication use and risk of complex traits using mendelian randomization approach with medication related genetic loci as instruments. 7) build disease prediction model using demographic data, lifestyle risk factors, biomarkers, and omics data.  Publication Statement: We will ensure that the research conducted using the datasets will only be published according to the regulations specified in the Data Use Certification (DUC).   Phenotypic and genomic data from TOPMed studies will be used for: (a) investigating genotype-phenotype associations; (b) variant summary statistics, (c) common controls in association tests, (d) reference panels to impute into un-sequenced individuals from TOPMed; and (e) population genetic studies.   TOPMed data will not be used to investigate individual identity. Consent type and other Data Use Limitations (DUL) for each study will be respected in all analyses. Data from an individual with disease-specific consent will not be used in analyses outside of that restriction, including indirect uses such as imputation reference panels, common controls for association studies and variant summary statistics. For example, an individual with consent restricted to lung function studies will not be used to impute data intended for analyses of diseases not clearly related to lung function. When an individual\\u00e2\\u0080\\u0099s DUL prohibits investigation of population genetics, population history or related issues, their data will be excluded from studies that address those issues. We intend to publish or otherwise broadly share any findings from this study with the scientific community.   TOPMed Generic Research Use Statement:  This application anticipates collaboration among TOPMed investigators from multiple institutions, who are named in a master list appended to the IRB document uploaded with this application. Within a set of applicants approved for access to the TOPMed Exchange Areas, data may be shared in a cloud environment or by direct transfer, provided that they all agree among themselves to allow such sharing, and that all have approved TOPMed Data Access Requests for the specific data sets to be shared. Otherwise, data will be shared only among Internal Collaborators \\u00e2\\u0080\\u0093 i.e. within an institution.   Data from TOPMed may be combined with non-TOPMed data from the same or other studies (obtained from dbGaP or other sources), to improve the power for novel genetic discoveries, while respecting the consent of all participants. We expect that this activity creates no additional risks to participants.   \",\n\"phs002363.v1.p1\",\n\"PETAL Repository of Electronic Data COVID-19 Observational Study (RED    CORAL)\"],\n [\"Chen, Weijie \",\n\"U.S. FOOD AND DRUG ADMINISTRATION\",\n\"Investigation of image-based biomarkers for radiogenomics of COVID-19\",\n\"Sep07, 2022\",\n\"approved\",\n\"Effective COVID-19 diagnosis and treatment rely on the information from multiple patient tests involving clinical, molecular, imaging, and genomic data. Radiomics, an expansion of computer-aided diagnosis, is a growing effort that involves these computerized image analyses in attempt to further relate quantitative image data to other \\u00e2\\u0080\\u009c-omic\\u00e2\\u0080\\u009d data such as clinical, pathologic, and genomic data. The integration of imaging data with genomic data allows for the study of associations between the imaging findings and the genomic measurements of the same disease site. Using the MIDC datasets of COVID-19, we will investigate relationships between computer-extracted quantitative radiomic COVID-19 features and various clinical, molecular, and genomics markers of prognosis and risk of recurrence, including gene expression profiles. We will assess the predictive ability of the quantitative COVID-19 CT/CXR features relative to multiple tasks: clinical stage, risk of recurrence, and genomics.\",\n\"Effective COVID-19 diagnosis and treatment rely on the information from multiple patient tests involving clinical, molecular, imaging, and genomic data. Radiomics, an expansion of computer-aided diagnosis, is a growing effort that involves these computerized image analyses in attempt to further relate quantitative image data to other \\u00e2\\u0080\\u009c-omic\\u00e2\\u0080\\u009d data such as clinical, pathologic, and genomic data [Zhu 2015]. The integration of imaging data (radiomics) with genomic data, referred to as \\u00e2\\u0080\\u009cimaging-genomics\\u00e2\\u0080\\u009d or \\u00e2\\u0080\\u009cradiogenomics\\u00e2\\u0080\\u009d, allows for the study of associations between the radiomic phenotypes and the genomic measurements of the same disease site. Radiomic phenotypes that are highly correlated with important clinical, molecular, or genomic biomarkers can potentially serve as diagnostic or prognostic tools for patient monitoring and assessing therapeutic response, and thus augment the utility of medical imaging as a non-invasive technology for COVID-19 care, like a \\u00e2\\u0080\\u009cvirtual digital biopsy\\u00e2\\u0080\\u009d. Using the MIDC datasets of COVID-19, we will investigate relationships between computer-extracted quantitative radiomic COVID-19 features and various clinical, molecular, and genomics markers of prognosis and risk of recurrence, including gene expression profiles. We will assess the predictive ability of the quantitative COVID-19 CT/CXR features relative to multiple tasks: clinical stage, risk of recurrence, and genomics. Aims: 1. To create a multi-omics dataset of cases, including radiomics of COVID-19 and non-COVID-19 patients from chest x-rays and/or CTs along with their molecular and genomic markers, which will be made openly available for discovery by computational biologists and machine learning experts. MIDRC has the chest x-ray image data corresponding to the EHR data in RED CORAL: PETAL Repository of Electronic Data COVID-19 Observational Study. Our first goal is to test the interoperability of MIDRC and the Biodata CATALYST. We will link the imaging data with the EHR data and investigate methods for use of data from both sources for machine learning development and assessment. 2. To determine if these quantitative radiomics (virtual biopsy) of COVID-19 CT/CXR can serve as a surrogate biomarker for assessing the molecular environment, enabling translation of virtual biopies. Push/Pull Relationships: The multi-institutional chest radiograph and CT datasets of the MIDC will enable the rapid investigation of radiogenomics for prognosis, prediction, and monitoring of response of the COVID patient. This project will deliver discovery findings for COVID-19 lungs to expedite the understanding and development of machine intelligence as well as a clinical tool. \",\n\"phs002363.v1.p1\",\n\"PETAL Repository of Electronic Data COVID-19 Observational Study (RED    CORAL)\"],\n [\"Gevaert, Olivier \",\n\"STANFORD UNIVERSITY\",\n\"Image-based biomarkers for radiogenomics of COVID19\",\n\"Oct04, 2022\",\n\"approved\",\n\"Effective COVID-19 diagnosis and treatment rely on the information from multiple patient tests involving clinical, molecular, imaging, and genomic data. Radiomics, an expansion of computer-aided diagnosis, is a growing effort that involves these computerized image analyses in attempt to further relate quantitative image data to other \\u00e2\\u0080\\u009c-omic\\u00e2\\u0080\\u009d data such as clinical, pathologic, and genomic data. The integration of imaging data with genomic data allows for the study of associations between the imaging findings and the genomic measurements of the same disease site. Using the MIDC datasets of COVID-19, we will investigate relationships between computer-extracted quantitative radiomic COVID-19 features and various clinical, molecular, and genomics markers of prognosis and risk of recurrence, including gene expression profiles. We will assess the predictive ability of the quantitative COVID-19 CT/CXR features relative to multiple tasks: clinical stage, risk of recurrence, and genomics.\",\n\"Effective COVID-19 diagnosis and treatment rely on the information from multiple patient tests involving clinical, molecular, imaging, and genomic data. Radiomics, an expansion of computer-aided diagnosis, is a growing effort that involves these computerized image analyses in attempt to further relate quantitative image data to other \\u00e2\\u0080\\u009c-omic\\u00e2\\u0080\\u009d data such as clinical, pathologic, and genomic data [Zhu 2015]. The integration of imaging data (radiomics) with genomic data, referred to as \\u00e2\\u0080\\u009cimaging-genomics\\u00e2\\u0080\\u009d or \\u00e2\\u0080\\u009cradiogenomics\\u00e2\\u0080\\u009d, allows for the study of associations between the radiomic phenotypes and the genomic measurements of the same disease site. Radiomic phenotypes that are highly correlated with important clinical, molecular, or genomic biomarkers can potentially serve as diagnostic or prognostic tools for patient monitoring and assessing therapeutic response, and thus augment the utility of medical imaging as a non-invasive technology for COVID-19 care, like a \\u00e2\\u0080\\u009cvirtual digital biopsy\\u00e2\\u0080\\u009d. Using the MIDC datasets of COVID-19, we will investigate relationships between computer-extracted quantitative radiomic COVID-19 features and various clinical, molecular, and genomics markers of prognosis and risk of recurrence, including gene expression profiles. We will assess the predictive ability of the quantitative COVID-19 CT/CXR features relative to multiple tasks: clinical stage, risk of recurrence, and genomics. Aims: 1. To create a multi-omics dataset of cases, including radiomics of COVID-19 and non-COVID-19 patients from chest x-rays and/or CTs along with their molecular and genomic markers, which will be made openly available for discovery by computational biologists and machine learning experts. MIDRC has the chest x-ray image data corresponding to the EHR data in RED CORAL: PETAL Repository of Electronic Data COVID-19 Observational Study. Our first goal is to test the interoperability of MIDRC and the Biodata CATALYST. We will link the imaging data with the EHR data and investigate methods for use of data from both sources for machine learning development and assessment. 2. To determine if these quantitative radiomics (virtual biopsy) of COVID-19 CT/CXR can serve as a surrogate biomarker for assessing the molecular environment, enabling translation of virtual biopies.  Push/Pull Relationships: The multi-institutional chest radiograph and CT datasets of the MIDC will enable the rapid investigation of radiogenomics for prognosis, prediction, and monitoring of response of the COVID patient. This project will deliver discovery findings for COVID-19 lungs to expedite the understanding and development of machine intelligence as well as a clinical tool.\",\n\"phs002363.v1.p1\",\n\"PETAL Repository of Electronic Data COVID-19 Observational Study (RED    CORAL)\"],\n [\"Ahuja, Sunil \",\n\"UNIVERSITY OF TEXAS HLTH SCIENCE CENTER\",\n\"Markers of immunologic resilience\",\n\"Sep16, 2022\",\n\"approved\",\n\"People vary in their susceptibility to develop diseases with features of autoinflammation (e.g., cardiovascular disease, Alzheimer\\u00e2\\u0080\\u0099s disease) versus autoimmunity (e.g., multiple sclerosis, lupus). The reasons for highly-variable disease susceptibility and progression are unknown and currently being explored. One possibility is that disease susceptibility relates to interpersonal differences (e.g., genotypes, epigenetics) in risk of a lowering in immunologic resilience in response to common environmental antigenic challenges accrued with age. Potentially, persons that fail to preserve immunologic resilience in response to chronic low-grade or spontaneous high-grade microbial exposures are at higher risk of developing diseases with features of autoinflammation, whereas the opposite is associated with a heightened risk of developing autoimmune diseases as a trade-off. In the proposed study, we will develop methods to assess immunologic resilience by use of genomic and transcriptomic profiling from the GTex, Framingham Heart Study, T2D-GENES, and Normal Twins & Major Depression cohorts.\",\n\"The primary objective of our studies is to develop quantitative measures of immunologic resilience, biomarkers of immunosenescence, and to identify genetic determinants of immunologic health. In ongoing studies, we have evaluated 20,000+ humans and results demonstrate that chronic antigenic stimuli have a negative impact on immunologic integrity. Using the laboratory variables of CD4+ and CD8+ T cell counts, we developed a method to grade immunologic resilience (immunologic health grades). This immune grading system highly correlated with adverse clinical outcomes such as poor vaccine responsiveness, COVID-19 severity, chronic disease progression, risk of cancer development and recurrence, and all-cause mortality.  Next, by using high-dimensional flow cytometry approaches, we identified immunologic traits (e.g., CD28-CD8dim senescent T cells, CD25++CD8bright proliferating T-cells, CD8+NKT-like cells) that associate with immunologic health grades, chronological age, and the interaction of the two. The identification and studies of these traits have contributed to improvement of our routine FACS analyses. In recent studies, we have identified cell subsets associated with IHG (e.g., CD8+ TEMRA, CD8+CD38+), COVID-19 hospitalization (CD4+ Treg suppressive, CD4+ EM), COVID-19 mortality (CD8+HLA-DR+, CD8+ EM anergic), and COVID-19 sVNT %inhibition (IgD- B cells). Additionally, key cytokines (e.g., ITAC/CXCL11, TNFa) were found to be associated with IHG and COVID-19 disease progression. Furthermore, by integrating RNA-seq transcriptomes and these immunologic traits, we developed transcriptomic signatures that highly associate with the immunologic health grades, T cell responsiveness & dysfunction, plasma IL-6 levels, COVID-19 hospitalization, and all-cause mortality. We used these genomic indexes to interrogate 24,000+ NCBI GEO and Array Express datasets and determined associations with different diseases (e.g., HIV, CMV, COVID-19, tuberculosis, lupus, respiratory diseases, aging) and the levels of these signatures in various cell types/tissues. Recently in our COVID-19 study, newly derived transcriptomic signatures associated with COVID-19 disease progression were found to be associated with the immune health grades in several diseases (e.g., COVID-19, lupus, therapy-naive and treated HIV), influenza severity independent of age, 30- and 120-day mortality independent of age and sex in hospitalized COVID-19 patients, and all-cause mortality independent of age and sex in the Framingham Heart Study.  Genetic polymorphisms associated with COVID-19 severity were assessed in our COVID-19 mechanistic cohort (n=1,826), with a prominent association of 3p21.31 rs17713054G>A proximal to the LZTFL1 gene. This marker was associated with a 2-fold excess of COVID-19 all-cause mortality hazard with each 10-year increase in age, with a dosage effect in European-Americans associating with accelerated mortality rates. Gene expression signatures were identified in the upper airways and peripheral blood to proxy the effect of genotype on COVID-19 outcomes, with multiple signatures associating with viral load in independent cohorts. These biomarkers and signatures will be further evaluated in the Normal Twins & Major Depression study (phs000486.v1.p1) and the T2D-GENES study (phs000847.v1.p1) upon renewal.  In addition, we have developed signatures of immune-checkpoint and immune-checkpoint blockade, which we would like to evaluate/validate in different cancer models and also assess the effectiveness of current treatment protocols.  With this updated background, we request access to these dbGaP datasets with exceptionally thorough phenotyping to identify the genetic determinants that associate with previously derived transcriptomic signatures in various cell types/tissues and previously studied genetic biomarkers. The study will test for associations between single nucleotide polymorphisms and our transcriptomic signatures, & then examine the allele-specific effects of each SNP and the individual genes used within each index. We will not be combining the dbGaP datasets with outside datasets and our study does not create any additional risks to participants due to de-identified status of the data. Where available, we are requesting access to BAM files, genotype calls, OMNI SNP arrays, Affymetrix expression data, allele-specific expression, gene expression matrices, sample attributes, subject phenotypes, laboratory measurements, & disease outcomes.\",\n\"phs002433.v1.p1\",\n\"Nasopharyngeal RNASeq Comparing SARS-CoV-2+ Patients and SARS-CoV-2    Negative Control Subjects\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Dec19, 2022\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002516.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: IBM Covid19    Contact Tracing and Data Exchange Tools\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer (2)\",\n\"Nov14, 2022\",\n\"approved\",\n\"I will be conducting testing to confirm that I, as a user of the RADx Data Hub, am only able to access data I have received prior approval for. \",\n\"I am a co-I of the RADx Data Hub program, and seek access to these program datasets to confirm researcher workflow functionality in the Hub is functioning properly. \",\n\"phs002516.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: IBM Covid19    Contact Tracing and Data Exchange Tools\"],\n [\"Claypool, Kajal \",\n\"MIT LINCOLN LABORATORY\",\n\"MIT LL Test Project\",\n\"Oct13, 2022\",\n\"approved\",\n\"This is a test request for the study to conduct a test for approval and notification to RAPIDS\",\n\"This is a test request for the study to conduct a test for approval and notification to RAPIDS\",\n\"phs002516.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: IBM Covid19    Contact Tracing and Data Exchange Tools\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Dec19, 2022\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002519.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Covidseeker and COVID-19 Citizen Science: Leveraging    Citizen Science and Real-Time Geospatial Temporal Mobile Data for    Digital Contact Tracing and SARS-CoV-2 Hotspotting\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer\",\n\"Nov09, 2022\",\n\"approved\",\n\"I will be testing whether I can see data in RAPIDS, download data from RAPIDS, open the data in the Data Hub and use cloud tools to analyze this information, and use this information to confirm Data Hub functionality, identify areas for improvements, and improve training materials for researchers. \",\n\"This request is to serve testing purposes of Data Hub functionality. \",\n\"phs002519.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Covidseeker and COVID-19 Citizen Science: Leveraging    Citizen Science and Real-Time Geospatial Temporal Mobile Data for    Digital Contact Tracing and SARS-CoV-2 Hotspotting\"],\n [\"Davis-Dusenbery, Brandi \",\n\"SEVEN BRIDGES GENOMICS, INC.\",\n\"RADx Data Hub Infrastructure Testing\",\n\"Nov14, 2022\",\n\"approved\",\n\"The RADx Data Hub provides a critical repository and ecosystem to support data access and analysis. This data request will allow testing and continued development of the RADx Data Hub in order to support the long term research objectives of RADx investigators. \",\n\"This data will not be used for research purposes, but rather to test the RADx Data Hub for accessing files from controlled datasets and uploading them to a Data Hub workbook.  This test will show that only authorized and authenticated researchers can access controlled data on the RADx Data Hub\",\n\"phs002519.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Covidseeker and COVID-19 Citizen Science: Leveraging    Citizen Science and Real-Time Geospatial Temporal Mobile Data for    Digital Contact Tracing and SARS-CoV-2 Hotspotting\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002522.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): MOF-SCENT:    Metal-Organic Frameworks for Screening COVID-19 by Electronic-Nose    Technology to Improve Selectivity and Time Response\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Mar27, 2023\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002523.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Multi-Modal    Wireless COVID Monitoring and Infection Alerts for Concentrated    Populations\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002523.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Multi-Modal    Wireless COVID Monitoring and Infection Alerts for Concentrated    Populations\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002524.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Validation of    Smart Masks for Surveillance of COVID-19\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Testing Data Access for RADx developer (2)\",\n\"Dec12, 2022\",\n\"approved\",\n\"I will be confirming data access using the Hub.    \",\n\"I am testing data access for select RADx program data. I will be using the Data Hub to access the data. \",\n\"phs002525.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): SF-RAD:    Development and Proof-of-Concept Implementation of the South Florida    Miami RADx-rad SARS-CoV-2 Wastewater-Based Surveillance Infrastructure\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002525.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): SF-RAD:    Development and Proof-of-Concept Implementation of the South Florida    Miami RADx-rad SARS-CoV-2 Wastewater-Based Surveillance Infrastructure\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002527.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Bioinformatics    Framework for Wastewater-based Surveillance of Infectious Diseases\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Dec19, 2022\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002533.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Digital Health Solutions for COVID-19: COVID Community    Action and Research Engagement (COVID-CARE)\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer (2)\",\n\"Nov14, 2022\",\n\"approved\",\n\"I will be conducting testing to confirm that I, as a user of the RADx Data Hub, am only able to access data I have received prior approval for. \",\n\"I am a co-I of the RADx Data Hub program, and seek access to these program datasets to confirm researcher workflow functionality in the Hub is functioning properly. \",\n\"phs002533.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Digital Health Solutions for COVID-19: COVID Community    Action and Research Engagement (COVID-CARE)\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Dec19, 2022\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002534.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: Team SAE\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer (2)\",\n\"Nov14, 2022\",\n\"approved\",\n\"I will be conducting testing to confirm that I, as a user of the RADx Data Hub, am only able to access data I have received prior approval for. \",\n\"I am a co-I of the RADx Data Hub program, and seek access to these program datasets to confirm researcher workflow functionality in the Hub is functioning properly. \",\n\"phs002534.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: Team SAE\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Feb07, 2023\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002535.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Personalized Analytics and Wearable Biosensor Platform for    Early Detection of Covid-19 Decompensation (DECODE)\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Dec19, 2022\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002537.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): COVID-19 Experience Study (C19EX) Survey\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer (2)\",\n\"Nov14, 2022\",\n\"approved\",\n\"I will be conducting testing to confirm that I, as a user of the RADx Data Hub, am only able to access data I have received prior approval for. \",\n\"I am a co-I of the RADx Data Hub program, and seek access to these program datasets to confirm researcher workflow functionality in the Hub is functioning properly. \",\n\"phs002537.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): COVID-19 Experience Study (C19EX) Survey\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Dec19, 2022\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002538.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): ILI Labels and Longitudinal Novel Engagement with Symptom    Surveillance (ILLNESS) Study\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer (2)\",\n\"Nov14, 2022\",\n\"approved\",\n\"I will be conducting testing to confirm that I, as a user of the RADx Data Hub, am only able to access data I have received prior approval for. \",\n\"I am a co-I of the RADx Data Hub program, and seek access to these program datasets to confirm researcher workflow functionality in the Hub is functioning properly. \",\n\"phs002538.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): ILI Labels and Longitudinal Novel Engagement with Symptom    Surveillance (ILLNESS) Study\"],\n [\"Krishnamurthy, Ashok \",\n\"UNIV OF NORTH CAROLINA CHAPEL HILL\",\n\"Testing RADx Data Hub and correlating ILI reports againts COVID-19 cases\",\n\"Nov16, 2022\",\n\"approved\",\n\"Understanding the correlation between the number of reports of flu-like symptoms and counts of COVID-19 cases.\",\n\"I am co-PI on the RADx Data Hub Project, which is creating a cloud-based infrastructure for the discovery and analysis of data related to COVID-19 testing. I am interested in understanding how checks for Influenza Like Illnesses (ILI) predict the number of COVID-19 cases. I am primarily also testing if the workflow to request a RADx dataset works.\",\n\"phs002538.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): ILI Labels and Longitudinal Novel Engagement with Symptom    Surveillance (ILLNESS) Study\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Dec19, 2022\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002539.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Large Scale Flu Surveillance Study (LSFS)\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer (2)\",\n\"Nov14, 2022\",\n\"approved\",\n\"I will be conducting testing to confirm that I, as a user of the RADx Data Hub, am only able to access data I have received prior approval for. \",\n\"I am a co-I of the RADx Data Hub program, and seek access to these program datasets to confirm researcher workflow functionality in the Hub is functioning properly. \",\n\"phs002539.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): Large Scale Flu Surveillance Study (LSFS)\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Dec19, 2022\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002540.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: SAFER-COVID -    Integration of Testing and Digital Health\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer (2)\",\n\"Nov14, 2022\",\n\"approved\",\n\"I will be conducting testing to confirm that I, as a user of the RADx Data Hub, am only able to access data I have received prior approval for. \",\n\"I am a co-I of the RADx Data Hub program, and seek access to these program datasets to confirm researcher workflow functionality in the Hub is functioning properly. \",\n\"phs002540.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: SAFER-COVID -    Integration of Testing and Digital Health\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002542.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Wastewater    Assessment for Coronavirus in Kentucky: Implementing Enhanced    Surveillance Technology\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002543.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Minimal    False-Alarm Touch-Based Detection of SARS-CoV-2 Virus Particles Using    Poly-Aptamers\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002544.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): AFS/SERS    Saliva-Based SARS-CoV-2 Earliest Infection and Antibodies Detection\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002546.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Development of an    Automated Diagnostic Platform for SARS-CoV-2 Monitoring in Vulnerable    Areas\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002549.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Severity    Predictors Using Immunology and Transcriptomics in Saliva Using Multi    Neural Network Intelligence in SARS-CoV2 Infection in Children (SPITS    MISC)\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002550.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Direct    Bioelectronic Detection of SARS-CoV-2 From Saliva Using Singlemolecule    Field-effect Transistor Array\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002551.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Marshallese:    Alternate Surveillance for COVID-19 in a Unique Population\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002553.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Characterization    of Multisystem Inflammatory Syndrome in Children and its Relationship    to Kawasaki Disease\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002561.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Rapid,    Sensitive, Point-of-Care, Antigen-Based Diagnostics for SARS-CoV-2\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002563.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Handheld    Microchip for GC Analysis of Breath to Screen for COVID-19\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002565.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad):    Touchscreen-Compatible, Real-Time Electrochemical Sensing of    SARS-CoV-2\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002569.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): COVID-19 Network    of Networks Expanding Clinical and Translational Approaches to Predict    Severe Illness in Children (CONNECT to Predict SIck Children)\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002570.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Multimodal    Platform for Oral Screening of COVID-19\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002572.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A SARS-CoV-2    Breathalyzer for Direct Virus Detection\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002573.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Detection and    Automatic Privacy-Protected Contact Tracing System Designed for    COVID-19\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002583.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Rapid    Breathalyzer Diagnostics Platform for COVID-19\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002585.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): AICORE-kids\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002588.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Effective,    Reagent-Free Detection of the Odor Signature of COVID-19 Infection    Using a Nano-Enabled Sensor Array\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002600.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Portable GC    Detector for COVID Diagnostics\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002602.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Virus Counter:    Rapid and Sensitive Diagnostics Based on Digital Detection of    Individual Pathogens\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002603.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Diagnosis of    MIS-C in Febrile Children\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002604.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Tracking the    COVID-19 Epidemic in Sewage (TRACES)\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002609.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Identifying    Biomarker Signatures of Prognostic Value for Multisystem Inflammatory    Syndrome in Children (MIS-C)\"],\n [\"Anwar, Mohd Mozharul\",\n\"NIH\",\n\"Exploration of Wearable Device Data for COVID-19 \",\n\"Dec19, 2022\",\n\"approved\",\n\"Wearable devices collect various physiological signals and measurements such as heart-beat rate, respiration rate, sleep, and body movement. This study explores wearable device data for detection, risk measurement, or prediction of disease like COVID-19.\",\n\"The objective of the proposed research is to explore how wearable device data from different sources can be aggregated and analyzed to draw meaningful scientific insights, such as detection, risk stratification, or prediction of disease. The datasets will be utilized to build various machine learning/deep learning models. In the process, the datasets will be standardized, normalized, and harmonized. Furthermore, it will be studied how to leverage survey data in interpreting wearable device data. The different models will be compared on the basis of various performance metrics.\",\n\"phs002628.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: Clear2Go - A    Digital Identity Wallet for Health Status\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Confirmation of RAS approval workflow for RADx Data Hub 1.0 developer (2)\",\n\"Nov14, 2022\",\n\"approved\",\n\"I will be conducting testing to confirm that I, as a user of the RADx Data Hub, am only able to access data I have received prior approval for. \",\n\"I am a co-I of the RADx Data Hub program, and seek access to these program datasets to confirm researcher workflow functionality in the Hub is functioning properly. \",\n\"phs002628.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Digital Health Technologies    (RADx-DHT): NIH Digital Health Solutions for COVID-19: Clear2Go - A    Digital Identity Wallet for Health Status\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002631.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Rapid Olfactory    Tools for Telemedicine-Friendly COVID-19 Screening and Survey\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002642.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A    Confectionary-Based Screening Tool for Assessing Chemosensory Loss in    COVID-19 Patients\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002657.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Early Detection,    Containment, and Management of COVID-19 in Dialysis Facilities Using    Multi-Modal Data Sources\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002679.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Wastewater    Detection of COVID-19\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002685.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): DNA Star    SAS-CoV-2 Rapid Test\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002689.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Rapid Saliva    Antigen Test for SARs-CoV-2 Detection\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002699.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): SCENTinel: A    Rapid Smell Test for COVID-19 Surveillance\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002700.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Nanobody-based    Electrochemical Biosensor for Real-Time Detection of Aerosolized    SARS-CoV2\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002702.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Scalable    Aptamer-Based Electrochemical Biosensor for Rapid Detection of    SARS-CoV-2 from Saliva\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002709.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Multiparametric    Integrated Molecular Detection of SARS-CoV-2 from Biofluids by    Adapting Single Extracellular Vesicle Characterization Technologies\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002729.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Improved    Scalability, Sensitivity, and Interpretability of Pathogen Detection,    Including SARS-CoV-2, in Wastewater using High-Throughput, Highly    Multiplexed Digital Array PCR Technology\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002744.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Rolosense: An    Innovative Platform for Automatic Mobile Phone Readout of Active    SARS-CoV-2\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002747.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): COVID-19    Detection through Scent Analysis with a Compact Gas Chromatography    Device\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002778.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Microfluidic    Isolation and Characterization of SARS-CoV-2 and Virus Related    Exosomes\"],\n [\"Jin, Shanzhao \",\n\"BIOMAP (BEIJING) INTELLIGENCE TECHNOLOGY LIMITED\",\n\"Integrated analysis of multi-omics data on human synovial tissue and PBMC in Rheumatoid Arthritis (RA)\",\n\"Feb14, 2023\",\n\"approved\",\n\"The main goal of this study is to reveal the underlining molecular mechanisms between treatment-responded and treatment resistant patients with RA. Specifically, we want to identify the critical factors that drive immune balance in RA patients with treatment. To explore potential targets or treatment strategies for RA, we plan to define and validate the cell subpopulations, cell states and gene pathways associated with RA pathogenicity and treatment response.\",\n\"The Target Discovery Research team at BioMap aims at leveraging multi-omic data for the novel drug discovery. Specifically, we are interested in auto-immune diseases such as Rheumatoid Arthritis. Our research objectives are to compare the RA synovial biopsy, PBMC, OA and healthy control datasets in public databases (including dbGaP) with our in-house data.    We want to make full use of datasets from dbGaP, particularly those from the Accelerating Medicines Partnership (AMP), as meaningful and significant references for our in-house single-cell RNA-seq (scRNA-seq) and spatial transcriptome datasets. First, we intend to integrate all the available datasets to uniformly define and validate cell subpopulations in RA tissues by different studies and use them as a solid reference for our in-house scRNA-seq dataset from synovial tissues and PBMC. Second, to identify the priority or major cell-cell interaction network hubs in RA progression and remission during treatment, including those cell types contributing to tissue repair/anti-inflammatory and tissue damage/pro-inflammatory, respectively. Finally, we will detect cellular responses to perturbations to identify relationships between genes and RA in silicon and in vitro. Overall, we request access to datasets from projects phs000983.v1.p1, phs001262.v1.p1.c2, phs001529.v1.p1.c1, phs001457.v1.p1, phs001459.v1.p1 and phs001615.v1.p1. The comparison and combination of these datasets with our in-house data will provide potential insights for RA pathogenesis and immunotherapies.   BioMap aims to accelerate novel drug discovery for unmet medical needs with secure IT systems and data storage. We will comply with all regulations for data access as set out in dbGaP, and we state that our institution's IT security requirements and policies are sufficient to protect the confidentiality and integrity of the NIH controlled-access data. \",\n\"phs002780.v1.p1\",\n\"IFN-&#x003b3; and TNF-&#x003b1; drive a CXCL10+ CCL2+ Macrophage Phenotype Expanded in    Severe COVID-19 Lungs and Inflammatory Diseases with Tissue    Inflammation\"],\n [\"Miguez, Maria-Jose \",\n\"FLORIDA INTERNATIONAL UNIVERSITY\",\n\"System analysis for COVID humoral response\",\n\"Feb10, 2023\",\n\"approved\",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Therefore, it is critical to understand who is at risk (e.g. those with allergies). Our long term goal is to improve early diagnosis.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely prepared given our medical and research experience.   \",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Indeed, in a sizable proportion of cases (up to 40%), the patient requires to be admitted to the ICU service. Therefore, it is critical to understand who is at risk (e.g. those with allergic or non-allergic diseases) those with alterations in the IgE network (platelets, eosinophils, specific Th2 or Th17 cytokines). Our long term objective is to develop new strategies for early diagnosis and to identify biomarkers of risk that can be easily deployed globally.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely qualified to do such analyses given the medical background of the PI, our ongoing work in COVID both in humans and with animal models, our unique expertise in IgE, and expertise using machine learning and model-based analyses.   \",\n\"phs002781.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Discovery and    Clinical Validation of Host Biomarkers of Disease Severity and    Multi-system Inflammatory Syndrome in Children (MIS-C) and COVID-19\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002781.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Discovery and    Clinical Validation of Host Biomarkers of Disease Severity and    Multi-system Inflammatory Syndrome in Children (MIS-C) and COVID-19\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002782.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): NIEHS    Diagnostic-Prognostic RNAseq\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002924.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Broad-Spectrum    Detection of VOC and Non-VOC Biomarkers from Patient Exhalant Using a    Biomimetic Multiplexed eNose Biosensor for COVID 19 Diagnosis\"],\n [\"Miguez, Maria-Jose \",\n\"FLORIDA INTERNATIONAL UNIVERSITY\",\n\"System analysis for COVID humoral response\",\n\"Feb10, 2023\",\n\"approved\",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Therefore, it is critical to understand who is at risk (e.g. those with allergies). Our long term goal is to improve early diagnosis.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely prepared given our medical and research experience.   \",\n\"Multisystem inflammatory syndrome in children (MIS-C) is a rare condition that emerged a couple weeks after a child is exposed to SARS-CoV-2. MIS-C compromised several organs (e.g. heart, lungs, kidneys, brain, skin, eyes, or gastrointestinal tract), and it can be deadly. Indeed, in a sizable proportion of cases (up to 40%), the patient requires to be admitted to the ICU service. Therefore, it is critical to understand who is at risk (e.g. those with allergic or non-allergic diseases) those with alterations in the IgE network (platelets, eosinophils, specific Th2 or Th17 cytokines). Our long term objective is to develop new strategies for early diagnosis and to identify biomarkers of risk that can be easily deployed globally.   Based on our strong preliminary data, we want to validate our findings in a pediatric cohort. The RADx hub will provide a unique opportunity to do so with the large data available. This team is uniquely qualified to do such analyses given the medical background of the PI, our ongoing work in COVID both in humans and with animal models, our unique expertise in IgE, and expertise using machine learning and model-based analyses.   \",\n\"phs002945.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Data Science    Approach to Identify and Manage Multisystem Inflammatory Syndrome in    Children (MIS-C) Associated with SARS-CoV-2 Infection and Kawasaki    Disease in Pediatric Patients\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002945.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): A Data Science    Approach to Identify and Manage Multisystem Inflammatory Syndrome in    Children (MIS-C) Associated with SARS-CoV-2 Infection and Kawasaki    Disease in Pediatric Patients\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs002964.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Longitudinal at    Home Smell Testing to Detect Infection by SARS-CoV-2\"],\n [\"Ciofani, Danielle \",\n\"BROAD INSTITUTE, INC.\",\n\"Testing Data Access for RADx developer\",\n\"Dec12, 2022\",\n\"approved\",\n\"I will be viewing data and loading it into a Jupyter Notebook to confirm functionality works. \",\n\"I am testing data access for select RADx program data. I will be using the Data Hub to access the data.\",\n\"phs003027.v1.p1\",\n\"Rapid Acceleration of Diagnostics - TECH (RADx-TECH): Test US Bank Study\"],\n [\"Rose, Peter \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Analysis and Evaluation of RADx-rad Datasets\",\n\"Mar07, 2023\",\n\"approved\",\n\"The National Institutes of Health launched the RADx? Radical program to support innovative, non-traditional diagnostic approaches to address gaps in COVID-19 testing and surveillance. This request is to support the evaluation of RADx-rad program outcomes by the RADx-rad Discoveries and Research Center (DCC).\",\n\"This request supports the RADx-rad program outcomes evaluation by the RADx-rad Discoveries and Research Center (DCC). DCC facilitates \\u00e2\\u0080\\u009cdata standardization, harmonization, integration, and analysis across RADx-rad projects and coordinates quality control, data curation, and analyses, and provides tools to monitor progress, performance, and use of the curated data\\u00e2\\u0080\\u009d (RFA-OD-20-019). To fulfill these requirements, we will perform the following evaluations: 1. Measure the extent of shared data elements across (i) projects within the same or related FOA areas, (ii), projects across all RADx-rad. 2. Evaluate the quality and consistency of data submissions. 3. Compare the performance of diagnostic methods for analytical and clinical performance. 4. Correlate analytical with clinical performance. 5. Develop data analysis templates and use cases to demonstrate data integration across RADx-rad projects. 6. Evaluate the ease of use and capabilities of the NIH Data Hub to (i) access datasets, (ii) data dictionaries, (iii) develop and run data analyses (for # 1. \\u00e2\\u0080\\u0093 5.) in the Data Hub workbench. \",\n\"phs003124.v1.p1\",\n\"Rapid Acceleration of Diagnostics - Radical (RADx-rad): Exosome-Based    Non-Traditional Technologies Towards Multi-Parametric and Integrated    Approaches for SARS-CoV-2\"],\n [\"Bendall, Matthew \",\n\"WEILL MEDICAL COLL OF CORNELL UNIV\",\n\"Investigating retroelement expression in cancer for biomarker discovery\",\n\"Aug27, 2021\",\n\"approved\",\n\"In human cancers, HERVs (fossil viruses which live in all of our DNA) have been suspected to be involved in pathogenesis of some cancers, with the detection of HERV expression in germ cell tumors, prostate and breast cancers, melanoma, renal cell carcinoma. However, HERV protein, and particle expression during cancers does not ascribe a definitive role for HERV activity in contributing to cancer etiology in humans. Whether or not HERVs play a causative role, HERV expression is an overlooked source of neoantigenic targets or biomarkers of tumor responsiveness. We wish to understand whether targeting specific HERV insertions could be used therapeutically in cancers.\",\n\"The goal of this project is to investigate how transcriptional activity of retroelements, including Human Endogenous Retroviruses (HERVs) and Long Interspersed Nuclear Elements (LINEs), may be involved in disease pathogenesis and prognosis. We have developed an analysis tool, Telescope, that can accurately quantify transposable element expression at a locus specific level. We will use Telescope to determine which HERVs and LINEs are expressed in several different cancers. This project extends our initial work in prostate, breast and colon cancers to also focus on cutaneous and uveal melanomas, lymphomas, and head and neck cancer. We intend to broadly share our results with the scientific community by publishing our findings in peer-reviewed journals.\",\n\"phs002315.v1.p1\",\n\"Integrated Analysis of Multimodal Single-Cell Data\"],\n [\"GETZ, GAD \",\n\"BROAD INSTITUTE, INC.\",\n\"Studying the cellular dynamics of response to CAR-T therapy\",\n\"May23, 2022\",\n\"approved\",\n\"CAR-T cell therapy has revolutionized the treatment of patients suffering from a wide class of cancers of the blood (leukemias) and lymph node (lymphomas). In this type of treatment, a patient\\u00e2\\u0080\\u0099s own immune cells are engineered to be able to recognize and kill the tumor cells. Despite their success, still about half of patients with advanced lymphomas ultimately relapse after treatment, and the reasons for this are not well understood. By sequencing the RNA from these CAR-T cells and other immune cells from lymphoma patients treated with this therapy, we identified the features of these cells that are characteristic of patients who do, or do not, respond, and show how these change across the course of treatment. \",\n\"Our study aims to understand the mechanisms of response to CAR-T therapy in patients with large B-cell lymphomas through single-cell RNA sequencing. A preliminary version of our study can be found on medRxiv (https://doi.org/10.1101/2022.04.04.22273422). We obtained a total of 105 samples, including pre- and post-treatment peripheral blood and infusion products, from 32 patients treated with the first two FDA-approved CD19 CAR-T products: axicabtagene ciloleucel (axi-cel) and tisagenlecleucel (tisa-cel). Through analysis of the CAR-T cell and other immune populations, we identify the key transcriptional features displayed by responders and non-responders to each product, as well as their dynamics across treatment timepoints. To characterize T-cell subpopulations in our data, we labeled cells based on expression of commonly used markers in FACs-based literature. We are applying for access to this dataset in order to have a reference 10x dataset with both CITE-seq and RNA-sequencing data that can serve to validate and enhance our classification method. We will pre-process this dataset using a pipeline identical to the one used in our project, and then classifying the cells based on both (1) the RNA-based classification method we used, and two (2) a protein-based classification using the CITE-seq data, and then compare the concordance of the two classification schemes. This dataset may also be used as training data to better parameterize our model, which we could then apply to future datasets. The clinical side of this project is in collaboration with the labs of Marcela Maus at Massachusetts General Hospital and Catherine Wu at Dana Farber Cancer Institute, but only the Getz lab will handle the raw data applied for here.\",\n\"phs002315.v1.p1\",\n\"Integrated Analysis of Multimodal Single-Cell Data\"],\n [\"Ratan, Aakrosh \",\n\"UNIVERSITY OF VIRGINIA\",\n\"Investigating Heterogeneity of LGL Leukemia\",\n\"Jan13, 2022\",\n\"approved\",\n\"Large granular lymphocytic (LGL) leukemia is a type of chronic leukemia that affects white blood cells called \\\"lymphocytes\\u00e2\\u0080\\u009d. The disease is heterogeneous both in terms of clinical presentations and the molecular phenotypes. We plan to generate single-cell data from LGL patients and will use publicly available datasets to provide context and understand the various cell types and states that play a role in LGL. \",\n\"LGL leukemia results from a persistent clonal expansion of cytotoxic lymphocytes, including from T-cell or NK-cell lineages. The disease is also associated with neutropenia, anemia and B-cell dyscrasias. The goal of this project is to use multi-modal single-cell sequencing to establish a high-dimensional single-cell reference atlas of LGL leukemia cell types and cell states. We aim to generate multi-modal sequencing datasets from LGL leukemia patients with various clinical presentations and molecular phenotypes. We plan to map new data onto available public reference single-cell and genomic datasets including the dataset generated for study accession phs002315.v1.p1 for contextual interpretation of aberrant cell states. We will then use the supervised annotations to test for differences in cell type abundance or cell type gene expression across the various clinical and molecular phenotypes. In order to control against technical artifacts, we will standardize analysis using the same software pipeline on each dataset, using raw reads as input. After analysis is complete, the generated sequencing data will be deposited in dbGaP, while processed counts data will be deposited to GEO. We intend to broadly share our results with the scientific community by publishing our findings in peer-reviewed journals. \",\n\"phs002315.v1.p1\",\n\"Integrated Analysis of Multimodal Single-Cell Data\"],\n [\"Labrecque, Nathalie \",\n\"HOPITAL MAISONNEUVE-ROSEMONT\",\n\"Application of a newly developed Notch T-cell specific transcriptional signature\",\n\"Apr30, 2023\",\n\"approved\",\n\"The immune system is devoted to fighting infectious agents. This is achieved by white blood cells that patrol the organism to eliminate pathogens. T cells, a type of white blood cells, are central players in an efficient immune response. These T cells recognize foreign materials via a cell surface receptor, the T cell receptor (TCR). This recognition allows for the expansion and differentiation of T cells that are specific to the infectious agent. This differentiation step is crucial to allow the T cells to eliminate the pathogen. We have previously shown that the evolutionarily conserved Notch receptor controls the differentiation of T cells. Identifying T cells that have active or inactive Notch signaling could be helpful to develop therapies where we manipulate Notch to control the immune response of T cells. In this proposal, we will determine whether a tool that we have developed, a T-cell specific Notch molecular signature, is detected in patients with multisystem inflammatory syndrome (MIS-C) following SARS-CoV-2 infection. These patients have been shown to have an affected Notch pathway. This would help us validate a tool that could simplify the identification of biological contexts where manipulation of the Notch pathway in T cells could be targeted for therapy.\",\n\"The evolutionary conserved Notch signaling pathway controls the differentiation fate in many cell types. In particular, our team has found that Notch plays a crucial role in the differentiation of effector CD8 T cells in the context of acute infections. Others have also described that the Notch pathway controls the CD4 T cell response during graft-versus-host disease (GVHD). Thus, the Notch pathway is important in the response of peripheral T cells. However, bioinformatic tools to identify Notch signalling often fail to identify the activity of this pathway in different T-cell responses. This is illustrated in a recent publication entitled \\\"Notch1-CD22-Dependent Immune Dysregulation in the SARSCoV2-Associated Multisystem Inflammatory Syndrome in Children.\\\" In this publication, the authors demonstrate an important role for Notch in Multisystem Inflammatory Syndrome, but the scRNA-seq data does not identify this pathway (see Fig S1). We propose that this is because the currently available Notch signatures in public databases are derived from tumour-derived data and Notch signalling in peripheral T cells regulates a distinct set of genes.   To address this issue, we developed our own Notch signature specific to activated peripheral T cells. We have validated this signature in several contexts of Notch deficiency or overexpression in mice. We have also shown that this signature is specific to T cells, as there was very little overlap with currently available Notch signatures developed from cancer cell lines.   We wish to evaluate whether our newly generated Notch signature is enriched during different types of human T-cell responses. In the above-mentionned study of Benamar M et al. (phs003086.v1.p1), single-cell RNA-seq was used to identify robust Notch signaling in Tregs of patients having developed multisystem inflammatory syndrome (MIS-C) following SARS-CoV-2 infection. Our objective is to determine whether our T-cell Notch Signature is enriched in these patients\\u00e2\\u0080\\u0099 Tregs. To this end, the scRNA-seq will be analyzed using Seurat v 4.0.0. The top 3000 variable genes will be used for PCA calculations and the first 25 PCs will be used for UMAP and for cluster identification (obtained with Louvain algorithm with a resolution of 2). Enrichment of the T-cell Notch Signature on each individual cell will be measured with the AddModuleScore function from Seurat. We will not be evaluating phenotypic characteristics or genetic variants of the participants?\",\n\"phs003086.v1.p1\",\n\"Single Cell Transcriptomic Data from CD4+ T Cells from Children with    MIS-C\"],\n [\"Aagaard, Kjersti \",\n\"BAYLOR COLLEGE OF MEDICINE\",\n\"Placental immunity at single-cell resolution\",\n\"Sep22, 2021\",\n\"approved\",\n\"The identification of microbes at the maternal-fetal interface in the placenta raises the question of whether they are transient, constant, or serve a function. We hypothesize these microbial challenges train the fetal immune system how to deal with beneficial or harmful pathogens. Access to this dataset will provide (1) cross-validation of our uninfected datasets and (2) allow for direct comparisons of many uninfected and infected cell types. The overarching objective of this proposed work is to determine unique transcription programs in the placenta that restrict (or permit) vertical transmission of microbes.\",\n\"The objective of this proposed work is to determine unique transcription programs in the placenta that restrict (or permit) vertical transmission of microbes. We plan to integrate the Pique-Rengi et al. (2019) and Pique-Rengi et al. (2020) scRNA-seq datasets from term and preterm chorioamniotic membranes, placental villi, and basal plate samples with our own scRNA-seq datasets. Our datasets are from patients with congenital viral infections, and we are aligning the raw data to custom human+virus transcriptomes. Thus, we need the raw data from the Pique-Rengi publications to make sure our viral transcripts are not seen in (presumably) uninfected samples. We will apply the same pre-processing (CellRanger) and downstream (Seurat in R) analyses to annotate our cell clusters and to do differential expression analyses. We anticipate finding infected and activated immune cells with distinct viral and host transcription responses. This proposed research will be consistent with the data use limitations and all analyses will be done in my lab and will not be shared in collaborations.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Adams, Erin \",\n\"UNIVERSITY OF CHICAGO\",\n\"Immune gene regulation at maternal-fetal interface\",\n\"Nov10, 2020\",\n\"closed\",\n\"The molecular abundance of a gene, or its gene expression level, can give us a snapshot of this gene's reaction to specific conditions. Specific conditions can include things like getting sick, intense exercise, or becoming pregnant. It has been especially difficult to obtain single snapshots, let alone multiple snapshots over time, of gene expression levels in the uterus during pregnancy for ethical, medical, and technological reasons. However, recent advances have provided us with exactly that. By using the data published herein, we hope to generate a data set that will allow us to look at three different gene expression level snapshots\\u00e2\\u0080\\u0093one during the first, one during the second, and one during the third trimester of pregnancy\\u00e2\\u0080\\u0093to study how genes react during a developing human pregnancy to learn more about which genes are important during healthy human development. \",\n\"We will combine these data with similar published data generated from women during the first trimester of pregnancy to generate a pregnancy transcript expression time-course data set in order to investigate the expression patterns of immune-related genes of interest within single cells. The same published methodology will be used to generate count matrices and perform additional downstream normalization for all samples. There are no plans to perform any type of genetic association analyses. There are no plans to associate expression patterns with phenotypic characteristics. The goal is to determine gene expression patterns over time during healthy human pregnancy. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Adibi, Jennifer \",\n\"UNIVERSITY OF PITTSBURGH AT PITTSBURGH\",\n\"Tissue type differences in placental transcriptome\",\n\"Dec14, 2021\",\n\"expired\",\n\"Maternal exposures including pre-pregnancy obesity predispose women to adverse pregnancy outcomes possibly caused by molecular changes in the placenta. Obesity may cause changes in placental cell types or through modifying multiple genetic pathways. This study will help characterize placental changes in gene expression and cell types that may explain associations between maternal obesity and gestational age at delivery. \",\n\"Our research interest is characterizing how maternal exposures predispose women to adverse pregnancy outcomes through changes in the placental transcriptome. The placenta is a heterogenous organ with multiple tissue and cell types that may be modified by maternal exposures differently. We have collected RNA sequencing data on three placental tissues including chorion smooth, chorion frondousm, and basal plate from a de-identified cohort of pregnant women. In this project, we are interested in identifying genes differentially expressed in the placenta that mediate the effects of maternal obesity on gestational age using high-dimensional mediation analyses (HIMA in R). It is possible that differential gene expression in bulk placental tissue may reflect heterogeneity in cell composition. To complement the mediation analysis, we aim to quantify differences in cell composition by cell deconvolution (EpiDish in R) using scRNA-seq datasets of placental and decidual tissues from Pique-Regi et al. (2019), Vento-Tormo et al. (2018), Suryawanshi et al (2018), and Tsang et al. (2017). Cell deconvolution of bulk placental tissue will additionally help identify cell-specific markers for future transcriptomic analyses of maternal exposures in our pregnancy cohort. Therefore, we are requesting the dbGap dataset (phs001886.v1.p1) of villous tissue, basal plate, and chorioamnionitic membranes in term and preterm deliveries to understand placental cell-type patterns associated with maternal exposures such as obesity. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Anchang, Benedict Nchang\",\n\"NIH\",\n\"The Materno-Fetal System during Pregnancy at Single-cell resolution\",\n\"Mar20, 2023\",\n\"approved\",\n\"This study aims to identify and characterize major cell types and their interactions representing placenta development across various ethnic groups and benchmarking the results with the Human Cell Atlas data and other environmental related databases. The tools and resources to be developed will be made publicly available to the research community as well as support innovative data generation and methods to reduce health disparities in the field of reproductive health.\",\n\"Prenatal environmental chemical exposures such endocrine disruptors and maternal infections (e.g., HIV and malaria) and morbidities (e.g., hypertensive disorders of pregnancy) are the leading causes of global perinatal (fetal and neonatal) and maternal morbidity and mortality. In addition, the exposure of these factors at different levels in different regions and ethnic groups results in global disparity of reproductive health outcomes. For example, Nigeria contributes the largest portion about 20% of the burden of maternal mortality worldwide, despite having just 2% of the global population. To improve understanding of how prenatal exposures affect fetal growth, maternal health and ultimately lifelong health outcomes, cellular reference maps of normal biological processes need to be developed during pregnancy. As a PI and part of the Ancestry network for Human cell atlas funded by the Chan Zuckerberg Initiative, the goal of this study is to provide a reference reproductive and perinatal developmental cell map of African and non-African ethnic groups that can be used to study early and long-term effects during normal and perturbed pregnancy from environmental and socioeconomic factors.  To account for environmental variations due to geography and ethnicity we plan to integrate single-cell RNA-sequencing placenta data from 3 Nigeria ethnic groups; Fulani, Ibo and Yoruba with other ancestry groups e.g., European ancestry. For example, the dbGaP single-cell RNA-sequencing data from the phs001886.v4.p1 study of human myometrium during spontaneous term labor include 113 placenta samples of various placenta regions: the placental villous tree, basal plate, and chorioamniotic membranes of women across a range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19. Integrating the control data from non-African ancestry with the African study data will increase sample size and diversity as well help generate a robust reference placenta cell atlas.  I will be responsible for accessing the controlled access data. The research use of the data is limited only by the terms of the model Data Use Certification.  Also, this research study is independent of collaboration with researchers at other institutions.    \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Bakulski, Kelly \",\n\"UNIVERSITY OF MICHIGAN AT ANN ARBOR\",\n\"Bulk villous tissue deconvolution in the term placenta: parallel single-cell and sorted cell type-specific gene expression\",\n\"Jul27, 2020\",\n\"approved\",\n\"The public health burden of negative pregnancy outcomes is substantial. The functional tissue of the placenta has been studied to better understand how the placenta works and leads to negative pregnancy outcomes. Tissue-level measures of gene activity represent a mixture of measurements from individual cell types that make up the tissue. Each cell type has a unique gene activity pattern. Failure to account for each cell type\\u00e2\\u0080\\u0099s pattern can cloud study results and limit our understanding of the placenta. Critically, we have a limited understanding of cell type gene activity patterns in the placenta. The overall goal of our study is to measure these patterns to strengthen studies of the placenta. We would like to compare our results and findings to a previous study. In addition, we would like to combine our datasets and reanalyze our results and findings in a combined dataset. \",\n\"The public health burden of adverse pregnancy outcomes is substantial. Villous tissue, the parenchyma of the placenta, has been characterized at term through gene expression to better understand placental function. However, tissue-level expression measurements represent a convolution of signals from individual cell types that compose the tissue. Cell types exhibit cell type-specific gene expression patterns. Failure to account for cell type heterogeneity could bias study results and limits interpretability. Critically, placental cell type-specific RNA references are limited or unavailable to deconvolve bulk placenta villous tissue measures. This research integrates RNA sequencing of seven flow-sorted placental cell types and single-cell RNA sequencing of term placental villous tissue to robustly generate cell type-specific gene expression profiles. Cell type-specific gene expression references will be publicly disseminated for bulk tissue deconvolution to advance molecular epidemiology studies of the placenta.  We request access to the phs001886.v1.p1 single-cell RNA sequencing dataset to compare our results to previously published findings and integrate the phs001886.v1.p1 dataset with primary single-cell RNA sequencing data to improve power in identifying cell type-specific gene expression, if appropriate and accounting for batch effects. The study by Pique-Regi et al. examined the transcriptional landscape of major placental compartments among preterm births, non-laboring term birth, and laboring term birth. Access to their findings would provide an opportunity to test for biological replication and integrate additional data to identify cell type-specific gene expression patterns. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Blelloch, Robert \",\n\"UNIVERSITY OF CALIFORNIA, SAN FRANCISCO\",\n\"Trophoblast gene expression programs in the chorionic villi and smooth chorion\",\n\"Jun27, 2022\",\n\"approved\",\n\"The human placenta contains two specialized regions: the villous chorion and the smooth chorion. In this study, we use single cell RNA sequencing to compare the cell types and molecular programs between these two regions in the second trimester human placenta. Access to these data will allow comparison 1) between the cell types we identify and those identified in these studies 2) comparison between cell types identified in the second trimester and term. Analyses of these data will improve analysis and definition of trophoblast subtypes and expand analysis into changes related to gestational age.\",\n\"We are using single cell RNA sequencing to compare the cell types and molecular programs between trophoblasts isolated from the chorionic villi and the smooth chorion regions of the second trimester human placenta. We will integrate our scRNA-seq data from these regions with the data in Pique-Regi et al., 2019 and 2020 from the healthy chorioamniotic membranes, placental villi, and basal plate samples collected at term. Our goal is to compare the cell types identified in our analysis of the second trimester to those found in these publications. To do so, we require access to the raw data from these studies so that they may be processed using matching methodologies to our data, which will minimize batch effects and improve integration of all datasets. Our proposed research plans are consistent with data use limitations and not be shared in other collaborations. No genetic variant association analyses will be performed. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Elo, Laura \",\n\"UNIVERSITY OF TURKU\",\n\"Computational methods for complex disease characterization \",\n\"Mar13, 2020\",\n\"closed\",\n\"With modern high-throughput biotechnologies it is possible to study biological systems in unprecedented insight and resolution. Specifically interesting is how the data will best be utilised in clinical practice and individualized prediction of diagnosis, prognosis or treatment of complex disorders such as immune-related diseases, neurodevelopmental disorders, and obesity. We plan to focus on developing and optimising computational data analysis tools and methods using large-scale data such as genotyping and next generation sequencing data with the aim of enabling robust and reproducible interpretation of the data. By applying mathematical, statistical and machine learning methods we aim to identify key features and patterns characterizing and revealing new insights into complex disorders.\",\n\"The objective of our research is to develop effective computational data analysis tools and mathematical modelling methods for biomedical research. A specific focus is on analyzing and interpreting data generated by modern high-throughput biotechnologies and on assessing their utility in clinical practice. Building on our previous computational, statistical and network-based studies, we aim at establishing a computational framework that allows optimized integration and analysis of large- scale data as well as heterogeneity between individuals to enable robust and reproducible interpretation of the data. The eventual goal is to improve the diagnosis, prognosis and treatment of complex diseases.  The objective of our research is to develop effective computational tools and mathematical models towards inferring optimal solutions for clinical problems at molecular level. In particular, we aim at enabling robust and reliable analysis and interpretation of data generated by modern high-throughput biotechnologies. A specific focus is on assessing the utility of next generation sequencing data in clinical practice and individualized prediction of diagnosis, prognosis or treatment of complex disorders such as immune-related diseases, neurodevelopmental disorders, and obesity. The data will be used for developing generally useful computational tools, specific models as well as for assessing a general background. We will develop and apply mathematical, statistical and machine learning methods to identify key features and patterns characterizing the particular disorder at multiple levels. An important prerequisite for developing generally useful computational solutions for biomedical applications is accessing large-scale well-designed datasets. Therefore, to ensure generally useful solutions that are unbiased and reliable, we would like to apply for the access to the comprehensive datasets featuring genotyping and next generation sequencing data. Such datasets fall within our requirements for the quality and the number of samples. Besides optimization of the data analysis procedures, consideration of multiple datasets holds potential to reveal also new insights into genetic abnormalities, genetic correlation between traits and heritability.  We do not reveal any information on any individual and the proposed research is consistent with the Use Restrictions. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Erlebacher, Adrian \",\n\"UNIVERSITY OF CALIFORNIA, SAN FRANCISCO\",\n\"Transcriptional regulation of human decidual stromal cells\",\n\"May13, 2020\",\n\"closed\",\n\"Successful pregnancy involves the coordinated interactions between multiple cell types at the interface between the mother\\u00e2\\u0080\\u0099s uterus and the fetus. It is currently thought that these interactions, when aberrantly controlled, cause a number of pregnancy complications, including preterm birth. The long-term goal of our work is to understand how gene expression within a key cell type of the pregnant uterus, called the decidual stromal cell, is controlled. The dataset we wish to gain access to in this application will allow us to identify key genes whose regulation might be controlled by a specialized \\u00e2\\u0080\\u009cepigenetic\\u00e2\\u0080\\u009d mechanism we recently discovered through our work on the maternal-fetal interface in mice. Understanding whether this mechanism operates during human pregnancy will open up entirely new ways of thinking about how human pregnancy complications come about and would lay the foundation for discovering new therapeutics to improve human pregnancy outcome, such as ones that prevent preterm birth.\",\n\"Successful pregnancy involves the coordinated interactions between multiple cell types at the maternal-fetal interface.  One cell type is the decidual stromal cell (DSC), which comprises the key parenchymal component of the decidua, i.e. the specialized endometrial tissue that surrounds the implanted conceptus. We are interested in understanding how the DSC phenotype is regulated at the transcriptional and epigenetic level in both normal and abnormal human pregnancy.  This project was motivated by our previous finding that a specific histone modification (H3K27me3) silences a large number (~800) of genes in mouse decidual stromal cells (Science 2012, J. Clin. Invest. 2018).  These genes appear in part to be ones that, if inappropriately expressed, might lead to pregnancy complications such as preterm labor. We thus wish to determine whether H3K27me3 silences a similar set of genes in human DSCs.  However, this is a long-term goal and a more immediate aim is to identify genes that are transcriptionally downregulated in human DSCs. We hypothesize that such genes, while silenced in early gestation, might be (re)-induced in late gestation as a part of the process that leads to labor induction.  Moreover, we hypothesize that premature induction of these genes might be a cause of preterm labor.  Accordingly, we are gathering together all published datasets on gene expression profiles of early and late gestation decidual tissue, with the hope that a thorough meta-analysis will allow us to discern key genes of interest for further investigation. The dataset we seek here is a single-cell RNA-seq analysis of the late gestation human decidua experiencing normal parturition timing or preterm labor.  \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Fisch, Kathleen \",\n\"UNIVERSITY OF CALIFORNIA, SAN DIEGO\",\n\"Comprehensive transcriptomic characterization of decidual leukocytes at the maternal-fetal interface at single cell resolution\",\n\"Apr16, 2021\",\n\"approved\",\n\" A successful pregnancy requires crosstalk between maternal immune cells and fetal-derived cells in the placenta (trophoblast). The goal of our research is to characterize these cells at the maternal-fetal interface in early and late pregnancy. Specifically in this project, we are interested in understanding the maternal immune cells and how these cells change as pregnancy progresses. \",\n\"Our research objective is to characterize the decidual leukocytes at the maternal-fetal interface. We propose to do this by single-cell RNA sequencing of both early and late gestation placenta. Our group has already sequenced early gestation decidua and the dbGaP dataset we are requesting access to are from term placental tissues. We will compare the leukocytes from each of the datasets to understand how the population changes throughout pregnancy. The requested dataset will also be used for validation of the datasets we plan to generate when we complete our single-cell RNA sequencing of term placenta.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Herse, Florian \",\n\"MAX DELBRUECK CENTRUM FOR MOLECULAR MED\",\n\"Placental cell atlas\",\n\"Sep03, 2020\",\n\"approved\",\n\"Preeclampsia (PE) is the sudden onset of high blood pressure after the 20th week of pregnancy. Severe PE causes intrauterine growth restriction (IUGR) of the fetus, preterm delivery or even stillbirth. PE is the leading cause of maternal and fetal morbidity and mortality. The underlying mechanism of PE is still unknown. However, it is evident that the placenta and its development (placentation) early in pregnancy plays a major role in the development of this disease. To analyze the underlying mechanisms in PE and to better understand human placentation, we recently conducted studies featuring placental tissues at single-cell resolution. These studies reveals the cell communication network as well as functionally distinct cell types of the maternal-fetal interface, namely placenta (tissue with fetal origin) and decidua (maternal tissue) and will describe human cell differentiation in first trimester and term placentas. Funded by the german research foundation \\u00e2\\u0080\\u009cDeutsche Forschungsgemeinschaft\\u00e2\\u0080\\u009d (DFG) (grants-No.: HE6249/7-1 and HE6249/7-2) our group plan to construct a placenta cell atlas resulting of first trimester and term placental and corresponding decidual tissue, including PE samples.\",\n\"Preeclampsia (PE) is the sudden onset of hypertension after the 20th week of pregnancy accompanied by proteinuria. Severe PE causes intrauterine growth restriction (IUGR) of the fetus, preterm delivery or even stillbirth. PE is the leading cause of maternal and fetal morbidity and mortality. The underlying pathomechanism of PE is still unknown. However, it is evident that the placenta plays a major role in the development of this disease. To analyze the underlying mechanisms in PE and to better understand human placentation, we recently conducted studies featuring placental tissues at single-cell resolution. These studies reveals the cell communication network as well as functionally distinct cell types of the maternal-fetal interface, namely placenta (fetal tissue) and decidua (maternal tissue) and describe human cell differentiation in first trimester and term placentas. Funded by the \\u00e2\\u0080\\u009cDeutsche Forschungsgemeinschaft\\u00e2\\u0080\\u009d (DFG) (grants-No.: HE6249/7-1 and HE6249/7-2) our group plan to construct a placenta cell atlas resulting of first trimester and term placental and corresponding decidual tissue, including PE samples: First trimester placenta control: 3 First trimester decidua control: 3 Term placenta control: 6 Term decidua control: 3 Term placenta PE: 3 Term decidua PE: 3  The data processing workflow consists of preprocessing via the CellRanger v3 workflow, followed by integration and comparisons using the R package Suerat v3, and further donstream analysis including differential expression, cell type abundance, receptor-ligand interactions and trajectory analysis.  Further bulk and single cell sequencing data from human tissue and cells will be integrated in the analysis to improve cell annotation and integrity of data.  \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Hillman, Sara \",\n\"UNIVERSITY COLLEGE LONDON\",\n\"Analysis of cell heterogeneity in human placenta \",\n\"Mar31, 2021\",\n\"expired\",\n\"During pregnancy there is a highly coordinated dialogue between mother and fetus and it is at the point of contact between both of them where very important communication is established to ensure the correct development of the baby. Recently, single-cell transcriptomics have allowed the study of the maternal-fetal interface at cellular resolution. We are requesting access to the data deposited at NIH dbGAP phs001886.v2.p1 to re-analyse the cellular composition of the tissues that encompasses the maternal-fetal interface using recent methods based on machine learning to classify cell types. The outcome of this proposal will increase our understanding of cellular composition at this important interface.\",\n\"The placenta is a dynamic and heterogenous organ including both maternal and fetal cell types. Single-cell RNA-seq has enabled the transcriptional profiling of this organ at single cell resolution [2-3, 5]. Using a combination of unsupervised clustering and label transfer, Pique-Regi et al., 2019 shown that there are 19 cell types in the basal plate, placental villous and chorionamniotic membranes.   Recently, new computational tools based on machine learning have been developed to probabilistically assign cell identities from sc-RNAseq datasets. We are requesting the libraries deposited at NIH dbGAP (accession number phs001886.v2.p1) to re-analyse the cellular composition in the placenta and benchmark some bioinformatics recent tools to classify cell types. Importantly, these tools have the potential of identifying novel cell types and resolve better the differences between term labor and pre-term labor.  Specifically, we will use supervised approaches (for instance Garnett[4] and CellAssign[6]) and unsupervised approaches (such as SCAFF[1]). We will analyse dbGap datasets separately and in combination with other datasets publicly available (Vento-Tormo et al., 2018).   The outcome of the proposed project will advance our understanding of the cellular composition at different trimesters during pregnancy and locations of the maternal and fetal interface. The analysis proposed will not create any additional risks to participants.  References  [1] Miao, Z., Nature Methods, 2020; 17(6), 621-628. DOI:10.1038/s41592-020-0825-9 [2] Pique-Regi R. Elife. 2019;8:e52004 DOI: 10.7554/eLife.52004 [3] Pique-Regi R. Elife. 2020; 9:e58716 DOI: 10.7554/eLife.58716 [4] Pliner, H. A. Nature Methods, 2019; 16(10), 983-986. DOI:10.1038/s41592-019-0535-3 [5] Vento-Tormo R. Nature. 2018;563(7731):347-53. DOI: 10.1038/s41586-018-0698-6  [6] Zhang, A. W. Nature Methods, 2019; 16(10),1007-1015. DOI:10.1038/s41592-019-0529-1 \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Huang, Frank \",\n\"CINCINNATI CHILDRENS HOSP MED CTR\",\n\"Identifying cell-type specific gene expression in preterm birth\",\n\"Apr28, 2020\",\n\"closed\",\n\"According to a recent investigation, more than 10 million preterm birth occur worldwide each year. We would like to analyze the data from women who are term and preterm parturition to find out the molecular mechanisms and risk genes that contribute to preterm birth.\",\n\"The placenta is of great significance for human parturition, while the cell types in the placenta are still puzzled, especially for preterm birth. Here we would like to apply for the access of single-cell RNA-seq data in the placenta. Our research mainly focuses on identifying the heterogeneous cell types of the placenta. We will also compare the gene expression of different cell types and their proposition between term and preterm birth sample. Moreover, we plan to apply the Seurat R package to study the cell type-specific differential expressed genes by comparing the single-cell RNA-seq data of term and preterm and identify if those genes are highly correlated to preterm birth. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"HURD, YASMIN \",\n\"ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI\",\n\"Single cell RNA-seq analysis\",\n\"Feb17, 2023\",\n\"approved\",\n\"We are looking to use molecular data from the human placenta, specifically mRNA expression from single cells in the human placenta, to deconvolve (more detail below) mRNA expression from whole placenta tissue. The data we are requesting has mRNA expression in single cells. The data our lab has is mRNA expression in whole placentas. By comparing the single-cell mRNA expression to our whole-placenta mRNA expression, we can estimate the percentages of different cell types in our whole-placenta mRNA expression, and this is what \\\"deconvolution\\\" means. The placenta contains a variety of cell types, and these cell types are labeled in the single-cell mRNA expression data we are requesting. By examining these different cell types in the requested data, we can find \\\"marker mRNAs\\\" that are selectively expressed in the different cell types in the placenta. With these marker mRNAs, we can estimate the percentages of the same cell types in our whole-placenta mRNA expression based on these marker mRNAs. Our whole-placenta mRNA expression comes from rats that were administered cannabinoids or no cannabinoids (i.e., control group). The data we are requesting will help us identify which cell types, if any, are affected in their placental expression by cannabinoids. \",\n\"Our goal is to use the human placenta single-cell RNA-sequencing data provided in this dataset for single-cell deconvolution of bulk placenta RNA-sequencing data generated by our lab. The purpose of this computational procedure is to use a single-cell RNA-seq reference dataset to estimate cell type fractions in bulk RNA-seq samples. Single-cell deconvolution is conducted using software packages such as dampened weighted least squares (DWLS). Our placenta RNA-seq samples were generated from rat dams that were exposed to cannabinoids during pregnancy. Our study is examining the effects of prenatal cannabinoid exposure on the placenta transcriptome, and part of our investigation is to understand how cannabinoids affect the abundance of different cell types. We plan to use the requested data as a reference single-cell RNA-seq dataset for single-cell deconvolution to obtain cell type fractions. These cell type fractions are calculated for each sample, and we can use these cell type fractions to measure differences in estimated cell type abundance in our bulk RNA-seq samples. Further, the first step of the single-cell deconvolution process is to identify marker genes of different cell types in a single-cell RNA-seq dataset. Using the provided data would allow us to identify marker genes in different placental cell types. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Isles, Anthony \",\n\"CARDIFF UNIVERSITY\",\n\"Investigating imprinted gene expression in the placenta at the single cell level\",\n\"May24, 2021\",\n\"expired\",\n\"We wish to study the activity of a group of genes called imprinted genes in the different cells of the placenta. We hope to use this dbGaP Single Cell Analysis of the Human Placenta dataset to identify in which specific cells of the placenta activity of imprinted genes is highest. This will give us an understanding of the possible role imprinted genes play in the placenta, and also hopefully help us understand what might be going wrong in disorders linked to imprinted genes, such as Prader-Willi syndrome. \",\n\"Genomic imprinting is the process whereby epigenetic imprints, established in the germline of the mother and father, survive epigenetic reprogramming in early development and are inherited by the somatic cells of the offspring. These imprints distinguish the maternally- and paternally-derived genomes and ultimately lead to some imprinted genes being expressed from the maternal allele only; whilst others are expressed from the paternal copy only. Imprinted genes have provided a template for understanding epigenetic processes throughout development generally. However, from a functional standpoint, although small in total number (~200 in the genome) imprinted genes, which are only found in mammalian animals (they are found in plants too), also pose a number of evolutionary questions and are also linked to several disorders, including Prader-Willi syndrome. My lab is interested in understanding the function of imprinted genes, and this project in particular takes advantage of a the growing number of single RNA-seq datasets to examine the enrichment of canonical imprinted gene expression as a group in the mouse and human tissues. These analyses identify \\u00e2\\u0080\\u0098expression hotspots\\u00e2\\u0080\\u0099 where strongly expressed imprinted gene are significantly over-represented in the transcriptome (i.e. more than by chance alone), which in turn suggests \\u00e2\\u0080\\u0098functional hotspots\\u00e2\\u0080\\u0099. We would like access to the dbGaP Single Cell Analysis of the Human Placenta dataset (phs001886.v2.p1) in order to examine imprinted gene expression and enrichment in the placenta. Although there are a number of studies examining imprinted gene expression in bulk placenta samples, placental cell type-specific RNA references are limited. Our analyses will involve screening the transcriptomes of different cell types (as identified and defined in Pique-Regi et al  eLife 2019;8:e52004) for the presence of imprinted gene expression, then comparing the relative proportion of imprinted genes in each cell type to establish if these genes, as a group, are enriched in any cell types specifically (see https://doi.org/10.1101/2020.07.27.222893 for methodological details).\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"KALLAPUR, SUHAS \",\n\"UNIVERSITY OF CALIFORNIA LOS ANGELES\",\n\"Single cell transcriptomics of the fetal membranes and placenta\",\n\"Feb01, 2022\",\n\"closed\",\n\"We are requesting access to the dataset so that we can compare our data that we will generate in Rhesus monkey placenta with those from human. This will ensure rigor of study and avoid duplication.\",\n\"We have performed single cell RNA seq analyses of preterm pregnant Rhesus macaques. We are requesting access to the human transcriptomic data set to compare our results in the Rhesus with those of human. We do not plan to combine datasets. If there are important data that are correlated, we may publish the human data finding as confirmatory. In such a case we will acknowledge the source and the publication. The published data will be of a global nature and not specific to any individual or study participant. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Koyuturk, Mehmet \",\n\"CASE WESTERN RESERVE UNIVERSITY\",\n\"Identification of causal drivers of labor in human pregnancy\",\n\"Aug24, 2020\",\n\"approved\",\n\"The process of parturition involves the transformation of the quiescent myometrium (uterine smooth muscle) to the highly contractile laboring state.  This is thought to be driven by changes in gene expression in myometrial cells.  Despite the existence of multiple myometrial gene expression studies, the gene expression changes that initiate labor are largely unknown.  This makes it difficult to understand and treat labor disorders such as preterm labor.   We will use this large-scale and high-resolution dataset to identify the drivers of labor in term and pre-term pregnancy. Such markers can potentially lead to the development of therapies for preventing and/or controlling pre-term birth, a public health problem that affects millions of infants and their families, many throughout their entire lifespan.\",\n\"While the interplay between progesterone signaling, inflammatory processes, and cAMP signaling in the onset of labor is relatively well-understood, the key drivers of this process are yet to be characterized. We are developing algorithms to construct functional state descriptors using single cell data and making causal inference to identify key drivers of cell state change using these functional state descriptors. As part of our ongoing collaboration with Dr. Sam Mesiano (Case Western Reserve Dept. of Reproductive Biology), we will use this dataset to identify functional state descriptors for cells representing different tissues, different cell types, and different times (labor vs. non-labor, term vs. pre-term).  We will then use counterfactual based causal inference to identify key drivers that drive state change. The high-resolution provided by this dataset will enable application of causal inference algorithms from observational data.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Li, Bin \",\n\"UNIVERSITY OF SCIENCE AND TECHNOLOGY OF CHINA\",\n\"Transcriptional landscape of the human placenta in ageing\",\n\"Aug13, 2021\",\n\"expired\",\n\"Older women are at high risk of miscarriage. Therefore, studying the changes in the maternal-fetal interface during aging is important to reveal age-related risks. We plan to apply the data from this application (phs001886.v1.p1) to this study to reveal changes in the placenta during aging.\",\n\"Exploring the cell type composition and function of human placenta is of great significance for studying the conditions required for a successful pregnancy. The risk of miscarriage in older women is very high. At present, no one has compared the transcriptome changes in the placenta of women during aging. We plan to sequence the transcriptome of the placenta of the young and old mothers and compare their differences. We intend to use the applied data (phs001886.v1.p1) as a control group.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Li, Jingjing \",\n\"UNIVERSITY OF CALIFORNIA, SAN FRANCISCO\",\n\"Functional Validation of Genetics Variants in Spontaneous Preterm Birth Using Single-Cell Genomic Data\",\n\"Feb06, 2020\",\n\"approved\",\n\"Preterm birth (PTB) affects ~11% newborns in the nation, and has a strong genetic basis. We have recently identified many new genes implicated in PTB, and we herein request this single-cell gene expression data to study the biological functions of our newly identified PTB-associated genes. Specifically, we will ask what specific cell types these genes expression, and whether these genes will be dysregulated in PTB clinical samples. This study will help identify novel mechanisms in PTB.\",\n\"Preterm birth (PTB) affects 15 million births worldwide annually, which is the leading cause of neonatal mortality and morbidity. PTB has a strong genetic component; however, its genetic basis has been largely unclear. We recently studied large-scale genome-wide association study (GWAS) datasets, developed a set of novel algorithms to integrate GWAS and many other functional genomic datasets, and identified candidate loci in PTB. As an immediate step, we need to experimentally validate these associated genes for their roles in PTB. We therefore request this dataset to study gene expression of the placenta at a single-cell resolution. For our identified genes in PTB, we will use this dataset to identify their expression specificities in particular cell types, and also identify their differential expression in PTB. This study will identify novel genetic elements in PTB, and will help foster developing prognostic and treatment strategies for PTB.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Male, Victoria \",\n\"U OF L IMPERIAL COL OF SCI/TECHNLGY/MED\",\n\"Changes in number and gene expression of decidual immune cells over gestation and in labour\",\n\"Sep03, 2020\",\n\"approved\",\n\"Preterm birth and complications from it are the leading cause of death in children under 5 worldwide. Approximately a third  of preterm births are classified as idiopathic \\u00e2\\u0080\\u0093 meaning none of the common reasons for preterm labour are present. Therefore, there is a lot to learn about the mechanisms of preterm labour and doing this will help design therapeutic strategies to delay labour.  Immune cells in the lining of the uterus (the decidua) may be involved in initiating labour and those in the muscle of the uterine wall (myometrium) may promote labour once it has begun. We want to take a closer look at the differences in the immune cells present in the decidua and myomedtrium of women in labour at term, compared to those who are not in labour, to learn more about the normal term labour process. We also want to compare term labouring patients to preterm labouring patients to look for differences in the preterm labour process. \",\n\"Our research interests lie in the immune populations present at the maternal-fetal interface and how they change in the term and preterm labour process. Currently we are studying this at a protein level in the decidua basalis, decidua parietalis and myometrium, and wish to access the phs001886.v1.p1 dataset to analyse complementary data at the scRNAseq level. We will use the R package Seurat to analyse the dataset.  Our goal is to reanalyse the data focusing on the maternal immune cell populations, and to determine whether broad immune cell types identified by Pique-Regi et al could be divided into meaningful subsets. This would be compared to immune cell populations identified by scRNAseq in the first trimester (Vento-Tormo, Nature 2018), to define changes in immune populations over the course of pregnancy and determine whether the subset classifications found in first trimester (dNK1, 2, 3 & M1, 2) are relevant at term. No T cell subsets were defined in the first trimester data set, potentially due to T cells being a small population early in pregnancy. This term data set would give us a higher resolution of the T cell population present in the decidua basalis and chorioamniotic membranes. Lastly, data presented in Pique-Regi et al suggested there may be some changes of immune populations in term labour, compared to term not in labour, and preterm labour compared to the term groups. We hope to confirm this, and identify which specific subsets of cells are involved in these. As well as considering the proportions of cell subsets present in each of these cases, we would also look for differentially expressed genes to see how each subset changes from first trimester to term, and between non-labouring and labouring patients.  The proposed reanalysis of the dataset would provide important information about how immune cells in the decidua differ between the first and third trimesters, and how maternal immune cells in the decidua and myometrium may be involved in labour at term and preterm.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Martins Jr, David \",\n\"FUNDACAO UNIVERSIDADE FEDERAL DO ABC - UFABC\",\n\"Impact of COVID-19 during pregnancy on infant neurodevelopment\",\n\"May09, 2023\",\n\"approved\",\n\"COVID-19 during pregnancy increases the risk of negative outcomes, such as maternal death. The in-utero exposure to some other viral infections cause long-lasting neuropsychiatric alterations. We do not know yet if gestational COVID-19 affects the offspring\\u00e2\\u0080\\u0099s neurodevelopment. This project aims at identifying genes related both to gestational COVID-19 and foetal neurodevelopment. First, we have searched the scientific literature for studies about gestational COVID-19 that provide large-scale protein and gene expression. Our search retrieved 12 high-quality, unique studies. We will re-analyse these datasets individually, since different studies use distinct methodologies, and we must assure that they are comparable. After that, for each study, we will have lists of proteins/genes altered in gestational COVID-19. Then, for each organ (cord blood, placenta, or maternal blood), we will integrate the datasets and find the proteins/genes altered in several studies. We will also find factors altered across distinct tissues. Departing from that, we hypothesize that these genes are altered in foetal brain as well. We will study their expression in foetal brain across the gestation. This analysis may help us understand the neurodevelopmental effects of gestational COVID-19.\",\n\"SARS-CoV-2 infection during gestation increases the risk of negative outcomes, such as maternal death and pre-eclampsia. Although transplacental SARS-CoV-2 transmission is very rare, gestational COVID-19 can still affect the foetus through maternal immune activation and other mechanisms. The in-utero exposure to some viral infections, such as the zika virus disease, cause long-lasting neurological and psychiatric alterations. There is no scientific consensus on whether gestational COVID-19 may affect the offspring\\u00e2\\u0080\\u0099s neurodevelopment, and little is known about its long-term effects. This RUS describes a subproject that aims at identifying protein-coding genes related both to gestational COVID-19 and foetal neurodevelopment. The subproject is part of a larger study that combines a clinical study, RNA-sequencing and bioinformatic analyses. First, we have performed a systematic review to identify proteomics and transcriptomics datasets on pregnant people with COVID-19 and their offspring. The inclusion criteria were: 1) The subjects must be human pregnant people, or neonates whose mothers had the investigated condition; 2) The investigated condition should be SARS-CoV-2 infection during gestation; 3) The study must provide original transcriptomics or proteomics data, produced through any methodology. Our search retrieved 194 records. After removal of duplicates, quality assessment and abstract and full-text screening, we remained with 12 records. This includes the dataset requested in this RUS, and other datasets.  Currently, we are in the data extraction phase, which attempts to obtain raw -omics data, or protein/gene frequency matrices. After obtaining the data, we will re-analyse them to identify differentially expressed genes and proteins, and the p-values of these altered factors. For each type of biological sample (cord blood, placenta or maternal peripheral blood), we will integrate the p-values from distinct studies with Liptak\\u00e2\\u0080\\u0099s weighted z-score. Therefore, for each type of sample, we will have a list of factors consistently altered in various studies. After that, we will intersect the lists from different samples and identify factors that may be consistently altered across distinct tissues. Departing from that, we hypothesize that these genes may be potentially altered in foetal brain as well. We will study their expression in foetal neural tissues across the gestation, using public transcriptomics datasets. We will use an algorithm developed by our research group that uses weighted graphs. This analysis may provide an insight on the potential neurodevelopmental effects of gestational COVID-19, and whether it may have an association to foetal sex and trimester of infection.  The re-analysis of the requested dataset, as well as its integration to other datasets, will not result in additional risk to the participants.  The external collaborators of this project are:  - Prof. Dr. Helena Brentani from Institute of Psychiatry - University of S\\u00c3\\u00a3o Paulo - MSc. Maria Laura Gabriel Kuniyoshi from Institute of Psychiatry - University of S\\u00c3\\u00a3o Paulo (PhD student advised by Helena Brentani and coadvised by David Correa Martins Jr) - Prof. Dr. Sergio Nery Sim\\u00c3\\u00b5es from Federal Institute of Esp\\u00c3\\u00adrito Santo  The collaborators aforementioned do not have access to the download dataset, so they do not need to file a request.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Merkus, Daphne \",\n\"ERASMUS MEDICAL CENTER\",\n\"Placental expression of kynurenine pathway enzymes\",\n\"Dec30, 2020\",\n\"closed\",\n\"Tryptophan is metabolized through the kynurenine pathway (KP). Indoleamine 2,3-dioxygenase (IDO1) is the first and rate-limiting enzyme of this pathway, and is highly expressed in healthy term placentas. In preeclampsia, placental IDO1 expression is decreased. This impaired metabolism of tryptophan by IDO1, and other KP enzymes might contribute to deficiencies in vascular regulation, immune regulation, and tolerance. However, there is a lot of controversy regarding the exact location of placental IDO1 expression, and we speculate that the function of IDO1 might depend on the cell type it is expressed in. Therefore, we would like to use this data base to explore which placental cells express KP enzymes.\",\n\"Objectives Tryptophan is metabolized through the kynurenine pathway (KP). The first and rate-limiting enzyme is indoleamine 2,3-dioxygenase (IDO1), which is highly expressed in healthy term placentas, but decreased in preeclampsia. Impaired tryptophan metabolism by IDO1, and other KP enzymes might contribute to deficiencies in vascular regulation, immune regulation, and tolerance. We speculate that the function of IDO1 might depend on the cell type it is expressed in. However, the exact location of placental IDO1 expression remains controversial. Based on publicly available data from first trimester placentas by Vento-Tormo et al.1, we have found that IDO1 is mainly expressed in endothelial cells, dendritic cells, monocytes and macrophages. However, IDO1 expression is known to increase throughout pregnancy, and it is therefore essential to confirm these result in term placentas. With the present requested dataset we aim to translate our results from first trimester to term, and preterm delivered placentas. Investigating cell type specific expression of KP enzymes can help us to direct future studies, to investigate cell type specific alterations in placental pathologies (eg preeclampsia), and give a functional meaning to the placental KP. Study design & analysis plan For the requested data set, we only need a count table which shows the expression of all genes per cell. The cells will be sorted based on expression of the enzyme of interest, and all negative cells (0 RNA counts per cell) will be excluded. Next, we will explore which cells are left in the positive fraction. For the phenotyping we would like to use the exact same cell types as described in the study for which we request access (Pique-Regi et al. eLIFE 20192). Adherence to the data use limitations Our study is consistent with the data use limitations as we work from a non-profit organisation. When sharing our data we will acknowledge the original authors of the requested dataset and the funders of their research. Collaborations We do not have any plans on collaboration on our own project. We have used/will use databases from three other studies1, 3, 4 to confirm our results.  References 1. Vento-Tormo R, Efremova M, Botting RA, Turco MY, Vento-Tormo M, Meyer KB, Park J-E, Stephenson E, Polanski K, Goncalves A, Gardner L, Holmqvist S, Henriksson J, Zou A, Sharkey AM, Millar B, Innes B, Wood L, Wilbrey-Clark A, Payne RP, Ivarsson MA, Lisgo S, Filby A, Rowitch DH, Bulmer JN, Wright GJ, Stubbington MJT, Haniffa M, Moffett A, Teichmann SA. Single-cell reconstruction of the early maternal\\u00e2\\u0080\\u0093fetal interface in humans. Nature. 2018;563:347-353 2. Pique-Regi R, Romero R, Tarca AL, Sendler ED, Xu Y, Garcia-Flores V, Leng Y, Luca F, Hassan SS, Gomez-Lopez N. Single cell transcriptional signatures of the human placenta in term and preterm parturition. Elife. 2019;8 3. Tsang JCH, Vong JSL, Ji L, Poon LCY, Jiang P, Lui KO, Ni YB, To KF, Cheng YKY, Chiu RWK, Lo YMD. Integrative single-cell and cell-free plasma rna transcriptomics elucidates placental cellular dynamics. Proc Natl Acad Sci U S A. 2017;114:E7786-E7795 4. Pavlicev M, Wagner GP, Chavan AR, Owens K, Maziarz J, Dunn-Fletcher C, Kallapur SG, Muglia L, Jones H. Single-cell transcriptomics of the human placenta: Inferring the cell communication network of the maternal-fetal interface. Genome Res. 2017;27:349-361\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Paik, Hyojung \",\n\"KOREA INSTITUTE/SCIENCE/TECH/INFORMATION\",\n\"Single-cell analysis of preeclampsia causing preterm birth\",\n\"Jan04, 2022\",\n\"closed\",\n\"The most distinctive feature of the placenta is the existence of cells with two or more heterogeneous origins: fetal and maternal cells in a single tissue. Therefore, de-multiplexing analysis according to cell origin is required for the identification of fetal and maternal characteristic differential genetic expression. Preeclampsia is a common obstetric disorder and is one of the causes of preterm birth, the integration of our preterm birth data and phs001886 data will enable the identification of the etiology of preeclampsia causing preterm birth.\",\n\"Preeclampsia is a common obstetric disorder and a leading cause of maternal and perinatal morbidity. Most of the adverse pregnancy outcomes in people with preeclampsia occur before the 34 weeks' gestation. Therefore, it is essential to identify preeclampsia-associated biomarkers in preterm that precede the previously known late-term markers of preeclampsia. We recently generated placental single-cell RNA-seq data from normal full-term pregnancies (37 weeks) and preeclampsia-induced preterm births (32 weeks), and confirmed the differences in immune cell population characteristics. However, since it was difficult to determine whether these results were due to preeclampsia or premature birth, we intend to analyze this by integrating the data of phs001886 with our dataset. To the integrated dataset, we will apply CellRanger-based quality control and Seurat-based downstream pipeline to annotate cell clusters and perform differential expression gene analysis.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Palin, Kimmo \",\n\"UNIVERSITY OF HELSINKI\",\n\"Mutational Processes and Genetic Predisposition in Colorectal Cancer and Other Tumor Types\",\n\"Nov29, 2022\",\n\"approved\",\n\"Each year, approximately 2600 Finns are diagnosed with colorectal cancer and fifth of them will ultimately die for it. Finland is a small country in the north eastern edge of Europe with exceptionally well characterized population history which is well suited for genetic risk studies. We are studying genetic colorectal cancer risks in Finnish population and will use data from dbGAP to evaluate the global effects of the discovered risk loci. Furthermore we study the somatic variations within the tumors to see what happens to a normal cell when it turns cancerous. \",\n\"Each year, approximately 2600 Finns are diagnosed with colorectal cancer and fifth of them will ultimately die for it. CRC risk has a substantial heritable component and total of 20 genetic loci are known to be associated with it in outbred population. We study genetic germline predisposition for CRC and other tumor types of interest in the isolated founder population of Finland in order to find rare (and common) variants associated with tumor predisposition.  In addition to germline predisposition, we will study molecular mechanisms and effects of somatic changes such as mutational patterns in different parts of the genome and signals driving excess proliferation.  All these studies will require development of novel methods which will be developed and tested by using the requested data.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Salomonis, Nathan \",\n\"CINCINNATI CHILDRENS HOSP MED CTR\",\n\"Understanding the mechanisms of preterm labor\",\n\"Sep22, 2021\",\n\"approved\",\n\"To understand the precise mechanisms of preterm birth we have developed mouse models of preterm labor which nominate new pharmacological targets for therapy. These analyses will help verify their predicted importance and relevance to human biology.\",\n\"We wish to simply use the non-identifiable count-matrix level data and previously produced cell-to-cluster associations to validate predictions from a mouse model of preterm label, to validate human relevance relative to bulk RNA-Seq (wild-type and KO). We will perform differential expression comparing term and preterm populations in the human samples and evaluate overlapping changes in mouse.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Schwartz, Robert \",\n\"WEILL MEDICAL COLL OF CORNELL UNIV\",\n\"Analysis of cell types in the placenta.\",\n\"Sep18, 2020\",\n\"approved\",\n\"The placenta is important and the cell constituents that give rise to the placenta are important to study. Several diseases impact placental formation, cell composition, and function. We plan to examine the placenta for changes in cell composition and gene expression in several disease processes.\",\n\"The placenta is of great significance for human parturition, while the cell types in the placenta are still relatively understudied. We are currently examinging and performing scRNAseq on the placenta and want to reeveluate and confirm our findings with an established dataset. CMV is a common viral infection, especially among young children. Congenital CMV occurs when the infection is passed across the placenta from a pregnant woman to her developing baby. Some babies with congenital CMV infection show signs of disability at birth, while others are born healthy. Other babies who do not have evidence of congenital CMV infection but do have evidence of placental abnormalities or malformations. Our proposed research requires using the dataset of healthy (normal) placenta to identify the heterogeneous cell types present and present a normal baseline gene expression for each cell type.  Once we have single cell RNA sequenced CMV-infected placenta, we intend to use markers gained from identifying cell types in this dataset to annotate the cell types in our own infected single cell data and thus once this is achieved also compare and produce differential gene expression in each cell type between infected and non-infected cells and also the cell type composition over time points pre-birth. The main data we will be using will be gene count matrices which will be analyzed in Seurat R package along with our own counts data. Processed data in terms of differentially expressed genes, UMAP plots and cluster markers will be accessible only to internal collaborators. All internal WCM collaborators are listed on this application. There are no external collaborators.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Straughen, Jennifer \",\n\"HENRY FORD HEALTH SYSTEM\",\n\"Characterize DNA Methylation alterations in placenta and their associations with gestational age\",\n\"Dec16, 2020\",\n\"approved\",\n\"The genomic and molecular characterization of placenta tissues is crucial to understand how placenta functions are related to childbirth outcomes. The traditional bulk mRNAseq measures gene expression of a mixture cells and placentas contains various types of cells. It is critical to characterize cell type specific expressions to enhance our understanding of placenta tissues. We will use single cell sequencing data from phs001886 as references to estimate cell populations in bulk RNA sequencing through mathematical algorithms. \",\n\"We are interested in characterizing molecular and genomic changes in placenta tissue with respect to gestational age and environmental exposures. Specifically, we have a project to study how placental changes in DNA methylation may mediate the effect of environmental exposures on gestational age at delivery. This is a recent field of study, and while it is appreciated from other areas of research that cellular heterogeneity can impact genomic studies of particular tissue types, very little information on this effect on genomic studies of placenta is available, despite the extreme heterogeneity in this vital developmental organ. In combination with another dbGaP dataset with bulk placental methylation and RNA-Seq data (phs001717), we intend to use the data from phs001886 to gain insight into the heterogeneous nature of placental tissue. Specifically, we will use this data to achieve two scientific objectives. First, using methods including CIBERSORT and MuSIC, we will use the placental single-cell RNA-Seq data from phs001886 as a reference to deconvolute the bulk placental RNA-Seq data from phs001717 and identify the individual cell types and their corresponding percentages in that data. Second, we will utilize these results to inform a deconvolution of our placental methylation data to explore how the heterogeneity in placental cellular subtypes may influence association analyses of methylation alterations and gestational age at delivery.  It is our intention to present all our analyses and findings of our data and the data from phs001886 to the scientific community. We will present these results in both national/international conferences and peer-reviewed scientific journal publications. This work will greatly boost our understanding of how biological changes in placenta impact preterm birth. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"St\\u00c3\\u00a5hl, Patrik \",\n\"KTH-ROYAL INSTITUTE OF TECHNOLOGY\",\n\"Spatial analysis of human placenta in pre-eclampsia\",\n\"Nov10, 2020\",\n\"closed\",\n\"Preeclampsia complicates about one in twenty pregnancies. This disease is associated with an increased risk of morbidity and mortality for both mother and baby. At present, we do not know what is causing preeclampsia. It is believed that poor placentation is a contributing factor behind the disease. What is known is that there are several changes in the placenta with preeclampsia. The placenta itself is a facinating organ with many internal structures. We have investigated how genes are expressed in different parts of these structures and will compare with the single-cell data to learn more of how the placenta is affected by preeclampsia.\",\n\"Preeclampsia (PE) complicates about 2-8% of all pregnancies worldwide. As such it is a major obstetric problem, with a high prevalence of maternal and perinatal mortality/morbidity. Although the underlying cause of preeclampsia is unknown, poor placentation has been suggested as an important contributing factor, especially in early-onset cases. Several alterations in the placenta has been reported with preeclampsia. Much of these results are based on histology or tissue homogenates, potentially missing alterations present in different parts of the villous tissue. Recently a method to determine spatial gene expression in tissues has been developed at KTH Royal Institute of Technology (Sweden). We have utilized this method on placental tissue from healthy pregnancies and pregnancies complicated by preeclampsia. We would like to request access to this dataset to compare single cell gene expression with our data on spatial transcriptomic gene expression, to validate/identify cell types detected in our data set.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"TUSCHL, THOMAS \",\n\"ROCKEFELLER UNIVERSITY\",\n\"Curated Human Transcriptome\",\n\"Aug17, 2020\",\n\"approved\",\n\"Many issues currently confound the quantification of Gene Expression. As yet,  reference transcript sets are not completely non-redundant (there are many issues of multi-mapping), they do not have a representative one transcript per gene (multiple transcripts are proposed to cater for alternate splicing) and variant nucleotide positions are computationally predicted and so tend to be incorrect fairly frequently. Accurate and unique mapping of sequence fragments from RNA sequencing datasets uniquely to transcripts is absolutely critical. And towards that goal, it is essential to have a well-curated and well-annotated set of transcript sequences and all their variants. We propose to create such a Curated, Annotated, Comprehensive, Non-Redundant, Coding and Non-Coding Human Reference Transcriptome. Accurate Gene Expression reports (and normal Gene Expression ranges of all genes across all known samples) and displays of alignments and resolving any multimapping is absolutely critical for making any biological hypotheses\",\n\"Mine datasets in the public domain to curate and annotate a Human Reference Transcriptome: As more diverse tissues and cell types are sequenced, genes that are expressed in rare conditions are available for curation and mining. Based on read evidence, allelic variations and junctions are created using our software platform, UTRs and ORF are annotated for coding genes. Precursors, mature non-coding sequences and special features like loop structures and other secondary structures are annotated on the sequence. We would be sequentially mining each dataset in our request separately for alternate splicing and allelic variations to enrich our reference transcriptome by extending our catalog of genetic variation and annotated transcripts. The datasets will be analyzed separately but kept together in one RU-located database. We are mining common (non-pathogenic) sequence variation and splice variation to complete a curated reference genome (and do not intend to discover genetic diseases outside dataset specific user agreements). The RU-located database is kept in a secure supercomputing cluster at RU. This repository contains primary sequence reads from controlled access RNA-seq datasets that have undergone minor processing. The processed and trimmed reads from the dbGAP datasets are made available for viewing one gene at a time using a small program; complete datasets are never exposed. This repository complies with NIH security guidelines. There are no collaborators.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"TUSCHL, THOMAS \",\n\"ROCKEFELLER UNIVERSITY\",\n\"Reanalyze single-cell data to better resolve cell types of the placenta\",\n\"Jul27, 2020\",\n\"closed\",\n\"Reanalysis of cell composition of placenta at different stages of pregnancy in preparation to evaluate if 2nd trimester intermediate state deserves future experimental analysis\",\n\"We wish to reanalyze single-cell and RNA-seq placenta data of the 3rd trimester for comparison to data we have generated for 1st and 2nd trimester. To best compare scRNA datasets they need to run through the same pipeline or be combined and then run through the same analysis. The statement above was requested to be expanded upon 1st review. In order to compare two single cell studies performed by 2 investigators at different sites, it is important to use the same bioinformatic pipeline for mapping sequence reads as well as setting thresholds to identify cell types and subtypes, e.g. separating cytotrophoblasts from syncytiotrophoblasts. also, we developed a analysis process, which requires multiple reads to support the unique molecular identifier (UMI) when counting transcripts per cell. in short, when comparing 2 datasets, both of which are published, access to primary sequencing data is necessary. we were planning to conduct a single-cell sequencing study using 2nd trimester placenta, and first determining if there were differences between 1st and 3 rd trimester placenta cell types is a critical step required to be shown as preliminary data in a NIH grant application for study of 2nd trimester placenta.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"van Rijn, Bas \",\n\"ERASMUS MEDICAL CENTER\",\n\"Placental expression of trophoblast subtypes\",\n\"Sep12, 2022\",\n\"approved\",\n\"Most children are born with a normal birth weight and have a healthy start to life. However, each year about 5% of children have not grown very well during pregnancy and are born too small and often too early. This is due to a problem with an organ called the placenta. The placenta is the organ by which the unborn baby makes contact with the mother and can extract food components and oxygen. In children that are born too small, the placenta has not developed very well. However, we do not fully understand how the cells in the placenta are formed in normal placentas and abnormal placentas. In this project, we compare data from analysis of cells that have been harvested from the placenta to figure out how these cells are formed during pregnancy. \",\n\"Regenerative capacity of the placenta relies on coordinated differentiation and growth of trophoblast subtypes. For this project we would like to investigate the proportions of trophoblast subtypes during gestation. The requested dataset allows for analysis of trophoblast subtypes which we will compare with our in-house sequencing data of first trimester placenta.  \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Vlachos, Ioannis \",\n\"BETH ISRAEL DEACONESS MEDICAL CENTER\",\n\"Effect of genetic, genomic, and microbiomic variation on complex traits\",\n\"Mar20, 2023\",\n\"approved\",\n\"Information is often lost in extensive cohort studies that can prove crucial to our understanding of the effects of the genotype on numerous quantitative traits and to complex disease phenotypes. We will use established and novel techniques that will enable the characterization of processes and traits across cohorts, uncover their in-between interactions and their associations with complex phenotypes\",\n\"In this project we aim to identify the effects of human variation on genetic, genomic and microbiomic level on complex phenotypes. We will perform this study by employing established techniques to assess the effects of genotype on different quantitative traits or develop new methods that can quantify and assess the associations between genotype and traits, as well as between the quantitative traits and the complex phenotype. We are requesting data from different study cohorts that will enable us to design, validate, and test our methods across phenotypes and populations.  The datasets will be used independently, combined and/or integrated. We will focus mostly on traits derived by analyzing RNA processing events, non-coding regulation, and microbiome profiling. The main research questions are:  a) Which are the genetic variants controlling quantitative traits that are until now lost in most analyses? How are they connected with the phenotype? b) Can we use common variants to reduce the search space for coding and non-coding functional events or loci? c) Can we identify interactions between different quantitative traits in order to maximize our understanding of the underlying complex phenotypes?  Access to the data will be limited only to members of the Vlachos Lab and collaborators who have completed human subjects and data safety training. All data users will agree to terms of data use certification restrictions and limit analyses accordingly and honor all publication embargo dates. Findings will be disseminated widely and shared with the scientific community by presenting results at national scientific meetings and publishing in peer-reviewed journals. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Wang, Kevin \",\n\"STANFORD UNIVERSITY\",\n\"long noncoding RNA (lncRNA) regulation of human preterm labor \",\n\"Feb01, 2022\",\n\"approved\",\n\"The mechanisms that trigger preterm labor have remained largely unknown, leading to a limited number of treatment options. Molecules called cytokines that are produced by the body\\u00e2\\u0080\\u0099s immune system in response to stress during pregnancy are thought to play important roles to jumpstart the uterus and the contraction process, leading to preterm labor. Understanding how cytokine release and control are regulated is critical to the development of better therapies. We have discovered a new class of molecules called long noncoding RNAs (lncRNAs) that play critical roles in the regulation of cytokine signals in humans. We will leverage the dbGAP data to identify and validate the existence and abnormal expression of these lncRNAs and begin to characterize how they affect the immune response.\",\n\"We are interested in analyzing preterm labor (PTL) datasets generated from different cell types like Whole blood, Monocytes, and chorioamniotic membrane to understand the cell type specific immune regulated changes happening in PTL condition and try to uncover long noncoding RNAs (lncRNAs) that change in expression through their interactions with epigenetically regulated immune genes in 3-dimensional chromatin space. We would like to reprocess this scRNA data to examine cluster specific differences. We will also compare and correlate the Monocyte cluster with the bulk RNAseq data from the Monocytes in order to identify Monocyte specific lncRNAs that are differentially expressed and regulated along with the differential immune genes.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Worthey, Elizabeth \",\n\"UNIVERSITY OF ALABAMA AT BIRMINGHAM\",\n\"Understanding the molecular underpinnings of chorangiomas of the placenta.\",\n\"Mar20, 2023\",\n\"approved\",\n\"Chorangiomas are benign placental capillary lesions associated with significant alteration of the placenta when they become extensive increasing risk of polyhydramnios, heart failure, growth restriction of the fetus, and/or preeclampsia in the mother. Despite the clear importance of these lesions, their molecular underpinnings are not understood. The relationships between development or expansion of chorangiomas, and adverse outcomes for mother or child remain unknown and are poorly studied. Only a single study of large chorangiomas has been performed and it identified no characteristic molecular variation. There are no early and accurate detection methods and treatments of these lesions are limited and varied in effect. The main objective of the research proposed is to use these dbGAP datasets as part of similar studies we are performing with internal cases to study the molecular underpinnings of chorangiomas. We use a multi-omics approach to identify and characterize deleterious molecular changes associated with development, vascularization, and growth in chorangiomas. We believe that our findings may also shed some light on mechanisms in preeclampsia and potentially other vascular tumors as these occurrences share overlap with chorangioma development.\",\n\"The main objective of the research proposed is to use these dbGAP datasets as part of similar studies we are performing with internal data. We are studying examples of chorangiomas identified within our affiliated women\\u00e2\\u0080\\u0099s and children\\u00e2\\u0080\\u0099s hospital referred to a collaborating pathologist. Our research focuses on analysis of these samples to identify molecular alterations and mechanism characteristic in chorangiomas. These generally benign placental capillary lesions are associated with significant hemodynamics alteration of the placenta when they become extensive increasing risk of polyhydramnios, heart failure, growth restriction of the fetus, and/or preeclampsia in the mother. Despite the clear importance of these lesions, their molecular underpinnings are poorly understood. We use a multi-omics approach to identify and characterize deleterious molecular alterations. We hypothesize that replication, vascularization, and growth pathways are altered in chorangiomas and share overlap with preeclampsia and potentially other vascular tumors.  We have already identified a number of variants and mutational signatures that we believe to be chorangioma specific and wish to use these external dbGAP datasets to validate and add to our findings. For instance, we will make use of transcriptome data requested to better understand the non-chorangioma expression landscape of the placenta as a control for our in-house generated data. We will also make use of the dbGAP expression data to compare and contrast regulatory networks from non chorangioma data with the networks already identified from our internal chorangioma and unaffected tissue. Utilization of the dbGAP methylation datasets will be used to identify associations between altered methylation sites and loci identified from internal and external genomic and expression data. We believe that by better understanding these distinct mutational signatures, the regulatory and others networks present in health and affected tissue, and the differences between these samples we will be able to advance our understanding of both healthy and chorangioma associated diseases processes in this critical and understudied organ. No individual participants will be identifiable from the analysis that we perform. We do not believe our project represents additional risks to participants. Our project will comply with all data use limitations for dbGaP studies. We will not combine datasets, but rather use one to validate findings from the other. Genetic ancestry will be considered in cleaning of the data and checking for sample relationships, but will not be a focus or primary outcome of our study. We will not seek or pursue findings that are beyond the respective scope of each data set. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Zhang, Liye \",\n\"SHANGHAITECH UNIVERSITY\",\n\"Explore human placenta in term and preterm parturition.\",\n\"Apr16, 2021\",\n\"closed\",\n\"Parturition is essential for the reproductive success of viviparous species, yet,the mechanisms responsible for the onset of labor remain to be elucidated. Clinically,preterm is common and may post danger to the newborn. Therefor, this project mainly wants to explore the molecular mechanism of preterm on the level of single cells and aims to find some molecular markers for the clinical diagnosis. We are also interested in comparing the difference in preterm birth between Asian and Caucasian groups. \",\n\"Research Objectives: Explore the mechanism of preterm birth and compare the single cell population of Asian and Caucasian population of preterm birth. Given the difficulties in obtaining the placenta samples, we want to reanalyze the data generated in this dataset and compare it with our small cohort on preterm birth in China.   Study design and analysis plan 1. Intergrate data with our own single cell data and validate whether some pattern we observed in our data is also observed in this dataset. In our datasets, we observe distinct population structure in different preterm birth cases. We want to check whether some of the patttern we observed in our sample is also observed in this dataset. For example, if we observed a increase in cell type X in our dataset in our preterm patient. We want to check whether such pattern can be observed at any patient in this dbgap dataset.   2. Compare aneuploidy of placenta between preterm and in-term. Recent study in Nature suggest common aneuploidy in human placenta. Given it is possible to infer aneuploidy based on the results in cancer, we want to analyze whether the aneuploidy is associated with preterm and interm birth.   3. explore the epigenetic dysregulation based on the eRNA captured in the single cell RNA-Seq.  We have established method to uncover eRNA from single cell RNA-Seq, we want to see whether the enhancer profiles predicted from eRNAs of preterm birth differs from the in term birth. \",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"],\n [\"Zhang, Zhongyang \",\n\"ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI\",\n\"Association study on placenta cell composition and gestational age\",\n\"Sep03, 2020\",\n\"approved\",\n\"Preterm birth (PTB) is the birth of a baby before 37 weeks of pregnancy. In 2018, preterm birth affected 1 of every 10 infants born in the United States, and the preterm birth rate rose for the fourth straight year. Additionally, racial and ethnic differences in preterm birth rates remain. The placenta is a complex organ relevant to PTB. However, little is known about the cell composition of placenta and pathologic process underlying PTB. We will use RNAseq data to investigate the cell composition of placenta and how it is relevant to PTB at the cellular and molecular level. This study has great public health significance especially in addressing the reproductive health disparities in the U.S. \",\n\"Preterm birth (PTB) is when a baby is born too early, before 37 weeks of pregnancy have been completed. In 2018, preterm birth affected 1 of every 10 infants born in the United States. The preterm birth rate rose for the fourth straight year in 2018. Additionally, racial and ethnic differences in preterm birth rates remain. For example, in 2018, the rate of preterm birth among African-American women (14%) was about 50 percent higher than the rate of preterm birth among white women (9%).  The cause of PTB molecular underpinnings of human parturition in gestational tissues, and in particular the placenta, are still poorly understood. The placenta is a complex heterogeneous organ consisting of both maternal and fetal cells. Little is known about the cell type composition and transcriptome of placenta and pathologic process underlying PTB at the cellular and molecular level. We will apply for NIH dbGaP single cell RNAseq data (accession number: phs001886) and plan to integrate it with bulk RNAseq data of placenta collected from term birth and from birth of short gestational age (SGA). We will deconvolve the bulk RNAseq data and infer their cell composition, and further associate them with SGA status and factors important in reproductive health. Potentially this proposed study will fill in the knowledge gap of PTB etiology at the cellular and molecular level. This study has great public health significance especially in addressing the reproductive health disparities in the U.S.\",\n\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\"]],\n        columns: [[\"string\", \"Requestor\"], [\"string\", \"Affiliation\"], [\"string\", \"Project\"], [\"string\", \"Date of approval\"], [\"string\", \"Request status\"], [\"string\", \"Public Research Use Statement\"], [\"string\", \"Technical Research Use Statement\"], [\"string\", \"accession\"], [\"string\", \"name\"]],\n        columnOptions: [],\n        rowsPerPage: 10,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n    ",
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Requestor</th>\n",
       "      <th>Affiliation</th>\n",
       "      <th>Project</th>\n",
       "      <th>Date of approval</th>\n",
       "      <th>Request status</th>\n",
       "      <th>Public Research Use Statement</th>\n",
       "      <th>Technical Research Use Statement</th>\n",
       "      <th>accession</th>\n",
       "      <th>name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Aday, Aaron</td>\n",
       "      <td>VANDERBILT UNIVERSITY MEDICAL CENTER</td>\n",
       "      <td>Peripheral Artery Disease and Major Depressive...</td>\n",
       "      <td>Mar03, 2023</td>\n",
       "      <td>approved</td>\n",
       "      <td>People with depression are at higher risk of d...</td>\n",
       "      <td>Overview Major depressive disorder (MDD) is ph...</td>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Adebamowo, Sally</td>\n",
       "      <td>UNIVERSITY OF MARYLAND BALTIMORE</td>\n",
       "      <td>NIH PRIMED Consortium Coordinated Application</td>\n",
       "      <td>Dec14, 2022</td>\n",
       "      <td>approved</td>\n",
       "      <td>Polygenic risk scores (PRS), are a genetic est...</td>\n",
       "      <td>The Polygenic Risk Methods in Diverse Populati...</td>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>AGRAWAL, ARPANA</td>\n",
       "      <td>WASHINGTON UNIVERSITY</td>\n",
       "      <td>Neurobiological bases of psychiatric traits</td>\n",
       "      <td>Apr08, 2020</td>\n",
       "      <td>approved</td>\n",
       "      <td>Genetic variation contributes to health outcom...</td>\n",
       "      <td>Genetic variation contributes to variability i...</td>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Almasy, Laura</td>\n",
       "      <td>CHILDREN'S HOSP OF PHILADELPHIA</td>\n",
       "      <td>Polygenic effects on complex traits across dev...</td>\n",
       "      <td>Jun22, 2022</td>\n",
       "      <td>approved</td>\n",
       "      <td>Genetic markers are an important source of var...</td>\n",
       "      <td>Summary statistics from genome-wide associatio...</td>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Andreassen, Ole</td>\n",
       "      <td>UNIVERSITY OF OSLO</td>\n",
       "      <td>TOP study</td>\n",
       "      <td>Apr21, 2020</td>\n",
       "      <td>approved</td>\n",
       "      <td>Severe mental disorders such as schizophrenia,...</td>\n",
       "      <td>The overall research objective is to gain more...</td>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>375</th>\n",
       "      <td>Vlachos, Ioannis</td>\n",
       "      <td>BETH ISRAEL DEACONESS MEDICAL CENTER</td>\n",
       "      <td>Effect of genetic, genomic, and microbiomic va...</td>\n",
       "      <td>Mar20, 2023</td>\n",
       "      <td>approved</td>\n",
       "      <td>Information is often lost in extensive cohort ...</td>\n",
       "      <td>In this project we aim to identify the effects...</td>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>376</th>\n",
       "      <td>Wang, Kevin</td>\n",
       "      <td>STANFORD UNIVERSITY</td>\n",
       "      <td>long noncoding RNA (lncRNA) regulation of huma...</td>\n",
       "      <td>Feb01, 2022</td>\n",
       "      <td>approved</td>\n",
       "      <td>The mechanisms that trigger preterm labor have...</td>\n",
       "      <td>We are interested in analyzing preterm labor (...</td>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>377</th>\n",
       "      <td>Worthey, Elizabeth</td>\n",
       "      <td>UNIVERSITY OF ALABAMA AT BIRMINGHAM</td>\n",
       "      <td>Understanding the molecular underpinnings of c...</td>\n",
       "      <td>Mar20, 2023</td>\n",
       "      <td>approved</td>\n",
       "      <td>Chorangiomas are benign placental capillary le...</td>\n",
       "      <td>The main objective of the research proposed is...</td>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>378</th>\n",
       "      <td>Zhang, Liye</td>\n",
       "      <td>SHANGHAITECH UNIVERSITY</td>\n",
       "      <td>Explore human placenta in term and preterm par...</td>\n",
       "      <td>Apr16, 2021</td>\n",
       "      <td>closed</td>\n",
       "      <td>Parturition is essential for the reproductive ...</td>\n",
       "      <td>Research Objectives: Explore the mechanism of ...</td>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>379</th>\n",
       "      <td>Zhang, Zhongyang</td>\n",
       "      <td>ICAHN SCHOOL OF MEDICINE AT MOUNT SINAI</td>\n",
       "      <td>Association study on placenta cell composition...</td>\n",
       "      <td>Sep03, 2020</td>\n",
       "      <td>approved</td>\n",
       "      <td>Preterm birth (PTB) is the birth of a baby bef...</td>\n",
       "      <td>Preterm birth (PTB) is when a baby is born too...</td>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>380 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "<google.colab.data_table.DataTable object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "#@title Detailed table of data access requests\n",
    "print(\"Number of data access requests :\", requests.shape[0])\n",
    "\n",
    "if requests.shape[0] > 0:\n",
    "    print(\"Number of unique requestors    :\", len(requests[\"Requestor\"].unique()))\n",
    "    print(\"Number of unique studies       :\", len(requests[\"accession\"].unique()))\n",
    "    display(data_table.DataTable(requests, include_index=False, num_rows_per_page=10))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "Jec8bu9uKIQf",
   "metadata": {
    "cellView": "form",
    "colab": {
     "base_uri": "https://localhost:8080/",
     "height": 662
    },
    "id": "Jec8bu9uKIQf",
    "outputId": "cd42d1b9-00c0-491d-b662-36b7a58a7f77"
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of publications: 59\n"
     ]
    },
    {
     "data": {
      "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/99dac6621f6ae8c4/data_table.js\";\n\n      window.createDataTable({\n        data: [[\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC7614108\",\n\"36601961\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9515437\",\n\"36174398\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9256707\",\n\"35790736\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9233006\",\n\"35449297\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9853312\",\n\"35422469\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8917986\",\n\"34865855\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8323712\",\n\"34139859\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8159880\",\n\"33853351\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8064427\",\n\"33629108\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC7485556\",\n\"32451486\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC6919570\",\n\"31151762\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC6365102\",\n\"30578418\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC7614108\",\n\"PMCPMC7614108\",\n\"CTX\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC10115959\",\n\"37091807\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC10065974\",\n\"36989840\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9894917\",\n\"36741942\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9715671\",\n\"36456565\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9729131\",\n\"36216889\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9496309\",\n\"35929454\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9419655\",\n\"35915156\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9282660\",\n\"35834561\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC10024253\",\n\"35654975\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9132265\",\n\"35613103\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9119501\",\n\"35587468\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8928056\",\n\"35267208\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9210390\",\n\"35241783\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8847690\",\n\"35186008\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8714681\",\n\"34844060\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8505959\",\n\"34650599\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8545250\",\n\"34496635\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC9328121\",\n\"34488071\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8214324\",\n\"34235496\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8429238\",\n\"34120358\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8645657\",\n\"34092032\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC8404304\",\n\"34045744\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC7972521\",\n\"33510476\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC7580856\",\n\"32981348\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC7785109\",\n\"32847391\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC7343592\",\n\"32541925\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC6858581\",\n\"31676865\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC6953633\",\n\"31358989\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC6768096\",\n\"31285632\",\n\"MED\"],\n [\"phs001672.v10.p1\",\n\"Veterans Administration (VA) Million Veteran Program (MVP) Summary    Results from Omics Studies\",\n\"MVP is an ongoing prospective cohort study and mega-biobank in the Department of Veterans Affairs Healthcare System designed to study genetic influences on health and disease among veterans. Eligible subjects  must be active users of the Veteran Health Administration (with available electronic health record) and willing to provide informed consent\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-MDS --- Health/medical/biomedical (mds)\",\n\"phs001672\",\n\"PMC7611456\",\n\"PMCPMC7611456\",\n\"CTX\"],\n [\"phs002300.v1.p1\",\n\"Successful Clinical Response in Pneumonia Therapy (SCRIPT)\",\n\"The goal of this study is to iteratively identify, validate and refine biomarkers to predict clinical failure, and identify novel targets for therapy, beyond traditional antibiotics, for patients with severe  pneumonia requiring intubation, including severe community-acquired pneumonia (CAP), hospital-acquired pneumonia, and ventilator-associated pneumonia\",\n\"Prospective Longitudinal Cohort\",\n\"HMB-IRB --- Health/medical/biomedical (irb)\",\n\"phs002300\",\n\"PMC7987233\",\n\"33429418\",\n\"MED\"],\n [\"phs002315.v1.p1\",\n\"Integrated Analysis of Multimodal Single-Cell Data\",\n\"The simultaneous measurement of multiple modalities, known as multimodal analysis, represents an exciting frontier for single cell genomics and necessitates new computational methods that can define cellular states based on   subpopulations. Moreover, we demonstrate how to leverage this reference to rapidly map new datasets, and to interpret immune responses to vaccination and COVID-19\",\n\"Case Set\",\n\"GRU --- General research use\",\n\"phs002315\",\n\"PMC10141847\",\n\"37117846\",\n\"MED\"],\n [\"phs002315.v1.p1\",\n\"Integrated Analysis of Multimodal Single-Cell Data\",\n\"The simultaneous measurement of multiple modalities, known as multimodal analysis, represents an exciting frontier for single cell genomics and necessitates new computational methods that can define cellular states based on   subpopulations. Moreover, we demonstrate how to leverage this reference to rapidly map new datasets, and to interpret immune responses to vaccination and COVID-19\",\n\"Case Set\",\n\"GRU --- General research use\",\n\"phs002315\",\n\"PMC8238499\",\n\"34062119\",\n\"MED\"],\n [\"phs002315.v1.p1\",\n\"Integrated Analysis of Multimodal Single-Cell Data\",\n\"The simultaneous measurement of multiple modalities, known as multimodal analysis, represents an exciting frontier for single cell genomics and necessitates new computational methods that can define cellular states based on   subpopulations. Moreover, we demonstrate how to leverage this reference to rapidly map new datasets, and to interpret immune responses to vaccination and COVID-19\",\n\"Case Set\",\n\"GRU --- General research use\",\n\"phs002315\",\nNaN,\n\"PPR629309\",\n\"PPR\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC10011423\",\n\"36914823\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC9245422\",\n\"35784314\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC8688323\",\n\"34651399\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC6949028\",\n\"31829938\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC9946687\",\n\"36599348\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC8976826\",\n\"35304419\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC8983148\",\n\"35260533\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC8582682\",\n\"34770059\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC8020997\",\n\"33821263\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC7465902\",\n\"32722449\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\n\"PMC7367681\",\n\"32662421\",\n\"MED\"],\n [\"phs001886.v4.p1\",\n\"Single Cell Analysis of Human Parturition: The Myometrium\",\n\"Parturition is a well-orchestrated process involving multiple tissues: myometrial contractility, cervical ripening, and activation of the chorioamniotic membranes. The transition from a quiescent to a contractile myometrium heralds the onset   range of pregnancy physiologic and pathologic conditions, including the processes of labor (term labor and preterm labor), and COVID-19 (version #3). In\",\n\"Case-Control\",\n\"GRU-NPU --- General research use (npu)\",\n\"phs001886\",\nNaN,\n\"PPR164481\",\n\"PPR\"]],\n        columns: [[\"string\", \"accession\"], [\"string\", \"name\"], [\"string\", \"description\"], [\"string\", \"Study Design\"], [\"string\", \"Study Consent\"], [\"string\", \"dbgap\"], [\"string\", \"PMCID\"], [\"string\", \"EXTID\"], [\"string\", \"SOURCE\"]],\n        columnOptions: [],\n        rowsPerPage: 10,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n    ",
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>accession</th>\n",
       "      <th>name</th>\n",
       "      <th>description</th>\n",
       "      <th>Study Design</th>\n",
       "      <th>Study Consent</th>\n",
       "      <th>dbgap</th>\n",
       "      <th>PMCID</th>\n",
       "      <th>EXTID</th>\n",
       "      <th>SOURCE</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC7614108</td>\n",
       "      <td>36601961</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9515437</td>\n",
       "      <td>36174398</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9256707</td>\n",
       "      <td>35790736</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9233006</td>\n",
       "      <td>35449297</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9853312</td>\n",
       "      <td>35422469</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8917986</td>\n",
       "      <td>34865855</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8323712</td>\n",
       "      <td>34139859</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8159880</td>\n",
       "      <td>33853351</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8064427</td>\n",
       "      <td>33629108</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC7485556</td>\n",
       "      <td>32451486</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC6919570</td>\n",
       "      <td>31151762</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC6365102</td>\n",
       "      <td>30578418</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC7614108</td>\n",
       "      <td>PMCPMC7614108</td>\n",
       "      <td>CTX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC10115959</td>\n",
       "      <td>37091807</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC10065974</td>\n",
       "      <td>36989840</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9894917</td>\n",
       "      <td>36741942</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9715671</td>\n",
       "      <td>36456565</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9729131</td>\n",
       "      <td>36216889</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9496309</td>\n",
       "      <td>35929454</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9419655</td>\n",
       "      <td>35915156</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9282660</td>\n",
       "      <td>35834561</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC10024253</td>\n",
       "      <td>35654975</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9132265</td>\n",
       "      <td>35613103</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9119501</td>\n",
       "      <td>35587468</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8928056</td>\n",
       "      <td>35267208</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9210390</td>\n",
       "      <td>35241783</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8847690</td>\n",
       "      <td>35186008</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8714681</td>\n",
       "      <td>34844060</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8505959</td>\n",
       "      <td>34650599</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8545250</td>\n",
       "      <td>34496635</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC9328121</td>\n",
       "      <td>34488071</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8214324</td>\n",
       "      <td>34235496</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8429238</td>\n",
       "      <td>34120358</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8645657</td>\n",
       "      <td>34092032</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC8404304</td>\n",
       "      <td>34045744</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC7972521</td>\n",
       "      <td>33510476</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC7580856</td>\n",
       "      <td>32981348</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC7785109</td>\n",
       "      <td>32847391</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC7343592</td>\n",
       "      <td>32541925</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC6858581</td>\n",
       "      <td>31676865</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC6953633</td>\n",
       "      <td>31358989</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC6768096</td>\n",
       "      <td>31285632</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>phs001672.v10.p1</td>\n",
       "      <td>Veterans Administration (VA) Million Veteran P...</td>\n",
       "      <td>MVP is an ongoing prospective cohort study and...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-MDS --- Health/medical/biomedical (mds)</td>\n",
       "      <td>phs001672</td>\n",
       "      <td>PMC7611456</td>\n",
       "      <td>PMCPMC7611456</td>\n",
       "      <td>CTX</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>phs002300.v1.p1</td>\n",
       "      <td>Successful Clinical Response in Pneumonia Ther...</td>\n",
       "      <td>The goal of this study is to iteratively ident...</td>\n",
       "      <td>Prospective Longitudinal Cohort</td>\n",
       "      <td>HMB-IRB --- Health/medical/biomedical (irb)</td>\n",
       "      <td>phs002300</td>\n",
       "      <td>PMC7987233</td>\n",
       "      <td>33429418</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>phs002315.v1.p1</td>\n",
       "      <td>Integrated Analysis of Multimodal Single-Cell ...</td>\n",
       "      <td>The simultaneous measurement of multiple modal...</td>\n",
       "      <td>Case Set</td>\n",
       "      <td>GRU --- General research use</td>\n",
       "      <td>phs002315</td>\n",
       "      <td>PMC10141847</td>\n",
       "      <td>37117846</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>phs002315.v1.p1</td>\n",
       "      <td>Integrated Analysis of Multimodal Single-Cell ...</td>\n",
       "      <td>The simultaneous measurement of multiple modal...</td>\n",
       "      <td>Case Set</td>\n",
       "      <td>GRU --- General research use</td>\n",
       "      <td>phs002315</td>\n",
       "      <td>PMC8238499</td>\n",
       "      <td>34062119</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>phs002315.v1.p1</td>\n",
       "      <td>Integrated Analysis of Multimodal Single-Cell ...</td>\n",
       "      <td>The simultaneous measurement of multiple modal...</td>\n",
       "      <td>Case Set</td>\n",
       "      <td>GRU --- General research use</td>\n",
       "      <td>phs002315</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PPR629309</td>\n",
       "      <td>PPR</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC10011423</td>\n",
       "      <td>36914823</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC9245422</td>\n",
       "      <td>35784314</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC8688323</td>\n",
       "      <td>34651399</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC6949028</td>\n",
       "      <td>31829938</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC9946687</td>\n",
       "      <td>36599348</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC8976826</td>\n",
       "      <td>35304419</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC8983148</td>\n",
       "      <td>35260533</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC8582682</td>\n",
       "      <td>34770059</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC8020997</td>\n",
       "      <td>33821263</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC7465902</td>\n",
       "      <td>32722449</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>PMC7367681</td>\n",
       "      <td>32662421</td>\n",
       "      <td>MED</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>phs001886.v4.p1</td>\n",
       "      <td>Single Cell Analysis of Human Parturition: The...</td>\n",
       "      <td>Parturition is a well-orchestrated process inv...</td>\n",
       "      <td>Case-Control</td>\n",
       "      <td>GRU-NPU --- General research use (npu)</td>\n",
       "      <td>phs001886</td>\n",
       "      <td>NaN</td>\n",
       "      <td>PPR164481</td>\n",
       "      <td>PPR</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "<google.colab.data_table.DataTable object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "#@title Publications that cite or mention dbGaP accession numbers\n",
    "studies[\"dbgap\"] = studies[\"accession\"].apply(lambda s: s.split(\".\")[0])\n",
    "# get list of publications from Europe PMC\n",
    "dbgap_pub = pd.read_csv(\"ftp://ftp.ebi.ac.uk/pub/databases/pmc/TextMinedTerms/dbgap.csv\")\n",
    "pubs = studies.merge(dbgap_pub, on=\"dbgap\")\n",
    "print(\"Number of publications:\", pubs.shape[0])\n",
    "display(data_table.DataTable(pubs, include_index=False, num_rows_per_page=10))"
   ]
  }
 ],
 "metadata": {
  "colab": {
   "provenance": []
  },
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
